<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.15</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../">The Antenna</a></li>
	<li><a href="../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../parks.html" title="new about parks">Parks</a></li>
	<li><a href="../../tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="../../libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="./">Archives</a></li>
	<li><a href="../search.html">Search</a></li>
	<li><a href="../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.15</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.15">health 2024.15</h1>
<p>(date: 2024-04-09 08:39:10)</p>
<hr />
<h2
id="causal-associations-between-human-gut-microbiota-and-osteomyelitis-a-mendelian-randomization-study">Causal
associations between human gut microbiota and osteomyelitis: a Mendelian
randomization study</h2>
<p>date: 2024-04-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundRecent studies have emphasized the role of gut microbiota
in the onset and progression of osteomyelitis. However, the exact types
of gut microbiota and their mechanisms of action remain unclear.
Additionally, there is a lack of theoretical support for treatments that
improve osteomyelitis by altering the gut microbiota.MethodsIn our
study, we utilized the largest genome-wide association study (GWAS)
meta-analysis to date from the MiBioGen consortium, involving 13,400
participants. The GWAS data for osteomyelitis were sourced from the UK
Biobank, which included 4,836 osteomyelitis cases and 486,484 controls.
We employed a two-sample Mendelian randomization framework for a
detailed investigation into the causal relationship between gut
microbiota and osteomyelitis. Our methods included inverse variance
weighting, MR-Egger, weighted median, and weighted mode approaches.
Additionally, we applied Cochran’s Q statistic to assess the
heterogeneity of the instrumental variable.ResultsAt the class level,
Bacilli and Bacteroidia were positively correlated with the risk of
osteomyelitis. At the order level, only Bacteroidales showed a positive
association with osteomyelitis. At the genus level, an increased
abundance of Butyricimonas, Coprococcus3, and Tyzzerella3 was positively
associated with the risk of osteomyelitis, whereas Lachnospira was
negatively associated. Sensitivity analyses showed no evidence of
heterogeneity or pleiotropy.ConclusionThis study reveals that classes
Bacilli and Bacteroidia, order Bacteroidales, and genera Butyricimonas,
Coprococcus3, and Tyzzerella3 are implicated in increasing the risk of
osteomyelitis, while the genus Lachnospira is associated with a reduced
risk. Future investigations are warranted to elucidate the precise
mechanisms through which these specific bacterial groups influence the
pathophysiology of osteomyelitis</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1338989"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1338989</a></p>
<hr />
<h2
id="endometrial-microbiota-alteration-in-female-patients-with-endometrial-polyps-based-on-16s-rrna-gene-sequencing-analysis">Endometrial
microbiota alteration in female patients with endometrial polyps based
on 16S rRNA gene sequencing analysis</h2>
<p>date: 2024-04-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe potential role of the endometrial microbiota in the
pathogenesis of endometrial polyps (EPs) warrants further investigation,
given the current landscape of limited and inconclusive research
findings. We aimed to explore the microecological characteristics of the
uterine cavity in patients with EPs and investigate the potential of
endometrial microbiota species as novel biomarkers for identifying
EPs.MethodsEndometrial samples were collected from 225 patients who
underwent hysteroscopies, of whom 167 had EPs, whereas 58 had non-
hyperproliferative endometrium status. The endometrial microbiota was
assessed using 16S rRNA gene sequencing. We characterized the
endometrial microbiota and identified microbial biomarkers for
predicting EPs.ResultsThe endometrial microbial diversity and
composition were significantly different between the EP and control
groups. Predictive functional analyses of the endometrial microbiota
demonstrated significant alterations in pathways involved in
sphingolipid metabolism, steroid hormone biosynthesis, and apoptosis
between the two groups. Moreover, a classification model based on
endometrial microbial ASV-based biomarkers along with the presence of
abnormal uterine bleeding symptoms achieved powerful classification
potential in identifying EPs in both the discovery and validation
cohorts.ConclusionOur study indicates a potential association between
altered endometrial microbiota and EPs. Endometrial microbiota-based
biomarkers may prove valuable for the diagnosis of EPs.Clinical trial
registrationChinese Clinical Trial Registry (ChiCTR2100052746).</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351329"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351329</a></p>
<hr />
<h2
id="beyond-pathogens-the-intriguing-genetic-legacy-of-endogenous-retroviruses-in-host-physiology">Beyond
pathogens: the intriguing genetic legacy of endogenous retroviruses in
host physiology</h2>
<p>date: 2024-04-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The notion that viruses played a crucial role in the evolution of
life is not a new concept. However, more recent insights suggest that
this perception might be even more expansive, highlighting the ongoing
impact of viruses on host evolution. Endogenous retroviruses (ERVs) are
considered genomic remnants of ancient viral infections acquired
throughout vertebrate evolution. Their exogenous counterparts once
infected the host’s germline cells, eventually leading to the permanent
endogenization of their respective proviruses. The success of ERV
colonization is evident so that it constitutes 8% of the human genome.
Emerging genomic studies indicate that endogenous retroviruses are not
merely remnants of past infections but rather play a corollary role,
despite not fully understood, in host genetic regulation. This review
presents some evidence supporting the crucial role of endogenous
retroviruses in regulating host genetics. We explore the involvement of
human ERVs (HERVs) in key physiological processes, from their precise
and orchestrated activities during cellular differentiation and
pluripotency to their contributions to aging and cellular senescence.
Additionally, we discuss the costs associated with hosting a substantial
amount of preserved viral genetic material.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1379962"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1379962</a></p>
<hr />
<h2
id="molecular-mechanisms-of-tigecycline-resistance-among-enterobacterales">Molecular
mechanisms of tigecycline-resistance among Enterobacterales</h2>
<p>date: 2024-04-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The global emergence of antimicrobial resistance to multiple
antibiotics has recently become a significant concern. Gram-negative
bacteria, known for their ability to acquire mobile genetic elements
such as plasmids, represent one of the most hazardous microorganisms.
This phenomenon poses a serious threat to public health. Notably, the
significance of tigecycline, a member of the antibiotic group
glycylcyclines and derivative of tetracyclines has increased.
Tigecycline is one of the last-resort antimicrobial drugs used to treat
complicated infections caused by multidrug-resistant (MDR) bacteria,
extensively drug-resistant (XDR) bacteria or even pan-drug-resistant
(PDR) bacteria. The primary mechanisms of tigecycline resistance include
efflux pumps’ overexpression, tet genes and outer membrane porins.
Efflux pumps are crucial in conferring multi-drug resistance by
expelling antibiotics (such as tigecycline by direct expelling) and
decreasing their concentration to sub-toxic levels. This review
discusses the problem of tigecycline resistance, and provides important
information for understanding the existing molecular mechanisms of
tigecycline resistance in Enterobacterales. The emergence and spread of
pathogens resistant to last-resort therapeutic options stands as a major
global healthcare concern, especially when microorganisms are already
resistant to carbapenems and/or colistin.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289396"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289396</a></p>
<hr />
<h2
id="deep-mutational-scanning-of-the-rnase-iii-like-domain-in-trypanosoma-brucei-rna-editing-protein-krepb4">Deep
mutational scanning of the RNase III-like domain in Trypanosoma brucei
RNA editing protein KREPB4</h2>
<p>date: 2024-04-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Kinetoplastid pathogens including Trypanosoma brucei, T. cruzi, and
Leishmania species, are early diverged, eukaryotic, unicellular
parasites. Functional understanding of many proteins from these
pathogens has been hampered by limited sequence homology to proteins
from other model organisms. Here we describe the development of a
high-throughput deep mutational scanning approach in T. brucei that
facilitates rapid and unbiased assessment of the impacts of many
possible amino acid substitutions within a protein on cell fitness, as
measured by relative cell growth. The approach leverages several
molecular technologies: cells with conditional expression of a wild-type
gene of interest and constitutive expression of a library of mutant
variants, degron-controlled stabilization of I-SceI meganuclease to
mediate highly efficient transfection of a mutant allele library, and a
high-throughput sequencing readout for cell growth upon conditional
knockdown of wild-type gene expression and exclusive expression of
mutant variants. Using this method, we queried the effects of amino acid
substitutions in the apparently non-catalytic RNase III-like domain of
KREPB4 (B4), which is an essential component of the RNA Editing
Catalytic Complexes (RECCs) that carry out mitochondrial RNA editing in
T. brucei. We measured the impacts of thousands of B4 variants on
bloodstream form cell growth and validated the most deleterious variants
containing single amino acid substitutions. Crucially, there was no
correlation between phenotypes and amino acid conservation,
demonstrating the greater power of this method over traditional sequence
homology searching to identify functional residues. The bloodstream form
cell growth phenotypes were combined with structural modeling, RECC
protein proximity data, and analysis of selected substitutions in
procyclic form T. brucei. These analyses revealed that the B4
RNaseIII-like domain is essential for maintenance of RECC integrity and
RECC protein abundances and is also involved in changes in RECCs that
occur between bloodstream and procyclic form life cycle stages.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1381155"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1381155</a></p>
<hr />
<h2
id="research-progress-on-the-microbiota-in-bladder-cancer-tumors">Research
progress on the microbiota in bladder cancer tumors</h2>
<p>date: 2024-04-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The microbiota, also referred to as the microbial community, is a
crucial component of the human microenvironment. It is located
predominantly in various organs, including the intestines, skin, oral
cavity, respiratory tract, and reproductive tract. The microbiota
maintains a symbiotic relationship with the human body, influencing
physiological and pathological functions to a significant degree. There
is increasing evidence linking the microbial flora to human cancers. In
contrast to the traditional belief that the urethra and urine of normal
individuals are sterile, recent advancements in high-throughput
sequencing technology and bacterial cultivation methods have led to the
discovery of specific microbial communities in the urethras of healthy
individuals. Given the prevalence of bladder cancer (BCa) as a common
malignancy of the urinary system, researchers have shifted their focus
to exploring the connection between disease development and the unique
microbial community within tumors. This shift has led to a deeper
investigation into the role of microbiota in the onset, progression,
metastasis, prognosis, and potential for early detection of BCa. This
article reviews the existing research on the microbiota within BCa
tumors and summarizes the findings regarding the roles of different
microbes in various aspects of this disease.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1374944"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1374944</a></p>
<hr />
<h2
id="isolation-of-mucosa-associated-microbiota-dysbiosis-in-the-ascending-colon-in-hepatitis-c-virus-post-sustained-virologic-response-cirrhotic-patients">Isolation
of mucosa-associated microbiota dysbiosis in the ascending colon in
hepatitis C virus post-sustained virologic response cirrhotic
patients</h2>
<p>date: 2024-04-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAchieving sustained virologic response (SVR) in patients
infected with hepatitis C virus (HCV) reduces all-cause mortality.
However, the mechanisms and risk factors for liver fibrosis and portal
hypertension post-SVR remain incompletely understood. In the gut-liver
axis, mucosa-associated microbiota (MAM) substantially influence immune
and metabolic functions, displaying spatial heterogeneity at the
anatomical intestinal site. We analyzed MAM composition and function to
isolate the locoregional MAM involved in chronic liver disease
progression in HCV post-SVR patients.MethodsWe collected MAM samples
from three intestinal sites (terminal ileum, ascending colon, and
sigmoid colon) via brushing during colonoscopy in 23 HCV post-SVR
patients and 25 individuals without liver disease (controls). The 16S
rRNA of bacterial DNA in specimens collected with a brush and in feces
was sequenced. The molecular expression of intestinal tissues and
hepatic tissues were evaluated by quantitative real-time PCR.ResultsIn
the post-SVR group, the microbial β-diversity of MAM, especially in the
ascending colon, differed from the control group and was associated with
liver fibrosis progression. In PICRUSt analysis, MAM in the ascending
colon in the liver cirrhosis (LC) group showed compromised functions
associated with the intestinal barrier and bile acid production, and
FGF19 expression was markedly decreased in the terminal ileum biopsy
tissue in the LC group. At the genus level, six short-chain fatty acid
(SCFA)-producing bacterial genera, Blautia, Alistipes, Roseburia,
Agathobaculum, Dorea, and Pseudoflavonifractor were reduced in the
ascending colon of post-SVR LC patients.ConclusionIn patients of HCV
post-SVR, we identified the association between the degree of liver
fibrosis and dysbiosis of mucosa-associated SCFA-producing bacterial
genera that may be related to intestinal barrier and bile acid
production in the ascending colon.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1371429"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1371429</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
