---
title: health 2024.08
updated: 2024-02-22 08:50:39
---

# health 2024.08

(date: 2024-02-22 08:50:39)

---

## Oral infectivity through carnivorism in murine model of Trypanosoma cruzi infection

date: 2024-02-22, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>Oral transmission of <italic>T. cruzi</italic> is probably the most frequent transmission mechanism in wild animals. This observation led to the hypothesis that consuming raw or undercooked meat from animals infected with <italic>T. cruzi</italic> may be responsible for transmitting the infection. Therefore, the general objective of this study was to investigate host-pathogen interactions between the parasite and gastric mucosa and the role of meat consumption from infected animals in the oral transmission of <italic>T. cruzi.</italic></p></sec><sec><title>Methods</title><p>Cell infectivity assays were performed on AGS cells in the presence or absence of mucin, and the roles of pepsin and acidic pH were determined. Moreover, groups of five female Balb/c mice were fed with muscle tissue obtained from mice in the acute phase of infection by the clone H510 C8C3<bold><italic>hvir</italic></bold> of <italic>T. cruzi</italic>, and the infection of the fed mice was monitored by a parasitemia curve. Similarly, we assessed the infective capacity of <italic>T. cruzi</italic> trypomastigotes and amastigotes by infecting groups of five mice Balb/c females, which were infected orally using a nasogastric probe, and the infection was monitored by a parasitemia curve. Finally, different trypomastigote and amastigote inoculums were used to determine their infective capacities. Adhesion assays of <italic>T. cruzi</italic> proteins to AGS stomach cells were performed, and the adhered proteins were detected by western blotting using monoclonal or polyclonal antibodies and by LC-MS/MS and bioinformatics analysis. </p></sec><sec><title>Results</title><p>Trypomastigote migration in the presence of mucin was reduced by approximately 30%, whereas in the presence of mucin and pepsin at pH 3.5, only a small proportion of parasites were able to migrate (∼6%). Similarly, the ability of TCTs to infect AGS cells in the presence of mucin is reduced by approximately 20%. In all cases, 60–100% of the animals were fed meat from mice infected in the acute phase or infected with trypomastigotes or amastigotes developed high parasitemia, and 80% died around day 40 post-infection. The adhesion assay showed that cruzipain is a molecule of trypomastigotes and amastigotes that binds to AGS cells. LC-MS/MS and bioinformatics analysis, also confirmed that transialidase, cysteine proteinases, and gp63 may be involved in TCTs attachment or invasion of human stomach cells because they can potentially interact with different proteins in the human stomach mucosa. In addition, several human gastric mucins have cysteine protease cleavage sites. </p></sec><sec><title>Discussion</title><p>Then, under our experimental conditions, consuming meat from infected animals in the acute phase allows the <italic>T. cruzi</italic> infection. Similarly, trypomastigotes and amastigotes could infect mice when administered orally, whereas cysteinyl proteinases and trans-sialidase appear to be relevant molecules in this infective process. </p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297099">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297099</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297099">Save to Pocket</a>
</span>

---

## Oral streptococci: modulators of health and disease

date: 2024-02-22, from: Frontiers in Cellular and Infection Microbiology

<p>Streptococci are primary colonizers of the oral cavity where they are ubiquitously present and an integral part of the commensal oral biofilm microflora. The role oral streptococci play in the interaction with the host is ambivalent. On the one hand, they function as gatekeepers of homeostasis and are a prerequisite for the maintenance of oral health - they shape the oral microbiota, modulate the immune system to enable bacterial survival, and antagonize pathogenic species. On the other hand, also recognized pathogens, such as oral <italic>Streptococcus mutans</italic> and <italic>Streptococcus sobrinus</italic>, which trigger the onset of dental caries belong to the genus <italic>Streptococcus</italic>. In the context of periodontitis, oral streptococci as excellent initial biofilm formers have an accessory function, enabling late biofilm colonizers to inhabit gingival pockets and cause disease. The pathogenic potential of oral streptococci fully unfolds when their dissemination into the bloodstream occurs; streptococcal infection can cause extra-oral diseases, such as infective endocarditis and hemorrhagic stroke. In this review, the taxonomic diversity of oral streptococci, their role and prevalence in the oral cavity and their contribution to oral health and disease will be discussed, focusing on the virulence factors these species employ for interactions at the host interface.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1357631">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1357631</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1357631">Save to Pocket</a>
</span>

---

## Comprehensive characterization of ERV-K (HML-8) in the chimpanzee genome revealed less genomic activity than humans

date: 2024-02-22, from: Frontiers in Cellular and Infection Microbiology

<p>Endogenous retroviruses (ERVs) originate from ancestral germline infections caused by exogenous retroviruses. Throughout evolution, they have become fixed within the genome of the animals into which they were integrated. As ERV elements coevolve with the host, they are normally epigenetically silenced and can become upregulated in a series of physiological and pathological processes. Generally, a detailed ERV profile in the host genome is critical for understanding the evolutionary history and functional performance of the host genome. We previously characterized and cataloged all the ERV-K subtype HML-8 loci in the human genome; however, this has not been done for the chimpanzee, the nearest living relative of humans. In this study, we aimed to catalog and characterize the integration of HML-8 in the chimpanzee genome and compare it with the integration of HML-8 in the human genome. We analyzed the integration of HML-8 and found that HML-8 pervasively invaded the chimpanzee genome. A total of 76 proviral elements were characterized on 23/24 chromosomes, including detailed elements distribution, structure, phylogeny, integration time, and their potential to regulate adjacent genes. The incomplete structure of HML-8 proviral LTRs will undoubtedly affect their activity. Moreover, the results indicated that HML-8 integration occurred before the divergence between humans and chimpanzees. Furthermore, chimpanzees include more HML-8 proviral elements (76 vs. 40) and fewer solo long terminal repeats (LTR) (0 vs. 5) than humans. These results suggested that chimpanzee genome activity is less than the human genome and that humans may have a better ability to shape and screen integrated proviral elements. Our work is informative in both an evolutionary and a functional context for ERVs.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349046">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349046</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349046">Save to Pocket</a>
</span>

---

## Natural products as a source of Coronavirus entry inhibitors

date: 2024-02-21, from: Frontiers in Cellular and Infection Microbiology

<p>The COVID-19 pandemic has had a significant and lasting impact on the world. Four years on, despite the existence of effective vaccines, the continuous emergence of new SARS-CoV-2 variants remains a challenge for long-term immunity. Additionally, there remain few purpose-built antivirals to protect individuals at risk of severe disease in the event of future coronavirus outbreaks. A promising mechanism of action for novel coronavirus antivirals is the inhibition of viral entry. To facilitate entry, the coronavirus spike glycoprotein interacts with angiotensin converting enzyme 2 (ACE2) on respiratory epithelial cells. Blocking this interaction and consequently viral replication may be an effective strategy for treating infection, however further research is needed to better characterize candidate molecules with antiviral activity before progressing to animal studies and clinical trials. In general, antiviral drugs are developed from purely synthetic compounds or synthetic derivatives of natural products such as plant secondary metabolites. While the former is often favored due to the higher specificity afforded by rational drug design, natural products offer several unique advantages that make them worthy of further study including diverse bioactivity and the ability to work synergistically with other drugs. Accordingly, there has recently been a renewed interest in natural product-derived antivirals in the wake of the COVID-19 pandemic. This review provides a summary of recent research into coronavirus entry inhibitors, with a focus on natural compounds derived from plants, honey, and marine sponges.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1353971">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1353971</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1353971">Save to Pocket</a>
</span>

---

## COVID-19 in pulmonary critically ill patients: metagenomic identification of fungi and characterization of pathogenic microorganisms

date: 2024-02-20, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Background</title><p>Fungal co-infection is prevalent in critically ill patients with COVID-19. The conventional approach applied to fungal identification has relatively low sensitivity and is time-consuming. The metagenomic next-generation sequencing (mNGS) technology can simultaneously detect a variety of microorganisms, and is increasingly being used for the rapid detection and diagnosis of pathogens.</p></sec><sec><title>Methods</title><p>In this single-center retrospective study, we described the clinical presentation and outcomes of COVID-19 and mNGS positive for fungi in pulmonary critically ill patients during the outbreak of Omicron infection from December 2022 to January 2023.</p></sec><sec><title>Results</title><p>Among 43 COVID-19 patients with acute respiratory distress syndrome (ARDS) on a single intensive care unit (ICU), 10 were reported to be fungal positive using the mNGS test. The number of pathogenic microorganisms detected by mNGS was significantly higher than that via traditional methods, especially in the detection of fungi and viruses. <italic>Aspergillus</italic> infection was dominant, and most of these patients also had concurrent bacterial or viral infections. Probable or possible COVID-19-associated pulmonary aspergillosis (CAPA) was diagnosed in all 10 patients, and the prognosis was poor.</p></sec><sec><title>Conclusion</title><p>Patients with COVID-19 may be at increased risk of developing fungal infections as well as concurrent bacterial or viral infections, and mNGS can be a powerful tool in identifying these infections. Clinicians should be aware of the increased risk of fungal infections in COVID-19 patients, particularly those who have underlying immunocompromising conditions, and should monitor for early signs of infection.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012">Save to Pocket</a>
</span>

---

## Effects of plant natural products on metabolic-associated fatty liver disease and the underlying mechanisms: a narrative review with a focus on the modulation of the gut microbiota

date: 2024-02-20, from: Frontiers in Cellular and Infection Microbiology

<p>Metabolic-associated fatty liver disease (MAFLD) is a chronic liver disease characterized by the excessive accumulation of fat in hepatocytes. However, due to the complex pathogenesis of MAFLD, there are no officially approved drugs for treatment. Therefore, there is an urgent need to find safe and effective anti-MAFLD drugs. Recently, the relationship between the gut microbiota and MAFLD has been widely recognized, and treating MAFLD by regulating the gut microbiota may be a new therapeutic strategy. Natural products, especially plant natural products, have attracted much attention in the treatment of MAFLD due to their multiple targets and pathways and few side effects. Moreover, the structure and function of the gut microbiota can be influenced by exposure to plant natural products. However, the effects of plant natural products on MAFLD through targeting of the gut microbiota and the underlying mechanisms are poorly understood. Based on the above information and to address the potential therapeutic role of plant natural products in MAFLD, we systematically summarize the effects and mechanisms of action of plant natural products in the prevention and treatment of MAFLD through targeting of the gut microbiota. This narrative review provides feasible ideas for further exploration of safer and more effective natural drugs for the prevention and treatment of MAFLD.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261">Save to Pocket</a>
</span>

---

## Advances in application of CRISPR-Cas13a system

date: 2024-02-19, from: Frontiers in Cellular and Infection Microbiology

<p>Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) and CRISPR-associated (Cas) proteins serve as an adaptive immune system that safeguards prokaryotes and some of the viruses that infect prokaryotes from foreign nucleic acids (such as viruses and plasmids). The genomes of the majority of archaea and about half of all bacteria contain various CRISPR-Cas systems. CRISPR-Cas systems depend on CRISPR RNAs (crRNAs). They act as a navigation system to specifically cut and destroy foreign nucleic acids by recognizing invading foreign nucleic acids and binding Cas proteins. In this review, we provide a brief overview of the evolution and classification of the CRISPR-Cas system, focusing on the functions and applications of the CRISPR-Cas13a system. We describe the CRISPR-Cas13a system and discuss its RNA-directed ribonuclease function. Meanwhile, we briefly introduce the mechanism of action of the CRISPR-Cas13a system and summarize the applications of the CRISPR-Cas13a system in pathogen detection, eukaryotes, agriculture, biosensors, and human gene therapy. We are right understanding of CRISPR-Cas13a has been broadened, and the CRISPR-Cas13a system will be useful for developing new RNA targeting tools. Therefore, understanding the basic details of the structure, function, and biological characterization of CRISPR-Cas13a effector proteins is critical for optimizing RNA targeting tools.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557">Save to Pocket</a>
</span>

---

## Excess fermentation and lactic acidosis as detrimental functions of the gut microbes in treatment-naive TB patients

date: 2024-02-19, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>The link between gut microbiota and host immunity motivated numerous studies of the gut microbiome in tuberculosis (TB) patients. However, these studies did not explore the metabolic capacity of the gut community, which is a key axis of impact on the host’s immunity.</p></sec><sec><title>Methods</title><p>We used deep sequencing of fecal samples from 23 treatment-naive TB patients and 48 healthy donors to reconstruct the gut microbiome’s metabolic capacity and strain/species-level content.</p></sec><sec><title>Results</title><p>We show that the systematic depletion of the commensal flora of the large intestine, <italic>Bacteroidetes</italic>, and an increase in <italic>Actinobacteria, Firmicutes</italic>, and <italic>Proteobacteria</italic> such as <italic>Streptococcaceae, Erysipelotrichaceae, Lachnospiraceae</italic>, and <italic>Enterobacteriaceae</italic> explains the strong taxonomic divergence of the gut community in TB patients. The cumulative expansion of diverse disease-associated pathobionts in patients reached 1/4 of the total gut microbiota, suggesting a heavy toll on host immunity along with MTB infection. Reconstruction of metabolic pathways showed that the microbial community in patients shifted toward rapid growth using glycolysis and excess fermentation to produce acetate and lactate. Higher glucose availability in the intestine likely drives fermentation to lactate and growth, causing acidosis and endotoxemia.</p></sec><sec><title>Discussion</title><p>Excessive fermentation and lactic acidosis likely characterize TB patients’ disturbed gut microbiomes. Since lactic acidosis strongly suppresses the normal gut flora, directly interferes with macrophage function, and is linked to mortality in TB patients, our findings highlight gut lactate acidosis as a novel research focus. If confirmed, gut acidosis may be a novel potential host-directed treatment target to augment traditional TB treatment.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521">Save to Pocket</a>
</span>

---

## Editorial: Precision medicine in viral hepatitis: progress and prospects towards elimination

date: 2024-02-19, from: Frontiers in Cellular and Infection Microbiology



<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423">Save to Pocket</a>
</span>

---

## Identification of potential novel proteomic markers of Leishmania spp.-derived exosomes

date: 2024-02-19, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>Extracellular vesicles (EVs) are heterogenous cell-derived membrane-bound structures which can be subdivided into three distinct classes according to distinct morphological characteristics, cellular origins, and functions. Small EVs, or exosomes, can be produced by the protozoan parasite <italic>Leishmania</italic> through the evolutionarily conserved ESCRT pathway, and act as effectors of virulence and drivers of pathogenesis within mammalian hosts. Techniques for the identification of EVs of non-mammalian origin, however, remain inaccurate in comparison to their well-characterized mammalian counterparts. Thus, we still lack reliable and specific markers for <italic>Leishmania</italic>-derived exosomes, which poses a significant challenge to the field.</p></sec><sec><title>Methods</title><p>Herein, we utilized serial differential ultracentrifugation to separate <italic>Leishmania</italic>-derived EV populations into three distinct fractions. Nanoparticle tracking analysis and transmission electron microscopy were used to validate their morphological characteristics, and bioinformatic analysis of LC-MS/MS proteomics corroborated cellular origins and function.</p></sec><sec><title>Discussion</title><p>Proteomic data indicated potential novel proteic markers of <italic>Leishmania</italic>-derived exosomes, including proteins involved in endosomal machinery and the ESCRT pathway, as well as the parasitic phosphatase PRL-1. Further investigation is required to determine the specificity and sensitivity of these markers.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636">Save to Pocket</a>
</span>



<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>

