<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.04</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../">The Antenna</a></li>
	<li><a href="../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../parks.html" title="new about parks">Parks</a></li>
	<li><a href="../../tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="../../libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="./">Archives</a></li>
	<li><a href="../search.html">Search</a></li>
	<li><a href="../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.04</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.04">health 2024.04</h1>
<p>(date: 2024-01-23 08:23:04)</p>
<hr />
<h2
id="editorial-interrelationships-between-human-microbiology-and-health-of-the-elderly">Editorial:
Interrelationships between human microbiology and health of the
elderly</h2>
<p>date: 2024-01-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345103">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345103</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345103">Save
to Pocket</a> </span></p>
<hr />
<h2
id="comparative-analysis-of-the-intestinal-microbiota-of-blacknecked-cranes-grus-nigricollis-in-different-wintering-areas">Comparative
analysis of the intestinal microbiota of black−necked cranes (Grus
nigricollis) in different wintering areas</h2>
<p>date: 2024-01-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Fecal microbiota is essential for host health because it increases
digestive effectiveness. The crane species <italic>Grus
nigricollis</italic> (<italic>G. nigricollis</italic>) is considered to
be near threatened. The fecal microbial composition of crane is less
understood, particularly in the Tibet, China. This study was performed
to investigate the differences in fecal microbial composition and
diversity of crane in different wintering areas using third-generation
single-molecule real-time sequencing technology in the Tibet, China.
According to the findings, 20 samples were used to generate 936
bacterial amplicon sequence variants (ASVs) and 1,800 fungal ASVs, only
4 bacterial ASVs and 20 fungal ASVs were shared in four distinct
locations. Firmicutes were the dominant bacterial phylum in all samples,
and Ascomycota and Basidiomycota were the dominant fungal phylum. At the
genus level, <italic>Lactobacillus</italic> was the dominant genus in
Linzhi City (LZ), Shannan City (SN), and Lasa City (LS), whereas
Megamonas was the dominant genus in Rikaze City (RKZ).
<italic>Naganishia</italic> and <italic>Mycosphaerella</italic> were the
dominant fungal genera in SN and RKZ. <italic>Mycosphaerella</italic>
and <italic>Tausonia</italic> were the dominant fungal genera in LZ.
<italic>Naganishia</italic> and <italic>Fusarium</italic> were the
dominant fungal genera in LS. And the fecal microbial composition varied
between the four groups, as shown by the underweighted pair-group method
with arithmetic means and principal coordinates analysis. This study
offers a theoretical basis for understanding the fecal microbial
composition of crane.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302785">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302785</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302785">Save
to Pocket</a> </span></p>
<hr />
<h2
id="invasive-streptococcus-suis-isolated-in-spain-contain-a-highly-promiscuous-and-dynamic-resistome">Invasive
Streptococcus suis isolated in Spain contain a highly promiscuous and
dynamic resistome</h2>
<p>date: 2024-01-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Streptococcus suis</italic> is a major pathogen for swine and
human. Here we aimed to know the rates of antimicrobial resistance (AMR)
in invasive <italic>S. suis</italic> isolates recovered along Spain
between 2016 – 2021 and elucidate their genetic origin.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Antibiotic susceptibility testing was performed for 116 isolates of
different genetic backgrounds and geographic origins against 18
antibiotics of 9 families. The association between AMR and genotypes and
the origin of the isolates were statistically analyzed using Pearson´s
chi-square test and the likelihood ratio. The antimicrobial resistant
genes were identified by whole genome sequencing analysis and PCR
screenings.
</p>
</sec><sec>
<title>
Results
</title>
<p>
High AMR rates (&gt;80%) were detected for tetracyclines, spectinomycin,
lincosamides, and marbofloxacin, medium (20-40%) for
sulphonamides/trimethoprim, tiamulin, penicillin G, and enrofloxacin,
and low (&lt; 20%) for florfenicol, and four additional β-lactams. The
occurrence of multidrug resistance was observed in 90% of isolates. For
certain antibiotics (penicillin G, enrofloxacin, marbofloxacin,
tilmicosin, and erythromycin), AMR was significantly associated with
particular sequence types (STs), geographic regions, age of pigs, and
time course. Whole genome sequencing comparisons and PCR screenings
identified 23 AMR genes, of which 19 were previously reported in
<italic>S. suis</italic> (<italic>aph</italic>(3’)-IIIa, <italic>sat4,
aadE, spw</italic>, aac(6’)-Ie-<italic>aph</italic>(2’’)-Ia,
<italic>fexA, optrA, erm</italic>(B), <italic>mef</italic>(A/E),
<italic>mrs</italic>(D), <italic>mph</italic>(C),
<italic>lnu</italic>(B), <italic>lsa</italic>(E),
<italic>vga</italic>(F), <italic>tet</italic>(M),
<italic>tet</italic>(O), <italic>tet</italic>(O/W/32/O),
<italic>tet</italic>(W)), and 4 were novel
(<italic>aph</italic>(2’’)-IIIa, <italic>apmA, erm</italic>(47),
<italic>tet</italic>(T)). These AMR genes explained the AMR to
spectinomycin, macrolides, lincosamides, tiamulin, and tetracyclines.
Several genes were located on mobile genetic elements which showed a
variable organization and composition. As AMR gene homologs were
identified in many human and animal pathogens, the resistome of
<italic>S. suis</italic> has a different phylogenetic origin. Moreover,
AMR to penicillin G, fluoroquinolones, and trimethoprim related to
mutations in genes coding for target enzymes (<italic>pbp1a, pbp2b,
pbp2x, mraY, gyrA</italic>, <italic>parC</italic>, and
<italic>dhfr)</italic>. Bioinformatic analysis estimated traits of
recombination on target genes, also indicative of gene transfer events.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
Our work evidences that <italic>S. suis</italic> is a major contributor
to AMR dissemination across veterinary and human pathogens. Therefore,
control of AMR in <italic>S. suis</italic> should be considered from a
One Health approach in regions with high pig production to properly
tackle the issue of antimicrobial drug resistance.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1329632">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1329632</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1329632">Save
to Pocket</a> </span></p>
<hr />
<h2
id="red-recombination-enables-a-wide-variety-of-markerless-manipulation-of-porcine-epidemic-diarrhea-virus-genome-to-generate-recombinant-virus">Red
recombination enables a wide variety of markerless manipulation of
porcine epidemic diarrhea virus genome to generate recombinant
virus</h2>
<p>date: 2024-01-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Porcine epidemic diarrhea virus (PEDV) is a member of the genera
<italic>Alphacoronavirus</italic> that has been associated with acute
watery diarrhea and vomiting in swine. Unfortunately, no effective
vaccines and antiviral drugs for PEDV are currently available. Reverse
genetics systems are crucial tools for these researches. Here, a PEDV
full-length cDNA clone was constructed. Furtherly, three PEDV reporter
virus plasmids containing red fluorescent protein (RFP), Nano luciferase
(Nluc), or green fluorescence protein (GFP) were generated using Red
recombination with the GS1783 <italic>E. coli</italic> strain. These
reporter-expressing recombinant (r) PEDVs showed similar growth
properties to the rPEDV, and the foreign genes were stable to culture up
to P9 in Vero cells. Using the Nluc-expressing rPEDV, the replication of
PEDV was easily quantified, and a platform for rapid anti-PEDV drug
screening was constructed. Among the three drugs, Bergenin, Umifenovir
hydrochloride (Arbidol), and <italic>Ganoderma lucidum</italic>
triterpenoids (GLTs), we found that GLTs inhibited PEDV replication
mainly after the stage of virus “Entry”. Overall, this study will
broaden insight into the method for manipulating the PEDV genome and
provide a powerful tool for screening anti-PEDV agents.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1338740">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1338740</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1338740">Save
to Pocket</a> </span></p>
<hr />
<h2
id="the-pediatric-gut-bacteriome-and-virome-in-response-to-sars-cov-2-infection">The
pediatric gut bacteriome and virome in response to SARS-CoV-2
infection</h2>
<p>date: 2024-01-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Since the beginning of the SARS-CoV-2 pandemic in early 2020, it has
been apparent that children were partially protected from both infection
and the more severe forms of the disease. Many different mechanisms have
been proposed to explain this phenomenon, including children’s frequent
exposure to other upper respiratory infections and vaccines, and which
inflammatory cytokines they are more likely to produce in response to
infection. Furthermore, given the presence of SARS-CoV-2 in the
intestine and its ability to infect enterocytes, combined with the well
described immunomodulatory capabilities of the microbiome, another
potential contributing factor may be the presence of certain protective
microbial members of the gut microbiota (GM).
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We performed shotgun metagenomic sequencing and profiled both the
bacteriome and virome of the GM of pediatric SARS-CoV-2 patients
compared to healthy, age-matched subjects.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We found that, while pediatric patients do share some pro-inflammatory
microbial signatures with adult patients, they also possess a distinct
microbial signature of protective bacteria previously found to be
negatively correlated with SARS-CoV-2 infectivity and COVID-19 severity.
COVID-19 was also associated with higher fecal
<italic>Cytomegalovirus</italic> load, and with shifts in the relative
abundances of bacteriophages in the GM. Furthermore, we address how the
preventative treatment of COVID-19 patients with antibiotics, a common
practice especially in the early days of the pandemic, affected the
bacteriome and virome, as well as the abundances of antimicrobial
resistance and virulence genes in these patients.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
To our knowledge, this is the first study to address the bacteriome,
virome, and resistome of pediatric patients in response to COVID-19 and
to preventative antibiotics use.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1335450">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1335450</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1335450">Save
to Pocket</a> </span></p>
<hr />
<h2
id="isobavachalcone-exhibits-antifungal-and-antibiofilm-effects-against-c.-albicans-by-disrupting-cell-wallmembrane-integrity-and-inducing-apoptosis-and-autophagy">Isobavachalcone
exhibits antifungal and antibiofilm effects against C. albicans by
disrupting cell wall/membrane integrity and inducing apoptosis and
autophagy</h2>
<p>date: 2024-01-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Isobavachalcone (IBC) is a natural flavonoid with multiple
pharmacological properties. This study aimed to evaluate the efficacy of
IBC against planktonic growth and biofilms of <italic>Candida
albicans</italic> (<italic>C. albicans</italic>) and the mechanisms
underlying its antifungal action. The cell membrane integrity, cell
metabolic viability, and cell morphology of <italic>C. albicans</italic>
treated with IBC were evaluated using CLSM and FESEM analyses. Crystal
violet staining, CLSM, and FESEM were used to assess the inhibition of
biofilm formation, as well as dispersal and killing effects of IBC on
mature biofilms. RNA-seq combined with apoptosis and autophagy assays
was used to examine the mechanisms underlying the antifungal action of
IBC. IBC exhibited excellent antifungal activity with 8 μg/mL of MIC for
<italic>C. albicans</italic>. IBC disrupted the cell membrane integrity,
and inhibited biofilm formation. IBC dispersed mature biofilms and
damaged biofilm cells of <italic>C. albicans</italic> at 32 μg/mL.
Moreover, IBC induced apoptosis and autophagy-associated cell death of
<italic>C. albicans</italic>. The RNA-seq analysis revealed upregulation
or downregulation of key genes involved in cell wall synthesis
(<italic>Wsc1</italic> and <italic>Fks1</italic>), ergosterol
biosynthesis (<italic>Erg3</italic>, and <italic>Erg11</italic>),
apoptisis (<italic>Hsp90</italic> and <italic>Aif1</italic>), as well as
autophagy pathways (<italic>Atg8</italic>, <italic>Atg13</italic>, and
<italic>Atg17</italic>), and so forth, in response to IBC, as evidenced
by the experiment-based phenotypic analysis. These results suggest that
IBC inhibits <italic>C. albicans</italic> growth by disrupting the cell
wall/membrane, caused by the altered expression of genes associated with
β-1,3-glucan and ergosterol biosynthesis. IBC induces apoptosis and
autophagy-associated cell death by upregulating the expression of
<italic>Hsp90</italic>, and altering autophagy-related genes involved in
the formation of the Atg1 complex and the pre-autophagosomal structure.
Together, our findings provide important insights into the potential
multifunctional mechanism of action of IBC.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336773">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336773</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336773">Save
to Pocket</a> </span></p>
<hr />
<h2
id="an-amplicon-based-nanopore-sequencing-workflow-for-rapid-tracking-of-avian-influenza-outbreaks-france-2020-2022">An
amplicon-based nanopore sequencing workflow for rapid tracking of avian
influenza outbreaks, France, 2020-2022</h2>
<p>date: 2024-01-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
During the recent avian influenza epizootics that occurred in France in
2020/21 and 2021/22, the virus was so contagiousness that it was
impossible to control its spread between farms. The preventive slaughter
of millions of birds consequently was the only solution available. In an
effort to better understand the spread of avian influenza viruses (AIVs)
in a rapid and innovative manner, we established an amplicon-based
MinION sequencing workflow for the rapid genetic typing of circulating
AIV strains. An amplicon-based MinION sequencing workflow based on a set
of PCR primers targeting primarily the hemagglutinin gene but also the
entire influenza virus genome was developed. Thirty field samples from
H5 HPAIV outbreaks in France, including environmental samples, were
sequenced using the MinION MK1C. A real-time alignment of the sequences
with MinKNOW software allowed the sequencing run to be stopped as soon
as enough data were generated. The consensus sequences were then
generated and a phylogenetic analysis was conducted to establish links
between the outbreaks. The whole sequence of the hemagglutinin gene was
obtained for the 30 clinical samples of H5Nx HPAIV belonging to clade
2.3.4.4b. The consensus sequences comparison and the phylogenetic
analysis demonstrated links between some outbreaks. While several
studies have shown the advantages of MinION for avian influenza virus
sequencing, this workflow has been applied exclusively to clinical field
samples, without any amplification step on cell cultures or embryonated
eggs. As this type of testing pipeline requires only a short amount of
time to link outbreaks or demonstrate a new introduction, it could be
applied to the real-time management of viral epizootics.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1257586">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1257586</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1257586">Save
to Pocket</a> </span></p>
<hr />
<h2
id="clinical-implications-of-trichomonads-detected-in-bronchoalveolar-fluid-by-metagenomic-next-generation-sequencing-a-multicenter-retrospective-study">Clinical
implications of trichomonads detected in bronchoalveolar fluid by
metagenomic next-generation sequencing: a multicenter retrospective
study</h2>
<p>date: 2024-01-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Pulmonary trichomoniasis is considered a neglected disease due to
failures in recognizing it, stemming from insensitive microbial methods
and a lack of specific clinical features. This study aims to analyze the
clinical implications of trichomonads detected in bronchoalveolar lavage
fluid (BALF) by metagenomic next-generation sequencing (mNGS).
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This multicenter retrospective study included patients diagnosed with
pneumonia, admitted to three tertiary hospitals in China from July 2018
to September 2022, with trichomonads detected in BALF through mNGS. The
analysis covered demographics, comorbidities, symptoms, laboratory
findings, mNGS results, clinical treatment, and outcomes of these
patients.
</p>
</sec><sec>
<title>
Results
</title>
<p>
A total of 17 patients were enrolled, comprising 14 males and 3 females.
Trichomonas tenax and Trichomonas vaginalis were detected by mNGS in
BALF samples of 15 and 2 patients, respectively. Patients were
categorized into two groups based on the presence of risk factors for
trichomonad infection, including immunocompromised conditions,
uncontrolled diabetes mellitus, oral/periodontal diseases, and
aspiration. Among 11 patients with risk factors (Case 1-11), 4 received
nitromidazoles as part of comprehensive treatment, achieving a 100%
treatment success rate. The remaining 7 patients, who did not receive
nitromidazoles, had only one achieving relief after broad-spectrum
antimicrobial therapy, resulting in a 14.3% treatment success rate. For
the 6 patients without any risk factors for trichomonad infection (Case
12-17), none received nitromidazoles during hospitalization. However, 4
out of these 6 patients (66.7%) eventually recovered.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
mNGS proves to be an efficient tool for detecting trichomonads in BALF
samples. Comprehensive analysis of clinical features and laboratory
indicators is essential to distinguish between infection and
colonization of trichomonads. Pulmonary trichomoniasis should not be
overlooked when trichomonads are detected in BALF from patients with
risk factors.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289231">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289231</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289231">Save
to Pocket</a> </span></p>
<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
