---
title: health 2024.16
updated: 2024-04-17 08:24:36
---

# health 2024.16

(date: 2024-04-17 08:24:36)

---

## Single nucleotide variants in the CCL2, OAS1 and DPP9 genes and their association with the severity of COVID-19 in an Ecuadorian population

date: 2024-04-17, from: Frontiers in Cellular and Infection Microbiology

COVID-19 has a broad clinical spectrum, ranging from asymptomatic-mild form to severe phenotype. The severity of COVID-19 is a complex trait influenced by various genetic and environmental factors. Ethnic differences have been observed in relation to COVID-19 severity during the pandemic. It is currently unknown whether genetic variations may contribute to the increased risk of severity observed in Latin-American individuals The aim of this study is to investigate the potential correlation between gene variants at CCL2, OAS1, and DPP9 genes and the severity of COVID-19 in a population from Quito, Ecuador. This observational case-control study was conducted at the Carrera de Biologia from the Universidad Central del Ecuador and the Hospital Quito Sur of the Instituto Ecuatoriano de Seguridad Social (Quito-SUR-IESS), Quito, Ecuador. Genotyping for gene variants at rs1024611 (A>G), rs10774671 (A>G), and rs10406145 (G>C) of CCL2, OAS1, and DPP9 genes was performed on 100 COVID-19 patients (43 with severe form and 57 asymptomatic-mild) using RFLP-PCR. The genotype distribution of all SNVs throughout the entire sample of 100 individuals showed Hardy Weinberg equilibrium (P=0.53, 0.35, and 0.4 for CCL2, OAS1, and DPP9, respectively). The HWE test did not find any statistically significant difference in genotype distribution between the study and control groups for any of the three SNVs. The multivariable logistic regression analysis showed that individuals with the GG of the CCL2 rs1024611 gene variant had an increased association with the severe COVID-19 phenotype in a recessive model (P = 0.0003, OR = 6.43, 95% CI 2.19-18.89) and for the OAS1 rs10774671 gene variant, the log-additive model showed a significant association with the severe phenotype of COVID-19 (P=0.0084, OR=3.85, 95% CI 1.33-11.12). Analysis of haplotype frequencies revealed that the coexistence of GAG at CCL2, OAS1, and DPP9 variants, respectively, in the same individual increased the presence of the severe COVID-19 phenotype (OR=2.273, 95% CI: 1.271-4.068, P=0.005305). The findings of the current study suggests that the ethnic background affects the allele and genotype frequencies of genes associated with the severity of COVID-19. The experience with COVID-19 has provided an opportunity to identify an ethnicity-based approach to recognize genetically high-risk individuals in different populations for emerging diseases. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1322882>

---

## Ocular surface immune transcriptome and tear cytokines in corneal infection patients

date: 2024-04-17, from: Frontiers in Cellular and Infection Microbiology

BackgroundMicrobial keratitis is one of the leading causes of blindness globally. An overactive immune response during an infection can exacerbate damage, causing corneal opacities and vision loss. This study aimed to identify the differentially expressed genes between corneal infection patients and healthy volunteers within the cornea and conjunctiva and elucidate the contributing pathways to these conditions’ pathogenesis. Moreover, it compared the corneal and conjunctival transcriptomes in corneal-infected patients to cytokine levels in tears.MethodsCorneal and conjunctival swabs were collected from seven corneal infection patients and three healthy controls under topical anesthesia. RNA from seven corneal infection patients and three healthy volunteers were analyzed by RNA sequencing (RNA-Seq). Tear proteins were extracted from Schirmer strips via acetone precipitation from 38 cases of corneal infection and 14 healthy controls. The cytokines and chemokines IL-1β, IL-6, CXCL8 (IL-8), CX3CL1, IL-10, IL-12 (p70), IL-17A, and IL-23 were measured using an antibody bead assay.ResultsA total of 512 genes were found to be differentially expressed in infected corneas compared to healthy corneas, with 508 being upregulated and four downregulated (fold-change (FC) <−2 or > 2 and adjusted p <0.01). For the conjunctiva, 477 were upregulated, and 3 were downregulated (FC <−3 or ≥ 3 and adjusted p <0.01). There was a significant overlap in cornea and conjunctiva gene expression in patients with corneal infections. The genes were predominantly associated with immune response, regulation of angiogenesis, and apoptotic signaling pathways. The most highly upregulated gene was CXCL8 (which codes for IL-8 protein). In patients with corneal infections, the concentration of IL-8 protein in tears was relatively higher in patients compared to healthy controls but did not show statistical significance.ConclusionsDuring corneal infection, many genes were upregulated, with most of them being associated with immune response, regulation of angiogenesis, and apoptotic signaling. The findings may facilitate the development of treatments for corneal infections that can dampen specific aspects of the immune response to reduce scarring and preserve sight. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1346821>

---

## Infection with the multidrug-resistant Klebsiella pneumoniae New Delhi metallo-B-lactamase strain in patients with COVID-19: Nec Hercules contra plures?

date: 2024-04-16, from: Frontiers in Cellular and Infection Microbiology

BackgroundDuring the coronavirus disease 2019 (COVID-19) pandemic, in patients treated for SARS-CoV-2 infection, infections with the Klebsiella pneumoniae bacteria producing New Delhi metallo-B-lactamase (NDM) carbapenemase in the USA, Brazil, Mexico, and Italy were observed, especially in intensive care units (ICUs). This study aimed to assess the impact of Klebsiella pneumoniae NDM infection and other bacterial infections on mortality in patients treated in ICUs due to COVID-19.MethodsThe 160 patients who qualified for the study were hospitalized in ICUs due to COVID-19. Three groups were distinguished: patients with COVID-19 infection only (N = 72), patients with COVID-19 infection and infection caused by Klebsiella pneumoniae NDM (N = 30), and patients with COVID-19 infection and infection of bacterial etiology other than Klebsiella pneumoniae NDM (N = 58). Mortality in the groups and chosen demographic data; biochemical parameters analyzed on days 1, 3, 5, and 7; comorbidities; and ICU scores were analyzed.ResultsBacterial infection, including with Klebsiella pneumoniae NDM type, did not elevate mortality rates. In the group of patients who survived the acute phase of COVID-19 the prolonged survival time was demonstrated: the median overall survival time was 13 days in the NDM bacterial infection group, 14 days in the other bacterial infection group, and 7 days in the COVID-19 only group. Comparing the COVID-19 with NDM infection and COVID-19 only groups, the adjusted model estimated a statistically significant hazard ratio of 0.28 (p = 0.002). Multivariate analysis revealed that age, APACHE II score, and CRP were predictors of mortality in all the patient groups.ConclusionIn patients treated for SARS-CoV-2 infection acquiring a bacterial infection due to prolonged hospitalization associated with the treatment of COVID-19 did not elevate mortality rates. The data suggests that in severe COVID-19 patients who survived beyond the first week of hospitalization, bacterial infections, particularly Klebsiella pneumoniae NDM, do not significantly impact mortality. Multivariate analysis revealed that age, APACHE II score, and CRP were predictors of mortality in all the patient groups. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297312>

---

## The complete genome sequence of unculturable Mycoplasma faucium obtained through clinical metagenomic next-generation sequencing

date: 2024-04-16, from: Frontiers in Cellular and Infection Microbiology

IntroductionDiagnosing Mycoplasma faucium poses challenges, and it's unclear if its rare isolation is due to infrequent occurrence or its fastidious nutritional requirements.MethodsThis study analyzes the complete genome sequence of M. faucium, obtained directly from the pus of a sternum infection in a lung transplant patient using metagenomic sequencing.ResultsGenome analysis revealed limited therapeutic options for the M. faucium infection, primarily susceptibility to tetracyclines. Three classes of mobile genetic elements were identified: two new insertion sequences, a new prophage (phiUMCG-1), and a species-specific variant of a mycoplasma integrative and conjugative element (MICE). Additionally, a Type I Restriction-Modification system was identified, featuring 5’-terminally truncated hsdS pseudogenes with overlapping repeats, indicating the potential for forming alternative hsdS variants through recombination.ConclusionThis study represents the first-ever acquisition of a complete circularized bacterial genome directly from a patient sample obtained from invasive infection of a primary sterile site using culture-independent, PCR-free clinical metagenomics. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368923>

---

## Comprehensive assessment of HF-rTMS treatment mechanism for post-stroke dysphagia in rats by integration of fecal metabolomics and 16S rRNA sequencing

date: 2024-04-15, from: Frontiers in Cellular and Infection Microbiology

BackgroundThe mechanism by which high-frequency repetitive transcranial magnetic stimulation (HF-rTMS) improves swallowing function by regulating intestinal flora remains unexplored. We aimed to evaluate this using fecal metabolomics and 16S rRNA sequencing.MethodsA Post-stroke dysphagia (PSD) rat model was established by middle cerebral artery occlusion. The magnetic stimulation group received HF-rTMS from the 7th day post-operation up to 14th day post-surgery. Swallowing function was assessed using a videofluoroscopic swallowing study (VFSS). Hematoxylin-eosin (H&E) staining was used to assess histopathological changes in the intestinal tissue. Intestinal flora levels were evaluated by sequencing the 16S rRNA V3-V4 region. Metabolite changes within the intestinal flora were evaluated by fecal metabolomics using liquid chromatography-tandem mass spectrometry.ResultsVFSS showed that the bolus area and pharyngeal bolus speed were significantly decreased in PSD rats, while the bolus area increased and pharyngeal transit time decreased after HF-rTMS administration (p < 0.05). In the PSD groups, H&E staining revealed damaged surface epithelial cells and disrupted cryptal glands, whereas HF-rTMS reinforced the integrity of the intestinal epithelial cells. 16S rRNA sequencing indicated that PSD can disturb the intestinal flora and its associated metabolites, whereas HF-rTMS can significantly regulate the composition of the intestinal microflora. Firmicutes and Lactobacillus abundances were lower in the PSD group than in the baseline group at the phylum and genus levels, respectively; however, both increased after HF-rTMS administration. Levels of ceramides (Cer), free fatty acids (FA), phosphatidylethanolamine (PE), triacylglycerol (TAG), and sulfoquinovosyl diacylglycerol were increased in the PSD group. The Cer, FA, and DG levels decreased after HF-rTMS treatment, whereas the TAG levels increased. Peptococcaceae was negatively correlated with Cer, Streptococcus was negatively correlated with DG, and Acutalibacter was positively correlated with FA and Cer. However, these changes were effectively restored by HF-rTMS, resulting in recovery from dysphagia.ConclusionThese findings suggest a synergistic role for the gut microbiota and fecal metabolites in the development of PSD and the therapeutic mechanisms underlying HF-rTMS. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1373737>

---

## Editorial: Identification and characterization of novel antigens of malarial parasites, volume II

date: 2024-04-15, from: Frontiers in Cellular and Infection Microbiology

 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408366>

---

## Editorial: One Health in clinical microbiology

date: 2024-04-15, from: Frontiers in Cellular and Infection Microbiology

 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1404276>

---

## Development of a new loop-mediated isothermal amplification test for the sensitive, rapid, and economic detection of different genotypes of Classical swine fever virus

date: 2024-04-15, from: Frontiers in Cellular and Infection Microbiology

BackgroundClassical swine fever virus (CSFV) remains one of the most important pathogens in animal health. Pathogen detection relies on viral RNA extraction followed by RT-qPCR. Novel technologies are required to improve diagnosis at the point of care.MethodsA loop-mediated isothermal amplification (LAMP) PCR technique was developed, with primers designed considering all reported CSFV genotypes. The reaction was tested using both fluorometric and colorimetric detection, in comparison to the gold standard technique. Viral strains from three circulating CSFV genotypes were tested, as well as samples from infected animals. Other pathogens were also tested, to determine the LAMP specificity. Besides laboratory RNA extraction methods, a heating method for RNA release, readily available for adaptation to field conditions was evaluated.ResultsThree primer sets were generated, with one of them showing better performance. This primer set proved capable of maintaining optimal performance at a wide range of amplification temperatures (60°C - 68°C). It was also able to detect CSFV RNA from the three genotypes tested. The assay was highly efficient in detection of samples from animals infected with field strains from two different genotypes, with multiple matrices being detected using both colorimetric and fluorometric methods. The LAMP assay was negative for all the unrelated pathogens tested, including Pestiviruses. The only doubtful result in both fluorometric and colorimetric LAMP was against the novel Pestivirus italiaense, ovine Italy Pestivirus (OVPV), which has proven to have cross-reaction with multiple CSFV diagnostic techniques. However, it is only possible to detect the OVPV in a doubtful result if the viral load is higher than 10000 viral particles.ConclusionThe results from the present study show that LAMP could be an important addition to the currently used molecular diagnostic techniques for CSFV. This technique could be used in remote locations, given that it can be adapted for successful use with minimal equipment and minimally invasive samples. The joined use of novel and traditional diagnostic techniques could prove to be a useful alternative to support the CSF control. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1372166>

