<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.02</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../">The Antenna</a></li>
	<li><a href="../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../parks.html" title="new about parks">Parks</a></li>
	<li><a href="../../tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="../../libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="./">Archives</a></li>
	<li><a href="../search.html">Search</a></li>
	<li><a href="../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.02</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.02">health 2024.02</h1>
<p>(date: 2024-01-11 14:45:08)</p>
<hr />
<h2
id="the-effect-and-mechanism-of-fushen-granule-on-gut-microbiome-in-the-prevention-and-treatment-of-chronic-renal-failure">The
effect and mechanism of Fushen Granule on gut microbiome in the
prevention and treatment of chronic renal failure</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Fushen Granule is an improved granule based on the classic formula
Fushen Formula, which is used for the treatment of peritoneal
dialysis-related intestinal dysfunction in patients with end-stage renal
disease. However, the effect and mechanism of this granule on the
prevention and treatment of chronic renal failure have not been fully
elucidated.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
A 5/6 nephrectomy model of CRF was induced and Fushen Granule was
administered at low and high doses to observe its effects on renal
function, D-lactate, serum endotoxin, and intestinal-derived metabolic
toxins. The 16SrRNA sequencing method was used to analyze the abundance
and structure of the intestinal flora of CRF rats. A FMT assay was also
used to evaluate the effects of transplantation of Fushen Granule fecal
bacteria on renal-related functional parameters and metabolic toxins in
CRF rats.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Gavage administration of Fushen Granule at low and high doses
down-regulated creatinine, urea nitrogen, 24-h urine microalbumin,
D-lactate, endotoxin, and the intestinal-derived toxins indophenol
sulphateand p-cresol sulphate in CRF rats. Compared with the
sham-operated group in the same period, CRF rats had a decreased
abundance of the firmicutes phylum and an increased abundance of the
bacteroidetes phylum at the phylum level, and a decreasing trend of the
lactobacillus genus at the genus level. Fushen Granule intervention
increased the abundance of the firmicutes phylum, decreased the
abundance of the bacteroidetes phylum, and increased the abundance of
the lactobacillus genus. The transplantation of Fushen Granule fecal
bacteria significantly reduced creatinine(Cr), blood urea nitrogen(Bun),
uric acid(UA), 24-h urinary microalbumin, D-lactate, serum endotoxin,
and enterogenic metabolic toxins in CRF rats. Compared with the
sham-operated group, the transplantation of Fushen Granule fecal
bacteria modulated the Firmicutes and Bacteroidetes phyla and the
Lactobacillus genus.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Fushen Granule improved renal function and intestinal barrier function
by regulating intestinal flora, inhibiting renal fibrosis, and delaying
the progression of chronic renal failure.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1334213">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1334213</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1334213">Save
to Pocket</a> </span></p>
<hr />
<h2
id="will-previous-antimicrobial-therapy-reduce-the-positivity-rate-of-metagenomic-next-generation-sequencing-in-periprosthetic-joint-infections-a-clinical-study">Will
previous antimicrobial therapy reduce the positivity rate of metagenomic
next-generation sequencing in periprosthetic joint infections? A
clinical study</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Metagenomic next-generation sequencing (mNGS) is a culture-independent
massively parallel DNA sequencing technology and it has been widely used
for rapid etiological diagnosis with significantly high positivity rate.
Currently, clinical studies on evaluating the influence of previous
antimicrobial therapy on positivity rate of mNGS in PJIs are rarely
reported. The present study aimed to investigate whether the positivity
rate of mNGS is susceptible to previous antimicrobial therapy.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We performed a prospective trial among patients who undergone hip or
knee surgery due to periprosthetic joint infection (PJI) to compare the
positivity rate of culture and mNGS between cases with and without
previous antimicrobial therapy, and the positivity rates between cases
with different antimicrobial-free intervals were also analysed.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Among 131 included PJIs, 91 (69.5%) had positive cultures and 115
(87.8%) had positive mNGS results. There was no significant difference
in the positivity rate of deep-tissue culture and synovial fluid mNGS
between cases with and without previous antimicrobial therapy. The
positivity rate of synovial fluid culture was higher in cases with
previous antimicrobial therapy. The positivity rates of mNGS in synovial
fluid decreased as the antimicrobial-free interval ranged from 4 to 14
days to 0 to 3 days.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
mNGS is more advantageous than culture with a higher pathogen detection
rate. However, our data suggested that antimicrobial agents may need to
be discontinued more than 3 days before sampling to further increase the
positivity rate of mNGS for PJIs.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1295962">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1295962</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1295962">Save
to Pocket</a> </span></p>
<hr />
<h2
id="characterization-of-klebsiella-pneumoniae-carrying-the-blandm-1-gene-in-incx3-plasmids-and-the-rare-in1765-in-an-incfib-inchi1b-plasmid">Characterization
of Klebsiella pneumoniae carrying the blaNDM-1 gene in IncX3 plasmids
and the rare In1765 in an IncFIB-IncHI1B plasmid</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Today, the <italic>bla</italic><sub>NDM</sub> gene is widely distributed
on several plasmids from a variety of Gram-negative bacteria, primarily
in transposons and gene cassettes within their multidrug-resistant (MDR)
regions. This has led to the global dissemination of the
<italic>bla</italic><sub>NDM</sub> gene.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
The determination of class A beta-lactamase, class B and D
carbapenemases was performed according to the recommendations of the
Clinical and Laboratory Standards Institute (CLSI). Antimicrobial
susceptibility testing was performed using both the BioMerieux VITEK2
system and antibiotic paper diffusion methods. Plasmid transfer was then
evaluated by conjugation experiments and plasmid electroporation assays.
To comprehensively analyze the complete genome of <italic>K.
pneumoniae</italic> strain F11 and to investigate the presence of mobile
genetic elements associated with antibiotic resistance and virulence
genes, Nanopore and Illumina sequencing platforms were used, and
bioinformatics methods were applied to analyze the obtained data.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Our findings revealed that <italic>K. pneumoniae</italic> strain F11
carried class A beta-lactamase and classes B+D carbapenemases, and
exhibited resistance to commonly used antibiotics, particularly
tigecycline and ceftazidime/avibactam, due to the presence of relevant
resistance genes. Plasmid transfer assays demonstrated successful
recovery of plasmids pA_F11 and pB_F11, with average conjugation
frequencies of 2.91×10<sup>-4</sup> and 1.56×10<sup>-4</sup>,
respectively. However, plasmids pC_F11 and pD_F11 failed in both
conjugation and electroporation experiments. The MDR region of plasmid
pA_F11 contained rare In1765, Tn<italic>As2</italic>, and
Tn<italic>As3</italic> elements. The MDR2 region of plasmid pB_F11
functioned as a mobile genetic “island” and lacked the
<italic>bla</italic><sub>NDM-1</sub> gene, serving as a “bridge”
connecting the early composite structure of
<italic>ble</italic><sub>MBL</sub> and
<italic>bla</italic><sub>NDM-1</sub> to the recent composite structure.
Additionally, the MDR1 region of plasmid pB_F11 comprised In27,
Tn<italic>As1</italic>, Tn<italic>As3</italic>, and
Tn<italic>2</italic>; and plasmid pC_F11 harbored the recent composite
structure of <italic>ble</italic><sub>MBL</sub> and
<italic>bla</italic><sub>NDM-1</sub> within Tn<italic>3000</italic>
which partially contained partial Tn<italic>125</italic>.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
This study demonstrated that complex combinations of transposons and
integron overlaps, along with the synergistic effects of different drug
resistance and virulence genes, led to a lack of effective therapeutic
agents for strain F11, therefore its dissemination and prevalence should
be strictly controlled.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324846">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324846</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324846">Save
to Pocket</a> </span></p>
<hr />
<h2
id="transcriptional-profiling-specifies-the-pathogen-specific-human-host-response-to-infectious-keratitis">Transcriptional
profiling specifies the pathogen-specific human host response to
infectious keratitis</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Purpose
</title>
<p>
Corneal infections are a leading cause of visual impairment and
blindness worldwide. Here we applied high-resolution transcriptomic
profiling to assess the general and pathogen-specific molecular and
cellular mechanisms during human corneal infection.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Clinical diagnoses of herpes simplex virus (HSV) (n=5) and
bacterial/fungal (n=5) keratitis were confirmed by histology. Healthy
corneas (n=7) and keratoconus (n=4) samples served as controls.
Formalin-fixed, paraffin-embedded (FFPE) human corneal specimens were
analyzed using the 3’ RNA sequencing method Massive Analysis of cDNA
Ends (MACE RNA-seq). The cellular host response was investigated using
comprehensive bioinformatic deconvolution (xCell and CYBERSORTx)
analyses and by integration with published single cell RNA-seq data of
the human cornea.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Our analysis identified 216 and 561 genes, that were specifically
overexpressed in viral or bacterial/fungal keratitis, respectively, and
allowed to distinguish the two etiologies. The virus-specific host
response was driven by adaptive immunity and associated molecular
signaling pathways, whereas the bacterial/fungal-specific host response
mainly involved innate immunity signaling pathways and cell types. We
identified several genes and pathways involved in the host response to
infectious keratitis, including CXCL9, CXCR3, and MMP9 for viral, and
S100A8/A9, MMP9, and the IL17 pathway for bacterial/fungal keratitis.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
High-resolution molecular profiling provides new insights into the human
corneal host response to viral and bacterial/fungal infection.
Pathogen-specific molecular profiles may provide the foundation for
novel diagnostic biomarker and therapeutic approaches that target
inflammation-induced damage to corneal host cells with the goal to
improve the outcome of infectious keratitis.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1285676">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1285676</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1285676">Save
to Pocket</a> </span></p>
<hr />
<h2
id="potential-roles-of-culicoides-spp.-culicoides-imicola-culicoides-oxystoma-as-biological-vectors-of-bluetongue-virus-in-yuanyang-of-yunnan-p.-r.-china">Potential
roles of Culicoides spp. (Culicoides imicola, Culicoides oxystoma) as
biological vectors of bluetongue virus in Yuanyang of Yunnan, P. R.
China</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Culicoides</italic> plays a crucial role as an insect vector in
the field of veterinary medicine. The transmission of significant
viruses such as bluetongue virus (BTV) and African horse sickness virus
(AHSV) by this insect poses a substantial threat, leading to the
development of severe diseases in domestic animals. This study aimed to
explore the <italic>Culicoides</italic> species, identify their
blood-meal sources, and assess the presence of BTV and AHSV carried by
<italic>Culicoides</italic> in Yuanyang County, Yunnan Province. The aim
was to gain insights into the potential vectors of these two viruses and
elucidate their potential roles in the transmission of pathogens.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
The midges were collected from cattle (<italic>Bos indicus</italic>),
pig (<italic>Sus scrofa</italic>), and goat (<italic>Capra
hircus</italic>) pens in Yuanyang County, Yunnan Province in June 2020.
Initial identification of midges was conducted through morphological
characteristics, followed by molecular identification using the
cytochrome C oxidase subunit I (COI) gene. The determination of
<italic>Culicoides</italic> blood-meal sources was accomplished using
specific primers targeting the cytochrome <italic>b</italic> (Cyt
<italic>b</italic>) gene from potential hosts. BTV and AHSV RNA were
identified in <italic>Culicoides</italic> pools through the application
of reverse transcriptase PCR and quantitative real-time PCR. Nucleotide
homology and phylogenetic analysis were performed using MegAlign
(DNAStar) and Mega 6.0 software.
</p>
</sec><sec>
<title>
Results
</title>
<p>
A total of 6,300 <italic>Culicoides</italic>, consisting of <italic>C.
oxystoma, C. arakawai, C. imicola</italic>, and <italic>C.
innoxius</italic>, were collected from cattle, pigs, and goat pens. The
engorgement rates for these species were 30.2%, 54.6%, 75%, and 66.7%,
respectively. In the cattle pen, the prevailing species is <italic>C.
oxystoma</italic> (100%). In the pig pen, <italic>C. arakawai </italic>
dominates (70%), with <italic>C. oxystoma</italic> following at 30%. In
the goat pen, <italic>C. imicola</italic> holds the majority (45.45%),
trailed by <italic>C. oxystoma</italic> (25%), <italic>C.
innoxius</italic> (20.45%), and <italic>C. arakawai</italic> (9.09%).
These <italic>Culicoides</italic> species were identified as feeding on
cattle, pigs, goats, chickens (<italic>Gallus gallus</italic>), and
humans (<italic>Homo sapiens</italic>). The positivity rates for BTV
were 20.00% and 11.54% in blood-fed specimens of <italic>C.
imicola</italic> and <italic>C. oxystoma</italic>, respectively.
Conversely, the positivity rates for BTV in non-blood-fed specimens were
0.00% and 6.67% for <italic>C. imicola</italic> and <italic>C.
oxystoma</italic>, respectively. BTV was not detected in <italic>C.
arakawai</italic> and <italic>C. innoxius</italic>. The specimens (YY86)
from <italic>C. imicola</italic> that tested positive for BTV had the
closest genetic relationship to YTS-4 isolated from Mangshi, Yunnan
Province in 1996. All test results for the nucleic acid of AHSV were
negative.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The study reveals variations in the species distribution, community
composition, blood sucking rate, and blood-feeding sources of
<italic>Culicoides</italic> across different habitats. Notably,
<italic>C. imicola</italic> and <italic>C. oxystoma</italic> emerge as
potential vectors for the transmission of BTV in local animals.
Accordingly, this investigation provides crucial insights that can serve
as a valuable reference for the prevention and control of BTV in local
animals, particularly from the perspective of vector management.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1283216">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1283216</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1283216">Save
to Pocket</a> </span></p>
<hr />
<h2
id="endogenous-retroviruses-suppressyn-and-syncytin-2-as-innovative-prognostic-biomarkers-in-acute-myeloid-leukemia">Endogenous
retroviruses Suppressyn and Syncytin-2 as innovative prognostic
biomarkers in Acute Myeloid Leukemia</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Emerging evidence has proven that human endogenous retroviruses (HERVs)
play a critical role in the pathogenesis of Acute Myeloid Leukemia
(AML), whereas the specific HERVs influencing the prognosis of AML
patients have yet to be fully understood.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
In this study, a systematic exploration was achieved to identify
potential prognostic HERVs for AML, sourced from TCGA and GTEx database.
Differential analysis and functional enrichment studies were conducted
using GO, KEGG, GSEA, and GSVA. The ESTIMATE algorithm was applied to
explore the immune infiltration of HERVs in AML. A prognostic risk-score
model was evaluated with predicted yearly accuracy using ROC analysis.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Two HERVs Suppressyn and Syncytin-2, were identified as promising
prognostic biomarkers, with high discrimination ability based on ROC
analysis between AML and healthy cohorts from TCGA. Their expression was
notably higher in AML patients compared to those in healthy individuals
but correlates with favorable clinical outcomes in sub-groups such as
white race, lower WBC counts, favorable and intermediate risks, and NPM1
or IDH1 mutation. Suppressyn and Syncytin-2 participated in
immune-related pathways and exhibited correlations with multiple immune
infiltration cells, such as T cells, mast cells, and tumor-associated
macrophages. Finally, we developed a prognostic risk-scoring model
combining Suppressyn and Syncytin-2, where a high risk-score is
associated with better prognosis.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Collectively, our findings revealed that Suppressyn and Syncytin-2 may
act as valuable diagnostic and prognostic biomarkers for individuals
with AML, while highlighting links between HERV activation,
immunogenicity, and future therapeutic targets.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1339673">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1339673</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1339673">Save
to Pocket</a> </span></p>
<hr />
<h2
id="a-guide-to-stenotrophomonas-maltophilia-virulence-capabilities-as-we-currently-understand-them">A
guide to Stenotrophomonas maltophilia virulence capabilities, as we
currently understand them</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
The Gram-negative pathogen <italic>Stenotrophomonas maltophilia</italic>
causes a wide range of human infections. It causes particularly serious
lung infections in individuals with cystic fibrosis, leading to high
mortality rates. This pathogen is resistant to most known antibiotics
and harbors a plethora of virulence factors, including lytic enzymes and
serine proteases, that cause acute infection in host organisms.
<italic>S. maltophilia</italic> also establishes chronic infections
through biofilm formation. The biofilm environment protects the bacteria
from external threats and harsh conditions and is therefore vital for
the long-term pathogenesis of the microbe. While studies have identified
several genes that mediate <italic>S. maltophilia’s</italic> initial
colonization and biofilm formation, the cascade of events initiated by
these factors is poorly understood. Consequently, understanding these
and other virulence factors can yield exciting new targets for novel
therapeutics.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1322853">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1322853</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1322853">Save
to Pocket</a> </span></p>
<hr />
<h2
id="dna-damage-response-coregulator-ddrr-affects-many-cellular-pathways-and-processes-in-acinetobacter-baumannii-17978">DNA
damage response coregulator ddrR affects many cellular pathways and
processes in Acinetobacter baumannii 17978</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Acinetobacter baumannii</italic> strain 17978 is an
opportunistic pathogen possessing a DNA damage response (DDR) in which
multiple error-prone polymerase genes are co-repressed by a UmuD
homolog, UmuDAb, and the small <italic>Acinetobacter</italic>-specific
protein DdrR. Additionally, these regulators coactivate nine other
genes. We identified the DNA damage-inducible transcriptome for
wildtype, <italic>umuDAb</italic>, and <italic>recA</italic> strains,
and later established the <italic>ddrR</italic> DDR transcriptome.
However, the ATCC 17978 reference genome had several assembly errors and
lacked the 44 kb virulence locus, AbaAL44, that is present in the strain
17978 UN.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
For this project, we combined our earlier single-end read RNAseq data
with the <italic>ddrR</italic> paired-end reads and aligned these data
to the improved 17978 UN genome assembly that resembled our laboratory
strain, 17978 JH.
</p>
</sec><sec>
<title>
Results
</title>
<p>
New DESeq2 analyses verified previous differentially expressed genes
(DEGs) but also found 339 genes in 17978 JH that were not annotated or
physically present in the older genome assembly. Sixty-three were
differentially expressed after DNA damage, and 182 had differential
basal expression when comparing <italic>umuDAb</italic>,
<italic>ddrR</italic>, or <italic>recA</italic> strains to wildtype,
with 94 genes’ expression unchanged. This work identified and
characterized the 55 gene DNA damage-repressible transcriptome, 98% of
which required either <italic>umuDAb</italic> or <italic>ddrR</italic>
for repression. Two-thirds of these DEGs required both regulators. We
also identified 110 genes repressed only in the <italic>ddrR</italic>
strain, ~50% of which were due to increased basal expression levels.
Basal gene expression in the <italic>ddrR</italic> mutant was further
dysregulated independent of the DDR. Over 800 genes were upregulated,
and over 1200 genes were downregulated compared to wildtype expression.
Half of <italic>A. baumannii’s</italic> essential genes were upregulated
in the <italic>ddrR</italic> strain, including cell division genes, and
two-thirds of these were downregulated in the <italic>umuDAb</italic>
strain.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
The <italic>ddrR</italic> mutant upregulated genes enriched in
translation, RNA metabolism, protein metabolism, AA/FA/cell-structure
synthesis, and transport, while downregulating genes enriched in quorum
sensing, biofilm production, secretion systems, pilus production, cell
adhesion, and aromatics and chlorine degradation. Our data underscore
the need for accurate and appropriately matched genome assemblies and
indicate that <italic>ddrR</italic> affects approximately 60% of the
genome, rendering it a potential target for <italic>Acinetobacter
baumannii</italic> infection treatment.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324091">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324091</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324091">Save
to Pocket</a> </span></p>
<hr />
<h2
id="urine-microbiome-in-individuals-with-an-impaired-immune-system">Urine
microbiome in individuals with an impaired immune system</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
With the advent of next generation sequencing, it is now appreciated
that human urine is not sterile. Recent investigations of the urinary
microbiome (urobiome) have provided insights into several urological
diseases. Urobiome dysbiosis, defined as non-optimal urine microbiome
composition, has been observed in many disorders; however, it is not
clear whether this dysbiosis is the cause of urinary tract disorders or
a consequence. In addition, immunologically altered disorders are
associated with higher rates of urinary tract infections. These
disorders include immunoproliferative and immunodeficiency diseases,
cancer, and immunosuppressant therapy in transplant recipients. In this
review, we examine the current state of knowledge of the urobiome in
immunologically altered diseases, its composition and metabolomic
consequences. We conclude that more data are required to describe the
urobiome in immune altered states, knowledge that could facilitate
understanding the role of the urobiome and its pathophysiological
effects on urinary tract infections and other disorders of the urinary
tract.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1308665">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1308665</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1308665">Save
to Pocket</a> </span></p>
<hr />
<h2
id="streptomyces-albireticuli-lung-infection-managed-as-a-pulmonary-air-cyst-a-case-report-and-literature-review">Streptomyces
albireticuli lung infection managed as a pulmonary air cyst: a case
report and literature review</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Streptomyces</italic>, the largest genus in the
<italic>Streptomycetaceae</italic> family and a prolific producer of
antibacterial drugs, is a saprophytic soil organism that rarely causes
invasive infections. Here we report a case of necrotic pneumonia caused
by <italic>Streptomyces albireticuli</italic> in a 75-year-old man who
presented with progressive chest tightness and dyspnea.
<italic>Streptomyces albireticuli</italic> was isolated from his
bronchoalveolar lavage fluid and identified through whole-genome
sequencing (WGS) and phylogenetic analysis. The patient responded
satisfactorily to clarithromycin therapy. The findings of this study may
enhance our vigilance in identifying visceral infections caused by
<italic>Streptomyces</italic>.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1296491">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1296491</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1296491">Save
to Pocket</a> </span></p>
<hr />
<h2
id="lysosomal-trafficking-regulator-restricts-intracellular-growth-of-coxiella-burnetii-by-inhibiting-the-expansion-of-coxiella-containing-vacuole-and-upregulating-nos2-expression">Lysosomal
trafficking regulator restricts intracellular growth of Coxiella
burnetii by inhibiting the expansion of Coxiella-containing vacuole and
upregulating nos2 expression</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Coxiella burnetii</italic> is an obligate intracellular
bacterium that causes Q fever, a zoonotic disease typically manifests as
a severe flu-illness. After invading into the host cells, <italic>C.
burnetii</italic> delivers effectors to regulate the vesicle trafficking
and fusion events to form a large and mature
<italic>Coxiella</italic>-containing vacuole (CCV), providing sufficient
space and nutrition for its intracellular growth and proliferation.
Lysosomal trafficking regulator (LYST) is a member of the Beige and
Chediak-Higashi syndrome (BEACH) family, which regulates the transport
of vesicles to lysosomes and regulates TLR signaling pathway, but the
effect of LYST on <italic>C. burnetii</italic> infection is unclear. In
this study, a series of experiments has been conducted to investigate
the influence of LYST on intracellular growth of <italic>C.
burnetii</italic>. Our results showed that <italic>lyst</italic>
transcription was up-regulated in the host cells after <italic>C.
burnetii</italic> infection, but there is no significant change in
<italic>lyst</italic> expression level after infection with the Dot/Icm
type IV secretion system (T4SS) mutant strain, while CCVs expansion and
significantly increasing load of <italic>C. burnetii</italic> appeared
in the host cells with a silenced <italic>lyst</italic> gene, suggesting
LYST inhibits the intracellular proliferation of <italic>C.
burnetii</italic> by reducing CCVs size. Then, the size of CCVs and the
load of <italic>C. burnetii</italic> in the HeLa cells pretreated with
E-64d were significantly decreased. In addition, the level of iNOS was
decreased significantly in LYST knockout THP-1 cells, which was
conducive to the intracellular replication of <italic>C.
burnetii</italic>. This data is consistent with the phenotype of
L-NMMA-treated THP-1 cells infected with <italic>C. burnetii</italic>.
Our results revealed that the upregulation of <italic>lyst</italic>
transcription after infection is due to effector secretion of <italic>C.
burnetii</italic> and LYST inhibit the intracellular replication of
<italic>C. burnetii</italic> by reducing the size of CCVs and inducing
<italic>nos2</italic> expression.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1336600">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1336600</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1336600">Save
to Pocket</a> </span></p>
<hr />
<h2
id="genetic-evidence-on-the-causality-between-gut-microbiota-and-various-asthma-phenotypes-a-two-sample-mendelian-randomization-study">Genetic
evidence on the causality between gut microbiota and various asthma
phenotypes: a two-sample Mendelian randomization study</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Asthma is a multifarious disease that manifests in various phenotypes.
Among the various factors that contribute to the development of asthma,
the gut microbiota has recently emerged as a compelling area of
investigation. This study aims to investigate the causal relationships
between gut microbiota and distinct asthma phenotypes.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
The genome-wide association study (GWAS) summary statistics for 211 gut
microbial taxa were used as study exposure. Five traits pertaining to
various asthma phenotypes (asthma, allergic asthma, childhood asthma,
suggestive for eosinophilic asthma and obesity-related asthma) were
included as study outcome. We conducted Mendelian randomization (MR)
analysis and sensitivity analysis for each bacterial taxa and asthma
phenotypes.
</p>
</sec><sec>
<title>
Result
</title>
<p>
We discovered a total of 58 associations that exhibited evidence of
causality. Out of these, 4 associations remained significant even after
applying multiple correction. An increased risk of asthma was causally
associated with higher abundance of genus <italic>Holdemanella</italic>
(OR = 1.11; CI: 1.05-1.17; p = 0.027), genus
<italic>Oxalobacter</italic> (OR = 1.09; CI: 1.04-1.15; p = 0.025) and
genus <italic>Butyricimonas</italic> (OR = 1.14; CI: 1.06-1.22; p =
0.027). Order <italic>NB1n</italic> was causally linked with an
increased risk of obesity-related asthma (OR = 1.17; CI: 1.07-1.29; p =
0.015). There was limited overlap among the taxa that exhibited
potential causal relationships with distinct asthma phenotypes.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Our research has provided genetic evidence that establishes multiple
causal relationships between the gut microbiota and distinct asthma
phenotypes, supporting the role of the gut microbiota in various asthma
phenotypes. It is possible that different taxa play a role in the
development of distinct asthma phenotypes. The causal relationships
identified in this study require further investigation.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1270067">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1270067</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1270067">Save
to Pocket</a> </span></p>
<hr />
<h2
id="gut-microbiome-based-noninvasive-diagnostic-model-to-predict-acute-coronary-syndromes">Gut
microbiome-based noninvasive diagnostic model to predict acute coronary
syndromes</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Previous studies have shown that alterations in the gut microbiota are
closely associated with Acute Coronary Syndrome (ACS) development.
However, the value of gut microbiota for early diagnosis of ACS remains
understudied.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We recruited 66 volunteers, including 29 patients with a first diagnosis
of ACS and 37 healthy volunteers during the same period, collected their
fecal samples, and sequenced the V4 region of the 16S rRNA gene.
Functional prediction of the microbiota was performed using PICRUSt2.
Subsequently, we constructed a nomogram and corresponding webpage based
on microbial markers to assist in the diagnosis of ACS. The diagnostic
performance and usefulness of the model were analyzed using boostrap
internal validation, calibration curves, and decision curve analysis
(DCA).
</p>
</sec><sec>
<title>
Results
</title>
<p>
Compared to that of healthy controls, the diversity and composition of
microbial community of patients with ACS was markedly abnormal.
Potentially pathogenic genera such as <italic>Streptococcus</italic> and
<italic>Acinetobacter</italic> were significantly increased in the ACS
group, whereas certain SCFA-producing genera such as
<italic>Blautia</italic> and <italic>Agathobacter</italic> were
depleted. In addition, in the correlation analysis with clinical
indicators, the microbiota was observed to be associated with the level
of inflammation and severity of coronary atherosclerosis. Finally, a
diagnostic model for ACS based on gut microbiota and clinical variables
was developed with an area under the receiver operating characteristic
(ROC) curve (AUC) of 0.963 (95% CI: 0.925–1) and an AUC value of 0.948
(95% CI: 0.549–0.641) for bootstrap internal validation. The calibration
curves of the model show good consistency between the actual and
predicted probabilities. The DCA showed that the model had a high net
clinical benefit for clinical applications.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Our study is the first to characterize the composition and function of
the gut microbiota in patients with ACS and healthy populations in
Southwest China and demonstrates the potential effect of the microbiota
as a non-invasive marker for the early diagnosis of ACS.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1305375">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1305375</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1305375">Save
to Pocket</a> </span></p>
<hr />
<h2
id="a-novel-enemy-of-cancer-recent-investigations-into-protozoan-anti-tumor-properties">A
novel enemy of cancer: recent investigations into protozoan anti-tumor
properties</h2>
<p>date: 2024-01-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Cancer remains a significant global health issue, despite advances in
screening and treatment. While existing tumor treatment protocols such
as surgery, chemotherapy, radiotherapy, targeted therapy, and
immunotherapy have proven effective in enhancing the prognosis for some
patients, these treatments do not benefit all patients. Consequently,
certain types of cancer continue to exhibit a relatively low 5-year
survival rate. Therefore, the pursuit of novel tumor intervention
strategies may help improve the current effectiveness of tumor
treatment. Over the past few decades, numerous species of protozoa and
their components have exhibited anti-tumor potential via immune and
non-immune mechanisms. This discovery introduces a new research
direction for the development of new and effective cancer treatments.
Through <italic>in vitro</italic> experiments and studies involving
tumor-bearing mice, the anti-tumor ability of <italic>Toxoplasma
gondii</italic>, <italic>Plasmodium</italic>, <italic>Trypanosoma
cruzi</italic>, and other protozoa have unveiled diverse mechanisms by
which protozoa combat cancer, demonstrating encouraging prospects for
their application. In this review, we summarize the anti-tumor ability
and anti-tumor mechanisms of various protozoa and explore the potential
for their clinical development and application.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325144">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325144</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325144">Save
to Pocket</a> </span></p>
<hr />
<h2
id="getting-clues-from-nature-the-impact-of-grass-hay-on-suckling-piglets-gastrointestinal-growth-and-colonic-microbiota">Getting
clues from nature: the impact of grass hay on suckling piglets’
gastrointestinal growth and colonic microbiota</h2>
<p>date: 2024-01-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
The effect of dietary fiber on pig production has been extensively
evaluated. Inspired by observations of the diet of wild, young piglets,
this study aimed to examine the possibility of feeding grass hay to
suckling piglets besides concentrated creep feed.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
The sow-nursed piglets in this study were divided into two groups based
on balanced sow parities. The control group (CON, <italic>n</italic> = 7
sows) only received a regular, concentrated creep feed, while the
treatment piglets (GH, <italic>n</italic> = 8 sows) were also provided
with chopped grass hay from 2 days of age until weaning (28 days). At
weaning, one piglet with a median weight was selected from each litter
for post-mortem evaluation. Subsequently, six pigs around median weight
per sow were grouped into nursery pens and monitored for their feed
intake and body weight gain until 9 weeks of age.
</p>
</sec><sec>
<title>
Results and discussion
</title>
<p>
Piglets in GH consumed, on average, 57 g of grass hay per piglet during
the entire lactation period. The emptied weight of the small and large
intestine was significantly greater in GH (280 vs. 228 g, 88.8 vs. 79.3
g, respectively, <italic>p</italic> &lt; 0.05), and the length of the
large intestine was stimulated by the grass hay (164 vs. 150 cm,
<italic>p</italic> &lt; 0.05). Morphologically, the villus height in the
jejunum was higher in GH (<italic>p</italic> &lt; 0.05). In the large
intestine, the crypt depth of the mid-colon was lower in GH. Moreover,
the short-chain fatty acid (SCFA) concentrations in the cecum were
increased in GH compared to CON (1,179 vs. 948 µmol/g dry matter,
<italic>p</italic> &lt; 0.05), whereas in the colon, SCFA concentrations
were lower in CON (341 vs. 278 µmol/g dry matter, <italic>p</italic>
&lt; 0.05). There was no major impact of grass hay inclusion on the
colonic microbiota composition. Only a trend was observed for a lower
inverse of the classical Simpson (InvSimpon) index and a higher
abundance of <italic>Lactobacillus</italic> genera in GH. After weaning,
no significant differences in feed intake and body weight gain were
observed. In conclusion, supplementing the grass hay to suckling piglets
led to alterations in intestinal morphology, increased SCFA fermentation
in proximal sections of large intestine, stimulation of gastrointestinal
tract growth, and subtle modifications in colonic microbiota.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1341147">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1341147</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1341147">Save
to Pocket</a> </span></p>
<hr />
<h2
id="in-vitro-inhibition-of-pseudomonas-aeruginosa-pao1-biofilm-formation-by-dz2002-through-regulation-of-extracellular-dna-and-alginate-production">In
vitro inhibition of Pseudomonas aeruginosa PAO1 biofilm formation by
DZ2002 through regulation of extracellular DNA and alginate
production</h2>
<p>date: 2024-01-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Pseudomonas aeruginosa</italic> (<italic>P. aeruginosa)</italic>
is a common pathogen associated with biofilm infections, which can lead
to persistent infections. Therefore, there is an urgent need to develop
new anti-biofilm drugs. DZ2002 is a reversible inhibitor that targets
S-adenosylhomocysteine hydrolase and possesses anti-inflammatory and
immune-regulatory activities. However, its anti-biofilm activity has not
been reported yet.
</p>
</sec><sec>
<title>
Methods and results
</title>
<p>
Therefore, we investigated the effect of DZ2002 on <italic>P.
aeruginosa</italic> PAO1 biofilm formation by crystal violet staining
(CV), real-time quantitative polymerase chain reaction (RT-qPCR) and
confocal laser scanning microscopy (CLSM). The results indicated that
although DZ2002 didn’t affect the growth of planktonic PAO1, it could
significantly inhibit the formation of mature biofilms. During the
inhibition of biofilm formation by DZ2002, there was a parallel decrease
in the synthesis of alginate and the expression level of alginate genes,
along with a weakening of swarming motility. However, these results were
unrelated to the expression of <italic>lasI, lasR, rhII, rhIR</italic>.
Additionally, we also found that after treatment with DZ2002, the
biofilms and extracellular DNA content of PAO1 were significantly
reduced. Molecular docking results further confirmed that DZ2002 had a
strong binding affinity with the active site of S-adenosylhomocysteine
hydrolase (SahH) of PAO1.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
In summary, our results indicated that DZ2002 may interact with SahH in
PAO1, inhibiting the formation of mature biofilms by downregulating
alginate synthesis, extracellular DNA production and swarming motility.
These findings demonstrate the potential value of DZ2002 in treating
biofilm infections associated with <italic>P. aeruginosa</italic>.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1333773">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1333773</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1333773">Save
to Pocket</a> </span></p>
<hr />
<h2
id="effects-of-oxyr-regulator-on-oxidative-stress-apx-toxin-secretion-and-virulence-of-actinobacillus-pleuropneumoniae">Effects
of OxyR regulator on oxidative stress, Apx toxin secretion and virulence
of Actinobacillus pleuropneumoniae</h2>
<p>date: 2024-01-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Actinobacillus pleuropneumoniae</italic>, the causative agent of
porcine pleuropneumonia, poses a significant threat to global swine
populations due to its high prevalence, mortality rates, and substantial
economic ramifications. Understanding the pathogen’s defense mechanisms
against host-produced reactive oxygen species is crucial for its
survival, with OxyR, a conserved bacterial transcription factor, being
pivotal in oxidative stress response.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This study investigated the presence and role of OxyR in <italic>A.
pleuropneumoniae</italic> serovar 1-12 reference strains. Transcriptomic
analysis was conducted on an oxyR disruption mutant to delineate the
biological activities influenced by OxyR. Additionally, specific assays
were employed to assess urease activity, catalase expression, ApxI toxin
secretion, as well as adhesion and invasion abilities of the oxyR
disruption mutant on porcine 3D4/21 and PT cells. A mice challenge
experiment was also conducted to evaluate the impact of oxyR
inactivation on <italic>A. pleuropneumoniae</italic> virulence.
</p>
</sec><sec>
<title>
Results
</title>
<p>
OxyR was identified as a conserved regulator present in <italic>A.
pleuropneumoniae</italic> serovar 1-12 reference strains. Transcriptomic
analysis revealed the involvement of OxyR in multiple biological
activities. The oxyR disruption resulted in decreased urease activity,
elevated catalase expression, enhanced ApxI toxin secretion—attributed
to OxyR binding to the apxIBD promoter—and reduced adhesion and invasion
abilities on porcine cells. Furthermore, inactivation of oxyR reduced
the virulence of <italic>A. pleuropneumoniae</italic> in a mice
challenge experiment.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
The findings highlight the pivotal role of OxyR in influencing the
virulence mechanisms of <italic>A. pleuropneumoniae</italic>. The
observed effects on various biological activities underscore OxyR as an
essential factor contributing to the pathogenicity of this bacterium.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324760">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324760</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324760">Save
to Pocket</a> </span></p>
<hr />
<h2
id="prevalence-and-resistance-characteristics-of-multidrug-resistant-streptococcus-pneumoniae-isolated-from-the-respiratory-tracts-of-hospitalized-children-in-shenzhen-china">Prevalence
and resistance characteristics of multidrug-resistant Streptococcus
pneumoniae isolated from the respiratory tracts of hospitalized children
in Shenzhen, China</h2>
<p>date: 2024-01-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
PCV13 introduction in China has led to a significant reduction of
vaccine serotype <italic>Streptococcus pneumoniae</italic>. However,
non-vaccine serotypes with highly resistance and invasiveness were often
reported in the post-pneumococcal conjugate vaccine era and there was
regional differences.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
A total of 669 <italic>S. pneumoniae</italic> strains were collected
from the respiratory tracts of hospitalized children at Shenzhen
Children’s Hospital in 2021 and 2022. Antimicrobial resistance (AMR)
characteristics were assessed through antibiotic susceptibility testing
performed with the VITEK 2 compact system. AMR genes and single
nucleotide polymorphisms (SNPs) in pbp1a, pbp2b, and pbp2x were
identified via analysis of whole genome sequencing data. Statistical
examination of the data was conducted employing chi-square and Fisher’s
exact tests.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We found that non-vaccine serotypes strains had accounted for 46.6% of
all the pneumococcal isolated strains. The most common non-vaccine
serotype is 23A, with a prevalence rate of 8.9%, followed by 15A (6.6%),
6E (5.7%), 34 (3.2%), and 15B (2.9%). The multidrug resistance rates
(MDR) of vaccine serotypes were 19F (99.36%), 19A (100%), 23F (98.08%),
6B (100%), and 6C (100%). Meanwhile, the MDR of non-vaccine serotypes
were 15B (100.00%), 6E (100%), 15C (100%), 34 (95.24%), and 23A
(98.31%). Resistance rates of 6E to more than six antibiotic classes
reached 89.47%, which is similar to 19F (83.33%) and 19A (90%). Unique
resistance profiles were also identified for non-vaccine serotypes,
including significantly higher resistance to chloramphenicol in 6E, 15B,
and 15C than in 19F and 19A. Furthermore, through genome sequencing, we
revealed strong correlation of <italic>cat-TC</italic> with
chloramphenicol resistance, <italic>patA/patB</italic> with tetracycline
resistance, <italic>ermB</italic> and <italic>pmrA</italic> with
erythromycin resistance.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The introduction of PCV13 into China from 2017 has led to a shift in the
dominant composition of pneumococcal strains. There has been a notable
rise and spread of multidrug-resistant non-vaccine serotypes among
children. Specifically, the non-vaccine serotype 6E, which was not
widely reported in China previously, has emerged. To comprehend the
resistance mechanisms, it is crucial to further investigate the
molecular and genetic characteristics of these non-vaccine serotypes.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1332472">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1332472</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1332472">Save
to Pocket</a> </span></p>
<hr />
<h2
id="case-report-enhancing-prognosis-in-severe-covid-19-through-human-herpes-virus-coinfection-treatment-strategies">Case
report: Enhancing prognosis in severe COVID-19 through human herpes
virus coinfection treatment strategies</h2>
<p>date: 2024-01-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
In the context of increasing reports of co-infection with coronavirus
disease 2019 (COVID-19), particularly with human herpes viruses (HHVs),
it is important to consider the appropriate treatment options for HHVs
that have been reactivated by COVID-19.
</p>
</sec><sec>
<title>
Case presentation
</title>
<p>
This study presents two cases of severe COVID-19 with HHV co-infection.
The first case involved a critically ill patient with COVID-19
co-infected with herpes simplex virus type 1, confirmed using
metagenomic next-generation sequencing, and another patient with severe
COVID-19 experiencing Epstein-Barr virus (EBV) reactivation, as
evidenced by elevated EBV-DNA levels in the serum. Treatment included
high-dose glucocorticoids and sivelestat sodium, with notable
improvements observed after initiating ganciclovir anti-herpesvirus
therapy.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
This study underscores the significance of recognizing HHV co-infections
in severe COVID-19 cases and highlights the potential of combining
anti-HHV treatment, increased glucocorticoid dosages, and anti-cytokine
storm therapy to enhance prognosis.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320933">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320933</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320933">Save
to Pocket</a> </span></p>
<hr />
<h2
id="stenotrophomonas-maltophilia-complex-insights-into-evolutionary-relationships-global-distribution-and-pathogenicity">Stenotrophomonas
maltophilia complex: insights into evolutionary relationships, global
distribution and pathogenicity</h2>
<p>date: 2024-01-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Stenotrophomonas maltophilia</italic> complex (Smc) comprises
opportunistic Gram-negative bacilli responsible for various nosocomial
infections. Limited data exists concerning its evolutionary lineage,
global prevalence and pathogenicity.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We conducted an extensive genomic analysis on 734 Smc genomes, of which
90 were newly sequenced and isolated from different patients. The
species composition and evolutionary relationships of Smc were examined
using core protein sequence analysis. Pathogenicity evaluation was used
by assays for swimming motility, biofilm formation and identification of
virulence factors. The broth microdilution method was used to evaluate
the drug resistance spectrum of clinical isolates.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Phylogenetic analyses delineated 24 species-level clades, dominated by
<italic>S. maltophilia</italic> (42.8%), <italic>S. sepilia</italic>
(13.6%) and <italic>S. geniculata</italic> (9.9%). Geographically,
strains were primarily distributed in Europe (34.2%), Asia (33.7%) and
North America (24.0%), with intricate global distribution patterns.
Meanwhile, 154 virulence-associated genes and 46 antimicrobial
resistance genes within Smc were identified. These genes encoded span
various functions, including motility, adherence, toxin, RND antibiotic
efflux pumps, beta-lactamases and aminoglycoside-modifying enzymes.
Moreover, significant variations were indicated in swimming motility and
biofilm-forming capability across the different species, with <italic>S.
sepilia</italic> exhibiting superior levels of both traits.
Additionally, no statistically significant discrepancy was detected
among Smc species to other antibiotics, despite the fact that all
<italic>S. geniculata</italic> isolates were resistant to Ceftazidime
and much higher than other species.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Our findings indicate the need to pay increased attention to other
mainstream species of Smc besides <italic>S. maltophilia</italic> in
order to better manage Smc-related infections and tailor effective
treatment strategies.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325379">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325379</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325379">Save
to Pocket</a> </span></p>
<hr />
<h2
id="etiology-of-lower-respiratory-tract-in-pneumonia-based-on-metagenomic-next-generation-sequencing-a-retrospective-study">Etiology
of lower respiratory tract in pneumonia based on metagenomic
next-generation sequencing: a retrospective study</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Pneumonia are the leading cause of death worldwide, and antibiotic
treatment remains fundamental. However, conventional sputum smears or
cultures are still inefficient for obtaining pathogenic
microorganisms.Metagenomic next-generation sequencing (mNGS) has shown
great value in nucleic acid detection, however, the NGS results for
lower respiratory tract microorganisms are still poorly studied.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This study dealt with investigating the efficacy of mNGS in detecting
pathogens in the lower respiratory tract of patients with pulmonary
infections. A total of 112 patients admitted at the First Affiliated
Hospital of Zhengzhou University between April 30, 2018, and June 30,
2020, were enrolled in this retrospective study. The bronchoalveolar
lavage fluid (BALF) was obtained from lower respiratory tract from each
patient. Routine methods (bacterial smear and culture) and mNGS were
employed for the identification of pathogenic microorganisms in BALF.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The average patient age was 53.0 years, with 94.6% (106/112) obtaining
pathogenic microorganism results. The total mNGS detection rate of
pathogenic microorganisms significantly surpassed conventional methods
(93.7% vs. 32.1%, P &lt; 0.05). Notably, 75% of patients (84/112) were
found to have bacteria by mNGS, but only 28.6% (32/112) were found to
have bacteria by conventional approaches. The most commonly detected
bacteria included <italic>Acinetobacter baumannii</italic> (19.6%),
<italic>Klebsiella pneumoniae</italic> (17.9%), <italic>Pseudomonas
aeruginosa</italic> (14.3%), <italic>Staphylococcus faecium</italic>
(12.5%), <italic>Enterococcus faecium</italic> (12.5%), and
<italic>Haemophilus parainfluenzae</italic> (11.6%). In 29.5% (33/112)
of patients, fungi were identified using mNGS, including 23 cases of
<italic>Candida albicans</italic> (20.5%), 18 of <italic>Pneumocystis
carinii</italic> (16.1%), and 10 of <italic>Aspergillus</italic> (8.9%).
However, only 7.1 % (8/112) of individuals were found to have fungi when
conventional procedures were used. The mNGS detection rate of viruses
was significantly higher than the conventional method rate (43.8%
vs. 0.9%, P &lt; 0.05). The most commonly detected viruses included
Epstein–Barr virus (15.2%), cytomegalovirus (13.4%), circovirus (8.9%),
human coronavirus (4.5%), and rhinovirus (4.5%). Only 29.4% (33/112) of
patients were positive, whereas 5.4% (6/112) of patients were negative
for both detection methods as shown by Kappa analysis, indicating poor
consistency between the two methods (P = 0.340; Kappa analysis).
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Significant benefits of mNGS have been shown in the detection of
pathogenic microorganisms in patients with pulmonary infection. For
those with suboptimal therapeutic responses, mNGS can provide an
etiological basis, aiding in precise anti-infective treatment.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1291980">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1291980</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1291980">Save
to Pocket</a> </span></p>
<hr />
<h2
id="possible-modulating-functions-of-probiotic-lactiplantibacillus-plantarum-in-particulate-matter-associated-pulmonary-inflammation">Possible
modulating functions of probiotic Lactiplantibacillus plantarum in
particulate matter-associated pulmonary inflammation</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Pulmonary disease represents a substantial global health burden.
Increased air pollution, especially fine particulate matter
(PM<sub>2.5</sub>) is the most concerned proportion of air pollutants to
respiratory health. PM<sub>2.5</sub> may carry or combine with other
toxic allergens and heavy metals, resulting in serious respiratory
allergies and anaphylactic reactions in the host. Available treatment
options such as antihistamines, steroids, and avoiding
allergens/dust/pollutants could be limited due to certain side effects
and immense exposure to air pollutants, especially in most polluted
countries. In this mini-review, we summarized how PM<sub>2.5</sub>
triggers respiratory hyperresponsiveness and inflammation, and the
probiotic <italic>Lactiplantibacillus plantarum</italic> supplementation
could minimize the risk of the same. <italic>L. plantarum</italic> may
confer beneficial effects in PM<sub>2.5</sub>-associated pulmonary
inflammation due to significant antioxidant potential. We discussed
<italic>L. plantarum’s</italic> effect on PM<sub>2.5</sub>-induced
reactive oxygen species (ROS), inflammatory cytokines, lipid
peroxidation, and DNA damage. Available preclinical evidence shows
<italic>L. plantarum</italic> induces gut-lung axis, SCFA, GABA, and
other neurotransmitter signaling via gut microbiota modulation. SCFA
signals are important in maintaining lung homeostasis and regulating
intracellular defense mechanisms in alveolar cells. However, significant
research is needed in this direction to contemplate <italic>L.
plantarum</italic>’s therapeutic potential in pulmonary allergies.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1290914">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1290914</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1290914">Save
to Pocket</a> </span></p>
<hr />
<h2
id="first-genome-assembly-and-annotation-of-sanghuangporus-weigelae-uncovers-its-medicinal-functions-metabolic-pathways-and-evolution">First
genome assembly and annotation of Sanghuangporus weigelae uncovers its
medicinal functions, metabolic pathways, and evolution</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Sanghuangporus</italic>, also known as “Sanghuang” in China, is
a well-known genus of traditional Chinese medicinal macrofungi. To make
more effective use of <italic>Sanghuangporus</italic> resources, we
completed the first genome assembly and annotation of a monokaryon
strain of <italic>S. weigelae</italic> in the present study. A 33.96-Mb
genome sequence was assembled as 13 contigs, leading to prediction of
9377 protein-coding genes. Phylogenetic and average nucleotide identity
analyses indicated that the <italic>S. weigelae</italic> genome is
closely related to those of other <italic>Sanghuangporus</italic>
species in evolutionary tree, which clustered in one clade. Collinearity
analysis revealed a high level of collinearity of <italic>S.
weigelae</italic> with <italic>S. baumii</italic>, <italic>S.
vaninii</italic>, and <italic>S. sanghuang</italic>. Biosynthesis
pathways potentially involved in medicinal properties, including
terpenoid and polysaccharide synthesis, were identified in <italic>S.
weigelae</italic>, while polysaccharides were identified as the main
medicinal metabolites in <italic>S. weigelae</italic>, with flavonoids
more important in <italic>Sanghuangporus</italic> than other medicinal
mushroom groups. Genes encoding 332 carbohydrate-active enzymes were
identified in the <italic>S. weigelae</italic> genome, including major
glycoside hydrolases and glycosyltransferases predicted, revealing the
robust lignocellulose degradation capacity of <italic>S.
weigelae</italic>. Further, 130 genes, clustered in seven classes were
annotated to encode cytochromes P450 in the <italic>S. weigelae</italic>
genome. Overall, our results reveal the remarkably medicinal capacity of
<italic>S. weigelae</italic> and provide new insights that will inform
the study of evolution and medicinal application of <italic>S.
weigelae</italic>. The data are a reference resource for the formulation
of scientific and rational ecological protection policies for
<italic>Sanghuangporus</italic> species.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325418">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325418</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1325418">Save
to Pocket</a> </span></p>
<hr />
<h2
id="xenophagy-receptors-optn-and-p62-and-autophagy-modulator-dram1-independently-promote-the-zebrafish-host-defense-against-mycobacterium-marinum">Xenophagy
receptors Optn and p62 and autophagy modulator Dram1 independently
promote the zebrafish host defense against Mycobacterium marinum</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Anti-bacterial autophagy, also known as xenophagy, is a crucial innate
immune process that helps maintain cellular homeostasis by targeting
invading microbes. This defense pathway is widely studied in the context
of infections with mycobacteria, the causative agents of human
tuberculosis and tuberculosis-like disease in animal models. Our
previous work in a zebrafish tuberculosis model showed that host defense
against <italic>Mycobacterium marinum</italic> (Mm) is impaired by
deficiencies in xenophagy receptors, optineurin (Optn) or sequestome 1
(p62), and Damage-regulated autophagy modulator 1 (Dram1). However, the
interdependency of these receptors and their interaction with Dram1
remained unknown. In the present study, we used single and double
knockout zebrafish lines in combination with overexpression experiments.
We show that Optn and p62 can compensate for the loss of each other’s
function, as their overexpression restores the infection susceptibility
of the mutant phenotypes. Similarly, Dram1 can compensate for
deficiencies in Optn and p62, and, vice versa, Optn and p62 compensate
for the loss of Dram1, indicating that these xenophagy receptors and
Dram1 do not rely on each other for host defense against Mm. In
agreement, Dram1 overexpression in <italic>optn/p62</italic> double
mutants restored the interaction of autophagosome marker Lc3 with Mm.
Finally, <italic>optn/p62</italic> double mutants displayed more severe
infection susceptibility than the single mutants. Taken together, these
results suggest that Optn and p62 do not function downstream of each
other in the anti-mycobacterial xenophagy pathway, and that the
Dram1-mediated defense against Mm infection does not rely on specific
xenophagy receptors.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1331818">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1331818</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1331818">Save
to Pocket</a> </span></p>
<hr />
<h2
id="eliminating-the-invading-extracellular-and-intracellular-fnbp-bacteria-from-respiratory-epithelial-cells-by-autophagy-mediated-through-fnbp-fn-integrin-α5β1-axis">Eliminating
the invading extracellular and intracellular FnBp+ bacteria from
respiratory epithelial cells by autophagy mediated through
FnBp-Fn-Integrin α5β1 axis</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
We previously found that the respiratory epithelial cells could
eliminate the invaded <italic>group A streptococcus</italic> (GAS)
through autophagy induced by binding a fibronectin (Fn) binding protein
(FnBp) expressed on the surface of GAS to plasma protein Fn and its
receptor integrin α5β1 of epithelial cells. Is autophagy initiated by
FnBp<sup>+</sup> bacteria via FnBp-Fn-Integrin α5β1 axis a common event
in respiratory epithelial cells?
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We chose <italic>Staphylococcus aureus</italic> (<italic>S. aureus/S.
a</italic>) and <italic>Listeria monocytogenes</italic> (<italic>L.
monocytogenes/L. m</italic>) as representatives of extracellular and
intracellular FnBp<sup>+</sup> bacteria, respectively. The FnBp of them
was purified and the protein function was confirmed by western blot,
viable bacteria count, confocal and pull-down. The key molecule
downstream of the action axis was detected by IP, mass spectrometry and
bio-informatics analysis.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We found that different FnBp from both <italic>S. aureus</italic> and
<italic>L. monocytogenes</italic> could initiate autophagy through
FnBp-Fn-integrin α5β1 axis and this could be considered a universal
event, by which host tries to remove invading bacteria from epithelial
cells. Importantly, we firstly reported that S100A8, as a key molecule
downstream of integrin β1 chain, is highly expressed upon activation of
integrin α5β1, which in turn up-regulates autophagy.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
Various FnBp from FnBp<sup>+</sup> bacteria have the ability to initiate
autophagy via FnBp-Fn-Integrin α5β1 axis to promote the removal of
invading bacteria from epithelial cells in the presence of fewer
invaders. S100A8 is a key molecule downstream of Integrin α5β1 in this
autophagy pathway.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324727">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324727</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324727">Save
to Pocket</a> </span></p>
<hr />
<h2
id="lipid-nanoparticles-based-on-natural-matrices-with-activity-against-multidrug-resistant-bacterial-species">Lipid
nanoparticles based on natural matrices with activity against multidrug
resistant bacterial species</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Lately, the bacterial multidrug resistance has been a reason to public
health concerning around world. The development of new pharmacology
therapies against infections caused by multidrug-resistant bacteria is
urgent. In this work, we developed 10 NLC formulations composed of
essential oils (EO), vegetable butter and surfactant. The formulations
were evaluated for long-term and thermal cycling stability studies in
terms of (particle size, polydispersion index and Zeta potential).
<italic>In vitro</italic> antimicrobial assays were performed using disk
diffusion test and by the determination of the minimum inhibitory
concentration (MIC) performed with fresh and a year-old NLC. The most
promising system and its excipients were structurally characterized
through experimental methodologies (FTIR-ATR, DSC and FE-SEM). Finally,
this same formulation was studied through nanotoxicity assays on the
chicken embryo model, analyzing different parameters, as viability and
weight changes of embryos and annexes. All the developed formulations
presented long-term physicochemical and thermal stability. The
formulation based on cinnamon EO presented <italic>in vitro</italic>
activity against strains of <italic>Acinetobacter baumannii</italic>,
<italic>Klebsiella pneumoniae</italic> and <italic>Pseudomonas
aeruginosa</italic> isolated from humans and <italic>in vivo</italic>
biocompatibility. Considering these promising results, such system is
able to be further tested on <italic>in vivo</italic> efficacy assays.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1328519">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1328519</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1328519">Save
to Pocket</a> </span></p>
<hr />
<h2
id="antibody-reactive-immunomes-of-ehrlichia-chaffeensis-and-e.-canis-are-diverse-and-defined-by-conformational-antigenic-determinants">Antibody
reactive immunomes of Ehrlichia chaffeensis and E. canis are diverse and
defined by conformational antigenic determinants</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
For decades, the defined antibody reactive proteins of <italic>Ehrlichia
chaffeensis</italic> and <italic>E. canis</italic> were limited to a
small group with linear antibody epitopes. Recently, our laboratory has
utilized an immunomics-based approach to rapidly screen and identify
undefined <italic>Ehrlichia chaffeensis</italic> and <italic>E.
canis</italic> antigenic proteins and antibody epitopes. In this study,
we analyzed the remaining portion (~50%) of the <italic>E.
chaffeensis</italic> and <italic>E. canis</italic> proteomes
(<italic>n</italic> = 444 and <italic>n =</italic> 405 proteins,
respectively), that were not examined in previous studies, to define the
complete immunomes of these important pathogens. Almost half of the
<italic>E. chaffeensis</italic> proteins screened (196/444) reacted with
antibodies in convalescent HME patient sera, while only 43 <italic>E.
canis</italic> proteins reacted with CME dog sera. New major
immunoreactive proteins were identified in <italic>E.
chaffeensis</italic> (<italic>n</italic> = 7) and <italic>E.
canis</italic> (<italic>n</italic> = 1), increasing the total number of
<italic>E. chaffeensis</italic> (n = 14) and <italic>E. canis</italic>
proteins (n = 18) that exhibited antibody reactivity comparable to
well-defined major antigenic proteins (TRP120 and TRP19). All of the
<italic>E. chaffeensis</italic> but only some <italic>E. canis</italic>
major immunoreactive proteins contained major conformation-dependent
antibody epitopes. The <italic>E. chaffeensis</italic> immunoreactive
proteins were generally small (&lt; 250 amino acids; ~27kDa) and the
<italic>E. canis</italic> proteins were slightly larger (&gt; 320 amino
acids; ~35 kDa). The majority of these new <italic>Ehrlichia</italic>
major immunoreactive proteins were predicted to be type I secreted
effectors, some of which contained transmembrane domains.
Characterization of the immunomes of <italic>E. chaffeensis</italic> and
<italic>E. canis</italic> and understanding the host specific
<italic>Ehrlichia</italic> immune responses will facilitate
identification of protective antigens and define the biophysical epitope
characteristics vital to effective vaccine development for the
ehrlichioses.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321291">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321291</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321291">Save
to Pocket</a> </span></p>
<hr />
<h2
id="clinical-features-and-risk-factors-of-liver-injury-in-patients-with-chlamydia-psittaci-pneumonia--a-retrospective-analysis">Clinical
features and risk factors of liver injury in patients with Chlamydia
psittaci pneumonia- a retrospective analysis</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Research into the effects of <italic>Chlamydia psittaci</italic>
pneumonia on the liver has emerged in the last few years. However, no
studies have systematically described liver injury in patients with
psittacosis. We present the first report on the clinical features and
risk factors of liver injury in patients with <italic>Chlamydia
psittaci</italic> pneumonia.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We retrospectively collected the clinical parameters for 46 patients
with <italic>Chlamydia psittaci</italic> pneumonia admitted to Jinhua
Central Hospital from January 2019 to February 2023. We analyzed the
liver function parameters and summarized the clinical characteristics
and risk factors of liver injury.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Among the 46 patients, 39 (84.8%) had abnormal liver function, and 23
(50.0%) had liver injury. The ratio of patients with a history of
alcohol consumption (39.1% vs. 4.3%, <italic>P</italic> =0.004) or
severe pneumonia (56.5% vs. 26.1%, <italic>P</italic> =0.036) was higher
in the liver injury group compared with the non-liver injury group.
Laboratory tests showed higher lactate dehydrogenase (LDH) levels in the
liver injury group (<italic>P &lt;</italic>0.001). The optimal cut-off
LDH level associated with liver injury was 473 IU/L as determined by ROC
curve analysis. Furthermore, multivariate logistic regression analysis
demonstrated that a history of alcohol consumption (odds ratio [OR] =
11.251; 95% confidence interval [CI] = 1.022 ~ 123.897,
<italic>P</italic> =0.048) and an LDH level of ≥ 473IU/L (OR = 11.635,
95% CI = 1.832 ~ 73.869, <italic>P</italic> =0.009) were independent
risk factors for liver injury.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
A history of alcohol consumption and an LDH level of over 473 IU/L are
independent risk factors for <italic>Chlamydia psittaci</italic>
pneumonia-related liver injury. It is recommended that particular
attention be given to monitoring and evaluating liver function
parameters when treating patients with <italic>Chlamydia
psittaci</italic> pneumonia who have a high LDH level and history of
alcohol consumption.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320758">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320758</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320758">Save
to Pocket</a> </span></p>
<hr />
<h2
id="characterization-of-nontyphoidal-salmonella-strains-from-a-tertiary-hospital-in-china-serotype-diversity-multidrug-resistance-and-genetic-insights">Characterization
of nontyphoidal Salmonella strains from a tertiary hospital in China:
serotype diversity, multidrug resistance, and genetic insights</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Objective
</title>
<p>
Nontyphoidal <italic>Salmonella</italic> is a significant public health
concern due to its ability to cause foodborne illnesses worldwide. This
study aims to characterize the nontyphoidal <italic>Salmonella</italic>
strains isolated from patients in China.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
A total of 19 nontyphoidal <italic>Salmonella</italic> strains were
characterized through serovar identification, antimicrobial
susceptibility testing (AST), biofilm formation assessment. Genetic
relatedness was determined using pulsed-field gel electrophoresis
(PFGE). WGS was employed to decipher the resistance mechanism and to
contextualize the <italic>S.</italic> serovar Mbandaka strains among
previously sequenced isolates in China. The biofilm associated
<italic>mrkA</italic> gene was examined by PCR.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The predominant serovar identified was <italic>S.</italic> Enteritidis,
followed by <italic>S.</italic> Mbandaka, <italic>S.</italic> Thompson,
<italic>S.</italic> Livingston, <italic>S.</italic> Alachua, and
<italic>S.</italic> Infantis. PFGE analysis indicated a notable genetic
similarity among the <italic>S.</italic> Mbandaka isolates. Phylogenetic
analysis suggested that these strains were likely derived from a single
source that had persisted in China for over five years. One multidrug
resistance (MDR) <italic>S.</italic> Enteritidis isolate carried a
highly transferable IncB/O/K/Z plasmid with
<italic>bla</italic><sub>CTX-M-15</sub>. One <italic>S.</italic>
Thompson strain, harboring the <italic>mrkABCDF</italic> operon in an
IncX1 plasmid, isolated from cutaneous lesions, demonstrated robust
biofilm formation. However, no <italic>mrkABCDF</italic> loci were
detected in other strains.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Our study emphasizes the importance of persisted surveillance and prompt
response to <italic>Salmonella</italic> infections to protect public
health. The dissemination of
<italic>bla</italic><sub>CTX-M-15</sub>-harboring IncB/O/K/Z plasmid and
the spread of virulent <italic>mrkABCDF</italic> operon among
<italic>Salmonella</italic> in China and other global regions warrant
close monitoring.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1327092">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1327092</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1327092">Save
to Pocket</a> </span></p>
<hr />
<h2
id="plasmid-content-of-carbapenem-resistant-acinetobacter-baumannii-isolates-belonging-to-five-international-clones-collected-from-hospitals-of-alexandria-egypt">Plasmid
content of carbapenem resistant Acinetobacter baumannii isolates
belonging to five International Clones collected from hospitals of
Alexandria, Egypt</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Multidrug resistant <italic>Acinetobacter baumannii</italic> is one of
the most important nosocomial pathogens worldwide. During the last
decades it has become a major threat for healthcare settings due to the
high antibiotic resistance rates among these isolates. Many resistance
determinants are coded by conjugative or mobilizable plasmids,
facilitating their dissemination. The majority of plasmids harbored by
<italic>Acinetobacter</italic> species are less than 20 Kb, however,
high molecular weight elements are the most clinically relevant since
they usually contain antibiotic resistance genes. The aim of this work
was to describe, classify and determine the genetic content of plasmids
harbored by carbapem resistant <italic>A. baumannii</italic> isolates
belonging to predominant clonal lineages circulating in hospitals from
Alexandria, Egypt. The isolates were subjected to S1-Pulsed Field Gel
Electrophoresis experiments to identify high molecular weight plasmids.
To further analyze the plasmid content and the genetic localization of
the antibiotic resistance genes, isolates were sequenced by Illumina
Miseq and MinION Mk1C and a hybrid assembly was performed using
Unicycler v0.5.0. Plasmids were detected with MOBsuite 3.0.3 and
Copla.py v.1.0 and mapped using the online software Proksee.ca.
Replicase genes were further analyzed through a BLAST against the
<italic>Acinetobacter</italic> Plasmid Typing database. Eleven plasmids
ranging in size from 4.9 to 205.6 Kb were characterized and mapped. All
isolates contained plasmids, and, in many cases, more than two elements
were identified. Antimicrobial resistance genes such as
<italic>bla</italic><sub>OXA-23</sub>,
<italic>bla</italic><sub>GES-like</sub>, <italic>aph(3’)-VI</italic> and
<italic>qacEΔ1</italic> were found in likely conjugative large plasmids;
while virulence determinants such as septicolysin or TonB-dependent
receptors were identified in plasmids of small size. Some of these
resistance determinants were, in turn, located within transposons and
class 1 integrons. Among the identified plasmids, the majority encoded
proteins belonging to the Rep_3 family, but replicases of the RepPriCT_1
(Aci6) family were also identified. Plasmids are of high interest as
antibiotic resistance control tools, since they may be used as genetic
markers for antibiotic resistance and virulence, and also as targets for
the development of compounds that can inhibit transfer processes.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1332736">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1332736</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1332736">Save
to Pocket</a> </span></p>
<hr />
<h2
id="commentary-non-pathogenic-neisseria-species-of-the-oropharynx-as-a-reservoir-of-antimicrobial-resistance-a-cross-sectional-study">Commentary:
Non-pathogenic Neisseria species of the oropharynx as a reservoir of
antimicrobial resistance: a cross-sectional study</h2>
<p>date: 2024-01-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1343608">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1343608</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1343608">Save
to Pocket</a> </span></p>
<hr />
<h2
id="the-cooperation-between-orf-virus-and-staphylococcus-aureus-leads-to-intractable-lesions-in-skin-infection">The
cooperation between orf virus and Staphylococcus aureus leads to
intractable lesions in skin infection</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
A large amount of evidence shows that different kinds of microorganisms
can jointly cope with environmental pressures including cell hosts. For
example, in many cases, it has been found that secondary or mixed
infection of animals caused by ORFV (an epitheliophilic Parapoxvirus)
and bacteria (such as Staphylococcus aureus or Streptococcus) shows a
mutual aid mode that indirectly leads to the deterioration of the
disease. However, the lack of research on the co-pathogenic mechanism,
including how to hijack and destroy the cell host in the pathological
microenvironment, has hindered the in-depth understanding of the
pathogenic process and consequences of this complex infection and the
development of clinical treatment methods. Here, we summarized the
current strategies of trapping cell hosts together, based on the
previously defined ORFV-Host (O-H) system. The opportunistic invasion of
<italic>S. aureus</italic> destroyed the delicate dynamic balance of the
O-H, thus aggravating tissue damage through bacterial products (mediated
by Agr), even causing sepsis or inducing cytokine storms. In fact, the
virus products from its adaptive regulatory system (VARS) weaken the
immune attacks and block molecular pathways, so that <italic>S.
aureus</italic> can settle there more smoothly, and the toxins can
penetrate into local tissues more quickly. This paper focuses on the
main challenges faced by cell hosts in dealing with mixed infection,
which provides a starting point for us to deal with this disease in the
future.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1213694">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1213694</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1213694">Save
to Pocket</a> </span></p>
<hr />
<h2
id="probiotics-for-functional-constipation-in-children-an-overview-of-overlapping-systematic-reviews">Probiotics
for functional constipation in children: an overview of overlapping
systematic reviews</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
This overview of systematic reviews (SRs) and meta-analysis (MAs) aimed
to systematically collate, appraise and synthesize evidence of
probiotics for functional constipation (FC) in children.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
SRs/MAs of probiotics for FC in children were systematic identified by
searching Cochrane Library, PubMed, Embase, and Web of science.
Assessment of Multiple Systematic Reviews 2 (AMSTAR-2), Preferred
Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA), and
Grading of Recommendations, Assessment, Development, and Evaluation
(GRADE) were unitized by two reviewers independently to assess the
methodological quality, reporting quality, and quality of evidence,
respectively.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Seven SRs/MAs met the eligibility criteria and were included in this
study. According to AMSTAR-2, a very low methodological quality
assessment was given to the included SRs/MAs due to the limitations of
items 2, 4 and 7. For the PRISMA statement, the overall quality of
reporting was unsatisfactory due to the lack of reporting on protocol,
risk of bias across studies, synthesis of results, and additional
analysis. According to GRADE, the quality of evidence for outcomes was
rated as very low to moderate.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
Probiotics may be beneficial in improving FC in children. Because of
limitations and inconsistent conclusions, further rigorous, normative
and comprehensive SRs/MAs are needed to provide robust evidence for
definitive conclusions.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1323521">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1323521</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1323521">Save
to Pocket</a> </span></p>
<hr />
<h2
id="anti-obesity-effects-by-parasitic-nematode-trichinella-spiralis-total-lysates">Anti-obesity
effects by parasitic nematode (Trichinella spiralis) total lysates</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Obesity is an inducible factor for the cause of chronic diseases and is
described by an increase in the size and number of adipocytes that
differentiate from precursor cells (preadipocytes). Parasitic helminths
are the strongest natural trigger of type 2 immune system, and several
studies have showed that helminth infections are inversely correlated
with metabolic syndromes.
</p>
</sec><sec>
<title>
Methodology/Principal findings
</title>
<p>
To investigate whether helminth-derived molecules have therapeutic
effects on high-fat diet (HFD)-induced obesity, we isolated total
lysates from <italic>Trichinella spiralis</italic> muscle larvae. We
then checked the anti-obesity effect after intraperitoneal
administration and intraoral administration of total lysate from
<italic>T. spiralis</italic> muscle larvae in a diet-induced obesity
model. <italic>T. spiralis</italic> total lysates protect against
obesity by inhibiting the proinflammatory response and/or enhancing M2
macrophages. In addition, we determined the effects of total lysates
from <italic>T. spiralis</italic> muscle larvae on anti-obesity
activities in 3T3-L1 preadipocytes by investigating the expression
levels of key adipogenic regulators, including peroxisome
proliferator-activated receptor gamma (PPARγ), CCAAT-enhancer-binding
protein alpha (C/EBPα) and adipocyte protein 2 (aP2). Oil Red O staining
showed that the total lysates from <italic>T. spiralis</italic> muscle
larvae decreased the differentiation of 3T3-L1 preadipocytes by
decreasing the number of lipid droplets. In addition, the production
levels of proinflammatory cytokines IL-1β, IL-6, IFN-γ and TNF-α were
examined by enzyme-linked immunosorbent assay (ELISA). <italic>T.
spiralis</italic> total lysates decreased intracellular lipid
accumulation and suppressed the expression levels of PPARγ, C/EBPα and
aP2.
</p>
</sec><sec>
<title>
Conclusion/Significance
</title>
<p>
These results show that <italic>T. spiralis</italic> total lysate
significantly suppresses the symptoms of obesity in a diet- induced
obesity model and 3T3-L1 cell differentiation and suggest that it has
potential for novel anti-obesity therapeutics.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1285584">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1285584</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1285584">Save
to Pocket</a> </span></p>
<hr />
<h2
id="unravelling-staphylococcal-small-colony-variants-in-cardiac-implantable-electronic-device-infections-clinical-characteristics-management-and-genomic-insights">Unravelling
staphylococcal small-colony variants in cardiac implantable electronic
device infections: clinical characteristics, management, and genomic
insights</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Objectives
</title>
<p>
Staphylococcal small-colony variants (SCVs) are common in cardiac
implantable electronic device (CIED) infections. This is the first
retrospective and multi-case study on CIED infections due to
staphylococcal SCVs, aiming to provide a theoretical basis for the
clinical management of CIED and device-related infections caused by
staphylococcal SCVs.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Ninety patients with culture positive CIED infections were enrolled
between 2021 and 2022. We compared the demographic and clinical
characteristics of patients with and without SCVs and performed genomic
studies on SCVs isolates.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Compared to patients without SCVs, those with SCVs had a longer primary
pacemaker implantation time and were more likely to have a history of
device replacement and infection. They showed upregulated inflammatory
indicators, especially higher NEUT% (52.6 vs. 26.8%, <italic>P</italic>
= 0.032) and they had longer hospital stays (median 13 vs. 12 days,
<italic>P</italic> = 0.012). Comparative genomics analysis was performed
on <italic>Staphylococcus epidermidis</italic> wild-type and SCVs. Some
genes were identified, including <italic>aap</italic>, genes encoding
adhesin, CHAP domain-containing protein, LPXTG cell wall anchor
domain-containing protein, and YSIRK-type signal peptide-containing
protein.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Staphylococcal SCVs affect the clinical characteristics of CIED
infections. The process of staphylococcal SCVs adherence, biofilm
formation, and interaction with neutrophils play a vital role.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321626">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321626</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321626">Save
to Pocket</a> </span></p>
<hr />
<h2
id="in-vitro-antibacterial-activity-of-antiretroviral-drugs-on-key-commensal-bacteria-from-the-human-microbiota">In
vitro antibacterial activity of antiretroviral drugs on key commensal
bacteria from the human microbiota</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Antiretroviral therapy has improved life expectancy in HIV-infected
patients. However, people living with HIV under antiretroviral therapy
are at higher risks of developing chronic complications and acquiring
multidrug resistant bacteria than healthy population. These factors have
been associated with shifts in gut microbiome composition and immune
activation. It is unclear how antiretroviral drugs affect gut microbiota
composition, but it has been observed that antiretroviral treatment is
not able to fully restore gut health after HIV infection. Additionally,
some antiretroviral drugs have shown antibacterial activity suggesting
that these drugs could have a direct impact on the human microbiome
composition.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We determined the <italic>in vitro</italic> antibacterial activity of 16
antiretroviral drugs against a set of key clinically relevant and human
commensal bacterial strains.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Our results demonstrate that 5 antiretroviral drugs have <italic>in
vitro</italic> antibacterial activity against gut and vaginal human
commensal bacteria. Zidovudine has antibacterial activity against
<italic>Escherichia coli</italic>, <italic>Klebsiella
pneumoniae</italic> and <italic>Prevotella bivia</italic>, abacavir
against <italic>Gardnerella vaginalis</italic>, efavirenz against
<italic>G. vaginalis</italic> and <italic>P. bivia</italic> and
bictegravir against <italic>Enterococcus</italic> spp. and <italic>G.
vaginalis</italic>. Moreover, we describe for the first time that
elvitegravir has antibacterial activity against <italic>G.
vaginalis</italic> and <italic>P. bivia</italic> and, most importantly,
against vancomycin-resistant <italic>Enterococcus</italic> spp. and
methicillin-resistant <italic>Staphylococcus aureus</italic> strains
with MIC values of 4-16 and 4 µg/mL, respectively showing high level of
effectiveness against the tested multidrug-resistant bacteria.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Our results underscore that some antiretroviral drugs may influence the
human microbiota composition. In addition, we report the potential use
of elvitegravir to treat multidrug-resistant Gram-positive bacteria
warranting the need of clinical studies to repurpose this antiretroviral
drug.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1306430">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1306430</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1306430">Save
to Pocket</a> </span></p>
<hr />
<h2
id="chronic-blue-light-emitting-diode-exposure-harvests-gut-dysbiosis-related-to-cholesterol-dysregulation">Chronic
blue light-emitting diode exposure harvests gut dysbiosis related to
cholesterol dysregulation</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Night shift workers have been associated with circadian dysregulation
and metabolic disorders, which are tightly coevolved with gut
microbiota. The chronic impacts of light-emitting diode (LED) lighting
at night on gut microbiota and serum lipids were investigated. Male
C57BL/6 mice were exposed to blue or white LED lighting at Zeitgeber
time 13.5-14 (ZT; ZT0 is the onset of “lights on” and ZT12 is the
“lights off” onset under 12-hour light, 12-hour dark schedule). After 33
weeks, only the high irradiance (7.2 J/cm<sup>2</sup>) of blue LED light
reduced the alpha diversity of gut microbiota. The high irradiance of
white LED light and the low irradiance (3.6 J/cm<sup>2</sup>) of both
lights did not change microbial alpha diversity. However, the low
irradiance, but not the high one, of both blue and white LED
illuminations significantly increased serum total cholesterol (TCHO),
but not triglyceride (TG). There was no significant difference of
microbial abundance between two lights. The ratio of beneficial to
harmful bacteria decreased at a low irradiance but increased at a high
irradiance of blue light. Notably, this ratio was negatively correlated
with serum TCHO but positively correlated with bile acid biosynthesis
pathway. Therefore, chronic blue LED lighting at a high irradiance may
harvest gut dysbiosis in association with decreased alpha diversity and
the ratio of beneficial to harmful bacteria to specifically dysregulates
TCHO metabolism in mice. Night shift workers are recommended to be avoid
of blue LED lighting for a long and lasting time.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320713">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320713</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1320713">Save
to Pocket</a> </span></p>
<hr />
<h2
id="real-time-detection-of-seneca-valley-virus-by-one-tube-rpa-crisprcas12a-assay">Real-time
detection of Seneca Valley virus by one-tube RPA-CRISPR/Cas12a
assay</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Senecavirus A (SVA) is a highly contagious virus that causes vesicular
disease in pigs. At present, laboratory detection methods, such as virus
isolation and polymerase chain reaction (PCR), required precision
instruments and qualified personnel, making them unsuitable for
point-of-care tests (POCT). Fortunately, the emergence of CRISPR/Cas
system has provided new opportunities for fast and efficient pathogen
detection.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This study successfully developed a precise and sensitive detection
platform for diagnosing SVA by combining the CRISPR system with
recombinase polymerase amplification (RPA).
</p>
</sec><sec>
<title>
Results
</title>
<p>
The minimum detection limit of the assay was 10 copies of the SVA
genome. Meanwhile, the assay demonstrated high specificity. To validate
the effectiveness of this system, we tested 85 swine clinical samples
and found that the fluorescence method had a 100% coincidence rate
compared to RT-qPCR.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Overall, the RPA-CRISPR/Cas12a assay established in our study is a
highly effective method for detecting SVA and holds great potential for
practical applications in the resource-limited settings.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1305222">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1305222</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1305222">Save
to Pocket</a> </span></p>
<hr />
<h2
id="ghrelin-regulating-liver-activity-and-its-potential-effects-on-liver-fibrosis-and-echinococcosis">Ghrelin
regulating liver activity and its potential effects on liver fibrosis
and Echinococcosis</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Ghrelin widely exists in the central nervous system and peripheral
organs, and has biological activities such as maintaining energy
homeostasis, regulating lipid metabolism, cell proliferation, immune
response, gastrointestinal physiological activities, cognition, memory,
circadian rhythm and reward effects. In many benign liver diseases, it
may play a hepatoprotective role against steatosis, chronic
inflammation, oxidative stress, mitochondrial dysfunction, endoplasmic
reticulum stress and apoptosis, and improve liver cell autophagy and
immune response to improve disease progression. However, the role of
Ghrelin in liver <italic>Echinococcosis</italic> is currently unclear.
This review systematically summarizes the molecular mechanisms by which
Ghrelin regulates liver growth metabolism, immune-inflammation,
fibrogenesis, proliferation and apoptosis, as well as its protective
effects in liver fibrosis diseases, and further proposes the role of
Ghrelin in liver <italic>Echinococcosis</italic> infection. During the
infectious process, it may promote the parasitism and survival of
parasites on the host by improving the immune-inflammatory
microenvironment and fibrosis state, thereby accelerating disease
progression. However, there is currently a lack of targeted <italic>in
vitro</italic> and <italic>in vivo</italic> experimental evidence for
this viewpoint.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324134">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324134</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1324134">Save
to Pocket</a> </span></p>
<hr />
<h2
id="multiple-cross-displacement-amplification-combined-with-nanoparticle-based-lateral-flow-biosensor-for-rapid-and-sensitive-detection-of-epstein-barr-virus">Multiple
cross displacement amplification combined with nanoparticle-based
lateral flow biosensor for rapid and sensitive detection of Epstein-Barr
virus</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Epstein-Barr virus (EBV) is a highly dangerous virus that is globally
prevalent and closely linked to the development of nasopharyngeal cancer
(NPC). Plasma EBV DNA analysis is an effective strategy for early
detection, prognostication and monitoring of treatment response of NPC.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Here, we present a novel molecular diagnostic technique termed
EBV-MCDA-LFB, which integrates multiple cross displacement amplification
(MCDA) with nanoparticle-based lateral flow (LFB) to enable simple,
rapid and specific detection of EBV. In the EBV-MCDA-LFB system, a set
of 10 primers was designed for rapidly amplifying the highly conserved
tandem repeat BamHI-W region of the EBV genome. Subsequently, the LFB
facilitate direct assay reading, eliminating the use of extra
instruments and reagents.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The outcomes showed that the 65°C within 40 minutes was the optimal
reaction setting for the EBV-MCDA system. The sensitivity of
EBV-MCDA-LFB assay reached 7 copies per reaction when using EBV
recombinant plasmid, and it showed 100% specificity without any
cross-reactivity with other pathogens. The feasibility of the
EBV-MCDA-LFB method for EBV detection was successfully validated by 49
clinical plasma samples. The complete detection process, consisting of
rapid template extraction (15 minutes), MCDA reaction (65°C for 40
minutes), and LFB result reading (2 minutes), can be finalized within a
60-minutes duration.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
EBV-MCDA-LFB assay designed here is a fast, extremely sensitive and
specific technique for detecting EBV in field and at the point-of-care
(PoC), which is especially beneficial for countries and regions with a
high prevalence of the disease and limited economic resources.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321394">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321394</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1321394">Save
to Pocket</a> </span></p>
<hr />
<h2
id="the-critical-role-of-neutrophil-endothelial-cell-interactions-in-sepsis-new-synergistic-approaches-employing-organ-on-chip-omics-immune-cell-phenotyping-and-in-silico-modeling-to-identify-new-therapeutics">The
critical role of neutrophil-endothelial cell interactions in sepsis: new
synergistic approaches employing organ-on-chip, omics, immune cell
phenotyping and in silico modeling to identify new therapeutics</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Sepsis is a global health concern accounting for more than 1 in 5 deaths
worldwide. Sepsis is now defined as life-threatening organ dysfunction
caused by a dysregulated host response to infection. Sepsis can develop
from bacterial (gram negative or gram positive), fungal or viral (such
as COVID) infections. However, therapeutics developed in animal models
and traditional <italic>in vitro</italic> sepsis models have had little
success in clinical trials, as these models have failed to fully
replicate the underlying pathophysiology and heterogeneity of the
disease. The current understanding is that the host response to sepsis
is highly diverse among patients, and this heterogeneity impacts immune
function and response to infection. Phenotyping immune function and
classifying sepsis patients into specific endotypes is needed to develop
a personalized treatment approach. Neutrophil-endothelium interactions
play a critical role in sepsis progression, and increased neutrophil
influx and endothelial barrier disruption have important roles in the
early course of organ damage. Understanding the mechanism of
neutrophil-endothelium interactions and how immune function impacts this
interaction can help us better manage the disease and lead to the
discovery of new diagnostic and prognosis tools for effective
treatments. In this review, we will discuss the latest research
exploring how <italic>in silico</italic> modeling of a synergistic
combination of new organ-on-chip models incorporating human
cells/tissue, omics analysis and clinical data from sepsis patients will
allow us to identify relevant signaling pathways and characterize
specific immune phenotypes in patients. Emerging technologies such as
machine learning can then be leveraged to identify druggable therapeutic
targets and relate them to immune phenotypes and underlying infectious
agents. This synergistic approach can lead to the development of new
therapeutics and the identification of FDA approved drugs that can be
repurposed for the treatment of sepsis.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1274842">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1274842</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1274842">Save
to Pocket</a> </span></p>
<hr />
<h2
id="anti-herpes-simplex-virus-activities-and-mechanisms-of-marine-derived-compounds">Anti-herpes
simplex virus activities and mechanisms of marine derived compounds</h2>
<p>date: 2024-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Herpes simplex virus (HSV) is the most widely prevalent herpes virus
worldwide, and the herpetic encephalitis and genital herpes caused by
HSV infection have caused serious harm to human health all over the
world. Although many anti-HSV drugs such as nucleoside analogues have
been ap-proved for clinical use during the past few decades, important
issues, such as drug resistance, toxicity, and high cost of drugs,
remain unresolved. Recently, the studies on the anti-HSV activities of
marine natural products, such as marine polysaccharides, marine peptides
and microbial secondary metabolites are attracting more and more
attention all over the world. This review discusses the recent progress
in research on the anti-HSV activities of these natural compounds
obtained from marine organisms, relating to their structural features
and the structure-activity relationships. In addition, the recent
findings on the different anti-HSV mechanisms and molecular targets of
marine compounds and their potential for therapeutic application will
also be summarized in detail.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302096">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302096</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302096">Save
to Pocket</a> </span></p>
<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
