---
title: health 2024.04
updated: 2024-01-24 07:16:25
---

# health 2024.04

(date: 2024-01-24 07:16:25)

---

## Clinical characterization of a hypersensitivity mixed bacterial and fungal dermatitis in a translational model of porcine NASH

date: 2024-01-24, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>The development of animal models of chronic liver disease via diet modification is a promising avenue for translational research but can lead to unexpected side effects that impact model adoption. While these side effects are well characterized in rodent models of nonalcoholic steatohepatitis (NASH), limited knowledge of these effects exists for novel porcine models of NASH. To close this gap, the present study investigates the side effects of diet-based NASH induction in pigs, with a systematic analysis of the pathologic mechanisms underlying dermatitis development and evaluation of treatment approaches.</p></sec><sec><title>Method</title><p>Twelve pigs (10 large domestic pigs, 2 Goettingen minipigs) were fed a methionine- and choline-deficient, high-fat diet for 8 weeks to induce NASH. A retrospective review of each animal’s clinical record was performed to identify the side effects of the diet. Following the identification of diet-associated dermatitis, severity was judged by using a novel gradation system that characterized the individual lesions and body regions resulting in a cumulative evaluation. In addition to this clinical assessment, the etiology of the dermatitis was investigated via histopathologic and microbiologic testing. Furthermore, the success of prophylactic and therapeutic treatment approaches was evaluated by considering dermatitis development and clinical course.</p></sec><sec><title>Results</title><p>All study animals demonstrated unexpected side effects of the methionine- and choline-deficient, high fat diet. In addition to marked dermatitis, study pigs showed impaired weight gain and developed steatorrhea and anemia. Based on the skin gradation system, five animals developed severe dermatitis, four animals moderate dermatitis, and three animals mild diet-associated dermatitis. Histological and microbiological evaluation of the affected skin showed signs of a hypersensitivity reaction with secondary infection by bacteria and fungi. The analysis showed that preemptive bathing extended the lesion-free duration by nearly 20 days. Furthermore, bathing in combination with a targeted antibiotic treatment represented a helpful treatment approach for diet-associated dermatitis.</p></sec><sec><title>Conclusion</title><p>The provision of a methionine- and choline-deficient, high fat diet represents an effective approach for inducing NASH liver disease in pigs but predisposes study animals to multiple side effects. These side effects are universal to animals on study but can be adequately managed and do not represent a significant limitation of this model.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1277045">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1277045</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1277045">Save to Pocket</a>
</span>

---

## Rational design of antibodies and development of a novel method for (1–3)-β-D glucan detection as an alternative to Limulus amebocyte lysate assay

date: 2024-01-24, from: Frontiers in Cellular and Infection Microbiology

<p>With advances in medicine, increasing medical interventions have increased the risk of invasive fungal disease development. (1-3)-β-D glucan (BDG) is a common fungal biomarker in serological tests. However, the scarcity of Limulus resources for BDG detection poses a challenge. This study addresses the need for an alternative to Limulus amebocyte lysate by using BDG mutant antibody for chemiluminescence detection. The wild-type BDG antibody was obtained by immunizing rabbits. An optimal V52<sup>H</sup>I/N34<sup>L</sup> Y mutant antibody, which has increased 3.7-fold of the testing efficiency compared to the wild-type antibody, was first achieved by mutating “hot-spot” residues that contribute to strong non-covalent bonds, as determined by alanine scanning and molecular dynamics simulation. The mutant was then applied to develop the magnetic particle chemiluminescence method. 574 clinical samples were tested using the developed method, with a cutoff value of 95 pg/mL set by Limulus amebocyte lysate. The receiver operating characteristic curve demonstrated an area under the curve of 0.905 (95% CI: 0.880–0.929). Chemiluminescence detected an antigen concentration of 89.98 pg/mL, exhibiting a sensitivity of 83.33% and specificity of 89.76%. In conclusion, the results showed a good agreement with Limulus amebocyte lysate and demonstrated the feasibility of using BDG mutant antibodies for invasive fungal disease diagnosis. The new method based on chemiluminescence for detecting BDG could shorten the sample-to-result time to approximately 30 min, rescue Limulus from being endangered and is resource efficient in terms of equipment and the non-use of a skilled technician.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1322264">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1322264</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1322264">Save to Pocket</a>
</span>

---

## Comparison of metagenomic next-generation sequencing and blood culture for diagnosis of bloodstream infections

date: 2024-01-24, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Objectives</title><p>This study aimed to evaluate the clinical performance of plasma cell-free DNA (cfDNA) next-generation sequencing (NGS) for pathogen detection in patients with sepsis.</p></sec><sec><title>Methods</title><p>A total of 43 pairs of blood and plasma samples form 33 blood culture-positive patients were used as testing samples in metagenomic NGS (mNGS) and NGS of 16S ribosomal RNA gene amplicons (16S rRNA NGS). The results of routine tests, including microbial culture, complete blood count, and biochemical tests, were collected from electronic medical records.</p></sec><sec><title>Results</title><p>Using blood as an mNGS testing sample, the proportion of host DNA was 99.9%, with only three bacteria and no fungi detected. When using plasma in mNGS, the proportion of host DNA was approximately 97%, with 84 bacteria and two fungi detected. Notably, 16S rRNA NGS detected 15 and 16 bacteria in 43 pairs of blood and plasma samples, respectively. Blood culture detected 49 bacteria (23 gram-negative bacilli and 26 gram-positive cocci) and four fungi, with 14 bacteria considered contaminants by clinical microbiologists. For all blood cultures, plasma cfDNA mNGS detected 78.26% (19/23) gram-negative rods, 17% (2/12) gram-positive cocci, and no fungi. Compared to blood cultures, the sensitivity and specificity of plasma cfDNA mNGS for detecting bacteria and fungi were 62.07% and 57.14%, respectively.</p></sec><sec><title>Conclusion</title><p>Compared to blood, plasma is more suitable for the detection of bloodstream infections using mNGS and is less affected by host DNA. The positive detection rate of plasma cfDNA mNGS for bloodstream infections caused by gram-negative bacteria was higher than that caused by gram-positive cocci.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1338861">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1338861</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1338861">Save to Pocket</a>
</span>

---

## The antibacterial activity and mechanism of a novel peptide MR-22 against multidrug-resistant Escherichia coli

date: 2024-01-24, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>Bacterial infections have become serious threats to human health, and the excessive use of antibiotics has led to the emergence of multidrug-resistant (MDR) bacteria. <italic>E. coli</italic> is a human bacterial pathogen, which can cause severe infectious. Antimicrobial peptides are considered the most promising alternative to traditional antibiotics.</p></sec><sec><title>Materials and methods</title><p>The minimum inhibitory concentration (MIC), minimum bactericidal concentration (MBC) and hemolytic activity were determined by the microdilution method. The antimicrobial kinetics of MR-22 against <italic>E. coli</italic> were studied by growth curves and time-killing curves. The cytotoxicity of MR-22 was detected by the CCK-8 assay. The antimicrobial activity of MR-22 in salt, serum, heat and trypsin was determined by the microdilution method. The antimicrobial mechanism of MR-22 against drug-resistant <italic>E. coli</italic> was studied by Scanning Electron Microscope, laser confocal microscopy, and Flow Cytometry. The <italic>in vivo</italic> antibacterial activity of MR-22 was evaluated by the mice model of peritonitis.</p></sec><sec><title>Results and discussion</title><p>In this study, MR-22 is a new antimicrobial peptide with good activity that has demonstrated against MDR <italic>E. coli</italic>. The antimicrobial activity of MR-22 exhibited stability under conditions of high temperature, 10% FBS, and Ca<sup>2+</sup>. However, a decline of the activity was observed in the presence of Na<sup>+</sup>, serum, and trypsin. MR-22 had no significant cytotoxicity or hemolysis <italic>in vitro</italic>. SEM and fluorescent images revealed that MR-22 could disrupt the integrity of cell membrane. DCFH-DA indicated that MR-22 increased the content of reactive oxygen species, while it decreased the content of intracellular ATP. In mice model of peritonitis, MR-22 exhibited potent antibacterial activity <italic>in vivo</italic>. These results indicated that MR-22 is a potential drug candidate against drug-resistant <italic>E. coli</italic>.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334378">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334378</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334378">Save to Pocket</a>
</span>

---

## Editorial: Interrelationships between human microbiology and health of the elderly

date: 2024-01-23, from: Frontiers in Cellular and Infection Microbiology



<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345103">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345103</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345103">Save to Pocket</a>
</span>

---

## Comparative analysis of the intestinal microbiota of black−necked cranes (Grus nigricollis) in different wintering areas

date: 2024-01-22, from: Frontiers in Cellular and Infection Microbiology

<p>Fecal microbiota is essential for host health because it increases digestive effectiveness. The crane species <italic>Grus nigricollis</italic> (<italic>G. nigricollis</italic>) is considered to be near threatened. The fecal microbial composition of crane is less understood, particularly in the Tibet, China. This study was performed to investigate the differences in fecal microbial composition and diversity of crane in different wintering areas using third-generation single-molecule real-time sequencing technology in the Tibet, China. According to the findings, 20 samples were used to generate 936 bacterial amplicon sequence variants (ASVs) and 1,800 fungal ASVs, only 4 bacterial ASVs and 20 fungal ASVs were shared in four distinct locations. Firmicutes were the dominant bacterial phylum in all samples, and Ascomycota and Basidiomycota were the dominant fungal phylum. At the genus level, <italic>Lactobacillus</italic> was the dominant genus in Linzhi City (LZ), Shannan City (SN), and Lasa City (LS), whereas Megamonas was the dominant genus in Rikaze City (RKZ). <italic>Naganishia</italic> and <italic>Mycosphaerella</italic> were the dominant fungal genera in SN and RKZ. <italic>Mycosphaerella</italic> and <italic>Tausonia</italic> were the dominant fungal genera in LZ. <italic>Naganishia</italic> and <italic>Fusarium</italic> were the dominant fungal genera in LS. And the fecal microbial composition varied between the four groups, as shown by the underweighted pair-group method with arithmetic means and principal coordinates analysis. This study offers a theoretical basis for understanding the fecal microbial composition of crane.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302785">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302785</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1302785">Save to Pocket</a>
</span>

---

## Invasive Streptococcus suis isolated in Spain contain a highly promiscuous and dynamic resistome

date: 2024-01-22, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p><italic>Streptococcus suis</italic> is a major pathogen for swine and human. Here we aimed to know the rates of antimicrobial resistance (AMR) in invasive <italic>S. suis</italic> isolates recovered along Spain between 2016 – 2021 and elucidate their genetic origin.</p></sec><sec><title>Methods</title><p>Antibiotic susceptibility testing was performed for 116 isolates of different genetic backgrounds and geographic origins against 18 antibiotics of 9 families. The association between AMR and genotypes and the origin of the isolates were statistically analyzed using Pearson´s chi-square test and the likelihood ratio. The antimicrobial resistant genes were identified by whole genome sequencing analysis and PCR screenings.</p></sec><sec><title>Results</title><p>High AMR rates (&gt;80%) were detected for tetracyclines, spectinomycin, lincosamides, and marbofloxacin, medium (20-40%) for sulphonamides/trimethoprim, tiamulin, penicillin G, and enrofloxacin, and low (&lt; 20%) for florfenicol, and four additional β-lactams. The occurrence of multidrug resistance was observed in 90% of isolates. For certain antibiotics (penicillin G, enrofloxacin, marbofloxacin, tilmicosin, and erythromycin), AMR was significantly associated with particular sequence types (STs), geographic regions, age of pigs, and time course. Whole genome sequencing comparisons and PCR screenings identified 23 AMR genes, of which 19 were previously reported in <italic>S. suis</italic> (<italic>aph</italic>(3’)-IIIa, <italic>sat4, aadE, spw</italic>, aac(6’)-Ie-<italic>aph</italic>(2’’)-Ia, <italic>fexA, optrA, erm</italic>(B), <italic>mef</italic>(A/E), <italic>mrs</italic>(D), <italic>mph</italic>(C), <italic>lnu</italic>(B), <italic>lsa</italic>(E), <italic>vga</italic>(F), <italic>tet</italic>(M), <italic>tet</italic>(O), <italic>tet</italic>(O/W/32/O), <italic>tet</italic>(W)), and 4 were novel (<italic>aph</italic>(2’’)-IIIa, <italic>apmA, erm</italic>(47), <italic>tet</italic>(T)). These AMR genes explained the AMR to spectinomycin, macrolides, lincosamides, tiamulin, and tetracyclines. Several genes were located on mobile genetic elements which showed a variable organization and composition. As AMR gene homologs were identified in many human and animal pathogens, the resistome of <italic>S. suis</italic> has a different phylogenetic origin. Moreover, AMR to penicillin G, fluoroquinolones, and trimethoprim related to mutations in genes coding for target enzymes (<italic>pbp1a, pbp2b, pbp2x, mraY, gyrA</italic>, <italic>parC</italic>, and <italic>dhfr)</italic>. Bioinformatic analysis estimated traits of recombination on target genes, also indicative of gene transfer events.</p></sec><sec><title>Conclusions</title><p>Our work evidences that <italic>S. suis</italic> is a major contributor to AMR dissemination across veterinary and human pathogens. Therefore, control of AMR in <italic>S. suis</italic> should be considered from a One Health approach in regions with high pig production to properly tackle the issue of antimicrobial drug resistance.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1329632">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1329632</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1329632">Save to Pocket</a>
</span>

---

## Red recombination enables a wide variety of markerless manipulation of porcine epidemic diarrhea virus genome to generate recombinant virus

date: 2024-01-22, from: Frontiers in Cellular and Infection Microbiology

<p>Porcine epidemic diarrhea virus (PEDV) is a member of the genera <italic>Alphacoronavirus</italic> that has been associated with acute watery diarrhea and vomiting in swine. Unfortunately, no effective vaccines and antiviral drugs for PEDV are currently available. Reverse genetics systems are crucial tools for these researches. Here, a PEDV full-length cDNA clone was constructed. Furtherly, three PEDV reporter virus plasmids containing red fluorescent protein (RFP), Nano luciferase (Nluc), or green fluorescence protein (GFP) were generated using Red recombination with the GS1783 <italic>E. coli</italic> strain. These reporter-expressing recombinant (r) PEDVs showed similar growth properties to the rPEDV, and the foreign genes were stable to culture up to P9 in Vero cells. Using the Nluc-expressing rPEDV, the replication of PEDV was easily quantified, and a platform for rapid anti-PEDV drug screening was constructed. Among the three drugs, Bergenin, Umifenovir hydrochloride (Arbidol), and <italic>Ganoderma lucidum</italic> triterpenoids (GLTs), we found that GLTs inhibited PEDV replication mainly after the stage of virus “Entry”. Overall, this study will broaden insight into the method for manipulating the PEDV genome and provide a powerful tool for screening anti-PEDV agents.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1338740">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1338740</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1338740">Save to Pocket</a>
</span>

---

## The pediatric gut bacteriome and virome in response to SARS-CoV-2 infection

date: 2024-01-22, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>Since the beginning of the SARS-CoV-2 pandemic in early 2020, it has been apparent that children were partially protected from both infection and the more severe forms of the disease. Many different mechanisms have been proposed to explain this phenomenon, including children’s frequent exposure to other upper respiratory infections and vaccines, and which inflammatory cytokines they are more likely to produce in response to infection. Furthermore, given the presence of SARS-CoV-2 in the intestine and its ability to infect enterocytes, combined with the well described immunomodulatory capabilities of the microbiome, another potential contributing factor may be the presence of certain protective microbial members of the gut microbiota (GM).</p></sec><sec><title>Methods</title><p>We performed shotgun metagenomic sequencing and profiled both the bacteriome and virome of the GM of pediatric SARS-CoV-2 patients compared to healthy, age-matched subjects.</p></sec><sec><title>Results</title><p>We found that, while pediatric patients do share some pro-inflammatory microbial signatures with adult patients, they also possess a distinct microbial signature of protective bacteria previously found to be negatively correlated with SARS-CoV-2 infectivity and COVID-19 severity. COVID-19 was also associated with higher fecal <italic>Cytomegalovirus</italic> load, and with shifts in the relative abundances of bacteriophages in the GM. Furthermore, we address how the preventative treatment of COVID-19 patients with antibiotics, a common practice especially in the early days of the pandemic, affected the bacteriome and virome, as well as the abundances of antimicrobial resistance and virulence genes in these patients. </p></sec><sec><title>Discussion</title><p>To our knowledge, this is the first study to address the bacteriome, virome, and resistome of pediatric patients in response to COVID-19 and to preventative antibiotics use.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1335450">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1335450</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1335450">Save to Pocket</a>
</span>

---

## Isobavachalcone exhibits antifungal and antibiofilm effects against C. albicans by disrupting cell wall/membrane integrity and inducing apoptosis and autophagy

date: 2024-01-22, from: Frontiers in Cellular and Infection Microbiology

<p>Isobavachalcone (IBC) is a natural flavonoid with multiple pharmacological properties. This study aimed to evaluate the efficacy of IBC against planktonic growth and biofilms of <italic>Candida albicans</italic> (<italic>C. albicans</italic>) and the mechanisms underlying its antifungal action. The cell membrane integrity, cell metabolic viability, and cell morphology of <italic>C. albicans</italic> treated with IBC were evaluated using CLSM and FESEM analyses. Crystal violet staining, CLSM, and FESEM were used to assess the inhibition of biofilm formation, as well as dispersal and killing effects of IBC on mature biofilms. RNA-seq combined with apoptosis and autophagy assays was used to examine the mechanisms underlying the antifungal action of IBC. IBC exhibited excellent antifungal activity with 8 μg/mL of MIC for <italic>C. albicans</italic>. IBC disrupted the cell membrane integrity, and inhibited biofilm formation. IBC dispersed mature biofilms and damaged biofilm cells of <italic>C. albicans</italic> at 32 μg/mL. Moreover, IBC induced apoptosis and autophagy-associated cell death of <italic>C. albicans</italic>. The RNA-seq analysis revealed upregulation or downregulation of key genes involved in cell wall synthesis (<italic>Wsc1</italic> and <italic>Fks1</italic>), ergosterol biosynthesis (<italic>Erg3</italic>, and <italic>Erg11</italic>), apoptisis (<italic>Hsp90</italic> and <italic>Aif1</italic>), as well as autophagy pathways (<italic>Atg8</italic>, <italic>Atg13</italic>, and <italic>Atg17</italic>), and so forth, in response to IBC, as evidenced by the experiment-based phenotypic analysis. These results suggest that IBC inhibits <italic>C. albicans</italic> growth by disrupting the cell wall/membrane, caused by the altered expression of genes associated with β-1,3-glucan and ergosterol biosynthesis. IBC induces apoptosis and autophagy-associated cell death by upregulating the expression of <italic>Hsp90</italic>, and altering autophagy-related genes involved in the formation of the Atg1 complex and the pre-autophagosomal structure. Together, our findings provide important insights into the potential multifunctional mechanism of action of IBC.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336773">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336773</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336773">Save to Pocket</a>
</span>

---

## An amplicon-based nanopore sequencing workflow for rapid tracking of avian influenza outbreaks, France, 2020-2022

date: 2024-01-22, from: Frontiers in Cellular and Infection Microbiology

<p>During the recent avian influenza epizootics that occurred in France in 2020/21 and 2021/22, the virus was so contagiousness that it was impossible to control its spread between farms. The preventive slaughter of millions of birds consequently was the only solution available. In an effort to better understand the spread of avian influenza viruses (AIVs) in a rapid and innovative manner, we established an amplicon-based MinION sequencing workflow for the rapid genetic typing of circulating AIV strains. An amplicon-based MinION sequencing workflow based on a set of PCR primers targeting primarily the hemagglutinin gene but also the entire influenza virus genome was developed. Thirty field samples from H5 HPAIV outbreaks in France, including environmental samples, were sequenced using the MinION MK1C. A real-time alignment of the sequences with MinKNOW software allowed the sequencing run to be stopped as soon as enough data were generated. The consensus sequences were then generated and a phylogenetic analysis was conducted to establish links between the outbreaks. The whole sequence of the hemagglutinin gene was obtained for the 30 clinical samples of H5Nx HPAIV belonging to clade 2.3.4.4b. The consensus sequences comparison and the phylogenetic analysis demonstrated links between some outbreaks. While several studies have shown the advantages of MinION for avian influenza virus sequencing, this workflow has been applied exclusively to clinical field samples, without any amplification step on cell cultures or embryonated eggs. As this type of testing pipeline requires only a short amount of time to link outbreaks or demonstrate a new introduction, it could be applied to the real-time management of viral epizootics.</p>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1257586">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1257586</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1257586">Save to Pocket</a>
</span>

---

## Clinical implications of trichomonads detected in bronchoalveolar fluid by metagenomic next-generation sequencing: a multicenter retrospective study

date: 2024-01-22, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Background</title><p>Pulmonary trichomoniasis is considered a neglected disease due to failures in recognizing it, stemming from insensitive microbial methods and a lack of specific clinical features. This study aims to analyze the clinical implications of trichomonads detected in bronchoalveolar lavage fluid (BALF) by metagenomic next-generation sequencing (mNGS).</p></sec><sec><title>Methods</title><p>This multicenter retrospective study included patients diagnosed with pneumonia, admitted to three tertiary hospitals in China from July 2018 to September 2022, with trichomonads detected in BALF through mNGS. The analysis covered demographics, comorbidities, symptoms, laboratory findings, mNGS results, clinical treatment, and outcomes of these patients.</p></sec><sec><title>Results</title><p>A total of 17 patients were enrolled, comprising 14 males and 3 females. Trichomonas tenax and Trichomonas vaginalis were detected by mNGS in BALF samples of 15 and 2 patients, respectively. Patients were categorized into two groups based on the presence of risk factors for trichomonad infection, including immunocompromised conditions, uncontrolled diabetes mellitus, oral/periodontal diseases, and aspiration. Among 11 patients with risk factors (Case 1-11), 4 received nitromidazoles as part of comprehensive treatment, achieving a 100% treatment success rate. The remaining 7 patients, who did not receive nitromidazoles, had only one achieving relief after broad-spectrum antimicrobial therapy, resulting in a 14.3% treatment success rate. For the 6 patients without any risk factors for trichomonad infection (Case 12-17), none received nitromidazoles during hospitalization. However, 4 out of these 6 patients (66.7%) eventually recovered.</p></sec><sec><title>Conclusion</title><p>mNGS proves to be an efficient tool for detecting trichomonads in BALF samples. Comprehensive analysis of clinical features and laboratory indicators is essential to distinguish between infection and colonization of trichomonads. Pulmonary trichomoniasis should not be overlooked when trichomonads are detected in BALF from patients with risk factors.</p></sec>

<span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289231">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289231</a> <a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1289231">Save to Pocket</a>
</span>



<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>

