---
title: health 2024.13
updated: 2024-03-27 08:14:48
---

# health 2024.13

(date: 2024-03-27 08:14:48)

---

## Staphylococcus aureus oleate hydratase produces ligands that activate host PPARα

date: 2024-03-27, from: Frontiers in Cellular and Infection Microbiology

<p>Commensal gut bacteria use oleate hydratase to release a spectrum of hydroxylated fatty acids using host-derived unsaturated fatty acids. These compounds are thought to attenuate the immune response, but the underlying signaling mechanism(s) remain to be established. The pathogen <italic>Staphylococcus aureus</italic> also expresses an oleate hydratase and 10-hydroxyoctadecanoic acid (<italic>h</italic>18:0) is the most abundant oleate hydratase metabolite found at Staphylococcal skin infection sites. Here, we show <italic>h</italic>18:0 stimulates the transcription of a set of lipid metabolism genes associated with the activation of peroxisome proliferator activated receptor (PPAR) in the RAW 264.7 macrophage cell line and mouse primary bone marrow-derived macrophages. Cell-based transcriptional reporter assays show <italic>h</italic>18:0 selectively activates PPARα. Radiolabeling experiments with bone marrow-derived macrophages show [1-<sup>14</sup>C]<italic>h</italic>18:0 is not incorporated into cellular lipids, but is degraded by β-oxidation, and mass spectrometry detected shortened fragments of <italic>h</italic>18:0 released into the media. The catabolism of <italic>h</italic>18:0 was &gt;10-fold lower in bone marrow-derived macrophages isolated from <italic>Ppara</italic><sup>−/−</sup> knockout mice, and we recover 74-fold fewer <italic>S. aureus</italic> cells from the skin infection site of <italic>Ppara</italic><sup>−/−</sup> knockout mice compared to wildtype mice. These data identify PPARα as a target for oleate hydratase-derived hydroxy fatty acids and support the existence of an oleate hydratase-PPARα signaling axis that functions to suppress the innate immune response to <italic>S. aureus</italic>.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1352810>

---

## Pathological proliferation: a potential mechanism for poor CD4+ T cell recovery in people living with HIV

date: 2024-03-27, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Background</title><p>People living with HIV (PLWH) fail to achieve normalization of CD4<sup>+</sup> T cell counts and function, especially in immunological non-responders (INRs). The frequencies of Ki67<sup>+</sup>CD4<sup>+</sup> T cells were inversely associated with CD4<sup>+</sup> T cell counts in HIV infected patients. Early ART did not normalize CD4<sup>+</sup> T cell proliferation. However, the features of the abnormal proliferation CD4<sup>+</sup> T cell in INRs are far from known.</p></sec><sec><title>Method</title><p>PLWH were divided into INRs (n= 16) and immunological responders (IRs, n= 53) groups. Mass cytometry was applied to peripheral blood T cells to profile the immune cells and liquid chip technique was used to measure plasma levels of cytokines and chemokines. Correlation analyses were conducted to evaluate associations between the degree of CD4<sup>+</sup> T cell proliferation and immune function.</p></sec><sec><title>Results</title><p>The percentage of Ki67<sup>+</sup> CD4<sup>+</sup> T cells were significant higher in INRs, and we defined these cells with significant higher level of Ki67, as over-proliferating cells. No significant difference of markers’ expression (HLA-DR, CD38, CD57, PD-1, PD-L1, CD107a, perforin) was found between INRs and IRs. Compared with naïve CD4<sup>+</sup> T cells in INRs, Ki67<sup>+</sup> CD4<sup>+</sup> T cells exhibited lower levels of CD57 and CD38. Whereas Ki67<sup>+</sup> T cells exhibited higher levels of CD38 and CD57 and activation compared with differentiated mature central memory CD4<sup>+</sup> T cells and effector memory CD4<sup>+</sup> T cells. Ki67<sup>+</sup> cells did not show higher levels of senescence and activation compared to certain Ki67<sup>-</sup> CD4<sup>+</sup> central memory T cells in IRs. Furthermore, Ki67<sup>+</sup> CD4<sup>+</sup> Tcm cells exhibited positive correlations with pro-inflammatory cytokines.</p></sec><sec><title>Conclusion</title><p>We proposed and validated the hypothesis of “pathological proliferation” in INRs: excessive proliferation of CD4<sup>+</sup> T cells in INRs may be accompanied by aberrant activation, senescence and loss of immune function. Eventually, such over-proliferating but poor-quality cells in INRs result in incomplete recovery of both CD4<sup>+</sup> T cell counts and function. An intervention that enhancing the proliferative capacity or functional ability or both of CD4<sup>+</sup> T cell in INRs might therefore be beneficial.</p></sec> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1344778>

---

## SARS-CoV-2–specific mucosal immune response in vaccinated versus infected children

date: 2024-03-27, from: Frontiers in Cellular and Infection Microbiology

<p>The anti-COVID-19 intramuscular vaccination induces a strong systemic but a weak mucosal immune response in adults. Little is known about the mucosal immune response in children infected or vaccinated against SARS-CoV-2. We found that 28% of children had detectable salivary IgA against SARS-CoV-2 even before vaccination, suggesting that, in children, SARS-CoV-2 infection may be undiagnosed. After vaccination, only receptor-binding domain (RBD)–specific IgA1 significantly increased in the saliva. Conversely, infected children had significantly higher salivary RBD-IgA2 compared to IgA1, indicating that infection more than vaccination induces a specific mucosal immune response in children. Future efforts should focus on development of vaccine technologies that also activate mucosal immunity.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1231697>

---

## Emergence of eravacycline heteroresistance in carbapenem-resistant Acinetobacter baumannii isolates in China

date: 2024-03-27, from: Frontiers in Cellular and Infection Microbiology

<p>Carbapenem-resistant <italic>Acinetobacter baumannii</italic> (CRAB) is resistant to almost all antibiotics. Eravacycline, a newer treatment option, has the potential to treat CRAB infections, however, the mechanism by which CRAB isolates develop resistance to eravacycline has yet to be clarified. This study sought to investigate the features and mechanisms of eravacycline heteroresistance among CRAB clinical isolates. A total of 287 isolates were collected in China from 2020 to 2022. The minimum inhibitory concentration (MIC) of eravacycline and other clinically available agents against <italic>A. baumannii</italic> were determined using broth microdilution. The frequency of eravacycline heteroresistance was determined by population analysis profiling (PAP). Mutations and expression levels of resistance genes in heteroresistant isolates were determined by polymerase chain reaction (PCR) and quantitative real-time PCR (qRT-PCR), respectively. Antisense RNA silencing was used to validate the function of eravacycline heteroresistant candidate genes. Twenty-five eravacycline heteroresistant isolates (17.36%) were detected among 144 CRAB isolates with eravacycline MIC values ≤4 mg/L while no eravacycline heteroresistant strains were detected in carbapenem-susceptible <italic>A. baumannii</italic> (CSAB) isolates. All eravacycline heteroresistant strains contained OXA-23 carbapenemase and the predominant multilocus sequence typing (MLST) was ST208 (72%). Cross-resistance was observed between eravacycline, tigecycline, and levofloxacin in the resistant subpopulations. The addition of efflux pump inhibitors significantly reduced the eravacycline MIC in resistant subpopulations and weakened the formation of eravacycline heteroresistance in CRAB isolates. The expression levels of <italic>adeABC</italic> and <italic>adeRS</italic> were significantly higher in resistant subpopulations than in eravacycline heteroresistant parental strains (<italic>P</italic> &lt; 0.05). An IS<italic>Aba1</italic> insertion in the <italic>adeS</italic> gene was identified in 40% (10/25) of the resistant subpopulations. Decreasing the expression of <italic>adeABC</italic> or <italic>adeRS</italic> by antisense RNA silencing significantly inhibited eravacycline heteroresistance. In conclusion, this study identified the emergence of eravacycline heteroresistance in CRAB isolates in China, which is associated with high expression of AdeABC and AdeRS.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356353>

---

## Klebsiella pneumoniae exhibiting a phenotypic hyper-splitting phenomenon including the formation of small colony variants

date: 2024-03-27, from: Frontiers in Cellular and Infection Microbiology

<p>In this study, we characterized a <italic>Klebsiella pneumoniae</italic> strain in a patient with shrapnel hip injury, which resulted in multiple phenotypic changes, including the formation of a small colony variant (SCV) phenotype. Although already described since the 1960s, there is little knowledge about SCV phenotypes in <italic>Enterobacteriaceae</italic>. The formation of SCVs has been recognized as a bacterial strategy to evade host immune responses and compromise the efficacy of antimicrobial therapies, leading to persistent and recurrent courses of infections. In this case, 14 isolates with different resisto- and morpho-types were distinguished from the patient’s urine and tissue samples. Whole genome sequencing revealed that all isolates were clonally identical belonging to the <italic>K. pneumoniae</italic> high-risk sequence type 147. Subculturing the SCV colonies consistently resulted in the reappearance of the initial SCV phenotype and three stable normal-sized phenotypes with distinct morphological characteristics. Additionally, an increase in resistance was observed over time in isolates that shared the same colony appearance. Our findings highlight the complexity of bacterial behavior by revealing a case of phenotypic “hyper-splitting” in a <italic>K. pneumoniae</italic> SCV and its potential clinical significance.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1372704>

---

## Case report: A rare case of skin abscess caused by coinfection of Actinobaculum schaalii and Actinomyces turicensis

date: 2024-03-27, from: Frontiers in Cellular and Infection Microbiology

<p>Skin abscess is one of the most common infections of the skin and soft tissues. However, anaerobic bacteria are infrequently identified as the causative agents of this particular form of abscess. In this case, a 34-year-old pregnant woman was diagnosed with a skin abscess with the use of ultrasonography. The microbiological analysis results of the purulent fluid revealed the coinfection of <italic>Actinobaculum schaalii</italic> and <italic>Actinomyces turicensis</italic>. The patient was first treated empirically with 3 days of cefathiamidine, which resulted in no symptom improvement. Subsequently, a surgical procedure involving incision and draining was performed, with the administration of ceftriaxone. After 7 days of antibiotic intervention, the patient exhibited a satisfactory recovery. Clinicians need to be aware of other types of infections that might be attributed to <italic>Actinobaculum schaalii</italic> and <italic>Actinomyces turicensis</italic>, in addition to urinary tract infections.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378197>

---

## Pore-forming peptide C14R exhibits potent antifungal activity against clinical isolates of Candida albicans and Candida auris

date: 2024-03-27, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>Invasive candidiasis is a global public health problem as it poses a significant threat in hospital-settings. The aim of this study was to evaluate C14R, an analog derived from peptide BP100, as a potential antimicrobial peptide against the prevalent opportunistic yeast <italic>Candida albicans</italic> and the emergent multidrug-resistant yeast <italic>Candida auris</italic>.</p></sec><sec><title>Methods</title><p>Antifungal susceptibility testing of C14R against 99 C<italic>. albicans</italic> and 105 C<italic>. auris</italic> clinical isolates from Colombia, was determined by broth microdilution. Fluconazole was used as a control antifungal. The synergy between C14R and fluconazole was assessed in resistant isolates. Assays against fungal biofilm and growth curves were also carried out. Morphological alterations of yeast cell surface were evaluated by scanning electron microscopy. A permeability assay verified the pore-forming ability of C14R.</p></sec><sec><title>Results</title><p><italic>C. albicans</italic> and <italic>C. auris</italic> isolates had a geometric mean MIC against C14R of 4.42 µg/ml and 5.34 µg/ml, respectively. Notably, none of the isolates of any species exhibited growth at the highest evaluated peptide concentration (200 µg/ml). Synergistic effects were observed when combining the peptide and fluconazole. C14R affects biofilm and growth of <italic>C. albicans</italic> and <italic>C. auris</italic>. Cell membrane disruptions were observed in both species after treatment with the peptide. It was confirmed that C14R form pores in <italic>C. albicans</italic>’ membrane.</p></sec><sec><title>Discussion</title><p>C14R has a potent antifungal activity against a large set of clinical isolates of both <italic>C. albicans</italic> and <italic>C. auris</italic>, showing its capacity to disrupt <italic>Candida</italic> membranes. This antifungal activity remains consistent across isolates regardless of their clinical source. Furthermore, the absence of correlation between MICs to C14R and resistance to fluconazole indicates the peptide’s potential effectiveness against fluconazole-resistant strains. Our results suggest the potential of C14R, a pore-forming peptide, as a treatment option for fungal infections, such as invasive candidiasis, including fluconazole and amphotericin B -resistant strains.</p></sec> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1389020>

---

## Association between gut microbiota and onset of type 2 diabetes mellitus: a two-sample Mendelian randomization study

date: 2024-03-26, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Aim</title><p>Mendelian randomization (MR) analysis has been used in the exploration of the role of gut microbiota (GM) in type 2 diabetes mellitus (T2DM); however, it was limited to the genus level. This study herein aims to investigate the relationship of GM, especially at the species level, with T2DM in order to provide some evidence for further exploration of more specific GM taxa and pathway abundance in T2DM.</p></sec><sec><title>Methods</title><p>This two-sample MR study was based on the summary statistics of GM from the available genome-wide association study (GWAS) meta-analysis conducted by the MiBioGen consortium as well as the Dutch Microbiome Project (DMP), whereas the summary statistics of T2DM were obtained from the FinnGen consortium released data. Inverse variance weighted (IVW), MR-Egger, strength test (<italic>F</italic>), and weighted median methods were used to examine the causal association between GM and the onset of T2DM. Cochran’s <italic>Q</italic> statistics was employed to quantify the heterogeneity of instrumental variables. Bonferroni’s correction was conducted to correct the bias of multiple testing. We also performed reverse causality analysis.</p></sec><sec><title>Results</title><p>The corrected IVW estimates suggested the increased relative abundance of family <italic>Oxalobacteraceae</italic> (OR = 1.0704) and genus <italic>Oxalobacter</italic> (OR = 1.0874), respectively, were associated with higher odds of T2DM, while that of species <italic>faecis</italic> (OR = 0.9460) had a negative relationship with T2DM. The relationships of class <italic>Betaproteobacteria</italic>, family <italic>Lactobacillaceae</italic>, species <italic>finegoldii</italic>, and species <italic>longum</italic> with T2DM were also significant according to the IVW results (all <italic>P</italic> &lt; 0.05).</p></sec><sec><title>Conclusions</title><p>GM had a potential causal association with T2DM, especially species <italic>faecis</italic>, <italic>finegoldii</italic>, and <italic>longum</italic>. Further studies are still needed to clarify certain results that are contradictory with previous findings.</p></sec> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1327032>

---

## A poisonous cocktail: interplay of cereulide toxin and its structural isomers in emetic Bacillus cereus

date: 2024-03-26, from: Frontiers in Cellular and Infection Microbiology

<p>Food intoxications evoked by emetic <italic>Bacillus cereus</italic> strains constitute a serious threat to public health, leading to emesis and severe organ failure. The emetic peptide toxin cereulide, assembled by the non-ribosomal peptide synthetase CesNRPS, cannot be eradicated from contaminated food by usual hygienic measures due to its molecular size and structural stability. Next to cereulide, diverse chemical variants have been described recently that are produced concurrently with cereulide by CesNRPS. However, the contribution of these isocereulides to the actual toxicity of emetic <italic>B. cereus</italic>, which produces a cocktail of these toxins in a certain ratio, is still elusive. Since cereulide isoforms have already been detected in food remnants from foodborne outbreaks, we aimed to gain insights into the composition of isocereulides and their impact on the overall toxicity of emetic <italic>B. cereus</italic>. The amounts and ratios of cereulide and isocereulides were determined in <italic>B. cereus</italic> grown under standard laboratory conditions and in a contaminated sample of fried rice balls responsible for one of the most severe food outbreaks caused by emetic <italic>B. cereus</italic> in recent years. The ratios of variants were determined as robust, produced either under laboratory or natural, food-poisoning conditions. Examination of their actual toxicity in human epithelial HEp2-cells revealed that isocereulides A-N, although accounting for only 10% of the total cereulide toxins, were responsible for about 40% of the total cytotoxicity. An this despite the fact that some of the isocereulides were less cytotoxic than cereulide when tested individually for cytotoxicity. To estimate the additive, synergistic or antagonistic effects of the single variants, each cereulide variant was mixed with cereulide in a 1:9 and 1:1 binary blend, respectively, and tested on human cells. The results showed additive and synergistic impacts of single variants, highlighting the importance of including not only cereulide but also the isocereulides in routine food and clinical diagnostics to achieve a realistic toxicity evaluation of emetic <italic>B. cereus</italic> in contaminated food as well as in patient samples linked to foodborne outbreaks. Since the individual isoforms confer different cell toxicity both alone and in association with cereulide, further investigations are needed to fully understand their cocktail effect.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1337952>

---

## AflaILVB/G/I and AflaILVD are involved in mycelial production, aflatoxin biosynthesis, and fungal virulence in Aspergillus flavus

date: 2024-03-26, from: Frontiers in Cellular and Infection Microbiology

<p>Aflatoxins (AFs) are produced by fungi such as <italic>Aspergillus flavus</italic> and <italic>A. parasiticus</italic> and are one of the most toxic mycotoxins found in agricultural products and food. Aflatoxin contamination, which requires the control of <italic>A. flavus</italic>, remains problematic because of the lack of effective strategies and the exploration of new compounds that can inhibit <italic>A. flavus</italic> growth and mycotoxin production is urgently required to alleviate potential deleterious effects. Acetohydroxy acid synthase (AHAS) and dihydroxy acid dehydratase are important enzymes in the biosynthetic pathways of branched-chain amino acids (BCAAs), including isoleucine, leucine, and valine. Enzymes involved in BCAA biosynthesis are present in bacteria, plants, and fungi, but not in mammals, and are therefore, attractive targets for antimicrobial and herbicide development. In this study, we characterized <italic>AflaILVB/G/I</italic> and <italic>AflaILVD</italic>, which encode the catalytic and regulatory subunits of AHAS and dihydroxy acid dehydratase, from the pathogenic fungus <italic>Aspergillus flavus</italic>. The <italic>AflaILVB/G/I</italic> and <italic>AflaILVD</italic> deletion mutant grew slower and produced smaller colonies than the wild-type strain when grown on glucose minimal medium, potato dextrose agar, and yeast extract medium for three days at 28°C, and disruption of <italic>AflaILVB/G/I</italic> caused a significant reduction in conidia production when grown on all kinds of media. Cellular stress assays determined that all strains were sensitive to H<sub>2</sub>O<sub>2</sub>. Importantly, the pathogenicity and aflatoxin production were affected when <italic>AflaILVB/G/I</italic> and <italic>AflaILVD</italic> were knocked out, particularly <italic>AflaILVB/G/I</italic>. A series of genes that encoded enzymes involved in aflatoxin synthesis were downregulated, meaning that the knockout of <italic>AflaILVB/G/I</italic> influenced aflatoxin synthesis in <italic>A. flavus</italic> strain WT. Collectively, our results demonstrate the potential value of antifungals targeting <italic>AflaILVB/G/I</italic> in <italic>A. flavus</italic>.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1372779>

---

## HemU and TonB1 contribute to hemin acquisition in Stenotrophomonas maltophilia

date: 2024-03-26, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>The hemin acquisition system is composed of an outer membrane TonB-dependent transporter that internalizes hemin into the periplasm, periplasmic hemin-binding proteins to shuttle hemin, an inner membrane transporter that transports hemin into the cytoplasm, and cytoplasmic heme oxygenase to release iron. Fur and HemP are two known regulators involved in the regulation of hemin acquisition. The hemin acquisition system of <italic>Stenotrophomonas maltophilia</italic> is poorly understood, with the exception of HemA as a TonB-dependent transporter for hemin uptake.</p></sec><sec><title>Methods</title><p>Putative candidates responsible for hemin acquisition were selected via a homolog search and a whole-genome survey of <italic>S. maltophilia</italic>. Operon verification was performed by reverse transcription-polymerase chain reaction. The involvement of candidate genes in hemin acquisition was assessed using an in-frame deletion mutant construct and iron utilization assays. The transcript levels of candidate genes were determined using quantitative polymerase chain reaction.</p></sec><sec><title>Results</title><p><italic>Smlt3896-hemU-exbB2-exbD2-tonB2</italic> and <italic>tonB1-exbB1-exbD1a-exbD1b</italic> operons were selected as candidates for hemin acquisition. Compared with the parental strain, <italic>hemU</italic> and <italic>tonB1</italic> mutants displayed a defect in their ability to use hemin as the sole iron source for growth. However, hemin utilization by the <italic>Smlt3896</italic> and <italic>tonB2</italic> mutants was comparable to that of the parental strain. <italic>HemA</italic> expression was repressed by Fur in iron-replete conditions and derepressed in iron-depleted conditions. HemP negatively regulated <italic>hemA</italic> expression. Like <italic>hemA</italic>, <italic>hemU</italic> was repressed by Fur in iron-replete conditions; however, <italic>hemU</italic> was moderately derepressed in response to iron-depleted stress and fully derepressed when hemin was present. Unlike <italic>hemA</italic> and <italic>hemU</italic>, the <italic>TonB1-exbB1-exbD1a-exbD1b</italic> operon was constitutively expressed, regardless of the iron level or the presence of hemin, and Fur and HemP had no influence on its expression.</p></sec><sec><title>Conclusion</title><p>HemA, HemU, and TonB1 contribute to hemin acquisition in <italic>S. maltophilia</italic>. Fur represses the expression of <italic>hemA</italic> and <italic>hemU</italic> in iron-replete conditions. <italic>HemA</italic> expression is regulated by low iron levels, and HemP acts as a negative regulator of this regulatory circuit. <italic>HemU</italic> expression is regulated by low iron and hemin levels in a <italic>hemP</italic>-dependent manner.</p></sec> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1380976>

---

## Association of Filifactor alocis and its RTX toxin gene ftxA with periodontal attachment loss, and in synergy with Aggregatibacter actinomycetemcomitans

date: 2024-03-26, from: Frontiers in Cellular and Infection Microbiology

<p>The Gram-positive bacterium, <italic>Filifactor alocis</italic> is an oral pathogen, and approximately 50% of known strains encode a recently identified repeat-in-toxin (RTX) protein, FtxA. By assessing a longitudinal Ghanaian study population of adolescents (10-19 years of age; mean age 13.2 years), we recently discovered a possible correlation between deep periodontal pockets measured at the two-year follow-up, presence of the <italic>ftxA</italic> gene, and a high quantity of <italic>F. alocis</italic>. To further understand the contribution of <italic>F. alocis</italic> and FtxA in periodontal disease, we used qPCR in the present study to assess the carriage loads of <italic>F. alocis</italic> and the prevalence of its <italic>ftxA</italic> gene in subgingival plaque specimens, sampled at baseline from the Ghanaian cohort (n=500). Comparing these results with the recorded clinical attachment loss (CAL) longitudinal progression data from the two-year follow up, we concluded that carriers of <italic>ftxA</italic>-positive <italic>F. alocis</italic> typically exhibited higher loads of the bacterium. Moreover, high carriage loads of <italic>F. alocis</italic> and concomitant presence of the <italic>ftxA</italic> gene were two factors that were both associated with an enhanced prevalence of CAL progression. Interestingly, CAL progression appeared to be further promoted upon the simultaneous presence of <italic>F. alocis</italic> and the non-JP2 genotype of <italic>Aggregatibacter actinomycetemcomitans</italic>. Taken together, our present findings are consistent with the notion that <italic>F. alocis</italic> and its <italic>ftxA</italic> gene promotes CAL during periodontal disease.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1376358>

---

## Influenza A virus replicates productively in primary human kidney cells and induces factors and mechanisms related to regulated cell death and renal pathology observed in virus-infected patients

date: 2024-03-25, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>Influenza A virus (IAV) infection can cause the often-lethal acute respiratory distress syndrome (ARDS) of the lung. Concomitantly, acute kidney injury (AKI) is frequently noticed during IAV infection, correlating with an increased mortality. The aim of this study was to elucidate the interaction of IAV with human kidney cells and, thereby, to assess the mechanisms underlying IAV-mediated AKI.</p></sec><sec><title>Methods</title><p>To investigate IAV effects on nephron cells we performed infectivity assays with human IAV, as well as with human isolates of either low or highly pathogenic avian IAV. Also, transcriptome and proteome analysis of IAV-infected primary human distal tubular kidney cells (DTC) was performed. Furthermore, the DTC transcriptome was compared to existing transcriptomic data from IAV-infected lung and trachea cells.</p></sec><sec><title>Results</title><p>We demonstrate productive replication of all tested IAV strains on primary and immortalized nephron cells. Comparison of our transcriptome and proteome analysis of H1N1-type IAV-infected human primary distal tubular cells (DTC) with existing data from H1N1-type IAV-infected lung and primary trachea cells revealed enrichment of specific factors responsible for regulated cell death in primary DTC, which could be targeted by specific inhibitors.</p></sec><sec><title>Discussion</title><p>IAV not only infects, but also productively replicates on different human nephron cells. Importantly, multi-omics analysis revealed regulated cell death as potential contributing factor for the clinically observed kidney pathology in influenza.</p></sec> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1363407>

---

## Inhibition of macrophage infectivity potentiator in Burkholderia pseudomallei suppresses pro-inflammatory responses in murine macrophages

date: 2024-03-25, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Introduction</title><p>Melioidosis, caused by the Gram-negative bacterium <italic>Burkholderia pseudomallei</italic>, is a disease endemic in many tropical countries globally. Clinical presentation is highly variable, ranging from asymptomatic to fatal septicemia, and thus the outcome of infection can depend on the host immune responses. The aims of this study were to firstly, characterize the macrophage immune response to <italic>B. pseudomallei</italic> and secondly, to determine whether the immune response was modified in the presence of novel inhibitors targeting the virulence factor, the macrophage infectivity potentiator (Mip) protein. We hypothesized that inhibition of Mip in <italic>B. pseudomallei</italic> would disarm the bacteria and result in a host beneficial immune response.</p></sec><sec><title>Methods</title><p>Murine macrophage J774A.1 cells were infected with <italic>B. pseudomallei</italic> K96243 in the presence of small-molecule inhibitors targeting the Mip protein. RNA-sequencing was performed on infected cells four hours post-infection. Secreted cytokines and lactose dehydrogenase were measured in cell culture supernatants 24 hours post-infection. Viable, intracellular <italic>B. pseudomallei</italic> in macrophages were also enumerated 24 hours post-infection.</p></sec><sec><title>Results</title><p>Global transcriptional profiling of macrophages infected with <italic>B. pseudomallei</italic> by RNA-seq demonstrated upregulation of immune-associated genes, in particular a significant enrichment of genes in the TNF signaling pathway. Treatment of <italic>B. pseudomallei</italic>-infected macrophages with the Mip inhibitor, AN_CH_37 resulted in a 5.3-fold reduction of <italic>il1b</italic> when compared to cells treated with DMSO, which the inhibitors were solubilized in. A statistically significant reduction in IL-1β levels in culture supernatants was seen 24 hours post-infection with AN_CH_37, as well as other pro-inflammatory cytokines, namely IL-6 and TNF-α. Treatment with AN_CH_37 also reduced the survival of <italic>B. pseudomallei</italic> in macrophages after 24 hours which was accompanied by a significant reduction in <italic>B. pseudomallei</italic>-induced cytotoxicity as determined by lactate dehydrogenase release.</p></sec><sec><title>Discussion</title><p>These data highlight the potential to utilize Mip inhibitors in reducing potentially harmful pro-inflammatory responses resulting from <italic>B. pseudomallei</italic> infection in macrophages. This could be of significance since overstimulation of pro-inflammatory responses can result in immunopathology, tissue damage and septic shock.</p></sec> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1353682>

---

## Clinical performance of metagenomic next-generation sequencing for diagnosis of invasive fungal disease after hematopoietic cell transplant

date: 2024-03-25, from: Frontiers in Cellular and Infection Microbiology

<sec><title>Background</title><p>Timely diagnosis and appropriate antifungal therapy are critical for improving the prognosis of patients with invasive fungal disease (IFD) after hematopoietic stem cell transplantation (HSCT). We evaluated the performance of metagenomic next-generation sequencing (mNGS) and conventional microbiological testing (CMT), as well as the diagnosis, therapeutic management, and outcomes of IFD after HSCT.</p></sec><sec><title>Methods</title><p>We retrospectively studied 189 patients who underwent HSCT and were considered at risk for IFD. In total, 46 patients with IFD were enrolled in this study. The IFD consensus was followed for classifying IFD incidents.</p></sec><sec><title>Results</title><p>Forty-six patients were diagnosed with proven/probable (n = 12), possible (n = 27), and undefined (n = 7) IFD. <italic>Aspergillus</italic> was the most commonly detected fungal genus. <italic>Mucormycosis</italic> was found in 15 patients; two had <italic>Aspergillus</italic>, and one had <italic>Candida</italic> infections. Compared to CMT, mNGS significantly reduced the time required to identify pathogens (<italic>P</italic> = 0.0016). mNGS had a much higher sensitivity than CMT (84.78% vs. 36.96%; <italic>P</italic> &lt; 0.0001). A total of 76.09% of patients received antifungal prophylaxis during fungal infections. All <italic>Pneumocystis</italic> infections occurred later than 100 days after transplantation. Among patients with <italic>Pneumocystis</italic> infection, 71.43% occurred following sulfonamide withdrawal, and subsequent treatment with sulfonamide alone or in combination with other drugs was effective. Based on the empirical antifungal treatment, the dosages, modes of administration, frequency of administration, or antifungal of 55.26% of the patients were changed according to the mNGS results. The 4-year overall survival rate of patients diagnosed with IFD after transplantation was 71.55% (95% CI, 55.18%–85.82%). Hypoproteinemia and corticosteroid use are independent risk factors for IFD.</p></sec><sec><title>Conclusion</title><p>mNGS, which has a high sensitivity and a short detection time, aids in the diagnosis and prognosis of pathogenic fungi. As a powerful technology, mNGS can influence treatment decisions in patients with IFD following HSCT.</p></sec> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1210857>

---

## Identification of kinase modulators as host-directed therapeutics against intracellular methicillin-resistant Staphylococcus aureus

date: 2024-03-25, from: Frontiers in Cellular and Infection Microbiology

<p>The increasing prevalence of antimicrobial-resistant <italic>Staphylococcus aureus</italic> strains, especially methicillin-resistant <italic>S. aureus</italic> (MRSA), poses a threat to successful antibiotic treatment. Unsuccessful attempts to develop a vaccine and rising resistance to last-resort antibiotics urge the need for alternative treatments. Host-directed therapy (HDT) targeting critical intracellular stages of <italic>S. aureus</italic> emerges as a promising alternative, potentially acting synergistically with antibiotics and reducing the risk of <italic>de novo</italic> drug resistance. We assessed 201 ATP-competitive kinase inhibitors from Published Kinase Inhibitor Sets (PKIS1 and PKIS2) against intracellular MRSA. Seventeen hit compounds were identified, of which the two most effective and well-tolerated hit compounds (i.e., GW633459A and GW296115X) were selected for further analysis. The compounds did not affect planktonic bacterial cultures, while they were active in a range of human cell lines of cervical, skin, lung, breast and monocyte origin, confirming their host-directed mechanisms. GW633459A, structurally related to lapatinib, exhibited an HDT effect on intracellular MRSA independently of its known human epidermal growth factor receptor (EGFR)/(HER) kinase family targets. GW296115X activated adenosine monophosphate-activated protein kinase (AMPK), thereby enhancing bacterial degradation via autophagy. Finally, GW296115X not only reduced MRSA growth in human cells but also improved the survival rates of MRSA-infected zebrafish embryos, highlighting its potential as HDT.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1367938>

---

## Minor envelope proteins from GP2a to GP4 contribute to the spread pattern and yield of type 2 PRRSV in MARC-145 cells

date: 2024-03-25, from: Frontiers in Cellular and Infection Microbiology

<p>In China, porcine reproductive and respiratory syndrome virus (PRRSV) vaccines are widely used. These vaccines, which contain inactivated and live attenuated vaccines (LAVs), are produced by MARC-145 cells derived from the monkey kidney cell line. However, some PRRSV strains in MARC-145 cells have a low yield. Here, we used two type 2 PRRSV strains (CH-1R and HuN4) to identify the genes responsible for virus yield in MARC-145 cells. Our findings indicate that the two viruses have different spread patterns, which ultimately determine their yield. By replacing the viral envelope genes with a reverse genetics system, we discovered that the minor envelope proteins, from GP2a to GP4, play a crucial role in determining the spread pattern and yield of type 2 PRRSV in MARC-145 cells. The cell-free transmission pattern of type 2 PRRSV appears to be more efficient than the cell-to-cell transmission pattern. Overall, these findings suggest that GP2a to GP4 contributes to the spread pattern and yield of type 2 PRRSV.</p> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1376725>

