<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.16</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../">The Antenna</a></li>
	<li><a href="../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../parks.html" title="new about parks">Parks</a></li>
	<li><a href="../../tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="../../libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="./">Archives</a></li>
	<li><a href="../search.html">Search</a></li>
	<li><a href="../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.16</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.16">health 2024.16</h1>
<p>(date: 2024-04-18 08:55:32)</p>
<hr />
<h2
id="tissue-specific-localization-of-tick-borne-pathogens-in-ticks-collected-from-camels-in-kenya-insights-into-vector-competence">Tissue-specific
localization of tick-borne pathogens in ticks collected from camels in
Kenya: insights into vector competence</h2>
<p>date: 2024-04-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundTick-borne pathogen (TBP) surveillance studies often use
whole-tick homogenates when inferring tick-pathogen associations.
However, localized TBP infections within tick tissues (saliva,
hemolymph, salivary glands, and midgut) can inform pathogen transmission
mechanisms and are key to disentangling pathogen detection from vector
competence.MethodsWe screened 278 camel blood samples and 504 tick
tissue samples derived from 126 camel ticks sampled in two Kenyan
counties (Laikipia and Marsabit) for Anaplasma, Ehrlichia, Coxiella,
Rickettsia, Theileria, and Babesia by PCR-HRM analysis.ResultsCandidatus
Anaplasma camelii infections were common in camels (91%), but absent in
all samples from Rhipicephalus pulchellus, Amblyomma gemma, Hyalomma
dromedarii, and Hyalomma rufipes ticks. We detected Ehrlichia
ruminantium in all tissues of the four tick species, but Rickettsia
aeschlimannii was only found in Hy. rufipes (all tissues). Rickettsia
africae was highest in Am. gemma (62.5%), mainly in the hemolymph (45%)
and less frequently in the midgut (27.5%) and lowest in Rh. pulchellus
(29.4%), where midgut and hemolymph detection rates were 17.6% and
11.8%, respectively. Similarly, in Hy. dromedarii, R. africae was mainly
detected in the midgut (41.7%) but was absent in the hemolymph.
Rickettsia africae was not detected in Hy. rufipes. No Coxiella,
Theileria, or Babesia spp. were detected in this study.ConclusionsThe
tissue-specific localization of R. africae, found mainly in the
hemolymph of Am. gemma, is congruent with the role of this tick species
as its transmission vector. Thus, occurrence of TBPs in the hemolymph
could serve as a predictor of vector competence of TBP transmission,
especially in comparison to detection rates in the midgut, from which
they must cross tissue barriers to effectively replicate and disseminate
across tick tissues. Further studies should focus on exploring the
distribution of TBPs within tick tissues to enhance knowledge of TBP
epidemiology and to distinguish competent vectors from dead-end
hosts.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382228"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382228</a></p>
<hr />
<h2
id="vitamin-d-effects-on-chlamydia-trachomatis-infection-a-case-control-and-experimental-study">Vitamin
D effects on Chlamydia trachomatis infection: a case-control and
experimental study</h2>
<p>date: 2024-04-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionVitamin D deficiency is the most common nutritional
deficiency worldwide. Chronic vitamin D deficiency causes immune system
dysfunction, which increases susceptibility to pathogens such as
bacteria, especially intracellular parasites, and viruses. Chlamydia
trachomatis (C. t) is an obligate intracellular parasitic bacterium that
causes a variety of sequelae. We speculated that vitamin D might be
associated with C. t infection. This study aimed to address this gap in
knowledge by investigating the relationship between vitamin D and C. t
infection using both in vitro and in vivo models.Methods and resultsThe
addition of calcitriol to McCoy cell culture in vitro delayed and
reduced the quantity and volume of inclusions compared to the control
group. Macrophages of peritoneally lavaged mice co-cultured with McCoy
decreased the infection rate and delayed the appearance of inclusions.
In mice models of vitamin D deficiency, mice in the VD-group exhibited
more severe genital tract inflammation and a longer duration of
infection after inoculation with C. t in the genital tract.
Supplementing these mice with vitamin D3 during treatment enhanced the
therapeutic effect of antibiotics. We also conducted a case-control
study involving 174 C. t-positive patients (95 males and 79 females) and
380 healthy volunteers (211 males and 169 females) aged 20–49 from
January 2016 to March 15, 2017. Serum 25-(OH)D concentration was
measured by assessing morning fasting blood samples of healthy
volunteers and C. t-positive patients 1 day before antibiotic treatment
and the next day after one course of treatment. The patients were
followed up for 1 month and evaluated for recovery. The results showed
that vitamin D deficiency was a risk factor for C. t infection and
treatment failure.ConclusionIn summary, findings from experimental and
clinical studies indicate a close association between vitamin D levels
and C. t infection and treatment outcomes. Given the affordability and
safety of vitamin D, both healthy individuals and patients should focus
on vitamin D intake. Vitamin D supplementation could enhance treatment
success and should be used as an adjunctive therapy alongside antibiotic
therapy for C. t infections, pending confirmation in larger,
prospective, randomized controlled trials.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366136"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366136</a></p>
<hr />
<h2
id="cellugyrin-synaptogyrin-2-dependent-pathways-are-used-by-bacterial-cytolethal-distending-toxin-and-sars-cov-2-virus-to-gain-cell-entry">Cellugyrin
(synaptogyrin-2) dependent pathways are used by bacterial cytolethal
distending toxin and SARS-CoV-2 virus to gain cell entry</h2>
<p>date: 2024-04-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Aggregatibacter actinomycetemcomitans cytolethal distending toxin
(Cdt) is capable of intoxicating lymphocytes macrophages, mast cells and
epithelial cells. Following Cdt binding to cholesterol, in the region of
membrane lipid rafts, the CdtB and CdtC subunits are internalized and
traffic to intracellular compartments. These events are dependent upon,
cellugyrin, a critical component of synaptic like microvesicles
(SLMVCg+). Target cells, such as Jurkat cells, rendered unable to
express cellugyrin are resistant to Cdt-induced toxicity. Similar to
Cdt, SARS-CoV-2 entry into host cells is initiated by binding to cell
surface receptors, ACE-2, also associated with cholesterol-rich lipid
rafts; this association leads to fusion and/or endocytosis of viral and
host cell membranes and intracellular trafficking. The similarity in
internalization pathways for both Cdt and SARS-CoV-2 led us to consider
the possibility that cellugyrin was a critical component in both
processes. Cellugyrin deficient Calu-3 cells (Calu-3Cg-) were prepared
using Lentiviral particles containing shRNA; these cells were resistant
to infection by VSV/SARS-CoV-2-spike pseudotype virus and partially
resistant to VSV/VSV-G pseudotype virus. Synthetic peptides representing
various regions of the cellugyrin protein were prepared and assessed for
their ability to bind to Cdt subunits using surface plasmon resonance.
Cdt was capable of binding to a region designated the middle outer loop
(MOL) which corresponds to a region extending into the cytoplasmic
surface of the SLMVCg+. SARS-CoV-2 spike proteins were assessed for
their ability to bind to cellugyrin peptides; SARS-CoV-2 full length
spike protein preferentially binds to a region within the SLMVCg+ lumen,
designated intraluminal loop 1A. SARS-CoV-2-spike protein domain S1,
which contains the receptor binding domains, binds to cellugyrin
N-terminus which extends out from the cytoplasmic surface of SLMV.
Binding specificity was further analyzed using cellugyrin scrambled
peptide mutants. We propose that SLMVCg+ represent a component of a
common pathway that facilitates pathogen and/or pathogen-derived toxins
to gain host cell entry.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334224"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334224</a></p>
<hr />
<h2
id="single-nucleotide-variants-in-the-ccl2-oas1-and-dpp9-genes-and-their-association-with-the-severity-of-covid-19-in-an-ecuadorian-population">Single
nucleotide variants in the CCL2, OAS1 and DPP9 genes and their
association with the severity of COVID-19 in an Ecuadorian
population</h2>
<p>date: 2024-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>COVID-19 has a broad clinical spectrum, ranging from
asymptomatic-mild form to severe phenotype. The severity of COVID-19 is
a complex trait influenced by various genetic and environmental factors.
Ethnic differences have been observed in relation to COVID-19 severity
during the pandemic. It is currently unknown whether genetic variations
may contribute to the increased risk of severity observed in
Latin-American individuals The aim of this study is to investigate the
potential correlation between gene variants at CCL2, OAS1, and DPP9
genes and the severity of COVID-19 in a population from Quito, Ecuador.
This observational case-control study was conducted at the Carrera de
Biologia from the Universidad Central del Ecuador and the Hospital Quito
Sur of the Instituto Ecuatoriano de Seguridad Social (Quito-SUR-IESS),
Quito, Ecuador. Genotyping for gene variants at rs1024611 (A&gt;G),
rs10774671 (A&gt;G), and rs10406145 (G&gt;C) of CCL2, OAS1, and DPP9
genes was performed on 100 COVID-19 patients (43 with severe form and 57
asymptomatic-mild) using RFLP-PCR. The genotype distribution of all SNVs
throughout the entire sample of 100 individuals showed Hardy Weinberg
equilibrium (P=0.53, 0.35, and 0.4 for CCL2, OAS1, and DPP9,
respectively). The HWE test did not find any statistically significant
difference in genotype distribution between the study and control groups
for any of the three SNVs. The multivariable logistic regression
analysis showed that individuals with the GG of the CCL2 rs1024611 gene
variant had an increased association with the severe COVID-19 phenotype
in a recessive model (P = 0.0003, OR = 6.43, 95% CI 2.19-18.89) and for
the OAS1 rs10774671 gene variant, the log-additive model showed a
significant association with the severe phenotype of COVID-19 (P=0.0084,
OR=3.85, 95% CI 1.33-11.12). Analysis of haplotype frequencies revealed
that the coexistence of GAG at CCL2, OAS1, and DPP9 variants,
respectively, in the same individual increased the presence of the
severe COVID-19 phenotype (OR=2.273, 95% CI: 1.271-4.068, P=0.005305).
The findings of the current study suggests that the ethnic background
affects the allele and genotype frequencies of genes associated with the
severity of COVID-19. The experience with COVID-19 has provided an
opportunity to identify an ethnicity-based approach to recognize
genetically high-risk individuals in different populations for emerging
diseases.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1322882"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1322882</a></p>
<hr />
<h2
id="ocular-surface-immune-transcriptome-and-tear-cytokines-in-corneal-infection-patients">Ocular
surface immune transcriptome and tear cytokines in corneal infection
patients</h2>
<p>date: 2024-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMicrobial keratitis is one of the leading causes of
blindness globally. An overactive immune response during an infection
can exacerbate damage, causing corneal opacities and vision loss. This
study aimed to identify the differentially expressed genes between
corneal infection patients and healthy volunteers within the cornea and
conjunctiva and elucidate the contributing pathways to these conditions’
pathogenesis. Moreover, it compared the corneal and conjunctival
transcriptomes in corneal-infected patients to cytokine levels in
tears.MethodsCorneal and conjunctival swabs were collected from seven
corneal infection patients and three healthy controls under topical
anesthesia. RNA from seven corneal infection patients and three healthy
volunteers were analyzed by RNA sequencing (RNA-Seq). Tear proteins were
extracted from Schirmer strips via acetone precipitation from 38 cases
of corneal infection and 14 healthy controls. The cytokines and
chemokines IL-1β, IL-6, CXCL8 (IL-8), CX3CL1, IL-10, IL-12 (p70),
IL-17A, and IL-23 were measured using an antibody bead assay.ResultsA
total of 512 genes were found to be differentially expressed in infected
corneas compared to healthy corneas, with 508 being upregulated and four
downregulated (fold-change (FC) &lt;−2 or &gt; 2 and adjusted p
&lt;0.01). For the conjunctiva, 477 were upregulated, and 3 were
downregulated (FC &lt;−3 or ≥ 3 and adjusted p &lt;0.01). There was a
significant overlap in cornea and conjunctiva gene expression in
patients with corneal infections. The genes were predominantly
associated with immune response, regulation of angiogenesis, and
apoptotic signaling pathways. The most highly upregulated gene was CXCL8
(which codes for IL-8 protein). In patients with corneal infections, the
concentration of IL-8 protein in tears was relatively higher in patients
compared to healthy controls but did not show statistical
significance.ConclusionsDuring corneal infection, many genes were
upregulated, with most of them being associated with immune response,
regulation of angiogenesis, and apoptotic signaling. The findings may
facilitate the development of treatments for corneal infections that can
dampen specific aspects of the immune response to reduce scarring and
preserve sight.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1346821"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1346821</a></p>
<hr />
<h2
id="infection-with-the-multidrug-resistant-klebsiella-pneumoniae-new-delhi-metallo-b-lactamase-strain-in-patients-with-covid-19-nec-hercules-contra-plures">Infection
with the multidrug-resistant Klebsiella pneumoniae New Delhi
metallo-B-lactamase strain in patients with COVID-19: Nec Hercules
contra plures?</h2>
<p>date: 2024-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundDuring the coronavirus disease 2019 (COVID-19) pandemic, in
patients treated for SARS-CoV-2 infection, infections with the
Klebsiella pneumoniae bacteria producing New Delhi metallo-B-lactamase
(NDM) carbapenemase in the USA, Brazil, Mexico, and Italy were observed,
especially in intensive care units (ICUs). This study aimed to assess
the impact of Klebsiella pneumoniae NDM infection and other bacterial
infections on mortality in patients treated in ICUs due to
COVID-19.MethodsThe 160 patients who qualified for the study were
hospitalized in ICUs due to COVID-19. Three groups were distinguished:
patients with COVID-19 infection only (N = 72), patients with COVID-19
infection and infection caused by Klebsiella pneumoniae NDM (N = 30),
and patients with COVID-19 infection and infection of bacterial etiology
other than Klebsiella pneumoniae NDM (N = 58). Mortality in the groups
and chosen demographic data; biochemical parameters analyzed on days 1,
3, 5, and 7; comorbidities; and ICU scores were
analyzed.ResultsBacterial infection, including with Klebsiella
pneumoniae NDM type, did not elevate mortality rates. In the group of
patients who survived the acute phase of COVID-19 the prolonged survival
time was demonstrated: the median overall survival time was 13 days in
the NDM bacterial infection group, 14 days in the other bacterial
infection group, and 7 days in the COVID-19 only group. Comparing the
COVID-19 with NDM infection and COVID-19 only groups, the adjusted model
estimated a statistically significant hazard ratio of 0.28 (p = 0.002).
Multivariate analysis revealed that age, APACHE II score, and CRP were
predictors of mortality in all the patient groups.ConclusionIn patients
treated for SARS-CoV-2 infection acquiring a bacterial infection due to
prolonged hospitalization associated with the treatment of COVID-19 did
not elevate mortality rates. The data suggests that in severe COVID-19
patients who survived beyond the first week of hospitalization,
bacterial infections, particularly Klebsiella pneumoniae NDM, do not
significantly impact mortality. Multivariate analysis revealed that age,
APACHE II score, and CRP were predictors of mortality in all the patient
groups.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297312"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297312</a></p>
<hr />
<h2
id="the-complete-genome-sequence-of-unculturable-mycoplasma-faucium-obtained-through-clinical-metagenomic-next-generation-sequencing">The
complete genome sequence of unculturable Mycoplasma faucium obtained
through clinical metagenomic next-generation sequencing</h2>
<p>date: 2024-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionDiagnosing Mycoplasma faucium poses challenges, and it’s
unclear if its rare isolation is due to infrequent occurrence or its
fastidious nutritional requirements.MethodsThis study analyzes the
complete genome sequence of M. faucium, obtained directly from the pus
of a sternum infection in a lung transplant patient using metagenomic
sequencing.ResultsGenome analysis revealed limited therapeutic options
for the M. faucium infection, primarily susceptibility to tetracyclines.
Three classes of mobile genetic elements were identified: two new
insertion sequences, a new prophage (phiUMCG-1), and a species-specific
variant of a mycoplasma integrative and conjugative element (MICE).
Additionally, a Type I Restriction-Modification system was identified,
featuring 5’-terminally truncated hsdS pseudogenes with overlapping
repeats, indicating the potential for forming alternative hsdS variants
through recombination.ConclusionThis study represents the first-ever
acquisition of a complete circularized bacterial genome directly from a
patient sample obtained from invasive infection of a primary sterile
site using culture-independent, PCR-free clinical metagenomics.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368923"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368923</a></p>
<hr />
<h2
id="comprehensive-assessment-of-hf-rtms-treatment-mechanism-for-post-stroke-dysphagia-in-rats-by-integration-of-fecal-metabolomics-and-16s-rrna-sequencing">Comprehensive
assessment of HF-rTMS treatment mechanism for post-stroke dysphagia in
rats by integration of fecal metabolomics and 16S rRNA sequencing</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe mechanism by which high-frequency repetitive
transcranial magnetic stimulation (HF-rTMS) improves swallowing function
by regulating intestinal flora remains unexplored. We aimed to evaluate
this using fecal metabolomics and 16S rRNA sequencing.MethodsA
Post-stroke dysphagia (PSD) rat model was established by middle cerebral
artery occlusion. The magnetic stimulation group received HF-rTMS from
the 7th day post-operation up to 14th day post-surgery. Swallowing
function was assessed using a videofluoroscopic swallowing study (VFSS).
Hematoxylin-eosin (H&amp;E) staining was used to assess
histopathological changes in the intestinal tissue. Intestinal flora
levels were evaluated by sequencing the 16S rRNA V3-V4 region.
Metabolite changes within the intestinal flora were evaluated by fecal
metabolomics using liquid chromatography-tandem mass
spectrometry.ResultsVFSS showed that the bolus area and pharyngeal bolus
speed were significantly decreased in PSD rats, while the bolus area
increased and pharyngeal transit time decreased after HF-rTMS
administration (p &lt; 0.05). In the PSD groups, H&amp;E staining
revealed damaged surface epithelial cells and disrupted cryptal glands,
whereas HF-rTMS reinforced the integrity of the intestinal epithelial
cells. 16S rRNA sequencing indicated that PSD can disturb the intestinal
flora and its associated metabolites, whereas HF-rTMS can significantly
regulate the composition of the intestinal microflora. Firmicutes and
Lactobacillus abundances were lower in the PSD group than in the
baseline group at the phylum and genus levels, respectively; however,
both increased after HF-rTMS administration. Levels of ceramides (Cer),
free fatty acids (FA), phosphatidylethanolamine (PE), triacylglycerol
(TAG), and sulfoquinovosyl diacylglycerol were increased in the PSD
group. The Cer, FA, and DG levels decreased after HF-rTMS treatment,
whereas the TAG levels increased. Peptococcaceae was negatively
correlated with Cer, Streptococcus was negatively correlated with DG,
and Acutalibacter was positively correlated with FA and Cer. However,
these changes were effectively restored by HF-rTMS, resulting in
recovery from dysphagia.ConclusionThese findings suggest a synergistic
role for the gut microbiota and fecal metabolites in the development of
PSD and the therapeutic mechanisms underlying HF-rTMS.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1373737"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1373737</a></p>
<hr />
<h2
id="editorial-identification-and-characterization-of-novel-antigens-of-malarial-parasites-volume-ii">Editorial:
Identification and characterization of novel antigens of malarial
parasites, volume II</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408366"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408366</a></p>
<hr />
<h2 id="editorial-one-health-in-clinical-microbiology">Editorial: One
Health in clinical microbiology</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1404276"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1404276</a></p>
<hr />
<h2
id="development-of-a-new-loop-mediated-isothermal-amplification-test-for-the-sensitive-rapid-and-economic-detection-of-different-genotypes-of-classical-swine-fever-virus">Development
of a new loop-mediated isothermal amplification test for the sensitive,
rapid, and economic detection of different genotypes of Classical swine
fever virus</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundClassical swine fever virus (CSFV) remains one of the most
important pathogens in animal health. Pathogen detection relies on viral
RNA extraction followed by RT-qPCR. Novel technologies are required to
improve diagnosis at the point of care.MethodsA loop-mediated isothermal
amplification (LAMP) PCR technique was developed, with primers designed
considering all reported CSFV genotypes. The reaction was tested using
both fluorometric and colorimetric detection, in comparison to the gold
standard technique. Viral strains from three circulating CSFV genotypes
were tested, as well as samples from infected animals. Other pathogens
were also tested, to determine the LAMP specificity. Besides laboratory
RNA extraction methods, a heating method for RNA release, readily
available for adaptation to field conditions was evaluated.ResultsThree
primer sets were generated, with one of them showing better performance.
This primer set proved capable of maintaining optimal performance at a
wide range of amplification temperatures (60°C - 68°C). It was also able
to detect CSFV RNA from the three genotypes tested. The assay was highly
efficient in detection of samples from animals infected with field
strains from two different genotypes, with multiple matrices being
detected using both colorimetric and fluorometric methods. The LAMP
assay was negative for all the unrelated pathogens tested, including
Pestiviruses. The only doubtful result in both fluorometric and
colorimetric LAMP was against the novel Pestivirus italiaense, ovine
Italy Pestivirus (OVPV), which has proven to have cross-reaction with
multiple CSFV diagnostic techniques. However, it is only possible to
detect the OVPV in a doubtful result if the viral load is higher than
10000 viral particles.ConclusionThe results from the present study show
that LAMP could be an important addition to the currently used molecular
diagnostic techniques for CSFV. This technique could be used in remote
locations, given that it can be adapted for successful use with minimal
equipment and minimally invasive samples. The joined use of novel and
traditional diagnostic techniques could prove to be a useful alternative
to support the CSF control.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1372166"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1372166</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
