<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.09</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../">The Antenna</a></li>
	<li><a href="../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../parks.html" title="new about parks">Parks</a></li>
	<li><a href="../../tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="../../libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="./">Archives</a></li>
	<li><a href="../search.html">Search</a></li>
	<li><a href="../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.09</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.09">health 2024.09</h1>
<p>(date: 2024-02-27 09:09:33)</p>
<hr />
<h2
id="deciphering-the-impact-of-endoparasitic-infection-on-immune-response-and-gut-microbial-composition-of-channa-punctata">Deciphering
the impact of endoparasitic infection on immune response and gut
microbial composition of Channa punctata</h2>
<p>date: 2024-02-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Intestinal parasitic infections caused by helminths are globally
distributed and are a major cause of morbidity worldwide. Parasites may
modulate the virulence, gut microbiota diversity and host responses
during infection. Despite numerous works, little is known about the
complex interaction between parasites and the gut microbiota. In the
present study, the complex interplay between parasites and the gut
microbiota was investigated. A total of 12 bacterial strains across four
major families, including Enterobacteriaceae, Morganellaceae,
Flavobacteriaceae, and Pseudomonadaceae, were isolated from
<italic>Channa punctata</italic>, infected with the nematode species
<italic>Aporcella</italic> sp., <italic>Axonchium</italic> sp.,
<italic>Tylencholaimus mirabilis</italic>, and <italic>Dioctophyme
renale</italic>. The findings revealed that nematode infection shaped
the fish gut bacterial microbiota and significantly affected their
virulence levels. Nematode-infected fish bacterial isolates are more
likely to be pathogenic, with elevated hemolytic activity and biofilm
formation, causing high fish mortality. In contrast, isolates recovered
further from non-parasitised <italic>C. punctata</italic> were observed
to be non-pathogenic and had negligible hemolytic activity and biofilm
formation. Antibiogram analysis of the bacterial isolates revealed a
disproportionately high percentage of bacteria that were either
marginally or multidrug resistant, suggesting that parasitic
infection-induced stress modulates the gut microenvironment and enables
colonization by antibiotic-resistant strains. This isolation-based study
provides an avenue to unravel the influence of parasitic infection on
gut bacterial characteristics, which is valuable for understanding the
infection mechanism and designing further studies aimed at optimizing
treatment strategies. In addition, the cultured isolates can supplement
future gut microbiome studies by providing wet lab specimens to compare
(meta)genomic information discovered within the gut microenvironment of
fish.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1296769">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1296769</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1296769">Save
to Pocket</a> </span></p>
<hr />
<h2
id="mechanisms-of-mesothelial-cell-response-to-viral-infections-hdac1-3-inhibition-blocks-polyic-induced-type-i-interferon-response-and-modulates-the-mesenchymalinflammatory-phenotype">Mechanisms
of mesothelial cell response to viral infections: HDAC1-3 inhibition
blocks poly(I:C)-induced type I interferon response and modulates the
mesenchymal/inflammatory phenotype</h2>
<p>date: 2024-02-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Infectious peritonitis is a leading cause of peritoneal functional
impairment and a primary factor for therapy discontinuation in
peritoneal dialysis (PD) patients. Although bacterial infections are a
common cause of peritonitis episodes, emerging evidence suggests a role
for viral pathogens. Toll-like receptors (TLRs) specifically recognize
conserved pathogen-associated molecular patterns (PAMPs) from bacteria,
viruses, and fungi, thereby orchestrating the ensuing
inflammatory/immune responses. Among TLRs, TLR3 recognizes viral dsRNA
and triggers antiviral response cascades upon activation. Epigenetic
regulation, mediated by histone deacetylase (HDAC), has been
demonstrated to control several cellular functions in response to
various extracellular stimuli. Employing epigenetic target modulators,
such as epidrugs, is a current therapeutic option in several cancers and
holds promise in treating viral diseases. This study aims to elucidate
the impact of TLR3 stimulation on the plasticity of human mesothelial
cells (MCs) in PD patients and to investigate the effects of HDAC1-3
inhibition. Treatment of MCs from PD patients with the TLR3 agonist
polyinosinic:polycytidylic acid (Poly(I:C)), led to the acquisition of a
bona fide mesothelial-to-mesenchymal transition (MMT) characterized by
the upregulation of mesenchymal genes and loss of epithelial-like
features. Moreover, Poly(I:C) modulated the expression of several
inflammatory cytokines and chemokines. A quantitative proteomic analysis
of MCs treated with MS-275, an HDAC1-3 inhibitor, unveiled altered
expression of several proteins, including inflammatory
cytokines/chemokines and interferon-stimulated genes (ISGs). Treatment
with MS-275 facilitated MMT reversal and inhibited the interferon
signature, which was associated with reduced STAT1 phosphorylation.
However, the modulation of inflammatory cytokine/chemokine production
was not univocal, as IL-6 and CXCL8 were augmented while TNF-Î± and
CXCL10 were decreased. Collectively, our findings underline the
significance of viral infections in acquiring a mesenchymal-like
phenotype by MCs and the potential consequences of virus-associated
peritonitis episodes for PD patients. The observed promotion of MMT
reversal and interferon response inhibition by an HDAC1-3 inhibitor,
albeit without a general impact on inflammatory cytokine production, has
translational implications deserving further analysis.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1308362">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1308362</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1308362">Save
to Pocket</a> </span></p>
<hr />
<h2
id="odorant-binding-protein-18-increases-the-pathogen-resistance-of-the-imported-willow-leaf-beetle-plagiodera-versicolora">Odorant
binding protein 18 increases the pathogen resistance of the imported
willow leaf beetle, Plagiodera versicolora</h2>
<p>date: 2024-02-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Insect odorant-binding proteins (OBPs) are a class of small molecular
weight soluble proteins. In the past few years, OBPs had been found to
work as carriers of ligands and play a crucial role in olfaction and
various other physiological processes, like immunity. A subset of insect
OBPs had been found to be expressed differently and play a function in
immunity of fungal infection. However, there are few studies on the role
of OBPs in immunity of bacterial infection.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
To identify the immune-related OBPs of <italic>Plagiodera
versicolora</italic> after infected by <italic>Pseudomonas
aeruginosa</italic>, we determined the mortality of <italic>P.
versicolora</italic> to <italic>P. aeruginosa</italic> and selected the
time point of 50% mortality of larvae to collect samples for RNA-seq.
RNAi technology was used to investigate the function of immune-related
OBPs after <italic>P. aeruginosa</italic> infection.
</p>
</sec><sec>
<title>
Results
</title>
<p>
RNA-seq data shows that <italic>PverOBP18</italic> gene significantly
up-regulated by 1.8-fold and further RT-qPCR affirmed its expression.
Developmental expression profile showed that the expression of
<italic>PverOBP18</italic> was highest in the pupae, followed by the
female adults, and lower in the 1st-3rd larvae and male adults with
lowest in eggs. Tissue expression profiling showed that
<italic>PverOBP18</italic> was dominantly expressed in the epidermis.
RNAi knockdown of <italic>PverOBP18</italic> significantly reduced the
expression of bacterial recognition receptor gene <italic>PGRP</italic>
and antibacterial peptide gene <italic>Attacin</italic> and reduced the
resistance of <italic>P. versicolora</italic> to <italic>P.
aeruginosa</italic> infection.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Our results indicated that <italic>PverOBP18</italic> gene increased the
pathogen resistance of <italic>P. versicolora</italic> by cooperating
with the immune genes and provided valuable insights into using OBPs as
targets to design novel strategies for management of <italic>P.
versicolora.</italic>
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360680">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360680</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360680">Save
to Pocket</a> </span></p>
<hr />
<h2
id="fluoroquinolone-resistance-in-complicated-urinary-tract-infections-association-with-the-increased-occurrence-and-diversity-of-escherichia-coli-of-clonal-complex-131-together-with-st1193">Fluoroquinolone
resistance in complicated urinary tract infections: association with the
increased occurrence and diversity of Escherichia coli of clonal complex
131, together with ST1193</h2>
<p>date: 2024-02-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Urinary tract infections (UTIs) are on3e of the leading causes of
multidrug-resistance (MDR) spread and infection-related deaths.
<italic>Escherichia coli</italic> is by far the main causative agent. We
conducted a prospective study on complicated urinary tract infections
(cUTIs) i) to monitor the high-risk clones that could be compromising
the therapeutic management and ii) to compare the cUTI etiology with
uncomplicated infections (uUTIs) occurring in the same period and health
area.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
154 non-duplicated <italic>E. coli</italic> recovered from cUTIs in 2020
at the Hospital Universitario Central de Asturias (Spain) constituted
the study collection.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Most cUTI isolates belonged to phylogroup B2 (72.1%) and met the
uropathogenic (UPEC) status (69.5%) (â¥3 of <italic>chuA</italic>,
<italic>fyuA</italic>, <italic>vat</italic>, and <italic>yfcV</italic>
genes). MDR was exhibited by 35.7% of the isolates, similarly to data
observed in the uUTI collection. A significant difference observed in
cUTI was the higher level of fluoroquinolone resistance (FQR) (47.4%),
where the pandemic clonal groups B2-CC131 and B2-ST1193 (CH14-64)
comprised 28% of the 154 <italic>E. coli</italic>, representing 52.1% of
the FQR isolates. Other prevalent FQR clones were D-ST69 (CH35-27),
D-ST405 (CH37-27), and B2-ST429 (CH40-20) (three isolates each). We
uncovered an increased genetic and genomic diversity of the CC131: 10
different virotypes, 8 clonotypes (CH), and 2 STs. The presence of
<italic>bla</italic><sub>CTX-M-15</sub> was determined in 12 (7.8%)
isolates (all CC131), which showed 10 different core genome (cg)STs and
2 <italic>fim</italic>H types (<italic>fim</italic>H30 and
<italic>fim</italic>H602) but the same set of chromosomal mutations
conferring FQR (<italic>gyrA</italic> p.S83L, <italic>gyrA</italic>
p.D87N, <italic>parC</italic> p.S80I, <italic>parC</italic> p.E84V, and
<italic>parE</italic> p.I529L). In addition, the plasmidome analysis
revealed 10 different IncF formulae in CC131 genomes.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
We proved here that non-lactose fermenting screening, together with the
detection of O25b (<italic>rfb</italic>O25b), H4
(<italic>fliC<sub>H4</sub></italic>), and H5
(<italic>fliC<sub>H5</sub></italic>) genes, and phylogroup and
clonotyping assignation, is a reasonable approach that can be easily
implemented for the surveillance of emerging high-risk clones associated
with FQR spread in cUTIs, such as the uncommonly reported
O25b:H4-B2-ST9126-CC131 (CH1267-30). Since <italic>E. coli</italic>
CC131 and ST1193 are also involved in the community uUTIs of this health
area, interventions to eradicate these MDR clones, along with
surveillance for other emerging ones, are essential for antibiotic use
optimization programs.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351618">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351618</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351618">Save
to Pocket</a> </span></p>
<hr />
<h2
id="solid-organ-transplantation-and-gut-microbiota-a-review-of-the-potential-immunomodulatory-properties-of-short-chain-fatty-acids-in-graft-maintenance">Solid
organ transplantation and gut microbiota: a review of the potential
immunomodulatory properties of short-chain fatty acids in graft
maintenance</h2>
<p>date: 2024-02-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Transplantation is the treatment of choice for several end-stage organ
defects: it considerably improves patient survival and quality of life.
However, post-transplant recipients may experience episodes of rejection
that can favor or ultimately lead to graft loss. Graft maintenance
requires a complex and life-long immunosuppressive treatment. Different
immunosuppressive drugs (<italic>i.e.</italic>, calcineurin inhibitors,
glucocorticoids, biological immunosuppressive agents, mammalian target
of rapamycin inhibitors, and antiproliferative or antimetabolic agents)
are used in combination to mitigate the immune response against the
allograft. Unfortunately, the use of these antirejection agents may lead
to opportunistic infections, metabolic (<italic>e.g.</italic>,
post-transplant diabetes mellitus) or cardiovascular
(<italic>e.g.</italic>, arterial hypertension) disorders, cancer
(<italic>e.g.</italic>, non-Hodgkin lymphoma) and other adverse effects.
Lately, immunosuppressive drugs have also been associated with gut
microbiome alterations, known as dysbiosis, and were shown to affect gut
microbiota-derived short-chain fatty acids (SCFA) production. SCFA play
a key immunomodulatory role in physiological conditions, and their
impairment in transplant patients could partly counterbalance the effect
of immunosuppressive drugs leading to the activation of deleterious
pathways and graft rejection. In this review, we will first present an
overview of the mechanisms of graft rejection that are prevented by the
immunosuppressive protocol. Next, we will explain the dynamic changes of
the gut microbiota during transplantation, focusing on SCFA. Finally, we
will describe the known functions of SCFA in regulating
immune-inflammatory reactions and discuss the impact of SCFA impairment
in immunosuppressive drug treated patients.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342354">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342354</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342354">Save
to Pocket</a> </span></p>
<hr />
<h2
id="pqsa-mutation-mediated-enhancement-of-phage-mediated-combat-against-pseudomonas-aeruginosa">PqsA
mutation-mediated enhancement of phage-mediated combat against
Pseudomonas aeruginosa</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Phage therapy is a potential approach in the biocontrol of foodborne
pathogens. However, the emergence of phage resistance and the narrow
host range of most phage isolates continue to limit the antimicrobial
efficacy of phages. Here, we investigated the potential of the
<italic>pqsA</italic> gene, encoding the anthranilate-CoA ligase enzyme,
as an adjuvant for phage therapy. The knockout of the
<italic>pqsA</italic> gene significantly enhanced the bactericidal
effect of phages vB_Pae_QDWS and vB_Pae_S1 against <italic>Pseudomonas
aeruginosa</italic>. Under phage infection pressure, the growth of the
PaÎ<italic>pqsA</italic> was significantly inhibited within 8Â h compared
to the wild-type PAO1. Furthermore, we found that altering phage
adsorption is not how PaÎ<italic>pqsA</italic> responds to phage
infection. Although <italic>pqsA</italic> represents a promising target
for enhancing phage killing, it may not be applicable to allÂ phages,
such as types vB_Pae_W3 and vB_Pae_TR. Our findings provideÂ newÂ material
reserves for the future design of novel phage-based therapeutic
strategies.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1296777">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1296777</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1296777">Save
to Pocket</a> </span></p>
<hr />
<h2
id="plasmid-encoded-gene-duplications-of-extended-spectrum-Î²-lactamases-in-clinical-bacterial-isolates">Plasmid-encoded
gene duplications of extended-spectrum Î²-lactamases in clinical
bacterial isolates</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
The emergence of extended-spectrum Î²-lactamase (ESBL)-producing
Enterobacteriaceae is an urgent and alarming One Health problem. This
study aimed to investigate duplications of plasmid-encoded ESBL genes
and their impact on antimicrobial resistance (AMR) phenotypes in
clinical and screening isolates.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Multi-drug-resistant bacteria from hospitalized patients were collected
during routine clinical surveillance from January 2022 to June 2023, and
their antimicrobial susceptibility patterns were determined. Genotypes
were extracted from long-read whole-genome sequencing data. Furthermore,
plasmids and other mobile genetic elements associated with ESBL genes
were characterized, and the ESBL genes were correlated to ceftazidime
minimal inhibitory concentration (MIC).
</p>
</sec><sec>
<title>
Results
</title>
<p>
In total, we identified four cases of plasmid-encoded ESBL gene
duplications that match four genetically similar plasmids during the
18-month surveillance period: five <italic>Escherichia coli</italic> and
three <italic>Klebsiella pneumoniae</italic> isolates. As the ESBL genes
were part of transposable elements, the surrounding sequence regions
were duplicated as well. In-depth analysis revealed insertion sequence
(IS)-mediated transposition mechanisms. Isolates with duplicated ESBL
genes exhibited a higher MIC for ceftazidime in comparison to isolates
with a single gene copy (3â256 <italic>vs.</italic> 1.5â32 mg/L,
respectively).
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
ESBL gene duplications led to an increased phenotypic resistance against
ceftazidime. Our data suggest that ESBL gene duplications by an
IS-mediated transposition are a relevant mechanism for how AMR develops
in the clinical setting and is part of the microevolution of plasmids.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1343858">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1343858</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1343858">Save
to Pocket</a> </span></p>
<hr />
<h2
id="distinct-fungal-microbiomes-of-two-thai-commercial-stingless-bee-species-lepidotrigona-terminata-and-tetragonula-pagdeni-suggest-a-possible-niche-separation-in-a-shared-habitat">Distinct
fungal microbiomes of two Thai commercial stingless bee species,
Lepidotrigona terminata and Tetragonula pagdeni suggest a possible niche
separation in a shared habitat</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Stingless bees, a social corbiculate bee member, play a crucial role in
providing pollination services. Despite their importance, the structure
of their microbiome, particularly the fungal communities, remains poorly
understood. This study presents an initial characterization of the
fungal community associated with two Thai commercial stingless bee
species, <italic>Lepidotrigona terminata</italic> (Smith) and
<italic>Tetragonula pagdeni</italic> (Schwarz) from Chiang Mai,
Thailand. Utilizing ITS amplicon sequencing, we identified distinct
fungal microbiomes in these two species. Notably, fungi from the phyla
Ascomycota, Basidiomycota, Mucoromycota, Mortierellomycota, and
Rozellomycota were present. The most dominant genera, which varied
significantly between species, included <italic>Candida</italic> and
<italic>Starmerella</italic>. Additionally, several key enzymes
associated with energy metabolism, structural strength, and host defense
reactions, such as adenosine triphosphatase, alcohol dehydrogenase,
<italic>Î²</italic>-glucosidase, chitinase, and peptidylprolyl isomerase,
were predicted. Our findings not only augment the limited knowledge of
the fungal microbiome in Thai commercial stingless bees but also provide
insights for their sustainable management through understanding their
microbiome.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1367010">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1367010</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1367010">Save
to Pocket</a> </span></p>
<hr />
<h2
id="droplet-digital-pcr-as-alternative-to-microbiological-culture-for-mycobacterium-tuberculosis-complex-detection-in-bovine-lymph-node-tissue-samples">Droplet
digital PCR as alternative to microbiological culture for Mycobacterium
tuberculosis complex detection in bovine lymph node tissue samples</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Bovine tuberculosis (bTB) caused by <italic>Mycobacterium
tuberculosis</italic> complex (MTC) remains a significant concern for
public health. Direct real-time PCR and droplet digital PCR (ddPCR) are
proposed as alternative tools to enhance diagnostic precision and
efficiency. This study aims to assess the diagnostic performance of a
ddPCR assay targeting IS<italic>6110</italic> for the detection of MTC
DNA in both microbiological culture and fresh lymph node (LN) tissue
samples obtained from cattle, in comparison with the established
reference standard, the microbiological culture followed by real-time
PCR.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
The fresh LNs (N=100) were collected each from a different cattle
carcass at the slaughterhouse. The limit of detection of
ddPCR-IS<italic>6110</italic> was set to 101 copies per 20 Î¼l reaction.
</p>
</sec><sec>
<title>
Results
</title>
<p>
DdPCR-IS<italic>6110</italic> detected 44 out of 49 reference-standard
positive samples and yielded negative results in 47 out of 51
reference-standard negative samples, resulting in adjusted sensitivity
(Se) and specificity (Sp) of 90.76% [95% confidence interval (CI): 82.58
- 98.96%)], and 100% (95% CI: 100%) respectively. The estimated adjusted
false negative rate (FNR) was 9.23% (95% CI: 1.04 - 17.42%) and the
false positive rate (FPR) was 0% (95% CI: 0%). When directly applied
from fresh bovine LN tissues, ddPCR-IS6110 identified 47 out of 49
reference-standard positive samples as ddPCR-IS6110-positive and 42 out
of 51 reference-standard negative samples as
ddPCR-IS<italic>6110</italic>-negative, resulting in adjusted Se and Sp
values of 94.80% [95% (CI): 88.52 - 100%] and 100% (95% CI: 100%),
respectively. The adjusted FNR was 5.20% (95% CI: 0 - 11.50%) and the
FPR was 0% (95% CI: 0%). Noteworthy, ddPCR-IS<italic>6110</italic>
disclosed as positive 9 samples negative to reference-standard.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
DdPCR-IS<italic>6110</italic> proved to be a rapid, highly sensitive,
and specific diagnostic tool as an alternative to reference-standard
method.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349999">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349999</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349999">Save
to Pocket</a> </span></p>
<hr />
<h2
id="strategies-of-helicobacter-pylori-in-evading-host-innate-and-adaptive-immunity-insights-and-prospects-for-therapeutic-targeting">Strategies
of Helicobacter pylori in evading host innate and adaptive immunity:
insights and prospects for therapeutic targeting</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Helicobacter pylori</italic> (<italic>H. pylori</italic>) is the
predominant pathogen causing chronic gastric mucosal infections
globally. During the period from 2011 to 2022, the global prevalence of
<italic>H. pylori</italic> infection was estimated at 43.1%, while in
China, it was slightly higher at approximately 44.2%. Persistent
colonization by <italic>H. pylori</italic> can lead to gastritis, peptic
ulcers, and malignancies such as mucosa-associated lymphoid tissue
(MALT) lymphomas and gastric adenocarcinomas. Despite eliciting robust
immune responses from the host, <italic>H. pylori</italic> thrives in
the gastric mucosa by modulating host immunity, particularly by altering
the functions of innate and adaptive immune cells, and dampening
inflammatory responses adverse to its survival, posing challenges to
clinical management. The interaction between <italic>H. pylori</italic>
and host immune defenses is intricate, involving evasion of host
recognition by modifying surface molecules, manipulating macrophage
functionality, and modulating T cell responses to evade immune
surveillance. This review analyzes the immunopathogenic and immune
evasion mechanisms of <italic>H. pylori</italic>, underscoring the
importance of identifying new therapeutic targets and developing
effective treatment strategies, and discusses how the development of
vaccines against <italic>H. pylori</italic> offers new hope for
eradicating such infections.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342913">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342913</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342913">Save
to Pocket</a> </span></p>
<hr />
<h2
id="a-dynamic-nomogram-to-predict-invasive-fungal-super-infection-during-healthcare-associated-bacterial-infection-in-intensive-care-unit-patients-an-ambispective-cohort-study-in-china">A
dynamic nomogram to predict invasive fungal super-infection during
healthcare-associated bacterial infection in intensive care unit
patients: an ambispective cohort study in China</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Objectives
</title>
<p>
Invasive fungal super-infection (IFSI) is an added diagnostic and
therapeutic dilemma. We aimed to develop and assess a nomogram of IFSI
in patients with healthcare-associated bacterial infection (HABI).
</p>
</sec><sec>
<title>
Methods
</title>
<p>
An ambispective cohort study was conducted in ICU patients with HABI
from a tertiary hospital of China. Predictors of IFSI were selected by
both the least absolute shrinkage and selection operator (LASSO) method
and the two-way stepwise method. The predictive performance of two
models built by logistic regression was internal-validated and compared.
Then external validity was assessed and a web-based nomogram was
deployed.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Between Jan 1, 2019 and June 30, 2023, 12,305 patients with HABI were
screened in 14 ICUs, of whom 372 (3.0%) developed IFSI. Among the fungal
strains causing IFSI, the most common was C.albicans (34.7%) with a
decreasing proportion, followed by C.tropicalis (30.9%), A.fumigatus
(13.9%) and C.glabrata (10.1%) with increasing proportions year by year.
Compared with LASSO-model that included five predictors (combination of
priority antimicrobials, immunosuppressant, MDRO, aCCI and S.aureus),
the discriminability of stepwise-model was improved by 6.8% after adding
two more predictors of COVID-19 and microbiological test before
antibiotics use (P&lt;0.01).And the stepwise-model showed similar
discriminability in the derivation (the area under curve, AUC=0.87) and
external validation cohorts (AUC=0.84, P=0.46). No significant gaps
existed between the proportion of actual diagnosed IFSI and the
frequency of IFSI predicted by both two models in derivation cohort and
by stepwise-model in external validation cohort (P=0.16, 0.30 and 0.35,
respectively).
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The incidence of IFSI in ICU patients with HABI appeared to be a
temporal rising, and our externally validated nomogram will facilitate
the development of targeted and timely prevention and control measures
based on specific risks of IFSI.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1281759">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1281759</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1281759">Save
to Pocket</a> </span></p>
<hr />
<h2
id="the-mla-system-and-its-role-in-maintaining-outer-membrane-barrier-function-in-stenotrophomonas-maltophilia">The
Mla system and its role in maintaining outer membrane barrier function
in Stenotrophomonas maltophilia</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Stenotrophomonas maltophilia</italic> are ubiquitous
Gram-negative bacteria found in both natural and clinical environments.
It is a remarkably adaptable species capable of thriving in various
environments, thanks to the plasticity of its genome and a diverse array
of genes that encode a wide range of functions. Among these functions,
one notable trait is its remarkable ability to resist various
antimicrobial agents, primarily through mechanisms that regulate the
diffusion across cell membranes. We have investigated the Mla ABC
transport system of <italic>S. maltophilia</italic>, which in other
Gram-negative bacteria is known to transport phospholipids across the
periplasm and is involved in maintaining outer membrane homeostasis.
First, we structurally and functionally characterized the periplasmic
substrate-binding protein MlaC, which determines the specificity of this
system. The predicted structure of the <italic>S. maltophilia</italic>
MlaC protein revealed a hydrophobic cavity of sufficient size to
accommodate the phospholipids commonly found in this species. Moreover,
recombinant MlaC produced heterologously demonstrated the ability to
bind phospholipids. Gene knockout experiments in <italic>S.
maltophilia</italic> K279a revealed that the Mla system is involved in
baseline resistance to antimicrobial and antibiofilm agents, especially
those with divalent-cation chelating activity. Co-culture experiments
with <italic>Pseudomonas aeruginosa</italic> also showed a significant
contribution of this system to the cooperation between both species in
the formation of polymicrobial biofilms. As suggested for other
Gram-negative pathogenic microorganisms, this system emerges as an
appealing target for potential combined antimicrobial therapies.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1346565">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1346565</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1346565">Save
to Pocket</a> </span></p>
<hr />
<h2
id="editorial-new-insights-into-the-role-of-tumor-microbial-microenvironment-in-tumor-immunotherapy">Editorial:
New insights into the role of tumor microbial microenvironment in tumor
immunotherapy</h2>
<p>date: 2024-02-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384645">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384645</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384645">Save
to Pocket</a> </span></p>
<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
