<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.10</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../">The Antenna</a></li>
	<li><a href="../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../parks.html" title="new about parks">Parks</a></li>
	<li><a href="../../tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="../../libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="./">Archives</a></li>
	<li><a href="../search.html">Search</a></li>
	<li><a href="../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.10</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.10">health 2024.10</h1>
<p>(date: 2024-03-04 08:41:58)</p>
<hr />
<h2
id="causal-relationship-between-the-gut-microbiota-and-insomnia-a-two-sample-mendelian-randomization-study">Causal
relationship between the gut microbiota and insomnia: a two-sample
Mendelian randomization study</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Changes in the gut microbiota are closely related to insomnia, but the
causal relationship between them is not yet clear.
</p>
</sec><sec>
<title>
Objective
</title>
<p>
To clarify the relationship between the gut microbiota and insomnia and
provide genetic evidence for them, we conducted a two-sample Mendelian
randomization study.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We used a Mendelian randomized two-way validation method to discuss the
causal relationship. First, we downloaded the data of 462,341
participants relating to insomnia, and the data of 18,340 participants
relating to the gut microbiota from a genome-wide association study
(GWAS). Then, we used two regression models, inverse-variance weighted
(IVW) and MR-Egger regression, to evaluate the relationship between
exposure factors and outcomes. Finally, we took a reverse MR analysis to
assess the possibility of reverse causality.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The combined results show 19 gut microbiotas to have a causal
relationship with insomnia (odds ratio (OR): 1.03; 95% confidence
interval (CI): 1.01, 1.05; p=0.000 for class. Negativicutes; OR: 1.03;
95% CI: 1.01, 1.05; p=0.000 for order.Selenomonadales; OR: 1.01; 95% CI:
1.00, 1.02; p=0.003 for genus.RikenellaceaeRC9gutgroup). The results
were consistent with sensitivity analyses for these bacterial traits. In
reverse MR analysis, we found no statistical difference between insomnia
and these gut microbiotas.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
This study can provide a new direction for the causal relationship
between the gut microbiota (class.Negativicutes, order.Selenomonadales,
genus.Lactococcus) and insomnia and the treatment or prevention
strategies of insomnia.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1279218">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1279218</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1279218">Save
to Pocket</a> </span></p>
<hr />
<h2
id="influenza-and-covid-19-co-infection-and-vaccine-effectiveness-against-severe-cases-a-mathematical-modeling-study">Influenza
and COVID-19 co-infection and vaccine effectiveness against severe
cases: a mathematical modeling study</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Influenza A virus have a distinctive ability to exacerbate SARS-CoV-2
infection proven by in vitro studies. Furthermore, clinical evidence
suggests that co-infection with COVID-19 and influenza not only
increases mortality but also prolongs the hospitalization of patients.
COVID-19 is in a small-scale recurrent epidemic, increasing the
likelihood of co-epidemic with seasonal influenza. The impact of
co-infection with influenza virus and SARS-CoV-2 on the population
remains unstudied.
</p>
</sec><sec>
<title>
Method
</title>
<p>
Here, we developed an age-specific compartmental model to simulate the
co-circulation of COVID-19 and influenza and estimate the number of
co-infected patients under different scenarios of prevalent virus type
and vaccine coverage. To decrease the risk of the population developing
severity, we investigated the minimum coverage required for the COVID-19
vaccine in conjunction with the influenza vaccine, particularly during
co-epidemic seasons.
</p>
</sec><sec>
<title>
Result
</title>
<p>
Compared to the single epidemic, the transmission of the SARS-CoV-2
exhibits a lower trend and a delayed peak when co-epidemic with
influenza. Number of co-infection cases is higher when SARS-CoV-2
co-epidemic with Influenza A virus than that with Influenza B virus. The
number of co-infected cases increases as SARS-CoV-2 becomes more
transmissible. As the proportion of individuals vaccinated with the
COVID-19 vaccine and influenza vaccines increases, the peak number of
co-infected severe illnesses and the number of severe illness cases
decreases and the peak time is delayed, especially for those &gt;60
years old.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
To minimize the number of severe illnesses arising from co-infection of
influenza and COVID-19, in conjunction vaccinations in the population
are important, especially priority for the elderly.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347710">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347710</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347710">Save
to Pocket</a> </span></p>
<hr />
<h2
id="detection-of-infectious-sars-cov-2-in-ocular-samples-is-linked-to-viral-load-in-the-nasopharynx">Detection
of infectious SARS-CoV-2 in ocular samples is linked to viral load in
the nasopharynx</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
SARS-CoV-2 is known to infect respiratory tissue cells. However, less is
known about infection of ocular tissue and potential infectivity of
lacrimal fluid. With this study, we want to compare viral loads in eye
and nasopharyngeal swabs and analyze these for infectious virus.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Between May 2020 and April 2021 ocular and nasopharyngeal swabs were
collected from 28 SARS-CoV-2 infected patients treated on the corona
virus disease 2019 (COVID-19)-ward of the University Hospital of
Innsbruck, Austria. Samples with PCR detectable SARS-CoV-2 were analyzed
via whole genome sequencing and an attempt was made to isolate
infectious virus.
</p>
</sec><sec>
<title>
Results
</title>
<p>
At the time point of sample collection, 22 individuals were still PCR
positive in nasopharyngeal samples and in 6 of these patients one or
both ocular samples were additionally positive. CT-values in eyes were
generally higher compared to corresponding nasopharyngeal samples and we
observed a tendency for lower CT-values, i.e. increased viral load, in
nasopharyngeal swabs of individuals with at least one infected eye,
compared to those where ocular samples were PCR negative. Ocular and
nasopharyngeal sequences from the same patient were assigned to the same
variant, either the D614G or the Alpha variant. Infectious virus was
successfully isolated from 9 nasopharyngeal swabs, however only from one
of the seven PCR positive ocular samples.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
We could detect SARS-CoV-2 in eyes of some of the infected patients
albeit at lower levels compared to nasopharyngeal swabs. However, our
results also indicate that lacrimal fluid might be infectious in
patients with high viral load.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1332157">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1332157</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1332157">Save
to Pocket</a> </span></p>
<hr />
<h2
id="development-and-validation-a-nomogram-prediction-model-for-early-diagnosis-of-bloodstream-infections-in-the-intensive-care-unit">Development
and validation a nomogram prediction model for early diagnosis of
bloodstream infections in the intensive care unit</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Purpose
</title>
<p>
This study aims to develop and validate a nomogram for predicting the
risk of bloodstream infections (BSI) in critically ill patients based on
their admission status to the Intensive Care Unit (ICU).
</p>
</sec><sec>
<title>
Patients and methods
</title>
<p>
Patients’ data were extracted from the Medical Information Mart for
Intensive Care−IV (MIMIC−IV) database (training set), the Beijing
Friendship Hospital (BFH) database (validation set) and the eICU
Collaborative Research Database (eICU−CRD) (validation set). Univariate
logistic regression analyses were used to analyze the influencing
factors, and lasso regression was used to select the predictive factors.
Model performance was assessed using area under receiver operating
characteristic curve (AUROC) and Presented as a Nomogram. Various
aspects of the established predictive nomogram were evaluated, including
discrimination, calibration, and clinical utility.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The model dataset consisted of 14930 patients (1444 BSI patients) from
the MIMIC-IV database, divided into the training and internal validation
datasets in a 7:3 ratio. The eICU dataset included 2100 patients (100
with BSI) as the eICU validation dataset, and the BFH dataset included
419 patients (21 with BSI) as the BFH validation dataset. The nomogram
was constructed based on Glasgow Coma Scale (GCS), sepsis related organ
failure assessment (SOFA) score, temperature, heart rate, respiratory
rate, white blood cell (WBC), red width of distribution (RDW), renal
replacement therapy and presence of liver disease on their admission
status to the ICU. The AUROCs were 0.83 (CI 95%:0.81-0.84) in the
training dataset, 0.88 (CI 95%:0.88-0.96) in the BFH validation dataset,
and 0.75 (95%CI 0.70-0.79) in the eICU validation dataset. The clinical
effect curve and decision curve showed that most areas of the decision
curve of this model were greater than 0, indicating that this model has
a certain clinical effectiveness.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The nomogram developed in this study provides a valuable tool for
clinicians and nurses to assess individual risk, enabling them to
identify patients at a high risk of bloodstream infections in the ICU.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348896">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348896</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348896">Save
to Pocket</a> </span></p>
<hr />
<h2
id="phage-p2-71-against-multi-drug-resistant-proteus-mirabilis-isolation-characterization-and-non-antibiotic-antimicrobial-potential">Phage
P2-71 against multi-drug resistant Proteus mirabilis: isolation,
characterization, and non-antibiotic antimicrobial potential</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Proteus mirabilis</italic>, a prevalent urinary tract pathogen
and formidable biofilm producer, especially in Catheter-Associated
Urinary Tract Infection, has seen a worrying rise in multidrug-resistant
(MDR) strains. This upsurge calls for innovative approaches in infection
control, beyond traditional antibiotics. Our research introduces
bacteriophage (phage) therapy as a novel non-antibiotic strategy to
combat these drug-resistant infections. We isolated P2-71, a lytic phage
derived from canine feces, demonstrating potent activity against MDR
<italic>P. mirabilis</italic> strains. P2-71 showcases a notably brief
10-minute latent period and a significant burst size of 228 particles
per infected bacterium, ensuring rapid bacterial clearance. The phage
maintains stability over a broad temperature range of 30-50°C and within
a pH spectrum of 4-11, highlighting its resilience in various
environmental conditions. Our host range assessment solidifies its
potential against diverse MDR <italic>P. mirabilis</italic> strains.
Through killing curve analysis, P2-71’s effectiveness was validated at
various MOI levels against <italic>P. mirabilis</italic> 37,
highlighting its versatility. We extended our research to examine
P2-71’s stability and bactericidal kinetics in artificial urine,
affirming its potential for clinical application. A detailed genomic
analysis reveals P2-71’s complex genetic makeup, including genes
essential for morphogenesis, lysis, and DNA modification, which are
crucial for its therapeutic action. This study not only furthers the
understanding of phage therapy as a promising non-antibiotic
antimicrobial but also underscores its critical role in combating
emerging MDR infections in both veterinary and public health contexts.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347173">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347173</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347173">Save
to Pocket</a> </span></p>
<hr />
<h2
id="airway-necrosis-and-granulation-tissue-formation-caused-by-rhizopus-oryzae-leading-to-severe-upper-airway-obstruction-a-case-report">Airway
necrosis and granulation tissue formation caused by Rhizopus oryzae
leading to severe upper airway obstruction: a case report</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Pulmonary Mucormycosis is a fatal infectious disease with high mortality
rate. The occurrence of Mucormycosis is commonly related to the fungal
virulence and the host’s immunological defenses against pathogens.
Mucormycosis infection and granulation tissue formation occurred in the
upper airway was rarely reported. This patient was a 60-year-old male
with diabetes mellitus, who was admitted to hospital due to progressive
cough, sputum and dyspnea. High-resolution computed tomography (HRCT)
and bronchoscopy revealed extensive tracheal mucosal necrosis,
granulation tissue proliferation, and severe airway stenosis. The
mucosal necrotic tissue was induced by the infection of <italic>Rhizopus
Oryzae</italic>, confirmed by metagenomic next-generation sequencing
(mNGS) in tissue biopsy. This patient was treated with the placement of
a covered stent and local instillation of amphotericin B via
bronchoscope. The tracheal mucosal necrosis was markedly alleviated, the
symptoms of cough, shortness of breath, as well as exercise tolerance
were significantly improved. The placement of airway stent and
transbronchial microtube drip of amphotericin B could conduce to rapidly
relieve the severe airway obstruction due to Mucormycosis infection.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366472">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366472</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366472">Save
to Pocket</a> </span></p>
<hr />
<h2
id="optimal-selection-of-specimens-for-metagenomic-next-generation-sequencing-in-diagnosing-periprosthetic-joint-infections">Optimal
selection of specimens for metagenomic next-generation sequencing in
diagnosing periprosthetic joint infections</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Objective
</title>
<p>
This study aimed to assess the diagnostic value of metagenomic
next-generation sequencing (mNGS) across synovial fluid, prosthetic
sonicate fluid, and periprosthetic tissues among patients with
periprosthetic joint infection (PJI), intending to optimize specimen
selection for mNGS in these patients.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This prospective study involved 61 patients undergoing revision
arthroplasty between September 2021 and September 2022 at the First
Affiliated Hospital of Zhengzhou University. Among them, 43 cases were
diagnosed as PJI, and 18 as aseptic loosening (AL) based on the American
Musculoskeletal Infection Society (MSIS) criteria. Preoperative or
intraoperative synovial fluid, periprosthetic tissues, and prosthetic
sonicate fluid were collected, each divided into two portions for mNGS
and culture. Comparative analyses were conducted between the
microbiological results and diagnostic efficacy derived from mNGS and
culture tests. Furthermore, the variability in mNGS diagnostic efficacy
for PJI across different specimen types was assessed.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The sensitivity and specificity of mNGS diagnosis was 93% and 94.4% for
all types of PJI specimens; the sensitivity and specificity of culture
diagnosis was 72.1% and 100%, respectively. The diagnostic sensitivity
of mNGS was significantly higher than that of culture (X<sup>2</sup> =
6.541, <italic>P</italic>=0.011), with no statistically significant
difference in specificity (X<sup>2</sup> = 1.029,
<italic>P</italic>=0.310). The sensitivity of the synovial fluid was
83.7% and the specificity was 94.4%; the sensitivity of the prosthetic
sonicate fluid was 90.7% and the specificity was 94.4%; and the
sensitivity of the periprosthetic tissue was 81.4% and the specificity
was 100%. Notably, the mNGS of prosthetic sonicate fluid displayed a
superior pathogen detection rate compared to other specimen types.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
mNGS can function as a precise diagnostic tool for identifying pathogens
in PJI patients using three types of specimens. Due to its superior
ability in pathogen identification, prosthetic sonicate fluid can
replace synovial fluid and periprosthetic tissue as the optimal sample
choice for mNGS.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356804">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356804</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356804">Save
to Pocket</a> </span></p>
<hr />
<h2
id="identifying-potential-monkeypox-virus-inhibitors-an-in-silico-study-targeting-the-a42r-protein">Identifying
potential monkeypox virus inhibitors: an in silico study targeting the
A42R protein</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Monkeypox (now Mpox), a zoonotic disease caused by the monkeypox virus
(MPXV) is an emerging threat to global health. In the time span of only
six months, from May to October 2022, the number of MPXV cases breached
80,000 and many of the outbreaks occurred in locations that had never
previously reported MPXV. Currently there are no FDA-approved
MPXV-specific vaccines or treatments, therefore, finding drugs to combat
MPXV is of utmost importance. The A42R profilin-like protein of the MPXV
is involved in cell development and motility making it a critical drug
target. A42R protein is highly conserved across orthopoxviruses, thus
A42R inhibitors may work for other family members. This study sought to
identify potential A42R inhibitors for MPXV treatment using
computational approaches. The energy minimized 3D structure of the A42R
profilin-like protein (PDB ID: 4QWO) underwent virtual screening using a
library of 36,366 compounds from Traditional Chinese Medicine (TCM),
AfroDb, and PubChem databases as well as known inhibitor tecovirimat via
AutoDock Vina. A total of seven compounds comprising PubChem CID:
11371962, ZINC000000899909, ZINC000001632866, ZINC000015151344,
ZINC000013378519, ZINC000000086470, and ZINC000095486204, predicted to
have favorable binding were shortlisted. Molecular docking suggested
that all seven proposed compounds have higher binding affinities to A42R
(–7.2 to –8.3 kcal/mol) than tecovirimat (–6.7 kcal/mol). This was
corroborated by MM/PBSA calculations, with tecovirimat demonstrating the
highest binding free energy of –68.694 kJ/mol (lowest binding affinity)
compared to the seven shortlisted compounds that ranged from –73.252 to
–97.140 kJ/mol. Furthermore, the 7 compounds in complex with A42R
demonstrated higher stability than the A42R-tecovirimat complex when
subjected to 100 ns molecular dynamics simulations. The protein-ligand
interaction maps generated using LigPlot+ suggested that residues Met1,
Glu3, Trp4, Ile7, Arg127, Val128, Thr131, and Asn133 are important for
binding. These seven compounds were adequately profiled to be potential
antivirals via PASS predictions and structural similarity searches. All
seven potential lead compounds were scored Pa &gt; Pi for antiviral
activity while ZINC000001632866 and ZINC000015151344 were predicted as
poxvirus inhibitors with Pa values of 0.315 and 0.215, and Pi values of
0.052 and 0.136, respectively. Further experimental validations of the
identified lead compounds are required to corroborate their predicted
activity. These seven identified compounds represent solid footing for
development of antivirals against MPXV and other orthopoxviruses.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351737">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351737</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351737">Save
to Pocket</a> </span></p>
<hr />
<h2
id="functional-characterization-of-francisella-tularensis-subspecies-holarctica-genotypes-during-tick-cell-and-macrophage-infections-using-a-proteogenomic-approach">Functional
characterization of Francisella tularensis subspecies holarctica
genotypes during tick cell and macrophage infections using a
proteogenomic approach</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Tularemia is a vector-borne disease caused by the Gram-negative
bacterium <italic>Francisella tularensis</italic>. Known hosts and
vectors in Europe are hare and ticks. <italic>F. tularensis</italic> is
transmitted from ticks and animals, but also from the hydrotelluric
environment and the consumption of contaminated water or food. A
changing climate expands the range in which ticks can live and
consequently might contribute to increasing case numbers of tularemia.
Two subspecies of <italic>F. tularensis</italic> are human pathogenic.
<italic>Francisella tularensis tularensis (Ftt)</italic> is endemic in
North America, while <italic>Francisella tularensis holarctica
(Fth)</italic> is the only subspecies causing tularemia in Europe.
<italic>Ft</italic> is classified as a category A bioterrorism agent due
to its low infectious dose, multiple modes of transmission, high
infectivity and potential for airborne transmission and has become a
global public health concern. In line with the European survey and
previous phylogenetic studies, Switzerland shows the co-distribution of
B.6 and B.12 strains with different geographical distribution and
prevalence within the country. To establish itself in different host
environments of ticks and mammals, <italic>F. tularensis</italic>
presumably undergoes substantial changes on the transcriptomics and
proteomic level. Here we investigate the transcriptomic and proteomic
differences of five strains of <italic>Fth</italic> upon infection of
rabbit macrophages and tick cells.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1355113">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1355113</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1355113">Save
to Pocket</a> </span></p>
<hr />
<h2
id="bumble-bee-microbiota-shows-temporal-succession-and-increase-of-lactic-acid-bacteria-when-exposed-to-outdoor-environments">Bumble
bee microbiota shows temporal succession and increase of lactic acid
bacteria when exposed to outdoor environments</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Question
</title>
<p>
The large earth bumble bee (<italic>Bombus terrestris</italic>)
maintains a social core gut-microbiota, similar as known from the honey
bee, which plays an important role for host health and resistance.
Experiments under laboratory conditions with commercial hives are
limited to vertically transmitted microbes and neglect influences of
environmental factors or external acquisition of microbes. Various
environmental and landscape-level factors may have an impact on the
gut-microbiota of pollinating insects, with consequences for pollinator
health and fitness in agroecosystems. Still, it is not fully clear
whether access to different flower diversities will have a significant
influence on the bumble bee microbiota. Here, we tested in a semi-field
experiment if the bumble bee microbiota changes over time when exposed
to different flower diversities within outdoor flight cages. We used
commercial hives to distinguish between vertically and horizontally
transmitted bacteria, respectively from the nest environment or the
exposed outside environment.
</p>
</sec><sec>
<title>
Result
</title>
<p>
The sequential sampling of foraging workers over a period of 35 days
indicated a temporal progression of the bumble bee microbiota when
placed outside. The microbiota increased in diversity and changed in
composition and variability over time. We observed a major increase in
relative abundance of the families <italic>Lactobacillaceae</italic>,
<italic>Bifidobacteriaceae</italic> and <italic>Weeksellaceae</italic>.
In contrast, major core-taxa like <italic>Snodgrassella</italic> and
<italic>Gilliamella</italic> declined in their relative abundance over
time. The genus <italic>Lactobacillus</italic> showed a high diversity
and strain specific turnover, so that only specific ASVs showed an
increase over time, while others had a more erratic occurrence pattern.
Exposure to different flower diversities had no significant influence on
the progression of the bumble bee microbiota.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The bumble bee microbiota showed a dynamic temporal succession with
distinct compositional changes and diversification over time when placed
outdoor. The exposure of bumble bees to environmental conditions, or
environmental microbes, increases dissimilarity and changes the
gut-community composition. This shows the importance of environmental
influences on the temporal dynamic and progression of the bumble bee
microbiota.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342781">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342781</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342781">Save
to Pocket</a> </span></p>
<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
