<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2025-05-13 14:07:51)</p>
<hr />
<h2
id="the-intratumoral-microbiota-heterogenicity-is-related-to-the-prognosis-and-tumorigenesis-of-cervical-cancer">The
intratumoral microbiota heterogenicity is related to the prognosis and
tumorigenesis of cervical cancer</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe intratumoral microbe-host interaction plays crucial
role in the development of cancer. The microbiome can influence cancer
development by modulating inflammation, immune responses and metabolic
pathways. Therefore, we aim to delineate the landscape and role of
intratumoral microbiota in cervical cancer (CC).MethodsThe presence of
bacterial community in CC tissues was confirmed by fluorescence in situ
hybridization (FISH). Then 16s rRNA and RNA-Seq were used to
characterize the composition of intratumoral microbiota. Combined with
cervical squamous cell carcinoma (CESC) data from the Tumor Cancer
Genome Atlas (TCGA), the clinical signatures of intratumoral microbiota
and DEGs were further analyzed. Finally, the effect of the up-regulated
Fibrinogen beta chain (FGB) expressed fragment peptide on the biological
behavior of cancer was verified in vitro.ResultsWe found the composition
heterogeneity of bacteria in CC tumors. Pseudomonas was most highly
enriched in CC tissues and grouped according to the relative abundance
level. The clinical characteristics of patients with relatively high
abundance of Pseudomonas had the higher levels of fibrinogen and lower
levels of white blood cell (WBC) and albumin (ALB) expression. Combining
transcriptome data from the two our collective CC and TCGA-CESC cohorts,
we found that Pseudomonas abundance was significantly associated with
fibrinogen beta peptide expression and worse overall survival in CC
patients. In vitro experiment revealed that Pseudomonas could promote
the proliferation and migration of cervical cancer cells through
overexpression of FGB.ConclusionsWe characterized the composition of the
intratumoral microbiota in CC tissues and identified the most
significantly differentially abundant bacteria between cancerous and
non-cancerous tissues. Our findings provide novel insights into the
relationship between intratumoral Pseudomonas and the tumorigenesis of
CC. A deeper understanding of the tumor microenvironment and its
associated microbiota may reveal new potential therapeutic targets and
improve clinical outcomes.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1574511"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1574511</a></p>
<hr />
<h2
id="involvement-of-the-blactx-m-3-gene-in-emergence-of-a-peculiar-resistance-phenotype-in-klebsiella-pneumoniae">Involvement
of the blaCTX-M-3 gene in emergence of a peculiar resistance phenotype
in Klebsiella pneumoniae</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThis study aimed to investigate the mechanism underlying
a peculiar resistance phenotype in Klebsiella pneumoniae, characterized
by reduced susceptibility to cefepime compared to
ceftazidime.MethodsAntimicrobial susceptibility testing, plasmid
conjugation experiments, whole-genome sequencing, and bioinformatic
analyses were employed to characterize the resistance phenotype and
identify genetic determinants.ResultsA total of 20 K. pneumoniae strains
exhibiting peculiar resistance phenotypes were collected and analyzed.
Ten distinct sequence types (STs) were identified, including ST25
(4/20), ST967 (3/20), ST65 (2/20), ST133 (2/20), ST48 (2/20), ST353
(1/20), ST628 (1/20), ST753 (1/20), ST792 (1/20), and ST254 (1/20). All
strains were resistant to FEP (MIC50 = 128 µg/mL) but not to CAZ (MIC50
= 8 µg/mL). This resistance was primarily attributed to the presence of
the blaCTX-M-3 (14/20) and blaOXA-10 (3/20). Conjugation experiments
demonstrated that 5 out of 14 blaCTX-M-3-positive K. pneumoniae strains
successfully acquired transconjugants, which exhibited the same peculiar
resistance phenotype. PCR analysis confirmed that the conjugates
contained the IncFII plasmid. To further elucidate the genetic basis of
the resistance phenotype, whole-genome long-read sequencing was
performed on three blaCTX-M-3-positive K. pneumoniae strains. The
sequencing results confirmed that blaCTX-M-3 was located on the IncFII
plasmid, and analysis of its genetic environment revealed a frequent
association with mobile genetic elements such as IS26, ISEcp1, and
Tn3.DiscussionThe primary driver of this phenotype in K. pneumoniae is
the presence of the IncFII plasmid carrying blaCTX-M-3, which contrasts
with the resistance mechanisms often reported in Pseudomonas aeruginosa
exhibiting similar phenotypes. This study emphasizes the critical role
of plasmid-mediated resistance in the spread of multidrug resistance in
K. pneumoniae and provides insights into strategies for combating
resistance in these pathogens.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545157"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545157</a></p>
<hr />
<h2
id="study-on-the-impact-of-covid-19-epidemic-and-agent-disease-risk-simulation-model-based-on-individual-factors-in-xian-city">Study
on the impact of COVID-19 epidemic and agent disease risk simulation
model based on individual factors in Xi’an City</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionSince the first discovery and reporting of the COVID - 19
pandemic towards the end of 2019, the virus has rapidly propagated
across the world. This has led to a remarkable spike in the number of
infections. Even now, doubt lingers over whether it has completely
disappeared. Moreover, the issue of restoring normal life while ensuring
safety continues to be a crucial challenge that public health agencies
and people globally are eager to tackle.MethodsTo thoroughly understand
the epidemic’s outbreak and transmission traits and formulate timely
prevention measures to fully safeguard human lives and property, this
paper presents an agent - based model incorporating individual - level
factors.ResultsThe model designates Xi’an—where a characteristic disease
outbreak occurred—as the research area. The simulation results
demonstrate substantial consistency with official records, effectively
validating the model’s applicability, adaptability, and
generalizability. This validated capacity enables accurate prediction of
epidemic trends and comprehensive assessment of disease
risks.DiscussionFrom late 2021 to early 2022, it employs a one - to -
one population simulation approach and simulates epidemic impacts and
disease risks. Initially, using building statistical data in the study
area, the model reconstructs the local real - world geographical
environment. Leveraging data from the seventh national population
census, it also replicates the study area’s population characteristics.
Next, the model takes into account population mobility, contact tracing,
patient treatment, and the diagnostic burden of COVID - 19 - like
influenza symptoms. It integrates epidemic transmission impact
parameters into the model framework. Eventually, the model’s results are
compared with official data for validation, and it’s applied to
hypothetical scenarios. It provides scientific theoretical tools to
support the implementation of government - driven prevention and control
measures. Additionally, it facilitates the adjustment of individual
behavioral guidelines, promoting more effective epidemic management.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547601"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547601</a></p>
<hr />
<h2
id="overexpression-in-plasmodium-falciparum-of-an-intrinsically-disordered-protein-segment-of-pfut-impairs-the-parasites-proteostasis-and-reduces-its-growth-rate">Overexpression
in Plasmodium falciparum of an intrinsically disordered protein segment
of PfUT impairs the parasite’s proteostasis and reduces its growth
rate</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The proteome of Plasmodium falciparum exhibits a marked propensity
for aggregation. This characteristic results from the parasite’s AT-rich
genome, which encodes numerous proteins with long asparagine-rich
stretches and low structural complexity, which lead to abundant
intrinsically disordered regions. While this poses challenges for the
parasite, the propensity for protein aggregation may also serve
functional roles, such as stress adaptation, and could therefore be
exploited by targeting it as a potential vulnerable spot in the
pathogen. Here, we overexpressed an aggregation-prone segment of the P.
falciparum ubiquitin transferase (PfUTf), an E3 ubiquitin ligase protein
that has been previously demonstrated to regulate the stability of
parasite proteins involved in invasion, development and drug metabolism.
Overexpression of PfUTf in P. falciparum had evident phenotypic effects
observed by transmission electron microscopy and confocal fluorescence
microscopy, increased endogenous protein aggregation, disrupted
proteostasis, and caused significant growth impairment in the parasite.
Combined with dihydroartemisinin treatment, PfUTf overexpression had a
synergistic effect that further compromised the parasite´s viability,
linking protein aggregation to proteasome dysfunction. Changes in the
distribution of aggregation-prone proteins, shown by the altered
subcellular fluorescent pattern of the new investigational aggregated
protein dye and antiplasmodial compound YAT2150 in the overexpressing P.
falciparum line, highlighted the critical balance between protein
aggregation, stress responses, and parasite viability, suggesting
proteostasis-targeting therapies as a good antimalarial strategy.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565814"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565814</a></p>
<hr />
<h2
id="visual-detection-of-coxsackievirus-a6-using-a-reverse-transcription-polymerase-spiral-reaction-method">Visual
detection of Coxsackievirus A6 using a reverse transcription polymerase
spiral reaction method</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Coxsackievirus A6 (CVA6) ranks as a primary enterovirus associated
with hand-foot-mouth disease (HFMD) and herpangina (HA). Given its
significant role in these diseases, there is an urgent need for an
efficient identification method. This study presents a novel visual
approach based on the reverse transcription polymerase spiral reaction
(RT-PSR) for the rapid detection of CVA6. We designed an RT-PSR assay
that targets and amplifies a segment of the VP1 gene. Hydroxy naphthol
blue (HNB) is incorporated as the detection agent in this assay. To
evaluate the performance of the RT-PSR assay, we analyzed 142 clinical
throat swab samples. The results were benchmarked against those obtained
using quantitative reverse transcription - polymerase chain reaction
(qRT - PCR). The RT-PSR assay operates at 65°C for 60 minutes and
exhibits a detection limit of 10 copies/μL. When tested against other
viruses, it consistently yielded negative results, demonstrating its
high specificity. Moreover, the RT - PSR assay showed excellent
agreement with a commercial qRT - PCR kit. In conclusion, by using HNB
as an indicator, the RT - PSR assay emerges as a straightforward and
highly sensitive method for detecting CVA6 in symptomatic throat
samples. This approach holds great potential for improving the diagnosis
and surveillance of CVA6 - related diseases.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563495"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563495</a></p>
<hr />
<h2
id="unveiling-the-role-of-the-upper-respiratory-tract-microbiome-in-susceptibility-and-severity-to-covid-19">Unveiling
the role of the upper respiratory tract microbiome in susceptibility and
severity to COVID-19</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>It is argued that commensal bacteria in the upper respiratory tract
(URT) protect against pathogen colonization and infection, including
respiratory viruses. Given that the microbiome can mediate immune
modulation, a link between the URT microbiome (URTM) and COVID-19
susceptibility and severity is expected. This 16S metagenomics
cross-sectional study assessed URTM composition, metabolic prediction,
and association with laboratory biomarkers in non-COVID-19 pneumonia
(NO-CoV), moderate (M-CoV), severe (S-CoV) COVID-19 patients, as well as
COVID-19-negative, asymptomatic (NC) patients. The S-CoV group exhibited
reduced URTM diversity, primarily due to a decreased abundance of
eubiotic taxa. Some of these taxa (e.g., Haemophilus sp., Neisseria sp.)
were also associated with inflammatory biomarkers. Multiple metabolic
pathways (e.g., short-chain fatty acids, vitamin B12) linked to immune
response, antiviral activity, and host susceptibility showed decreased
abundance in S-CoV. These pathways could suggest potential alternatives
for the therapeutic arsenal against COVID-19, providing reassurance
about the progress in understanding and treating this disease.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1531084"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1531084</a></p>
<hr />
<h2
id="chlamydia-muridarum-mutant-cm-pgp3s-as-a-novel-attenuated-live-rectal-vaccine-protects-against-genital-tract-infection">Chlamydia
muridarum mutant CM-pGP3S as a novel attenuated live rectal vaccine
protects against genital tract infection</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionChlamydia trachomatis (CT) is a major sexually
transmitted pathogen with severe complications. Using Chlamydia
muridarum (CM) as a model, this study evaluates the attenuated mutant
Chlamydia muridarum (CM-pGP3S) as a novel rectal vaccine to protect
against genital tract infection and pathology.MethodsFemale C57BL/6 mice
were rectally immunized with low (1×103), middle (1×105), or high
(1×107) doses of CM-pGP3S. Mice were challenged intravaginally with
wild-type Chlamydia muridarum 63 days post-immunization. Protection was
assessed via genital Chlamydia shedding, hydrosalpinx incidence
(gross/histopathology), serum IgG, fecal IgA, and T cell responses. Gut
microbiota stability was analyzed using qPCR.ResultsCM-pGP3S
immunization significantly reduced CM-WT genital shedding duration (3–7
days vs. 21 days in controls, p &lt; 0.01) and hydrosalpinx incidence
(0% vs. 80% in controls, p &lt; 0.01). Elevated systemic and mucosal
immunity were observed, including higher serum IgG (1:100–1:1600
dilutions, p &lt; 0.05–0.01) and fecal IgA (p &lt; 0.05–0.01). CD4+ and
CD8+ T cells exhibited increased IFN-γ (p &lt; 0.01), while CD8+ T cells
showed elevated TNF-α and IL-2 (p &lt; 0.05). No colitis or significant
gut microbiota disruption occurred post-immunization.DiscussionRectal
CM-pGP3S vaccination induces robust transmucosal immunity, protecting
against genital Chlamydia infection and pathology without
gastrointestinal adverse effects. This highlights its potential as a
safe and effective mucosal vaccine strategy to combat CT genital tract
infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1550455"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1550455</a></p>
<hr />
<h2
id="editorial-can-chinese-medicines-affect-diarrhea-via-effects-of-the-intestinal-microbiota-on-the-renal-intestinal-axis">Editorial:
Can Chinese medicines affect diarrhea via effects of the intestinal
microbiota on the renal-intestinal axis?</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593758"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593758</a></p>
<hr />
<h2
id="the-role-of-periostin-osf-2-in-the-cytoadherence-phenomena-mediated-by-malaria-parasites">The
role of periostin (OSF-2) in the cytoadherence phenomena mediated by
malaria parasites</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe pathogenesis of severe malaria is primarily
attributed to the cytoadherence properties of Plasmodium-infected
erythrocytes (IRBC), which include rosetting and IRBC-endothelial
cytoadherence. These cytoadherence events are influenced by various
parasite- and host-derived factors. Previously, antibodies against human
periostin (OSF-2), an inflammation-associated protein, were reported to
inhibit rosetting. In this study, we aimed to characterize the
OSF-2-mediated cytoadherence in infections caused by Plasmodium
falciparum (the most fatal human malaria parasite) and P. knowlesi (an
emerging, potentially fatal zoonotic malaria
parasite).MethodsLaboratory-adapted P. falciparum and P. knowlesi
isolates were cultured, and the late-stage parasites were purified for
experiments using recombinant human OSF-2.ResultsWe found that OSF-2 at
a concentration of 200 ng/ml induced rosette-stimulation in both
parasite species. Furthermore, we demonstrated the serum dependency of
OSF-2-mediated rosetting. The rosette-stimulating effect of OSF-2 was
completely abolished when IRBC were treated with a low concentration of
trypsin. This suggests a role for P. falciparum erythrocyte membrane
protein 1 (PfEMP1) in OSF-2-mediated rosetting by P. falciparum, and
reveals the trypsin-sensitive nature of the P. knowlesi-derived ligands
involved in OSF-2-mediated rosetting. We also found that OSF-2-mediated
rosetting was independent of the ABO blood group. Additionally, we
demonstrated the ability of OSF-2 to disrupt the IRBC-endothelial
binding.DiscussionThis work contributes to our understanding of the
host-parasite interactions in malaria pathobiology.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1599872"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1599872</a></p>
<hr />
<h2
id="the-progress-of-the-microbe-gut-brain-axis-in-sepsis-associated-encephalopathy">The
progress of the microbe-gut-brain axis in sepsis-associated
encephalopathy</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Sepsis-associated encephalopathy (SAE) is a diffuse brain dysfunction
that is caused by sepsis without direct brain injury or central nervous
system infection and is manifested as anxiety-like behavior and
cognitive dysfunction. The microbiota-gut-brain axis, on the other hand,
is a bidirectional communication network between the gut and the brain
that modulates host behavior and cognitive function in many ways and is
of central importance in the preservation of general health and
homeostasis. Given the functional roles attributed to the
microbiota-gut-brain axis (MGBA), contemporary research is progressively
focused on elucidating relationships between SAE and alterations in
compositional and quantitative intestinal microbiota profiles. This
review consolidates interdisciplinary insights from immunology,
microbiology, neuroendocrine signaling, and neural pathophysiology to
evaluate the mechanistic contribution of the MGBA to the relief of
cognitive impairments in SAE. By unifying these perspectives, with the
aim of preventing or enhancing SAE-related neurological dysfunction for
the formulation of MGBA-targeted therapeutic strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587463"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587463</a></p>
<hr />
<h2
id="association-between-oral-dysbiosis-and-parkinsons-disease-a-systematic-review">Association
between oral dysbiosis and Parkinson’s disease: a systematic review</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The oral cavity serves as the gateway to the human organism, hosting
a diverse community of microorganisms that coexist in a state of
symbiosis. Disruption of this balance leads to oral dysbiosis, a
condition associated with infections and oral pathologies, which may
contribute to the etiopathogenesis of systemic disorders such as
Parkinson’s disease, a neurodegenerative movement disorder characterized
by resting tremor, rigidity, and bradykinesia. While oral dysbiosis is
recognized as a risk factor and an aggravating element for Parkinson’s
disease, it is not regarded as a direct cause. This systematic review
aims to synthesize existing research exploring the potential
relationship between oral dysbiosis and the development of Parkinson’s
disease. Following a comprehensive analysis, 12 studies were selected,
comprising 11 case-control studies and one observational analytical
study. These studies investigated the composition of oral microbiota in
different sample groups, revealing a higher abundance of pathogenic oral
bacteria in individuals diagnosed with Parkinson’s disease. The findings
suggest that oral dysbiosis may influence both the onset of Parkinson’s
disease and the progression of symptoms such as cognitive decline. These
results pave the way for future research, particularly regarding
alterations in oral microbiota as potential biomarkers for early
diagnosis and disease monitoring.Systematic review
registrationhttps://www.crd.york.ac.uk/prospero/, identifier
CRD42024540056.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1564362"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1564362</a></p>
<hr />
<h2
id="screening-and-identification-of-protein-interacting-with-goose-astrovirus">Screening
and identification of protein interacting with goose astrovirus</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionGoose Astrovirus (GoAstV), a recently identified member
of the Astroviridae family in China, predominantly affects goslings,
resulting in substantial economic losses to the goose farming industry
due to its high infection and mortality rates. Currently, the infection
mechanism and pathogenesis of GoAstV remain unknown.MethodsGiven this,
the Viral Overlay Protein Blot Assay was utilized to identify and
characterize proteins on the LMH (Leghorn Male Hepatoma) cell
membrane that interact with Goose Astrovirus. The identities of the
candidate proteins were determined via LC-MS mass spectrometry analysis,
bioinformatics analysis, and UniProt database search. The interaction
between HSPA5 and the astrovirus protein was further validated in vitro
through Western blot and Coimmunoprecipitation experiments. Finally,
bioinformatics tools such as SWISSMODEL, AlphaFold, and ZDOCK were
employed to construct and analyze the docking complex model between the
candidate protein and GoAstV protein, including their key binding
residue sites.ResultsWe successfully identified a 70 kDa protein in the
plasma membrane protein extracts of LMH cells and confirmed the identity
of this candidate protein as HSPA5. Meanwhile, in vitro experiments
further validated the interaction between HSPA5 and astrovirus proteins.
Subsequently, we successfully predicted the docking complex model of
HSPA5 protein with GoAstV protein. Further prediction of the binding
residue sites revealed that seven residues of the GoAstV-P2 protein
(THR124, ILE22, VAL24, TRP51, PRO66, GLN100, and VAL125) and twelve
residues of the HSPA5 protein (ARG2, HIS3, LEU4, LEU6, ALA7, LEU8, LEU9,
LEU10, LEU11, ASP411, VAL413, and LEU415) may be involved in the
interaction between these two proteins.DiscussionOur research results
have preliminarily elucidated the interaction mechanisms between viral
proteins and receptors, facilitating exploration from multiple angles of
the roles of candidate protein in the process of GoAstV infecting host
cells. This provides a theoretical basis for further identification of
GoAstV receptors and clarification of its infection mechanisms.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1595736"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1595736</a></p>
<hr />
<h2
id="hmuy-proteins-of-the-porphyromonas-genus-show-diversity-in-heme-binding-properties">HmuY
proteins of the Porphyromonas genus show diversity in heme-binding
properties</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionBacteria of the Porphyromonas genus, belonging to the
Bacteroidota phylum, colonize various host niches in health and disease.
As heme auxotrophs, they rely on heme uptake for iron and protoporphyrin
IX. A key heme acquisition system in Porphyromonas gingivalis is the Hmu
system, where the hemophore-like HmuYPg protein plays a major role.
HmuYPg coordinates heme-iron using two histidines, whereas other known
HmuY proteins produced by other Bacteroidota members prefer a pair of
histidine-methionine or two methionines. Some of them bind heme via the
protoporphyrin ring without heme-iron coordination, similar to the P.
gingivalis HusA protein.MethodsThis study used bioinformatics,
spectroscopic, and electrophoretic methods to compare the genomic
organization of the Hmu system and the structural and functional
properties of HmuY proteins within the Porphyromonas genus.Results and
DiscussionWe revealed variations in the heme-binding properties of
proteins belonging to the HmuY family and susceptibility to
modifications in their heme-binding pockets. These findings suggest that
HmuY proteins may have undergone evolutionary adaptations to enhance
bacterial survival in the human microbiome, contributing to dysbiosis
and disease development. These evolutionary changes may explain the
superior heme-binding ability of P. gingivalis HmuYPg compared to HmuY
homologs produced by other Porphyromonas species.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560779"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560779</a></p>
<hr />
<h2
id="a-pharmacovigilance-study-on-probiotic-preparations-based-on-the-fda-adverse-event-reporting-system-from-2005-to-2023">A
pharmacovigilance study on probiotic preparations based on the FDA
Adverse Event Reporting System from 2005 to 2023</h2>
<p>date: 2025-05-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundProbiotics are recognized as beneficial foods, but adverse
reactions reported by individuals still exist. This study aims to
analysis adverse events (AE) related to probiotics from the FAERS
database from the first quarter (Q1) of 2005 to the fourth quarter (Q4)
of 2023.MethodsThe AE data related to probiotic from the 2005 Q1 to the
2023 Q4 were collected. R language was applied to analyze the
standardized AE data and three algorithms including the reporting odds
ratio (ROR), the proportional reporting ratio (PRR) and the empirical
Bayes geometric mean (EBGM) were used to identify AE signals.ResultsIn
this study, 10,698,312 reports were collected from the FAERS database,
of which 74 probiotic-related adverse events were reported. About one
third of the reported cases were older than 60 years.36.36% of the
reported cases required Hospitalization. A total of 285 preference terms
(PTS) and 15 system organ classes (SOC) were identified. In the overall
analysis, only 9 PTs and 2 SOCs met significant disproportionality for
all three algorithms simultaneously. SOCs included Gastrointestinal
disorders (N=97, ROR=5.3, PRR=3.84, EBGM=3.84) and Hepatobiliary
disorders (N=9, ROR =3.39, PRR=3.32, EBGM=3.32). PTs included
Gastrointestinal pain (ROR=77.76, PRR=76.69, EBGM=76.63), Hypophagia
(ROR=24.13, PRR=23.88, EBGM=28.88), and Hepatobiliary disorders (N=97,
ROR=5.3, PRR=3.84, EBGM=3.84) and Flatulence (ROR=23.75, PRR=23.28,
EBGM=23.27) were the top four highest. Meanwhile, s found new unique
adverse signals such as Agitation (ROR=12.48, PRR=12.32, EBGM=12.32) and
Anxiety (ROR=4.10, PRR=4.04, EBGM=4.04). Additionally, subgroup analyses
were performed to identify AE signals based on gender and age.
Metabolism and nutrition disorders (N=6, ROR=3.21, PRR=3.04, EBGM=3.04)
and Asthenia (N=3, ROR=5.9, PRR=5.71, EBGM=5.71) were unique AE signal
for the male group.ConclusionAlthough, the risk of adverse reactions
arising from the application of probiotics cannot be ignored. However,
However, the results of this FAERS-based study continue to support the
overall safety of probiotic preparations. It is necessary to pay
attention to the potential influence of factors such as gender and age
on the effects and adverse reactions of probiotic application in basic
research and clinical application.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1455735"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1455735</a></p>
<hr />
<h2
id="insights-into-the-protein-domains-of-c-vi-trim-subfamily-in-viral-infection">Insights
into the protein domains of C-VI TRIM subfamily in viral infection</h2>
<p>date: 2025-05-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Tripartite motif (TRIM) proteins, defined by their conserved RBCC
domain architecture, play key roles in various cellular processes and
virus-host interactions. In this review, we focus on Class VI TRIM
proteins, including TRIM24, TRIM28, and TRIM33, highlighting the
distinct functional attributes of their RING, B-BOX1, B-BOX2, COILED
COIL, PHD, and BRD domains in viral infection. Through multiple sequence
alignment, we delineate both the conserved and divergent features within
this subclass, underscoring the uniqueness of Class VI TRIM protein.
Additionally, we explore the post-translational modifications (PTMs) of
Class VI TRIM proteins including their functional differences in
modulating viral infection. Moreover, we examine the C-VI TRIM protein
complexes and their significant contributions to the antiviral response.
Furthermore, we discuss small molecule ligands targeting Class VI TRIM
domains, with a focus on druggable structural motifs. Understanding the
multi-domain features of TRIM proteins is crucial for developing
effective antiviral strategies and the therapeutic modulation of their
activity.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1573422"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1573422</a></p>
<hr />
<h2
id="ganodermataceaecurrent-status-research-and-development-in-lower-mekong-basin">Ganodermataceae—current
status, research, and development in Lower Mekong Basin</h2>
<p>date: 2025-05-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The family Ganodermataceae encompasses several genera, including the
widely studied Ganoderma, which is prominent in traditional medicine for
its therapeutic properties. Species within this family, particularly
Ganoderma lucidum, have been valued for centuries in regions such as
China, Korea, and Japan for enhancing vitality, longevity, and overall
health. However, the taxonomy of Ganodermataceae remains complex, with
ongoing debates about species identification and classification. Members
of this family are globally distributed, with the Lower Mekong
Basin—comprising Laos, Thailand, Cambodia, and Vietnam—offering optimal
conditions for their growth due to its warm, humid climate. In the Lower
Mekong Basin, the species of Ganodermataceae are significant for their
medicinal applications in treating conditions such as bronchitis,
hepatitis, diabetes, and cancer. They also hold significant economic
value, being used in products like teas, dietary supplements, and
cosmetics. Ganoderma lucidum is particularly notable as a high-value
market product in this region. Recent research has revealed a rich
diversity of Ganodermataceae species in the region, highlighting their
ecological roles, medicinal properties, and importance in plant
pathology, particularly in addressing diseases in crops such as oil
palm. These findings underscore the need for further research into the
taxonomy, ecological functions, and potential applications of
Ganodermataceae species. Advancing our understanding will support
sustainable utilization, conservation efforts, and the maximization of
their medicinal and commercial benefits.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545135"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545135</a></p>
<hr />
<h2
id="antibiotic-resistance-virulence-gene-phylogenetic-group-and-genetic-diversity-of-escherichia-coli-isolated-from-tibetan-pig-farms-in-garze-tibetan-autonomous-prefecture-sichuan-china">Antibiotic
resistance, virulence gene, phylogenetic group and genetic diversity of
Escherichia coli isolated from Tibetan pig farms in Garze Tibetan
Autonomous Prefecture, Sichuan, China</h2>
<p>date: 2025-05-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionTo investigate the correlations among antibiotic
resistance, virulence gene, phylogenetic group, and genetic diversity,
providing essential data for Escherichia coli (E. coli) infection
prevention and control in Tibetan pigs.MethodsA total of 244 E. coli
isolates were collected. Antimicrobial susceptibility was assessed using
the microdilution method. PCR was used to detect antibiotic resistance
genes (ARGs), virulence genes, and phylogenetic groups. Genetic
diversity was analyzed using enterobacterial repetitive element
sequence-based PCR. Enteroaggregative E. coli (EAEC) 5-12, a
representative strain with multidrug resistance and strong
biofilm-forming ability, harboring abundant virulence genes, was
selected for whole-genome sequencing (WGS) to validate PCR
results.ResultsAmong the 244 isolates, 84.43% showed multidrug
resistance (MDR), with the highest resistance rates for chloramphenicol
(99.59%), sulfadiazine (96.31%), and sulfamethoxazole (93.85%).
Twenty-five ARGs were detected, with ant(3’)-Ia, blaTEM, aac(3’)-II,
floR, and qnrS exceeding 80% detection rates. Integrase genes intl1 and
intl2 were found in 90.16% and 15.16% of isolates, respectively.
Seventeen virulence genes were detected; bcsA (98.77%), fimC (89.75%),
and agn43 (59.43%) were the most prevalent. A total of 106 virulence
patterns were identified, with agn43/bcsA/fimC being predominant
(17.92%). Most strains belonged to phylogenetic group A (45.90%),
followed by B1 (34.43%), while 29 strains were unclassified. Sixty-four
isolates were identified as diarrheagenic E. coli (DEC), predominantly
enteroaggregative E. coli (EAEC, 90.63%). Biofilm-forming ability was
categorized as strong (4.69%), moderate (21.88%), weak (59.38%), or
absent (14.06%). Clustering based on 61.2% similarity grouped the 64 DEC
into five clusters, with 84.38% in cluster II, which contained all
strong biofilm producers.DiscussionAntimicrobial resistance profiles of
EAEC 5–12 confirmed that primarily confer resistance through antibiotic
efflux, target alteration, and reduced permeability. These findings will
contribute to further understanding the positive correlation between
antibiotic resistance and pathogenicity in E. coli from Tibetan pig
farms, shedding light on the rational use of antimicrobial agents and
tackling the antibiotic resistance crisis in Tibetan pig breeding in
Garze Tibetan Autonomous Prefecture, Sichuan, China.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1526028"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1526028</a></p>
<hr />
<h2
id="predicting-ventilator-associated-lower-respiratory-tract-infection-outcomes-using-sequencing-based-early-microbiological-response-a-proof-of-concept-prospective-study">Predicting
ventilator-associated lower respiratory tract infection outcomes using
sequencing-based early microbiological response: a proof-of-concept
prospective study</h2>
<p>date: 2025-05-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesVentilator-associated lower respiratory tract infections
(VA-LRTIs) cause significant mortality. This study was assigned to
explore the association between early microbiological responses defined
by quantitative targeted amplicon-based next-generation sequencing
(QtNGS) and clinical outcomes in patients with Acinetobacter
baumannii-dominant VA-LRTI.Measurements and main resultsA prospective
observational study including 34 participants was conducted to assess
the probability of predicting clinical outcomes using sequencing-based
early microbiological response. Bronchoalveolar lavage fluids (BALFs)
were collected at admission and 3 days post-treatment from these
patients for QtNGS to determine the relative quantification ratio (RQR)
of A. baumannii. Patients were categorized into survival (n=26) and
non-survival (n=8) groups. The RQR was calculated as the quantification
of A. baumannii determined by QtNGS after treatment to pretreatment. RQR
significantly increased on day 4 in the non-survival group (median
5.285), and decreased in the survival group (median 0.1864). Receiver’s
operation characteristic curves revealed that an RQR ≥1.41 was
predictive of poor outcomes, with an area under the curve of 0.9471
(0.8759-1). The accuracy of the RQR determined by QtNGS was further
evaluated by retesting the same specimen using digital droplet PCR, and
the linear correlation was confirmed in the RQR calculated by two
methods. The 23 patients with RQR&lt;1.41 all survived for 28 days,
whereas the survival rate for the 11 patients with RQR ≥1.41 was 27.27%.
RQR was significantly and positively correlated with the length of ICU
stay in survivors.ConclusionsThe RQR of A. baumannii detected by QtNGS
correlates with the prognosis of VA-LRTI patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547998"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547998</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
