---
title: health 2024.19
updated: 2024-05-10 11:11:56
---

# health 2024.19

(date: 2024-05-10 11:11:56)

---

## Unlocking the antiviral potential of rosmarinic acid against chikungunya virus via IL-17 signaling pathway

date: 2024-05-10, from: Frontiers in Cellular and Infection Microbiology

BackgroundThe Chikungunya virus is an Alphavirus that belongs to the Togaviridae family and is primarily transmitted by mosquitoes. It causes acute infection characterized by fever, headache, and arthralgia. Some patients also experience persistent chronic osteoarthritis-like symptoms. Dedicated antiviral treatments are currently unavailable for CHIKV. This study aims to explore the potential anti-CHIKV effect of rosmarinic acid using network pharmacology.MethodsThis study employed network pharmacology to predict and verify the molecular targets and pathways associated with ROSA in the context of CHIKV. The analysis outcomes were further validated using molecular docking and in vitro experiments.ResultsThe analysis of CHIKV targets using the Kyoto Encyclopedia of Genes and Genomes and MCODE identified IL-17 as an important pathogenic pathway in CHIKV infection. Among the 30 targets of ROSA against CHIKV, nearly half were found to be involved in the IL-17 signaling pathway. This suggests that ROSA may help the host in resisting CHIKV invasion by modulating this pathway. Molecular docking validation results showed that ROSA can stably bind to 10 core targets out of the 30 identified targets. In an in vitro CHIKV infection model developed using 293T cells, treatment with 60 μM ROSA significantly improved the survival rate of infected cells, inhibited 50% CHIKV proliferation after CHIKV infection, and reduced the expression of TNF-α in the IL-17 signaling pathway.ConclusionThis study provides the first confirmation of the efficacy of ROSA in suppressing CHIKV infection through the IL-17 signaling pathway. The findings warrant further investigation to facilitate the development of ROSA as a potential treatment for CHIKV infection. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1396279>

---

## Characterization and functional prediction of the dental plaque microbiome in patients with alveolar clefts

date: 2024-05-10, from: Frontiers in Cellular and Infection Microbiology

IntroductionAlveolar cleft (AC) is a common congenital defect in people with cleft lip and palate (CLP). Alveolar bone grafting (ABG) is typically performed during adolescence, resulting in the fissure remaining in the mouth for a longer length of time. Patients with AC have a greater rate of oral diseases such as dental caries than the normal population, and the precise characteristics of the bacterial alterations caused by AC are unknown.MethodsWe recruited a total of 87 subjects and collected dental plaque samples from AC adolescents (AAP), post-operative ABG adolescents (PAP), healthy control adolescents (CAP), AC young adults (AYP), post-operative ABG young adults (PYP), and healthy control young adults (CYP). The sequencing of 16S rRNA genes was performed.ResultsThe microbial composition of plaque from alveolar cleft patients differed significantly from age-matched healthy controls. Linear discriminant analysis effect size (LEfSe) analysis revealed that AAP was enriched for Neisseria, Haemophilus, Fusobacterium, Rhodococcus, Aggregatibacter, Gemella, and Porphyromonas, whereas AYP was enriched for Capnocytophaga, Rhodococcus, and Actinomyces-f0332. There were phenotypic differences in facultatively anaerobic, Gram-negative, Gram-positive, and oxidative stress tolerance between the AYP group with longer alveolar cleft and the healthy control group according to Bugbase phenotypic predictions. Alveolar bone grafting did not alter the functional phenotype of alveolar cleft patients but reduced the number of differential genera between alveolar cleft patients and healthy controls at both ages.ConclusionsOur study systematically characterized the supragingival plaque microbiota of alveolar cleft patients, post-alveolar bone grafting patients, and matched healthy controls in two ages to gain a better understanding of plaque ecology and microbiology associated with alveolar clefts. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361206>

---

## HSP90 is part of a protein complex with the L polymerase of Rift Valley fever phlebovirus and prevents its degradation by the proteasome during the viral genome replication/transcription stage

date: 2024-05-10, from: Frontiers in Cellular and Infection Microbiology

The mosquito-borne Rift Valley fever virus (RVFV) from the Phenuiviridae family is a single-stranded RNA virus that causes the re-emerging zoonotic disease Rift Valley fever (RVF). Classified as a Category A agent by the NIH, RVFV infection can cause debilitating disease or death in humans and lead to devastating economic impacts by causing abortion storms in pregnant cattle. In a previous study, we showed that the host chaperone protein HSP90 is an RVFV-associated host factor that plays a critical role post viral entry, during the active phase of viral genome replication/transcription. In this study, we have elucidated the molecular mechanisms behind the regulatory effect of HSP90 during infection with RVFV. Our results demonstrate that during the early infection phase, host HSP90 associates with the viral RNA-dependent RNA polymerase (L protein) and prevents its degradation through the proteasome, resulting in increased viral replication. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331755>

---

## Crosstalk between human immunodeficiency virus infection and salivary bacterial function in men who have sex with men

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

BackgroundEngaging in anal sexual intercourse markedly increases the risk of developing HIV among men who have sex with men (MSM); oral sexual activities tend to uniquely introduce gut-derived microbes to salivary microbiota, which, combined with an individual’s positive HIV status, may greatly perturb oral microecology. However, till date, only a few published studies have addressed this aspect.MethodsBased on 16S rRNA sequencing data of bacterial taxa, MicroPITA picks representative samples for metagenomic analysis, effectively revealing how the development and progression of the HIV disease influences oral microbiota in MSM. Therefore, we collected samples from 11 HIV-negative and 44 HIV-positive MSM subjects (stage 0 was defined by HIV RNA positivity, but negative or indeterminate antibody status; stages 1, 2, and 3 were defined by CD4+ T lymphocyte counts ≥ 500, 200–499, and ≤ 200 or opportunistic infection) and selected 25 representative saliva samples (5 cases/stage) using MicroPITA. Metagenomic sequencing analysis were performed to explore whether positive HIV status changes salivary bacterial KEGG function and metabolic pathway in MSM.ResultsThe core functions of oral microbiota were maintained across each of the five groups, including metabolism, genetic and environmental information processing. All HIV-positive groups displayed KEGG functions of abnormal proliferation, most prominently at stage 0, and others related to metabolism. Clustering relationship analysis tentatively identified functional relationships between groups, with bacterial function being more similar between stage 0-control groups and stage 1-2 groups, whereas the stage 3 group exhibited large functional changes. Although we identified most metabolic pathways as being common to all five groups, several unique pathways formed clusters for certain groups; the stage 0 group had several, while the stage 2 and 3 groups had few, such clusters. The abundance of K03046 was positively correlated with CD4 counts.ConclusionAs HIV progresses, salivary bacterial function and metabolic pathways in MSM progressively changes, which may be related to HIV promoting abnormal energy metabolism and exacerbate pathogen virulence. Further, infection and drug resistance of acute stage and immune cell destruction of AIDS stage were abnormally increased, predicting an increased risk for MSM individuals to develop systemic and oral diseases. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1341545>

---

## Clinical outcomes and risk factors for mortality in recipients with carbapenem-resistant gram-negative bacilli infections after kidney transplantation treated with ceftazidime-avibactam: a retrospective study

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

BackgroundCeftazidime-avibactam is a treatment option for carbapenem-resistant gram-negative bacilli (CR-GNB) infections. However, the risk factors associated with ceftazidime-avibactam (CAZ-AVI) treatment failure in kidney transplant (KT) recipients and the need for CAZ-AVI-based combination therapy remain unclear.MethodsFrom June 2019 to December 2023, a retrospective observational study of KT recipients with CR-GNB infection treated with CAZ-AVI was conducted, with the primary outcome being 30-day mortality and secondary outcomes being clinical cure, microbiological cure, and safety. Risk factors for 30-day mortality and clinical failure were also investigated.ResultsA total of 81 KT recipients treated with CAZ-AVI were included in this study. Forty recipients (49.4%) received CAZ-AVI monotherapy, with a 30-day mortality of 22.2%. The clinical cure and microbiological cure rates of CAZ/AVI therapy were 72.8% and 66.7%, respectively. CAZ-AVI alone or in combination with other medications had no effect on clinical cure or 30-day mortality. Multivariate logistic regression analysis revealed that a higher Acute Physiology and Chronic Health Evaluation (APACHE) II score (odds ratio [OR]: 4.517; 95% confidence interval [CI]: 1.397-14.607; P = 0.012) was an independent risk factor for 30-day mortality. Clinical cure was positively associated with the administration of CAZ-AVI within 48 hours of infection onset (OR: 11.009; 95% CI: 1.344-90.197; P=0.025) and negatively associated with higher APACHE II scores (OR: 0.700; 95% CI: 0.555-0.882; P=0.002). Four (4.9%) recipients experienced recurrence within 90 days after the initial infection, 3 (3.7%) recipients experienced CAZ-AVI-related adverse events, and no CAZ-AVI resistance was identified.ConclusionCAZ-AVI is an effective medication for treating CR-GNB infections following kidney transplantation, even as monotherapy. Optimization of CAZ/AVI therapy (used within 48 hours of infection onset) is positively associated with potential clinical benefit. Further larger-scale studies are needed to validate these findings. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1404404>

---

## Gut microbiota microbial metabolites in diabetic nephropathy patients: far to go

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

Diabetic nephropathy (DN) is one of the main complications of diabetes and a major cause of end-stage renal disease, which has a severe impact on the quality of life of patients. Strict control of blood sugar and blood pressure, including the use of renin–angiotensin–aldosterone system inhibitors, can delay the progression of diabetic nephropathy but cannot prevent it from eventually developing into end-stage renal disease. In recent years, many studies have shown a close relationship between gut microbiota imbalance and the occurrence and development of DN. This review discusses the latest research findings on the correlation between gut microbiota and microbial metabolites in DN, including the manifestations of the gut microbiota and microbial metabolites in DN patients, the application of the gut microbiota and microbial metabolites in the diagnosis of DN, their role in disease progression, and so on, to elucidate the role of the gut microbiota and microbial metabolites in the occurrence and prevention of DN and provide a theoretical basis and methods for clinical diagnosis and treatment. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1359432>

---

## Clinical application value of metagenome next-generation sequencing in pulmonary diffuse exudative lesions: a retrospective study

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

ObjectiveThis study aims to investigate the clinical application value of Metagenome Next-Generation Sequencing (mNGS) for pulmonary diffuse exudative lesions.MethodsFrom January 1, 2014, to November 31, 2021, 136 cases with chest radiologic presentations of pulmonary diffuse exudative lesions admitted to Fujian Provincial Hospital were included in the study; of those, 77 patients underwent mNGS pathogen detection. Based on the pathogen detection outcomes and clinical diagnoses, patients were categorized into an infection group (IG) and a non-infection group (NIG). A comparison was made between the diagnostic efficacy of the mNGS technique and traditional culture methods. Meanwhile, 59 patients clinically identified as having infectious pulmonary diffuse exudative lesions but who did not receive mNGS testing were designated as the non-NGS infection group (non-IG). A retrospective cohort study was conducted on patients in both the IG and non-IG, with a 30-day all-cause mortality endpoint used for follow-up.OutcomesWhen compared to conventional culture methods, mNGS demonstrated an approximate 35% increase in sensitivity (80.0% vs 45.5%, P<0.001), without significant disparity in specificity (77.3% vs 95.5%, P=0.185). Under antibiotic exposure, the positivity rate detected by mNGS was notably higher than that by traditional culture methods, indicating that mNGS is less affected by exposure to antibiotics (P<0.05). Within 30 days, the all-cause mortality rate for patients in the IG versus the non-IG was 14.55% and 37.29%, respectively (P<0.05). Following a COX regression analysis to adjust for confounding factors, the analysis revealed that a CURB-65 score ≥3 points (HR=3.348, P=0.001) and existing cardiovascular disease (HR=2.473, P=0.026) were independent risk factors for these patients. Conversely, mNGS testing (HR=0.368, P=0.017) proved to be an independent protective factor.ConclusionmNGS technology makes it easier to pinpoint the cause of pulmonary diffuse infectious exudative lesions without much interference from antibiotics, helping doctors spot and diagnose these issues early on, thereby playing a key role in helping them decide the best treatment approach for patients. Such conclusions may have a bias, as the performance of traditional methods might be underestimated due to the absence of complete results from other conventional diagnostic techniques like serological testing and PCR. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1367885>

---

## An integrative multi-omic analysis defines gut microbiota, mycobiota, and metabolic fingerprints in ulcerative colitis patients

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

BackgroundUlcerative colitis (UC) is a multifactorial chronic inflammatory bowel disease (IBD) that affects the large intestine with superficial mucosal inflammation. A dysbiotic gut microbial profile has been associated with UC. Our study aimed to characterize the UC gut bacterial, fungal, and metabolic fingerprints by omic approaches.MethodsThe 16S rRNA- and ITS2-based metataxonomics and gas chromatography–mass spectrometry/solid phase microextraction (GC–MS/SPME) metabolomic analysis were performed on stool samples of 53 UC patients and 37 healthy subjects (CTRL). Univariate and multivariate approaches were applied to separated and integrated omic data, to define microbiota, mycobiota, and metabolic signatures in UC. The interaction between gut bacteria and fungi was investigated by network analysis.ResultsIn the UC cohort, we reported the increase of Streptococcus, Bifidobacterium, Enterobacteriaceae, TM7-3, Granulicatella, Peptostreptococcus, Lactobacillus, Veillonella, Enterococcus, Peptoniphilus, Gemellaceae, and phenylethyl alcohol; and we also reported the decrease of Akkermansia; Ruminococcaceae; Ruminococcus; Gemmiger; Methanobrevibacter; Oscillospira; Coprococus; Christensenellaceae; Clavispora; Vishniacozyma; Quambalaria; hexadecane; cyclopentadecane; 5-hepten-2-ol, 6 methyl; 3-carene; caryophyllene; p-Cresol; 2-butenal; indole, 3-methyl-; 6-methyl-3,5-heptadiene-2-one; 5-octadecene; and 5-hepten-2-one, 6 methyl. The integration of the multi-omic data confirmed the presence of a distinctive bacterial, fungal, and metabolic fingerprint in UC gut microbiota. Moreover, the network analysis highlighted bacterial and fungal synergistic and/or divergent interkingdom interactions.ConclusionIn this study, we identified intestinal bacterial, fungal, and metabolic UC-associated biomarkers. Furthermore, evidence on the relationships between bacterial and fungal ecosystems provides a comprehensive perspective on intestinal dysbiosis and ecological interactions between microorganisms in the framework of UC. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366192>

---

## Quorum-quenching enzyme Est816 assisted antibiotics against periodontitis induced by Aggregatibacter actinomycetemcomitans in rats

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

IntroductionQuorum-quenching enzyme Est816 hydrolyzes the lactone rings of N-acyl homoserine lactones, effectively blocking the biofilm formation and development of Gram-negative bacteria. However, its applications in the oral field is limited. This study aimed to evaluate the efficacy of enzyme Est816 in combination with antibiotics against periodontitis induced by Aggregatibacter actinomycetemcomitans in vitro and in vivo.MethodsThe antimicrobial efficacy of enzyme Est816 in combination with minocycline, metronidazole, and amoxicillin was determined using the minimum inhibitory concentration test. The anti-biofilm effect of enzyme Est816 was assessed using scanning electron microscopy, live/dead bacterial staining, crystal violet staining, and real-time quantitative PCR. Biocompatibility of enzyme Est816 was assessed in human gingival fibroblasts (HGF) by staining. A rat model of periodontitis was established to evaluate the effect of enzyme Est816 combined with minocycline using micro-computed tomography and histological staining.ResultsCompared to minocycline, metronidazole, and amoxicillin treatment alone, simultaneous treatment with enzyme Est816 increased the sensitivity of biofilm bacteria to antibiotics. Enzyme Est816 with minocycline exhibited the highest rate of biofilm clearance and high biocompatibility. Moreover, the combination of enzyme Est816 with antibiotics improved the antibiofilm effects of the antibiotics synergistically, reducing the expression of the virulence factor leukotoxin gene (ltxA) and fimbria-associated gene (rcpA). Likewise, the combination of enzyme Est816 with minocycline exhibited a remarkable inhibitory effect on bone resorption and inflammation damage in a rat model of periodontitis.DiscussionThe combination of enzyme Est816 with antibiotics represents a prospective anti-biofilm strategy with the potential to treat periodontitis. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368684>

---

## Transcriptional regulation of TacL-mediated lipoteichoic acids biosynthesis by ComE during competence impacts pneumococcal transformation

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

Competence development is essential for bacterial transformation since it enables bacteria to take up free DNA from the surrounding environment. The regulation of teichoic acid biosynthesis is tightly controlled during pneumococcal competence; however, the mechanism governing this regulation and its impact on transformation remains poorly understood. We demonstrated that a defect in lipoteichoic acid ligase (TacL)-mediated lipoteichoic acids (LTAs) biosynthesis was associated with impaired pneumococcal transformation. Using a fragment of tacL regulatory probe as bait in a DNA pulldown assay, we successfully identified several regulatory proteins, including ComE. Electrophoretic mobility shift assays revealed that phosphomimetic ComE, but not wild-type ComE, exhibited specific binding to the probe. DNase I footprinting assays revealed the specific binding sequences encompassing around 30 base pairs located 31 base pairs upstream from the start codon of tacL. Expression of tacL was found to be upregulated in the ΔcomE strain, and the addition of exogenous competence-stimulating peptide repressed the tacL transcription in the wild-type strain but not the ΔcomE mutant, indicating that ComE exerted a negative regulatory effect on the transcription of tacL. Mutation in the JH2 region of tacL upstream regulatory sequence led to increased LTAs abundance and displayed higher transformation efficiency. Collectively, our work identified the regulatory mechanisms that control LTAs biosynthesis during competence and thereby unveiled a repression mechanism underlying pneumococcal transformation. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1375312>

---

## Shifts in the functional capacity and metabolite composition of the gut microbiome during recovery from enteric infection

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

While enteric pathogens have been widely studied for their roles in causing foodborne infection, their impacts on the gut microbial community have yet to be fully characterized. Previous work has identified notable changes in the gut microbiome related to pathogen invasion, both taxonomically and genetically. Characterization of the metabolic landscape during and after enteric infection, however, has not been explored. Consequently, we investigated the metabolome of paired stools recovered from 60 patients (cases) during and after recovery from enteric bacterial infections (follow-ups). Shotgun metagenomics was applied to predict functional microbial pathways combined with untargeted metametabolomics classified by Liquid Chromatography Mass Spectrometry. Notably, cases had a greater overall metabolic capacity with significantly higher pathway richness and evenness relative to the follow-ups (p<0.05). Metabolic pathways related to central carbon metabolism, amino acid metabolism, and lipid and fatty acid biosynthesis were more highly represented in cases and distinct signatures for menaquinone production were detected. By contrast, the follow-up samples had a more diverse metabolic landscape with enhanced richness of polar metabolites (p<0.0001) and significantly greater richness, evenness, and overall diversity of nonpolar metabolites (p<0.0001). Although many metabolites could not be annotated with existing databases, a marked increase in certain clusters of metabolites was observed in the follow-up samples when compared to the case samples and vice versa. These findings suggest the importance of key metabolites in gut health and recovery and enhance understanding of metabolic fluctuations during enteric infections. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1359576>

---

## Amyloid, Crohn’s disease, and Alzheimer’s disease - are they linked?

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

Crohn’s disease (CD) is a chronic inflammatory disease that most frequently affects part of the distal ileum, but it may affect any part of the gastrointestinal tract. CD may also be related to systemic inflammation and extraintestinal manifestations. Alzheimer’s disease (AD) is the most common neurodegenerative disease, gradually worsening behavioral and cognitive functions. Despite the meaningful progress, both diseases are still incurable and have a not fully explained, heterogeneous pathomechanism that includes immunological, microbiological, genetic, and environmental factors. Recently, emerging evidence indicates that chronic inflammatory condition corresponds to an increased risk of neurodegenerative diseases, and intestinal inflammation, including CD, increases the risk of AD. Even though it is now known that CD increases the risk of AD, the exact pathways connecting these two seemingly unrelated diseases remain still unclear. One of the key postulates is the gut-brain axis. There is increasing evidence that the gut microbiota with its proteins, DNA, and metabolites influence several processes related to the etiology of AD, including β-amyloid abnormality, Tau phosphorylation, and neuroinflammation. Considering the role of microbiota in both CD and AD pathology, in this review, we want to shed light on bacterial amyloids and their potential to influence cerebral amyloid aggregation and neuroinflammation and provide an overview of the current literature on amyloids as a potential linker between AD and CD. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1393809>

---

## Extracellular host DNA contributes to pathogenic biofilm formation during periodontitis

date: 2024-05-08, from: Frontiers in Cellular and Infection Microbiology

IntroductionPeriodontal diseases are known to be associated with polymicrobial biofilms and inflammasome activation. A deeper understanding of the subgingival cytological (micro) landscape, the role of extracellular DNA (eDNA) during periodontitis, and contribution of the host immune eDNA to inflammasome persistence, may improve our understanding of the mechanisms underlaying severe forms of periodontitis.MethodsIn this work, subgingival biolfilms developing on biologically neutral polyethylene terephthalate films placed in gingival cavities of patients with chronic periodontitis were investigated by confocal laser scanning microscopy (CLSM). This allowed examination of realistic cytological landscapes and visualization of extracellular polymeric substances (EPS) including amyloids, total proteins, carbohydrates and eDNA, as well as comparison with several single-strain in vitro model biofilms produced by oral pathogens such as Klebsiella pneumoniae, Pseudomonas aeruginosa, Staphylococcus aureus, Streptococcus gordonii, S. sanguinis and S. mitis. Fluorescence in situ hybridization (FISH) analysis was also used to identify eDNA derived from eubacteria, streptococci and members of the Bacteroides–Porphyromonas–Prevotella (BPP) group associated with periodontitis.ResultsAnalysis of subgingival biofilm EPS revealed low levels of amyloids and high levels of eDNA which appears to be the main matrix component. However, bacterial eDNA contributed less than a third of the total eDNA observed, suggesting that host-derived eDNA released in neutrophil extracellular traps may be of more importance in the development of biofilms causing periodontitis.DiscussioneDNA derived from host immunocompetent cells activated at the onset of periodontitis may therefore be a major driver of bacterial persistence and pathogenesis. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1374817>

---

## Pleiotropic roles of LAMMER kinase, Lkh1 in stress responses and virulence of Cryptococcus neoformans

date: 2024-05-07, from: Frontiers in Cellular and Infection Microbiology

Dual-specificity LAMMER kinases are highly evolutionarily conserved in eukaryotes and play pivotal roles in diverse physiological processes, such as growth, differentiation, and stress responses. Although the functions of LAMMER kinase in fungal pathogens in pathogenicity and stress responses have been characterized, its role in Cryptococcus neoformans, a human fungal pathogen and a model yeast of basidiomycetes, remains elusive. In this study, we identified a LKH1 homologous gene and constructed a strain with a deleted LKH1 and a complemented strain. Similar to other fungi, the lkh1Δ mutant showed intrinsic growth defects. We observed that C. neoformans Lkh1 was involved in diverse stress responses, including oxidative stress and cell wall stress. Particularly, Lkh1 regulates DNA damage responses in Rad53-dependent and -independent manners. Furthermore, the absence of LKH1 reduced basidiospore formation. Our observations indicate that Lkh1 becomes hyperphosphorylated upon treatment with rapamycin, a TOR protein inhibitor. Notably, LKH1 deletion led to defects in melanin synthesis and capsule formation. Furthermore, we found that the deletion of LKH1 led to the avirulence of C. neoformans in a systemic cryptococcosis murine model. Taken together, Lkh1 is required for the stress response, sexual differentiation, and virulence of C. neoformans. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1369301>

---

## FAIR compliant database development for human microbiome data samples

date: 2024-05-07, from: Frontiers in Cellular and Infection Microbiology

IntroductionSharing microbiome data among researchers fosters new innovations and reduces cost for research. Practically, this means that the (meta)data will have to be standardized, transparent and readily available for researchers. The microbiome data and associated metadata will then be described with regards to composition and origin, in order to maximize the possibilities for application in various contexts of research. Here, we propose a set of tools and protocols to develop a real-time FAIR (Findable. Accessible, Interoperable and Reusable) compliant database for the handling and storage of human microbiome and host-associated data.MethodsThe conflicts arising from privacy laws with respect to metadata, possible human genome sequences in the metagenome shotgun data and FAIR implementations are discussed. Alternate pathways for achieving compliance in such conflicts are analyzed. Sample traceable and sensitive microbiome data, such as DNA sequences or geolocalized metadata are identified, and the role of the GDPR (General Data Protection Regulation) data regulations are considered. For the construction of the database, procedures have been realized to make data FAIR compliant, while preserving privacy of the participants providing the data.Results and discussionAn open-source development platform, Supabase, was used to implement the microbiome database. Researchers can deploy this real-time database to access, upload, download and interact with human microbiome data in a FAIR complaint manner. In addition, a large language model (LLM) powered by ChatGPT is developed and deployed to enable knowledge dissemination and non-expert usage of the database. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384809>

---

## An altered gut microbiome in pre-eclampsia: cause or consequence

date: 2024-05-07, from: Frontiers in Cellular and Infection Microbiology

Hypertensive disorders of pregnancy, including pre-eclampsia, are a leading cause of serious and debilitating complications that affect both the mother and the fetus. Despite the occurrence and the health implications of these disorders there is still relatively limited evidence on the molecular underpinnings of the pathophysiology. An area that has come to the fore with regard to its influence on health and disease is the microbiome. While there are several microbiome niches on and within the body, the distal end of the gut harbors the largest of these impacting on many different systems of the body including the central nervous system, the immune system, and the reproductive system. While the role of the microbiome in hypertensive disorders, including pre-eclampsia, has not been fully elucidated some studies have indicated that several of the symptoms of these disorders are linked to an altered gut microbiome. In this review, we examine both pre-eclampsia and microbiome literature to summarize the current knowledge on whether the microbiome drives the symptoms of pre-eclampsia or if the aberrant microbiome is a consequence of this condition. Despite the paucity of studies, obvious gut microbiome changes have been noted in women with pre-eclampsia and the individual symptoms associated with the condition. Yet further research is required to fully elucidate the role of the microbiome and the significance it plays in the development of the symptoms. Regardless of this, the literature highlights the potential for a microbiome targeted intervention such as dietary changes or prebiotic and probiotics to reduce the impact of some aspects of these disorders. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1352267>

---

## The footprint of gut microbiota in gallbladder cancer: a mechanistic review

date: 2024-05-07, from: Frontiers in Cellular and Infection Microbiology

Gallbladder cancer (GBC) is the most common malignant tumor of the biliary system with the worst prognosis. Even after radical surgery, the majority of patients with GBC have difficulty achieving a clinical cure. The risk of tumor recurrence remains more than 65%, and the overall 5-year survival rate is less than 5%. The gut microbiota refers to a variety of microorganisms living in the human intestine, including bacteria, viruses and fungi, which profoundly affect the host state of general health, disease and even cancer. Over the past few decades, substantial evidence has supported that gut microbiota plays a critical role in promoting the progression of GBC. In this review, we summarize the functions, molecular mechanisms and recent advances of the intestinal microbiota in GBC. We focus on the driving role of bacteria in pivotal pathways, such as virulence factors, metabolites derived from intestinal bacteria, chronic inflammatory responses and ecological niche remodeling. Additionally, we emphasize the high level of correlation between viruses and fungi, especially EBV and Candida spp., with GBC. In general, this review not only provides a solid theoretical basis for the close relationship between gut microbiota and GBC but also highlights more potential research directions for further research in the future. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1374238>

---

## Untargeted metabolomics in gastric and colorectal cancer patients – preliminary results

date: 2024-05-07, from: Frontiers in Cellular and Infection Microbiology

IntroductionRecent years, microbiota-associated aspects have been analysed in multiple disorders regarding cancers. Existing evidence pints that gut microorganisms might take part in tumour origin and therapy efficacy. Nevertheless, to date, data on faecal metabolomics in cancer patients is still strongly limited. Therefore, we aimed to analyse gut untargeted metabolome in gastrointestinal cancer patients (i.e., gastric and colorectal cancer).Patients and methodsThere were 12 patients with either gastric (n=4) or colorectal cancer (n=8) enrolled and 8 analysed (n=4 each). Stool samples were collected prior to anti-cancer treatments. Untargeted metabolomics analyses were conducted by means of mass spectrometry.ResultsA plethora of metabolites in cancer patients we analysed were noted, with higher homogenity in case of gastric cancer patients. We found that the level of Deoxyguanosine,m/z 266.091,[M-H]-, Uridine,m/z 245.075,[M+H]+, Deoxyguanosine,m/z 268.104,[M]+, 3-Indoleacetic acid,m/z 176.07,[M+H]+, Indoxyl,m/z 132.031,[M-H]-, L-Phenylalanine,m/z 164.073,[M-H]-, L-Methionine,m/z 150.058,[M+NH4]+, was significantly higher in colorectal cancer patients and Ethyl hydrogen malonate,m/z 133.031,[M+H]+ in gastric cancer.ConclusionThe overall insights into untargeted metabolomics showed that most often higher levels of analysed metabolites were detected in colorectal cancer patients compared to gastric cancer patients. The link between gut metabolome and both local and distal metastasis might exist, however it requires confirmation in further multi-centre studies regarding larger sample size. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1394038>

---

## Editorial: Stem cell technologies meet stem cell biology to shine new light into tropical infectious diseases

date: 2024-05-07, from: Frontiers in Cellular and Infection Microbiology

 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411728>

---

## The diagnostic value of metagenomics next-generation sequencing in HIV-infected patients with suspected pulmonary infections

date: 2024-05-07, from: Frontiers in Cellular and Infection Microbiology

BackgroundTraditional microbiological detection methods used to detect pulmonary infections in people living with HIV (PLHIV) are usually time-consuming and have low sensitivity, leading to delayed treatment. We aimed to evaluate the diagnostic value of metagenomics next-generation sequencing (mNGS) for microbial diagnosis of suspected pulmonary infections in PLHIV.MethodsWe retrospectively analyzed PLHIV who were hospitalized due to suspected pulmonary infections at the sixth people hospital of Zhengzhou from November 1, 2021 to June 30, 2022. Bronchoalveolar lavage fluid (BALF) samples of PLHIV were collected and subjected to routine microbiological examination and mNGS detection. The diagnostic performance of the two methods was compared to evaluate the diagnostic value of mNGS for unknown pathogens.ResultsThis study included a total of 36 PLHIV with suspected pulmonary infections, of which 31 were male. The reporting period of mNGS is significantly shorter than that of CMTs. The mNGS positive rate of BALF samples in PLHIV was 83.33%, which was significantly higher than that of smear and culture (44.4%, P<0.001). In addition, 11 patients showed consistent results between the two methods. Futhermore, mNGS showed excellent performance in identifying multi-infections in PLHIV, and 27 pathogens were detected in the BALF of 30 PLHIV by mNGS, among which 15 PLHIV were found to have multiple microbial infections (at least 3 pathogens). Pneumocystis jirovecii, human herpesvirus type 5, and human herpesvirus type 4 were the most common pathogen types.ConclusionsFor PLHIV with suspected pulmonary infections, mNGS is capable of rapidly and accurately identifying the pathogen causing the pulmonary infection, which contributes to implement timely and accurate anti-infective treatment. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1395239>

---

## Editorial: COVID-19 and vaccination-associated ocular complications

date: 2024-05-06, from: Frontiers in Cellular and Infection Microbiology

 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1413435>

