<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.35</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.35">health 2024.35</h1>
<p>(date: 2024-08-29 07:12:05)</p>
<hr />
<h2
id="pythium-insidiosum-an-emerging-pathogen-that-is-easily-misdiagnosed-and-given-treatment-as-a-fungus">Pythium
insidiosum: an emerging pathogen that is easily misdiagnosed and given
treatment as a fungus</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPythium insidiosum (P. insidiosum) is the causative agent
of pythiosis, an infectious disease with a high morbidity and fatality
rate. Pythiosis cases have increased dramatically during the past ten
years, particularly in tropical and subtropical areas. Sadly,
microbiologists and medical professionals know very little about
pythiosis, and the disease is frequently challenging to identify. It is
frequently misdiagnosed as a fungal infection.MethodsWe report two cases
of pythiosis, one was Pythium keratitis, the other was cutaneous
pythiosis. The patient with corneal infection had no underlying disease,
while the patient with cutaneous pythiosis had a history of liver
cirrhosis, diabetes, and psoriasis. The corneal sample and subcutaneous
pus were sent for metagenomic Next-Generation Sequencing (mNGS). To
further diagnose the isolated strain, P. insidiosum zoospores were
induced to produce by co-incubation with sterile grass leaves in sterile
pond water. Their zoospores were used as an inoculum for drug
susceptibility testing by disk diffusion and broth microdilution
method.ResultsThe mNGS of two cases were reported as P. insidiosum.
Zoospores were produced after incubation 48h. The zoospores were
collected for drug susceptibility assay. All antifungal drugs,
antibacterial drugs of β-Lactams, vancomycin, levofloxacin,
ciprofloxacin, gentamicin, trimethoprim-sulfamethoxazole, clindamycin
have no inhibitory activity against P. insidiosum in vitro. Minocycline,
tigecycline, linezolid, erythromycin and azithromycin have significant
in vitro activity against P. insidiosum. Based on the susceptibility
results, the drug was changed from itraconazole to linezolid and
minocycline, along with multiple debridements and drainage for cutaneous
pythiosis. The patient was discharged after 24 days of
treatment.ConclusionsEarly and accurate identification, combined with
aggressive surgical debridement and appropriate drug therapy, can
greatly improve patient managements. Conventional culture and zoospore
induction remain gold standard for diagnosis; however, DNA-based method
should be performed simultaneously. The drug susceptibility testing
provides profound effects on proper drug selection against P.
insidiosum.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1430032"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1430032</a></p>
<hr />
<h2
id="clinical-outcomes-and-risk-factors-of-acinetobacter-baumannii-meningitis-in-pediatric-patients-at-a-tertiary-hospital-in-china">Clinical
outcomes and risk factors of Acinetobacter baumannii meningitis in
pediatric patients at a tertiary hospital in China</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesTo summarize the clinical characteristics, outcomes and
identify risk factors of Acinetobacter baumannii (AB) meningitis in
children.MethodsThis was a single-center, retrospective study. Children
hospitalized between January 2016 and December 2021 who were diagnosed
with AB meningitis were included. The clinical characteristics and
outcomes were reviewed. Risk factors were determined using univariate
analyses (chi-square and Mann-Whitney U tests).ResultsSeventeen patients
were included; 15 cases were secondary to neurosurgery, and two were
neonates with primary bacterial meningitis. Common symptoms included
fever, convulsions and nervous system abnormalities. Cerebrospinal fluid
(CSF) tests typically showed increased white blood cell counts dominated
by neutrophils, reduced glucose levels and elevated protein levels. Ten
patients were successfully treated (successful treatment [ST] group);
seven had failed treatment (failed treatment [FT] group). Univariate
analyses revealed that mechanical ventilation, routine white cell counts
in the peripheral blood, procalcitonin, protein in the CSF, septic shock
and carbapenem-resistant AB (CRAB) differed significantly between the
groups.ConclusionAB meningitis in children has a high mortality rate. FT
was associated with mechanical ventilation, septic shock, CRAB, lower
peripheral leukocyte counts, higher protein levels in the CSF and
procalcitonin. Larger studies are needed to identify independent risk
factors for adverse outcomes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408959"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408959</a></p>
<hr />
<h2
id="early-prediction-of-the-bactericidal-and-bacteriostatic-effect-of-imipenem-and-doxycycline-using-tabletop-scanning-electron-microscopy">Early
prediction of the bactericidal and bacteriostatic effect of imipenem and
doxycycline using tabletop scanning electron microscopy</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionOur work aims at establishing a proof-of-concept for a
method that allows the early prediction of the bactericidal and
bacteriostatic effects of antibiotics on bacteria using scanning
electron microscopy (SEM) as compared to traditional culture-based
methods.MethodsWe tested these effects using Imipenem (bactericidal) and
Doxycycline (bacteriostatic) with several strains of sensitive and
resistant Escherichia coli. We developed a SEM-based predictive score
based on three main criteria: Bacterial Density,
Morphology/Ultrastructure, and Viability. We determined the results for
each of these criteria using SEM micrographs taken with the TM4000Plus
II-Tabletop-SEM (Hitachi, Japan) following an optimized, rapid, and
automated acquisition and analysis protocol. We compared our method with
the traditional culture colony counting gold standard method and classic
definitions of the two effects. ResultsOur method revealed total
agreement with the CFU method and classic definition by visualizing the
effect of the antibiotic at 60 minutes and 120 minutes using SEM.
DiscussionThis early prediction allows a rapid and early identification
of the bactericidal and bacteriostatic effects as compared to culture
that would take a minimum of 18 hours. This has several future
applications in the development of SEM-automated assays coupled to
machine learning models that identify the antibiotic effect and
facilitate determination of bacterial susceptibility.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431141"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431141</a></p>
<hr />
<h2
id="development-and-validation-of-next-generation-sequencing-panel-for-personalized-helicobacter-pylori-eradication-treatment-targeting-multiple-species">Development
and validation of next-generation sequencing panel for personalized
Helicobacter pylori eradication treatment targeting multiple
species</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe decreasing Helicobacter pylori eradication rate is
primarily attributed to antibiotic resistance, and further exacerbated
by uniform drug administration disregarding a host’s metabolic
capability. Consequently, applying personalized treatment based on
antibiotic resistance-associated variants and the host’s metabolic
phenotype can potentially increase the eradication rate.MethodA custom
next-generation sequencing panel for personalized H. pylori eradication
treatment (NGS-PHET) was designed which targeted the regions for
amoxicillin, clarithromycin, metronidazole, tetracycline, and
levofloxacin-resistance in H. pylori and human proton-pump inhibitor
(PPI) metabolism. The libraries were constructed following customized
methods and sequenced simultaneously. The customized framework criteria,
grounded in previously reported antibiotic resistance associated
variants and the host’s PPI metabolism, was applied to the NGS-PHET
results and suggested a personalized treatment for each subject, which
was validated through each subject’s actual eradication
outcome.ResultsBoth previously reported and novel variants were
identified from H. pylori sequencing results. Concurrently, five CYP2C19
homozygous extensive metabolizers and three CYP3A4 intermediate
metabolizers were identified. Among the total of 12 subjects,
clarithromycin triple therapy was suggested for five subjects, bismuth
quadruple therapy was suggested for six subjects, and rifabutin triple
therapy was suggested for one subject by following the customized
framework criteria. The treatment suggestion for nine of the 12 subjects
was consistent with the treatment that each subject achieved eradication
with.DiscussionApplying the methodology using the NGS-PHET and
customized framework helps to perform eradication treatment quickly and
effectively in most patients with antibiotic-resistant H. pylori
strains, and is also useful in research to find novel
antibiotic-resistance candidates.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1379790"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1379790</a></p>
<hr />
<h2
id="isolation-and-characterization-of-two-novel-bacteriophages-against-carbapenem-resistant-klebsiella-pneumoniae">Isolation
and characterization of two novel bacteriophages against
carbapenem-resistant Klebsiella pneumoniae</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The increase of antibiotic-resistant bacteria has become a global
health emergency and the need to explore alternative therapeutic options
arises. Phage therapy uses bacteriophages to target specific bacterial
strains. Phages are highly specific and can target resistant bacteria.
Currently, research in this regard is focused on ensuring reliability
and safety to bring this tool into clinical practice. The first step is
to conduct comprehensive preclinical research. In this work, we present
two novel bacteriophages vB_Kpn_F13 and vB_Kpn_F14 isolated against
clinical carbapenem-resistant Klebsiella pneumoniae strains obtained
from hospital sewage. Multiple studies in vitro were conducted, such as
sequencing, electron microscopy, stability, host range infectivity,
planktonic effect and biofilm inhibition in order to discover their
ability to be used against carbapenem-resistant K. pneumoniae pathogens
causing difficult-to-treat infections.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1421724"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1421724</a></p>
<hr />
<h2
id="microbiome-of-diseased-and-healthy-implantsa-comprehensive-microbial-data-analysis">Microbiome
of diseased and healthy implants—a comprehensive microbial data
analysis</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe purpose of this systematic bioinformatics analysis was
to describe the compositions and differences in submucosal microbial
profiles of peri-implants’ diseases and healthy implant.Material and
methodsPubMed, Embase, ETH Z, Scopus, CNKI, and Wanfang databases were
searched to screen relevant literature on the analysis of peri-implant
microflora based on the sequencing analysis technique of 16S ribosomal
RNA (16S rRNA) gene. High-throughput sequencing of the 16S rRNA gene of
microorganisms from healthy implants, peri-implant mucositis, and
peri-implantitis was downloaded from the screened articles. EasyAmplicon
and Usearch global algorithm were used to match the reads from each
dataset to a full length of 16S rRNA or ITS gene sequence. The
microorganisms based on the Human Oral Microbiome Database (HOMD) were
re-classified, and the microbial diversity, flora composition, and
differential species of the samples were re-analyzed, including
taxonomic classification and alpha and beta diversity calculations. The
co-occurrence network was also re-analyzed.ResultsA total of seven
articles with 240 implants were included. Among them, 51 were healthy
implants (HI), 43 were in the peri-implant mucositis (PM) group, and 146
were in the peri-implantitis (PI) group. A total of 26,483 OTUs were
obtained, and 877 microorganisms were annotated. The alpha diversity
including Chao1 (healthy implants, 121.04 ± 92.76; peri-implant
mucositis, 128.21 ± 66.77; peri-implantitis, 131.15 ± 84.69) and Shannon
(healthy implants, 3.25 ± 0.65; peri-implant mucositis, 3.73 ± 0.61;
peri-implantitis, 3.53 ± 0.67) of the samples from the three groups
showed a significant difference. The beta diversity of the three samples
was statistically different among groups. The genera of Treponema and
Fretibacterium were significantly more abundant in the PI group than in
the other two groups, and the genus of Streptococcus was more abundant
in the HI group. The relative abundance of Porphyromonas in the
peri-implantitis group was 6.1%. The results of the co-occurrence
network showed differences in the network topology among the three
groups of samples. The most connected three genera in the healthy
implants were Halomonas, Fusobacterium, and Fretibacterium. The most
connected three genera in peri-implant mucositis were Alistipes,
Clostridia UCG-014, and Candidatus Saccharimonas. The most connected
three genera in the peri-implantitis group were Lachnoanaerobaculum,
Fusobacterium, and Atopobium. The betweenness of Porphvromonas
gingivalis (red complex) in the PI group (7,900) was higher than in the
HI group (23).ConclusionsThe community compositions of peri-implant
submucosal microorganisms were significantly different in healthy
implants, peri-implant mucositis, and peri-implantitis. The submucosal
microbial communities in peri-implantitis were characterized by high
species richness and diversity compared with the healthy implants; the
relative abundance of red complex, some members of the yellow complex,
and some novel periodontal pathogens was higher in the peri-implantitis
and peri-implant mucositis groups than in the healthy implant group. The
core flora of the co-occurrence network of healthy implants,
peri-implant mucositis, and peri-implantitis varied considerably. The
peri-implantitis site presented a relative disequilibrium microbial
community, and Porphyromonas may play an important role in the
co-occurrence network.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445751"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445751</a></p>
<hr />
<h2
id="age-specific-performance-of-human-papillomavirus-e6e7-mrna-assay-versus-cytology-for-primary-cervical-cancer-screening-and-triage-community-based-screening-in-china">Age-specific
performance of human papillomavirus E6/E7 mRNA assay versus cytology for
primary cervical cancer screening and triage: community-based screening
in China</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundIn the general population, primary human papillomavirus
(HPV) testing is advocated for cervical cancer (CC) screening. HPV E6/E7
mRNA (Aptima HPV, AHPV) assays have garnered considerable traction due
to their higher specificity when compared with HPV DNA assays. Here, we
investigated age-specific primary AHPV screening assays and different
triage strategies versus cytology to identify the best
approach.MethodsBetween April 2018 and December 2021, we recruited
female participants from 34 communities across Liaoning province and
Qingdao City, China. Primary cervical screening protocols included
liquid-based cytology (LBC) and AHPV assays, with females positive for
any assays undergoing colposcopy. Genotyping (AHPV-GT) was conducted on
all HPV-positive samples. Our primary outcomes were the identification
of age-specific detection rates, colposcopy referral rates, and
sensitivity and specificity values for high-grade squamous
intraepithelial lesions or worse (HSIL+). AHPV and different triage
strategy performances were also examined across different age
cohorts.ResultsOur investigation included 9911 eligible females.
Age-specific abnormal cytology rates were in the 6.1%–8.0% range, and
were highest in 45–54-year olds. When compared with 35–44-or 45–54-year
olds, HPV prevalence was highest in 55–64-year olds (12.2% or 11.6%
vs.14.1%, P = 0.048 and P = 0.002, respectively). In 35–44-year olds,
AHPV sensitivity for detecting HSIL+ was 96.6 (95% confidence interval
[CI]: 89.7–100) - significantly higher than LBC sensitivity (65.5 [95%
CI: 48.3–82.8], P &lt; 0.001). When compared with LBC, HSIL+ detection
rates by AHPV-GT using reflex LBC triage increased by 31.5% (9.6‰
vs. 7.3‰), and colposcopy referral rates decreased by 16.4% (5.1%
vs. 6.1%). In 45–54-year olds, HSIL+ detection rates for AHPV-GT using
reflex LBC triage were lower than LBC rates (6.2‰ vs. 6.6‰). In
55–64-year olds, AHPV sensitivity (97.2 [95% CI: 91.7–100.0]) was higher
than LBC sensitivity (66.7 [95% CI: 50.0–80.6], P = 0.003). The area
under the curve (AUC) value was not significantly different between
AHPV-GT with reflex LBC triage and LBC (0.845 [95% CI: 0.771–0.920]
vs. 0.812 [95% CI: 0.734–0.891], P = 0.236).ConclusionsPrimary AHPV
screening using different triage strategies were different across
different age cohorts. Thus, AHPV may be an appropriate primary
screening method for 35–44 and 55–64 year old females, while AHPV-GT
with reflex LBC triage may be more apt for 35–44 year old females.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1428071"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1428071</a></p>
<hr />
<h2
id="delivery-mode-and-maternal-gestational-diabetes-are-important-factors-in-shaping-the-neonatal-initial-gut-microbiota">Delivery
mode and maternal gestational diabetes are important factors in shaping
the neonatal initial gut microbiota</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe infant gut microbiome’s establishment is pivotal for
health and immune development. Understanding it unveils insights into
growth, development, and maternal microbial interactions. Research often
emphasizes gut bacteria, neglecting the phageome.MethodsTo investigate
the influence of geographic or maternal factors (mode of delivery, mode
of breastfeeding, gestational diabetes mellitus) on the gut microbiota
and phages of newborns, we collected fecal samples from 34 pairs of
mothers and their infants within 24 hours of delivery from three regions
(9 pairs from Enshi, 7 pairs from Hohhot, and 18 pairs from Hulunbuir)
using sterile containers. Gut microbiota analysis by Shotgun sequencing
was subsequently performed.ResultsOur results showed that geographic
location affects maternal gut microbiology (P &lt; 0.05), while the
effect on infant gut microbiology was not significant (P = 0.184). Among
the maternal factors, mode of delivery had a significant (P &lt; 0.05)
effect on the newborn. Specific bacteria (e.g., Bacteroides, Escherichia
spp., Phocaeicola vulgatus, Escherichia coli, Staphylococcus hominis,
Veillonella spp.), predicted active metabolites, and bacteriophage vOTUs
varied with delivery mode. Phocaeicola vulgatus significantly correlated
with some metabolites and bacteriophages in the early infant gut (P &lt;
0.05). In the GD group, a strong negative correlation of phage diversity
between mother and infants was observed (R = -0.58, P=0.04).ConclusionIn
conclusion, neonatal early gut microbiome (including bacteria and
bacteriophages) colonization is profoundly affected by the mode of
delivery, and maternal gestational diabetes mellitus. The key bacteria
may interact with bacteriophages to influence the levels of specific
metabolites. Our study provides new evidence for the study of the infant
microbiome, fills a gap in the analysis of the infant gut microbiota
regarding the virome, and emphasizes the importance of maternal health
for the infant initial gut virome.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397675"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397675</a></p>
<hr />
<h2
id="synergistic-in-vitro-activity-and-mechanism-of-kbn-lotion-and-miconazole-nitrate-against-drug-resistant-candida-albicans-biofilms">Synergistic
in vitro activity and mechanism of KBN lotion and miconazole nitrate
against drug-resistant Candida albicans biofilms</h2>
<p>date: 2024-08-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundIn the face of increasing antifungal resistance among
Candida albicans biofilms, this study explores the efficacy of a
combined treatment using Kangbainian lotion (KBN) and miconazole nitrate
(MN) to address this challenge.MethodsUsing UPLC-Q-TOF/MS Analysis for
Identification of Active Compounds in KBN Lotion; FICI for synergy
evaluation, XTT and ROS assays for biofilm viability and oxidative
stress, fluorescence and confocal laser scanning microscopy (CLSM) for
structural and viability analysis, and real-time fluorescence for gene
expression.ConclusionOur study indicates that the combined application
of KBN and MN somewhat impacts the structural integrity of Candida
albicans biofilms and affects the expression of several key genes
involved in biofilm formation, including ALS1, ALS3, HWP1, HSP90, and
CSH1. These preliminary findings suggest that there may be a synergistic
effect between KBN and MN, potentially influencing not only the
structural aspects of fungal biofilms but also involving the modulation
of genetic pathways during their formation.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1426791"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1426791</a></p>
<hr />
<h2
id="editorial-role-of-microbes-in-ocular-surface-health-and-diseases">Editorial:
Role of microbes in ocular surface health and diseases</h2>
<p>date: 2024-08-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1462752"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1462752</a></p>
<hr />
<h2
id="editorial-designing-metal-complexes-metallo-drugs-against-infectious-diseases-and-understanding-their-metal-toxicity">Editorial:
Designing metal-complexes (metallo-drugs) against infectious diseases
and understanding their metal toxicity</h2>
<p>date: 2024-08-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1463451"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1463451</a></p>
<hr />
<h2 id="is-orange-juice-good-for-kidney-stones">Is Orange Juice Good For
Kidney Stones?</h2>
<p>date: 2024-08-27, from: The Kidney Dietitian</p>
<p>
Is orange juice actually good for kidney stones? Or, is this an old
wives tale? Orange juice can impact our health in more ways than one,
especially if we have kidney stones.  In this blog post, we’ll explore
the nutritional profile of orange juice, discuss the benefits and
potential concerns related to kidney stones, investigate […]
</p>
<p>
The post
<a href="https://www.thekidneydietitian.org/is-orange-juice-good-for-kidney-stones/">Is
Orange Juice Good For Kidney Stones?</a> appeared first on
<a href="https://www.thekidneydietitian.org">The Kidney Dietitian</a>.
</p>
<p><a
href="https://www.thekidneydietitian.org/is-orange-juice-good-for-kidney-stones/"
class="uri">https://www.thekidneydietitian.org/is-orange-juice-good-for-kidney-stones/</a></p>
<hr />
<h2
id="unveiling-microbial-dynamics-in-lung-adenocarcinoma-and-adjacent-nontumor-tissues-insights-from-nicotine-exposure-and-diverse-clinical-stages-via-nanopore-sequencing-technology">Unveiling
microbial dynamics in lung adenocarcinoma and adjacent nontumor tissues:
insights from nicotine exposure and diverse clinical stages via nanopore
sequencing technology</h2>
<p>date: 2024-08-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundLung is the largest mucosal area of the human body and
directly connected to the external environment, facing microbial
exposure and environmental stimuli. Therefore, studying the internal
microorganisms of the lung is crucial for a deeper understanding of the
relationship between microorganisms and the occurrence and progression
of lung cancer.MethodsTumor and adjacent nontumor tissues were collected
from 38 lung adenocarcinoma patients and used nanopore sequencing
technology to sequence the 16s full-length sequence of bacteria, and
combining bioinformatics methods to identify and quantitatively analyze
microorganisms in tissues, as well as to enrich the metabolic pathways
of microorganisms.Resultsthe microbial composition in lung
adenocarcinoma tissues is highly similar to that in adjacent tissues,
but the alpha diversity is significantly lower than that in adjacent
tissues. The difference analysis results show that the bacterial
communities of Streptococcaceae, Lactobacillaceae, and Neisseriales were
significantly enriched in cancer tissues. The results of metabolic
pathway analysis indicate that pathways related to cellular
communication, transcription, and protein synthesis were significantly
enriched in cancer tissue. In addition, clinical staging analysis of
nicotine exposure and lung cancer found that Haemophilus,
paralinfluenzae, Streptococcus gordonii were significantly enriched in
the nicotine exposure group, while the microbiota of Cardiobactereae and
Cardiobacterales were significantly enriched in stage II tumors. The
microbiota significantly enriched in IA-II stages were Neisseriaeae,
Enterobacteriales, and Cardiobacterales, respectively.ConclusionNanopore
sequencing technology was performed on the full length 16s sequence,
which preliminarily depicted the microbial changes and enrichment of
microbial metabolic pathways in tumor and adjacent nontumor tissues. The
relationship between nicotine exposure, tumor progression, and
microorganisms was explored, providing a theoretical basis for the
treatment of lung cancer through microbial targets.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397989"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397989</a></p>
<hr />
<h2
id="immunogenicity-and-transmission-blocking-potential-of-quiescin-sulfhydryl-oxidase-in-plasmodium-vivax">Immunogenicity
and transmission-blocking potential of quiescin sulfhydryl oxidase in
Plasmodium vivax</h2>
<p>date: 2024-08-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundTransmission-blocking vaccines (TBVs) can effectively
prevent the community’s spread of malaria by targeting the antigens of
mosquito sexual stage parasites. At present, only a few candidate
antigens have demonstrated transmission-blocking activity (TBA)
potential in P. vivax. Quiescin-sulfhydryl oxidase (QSOX) is a sexual
stage protein in the rodent malaria parasite Plasmodium berghei and is
associated with a critical role in protein folding by introducing
disulfides into unfolded reduced proteins. Here, we reported the
immunogenicity and transmission-blocking potency of the PvQSOX in P.
vivax.Methods and findingsThe full-length recombinant PvQSOX protein
(rPvQSOX) was expressed in the Escherichia coli expression system. The
anti-rPvQSOX antibodies were generated following immunization with the
rPvQSOX in rabbits. A parasite integration of the pvqsox gene into the
P. berghei pbqsox gene knockout genome was developed to express
full-length PvQSOX protein in P. berghei (Pv-Tr-PbQSOX). In western
blot, the anti-rPvQSOX antibodies recognized the native PvQSOX protein
expressed in transgenic P. berghei gametocyte and ookinete. In indirect
immunofluorescence assays, the fluorescence signal was detected in the
sexual stages, including gametocyte, gamete, zygote, and ookinete.
Anti-rPvQSOX IgGs obviously inhibited the ookinetes and oocysts
development both in vivo and in vitro using transgenic parasites. Direct
membrane feeding assays of anti-rPvQSOX antibodies were conducted using
four field P. vivax isolates (named isolates #1–4) in Thailand. Oocyst
density in mosquitoes was significantly reduced by 32.00, 85.96, 43.52,
and 66.03% with rabbit anti-rPvQSOX antibodies, respectively. The
anti-rPvQSOX antibodies also showed a modest reduction of infection
prevalence by 15, 15, 20, and 22.22%, respectively, as compared to the
control, while the effect was insignificant. The variation in the DMFA
results may be unrelated to the genetic polymorphisms. Compared to the
P.vivax Salvador (Sal) I strain sequences, the pvqsox in isolate #1
showed no amino acid substitution, whereas isolates #2, #3, and #4 all
had the M361I substitution.ConclusionsOur results suggest that PvQSOX
could serve as a potential P. vivax TBVs candidate, which warrants
further evaluation and optimization.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451063"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451063</a></p>
<hr />
<h2
id="comparative-analysis-of-metagenomic-and-targeted-next-generation-sequencing-for-pathogens-diagnosis-in-bronchoalveolar-lavage-fluid-specimens">Comparative
analysis of metagenomic and targeted next-generation sequencing for
pathogens diagnosis in bronchoalveolar lavage fluid specimens</h2>
<p>date: 2024-08-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAlthough the emerging NGS-based assays, metagenomic
next-generation sequencing (mNGS) and targeted next-generation
sequencing (tNGS), have been extensively utilized for the identification
of pathogens in pulmonary infections, there have been limited studies
systematically evaluating differences in the efficacy of mNGS and
multiplex PCR-based tNGS in bronchoalveolar lavage fluid (BALF)
specimens.MethodsIn this study, 85 suspected infectious BALF specimens
were collected. Parallel mNGS and tNGS workflows to each sample were
performed; then, we comparatively compared their consistency in
detecting pathogens. The differential results for clinically key
pathogens were confirmed using PCR.ResultsThe microbial detection rates
of BALF specimens by the mNGS and tNGS workflows were 95.18% (79/83) and
92.77% (77/83), respectively, with no significant difference. mNGS
identified 55 different microorganisms, whereas tNGS detected 49
pathogens. The comparative analysis of mNGS and tNGS revealed that
86.75% (72/83) of the specimens were complete or partial concordance.
Particularly, mNGS and tNGS differed significantly in detection rates
for some of the human herpesviruses only, including Human
gammaherpesvirus 4 (P&lt;0.001), Human betaherpesvirus 7 (P&lt;0.001),
Human betaherpesvirus 5 (P&lt;0.05) and Human betaherpesvirus 6
(P&lt;0.01), in which tNGS always had higher detection rates. Orthogonal
testing of clinically critical pathogens showed a total coincidence rate
of 50% for mNGS and PCR, as well as for tNGS and PCR.ConclusionsOverall,
the performance of mNGS and multiplex PCR-based tNGS assays was similar
for bacteria and fungi, and tNGS may be superior to mNGS for the
detection of DNA viruses. No significant differences were seen between
the two NGS assays compared to PCR.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451440"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451440</a></p>
<hr />
<h2
id="editorial-exploring-genetic-characteristics-and-molecular-mechanisms-of-host-adaptation-of-viruses-with-artificial-intelligence-ai-or-and-biological-bio-approaches">Editorial:
Exploring genetic characteristics and molecular mechanisms of host
adaptation of viruses with artificial intelligence (AI) or (and)
biological (BIO) approaches</h2>
<p>date: 2024-08-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474097"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474097</a></p>
<hr />
<h2
id="proven-anti-virulence-therapies-in-combating-methicillin--and-vancomycin-resistant-staphylococcus-aureus-infections">Proven
anti-virulence therapies in combating methicillin- and
vancomycin-resistant Staphylococcus aureus infections</h2>
<p>date: 2024-08-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionDespite years of efforts to develop new antibiotics for
eradicating multidrug-resistant (MDR) and multi-virulent
Methicillin-Resistant Staphylococcus aureus (MRSA) and
Vancomycin-Resistant Staphylococcus aureus (VRSA) infections, treatment
failures and poor prognoses in most cases have been common. Therefore,
there is an urgent need for new therapeutic approaches targeting
virulence arrays. Our aim is to discover new anti-virulence therapies
targeting MRSA and VRSA virulence arrays.MethodologyWe employed
phenotypic, molecular docking, and genetic studies to screen for
anti-virulence activities among selected promising compounds: Coumarin,
Simvastatin, and Ibuprofen.ResultsWe found that nearly all detected MRSA
and VRSA strains exhibited MDR and multi-virulent profiles. The
molecular docking results aligned with the phenotypic and genetic
assessments of virulence production. Biofilm and hemolysin productions
were inhibited, and all virulence genes were downregulated upon
treatment with sub-minimum inhibitory concentration (sub-MIC) of these
promising compounds. Ibuprofen was the most active compound, exhibiting
the highest inhibition and downregulation of virulence gene products.
Moreover, in vivo and histopathological studies confirmed these results.
Interestingly, we observed a significant decrease in wound area and
improvements in re-epithelialization and tissue organization in the
Ibuprofen and antimicrobial treated group compared with the group
treated with antimicrobial alone. These findings support the idea that a
combination of Ibuprofen and antimicrobial drugs may offer a promising
new therapy for MRSA and VRSA infections.ConclusionWe hope that our
findings can be implemented in clinical practice to assist physicians in
making the most suitable treatment decisions.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1403219"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1403219</a></p>
<hr />
<h2
id="metagenomic-next-generation-sequencing-promotes-pathogen-detection-over-culture-in-joint-infections-with-previous-antibiotic-exposure">Metagenomic
next-generation sequencing promotes pathogen detection over culture in
joint infections with previous antibiotic exposure</h2>
<p>date: 2024-08-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveTo investigate the diagnostic value of metagenomic
next-generation sequencing (mNGS) in detecting pathogens from joint
infection (JI) synovial fluid (SF) samples with previous antibiotic
exposure.MethodsFrom January 2019 to January 2022, 59 cases with
suspected JI were enrolled. All cases had antibiotic exposure within 2
weeks before sample collection. mNGS and conventional culture were
performed on SF samples. JI was diagnosed based on history and clinical
symptoms in conjunction with MSIS criteria. The diagnostic values,
including sensitivity, specificity, positive/negative predictive values
(PPV/NPV), and accuracy, were in comparison with mNGS and
culture.ResultsThere were 47 of the 59 cases diagnosed with JI, while
the remaining 12 were diagnosed with non-infectious diseases. The
sensitivity of mNGS was 68.1%, which was significantly higher than that
of culture (25.5%, p&lt;0.01). The accuracy of mNGS was significantly
higher at 71.2% compared to the culture at 39.0% (p &lt;0.01). Eleven
pathogenic strains were detected by mNGS but not by microbiological
culture, which included Staphylococcus lugdunensis, Staphylococcus
cohnii, Finegoldia magna, Enterococcus faecalis, Staphylococcus
saprophytics, Escherichia coli, Salmonella enterica, Pseudomonas
aeruginosa, Acinetobacter pittii, Brucella ovis, andCoxiella burnetii.
Antibiotic therapy was adjusted based on the mNGS results in 32 (68.1%)
patients, including 12 (25.5%) and 20 (42.6%) patients, in whom
treatment was upgraded and changed, respectively. All JI patients
underwent surgery and received subsequent antibiotic therapy. They were
followed up for an average of 23 months (20-27 months), and the success
rate of treatment was 89.4%. Out of the 33 patients who had positive
results for pathogens, reoperation was performed in 1 case (3.03%),
while out of the 14 cases with negative results for both mNGS and
cultures, reoperation was performed in 4 cases (28.6%).ConclusionsmNGS
has advantages over conventional culture in detecting pathogens in SF
samples from JI patients previously treated with antibiotics,
potentially improving clinical outcomes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1388765"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1388765</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
