<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2024-12-03 07:05:37)</p>
<hr />
<h2 id="nonhuman-primate-models-of-pediatric-viral-diseases">Nonhuman
primate models of pediatric viral diseases</h2>
<p>date: 2024-12-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Infectious diseases are the leading cause of death in infants and
children under 5 years of age. In utero exposure to viruses can lead to
spontaneous abortion, preterm birth, congenital abnormalities or other
developmental defects, often resulting in lifelong health sequalae. The
underlying biological mechanisms are difficult to study in humans due to
ethical concerns and limited sample access. Nonhuman primates (NHP) are
closely related to humans, and pregnancy and immune ontogeny in infants
are very similar to humans. Therefore, NHP are a highly relevant model
for understanding fetal and postnatal virus-host interactions and to
define immune mechanisms associated with increased morbidity and
mortality in infants. We will discuss NHP models of viruses causing
congenital infections, respiratory diseases in early life, and HIV.
Cytomegalovirus (CMV) remains the most common cause of congenital
defects worldwide. Measles is a vaccine-preventable disease, yet measles
cases are resurging. Zika is an example of an emerging arbovirus with
devastating consequences for the developing fetus and the surviving
infant. Among the respiratory viruses, we will discuss influenza and
Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2). We will
finish with HIV as an example of a lifelong infection without a cure or
vaccine. The review will highlight (i) the impact of viral infections on
fetal and infant immune development, (ii) how differences in infant and
adult immune responses to infection alter disease outcome, and emphasize
the invaluable contribution of pediatric NHP infection models to the
design of effective treatment and prevention strategies, including
vaccines, for human infants.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1493885"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1493885</a></p>
<hr />
<h2
id="the-efficacy-of-a-nitric-oxide-releasing-formulation-on-nares-isolated-methicillin-resistant-staphylococcus-aureus-in-porcine-wound-infection-model">The
efficacy of a nitric oxide-releasing formulation on nares isolated
Methicillin-Resistant Staphylococcus aureus in porcine wound infection
model</h2>
<p>date: 2024-12-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe colonization of Staphylococcus aureus (SA) acquired in
nosocomial infections may develop acute and chronic infections such as
Methicillin-Resistant Staphylococcus aureus (MRSA) in the nose. As a
commensal microorganism with the ability to form a biofilm, SA can dwell
on the skin, nostrils, throat, perineum, and axillae of healthy humans.
Nitric oxide (NO) is an unstable gas with various molecular functions
and has antimicrobial properties which are converted into many potential
treatments.MethodsMethicillin-Resistant Staphylococcus aureus MRSA
BAA1686 isolated from nasal infection was used in a porcine wound
infection model. Deep partial-thickness wounds (10mm x 7mm x 0.5mm) were
made on three animals using a specialized electrokeratome. All wounds
were inoculated and then covered with polyurethane film dressings for
biofilm formation. After 48 hours, three wounds were recovered from each
animal for baseline enumeration. The remaining wounds were randomly
assigned to six treatment groups and treated once daily. The treatment
groups are as follows: NO topical ointments concentrations of 0.3, 0.9
and 1.8%, Vehicle Ointment, Mupirocin 2%, and Untreated Control.
Microbiological recoveries were conducted on day 4 and day 7.ResultsThe
greatest efficacy observed from the NO formulations against MRSA BAA1686
was the 1.8% concentration. This agent was able to reduce more than 99%
of bacterial counts when compared to Baseline, Vehicle Ointment, and
Untreated Control wounds on both assessment days. Mupirocin 2% was the
overall best treatment against MRSA BAA1686 on both assessment days,
with a significant reduction (p ≤ 0.05) of 4.70 ± 0.13 Log CFU/mL from
day 4 to day 7.ConclusionsOverall, the positive control Mupirocin 2% was
the most effective in eliminating MRSA BAA1686 throughout the study.
This experiment demonstrated a downward trend from the highest
concentration of NO topical ointment formulations to the lowest
concentrations on both assessment days (0.3% - 1.8%). Out of all NO
topical ointments, the highest concentration (1.8%) was the most
effective with the potential to be an alternative treatment against a
MRSA nasal strain biofilm.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1501360"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1501360</a></p>
<hr />
<h2
id="resistome-in-the-indoor-dust-samples-from-workplaces-and-households-a-pilot-study">Resistome
in the indoor dust samples from workplaces and households: a pilot
study</h2>
<p>date: 2024-12-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The antibiotic resistance genes (ARGs) limit the susceptibility of
bacteria to antimicrobials, representing a problem of high importance.
Current research on the presence of ARGs in microorganisms focuses
mainly on humans, livestock, hospitals, or wastewater. However, the
spectrum of ARGs in the dust resistome in workplaces and households has
gone relatively unexplored. This pilot study aimed to analyze resistome
in indoor dust samples from participants’ workplaces (a pediatric
hospital, a maternity hospital, and a research center) and households
and compare two different approaches to the ARGs analysis;
high-throughput quantitative PCR (HT-qPCR) and whole metagenome shotgun
sequencing (WMGS). In total, 143 ARGs were detected using HT-qPCR, with
ARGs associated with the macrolides, lincosamides, and streptogramin B
(MLSB) phenotype being the most abundant, followed by MDR (multi-drug
resistance) genes, and genes conferring resistance to aminoglycosides. A
higher overall relative quantity of ARGs was observed in indoor dust
samples from workplaces than from households, with the pediatric
hospital being associated with the highest relative quantity of ARGs.
WMGS analysis revealed 36 ARGs, of which five were detected by both
HT-qPCR and WMGS techniques. Accordingly, the efficacy of the WMGS
approach to detect ARGs was lower than that of HT-qPCR. In summary, our
pilot data revealed that indoor dust in buildings where people spend
most of their time (workplaces, households) can be a significant source
of antimicrobial-resistant microorganisms, which may potentially pose a
health risk to both humans and animals.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1484100"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1484100</a></p>
<hr />
<h2
id="cell-free-supernatant-of-lactobacillus-gasseri-1a-tv-shows-a-promising-activity-to-eradicate-carbapenem-resistant-klebsiella-pneumoniae-colonization">Cell-free
supernatant of Lactobacillus gasseri 1A-TV shows a promising activity to
eradicate carbapenem-resistant Klebsiella pneumoniae colonization</h2>
<p>date: 2024-12-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe use of beneficial bacteria like Lactobacillus spp. is a
potential innovative approach to fight antibiotic-resistant pathogens.
Klebsiella pneumoniae is one of the most concerning multi drug-resistant
(MDR) pathogens, and its ability to colonize the human gut is considered
to be the main reason for recurrent infections in critically ill
patients.MethodsIn this study, Lactobacillus gasseri 1A-TV, already
described for its probiotic activity, was characterized at the genomic
level. Moreover, its cell-free supernatant (CFS) was tested for
antimicrobial activity against extended-spectrum β-lactamase (ESBL)- and
carbapenemase (KPC)-producing K. pneumoniae clinical
isolates.ResultsWhole-genome sequencing showed that the L. gasseri 1A-TV
genome was of 2,018,898 bp in size with 34.9% GC content, containing
1,937 putative protein coding sequences, 55 tRNA, and 4 rRNA detected by
RAST and classified in 20 functional groups by Cluster of Orthologous
Genes (COG). BAGEL4 (BActeriocin GEnome minimal tooL) and the antiSMASH
7.0 pipeline identified two bacteriocin biosynthetic gene clusters
(BBGCs), namely, BBGC1 that comprises two class IIc bacteriocins
including gassericin A-like bacteriocin, and BBGC2 carrying the class
III bacteriocin helveticin J. Strikingly, 1A-TV CFS inhibited the growth
of all K. pneumoniae isolates only after 8 h of incubation, showing a
bactericidal effect at 24 h and interfering, even at lower
concentrations, with the biofilm production of biofilm-producer strains
independently of a bactericidal effect. NMR analysis of CFS identified
and quantified several metabolites involved in carbohydrate metabolism
and amino acid metabolism, and organic acids like ethanol, lactate,
acetate, and succinate. Finally, in vitro assays of 1A-TV showed
significant co-aggregation effects against carbapenem-resistant K.
pneumoniae, namely, strains 1, 2, 3, and 7.ConclusionsOur findings
highlight the antimicrobial activity of 1A-TV as a probiotic candidate
or its CFS as a natural bioproduct active against MDR K. pneumoniae
strains, underlining the importance of novel therapeutic strategies for
prevention and control of ESBL- and carbapenemase-producing K.
pneumoniae colonization.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1471107"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1471107</a></p>
<hr />
<h2
id="urinary-microbiome-profiling-as-a-non-invasive-tool-for-identifying-biomarkers-in-systemic-lupus-erythematosus-and-lupus-nephritis">Urinary
microbiome profiling as a non-invasive tool for identifying biomarkers
in systemic lupus erythematosus and lupus nephritis</h2>
<p>date: 2024-12-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionBacteriome alterations have been implicated in the
pathogenesis of systemic lupus erythematosus (SLE). However, the
relationship between SLE and the urinary microbiome remains
underexplored. This study aimed to characterize the urinary microbiome
of SLE patients using 16S rRNA sequencing and to investigate its
correlations with clinical parameters through integrative
analyses.MethodsUrine sediment samples were collected from individuals
with SLE and lupus nephritis (LN) (n = 20), SLE without LN (n = 22), and
healthy controls (HCs) (n = 23). DNA was extracted and subjected to 16S
rRNA sequencing to profile the urinary microbiome. Receiver operating
characteristic (ROC) curve analysis was conducted to evaluate the
diagnostic efficacy of urinary microbiota, while Spearman’s correlation
analysis was employed to identify links between specific microbial taxa
and clinical parameters. Functional predictions of bacterial roles were
performed using Picrust2.ResultsThe urinary microbiota diagnostic model
exhibited excellent performance in distinguishing SLE patients from HCs.
Spearman’s analysis revealed significant correlations between the
urinary microbiome and clinical parameters. Specifically, Sphingomonas
and Lachnospiraceae genera showed positive correlations with vitamin D
levels, cylinderuria, and proteinuria, while Pedobacter, Aquabacterium,
Delftia, and Achromobacter displayed negative correlations with
proteinuria and albumin-to-creatinine ratio (ACR). Functional
predictions indicated that the urinary microbiome might influence immune
regulation through modulation of signaling pathways and metabolic
processes.DiscussionOur study is the first to reveal dysbiosis in the
urinary microbiome of patients with SLE. Certain bacterial taxa in the
urinary microbiome were identified as potential diagnostic biomarkers
for SLE. Furthermore, the functional implications of these bacterial
communities suggest their involvement in immune modulation, highlighting
the potential for further investigation into their roles in SLE
pathogenesis and diagnosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1364333"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1364333</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2024-12-02, from: People’s CDC</p>
<p>This is the People’s CDC bi-weekly update for December 2, 2024! This
Weather Report from the People’s CDC sheds light on the ongoing COVID
situation in the US.</p>
<p><br></p>
<p><a
href="https://peoplescdc.org/2024/12/02/peoples-cdc-covid-19-weather-report-87/"
class="uri">https://peoplescdc.org/2024/12/02/peoples-cdc-covid-19-weather-report-87/</a></p>
<hr />
<h2
id="pneumococcal-carriage-in-a-large-sicilian-sample-population-impact-on-the-current-epidemiological-scenario-and-implications-for-future-vaccination-strategies">Pneumococcal
carriage in a large Sicilian sample population: impact on the current
epidemiological scenario and implications for future vaccination
strategies</h2>
<p>date: 2024-12-02, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionStreptococcus pneumoniae is a prevalent and virulent
global pathogen, with colonization being considered a precondition for
pneumococcal disease. Understanding colonization is critical for gaining
insights into transmission dynamics and developing effective
interventions. This study aimed to determine the prevalence of
nasopharyngeal colonization and serotype distribution in the Sicilian
population.MethodsObservational study randomly selecting samples
belonging to Sicilian individuals whose nasopharyngeal swabs were
collected between February 1, 2020, and December 31, 2022. Pneumococcal
colonization was determined using PCR for the pneumococcal autolysin
(LytA) gene, and positive samples were serotyped.ResultsThe study sample
consisted of 1,196 individuals, with 17.4% testing positive for the LytA
gene. Pneumococcal colonization rates fell from birth to 24 years, with
a peak in 0-4-year-olds (aOR=6.9; p&lt;0.001). Colonization was higher
in colder months, particularly in December (aOR=2.9, p&lt;0.05) and
February (aOR=4, p&lt;0.05). Serotypes 22F and 24ABF exhibited strong
colonization and an invasive pneumococcal disease (IPD) risk, whereas
serotypes 4, 6AB, 9VA, and 13 had high colonization but a low IPD risk.
Serotypes 3 and 8 exhibited considerable IPD risk but low
colonization.ConclusionOur findings provide insights into pneumococcal
colonization mechanisms, influencing serotype prevalence, colonization
risk variables, and serotype comparisons for colonization and
pathogenicity propensity.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1467320"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1467320</a></p>
<hr />
<h2
id="genomic-characterization-and-tissue-tropism-variations-of-two-porcine-delta-coronavirus-strains-isolated-in-china">Genomic
characterization and tissue tropism variations of two porcine delta
coronavirus strains isolated in China</h2>
<p>date: 2024-12-02, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The porcine delta coronavirus (PDCoV) is a member of the Delta
coronavirus genus, which can lead to diarrhea, vomiting, and mortality
in piglets. First detected in Hong Kong in 2012, PDCoV has since spread
globally. In January 2024, two strains, CHN-ANHZ-2024 and CHN-JSSQ-2024,
were isolated from diarrheal piglets in Anhui and Jiangsu provinces.
Immunofluorescence assays, electron microscopy, and genome sequencing
were performed. Genome analysis revealed that both PDCoV strains
belonged to the Chinese lineage, exhibiting amino acid mutations in the
S1 region compared to other strains within the lineage. Amino acid
mutation at position 530L is uniquely associated with the Thai strain.
Notably, CHN-JSSQ-2024 was identified as a recombinant strain of DH1 and
CHN-AHHN-2024, with the recombination occurring in the S2 subunit.
CHN-ANHZ-2024 caused severe diarrhea with an 80% mortality rate, whereas
CHN-JSSQ-2024 resulted in mild diarrhea without mortality. Viral load
analysis showed CHN-ANHZ-2024 primarily infecting the brain and kidneys,
while CHN-JSSQ-2024 targeted the lungs, revealing notable differences in
tissue tropism. We designed the RNA scope Probe-PDCoV-N to visualize
viral RNA in the positively detected organs, viral RNA was detected in
the brain, cerebellum, kidneys, and lungs of the infected piglets. This
study highlights significant differences in the pathogenicity and organ
tropism of two PDCoV strains. The CHN-ANHZ-2024 strain caused severe
diarrhea and high mortality in piglets, while the CHN-JSSQ-2024 strain
exhibited much milder symptoms. Additionally, the study elucidated
notable differences in organ tropism between the strains, offering
valuable insights into the epidemiological characteristics and
pathogenic mechanisms of PDCoV. These findings provide a foundation for
the development of targeted prevention and treatment strategies tailored
to specific strains in the future.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1507277"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1507277</a></p>
<hr />
<h2 id="editorial-rising-stars-in-parasite-and-host-2023">Editorial:
Rising stars in parasite and host: 2023</h2>
<p>date: 2024-12-02, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1509107"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1509107</a></p>
<hr />
<h2
id="assessing-the-impact-of-gut-microbiota-and-metabolic-products-on-acute-lung-injury-following-intestinal-ischemia-reperfusion-injury-harmful-or-helpful">Assessing
the impact of gut microbiota and metabolic products on acute lung injury
following intestinal ischemia-reperfusion injury: harmful or
helpful?</h2>
<p>date: 2024-12-02, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Ischemia-reperfusion injury (IRI) is a common and clinically
significant form of tissue damage encountered in medical practice. This
pathological process has been thoroughly investigated across a variety
of clinical settings, including, but not limited to, sepsis, organ
transplantation, shock, myocardial infarction, cerebral ischemia, and
stroke. Intestinal IRI, in particular, is increasingly recognized as a
significant clinical entity due to marked changes in the gut microbiota
and their metabolic products, often described as the body’s “second
genome.” These changes in intestinal IRI lead to profound alterations in
the gut microbiota and their metabolic outputs, impacting not only the
pathology of intestinal IRI itself but also influencing the function of
other organs through various mechanisms. Notable among these are brain,
liver, and kidney injuries, with acute lung injury being especially
significant. This review seeks to explore in depth the roles and
mechanisms of the gut microbiota and their metabolic products in the
progression of acute lung injury initiated by intestinal IRI, aiming to
provide a theoretical basis and directions for future research into the
treatment of related conditions.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1491639"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1491639</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
