<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.16</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="parks.html" title="News about Parks">Parks</a></li>
	<li><a href="tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="2023/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.16">health 2024.16</h1>
<p>(date: 2024-04-16 14:55:23)</p>
<hr />
<h2
id="infection-with-the-multidrug-resistant-klebsiella-pneumoniae-new-delhi-metallo-b-lactamase-strain-in-patients-with-covid-19-nec-hercules-contra-plures">Infection
with the multidrug-resistant Klebsiella pneumoniae New Delhi
metallo-B-lactamase strain in patients with COVID-19: Nec Hercules
contra plures?</h2>
<p>date: 2024-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundDuring the coronavirus disease 2019 (COVID-19) pandemic, in
patients treated for SARS-CoV-2 infection, infections with the
Klebsiella pneumoniae bacteria producing New Delhi metallo-B-lactamase
(NDM) carbapenemase in the USA, Brazil, Mexico, and Italy were observed,
especially in intensive care units (ICUs). This study aimed to assess
the impact of Klebsiella pneumoniae NDM infection and other bacterial
infections on mortality in patients treated in ICUs due to
COVID-19.MethodsThe 160 patients who qualified for the study were
hospitalized in ICUs due to COVID-19. Three groups were distinguished:
patients with COVID-19 infection only (N = 72), patients with COVID-19
infection and infection caused by Klebsiella pneumoniae NDM (N = 30),
and patients with COVID-19 infection and infection of bacterial etiology
other than Klebsiella pneumoniae NDM (N = 58). Mortality in the groups
and chosen demographic data; biochemical parameters analyzed on days 1,
3, 5, and 7; comorbidities; and ICU scores were
analyzed.ResultsBacterial infection, including with Klebsiella
pneumoniae NDM type, did not elevate mortality rates. In the group of
patients who survived the acute phase of COVID-19 the prolonged survival
time was demonstrated: the median overall survival time was 13 days in
the NDM bacterial infection group, 14 days in the other bacterial
infection group, and 7 days in the COVID-19 only group. Comparing the
COVID-19 with NDM infection and COVID-19 only groups, the adjusted model
estimated a statistically significant hazard ratio of 0.28 (p = 0.002).
Multivariate analysis revealed that age, APACHE II score, and CRP were
predictors of mortality in all the patient groups.ConclusionIn patients
treated for SARS-CoV-2 infection acquiring a bacterial infection due to
prolonged hospitalization associated with the treatment of COVID-19 did
not elevate mortality rates. The data suggests that in severe COVID-19
patients who survived beyond the first week of hospitalization,
bacterial infections, particularly Klebsiella pneumoniae NDM, do not
significantly impact mortality. Multivariate analysis revealed that age,
APACHE II score, and CRP were predictors of mortality in all the patient
groups.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297312"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297312</a></p>
<hr />
<h2
id="the-complete-genome-sequence-of-unculturable-mycoplasma-faucium-obtained-through-clinical-metagenomic-next-generation-sequencing">The
complete genome sequence of unculturable Mycoplasma faucium obtained
through clinical metagenomic next-generation sequencing</h2>
<p>date: 2024-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionDiagnosing Mycoplasma faucium poses challenges, and it’s
unclear if its rare isolation is due to infrequent occurrence or its
fastidious nutritional requirements.MethodsThis study analyzes the
complete genome sequence of M. faucium, obtained directly from the pus
of a sternum infection in a lung transplant patient using metagenomic
sequencing.ResultsGenome analysis revealed limited therapeutic options
for the M. faucium infection, primarily susceptibility to tetracyclines.
Three classes of mobile genetic elements were identified: two new
insertion sequences, a new prophage (phiUMCG-1), and a species-specific
variant of a mycoplasma integrative and conjugative element (MICE).
Additionally, a Type I Restriction-Modification system was identified,
featuring 5’-terminally truncated hsdS pseudogenes with overlapping
repeats, indicating the potential for forming alternative hsdS variants
through recombination.ConclusionThis study represents the first-ever
acquisition of a complete circularized bacterial genome directly from a
patient sample obtained from invasive infection of a primary sterile
site using culture-independent, PCR-free clinical metagenomics.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368923"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368923</a></p>
<hr />
<h2
id="comprehensive-assessment-of-hf-rtms-treatment-mechanism-for-post-stroke-dysphagia-in-rats-by-integration-of-fecal-metabolomics-and-16s-rrna-sequencing">Comprehensive
assessment of HF-rTMS treatment mechanism for post-stroke dysphagia in
rats by integration of fecal metabolomics and 16S rRNA sequencing</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe mechanism by which high-frequency repetitive
transcranial magnetic stimulation (HF-rTMS) improves swallowing function
by regulating intestinal flora remains unexplored. We aimed to evaluate
this using fecal metabolomics and 16S rRNA sequencing.MethodsA
Post-stroke dysphagia (PSD) rat model was established by middle cerebral
artery occlusion. The magnetic stimulation group received HF-rTMS from
the 7th day post-operation up to 14th day post-surgery. Swallowing
function was assessed using a videofluoroscopic swallowing study (VFSS).
Hematoxylin-eosin (H&amp;E) staining was used to assess
histopathological changes in the intestinal tissue. Intestinal flora
levels were evaluated by sequencing the 16S rRNA V3-V4 region.
Metabolite changes within the intestinal flora were evaluated by fecal
metabolomics using liquid chromatography-tandem mass
spectrometry.ResultsVFSS showed that the bolus area and pharyngeal bolus
speed were significantly decreased in PSD rats, while the bolus area
increased and pharyngeal transit time decreased after HF-rTMS
administration (p &lt; 0.05). In the PSD groups, H&amp;E staining
revealed damaged surface epithelial cells and disrupted cryptal glands,
whereas HF-rTMS reinforced the integrity of the intestinal epithelial
cells. 16S rRNA sequencing indicated that PSD can disturb the intestinal
flora and its associated metabolites, whereas HF-rTMS can significantly
regulate the composition of the intestinal microflora. Firmicutes and
Lactobacillus abundances were lower in the PSD group than in the
baseline group at the phylum and genus levels, respectively; however,
both increased after HF-rTMS administration. Levels of ceramides (Cer),
free fatty acids (FA), phosphatidylethanolamine (PE), triacylglycerol
(TAG), and sulfoquinovosyl diacylglycerol were increased in the PSD
group. The Cer, FA, and DG levels decreased after HF-rTMS treatment,
whereas the TAG levels increased. Peptococcaceae was negatively
correlated with Cer, Streptococcus was negatively correlated with DG,
and Acutalibacter was positively correlated with FA and Cer. However,
these changes were effectively restored by HF-rTMS, resulting in
recovery from dysphagia.ConclusionThese findings suggest a synergistic
role for the gut microbiota and fecal metabolites in the development of
PSD and the therapeutic mechanisms underlying HF-rTMS.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1373737"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1373737</a></p>
<hr />
<h2
id="editorial-identification-and-characterization-of-novel-antigens-of-malarial-parasites-volume-ii">Editorial:
Identification and characterization of novel antigens of malarial
parasites, volume II</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408366"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408366</a></p>
<hr />
<h2 id="editorial-one-health-in-clinical-microbiology">Editorial: One
Health in clinical microbiology</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1404276"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1404276</a></p>
<hr />
<h2
id="development-of-a-new-loop-mediated-isothermal-amplification-test-for-the-sensitive-rapid-and-economic-detection-of-different-genotypes-of-classical-swine-fever-virus">Development
of a new loop-mediated isothermal amplification test for the sensitive,
rapid, and economic detection of different genotypes of Classical swine
fever virus</h2>
<p>date: 2024-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundClassical swine fever virus (CSFV) remains one of the most
important pathogens in animal health. Pathogen detection relies on viral
RNA extraction followed by RT-qPCR. Novel technologies are required to
improve diagnosis at the point of care.MethodsA loop-mediated isothermal
amplification (LAMP) PCR technique was developed, with primers designed
considering all reported CSFV genotypes. The reaction was tested using
both fluorometric and colorimetric detection, in comparison to the gold
standard technique. Viral strains from three circulating CSFV genotypes
were tested, as well as samples from infected animals. Other pathogens
were also tested, to determine the LAMP specificity. Besides laboratory
RNA extraction methods, a heating method for RNA release, readily
available for adaptation to field conditions was evaluated.ResultsThree
primer sets were generated, with one of them showing better performance.
This primer set proved capable of maintaining optimal performance at a
wide range of amplification temperatures (60°C - 68°C). It was also able
to detect CSFV RNA from the three genotypes tested. The assay was highly
efficient in detection of samples from animals infected with field
strains from two different genotypes, with multiple matrices being
detected using both colorimetric and fluorometric methods. The LAMP
assay was negative for all the unrelated pathogens tested, including
Pestiviruses. The only doubtful result in both fluorometric and
colorimetric LAMP was against the novel Pestivirus italiaense, ovine
Italy Pestivirus (OVPV), which has proven to have cross-reaction with
multiple CSFV diagnostic techniques. However, it is only possible to
detect the OVPV in a doubtful result if the viral load is higher than
10000 viral particles.ConclusionThe results from the present study show
that LAMP could be an important addition to the currently used molecular
diagnostic techniques for CSFV. This technique could be used in remote
locations, given that it can be adapted for successful use with minimal
equipment and minimally invasive samples. The joined use of novel and
traditional diagnostic techniques could prove to be a useful alternative
to support the CSF control.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1372166"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1372166</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
