<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2024-11-11 09:13:39)</p>
<hr />
<h2
id="intestinal-microbiome-changes-and-mechanisms-of-maintenance-hemodialysis-patients-with-constipation">Intestinal
microbiome changes and mechanisms of maintenance hemodialysis patients
with constipation</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundConstipation is a common symptom in maintenance
hemodialysis patients and greatly affects the quality of survival of
hemodialysis patients. Fecal microbiota transplantation and probiotics
are feasible treatments for functional constipation, but there is still
a gap in the research on the characteristics of gut flora in patients
with maintenance hemodialysis combined with constipation. The aim of
this study is to clarify the characteristics of the intestinal flora and
its changes in maintenance hemodialysis patients with
constipation.MethodsFecal samples were collected from 45 participants,
containing 15 in the maintenance hemodialysis constipation group,15 in
the maintenance hemodialysis non-constipation group and 15 in the
healthy control group. These samples were analyzed using 16S rRNA gene
sequencing. The feature of the intestinal microbiome of maintenance
hemodialysis constipation group and the microbiome differences among the
three groups were elucidated by species annotation analysis, α-diversity
analysis, β-diversity analysis, species difference analysis, and
predictive functional analysis.ResultsThe alpha diversity analysis
indicated that maintenance hemodialysis constipation group was less
diverse and homogeneous than maintenance hemodialysis non-constipation
group and healthy control group. At the genus level, the top ten
dominant genera in maintenance hemodialysis constipation group patients
were Enterococcus, Escherichia-Shigella, Bacteroides, Streptococcus,
Bifidobacterium, Ruminococcus_gnavus_group,
Lachnospiraceae_unclassified, Faecalibacterium, Akkermansia and UCG-002.
Compared with non-constipation group, the Enterococcus,
Rhizobiales_unclassified, Filomicrobium, Eggerthella, Allobaculum,
Prevotella_7, Gordonibacter, Mitochondria_unclassified,
Lachnoanaerobaculum were significantly higher in constipation group
(p&lt;0.05). Compared with non-constipation group, the Kineothrix,
Rhodopirellula, Weissella were significantly lower in constipation group
(p&lt;0.05). The predictive functional analysis revealed that compared
with non-constipation group, constipation group was significantly
enriched in pathways associated with pyruate metabolism, flavonoid
biosynthesis.ConclusionsThis study describes for the first time the
intestinal microbiome characteristics of maintenance hemodialysis
patients with constipation. The results of this study suggest that there
is a difference in the intestinal flora between maintenance hemodialysis
patients with constipation and maintenance hemodialysis patients without
constipation.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1495364"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1495364</a></p>
<hr />
<h2
id="apigeninidin-chloride-disrupts-toxoplasma-gondii-mitochondrial-membrane-potential-and-induce-reactive-oxygen-species-and-metabolites-production">Apigeninidin
chloride disrupts Toxoplasma gondii Mitochondrial membrane potential and
induce reactive oxygen species and metabolites production</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionApigeninidin chloride (APi) is a form of
3-deoxyanthrocyanidins (3-DAs) abundantly produced by the red Sorghum
bicolor plant. It has been previously reported to be effective against
Toxoplasma gondii (T. gondii) tachyzoites grown in vitro with less
cytotoxic effect. However, its possible mechanism(s) of action has not
been elucidated. Biochemically, we discovered that APi induced high
reactive oxygen species (ROS) and mitochondria superoxide (MitoSOX)
productions in tachyzoites, leading to mitochondrial membrane potential
(MMP) disruption in vitro.MethodsTo confirm our biochemical results at
the molecular level, we performed a liquid chromatography-mass
spectrometry (LC-MS) analysis on APi-treated parasites to assess any
metabolite and lipid alterations often associated with high ROS/MitoSOX
production in cells. ResultsNoteworthy is that we detected several
important oxidative stress-induced metabolites such as hexanal,
aldehydes, methyl undeo10-enoate, butadiynyl phenyl ketone,
16-hydroxyhexadecanoic acid (16-OH, 16:0), 2-hydroxytricosanoic acid
(C23:0; O), 3-oxodecanosanoic acid (C22:1; O), 2-hydroxypropylsterate,
and furan fatty acids F6 (19FU-FA). DiscussionThese metabolites are
associated with lipid, protein, and nucleic acid disruptions. Using
atovaquone (Atov) as a control, we observed that it disrupted
intracellular tachyzoites’ mitochondrial membrane potential, increased
ROS and MitoSOX production, and altered metabolite and lipid production
similar to what was observed with our experimental compound APi.
Overall, our results indicated that APi targets T. gondii tachyzoite
growth through inducing oxidative stress, mitochondrial dysfunction, and
eventually parasite death.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368019"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368019</a></p>
<hr />
<h2
id="a-novel-rapid-ultrasensitive-diagnosis-platform-for-detecting-candida-albicans-using-restriction-endonucleasemediated-real-time-loop-mediated-isothermal-amplification">A
novel, rapid, ultrasensitive diagnosis platform for detecting Candida
albicans using restriction endonuclease‐mediated real-time loop-mediated
isothermal amplification</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCandida albicans (C. albicans, CA) is an essential
invasive fungus in clinical diagnosis. Although several detection
methods exist, none meet the need for early diagnosis. A rapid,
sensitive, and specific diagnostic tool is crucial for effective
prevention and control of C. albicans infections.MethodsThis study aimed
to develop a new, rapid, and ultrasensitive diagnostic tool for C.
albicans detection based on restriction endonuclease-mediated real-time
loop-mediated isothermal amplification (ERT-LAMP-CA). The ERT-LAMP-CA
technology combines LAMP amplification, restriction endonuclease
cleavage, and real-time fluorescence detection in a single reaction
tube, which can complete a diagnosis of C. albicans in a short time
(approximately 1 h).ResultsHerein, we developed the primer sequences
required for ERT-LAMP-CA based on the ITS2 gene of C. albicans and found
that ERT-LAMP-CA limit of detection was approximately 500 ag/μL genomic
DNA and can present negative results for non-C. albicans templates. We
tested sputum samples from 64 patients with suspected C. albicans
infections to validate ERT-LAMP-CA applicability in clinical sample
testing and found that ERT-LAMP-CA was consistent with multiplex
PCR-capillary electrophoresis.DiscussionIn conclusion, ERT-LAMP-CA is a
rapid, accurate, and sensitive assay with excellent potential for
clinical and basic laboratory diagnosis and an efficient screening
strategy.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1450199"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1450199</a></p>
<hr />
<h2
id="impact-of-endogenous-viral-elements-on-glioma-clinical-phenotypes-by-inducing-oct4-in-the-host">Impact
of endogenous viral elements on glioma clinical phenotypes by inducing
OCT4 in the host</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionEndogenous viral elements (EVEs) are viral sequences
integrated within the host genome that can influence gene regulation and
tumor development. While EVEs have been implicated in cancer, their role
in regulating key transcription factors in glioblastoma (GBM) remains
underexplored. This study investigates the relationship between EVEs and
the activation of OCT4, a critical transcription factor in GBM
progression.MethodsWe utilized CancerHERVdb and HervD Atlas databases to
identify potential interactions between EVEs and key genes involved in
GBM. Data from 273 GBM patient samples in the TCGA database were
analyzed to examine the correlation between OCT4 expression and
mutations in glioma-related genes. Furthermore, glioblastoma stem cells
(GSCs) were assessed for the expression levels of OCT4 and SOX2, and
Pearson correlation analysis was performed.ResultsOur analysis revealed
that OCT4 is a pivotal gene activated by EVEs in GBM. OCT4 expression
was significantly correlated with mutations in key glioma-associated
genes. Higher OCT4 levels were associated with poorer patient prognosis,
higher tumor grades, and older age. Additionally, GSCs exhibited
elevated expression of both OCT4 and SOX2, with a positive correlation
observed between these two genes in GBM patients.DiscussionThis study
highlights the potential role of EVEs in driving GBM progression through
the activation of OCT4. The findings emphasize the importance of OCT4 in
GBM malignancy and suggest that targeting EVE-mediated pathways may
provide new therapeutic approaches for GBM treatment.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474492"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474492</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
