<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.30</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.30">health 2024.30</h1>
<p>(date: 2024-07-22 08:22:26)</p>
<hr />
<h2
id="elevation-of-il-8-secretion-induced-by-pedv-infection-via-nf-κb-signaling-pathway">Elevation
of IL-8 secretion induced by PEDV infection via NF-κB signaling
pathway</h2>
<p>date: 2024-07-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Porcine epidemic diarrhea virus (PEDV) is associated with severe
enteritis, which contributes to high mortality in piglets. The aim of
this study was to describe molecular mechanisms associated with
proinflammatory cytokine(s) production during PEDV infection. We showed
that infection of porcine intestine epithelial cell clone J2 (IPEC-J2)
with PEDV induces a gradual increase in interleukin 8 (IL-8) production
at different time points, as well as infection of Vero E6 with PEDV. The
secretion of IL-8 in these two cell lines infected with PEDV is related
to the activation of NF-κB. Furthermore, the cells expressing PEDV M or
E protein can induce the upregulation of IL-8. These findings suggest
that the IL-8 production can be the initiator of inflammatory response
by the host cells upon PEDV infection.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1422560"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1422560</a></p>
<hr />
<h2
id="biliary-atresia-the-role-of-gut-microbiome-and-microbial-metabolites">Biliary
atresia: the role of gut microbiome, and microbial metabolites</h2>
<p>date: 2024-07-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Biliary atresia (BA) is a progressive fibroinflammatory disease
affecting both the extrahepatic and intrahepatic bile ducts, potentially
leading to chronic cholestasis and biliary cirrhosis. Despite its
prevalence, the exact mechanisms behind BA development remain
incompletely understood. Recent research suggests that the gut
microbiota and its metabolites may play significant roles in BA
development. This paper offers a comprehensive review of the changing
characteristics of gut microbiota and their metabolites at different
stages of BA in children. It discusses their influence on the host’s
inflammatory response, immune system, and bile acid metabolism. The
review also explores the potential of gut microbiota and metabolites as
a therapeutic target for BA, with interventions like butyrate and gut
microbiota preparations showing promise in alleviating BA symptoms.
While progress has been made, further research is necessary to untangle
the complex interactions between gut microbiota and BA, paving the way
for more effective prevention and treatment strategies for this
challenging condition.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411843"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411843</a></p>
<hr />
<h2
id="molecular-and-phenotypic-characterization-of-streptococcus-pneumoniae-isolates-in-a-japanese-tertiary-care-hospital">Molecular
and phenotypic characterization of Streptococcus pneumoniae isolates in
a Japanese tertiary care hospital</h2>
<p>date: 2024-07-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>This study aimed to investigate the bacterial characteristics of
pneumococcal isolates obtained from a tertiary care hospital in Japan.
We analyzed the antimicrobial susceptibility, possession of macrolide
resistance genes, pneumococcal serogroup/serotype, and sequence type
(ST) of pneumococcal isolates from patients aged 15 years or older
between 2011 and 2020 at Nagasaki University Hospital. Of the 73
isolates analyzed, 86.3% showed resistance to macrolides, and 28.8%,
46.6%, and 11.0% harbored mefA, ermB, and both, respectively. Of the
isolates possessing ermB, 97.6% showed high levels of macrolide
resistance [minimal inhibitory concentration (MIC) range, &gt; 16
µg/mL]. Solithromycin (MIC range, 0.03–0.25 µg/mL), regardless of the
presence of macrolide resistance genes, and lascufloxacin (MIC range,
0.06–0.5 µg/mL) showed potent in vitro activity against pneumococci.
Serotype 19A was the most prevalent (six isolates), followed by
serotypes 10A, 15A, and 15B/C (five isolates each). Four serotypes (11A,
19A, 22F, and 23B) and five STs (36, 99, 433, 558, and 3111) were
significantly correlated with the presence of macrolide resistance
genes. All four isolates with serotype 11A/ST99 and three isolates with
serotype 19A/ST3111 harbored both mefA and ermB. No macrolide resistance
genes were detected in either of the two isolates with serotype
22F/ST433, while all ten isolates with serogroup 15 (serotypes 15A and
15B/C, five isolates each) possessed ermB alone. Our study revealed the
bacterial characteristics of the pneumococcal isolates obtained from our
hospital. In vitro activity of solithromycin and lascufloxacin against
these isolates was confirmed.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391879"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391879</a></p>
<hr />
<h2
id="genome-composition-based-deep-learning-predicts-oncogenic-potential-of-hpvs">Genome
composition-based deep learning predicts oncogenic potential of
HPVs</h2>
<p>date: 2024-07-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Human papillomaviruses (HPVs) account for more than 30% of cancer
cases, with definite identification of the oncogenic role of viral E6
and E7 genes. However, the identification of high-risk HPV genotypes has
largely relied on lagged biological exploration and clinical
observation, with types unclassified and oncogenicity unknown for many
HPVs. In the present study, we retrieved and cleaned HPV sequence
records with high quality and analyzed their genomic compositional
traits of dinucleotide (DNT) and DNT representation (DCR) to overview
the distribution difference among various types of HPVs. Then, a deep
learning model was built to predict the oncogenic potential of all HPVs
based on E6 and E7 genes. Our results showed that the main three groups
of Alpha, Beta, and Gamma HPVs were clearly separated between/among
types in the DCR trait for either E6 or E7 coding sequence (CDS) and
were clustered within the same group. Moreover, the DCR data of either
E6 or E7 were learnable with a convolutional neural network (CNN) model.
Either CNN classifier predicted accurately the oncogenicity label of
high and low oncogenic HPVs. In summary, the compositional traits of HPV
oncogenicity-related genes E6 and E7 were much different between the
high and low oncogenic HPVs, and the compositional trait of the
DCR-based deep learning classifier predicted the oncogenic phenotype
accurately of HPVs. The trained predictor in this study will facilitate
the identification of HPV oncogenicity, particularly for those HPVs
without clear genotype or phenotype.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1430424"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1430424</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
