<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2025-01-08 07:05:56)</p>
<hr />
<h2
id="exosomes-derived-from-syncytia-induced-by-sars-2-s-promote-the-proliferation-and-metastasis-of-hepatocellular-carcinoma-cells">Exosomes
derived from syncytia induced by SARS-2-S promote the proliferation and
metastasis of hepatocellular carcinoma cells</h2>
<p>date: 2025-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCoronavirus disease 2019 (COVID-19) is characterized by
fever, fatigue, dry cough, dyspnea, mild pneumonia and acute lung injury
(ALI), which can lead to acute respiratory distress syndrome (ARDS), and
SARS-CoV-2 can accelerate tumor progression. However, the molecular
mechanism for the increased mortality in cancer patients infected with
COVID-19 is unclear.MethodsColony formation and wound healing assays
were performed on Huh-7 cells cocultured with syncytia. Exosomes were
purified from the cell supernatant and verified by nanoparticle tracking
analysis (NTA), Western blot (WB) analysis and scanning electron
microscopy (SEM). Differentially expressed proteins in syncytia-derived
exosomes (Syn-Exos) and their functions was analyzed by Proteomic
sequencing. Syn-Exo-mediated promotion of hepatocellular carcinoma cells
was measured by CCK-8 and Transwell migration assays. The mechanism by
which Syn-Exos promote tumor growth was analyzed by Western blotting. A
patient-derived xenotransplantation (PDX) mouse model was constructed to
evaluate the pathological role of the SARS-CoV-2 spike protein
(SARS-2-S). The number of syncytia in the tumor tissue sections was
determined by immunofluorescence analysis.ResultsSyncytium formation
promoted the proliferation and migration of hepatocellular carcinoma
cells. Proteomic sequencing revealed that proteins that regulate cell
proliferation and metastasis in Syn-Exos were significantly upregulated.
Syn-Exos promote the proliferation and migration of hepatocellular
carcinoma cells. Animal experiments showed that a pseudotyped lentivirus
bearing SARS-2-S (SARS-2-Spp) promoted tumor development in PDX mice.
More syncytia were found in tumor tissue from SARS-2-Spp mice than from
VSV-Gpp mice.ConclusionsSyn-Exos induced by SARS-2-S can promote the
proliferation and metastasis of hepatocellular carcinoma cells.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1415356"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1415356</a></p>
<hr />
<h2
id="interpretable-machine-learning-based-prediction-of-28-day-mortality-in-icu-patients-with-sepsis-a-multicenter-retrospective-study">Interpretable
machine learning-based prediction of 28-day mortality in ICU patients
with sepsis: a multicenter retrospective study</h2>
<p>date: 2025-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSepsis is a major cause of mortality in intensive care
units (ICUs) and continues to pose a significant global health
challenge, with sepsis-related deaths contributing substantially to the
overall burden on healthcare systems worldwide. The primary objective
was to construct and evaluate a machine learning (ML) model for
forecasting 28-day all-cause mortality among ICU sepsis
patients.MethodsData for the study was sourced from the eICU
Collaborative Research Database (eICU-CRD) (version 2.0). The main
outcome was 28-day all-cause mortality. Predictor selection for the
final model was conducted using the least absolute shrinkage and
selection operator (LASSO) regression analysis and the Boruta feature
selection algorithm. Five machine learning algorithms including logistic
regression (LR), decision tree (DT), extreme gradient boosting
(XGBoost), support vector machine (SVM), and light gradient boosting
machine (lightGBM) were employed to construct models using 10-fold
cross-validation. Model performance was evaluated using AUC, accuracy,
sensitivity, specificity, recall, and F1 score. Additionally, we
performed an interpretability analysis on the model that showed the most
stable performance.ResultsThe final study cohort comprised 4564
patients, among whom 568 (12.4%) died within 28 days of ICU admission.
The XGBoost algorithm demonstrated the most reliable performance,
achieving an AUC of 0.821, balancing sensitivity (0.703) and specificity
(0.798). The top three risk predictors of mortality included APACHE
score, serum lactate levels, and AST.ConclusionML models reliably
predicted 28-day mortality in critically ill sepsis patients. Of the
models evaluated, the XGBoost algorithm exhibited the most stable
performance in identifying patients at elevated mortality risk. Model
interpretability analysis identified crucial predictors, potentially
informing clinical decisions for sepsis patients in the ICU.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1500326"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1500326</a></p>
<hr />
<h2
id="ectophoma-salviniae-sp.-nov.-neottiosporina-mihintaleensis-sp.-nov.-and-four-other-endophytes-associated-with-aquatic-plants-from-sri-lanka-and-their-extracellular-enzymatic-potential">Ectophoma
salviniae sp. nov., Neottiosporina mihintaleensis sp. nov. and four
other endophytes associated with aquatic plants from Sri Lanka and their
extracellular enzymatic potential</h2>
<p>date: 2025-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Endophytic fungi associated with selected aquatic plants, Eichhornia
crassipes, Nymphaea nouchali, Salvinia minima and S. molesta were
evaluated. Ectophoma salviniae sp. nov. and Neottiosporina
mihintaleensis sp. nov. are introduced as novel taxa from Salvinia spp.
from Sri Lanka. Chaetomella raphigera is reported as a new geographical
record, Colletotrichum siamense and C. truncatum are reported as novel
host records in aquatic plants, while Phyllosticta capitalensis has been
identified on the same host (Nymphaea nouchali) in the North-Central
Province of Sri Lanka. Identification of the fungi was based on
morphological characteristics and multi-locus phylogenetic analyses
using ITS, LSU, SSU, ACT, CHS-1, GAPDH, tub2, rpb2, and tef1-Î± molecular
markers. The identified fungi were analysed for extracellular enzymatic
properties. According to the qualitative analysis, Ectophoma salviniae
sp. nov. exhibited the highest amylase production, Chaetomella raphigera
exhibited the highest cellulase enzyme production, and Neottiosporina
mihintaleensis sp. nov. exhibited the highest laccase production. The
results demonstrate the aquatic fungal diversity in this region and
their extracellular enzymatic potentials, providing valuable insights
for future biotechnological approaches.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1475114"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1475114</a></p>
<hr />
<h2
id="genetic-insights-into-mis-c-post-covid-19-in-kuwaiti-children-investigating-monogenic-factors">Genetic
insights into MIS-C Post-COVID-19 in Kuwaiti children: investigating
monogenic factors</h2>
<p>date: 2025-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMultisystem inflammatory syndrome in children (MIS-C) is a
severe complication arising from SARS-CoV-2 infection, with indications
that rare inborn errors of immunity may play a role in its pathogenesis.
Recent studies suggest that genetic predispositions, particularly
monogenic forms, could significantly influence the immune responses to
SARS-CoV-2 in MIS-C.MethodsWe analysed 24 children under 12 years old,
all of whom met the criteria provided by the World Health Organization,
2020 for MIS-C diagnosis, from the Paediatric COVID-19 Registry in
Kuwait (PCR-Q8). Demographic and clinical data were collected from
medical records, and exome sequencing was performed on the children and
their parents to identify rare exonic variants. These variants were
prioritized using two approaches: a candidate genes approach employing
trio segregation analysis, and a candidate variants approach using a
gene panel informed by previous studies on MIS-C-related genetic
variants and datasets of differentially expressed genes in MIS-C
patients.ResultsThe candidate genes approach identified 53 unique genes
in 20 of the 24 probands, including DDX60 and TMEM154, which were also
differentially expressed between MIS-C and control groups. The candidate
variants approach identified 33 rare, predicted deleterious heterozygous
variants across 19 unique genes in 19 of the 24 probands, including both
previously described and novel candidate variants for MIS-C. Pathway
analysis of the identified genes from both approaches revealed
significant involvement in immune response, viral defence, and
inflammatory pathways.ConclusionThis study underscores the monogenic
susceptibility to MIS-C, enhancing the evidence base through
comprehensive genetic analysis. The findings highlight the critical role
of genetic predispositions in MIS-C and suggest that further functional
genomics work is necessary to explore the mechanistic contributions of
these genes, facilitating the development of targeted diagnostic
strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1444216"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1444216</a></p>
<hr />
<h2
id="single-nucleotide-polymorphisms-in-genes-associated-with-the-vitamin-d-pathway-related-to-clinical-and-therapeutic-outcomes-of-american-tegumentary-leishmaniasis">Single-nucleotide
polymorphisms in genes associated with the vitamin D pathway related to
clinical and therapeutic outcomes of American tegumentary
leishmaniasis</h2>
<p>date: 2025-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe vitamin D pathway contributes to the microbicidal
activity of macrophages against Leishmania infection. In addition to
induction of this pathway, interferon-gamma (IFNÎ³), interleukin (IL)-15,
and IL32Î³ are part of a network of pro-inflammatory cytokines. The aim
of this study was to evaluate single-nucleotide polymorphisms (SNPs) in
the components of the vitamin D pathway and associated cytokine genes
that could be related to resistance or susceptibility to American
tegumentary leishmaniasis (ATL).MethodsThe expressions of IFNG, IL15,
IL32, CYP27B1, VDR, and other pro-inflammatory cytokines TNF, IL6, and
IL17 genes were evaluated using real-time polymerase chain reaction
(qPCR) in lesions of patients with localized cutaneous leishmaniasis
(LCL) or mucosal leishmaniasis (ML). SNP genotypes/alleles (in IL15,
IL32, CYP27B1, and VDR) were evaluated by TaqMan PCR assays using DNA
from the blood of patients and healthy individuals. Serum vitamin D
levels were determined by chemiluminescence.ResultsVitamin D
pathway-associated genes were expressed in cutaneous as well as mucosal
lesions. IFNG, IL6, and IL17 were more highly expressed in ML than in
LCL. In contrast, IL32Î³/CYP27B1/VDR mRNAs were mainly correlated in LCL,
and IL32Î³ in ML makes strong connections with all cytokines. The SNP
IL32 rs1555001 was less frequent in patients with ML. In addition, some
SNPs appear to influence the VDR and CYP27B1 (IL15 rs10519613 and IL15
rs3775597) and IL6 (VDR rs7975232) expressions in LCL and the IL17
expression in ML (IL15 rs3775597). Gene expression was also correlated
with clinical parameters, such as number of lesions (CYP27B1 mRNA) and
treatment failure (VDR mRNA). In addition, one SNP was associated with
treatment failure in ML (VDR rs7975232).ConclusionsOur findings
suggested that some SNPs in the vitamin D pathway-associated genes can
be related to resistance and therapeutic outcomes of ATL. They are
promising candidates that need to be further evaluated to understand
their biological effects in the control or immunopathogenesis of
ATL.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1487255"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1487255</a></p>
<hr />
<h2
id="application-of-metagenomic-next-generation-sequencing-in-the-diagnosis-of-post-stroke-infections-a-case-series-study-using-multiple-sample-types">Application
of metagenomic next-generation sequencing in the diagnosis of
post-stroke infections: a case series study using multiple sample
types</h2>
<p>date: 2025-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMetagenomic high-throughput sequencing (mNGS) represents a
powerful tool for detecting nucleic acids from various pathogens, such
as bacteria, fungi, viruses and parasites, in clinical samples. Despite
its extensively employed in the pathogen diagnosis for various
infectious diseases, its application in diagnosing stroke-related
infection, and its potential impact on clinical decision-making,
anti-infection treatment, clinical intervention, and patient prognosis
remain insufficiently explored. Additionally, while mNGS offers
promising potential, it facts limitations related to sensitivity,
specificity, cost, and standardization, which could influence its
integration into routine clinical practice.MethodsWe retrospectively
analyzed 18 stroke patients admitted to the First Affiliated Hospital of
Medical College of Shantou University from January to February 2023,
comparing culture-based methods with mNGS detection, and assessing its
significance in etiological diagnosis. Additionally, we evaluated the
performance differences among various sequencing platforms.ResultsAmong
the 18 stroke patients enrolled, pulmonary infections were identified in
7 cases, urinary tract infections in 1 case, central nervous system
infections in 10 cases, and combined pulmonary and central nervous
system infections in 2 cases, with 2 cases yielding negative results.
mNGS detected pathogens in 13 cases, aligning with clinical diagnoses
(75% concordance), whereas culture-based methods yielded positive
results in only 6 cases (22% concordance). Importantly, for 9 of the 18
patients, adjustments to anti-infective treatment regimens based on mNGS
results led to improved symptomatic relief and infection control. This
suggests that mNGS can contribute to more timely and precise treatment
modifications, particularly for infections with low pathogen loads,
potentially enhancing clinical outcomes.ConclusionOur findings
highlights the utility of mNGS in diagnosing stroke-associated
infections by providing a more comprehensive etiological diagnosis
compared to traditional method. While mNGS shows promise in enhancing
diagnostic accuracy and guiding clinical treatment, it high cost and
technical challenges need addressing before widespread clinical
adoption. Future research should focus on optimizing mNGS protocols,
integration it with convertional diagnostic tools, and evaluating its
cost-effectiveness and clinical impact through larger, multicentric
studies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1386377"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1386377</a></p>
<hr />
<h2
id="dynamics-of-coagulation-proteins-upon-icu-admission-and-after-one-year-of-recovery-from-covid-19-a-preliminary-study">Dynamics
of coagulation proteins upon ICU admission and after one year of
recovery from COVID-19: a preliminary study</h2>
<p>date: 2025-01-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesThis study aimed to investigate the association of baseline
coagulation proteins with hospitalization variables in COVID-19 patients
admitted to ICU, as well as coagulation system changes after one-year
post-discharge, taking into account gender-specific bias in the
coagulation profile.MethodsWe conducted a prospective longitudinal study
on 49 ICU-admitted COVID-19 patients. Proteins were measured using a
Luminex 200â¢. The association between coagulation protein levels and
hospitalization variables was carried out by generalized linear models
adjusted by the most relevant covariates.ResultsAt ICU admission, lower
factor XII, antithrombin, and protein C levels were linked to the need
for invasive mechanical ventilation (IMV) or its duration (p=0.028;
p=0.047 and p=0.015, respectively). Likewise, lower factor XII,
antithrombin, and prothrombin levels were associated with longer ICU
length of stay (ICU LOS) (p=0.045; p=0.022; p=0.036, respectively). From
baseline to the end of the follow-up, factor XII, antithrombin,
prothrombin, and protein C levels notably increased in patients with
longer ICU LOS. One-year post-discharge, differences were found for
factor IX, aPTT, and INR. Gender-stratified analysis showed sustained
alterations in males.ConclusionsDepleted specific coagulation factors on
ICU admission are associated with increased severity in critically ill
COVID-19 patients. Most coagulation alterations recover one-year
post-discharge, except for factor IX, aPTT and INR, which remain
reduced.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1489936"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1489936</a></p>
<hr />
<h2
id="metagenomic-next-generation-sequencing-for-the-clinical-identification-of-spinal-infection-associated-pathogens">Metagenomic
next-generation sequencing for the clinical identification of spinal
infection-associated pathogens</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThis study aimed to evaluate the efficacy of metagenomic
next-generation sequencing (mNGS) technology for identifying pathogens
associated with spinal infection (SI).MethodsA retrospective analysis
was conducted on clinical data from 193 patients with suspected SI
between August 2020 and September 2024. Based on histopathological
results, the patients were divided into the SI group (n=162) and the
non-SI group (n=31). The diagnostic performance of mNGS technology was
compared with that of laboratory examination, imaging examination, and
microbial culture.ResultsAmong SI group, mNGS detected 135 pathogens in
77.78% (126/162) of the cases, including nine cases of multiple
infections. One or more pathogens were detected using mNGS in 86
patients with SI and negative microbial cultures. Staphylococcus aureus
(22.22%, n=30) and Mycobacterium tuberculosis (22.22%, n=30) were the
major pathogens, while various rare pathogens such as anaerobes,
Brucella, and Coxiella burnetii were also detected. For the 40 cases
with positive results for both culture- and mNGS-based identification,
high consistency (77.50%) was observed. Antibiotic use did not
significantly affect the mNGS detection rate (P = 0.45). There was no
significant difference in the positivity rate of mNGS between CT-guided
needle biopsy (80.00%) and surgical sampling (77.17%) (P = 0.72). The
sensitivity of mNGS (77.78%) was significantly higher than that of
traditional microbial culture (27.16%), and the specificity was similar
(90.32% vs.Â 96.77%). Although the sensitivities of erythrocyte
sedimentation rate-based assay (91.36%), magnetic resonance imaging
(88.27%), and C-reactive protein-based assay (87.65%) were better than
those of mNGS, their specificities were generally low
(20%-40%).ConclusionThe pathogens responsible for SI are complex and
diverse. As a novel diagnostic method, mNGS exhibits a high sensitivity
and extensive pathogen coverage for SI diagnosis. When combined with
imaging and laboratory indicators, mNGS can significantly improve the
accuracy of SI diagnosis and provide strong support for clinical
treatment.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1437665"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1437665</a></p>
<hr />
<h2
id="the-role-of-metagenomic-next-generation-sequencing-in-diagnosing-and-managing-post-kidney-transplantation-infections">The
role of metagenomic next-generation sequencing in diagnosing and
managing post-kidney transplantation infections</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Kidney transplantation (KT) is a life-saving treatment for patients
with end-stage renal disease, but post-transplant infections remain one
of the most significant challenges. These infections, caused by a
variety of pathogens, can lead to prolonged hospitalization, graft
dysfunction, and even mortality, particularly in immunocompromised
patients. Traditional diagnostic methods often fail to identify the
causative organisms in a timely manner, leading to delays in treatment
and poorer patient outcomes. This review explores the application of
metagenomic next-generation sequencing (mNGS) in the diagnosis of
post-KT infections. mNGS allows for the rapid, comprehensive detection
of a wide range of pathogens, including bacteria, viruses, fungi, and
parasites, without the need for culture-based techniques. We discuss the
advantages of mNGS in early and accurate pathogen identification, its
role in improving patient management, and the potential challenges in
its clinical implementation. Additionally, we consider the future
prospects of mNGS in overcoming current diagnostic limitations and its
potential for guiding targeted therapies, particularly in detecting
antimicrobial resistance and emerging pathogens. This review emphasizes
the promise of mNGS as an essential tool in improving the diagnosis and
treatment of infections in KT recipients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1473068"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1473068</a></p>
<hr />
<h2
id="antibodies-directed-against-bacterial-antigens-in-sera-of-polish-patients-with-primary-biliary-cholangitis">Antibodies
directed against bacterial antigens in sera of Polish patients with
primary biliary cholangitis</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPrimary biliary cholangitis (PBC) is a cholestatic,
autoimmune liver disease with the presence of characteristic
autoantibodies. The aim of the work was to determine the level of
antibodies directed against bacterial antigens: Chlamydia pneumoniae
(anti-Cpn), Yersinia enterolitica (anti-Y.e), Helicobacter pylori
(anti-Hp), Mycoplasma pneumoniae (anti- Mp.) and Escherichia coli
(E.coli) in sera of PBC patients. We also performed in vitro studies on
the impact of the bacterial peptides on the specific antigen-antibody
binding.MethodWe screened 92 Polish PBC patients and sera samples from
healthy donors and pathological controls. Autoantibodies and
anti-bacterial antibodies were determined by commercially available
ELISA kits. Specific inhibition of antibody binding was also detected by
the in house ELISA method.ResultsAnti-Cpn, anti-Y. enterolitica,
anti-Hp, anti-M. pneumoniae and anti-E. coli antibodies were
significantly more common in the group of PBC patients than in the
pathological and healthy control groups: 74%, 40%, 84%, 39% and 69%
respectively. The mean level of anti-Cpn, anti- Y.e, anti-Hp and anti-
M.p in the PBC group was significantly higher than those in the healthy
group (p &lt; 0.001). and in patients with other liver diseases. In sera
of patients with the presence of positive anti-mitochondrial antibodies
(AMA), specific for PBC, anti-bacterial antibodies have been found in
80% vs.Â 50% in sera with AMA negative. We observed inhibition of
specific antigen-antibody binding by the bacterial peptide: EClpP (E.
coli caseinolytic protease) and adenine glycosylase from E. coli
caseinolytic protease P, ClpP Y.e from peptide of Y. enterolitica, Mp
PDC from M. pneumonia peptide and adenine glycosylase of E. coli.
Bacterial factors influence the specific binding of antibodies to
pyruvate dehydrogenase (PDC-E2), gp210 and KLHL12 (kelch-like peptide
12) antigens.ConclusionMicrobial mimics may be the major targets of
cross-reactivity with human pyruvate dehydrogenase, gp210, and KLHL12 in
PBC.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1410282"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1410282</a></p>
<hr />
<h2
id="study-on-the-effects-of-massa-medicata-fermentata-with-different-formulations-on-the-intestinal-microbiota-and-enzyme-activities-in-mice-with-spleen-deficiency-constipation">Study
on the effects of Massa Medicata Fermentata with different formulations
on the intestinal microbiota and enzyme activities in mice with spleen
deficiency constipation</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study aims to explore the therapeutic mechanism of
Massa Medicata Fermentata (MMF) with different formulations on spleen
deficiency constipation in mice by analyzing gastrointestinal hormones,
D-xylose, intestinal microbiota, and intestinal enzyme
activities.MethodsA spleen deficiency constipation model was established
using an oral administration of Sennae Folium decoction combined with
controlled diet and water intake. After successful model establishment,
the mice with spleen deficiency constipation were treated with MMF S1,
S2, S3. Following the intervention, serum samples from each group of
mice were collected to measure VIP, 5-HT, and D-xylose. Additionally,
small intestine contents were analyzed for intestinal enzyme activity
and subjected to 16S rRNA high-throughput sequencing.ResultsMice with
spleen deficiency constipation showed significant decreases in body
weight and fecal water content. In contrast, the body weight of the CS2
and CS3 groups returned to normal levels, and fecal water content in the
CS2 and CS3 groups also returned to normal. The MMF S2 and S3
significantly increased protease and sucrase enzymes levels compared
with CM group. Serum D-xylose levels were significantly reduced in the
CM and CS2 group. VIP levels increased significantly in the CM group but
decreased in the CS2 and CS3 groups. Additionally, 5-HT levels in the CM
and CS1 groups decreased significantly, with the CS2 group returning to
normal and the CS3 group showing significant increases. 16S rRNA
sequencing analysis revealed that all three MMF formulations effectively
restored the intestinal microbiota composition in mice. LEfSe analysis
identified characteristic microbiota linked to different intervention
groups. The CS3 group significantly upregulated the chloroalkane and
chloroalkene degradation and vibrio cholerae pathogenic cycle pathways
compared to the CM group. Candidatus_Arthromitus in the CS3 group and
Psychrobacter in the CS2 group were positive and negative correlations
with 5-HT and VIP, respectively.ConclusionThe three formulations of MMF
significantly alleviated spleen deficiency constipation symptoms by
modulating intestinal enzyme activities, D-xylose, VIP, and 5-HT levels,
and restoring intestinal microbiota balance. Psychrobacter and
Candidatus_Arthromitus were identified as potential biomarkers for the
treatment of spleen deficiency constipation. Different formulations of
MMF have different mechanisms of regulating constipation through
intestinal microbiota.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1524327"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1524327</a></p>
<hr />
<h2
id="rising-prevalence-and-drug-resistance-of-corynebacterium-striatum-in-lower-respiratory-tract-infections">Rising
prevalence and drug resistance of Corynebacterium striatum in lower
respiratory tract infections</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Corynebacterium striatum (C. striatum) is a Gram-positive bacterium
commonly colonizing the skin and mucosa in healthy individuals and
hospitalized patients. Traditionally regarded as a contaminant, C.
striatum is now increasingly recognized as a potential cause of clinical
infections, especially after the coronavirus disease pandemic. It has
emerged as a pathogen implicated in severe infections, including
pneumonia, bacteremia, meningitis, artificial joint infections,
abdominal infections, and endocarditis. C. striatum has been reported in
lower respiratory tract infections, mostly as a conditioned pathogen in
immunocompromised individuals, particularly in those with chronic
structural lung diseases, such as chronic obstructive pulmonary disease,
leading to severe pneumonia or exacerbation of the existing disease and
high mortality. Additionally, C striatum has been implicated in the
community-acquired pneumonia among immunocompetent individuals and
nosocomial lung infections, with evidence of person-to-person
transmission through caregivers. C. striatum may exhibit multidrug
resistance. Vancomycin, alone or in combination, is currently considered
the most effective treatment for C. striatum. This review highlights the
epidemiological characteristics, drug resistance mechanisms, diagnostics
approaches, and treatment options for C. striatum lower respiratory
tract infections to enhance clinician awareness and improve patient
management strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1526312"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1526312</a></p>
<hr />
<h2
id="exploring-the-mediating-role-of-blood-metabolites-in-the-relationship-between-gut-microbiota-and-gastric-cancer-risk-a-mendelian-randomization-study">Exploring
the mediating role of blood metabolites in the relationship between gut
microbiota and gastric cancer risk: a Mendelian randomization study</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPrior studies have established correlations between gut
microbiota (GM) dysbiosis, circulating metabolite alterations, and
gastric cancer (GC) risk. However, the causal nature of these
associations remains uncertain.MethodsWe utilized summary data from
genome-wide association studies (GWAS) on GM (European, n=8,956), blood
metabolites (European, n=120,241; East Asian, n=4,435), and GC
(European, n=476,116; East Asian, n=167,122) to perform a bidirectional
Mendelian randomization (MR) analysis, investigating the causal effects
of GM and metabolites on GC risk. Additionally, we conducted mediation
analysis (two-step MR) to identify potential metabolite mediators in the
GM-GC relationship.ResultsWe identified twelve negative and seven
positive associations between specific GM taxa and GC risk. For blood
metabolites, seven traits were found to be significantly associated with
reduced GC risk in the European population, with these findings
subsequently validated in the East Asian cohort. Three GM taxa showed
potential causal associations with five metabolic traits: the
Bacteroidia class and Bacteroidales order were positively correlated
with five metabolites (all P &lt; 0.013), while Bacteroides OTU97_27
exhibited a negative correlation with one metabolite (P = 0.007).
Two-step MR analysis indicated that total lipids in intermediate-density
lipoprotein (IDL), IDL particle concentration, phospholipids in medium
low-density lipoprotein (LDL), phospholipids in small LDL, and free
cholesterol in small LDL indirectly influenced the association between
Bacteroidia class/Bacteroidales order and GC, with mediation proportions
of 1.71% (P = 0.048), 1.69% (P = 0.048), 2.05% (P = 0.045), 1.85% (P =
0.048), and 1.99% (P = 0.045), respectively.ConclusionThe present study
provides suggestive evidence of a causal relationship between specific
GM, blood metabolites, and GC risk, potentially offering new insights
into GC etiology.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1453286"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1453286</a></p>
<hr />
<h2
id="corrigendum-viable-but-nonculturable-state-in-the-zoonotic-pathogen-bartonella-henselae-induced-by-low-grade-fever-temperature-and-antibiotic-treatment">Corrigendum:
Viable but nonculturable state in the zoonotic pathogen Bartonella
henselae induced by low-grade fever temperature and antibiotic
treatment</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1544596"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1544596</a></p>
<hr />
<h2
id="identifying-subgroup-of-severe-community-acquired-pneumonia-based-on-clinical-metagenomics-a-multicenter-retrospective-cohort-study">Identifying
subgroup of severe community-acquired pneumonia based on clinical
metagenomics, a multicenter retrospective cohort study</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveSevere community-acquired pneumonia (sCAP) is one of the
major diseases within the ICU. We hypothesize that subtyping sCAP based
on simple inflammatory markers, organ dysfunction, and clinical
metagenomics results is feasible.MethodIn this study, we retrospectively
enrolled immunocompetent sCAP patients requiring invasive mechanical
ventilation, who underwent clinical metagenomics from 17 medical
centers. We collected data on potentially pathogenic species reported by
clinical metagenomics and clinical information for all patients. Latent
class analysis (LCA) was applied to routine clinical parameters such as
gender, age, white blood cell (WBC), lymphocytes, C-reactive protein
(CRP), and Procalcitonin (PCT), identifying two optimally fitting
models.ResultsA total of 569 patients were enrolled. Compared to class
B, class A was characterized by a younger age, higher CRP and PCT
levels, and a higher incidence of coagulation dysfunction, liver
failure, circulatory failure, and renal failure. However, the mortality
rates were similar between the two groups. In class A, more cases of
Streptococcus spp. and fewer cases of HSV-1 and Candida spp. were
detected. Among the patients in the two phenotypes, 48.7% and 57.5%
received corticosteroid treatment, respectively. In the class A,
corticosteroid treatment was not associated with patient mortality
(unadjusted hazard ratio (HR)=0.988; 95% confidence interval (CI),
0.634â1.541; p=0.959). In contrast, in the class B group, the use of
corticosteroids was associated with a reduced mortality rate (adjusted
HR=0.719; 95% CI, 0.525â0.986; p=0.04). Additional analysis showed that
in class B, methylprednisolone was associated with reduced mortality
(adjusted HR=0.61; 95% CI, 0.44â0.86; p=0.005), while dexamethasone was
not associated with mortality (adjusted HR=1.4; 95% CI, 0.89â2.22;
p=0.148). In addition, after dose conversion, the results showed that
higher doses of corticosteroids in class B were associated with
increased mortality (adjusted HR=1.01; 95% CI, 1.00â1.01;
p=0.005).ConclusionWe identified two classes based on clinical
metagenomics and clinical features. Class B exhibited a better response
to corticosteroid compared to class A. The rapid identification of these
phenotypes could facilitate the screening of sCAP patients responsive to
corticosteroid in future prospective clinical trials.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1516620"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1516620</a></p>
<hr />
<h2
id="diagnostic-value-of-third-generation-nanopore-sequencing-in-extrapulmonary-tuberculosis">Diagnostic
value of third-generation nanopore sequencing in extrapulmonary
tuberculosis</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThis study aimed to explore the accuracy of
third-generation nanopore sequencing to diagnose extrapulmonary
tuberculosis (EPTB).MethodsSamples were collected from the lesions of 67
patients with suspected EPTB admitted between April 2022 and August
2023. Nanopore sequencing, acid-fast bacilli (AFB) staining, DNA
testing, and X-pert and mycobacterial cultures were performed. The
sensitivity, specificity, positive predictive value (PPV), negative
predictive value (NPV) and area under the receiver operating
characteristic curve (AUC) were calculated for different diagnostic
methods, and their diagnostic accuracies were compared.ResultsNanopore
sequencing demonstrated the highest correct diagnosis rate among 50
positive EPTB cases, independently diagnosing 19 positive cases missed
by conventional methods. Its sensitivity (62.00%), specificity (94.10%),
PPV (96.90%), NPV (45.70%) and AUC (0.781, 95% CI: 0.67â0.89) were
superior to those of conventional methods, such as AFB staining, DNA
testing, X-pert, and solid culture, indicating its significantly
efficient advantage in EPTB detection.ConclusionNanopore sequencing
technology significantly outperforms conventional methods such as AFB
staining, DNA testing, X-pert, and mycobacterial culture to diagnose
EPTB, promising to improve the diagnosis of EPTB.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1432446"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1432446</a></p>
<hr />
<h2
id="plasma-epstein-barr-virus-dna-load-for-diagnostic-and-prognostic-assessment-in-intestinal-epstein-barr-virus-infection">Plasma
Epstein-Barr Virus DNA load for diagnostic and prognostic assessment in
intestinal Epstein-Barr Virus infection</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe prospective application of plasma Epstein-Barr virus
(EBV) DNA load as a noninvasive measure of intestinal EBV infection
remains unexplored. This study aims to identify ideal threshold levels
for plasma EBV DNA loads in the diagnosis and outcome prediction of
intestinal EBV infection, particularly in cases of primary intestinal
lymphoproliferative diseases and inflammatory bowel disease
(IBD).MethodsReceiver operating characteristic (ROC) curves were
examined to determine suitable thresholds for plasma EBV DNA load in
diagnosing intestinal EBV infection and predicting its
prognosis.Results108 patients were retrospectively assigned to the test
group, while 56 patients were included in the validation group. Plasma
EBV DNA loads were significantly higher in the intestinal EBV infection
group compared to the non-intestinal EBV infection group (Median: 2.02 Ã
102 copies/mL, interquartile range [IQR]: 5.49 Ã 101-6.34Ã103 copies/mL
versus 4.2Ã101 copies/mL, IQR: 1.07 Ã101-6.08Ã101 copies/mL; P &lt;
0.0001). Plasma EBV DNA levels at 9.21Ã101 and 6.77Ã101 copies/mL proved
beneficial for the identification and prognostication in intestinal EBV
infection, respectively. Values of 0.82 and 0.71 were yielded by the
area under the ROC curve (AUC) in the test cohort, corresponding to
sensitivities of 84.38% (95% confidence interval [95%CI]: 68.25%-93.14%)
and 87.5% (95%CI: 69%-95.66%), specificities of 83.33% (95%CI:
64.15%-93.32%) and 68.09% (95%CI: 53.83%-79.6%), positive predictive
values (PPV) of 87.1% (95%CI: 71.15%-94.87%) and 58.33% (95%CI:
42.2%-72.86%), and positive likelihood ratios (LR+) of 5.06 and 2.74 in
the validation cohort, respectively. Furthermore, a plasma EBV DNA load
of 5.4Ã102 copies/mL helped differentiate IBD with intestinal EBV
infection from primary intestinal EBV-positive lymphoproliferative
disorders (PIEBV+LPDs), achieving an AUC of 0.85 within the test cohort,
as well as 85% sensitivity (95%CI: 63.96%-94.76%), 91.67% specificity
(95%CI: 64.61%-99.57%), 94.44% PPV (95%CI: 74.24%-99.72%), and an LR+ of
10.2 in the validation cohort.ConclusionsPlasma EBV DNA load
demonstrates notable potential in distinguishing between different
patient cohorts with intestinal EBV infection, although its sensitivity
requires further optimization for clinical application.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1526633"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1526633</a></p>
<hr />
<h2
id="clinical-features-and-risk-factors-of-hiv-infected-patients-with-intracerebral-hemorrhage-a-retrospective-study-with-propensity-score-matching-analysis">Clinical
features and risk factors of HIV-infected patients with intracerebral
hemorrhage: a retrospective study with propensity score matching
analysis</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>PurposeTo investigate the clinical features and risk factors of the
human immunodeficiency virus (HIV)-infected patients with intracerebral
hemorrhage (ICH).Patients and methodsThe patients with HIV-infected
without ICH group were matched to the group of HIV-infected ICH
patients. Logistic regression analysis using 1:1 propensity score
matching (PSM) was performed to investigate the independent risk factors
for ICH in HIV-infected patients. The receiver operating characteristic
(ROC) curve was configured to calculate the optimal predictors of ICH in
HIV-infected patients.ResultsA total of 59 HIV-infected patients with
ICH and 180 HIV-infected patients without ICH were included. A cohort of
118 patients was ascertained utilizing PSM. Multivariate binary logistic
regression analysis revealed that drug abuse-related HIV-infected,
prolonged prothrombin time (PT), and elevated triglyceride (TG) levels
were independent risk factors of ICH in HIV-infected patients. The ROC
curve demonstrated that the combined predictor, composed of drug
abuse-related HIV-infected, prolonged PT, and elevated TG levels,
exhibited the highest area under the curve (AUC), with a cut-off point
at 0.426, sensitivity of 78%, and specificity of 81.4%.ConclusionThe
present study revealed that a valuable factor combined with drug
abuse-related HIV-infected, prolonged PT, and elevated serum TG levels
could serve as predictors of ICH in HIV-infected patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1498327"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1498327</a></p>
<hr />
<h2
id="effects-of-fermented-artemisia-annua-on-the-intestinal-microbiota-and-metabolites-of-hu-lambs-with-naturally-infected-with-eimeria-spp.">Effects
of fermented Artemisia annua on the intestinal microbiota and
metabolites of Hu lambs with naturally infected with Eimeria spp.</h2>
<p>date: 2025-01-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSheep coccidiosis could disturb the balance of intestinal
microbiota, causing diarrhea, and even death in lambs. Chemical drugs
are the primary method of treating sheep coccidiosis, but their use will
bring drug resistance, toxic side effects, drug residues, and other
problems. Chinese herbal medicines are investigated as alternative
methods for controlling coccidian infections.MethodsIn this study, the
effect of fermented Artemisia annua (FA) on oocysts per gram (OPG),
average daily gain (ADG), and expression of inflammatory factors were
investigated in lambs that were naturally infected with
coccidia.ResultsThe results showed that the FA had similar
anti-coccidiosis effect to the original drug, while the FA demonstrated
a more significant effect on weight gain, and a better ability to reduce
the inflammatory response compared to the unfermented drug during the
treatment period (P &lt; 0.05). Furthermore, High-throughput sequencing
technology was used to study the effects of FA on intestinal microbiota,
and fecal metabolites of naturally infected lambs. The species richness
of intestinal microbiota of lambs was significantly improved by FA. The
abundance of bacteria unclassified_Muribaculaceae, and UCG_005 were
increased by fermentation of A. annua. The abundance of bacteria
Escherichia_Shigella, unclassified_Clostridia_UCG_014, and Alistipes was
reduced. The prevention, and treatment of coccidiosis by fermentation of
A. annua may also be related to a series of metabolites affected by
intestinal microbiota, including artemisinin, Lysyl-Proline, and
TRP-tyrosine.ConclusionFA was found to have superior anti-coccidiosis,
anti-inflammatory, and weight gain effects compared to the original
Artemisia annua. Intestinal microbes and metabolites such as
unclassified_Muribaculaceae, UCG-005, and Artemisinin were identified,
suggesting their potential significance. Alistipes was proposed as a
biomarker for predicting intestinal coccidia outbreak risk in lambs,
pending further validation. The correlation between microbiota, and
metabolites may provide new insights into pathogenic changes associated
with Eimeria spp.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1448516"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1448516</a></p>
<hr />
<h2
id="human-milk-oligosaccharides-bridging-the-gap-in-intestinal-microbiota-between-mothers-and-infants">Human
milk oligosaccharides: bridging the gap in intestinal microbiota between
mothers and infants</h2>
<p>date: 2025-01-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Breast milk is an essential source of infant nutrition. It is also a
vital determinant of the structure and function of the infant intestinal
microbial community, and it connects the mother and infant intestinal
microbiota. Human milk oligosaccharides (HMOs) are a critical component
in breast milk. HMOs can reach the babyâs colon entirely from milk and
become a fermentable substrate for some intestinal microorganisms. HMOs
can enhance intestinal mucosal barrier function and affect the
intestinal function of the host through immune function, which has a
therapeutic effect on specific infant intestinal diseases, such as
necrotizing enterocolitis. In addition, changes in infant intestinal
microbiota can reflect the maternal intestinal microbiota. HMOs are a
link between the maternal intestinal microbiota and infant intestinal
microbiota. HMOs affect the intestinal microbiota of infants and are
related to the maternal milk microbiota. Through breastfeeding, maternal
microbiota and HMOs jointly affect infant intestinal bacteria.
Therefore, HMOs positively influence the establishment and balance of
the infant microbial community, which is vital to ensure infant
intestinal function. Therefore, HMOs can be used as a supplement and
alternative therapy for infant intestinal diseases.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1386421"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1386421</a></p>
<hr />
<h2
id="plasmodium-falciparum-transcription-factor-ap2-06b-is-mutated-at-high-frequency-in-southeast-asia-but-does-not-associate-with-drug-resistance">Plasmodium
falciparum transcription factor AP2-06B is mutated at high frequency in
Southeast Asia but does not associate with drug resistance</h2>
<p>date: 2025-01-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionA continuing challenge for malaria control is the ability
of Plasmodium falciparum to develop resistance to antimalarial drugs.
Members within the Plasmodium transcription factor family AP2 regulate
the growth and development of the parasite, and are also thought to be
involved in unclear aspects of drug resistance. Here we screened for
single nucleotide polymorphisms (SNPs) within the AP2 family and
identified 6 non-synonymous mutations within AP2-06B (PF3D7_0613800),
with allele frequencies greater than 0.05. One mutation, K3124R, was
located in a PfAP2-06B AP2 domain.MethodsTo investigate transcriptional
regulation by PfAP2-06B, ChIP-seq assays were performed on
3D7/PfAP2-06B-GFP schizonts using antibodies against GFP. The DNA
sequences of the artemisinin-resistant CWX and the quinoline-resistant
strains PfDd2 and Pf7G8 were analyzed for the genetic diversity of
AP2-06B, compared with the Pf3D7 strain as a reference sequence. To
determine whether AP2-06B can alter the expression of pfk13 and pfcrt,
as well as cause artemisinin and quinoline resistance in Plasmodium, we
generated both a K3124R mutation and conditional knockdown of AP2-06B in
Pf3D7 using CRISPR/Cas9-mediated genome editing.ResultsChIP-Seq analysis
showed that AP2-06B can bind to the loci of the Plasmodium genes pfk13
and pfcrt. The AP2-06B K3124R mutation was also found in the
artemisinin-resistant parasite strain CWX and the chloroquine-resistant
strains Dd2 and 7G8. Contrary to expectation, Pf3D7 Plasmodium lines
modified by either K3124R mutation of AP2-06B or conditional knockdown
of AP2-06B did not have altered sensitivity to artemisinin or quinolines
by modulating pfk13 or pfcrt expression.DiscussionAP2-06B was predicted
to be associated with artemisinin and quinoline resistance, but no
change in resistance was observed after mutation or conditional
knockdown. Given the multigenic nature of resistance, it might be
difficult to recreate a resistance phenotype. In conclusion, whether
AP2-06B regulates the development of artemisinin or quinoline resistance
remains to be studied.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1521152"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1521152</a></p>
<hr />
<h2
id="optimizing-the-production-and-efficacy-of-antimicrobial-bioactive-compounds-from-streptomyces-kanamyceticus-in-combating-multi-drug-resistant-pathogens">Optimizing
the production and efficacy of antimicrobial bioactive compounds from
Streptomyces kanamyceticus in combating multi-drug-resistant
pathogens</h2>
<p>date: 2025-01-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe rise of antibiotic-resistant pathogens has intensified
the search for novel antimicrobial agents. This study aimed to isolate
Streptomyces kanamyceticus from local soil samples and evaluate its
antimicrobial properties, along with optimizing the production of
bioactive compounds.MethodsSoil samples were collected from local
regions, processed, and analysed for Streptomyces strains isolation
using morphological characteristics and molecular identification through
16S rRNA gene PCR assay. Antimicrobial activity was assessed against
Escherichia coli, Staphylococcus aureus, Bacillus subtilis, and Candida
albicans using the double-layer method, while Minimum Inhibitory
Concentration (MIC) values were determined. The extracted compounds
underwent Fourier Transform Infrared Spectroscopy (FTIR) analysis for
functional group identification. Optimization of bioactive compound
production was performed using a Central Composite Design (CCD) coupled
with Partial Least Squares Regression (PLSR).ResultsA total of 25
distinct Streptomyces strains were isolated, with seven confirmed as S.
kanamyceticus. These strains exhibited antimicrobial activity, with
inhibition zones reaching 30 mm and MIC values between 20 and 70 Âµg/mL.
The extraction yielded 150-200 mL of bioactive compounds. Optimization
studies revealed that a medium containing 10 g/L glucose and 10 g/L
glycine max meal maximized antibiotic production.ConclusionThis study
confirmed that S. kanamyceticus is a promising source of novel
antibiotics. The combination of microbial isolation, antimicrobial
testing, and statistical optimization successfully enhanced the
production of bioactive compounds, contributing to the search for
effective antimicrobial agents against resistant pathogens.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1500440"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1500440</a></p>
<hr />
<h2
id="ammonia-and-hydrogen-sulfide---new-insights-into-gut-microbiota-and-male-infertility-through-meta-analysis">Ammonia
and hydrogen sulfide - new insights into gut microbiota and male
infertility through meta-analysis</h2>
<p>date: 2025-01-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAmmonia (NH3) and hydrogen sulfide (H2S) are produced
during digestion in the human gut, yet the impact of these internally
generated gases on male reproduction have received limited attention in
scientific research.MethodsWe systematically reviewed 935 scientific
publications, spanning from 1947 to 2023, focusing on external or
internal NH3 and/or H2S, male infertility, and gut microbiota.
Meta-analysis was conducted to evaluate the summary relative risk (RR)
and 95% confidence intervals (CIs) of combined studies.ResultsOur
findings revealed that the internal NH3 and/or H2S were negatively
related to the Lactobacillus, which is beneficial to male fertility,
whereas NH3 and H2S were positively related to Bacteroides, which showed
negative effects on male fertility. The meta-analysis comparing
Lactobacillus and Bacteroides levels with NH3 showed statistically
significant results (p&lt;0.001).ConclusionsThe meta-analysis is the
first to confirm these facts and explored the potential existence of a
gut microbiota-inner gases-male fertility axis in the human gut.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1449453"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1449453</a></p>
<hr />
<h2
id="editorial-mpox-understanding-the-scientific-gaps-to-combat-the-threat">Editorial:
Mpox: understanding the scientific gaps to combat the threat</h2>
<p>date: 2025-01-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1538718"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1538718</a></p>
<hr />
<h2
id="increased-rate-of-multidrug-resistant-gram-negative-bacterial-infections-in-hospitalized-immunocompromised-pediatric-patients">Increased
rate of multidrug-resistant gram-negative bacterial infections in
hospitalized immunocompromised pediatric patients</h2>
<p>date: 2025-01-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMultidrug resistant Gram-negative bacterial infections
are considered a major public health threat. Immunocompromised pediatric
patients are at a great risk of severe or overwhelming infections. The
aim of this study was to describe the frequency of infections with
multidrug resistant (MDR) Gram-negative bacteria (GNB) in
immunocompromised pediatric patients and to determine the risk factors.
In addition, we aimed to identify the antimicrobial resistance patterns
of these isolates.Materials and methodsThis was a retrospective
observational study conducted at the American University of Beirut
Medical Center (AUBMC) from 2009 to 2017. The study included
immunocompromised patients 18 years of age or younger with infections
caused by Gram-negative bacteria isolated from a sterile site, or
nonsterile site in the setting of clinical infection.ResultsA total of
381 episodes of infection with GNB in 242 immunocompromised pediatric
patients were identified. The mean age was 7.7 years. The most common
pathogens were Enterobacterales followed by Pseudomonas and
Acinetobacter spp. MDR GNB infections predominated causing 72% of the
episodes, with alarming MDR rates among Escherichia coli (95.7%) and
Klebsiella pneumoniae (82.7%). The overall rate of MDR GNB isolated
increased from 62.7% in 2015 to 90% in 2017. Thrombocytopenia,
chemotherapy and previous colonization or infection with the same
organism during the past 12 months were found to be independent risk
factors for infection with MDR GNB.ConclusionThis study provides data on
the epidemiology of infections with MDR GNB in immunocompromised
pediatric patients and illustrates the alarmingly high prevalence of
these infections. This necessitates the frequent revisiting of treatment
guidelines in these high-risk patients and the implementation of
judicious antimicrobial stewardship programs and infection control
policies to stabilize or decrease the prevalence of these
infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382500"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382500</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
