<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.17</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="parks.html" title="News about Parks">Parks</a></li>
	<li><a href="tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="2023/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.17">health 2024.17</h1>
<p>(date: 2024-04-25 06:01:51)</p>
<hr />
<h2
id="application-value-of-metagenomic-next-generation-sequencing-in-hematological-patients-with-high-risk-febrile-neutropenia">Application
value of metagenomic next-generation sequencing in hematological
patients with high-risk febrile neutropenia</h2>
<p>date: 2024-04-25, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMetagenomic next-generation sequencing (mNGS) is a novel
non-invasive and comprehensive technique for etiological diagnosis of
infectious diseases. However, its practical significance has been seldom
reported in the context of hematological patients with high-risk febrile
neutropenia, a unique patient group characterized by neutropenia and
compromised immune responses.MethodsThis retrospective study evaluated
the results of plasma cfDNA sequencing in 164 hematological patients
with high-risk febrile neutropenia. We assessed the diagnostic efficacy
and clinical impact of mNGS, comparing it with conventional
microbiological tests.ResultsmNGS identified 68 different pathogens in
111 patients, whereas conventional methods detected only 17 pathogen
types in 36 patients. mNGS exhibited a significantly higher positive
detection rate than conventional methods (67.7% vs. 22.0%, P &lt;
0.001). This improvement was consistent across bacterial (30.5%
vs. 9.1%), fungal (19.5% vs. 4.3%), and viral (37.2% vs. 9.1%)
infections (P &lt; 0.001 for all comparisons). The anti-infective
treatment strategies were adjusted for 51.2% (84/164) of the patients
based on the mNGS results.ConclusionsmNGS of plasma cfDNA offers
substantial promise for the early detection of pathogens and the timely
optimization of anti-infective therapies in hematological patients with
high-risk febrile neutropenia.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366908"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366908</a></p>
<hr />
<h2
id="survey-of-rickettsia-species-in-hematophagous-arthropods-from-endemic-areas-for-japanese-spotted-fever-in-china">Survey
of Rickettsia species in hematophagous arthropods from endemic areas for
Japanese spotted fever in China</h2>
<p>date: 2024-04-25, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Japanese spotted fever (JSF) is caused by Rickettsia japonica, mainly
vectored by hard ticks. However, whether R. japonica can be transmitted
by other arthropods remains unknown. Moreover, it is of interest to
investigate whether other Rickettsia species cause spotted fever in
endemic areas. In this study, a survey of Rickettsia species was
performed in hematophagous arthropods (mosquitoes, tabanids, and ticks)
from endemic areas for JSF in Hubei Province, central China. The results
showed that the diversity and prevalence of Rickettsia species in
mosquitoes are low, suggesting that mosquitoes may not be the vector of
zoonotic Rickettsia species. A novel Rickettsia species showed a high
prevalence (16.31%, 23/141) in tabanids and was named “Candidatus
Rickettsia tabanidii.” It is closely related to Rickettsia from fleas
and mosquitoes; however, its pathogenicity in humans needs further
investigation. Five Rickettsia species were identified in ticks.
Rickettsia japonica, the agent of JSF, was detected only in
Haemaphysalis longicornis and Haemaphysalis hystricis, suggesting that
they may be the major vectors of R. japonica. Notably, two novel species
were identified in H. hystricis ticks, one belonging to the spotted
fever group and the other potentially belonging to the ancestral group.
The latter one named “Candidatus Rickettsia hubeiensis” may provide
valuable insight into the evolutionary history of Rickettsia.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384284"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384284</a></p>
<hr />
<h2
id="rutin-prevents-eqhv-8-induced-infection-and-oxidative-stress-via-nrf2ho-1-signaling-pathway">Rutin
prevents EqHV-8 induced infection and oxidative stress via Nrf2/HO-1
signaling pathway</h2>
<p>date: 2024-04-25, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe Nuclear factor erythroid 2-related factor 2
(Nrf2)/heme oxygenase-1 (HO-1) signaling pathway has been extensively
studied for its role in regulating antioxidant and antiviral responses.
The Equid herpesvirus type 8 (EqHV-8) poses a significant threat to the
equine industry, primarily manifesting as respiratory disease,
abortions, and neurological disorders in horses and donkeys. Oxidative
stress is considered a key factor associated with pathogenesis of EqHV-8
infection. Unfortunately, there is currently a dearth of therapeutic
interventions available for the effective control of EqHV-8. Rutin has
been well documented for its antioxidant and antiviral potential. In
current study we focused on the evaluation of Rutin as a potential
therapeutic agent against EqHV-8 infection.MethodsFor this purpose, we
encompassed both in-vitro and in-vivo investigations to assess the
effectiveness of Rutin in combatting EqHV-8 infection.Results and
DiscussionThe results obtained from in vitro experiments demonstrated
that Rutin exerted a pronounced inhibitory effect on EqHV-8 at multiple
stages of the viral life cycle. Through meticulous experimentation, we
elucidated that Rutin’s antiviral action against EqHV-8 is intricately
linked to the Nrf2/HO-1 signaling pathway-mediated antioxidant response.
Activation of this pathway by Rutin was found to significantly impede
EqHV-8 replication, thereby diminishing the viral load. This mechanistic
insight not only enhances our understanding of the antiviral potential
of Rutin but also highlights the significance of antioxidant stress
responses in combating EqHV-8 infection. To complement our in vitro
findings, we conducted in vivo studies employing a mouse model. These
experiments revealed that Rutin administration resulted in a substantial
reduction in EqHV-8 infection within the lungs of the mice, underscoring
the compound’s therapeutic promise in vivo.ConclusionIn summation, our
finding showed that Rutin holds promise as a novel and effective
therapeutic agent for the prevention and control of EqHV-8
infections.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1386462"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1386462</a></p>
<hr />
<h2 id="editorial-bacterial-surface-polymers">Editorial: Bacterial
surface polymers</h2>
<p>date: 2024-04-25, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1415799"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1415799</a></p>
<hr />
<h2
id="antimicrobial-peptides-in-livestock-a-review-with-a-one-health-approach">Antimicrobial
peptides in livestock: a review with a one health approach</h2>
<p>date: 2024-04-24, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Antimicrobial peptides (AMPs), often referred to as nature’s
antibiotics, are ubiquitous in living organisms, spanning from bacteria
to humans. Their potency, versatility, and unique mechanisms of action
have garnered significant research attention. Unlike conventional
antibiotics, peptides are biodegradable, adding to their appeal as
potential candidates to address bacterial resistance in livestock
farming—a challenge that has been under scrutiny for decades. This issue
is complex and multifactorial, influenced by a variety of components.
The World Health Organization (WHO) has proposed a comprehensive
approach known as One Health, emphasizing the interconnectedness of
human-animal-environment relationships in tackling such challenges. This
review explores the application of AMPs in livestock farming and how
they can mitigate the impact of this practice within the One Health
framework.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1339285"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1339285</a></p>
<hr />
<h2
id="a-combination-of-gra3-gra6-and-gra7-peptides-offer-a-useful-tool-for-serotyping-type-ii-and-iii-toxoplasma-gondii-infections-in-sheep-and-pigs">A
combination of GRA3, GRA6 and GRA7 peptides offer a useful tool for
serotyping type II and III Toxoplasma gondii infections in sheep and
pigs</h2>
<p>date: 2024-04-24, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The clinical consequences of toxoplasmosis are greatly dependent on
the Toxoplasma gondii strain causing the infection. To better understand
its epidemiology and design appropriate control strategies, it is
important to determine the strain present in infected animals.
Serotyping methods are based on the detection of antibodies that react
against segments of antigenic proteins presenting strain-specific
polymorphic variations, offering a cost-effective, sensitive, and
non-invasive alternative to genotyping techniques. Herein, we evaluated
the applicability of a panel of peptides previously characterized in
mice and humans to serotype sheep and pigs. To this end, we used 51
serum samples from experimentally infected ewes (32 type II and 19 type
III), 20 sheep samples from naturally infected sheep where the causative
strain was genotyped (18 type II and 2 type III), and 40 serum samples
from experimentally infected pigs (22 type II and 18 type III). Our
ELISA test results showed that a combination of GRA peptide homologous
pairs can discriminate infections caused by type II and III strains of
T. gondii in sheep and pigs. Namely, the GRA3-I/III-43 vs. GRA3-II-43,
GRA6-I/III-213 vs. GRA6-II-214 and GRA6-III-44 vs. GRA6-II-44 ratios
showed a statistically significant predominance of the respective
strain-type peptide in sheep, while in pigs, in addition to these three
peptide pairs, GRA7-II-224 vs. GRA7-III-224 also showed promising
results. Notably, the GRA6-44 pair, which was previously deemed
inefficient in mice and humans, showed a high prediction capacity,
especially in sheep. By contrast, GRA5-38 peptides failed to correctly
predict the strain type in most sheep and pig samples, underpinning the
notion that individual standardization is needed for each animal
species. Finally, we recommend analyzing for each animal at least 2
samples taken at different time points to confirm the obtained
results.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384393"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1384393</a></p>
<hr />
<h2
id="integrating-environmental-conservation-and-public-health-strategies-to-combat-zoonotic-disease-emergence-a-call-to-action-from-the-amazon-rainforest">Integrating
environmental conservation and public health strategies to combat
zoonotic disease emergence: a call to action from the Amazon
rainforest</h2>
<p>date: 2024-04-24, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1405472"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1405472</a></p>
<hr />
<h2
id="updates-on-traditional-methods-for-combating-malaria-and-emerging-wolbachia-based-interventions">Updates
on traditional methods for combating malaria and emerging
Wolbachia-based interventions</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The escalating challenge of malaria control necessitates innovative
approaches that extend beyond traditional control strategies. This
review explores the incorporation of traditional vector control
techniques with emerging Wolbachia-based interventions. Wolbachia, a
naturally occurring bacteria, offers a novel approach for combatting
vector-borne diseases, including malaria, by reducing the mosquitoes’
ability to transmit these diseases. The study explores the rationale for
this integration, presenting various case studies and pilot projects
that have exhibited significant success. Employing a multi-dimensional
approach that includes community mobilization, environmental
modifications, and new biological methods, the paper posits that
integrated efforts could mark a turning point in the struggle against
malaria. Our findings indicate that incorporating Wolbachia-based
strategies into existing vector management programs not only is feasible
but also heightens the efficacy of malaria control initiatives in
different countries especially in Pakistan. The paper concludes that
continued research and international collaboration are imperative for
translating these promising methods from the laboratory to the field,
thereby offering a more sustainable and effective malaria control
strategy.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1330475"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1330475</a></p>
<hr />
<h2
id="lactobacillus-rhamnosus-gg-powder-supplementation-alleviates-intestinal-injury-in-piglets-challenged-by-porcine-epidemic-diarrhea-virus">Lactobacillus
rhamnosus GG powder supplementation alleviates intestinal injury in
piglets challenged by porcine epidemic diarrhea virus</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Porcine epidemic diarrhea virus (PEDV) has become a challenging
problem in pig industry worldwide, causing significant profit losses.
Lactobacillus rhamnosus GG (LGG) has been regarded as a safe probiotic
strain and has been shown to exert protective effects on the intestinal
dysfunction caused by PEDV. This study evaluated the effect of LGG on
the gut health of lactating piglets challenged with PEDV. Fifteen
piglets at 7 days of age were equally assigned into 3 groups (5 piglets
per group): 1) control group (basal diet); 2) PEDV group: (basal diet +
PEDV challenged); 3) LGG + PEDV group (basal diet + 3×109 CFU/pig/day
LGG + PEDV). The trial lasted 11 days including 3 days of adaptation.
The treatment with LGG was from D4 to D10. PEDV challenge was carried
out on D8. PEDV infection disrupted the cell structure, undermined the
integrity of the intestinal tract, and induced oxidative stress, and
intestinal damage of piglets. Supplementation of LGG improved intestinal
morphology, enhanced intestinal antioxidant capacity, and alleviated
jejunal mucosal inflammation and lipid metabolism disorders in
PEDV-infected piglets, which may be regulated by LGG by altering the
expression of TNF signaling pathway, PPAR signaling pathway, and fat
digestion and absorption pathway.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1371916"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1371916</a></p>
<hr />
<h2
id="chikungunya-virus-infection-in-human-microglial-c20-cells-induces-mitochondria-mediated-apoptosis">Chikungunya
virus infection in human microglial C20 cells induces
mitochondria-mediated apoptosis</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionChikungunya virus (CHIKV) infection is associated with
acute clinical manifestations and chronic joint inflammation. CHIKV has
emerged as a significant causative agent of central nervous system (CNS)
complications, including encephalitis and related sequelae. Microglial
cells, crucial for immune responses and tissue repair in the CNS, play a
vital role in the host response to viral infections, with their
activation potentially leading to either protection or pathology. In
this study, the infection biology of CHIKV in the C20 human microglial
cell line was investigated.MethodsThe permissiveness of C20 cells to
CHIKV infection was assessed, and viral replication kinetics were
compared to Vero E6 cells. Cytopathic effects of CHIKV infection on C20
cells were examined, along with ultrastructural changes using
transmission electron microscopy. Additionally, apoptosis induction,
mitochondrial membrane potential, and alterations in cell surface marker
expression were evaluated by flow cytometry.ResultsCHIKV infection
demonstrated permissiveness in C20 cells, similar to Vero cells,
resulting in robust viral replication and cytopathic effects.
Ultrastructural analysis revealed viral replication, mature virion
formation, and distinctive cytoplasmic and nuclear changes in infected
C20 cells. CHIKV infection induced significant apoptosis in C20 cells,
accompanied by mitochondrial membrane depolarization and altered
expression of cell surface markers such as CD11c, CD14, and HLA-DR.
Notably, decreased CD14 expression was observed in CHIKV-infected C20
cells.DiscussionThe study findings suggest that CHIKV infection induces
apoptosis in C20 microglial cells via the mitochondrial pathway, with
significant alterations in cell surface marker expression, particularly
CD14 that is linked with apoptosis induction. These observations provide
valuable insights into the role of human microglial cells in the host
response to CHIKV infection and contribute to the knowledge on the
neuropathogenesis of this virus.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1380736"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1380736</a></p>
<hr />
<h2
id="editorial-host-pathogen-interactions-at-the-blood-brain-barriers">Editorial:
Host-pathogen interactions at the blood-brain barriers</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1412140"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1412140</a></p>
<hr />
<h2
id="integrating-machine-learning-algorithms-and-single-cell-analysis-to-identify-gut-microbiota-related-macrophage-biomarkers-in-atherosclerotic-plaques">Integrating
machine learning algorithms and single-cell analysis to identify gut
microbiota-related macrophage biomarkers in atherosclerotic plaques</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe relationship between macrophages and the gut microbiota
in patients with atherosclerosis remains poorly defined, and effective
biological markers are lacking. This study aims to elucidate the
interplay between gut microbial communities and macrophages, and to
identify biomarkers associated with the destabilization of
atherosclerotic plaques. The goal is to enhance our understanding of the
underlying molecular pathways and to pave new avenues for diagnostic
approaches and therapeutic strategies in the disease.MethodsThis study
employed Weighted Gene Co-expression Network Analysis (WGCNA) and
differential expression analysis on atherosclerosis datasets to identify
macrophage-associated genes and quantify the correlation between these
genes and gut microbiota gene sets. The Random Forest algorithm was
utilized to pinpoint PLEK, IRF8, BTK, CCR1, and CD68 as gut
microbiota-related macrophage genes, and a nomogram was constructed.
Based on the top five genes, a Non-negative Matrix Factorization (NMF)
algorithm was applied to construct gut microbiota-related macrophage
clusters and analyze their potential biological alterations. Subsequent
single-cell analyses were conducted to observe the expression patterns
of the top five genes and the interactions between immune cells.
Finally, the expression profiles of key molecules were validated using
clinical samples from atherosclerosis patients.ResultsUtilizing the
Random Forest algorithm, we ultimately identified PLEK, IRF8, CD68,
CCR1, and BTK as gut microbiota-associated macrophage genes that are
upregulated in atherosclerotic plaques. A nomogram based on the
expression of these five genes was constructed for use as an auxiliary
tool in clinical diagnosis. Single-cell analysis confirmed the specific
expression of gut microbiota-associated macrophage genes in macrophages.
Clinical samples substantiated the high expression of PLEK in unstable
atherosclerotic plaques.ConclusionGut microbiota-associated macrophage
genes (PLEK, IRF8, CD68, CCR1, and BTK) may be implicated in the
pathogenesis of atherosclerotic plaques and could serve as diagnostic
markers to aid patients with atherosclerosis.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1395716"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1395716</a></p>
<hr />
<h2
id="detection-of-novel-plasmodium-falciparum-coronin-gene-mutations-in-a-recrudescent-act-treated-patient-in-south-western-nigeria">Detection
of novel Plasmodium falciparum coronin gene mutations in a recrudescent
ACT-treated patient in South-Western Nigeria</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundRoutine surveillance for antimalarial drug resistance is
critical to sustaining the efficacy of artemisinin-based Combination
Therapies (ACTs). Plasmodium falciparum kelch-13 (Pfkelch-13) and
non-Pfkelch-13 artemisinin (ART) resistance-associated mutations are
uncommon in Africa. We investigated polymorphisms in Plasmodium
falciparum actin-binding protein (Pfcoronin) associated with in vivo
reduced sensitivity to ART in Nigeria.MethodsFifty-two P. falciparum
malaria subjects who met the inclusion criteria were followed up in a
28-day therapeutic efficacy study of artemether-lumefantrine in Lagos,
Nigeria. Parasite detection was done by microscopy and molecular
diagnostic approaches involving PCR amplification of genes for Pf18S
rRNA, varATS, telomere-associated repetitive elements-2 (TARE-2).
Pfcoronin and Pfkelch-13 genes were sequenced bi-directionally while
clonality of infections was determined using 12 neutral P. falciparum
microsatellite loci and msp2 analyses. Antimalarial drugs
(sulfadoxine-pyrimethamine, amodiaquine, chloroquine and some
quinolones) resistance variants (DHFR_51, DHFR_59, DHFR_108, DHFR_164,
MDR1_86, MDR1_184, DHPS_581 and DHPS_613) were genotyped by
high-resolution melting (HRM) analysis.ResultsA total of 7 (26.92%)
cases were identified either as early treatment failure, late
parasitological failure or late clinical failure. Of the four
post-treatment infections identified as recrudescence by msp2 genotypes,
only one was classified as recrudescence by multilocus microsatellites
genotyping. Microsatellite analysis revealed no significant difference
in the mean allelic diversity, He, (P = 0.19, Mann-Whitney test). Allele
sizes and frequency per locus implicated one isolate. Genetic analysis
of this isolate identified two new Pfcoronin SNVs (I68G and L173F) in
addition to the P76S earlier reported. Linkage-Disequilibrium as a
standardized association index, IAS, between multiple P. falciparum loci
revealed significant LD (IAS = 0.2865, P=0.02, Monte-Carlo simulation)
around the neutral microsatellite loci. The pfdhfr/pfdhps/pfmdr1 drug
resistance-associated haplotypes combinations,
(108T/N/51I/164L/59R/581G/86Y/184F), were observed in two
samples.ConclusionPfcoronin mutations identified in this study, with
potential to impact parasite clearance, may guide investigations on
emerging ART tolerance in Nigeria, and West African endemic
countries.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366563"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366563</a></p>
<hr />
<h2
id="high-fat-diets-induce-inflammatory-imdnfκb-signaling-via-gut-microbiota-remodeling-in-drosophila">High-fat
diets induce inflammatory IMD/NFκB signaling via gut microbiota
remodeling in Drosophila</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>High-fat diets (HFDs), a prevailing daily dietary style worldwide,
induce chronic low-grade inflammation in the central nervous system and
peripheral tissues, promoting a variety of diseases including
pathologies associated with neuroinflammation. However, the mechanisms
linking HFDs to inflammation are not entirely clear. Here, using a
Drosophila HFD model, we explored the mechanism of HFD-induced
inflammation in remote tissues. We found that HFDs activated the
IMD/NFκB immune pathway in the head through remodeling of the commensal
gut bacteria. Removal of gut microbiota abolished such HFD-induced
remote inflammatory response. Further experiments revealed that HFDs
significantly increased the abundance of Acetobacter malorum in the gut,
and the re-association of this bacterium was sufficient to elicit
inflammatory response in remote tissues. Mechanistically, Acetobacter
malorum produced a greater amount of peptidoglycan (PGN), a well-defined
microbial molecular pattern that enters the circulation and remotely
activates an inflammatory response. Our results thus show that HFDs
trigger inflammation mediated by a bacterial molecular pattern that
elicits host immune response.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347716"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347716</a></p>
<hr />
<h2
id="editorial-combatting-tropical-diseases-a-multi-level-approach">Editorial:
Combatting tropical diseases: a multi-level approach</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1406964"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1406964</a></p>
<hr />
<h2
id="editorial-mechanobiology-of-biofilms-and-associated-host--pathogen-interactions">Editorial:
Mechanobiology of biofilms and associated host -pathogen
interactions</h2>
<p>date: 2024-04-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416131"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416131</a></p>
<hr />
<h2
id="genomic-insights-into-the-diversity-virulence-and-antimicrobial-resistance-of-group-b-streptococcus-clinical-isolates-from-saudi-arabia">Genomic
insights into the diversity, virulence, and antimicrobial resistance of
group B Streptococcus clinical isolates from Saudi Arabia</h2>
<p>date: 2024-04-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionDetailed assessment of the population structure of group
B Streptococcus (GBS) among adults is still lacking in Saudi Arabia.
Here we characterized a representative collection of isolates from
colonized and infected adults.MethodsGBS isolates (n=89) were sequenced
by Illumina and screened for virulence and antimicrobial resistance
determinants. Genetic diversity was assessed by single nucleotide
polymorphisms and core-genome MLST analyses.ResultsGenome sequences
revealed 28 sequence types (STs) and nine distinct serotypes, including
uncommon serotypes VII and VIII. Majority of these STs (n=76) belonged
to the human-associated clonal complexes (CCs) CC1 (33.71%), CC19
(25.84%), CC17 (11.24%), CC10/CC12 (7.87%), and CC452 (6.74%). Major CCs
exhibited intra-lineage serotype diversity, except for the hypervirulent
CC17, which exclusively expressed serotype III. Virulence profiling
revealed that nearly all isolates (94.38%) carried at least one of the
four alpha family protein genes (i.e., alphaC, alp1, alp2/3, and rib),
and 92.13% expressed one of the two serine-rich repeat surface proteins
Srr1 or Srr2. In addition, most isolates harbored the pilus island
(PI)-2a alone (15.73%) or in combination with PI-1 (62.92%), and those
carrying PI-2b alone (10.11%) belonged to CC17. Phylogenetic analysis
grouped the sequenced isolates according to CCs and further subdivided
them along with their serotypes. Overall, isolates across all CC1
phylogenetic clusters expressed Srr1 and carried the PI-1 and PI-2a
loci, but differed in genes encoding the alpha-like proteins. CC19
clusters were dominated by the III/rib/srr1/PI-1+PI-2a (43.48%, 10/23)
and V/alp1/srr1/PI-1+PI-2a (34.78%, 8/23) lineages, whereas most CC17
isolates (90%, 9/10) had the same III/rib/srr2/P1-2b genetic background.
Interestingly, genes encoding the CC17-specific adhesins HvgA and Srr2
were detected in phylogenetically distant isolates belonging to ST1212,
suggesting that other highly virulent strains might be circulating
within the species. Resistance to macrolides and/or lincosamides across
all major CCs (n=48) was associated with the acquisition of erm(B)
(62.5%, 30/48), erm(A) (27.1%, 13/48), lsa(C) (8.3%, 4/48), and mef(A)
(2.1%, 1/48) genes, whereas resistance to tetracycline was mainly
mediated by presence of tet(M) (64.18%, 43/67) and tet(O) (20.9%, 14/67)
alone or in combination (13.43%, 9/67).DiscussionThese findings
underscore the necessity for more rigorous characterization of GBS
isolates causing infections.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1377993"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1377993</a></p>
<hr />
<h2
id="characterization-and-efficacy-against-carbapenem-resistant-acinetobacter-baumannii-of-a-novel-friunavirus-phage-from-sewage">Characterization
and efficacy against carbapenem-resistant Acinetobacter baumannii of a
novel Friunavirus phage from sewage</h2>
<p>date: 2024-04-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Carbapenem-resistant Acinetobacter baumannii (CRAB) has become a new
threat in recent years, owing to its rapidly increasing resistance to
antibiotics and new effective therapies are needed to combat this
pathogen. Phage therapy is considered to be the most promising
alternative for treating CRAB infections. In this study, a novel phage,
Ab_WF01, which can lyse clinical CRAB, was isolated and characterized
from hospital sewage. The multiplicity of infection, morphology,
one-step growth curve, stability, sensitivity, and lytic activity of the
phage were also investigated. The genome of phage Ab_WF01 was 41, 317 bp
in size with a GC content of 39.12% and encoded 51 open reading frames
(ORFs). tRNA, virulence, and antibiotic resistance genes were not
detected in the phage genome. Comparative genomic and phylogenetic
analyses suggest that phage Ab_WF01 is a novel species of the genus
Friunavirus, subfamily Beijerinckvirinae, and family Autographiviridae.
The in vivo results showed that phage Ab_WF01 significantly increased
the survival rate of CRAB-infected Galleria mellonella (from 0% to 70%
at 48 h) and mice (from 0% to 60% for 7 days). Moreover, after day 3
post-infection, phage Ab_WF01 reduced inflammatory response, with
strongly ameliorated histological damage and bacterial clearance in
infected tissue organs (lungs, liver, and spleen) in mouse CRAB
infection model. Taken together, these results show that phage Ab_WF01
holds great promise as a potential alternative agent with excellent
stability for against CRAB infections.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382145"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382145</a></p>
<hr />
<h2
id="modulation-of-campylobacter-jejuni-adhesion-to-biotic-model-surfaces-by-fungal-lectins-and-protease-inhibitors">Modulation
of Campylobacter jejuni adhesion to biotic model surfaces by fungal
lectins and protease inhibitors</h2>
<p>date: 2024-04-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Campylobacter jejuni, a Gram-negative bacterium, is one of the most
common causes of foodborne illness worldwide. Its adhesion mechanism is
mediated by several bacterial factors, including flagellum, protein
adhesins, lipooligosaccharides, proteases, and host factors, such as
surface glycans on epithelial cells and mucins. Fungal lectins,
specialized carbohydrate-binding proteins, can bind to specific glycans
on host and bacterial cells and thus influence pathogenesis. In this
study, we investigated the effects of fungal lectins and protease
inhibitors on the adhesion of C. jejuni to model biotic surfaces (mucin,
fibronectin, and collagen) and Caco-2 cells as well as the invasion of
Caco-2 cells. The lectins Marasmius oreades agglutinin (MOA) and
Laccaria bicolor tectonin 2 (Tec2) showed remarkable efficacy in all
experiments. In addition, different pre-incubations of lectins with C.
jejuni or Caco-2 cells significantly inhibited the ability of C. jejuni
to adhere to and invade Caco-2 cells, but to varying degrees.
Pre-incubation of Caco-2 cells with selected lectins reduced the number
of invasive C. jejuni cells the most, while simultaneous incubation
showed the greatest reduction in adherent C. jejuni cells. These results
suggest that fungal lectins are a promising tool for the prevention and
treatment of C. jejuni infections. Furthermore, this study highlights
the potential of fungi as a rich reservoir for novel anti-adhesive
agents.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391758"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391758</a></p>
<hr />
<h2
id="innate-immune-response-against-vector-borne-bunyavirus-infection-and-viral-countermeasures">Innate
immune response against vector-borne bunyavirus infection and viral
countermeasures</h2>
<p>date: 2024-04-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Bunyaviruses are a large group of important viral pathogens that
cause significant diseases in humans and animals worldwide. Bunyaviruses
are enveloped, single-stranded, negative-sense RNA viruses that infect a
wide range of hosts. Upon entry into host cells, the components of
viruses are recognized by host innate immune system, leading to the
activation of downstream signaling cascades to induce interferons (IFNs)
and other proinflammatory cytokines. IFNs bind to their receptors and
upregulate the expression of hundreds of interferon-stimulated genes
(ISGs). Many ISGs have antiviral activities and confer an antiviral
state to host cells. For efficient replication and spread, viruses have
evolved different strategies to antagonize IFN-mediated restriction.
Here, we discuss recent advances in our understanding of the
interactions between bunyaviruses and host innate immune response.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1365221"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1365221</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
