<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2024-11-12 11:20:05)</p>
<hr />
<h2
id="the-mammalian-ire1-inhibitor-4µ8c-exhibits-broad-anti-aspergillus-activity-in-vitro-and-in-a-treatment-model-of-fungal-keratitis">The
mammalian Ire1 inhibitor, 4µ8C, exhibits broad anti-Aspergillus activity
in vitro and in a treatment model of fungal keratitis</h2>
<p>date: 2024-11-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe fungal unfolded protein response consists of a
two-component relay in which the ER-bound sensor, IreA, splices and
activates the mRNA of the transcription factor, HacA. Previously, we
demonstrated that hacA is essential for Aspergillus fumigatus virulence
in a murine model of fungal keratitis (FK), suggesting the pathway could
serve as a therapeutic target. Here we investigate the antifungal
properties of known inhibitors of the mammalian Ire1 protein both in
vitro and in a treatment model of FK.MethodsThe antifungal activity of
Ire1 inhibitors was tested against conidia of several A. fumigatus
isolates by a broth microdilution assay and against fungal biofilm by
XTT reduction. The influence of 4μ8C on hacA mRNA splicing in A.
fumigatus was assessed through gel electrophoresis and qRT-PCR of UPR
regulatory genes. The toxicity and antifungal profile of 4μ8C in the
cornea was assessed by applying drops to uninfected or A.
fumigatus-infected corneas 3 times daily starting 4 hours
post-inoculation. Corneas were evaluated daily through slit-lamp imaging
and optical coherence tomography, or at endpoint through histology or
fungal burden quantification via colony forming units.ResultsAmong six
Ire1 inhibitors screened, the endonuclease inhibitor 4μ8C displayed the
strongest antifungal profile with an apparent fungicidal action. The
compound both blocked conidial germination and hyphal metabolism of A.
fumigatus Af293 in the same concentration range that blocked hacA
splicing and UPR gene induction (60-120 µM). Topical treatment of
sham-inoculated corneas with 0.5 and 2.5 mM 4μ8C did not impact corneal
clarity, but did transiently inhibit epithelialization of corneal
ulcers. Relative to vehicle-treated Af293-infected corneas, treatment
with 0.5 and 2.5 mM drug resulted in a 50% and &gt;90% reduction in
fungal load, respectively, the latter of which corresponded to an
absence of clinical signs of infection or corneal
pathology.ConclusionThe in vitro data suggest that 4μ8C displays
antifungal activity against A. fumigatus through the specific inhibition
of IreA. Topical application of the compound to the murine cornea can
furthermore block the establishment of infection, suggesting this class
of drugs can be developed as novel antifungals that improve visual
outcomes in FK patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477463"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477463</a></p>
<hr />
<h2
id="genomic-characterization-of-a-blakpc-2producing-incm2-plasmid-harboring-transposon-δtn6296-in-klebsiella-michiganensis">Genomic
characterization of a blaKPC-2–producing IncM2 plasmid harboring
transposon ΔTn6296 in Klebsiella michiganensis</h2>
<p>date: 2024-11-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Klebsiella michiganensis is an emerging hospital-acquired bacterial
pathogen, particularly strains harboring plasmid-mediated carbapenemase
genes. Here, we recovered and characterized a multidrug-resistant
strain, blaKPC-2–producing Klebsiella michiganensis LS81, which was
isolated from the abdominal drainage fluid of a clinical patient in
China, and further characterized the co-harboring plasmid. K.
michiganensis LS81 tested positive for the blaKPC-2 genes by PCR
sequencing, with blaKPC-2 located on a plasmid as confirmed by S1
nuclease pulsed-field gel electrophoresis combined with Southern
blotting. In the transconjugants, the blaKPC-2 genes were successfully
transferred to the recipient strain E. coli EC600. Whole-genome
sequencing and bioinformatics analysis confirmed that this strain
belongs to sequence type 196 (ST196), with a complete genome comprising
a 5,926,662bp circular chromosome and an 81,451bp IncM2 plasmid encoding
blaKPC-2 (designated pLS81-KPC). The IncM2 plasmid carried multiple
β-lactamase genes such as blaTEM-1B, blaCTX-M-3, and blaKPC-2 inserted
in truncated Tn6296 with the distinctive core structure
ISKpn27–blaKPC-2–ISKpn6. A comparison with 46 K. michiganensis genomes
available in the NCBI database revealed that the closest phylogenetic
relative of K. michiganensis LS81 is a clinical isolate from a wound
swab in the United Kingdom. Ultimately, the pan-genomic analysis
unveiled a substantial accessory genome within the strain, alongside
significant genomic plasticity within the K. michiganensis species,
emphasizing the necessity for continuous surveillance of this pathogen
in clinical environments.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1492700"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1492700</a></p>
<hr />
<h2
id="assessment-of-babesia-ovis-pathogenicity-in-goats-implications-for-transmission-dynamics-and-host-resistant">Assessment
of Babesia ovis pathogenicity in goats: implications for transmission
dynamics and host resistant</h2>
<p>date: 2024-11-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Babesia ovis, commonly associated with ovine babesiosis, poses a
significant threat to sheep health, often resulting in severe clinical
manifestations and high mortality rates. However, the impact of B. ovis
on goats has remained uncertain, prompting us to investigate its
pathogenicity in caprine hosts. Experimental infections using B.
ovis-infected blood inoculation and infected tick infestation, were
conducted on spleen-intact (n=5) and splenectomized (n=5) goats. The
experimental infection was performed using fresh blood obtained from a
B. ovis-infected splenectomized sheep. One spleen-intact sheep served as
a control for the experimental infection with B. ovis-infected
Rhipicephalus bursa ticks. While all experimentally infected sheep
(#501, #575) displayed severe clinical symptoms and high parasitemia,
goats exhibited resistance, showing no significant clinical
manifestations or sustained parasitemia. Notably, B. ovis was detected
in two spleen-intact goats via nested PCR, prompting further
investigation into their role as reservoirs for tick-borne transmission.
These goats were then infested with Babesia spp.-free R. bursa larvae
(0.1 gr) and adults (50 females and 50 males) for transstadial and
transovarial transmission experiments respectively. Results indicated
that chronically B. ovis-infected spleen-intact goats are not
significant sources for maintaining the tick-borne transmission cycle of
the parasite. These findings highlight the differential susceptibility
of goats to B. ovis infection compared to sheep and their limited role
as reservoirs for parasite transmission. Understanding the role of goats
in B. ovis transmission and their resistance mechanisms can inform
effective control measures and reduce economic losses in affected
regions. Further research into caprine babesiosis and host immunological
responses is essential to fully elucidate their possible role as
reservoirs of the parasite, and underlying mechanisms of host
susceptibility and parasite pathogenesis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1480347"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1480347</a></p>
<hr />
<h2
id="advances-in-the-study-of-oral-microbiota-and-metabolism-associated-fatty-liver-disease-a-systematic-review">Advances
in the study of oral microbiota and metabolism associated fatty liver
disease: a systematic review</h2>
<p>date: 2024-11-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe oral microbiota is the second largest microbiota in the
human body and has a significant impact on human health. Recent evidence
suggests that dysbiosis of the oral microbiota may be associated with
the development of metabolism-associated fatty liver disease (MAFLD).
This review aimed to validate the relationship between oral microbial
diversity and the development of MAFLD.MethodsA systematic evaluation
was performed based on PRISMA guidelines. Three independent reviewers
searched for relevant literature in several databases, including
PubMed/Medline, Web of Science, and Scopus, with a search date ranging
from the establishment of the databases to June 2024.ResultsA total of
1278 publications were initially screened, including five
cross-sectional studies, seven case-control studies, one cohort study,
and one retrospective study. These studies included a total of 3335
patients with MAFLD, 254 patients with MASH, and 105 patients with liver
cirrhosis. All 14 included studies concluded that there was a
correlation or potential correlation between oral microbiota and MAFLD.
Seven studies found that the composition of the oral microbiota in MAFLD
patients differed from that of healthy controls, and specific oral
bacteria may be associated with an increased incidence of MAFLD. At the
phylum level, several studies found differences in the abundance of the
phyla Firmicutes, Proteobacteria, and Clostridia compared to healthy
controls. Additionally, a study on oral fungi found significant
differences in the phyla Proteobacteria and in the genus Staphylococcus
between patients with MAFLD and healthy controls. At the genus level,
Porphyromonas was studied most frequently, with all 8 studies
identifying infection with Porphyromonas as a significant risk factor
for pathological progression in MAFLD. Furthermore, a dysbiosis in the
ratio of Porphyromonas gingivalis./Porphyromonas anomalies may be an
important marker of MAFLD progression.ConclusionThere is an important
association between the diversity of oral microbiota composition and
MAFLD. This finding suggests the importance of oral health assessment
and monitoring for the prevention or intervention of MAFLD.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1491696"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1491696</a></p>
<hr />
<h2
id="koala-ocular-disease-grades-are-defined-by-chlamydial-load-changes-and-increases-in-th2-immune-responses">Koala
ocular disease grades are defined by chlamydial load changes and
increases in Th2 immune responses</h2>
<p>date: 2024-11-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThis study employs bulk RNA sequencing, PCR, and ELISA
assays to analyze the pathological factors affecting the outcomes of C.
pecorum ocular infections in koalas. It investigates the immune
responses and gene expression profiles associated with various stages of
koala ocular chlamydiosis.MethodsA cohort of 114 koalas from Queensland,
Australia were assessed, with 47% displaying clinical signs of ocular
disease. Animals were classified into three cohorts: acute active
disease (G1), chronic active disease (G2), and chronic inactive disease
(G3), along with subclinical Chlamydia pecorum positive (H2) and healthy
(H1) cohorts.ResultsAnalysis of clinical, microbiological, humoral
immune and cellular immune biomarkers revealed varying chlamydial loads
and anti-chlamydial IgG levels across disease grades, with a negative
correlation observed between ocular chlamydial load and anti-chlamydial
IgG. Koala ocular mucosa gene expression analysis from 27 koalas
identified shared expression pathways across disease cohorts, with a
significant upregulation of IFNγ expression and tryptophan metabolism in
all disease stages.DiscussionThese findings help elucidate immune
response dynamics and molecular pathways underlying koala ocular
chlamydiosis, providing insights crucial for disease management
strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1447119"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1447119</a></p>
<hr />
<h2
id="intestinal-microbiome-changes-and-mechanisms-of-maintenance-hemodialysis-patients-with-constipation">Intestinal
microbiome changes and mechanisms of maintenance hemodialysis patients
with constipation</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundConstipation is a common symptom in maintenance
hemodialysis patients and greatly affects the quality of survival of
hemodialysis patients. Fecal microbiota transplantation and probiotics
are feasible treatments for functional constipation, but there is still
a gap in the research on the characteristics of gut flora in patients
with maintenance hemodialysis combined with constipation. The aim of
this study is to clarify the characteristics of the intestinal flora and
its changes in maintenance hemodialysis patients with
constipation.MethodsFecal samples were collected from 45 participants,
containing 15 in the maintenance hemodialysis constipation group,15 in
the maintenance hemodialysis non-constipation group and 15 in the
healthy control group. These samples were analyzed using 16S rRNA gene
sequencing. The feature of the intestinal microbiome of maintenance
hemodialysis constipation group and the microbiome differences among the
three groups were elucidated by species annotation analysis, α-diversity
analysis, β-diversity analysis, species difference analysis, and
predictive functional analysis.ResultsThe alpha diversity analysis
indicated that maintenance hemodialysis constipation group was less
diverse and homogeneous than maintenance hemodialysis non-constipation
group and healthy control group. At the genus level, the top ten
dominant genera in maintenance hemodialysis constipation group patients
were Enterococcus, Escherichia-Shigella, Bacteroides, Streptococcus,
Bifidobacterium, Ruminococcus_gnavus_group,
Lachnospiraceae_unclassified, Faecalibacterium, Akkermansia and UCG-002.
Compared with non-constipation group, the Enterococcus,
Rhizobiales_unclassified, Filomicrobium, Eggerthella, Allobaculum,
Prevotella_7, Gordonibacter, Mitochondria_unclassified,
Lachnoanaerobaculum were significantly higher in constipation group
(p&lt;0.05). Compared with non-constipation group, the Kineothrix,
Rhodopirellula, Weissella were significantly lower in constipation group
(p&lt;0.05). The predictive functional analysis revealed that compared
with non-constipation group, constipation group was significantly
enriched in pathways associated with pyruate metabolism, flavonoid
biosynthesis.ConclusionsThis study describes for the first time the
intestinal microbiome characteristics of maintenance hemodialysis
patients with constipation. The results of this study suggest that there
is a difference in the intestinal flora between maintenance hemodialysis
patients with constipation and maintenance hemodialysis patients without
constipation.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1495364"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1495364</a></p>
<hr />
<h2
id="apigeninidin-chloride-disrupts-toxoplasma-gondii-mitochondrial-membrane-potential-and-induce-reactive-oxygen-species-and-metabolites-production">Apigeninidin
chloride disrupts Toxoplasma gondii Mitochondrial membrane potential and
induce reactive oxygen species and metabolites production</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionApigeninidin chloride (APi) is a form of
3-deoxyanthrocyanidins (3-DAs) abundantly produced by the red Sorghum
bicolor plant. It has been previously reported to be effective against
Toxoplasma gondii (T. gondii) tachyzoites grown in vitro with less
cytotoxic effect. However, its possible mechanism(s) of action has not
been elucidated. Biochemically, we discovered that APi induced high
reactive oxygen species (ROS) and mitochondria superoxide (MitoSOX)
productions in tachyzoites, leading to mitochondrial membrane potential
(MMP) disruption in vitro.MethodsTo confirm our biochemical results at
the molecular level, we performed a liquid chromatography-mass
spectrometry (LC-MS) analysis on APi-treated parasites to assess any
metabolite and lipid alterations often associated with high ROS/MitoSOX
production in cells. ResultsNoteworthy is that we detected several
important oxidative stress-induced metabolites such as hexanal,
aldehydes, methyl undeo10-enoate, butadiynyl phenyl ketone,
16-hydroxyhexadecanoic acid (16-OH, 16:0), 2-hydroxytricosanoic acid
(C23:0; O), 3-oxodecanosanoic acid (C22:1; O), 2-hydroxypropylsterate,
and furan fatty acids F6 (19FU-FA). DiscussionThese metabolites are
associated with lipid, protein, and nucleic acid disruptions. Using
atovaquone (Atov) as a control, we observed that it disrupted
intracellular tachyzoites’ mitochondrial membrane potential, increased
ROS and MitoSOX production, and altered metabolite and lipid production
similar to what was observed with our experimental compound APi.
Overall, our results indicated that APi targets T. gondii tachyzoite
growth through inducing oxidative stress, mitochondrial dysfunction, and
eventually parasite death.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368019"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368019</a></p>
<hr />
<h2
id="a-novel-rapid-ultrasensitive-diagnosis-platform-for-detecting-candida-albicans-using-restriction-endonucleasemediated-real-time-loop-mediated-isothermal-amplification">A
novel, rapid, ultrasensitive diagnosis platform for detecting Candida
albicans using restriction endonuclease‐mediated real-time loop-mediated
isothermal amplification</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCandida albicans (C. albicans, CA) is an essential
invasive fungus in clinical diagnosis. Although several detection
methods exist, none meet the need for early diagnosis. A rapid,
sensitive, and specific diagnostic tool is crucial for effective
prevention and control of C. albicans infections.MethodsThis study aimed
to develop a new, rapid, and ultrasensitive diagnostic tool for C.
albicans detection based on restriction endonuclease-mediated real-time
loop-mediated isothermal amplification (ERT-LAMP-CA). The ERT-LAMP-CA
technology combines LAMP amplification, restriction endonuclease
cleavage, and real-time fluorescence detection in a single reaction
tube, which can complete a diagnosis of C. albicans in a short time
(approximately 1 h).ResultsHerein, we developed the primer sequences
required for ERT-LAMP-CA based on the ITS2 gene of C. albicans and found
that ERT-LAMP-CA limit of detection was approximately 500 ag/μL genomic
DNA and can present negative results for non-C. albicans templates. We
tested sputum samples from 64 patients with suspected C. albicans
infections to validate ERT-LAMP-CA applicability in clinical sample
testing and found that ERT-LAMP-CA was consistent with multiplex
PCR-capillary electrophoresis.DiscussionIn conclusion, ERT-LAMP-CA is a
rapid, accurate, and sensitive assay with excellent potential for
clinical and basic laboratory diagnosis and an efficient screening
strategy.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1450199"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1450199</a></p>
<hr />
<h2
id="impact-of-endogenous-viral-elements-on-glioma-clinical-phenotypes-by-inducing-oct4-in-the-host">Impact
of endogenous viral elements on glioma clinical phenotypes by inducing
OCT4 in the host</h2>
<p>date: 2024-11-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionEndogenous viral elements (EVEs) are viral sequences
integrated within the host genome that can influence gene regulation and
tumor development. While EVEs have been implicated in cancer, their role
in regulating key transcription factors in glioblastoma (GBM) remains
underexplored. This study investigates the relationship between EVEs and
the activation of OCT4, a critical transcription factor in GBM
progression.MethodsWe utilized CancerHERVdb and HervD Atlas databases to
identify potential interactions between EVEs and key genes involved in
GBM. Data from 273 GBM patient samples in the TCGA database were
analyzed to examine the correlation between OCT4 expression and
mutations in glioma-related genes. Furthermore, glioblastoma stem cells
(GSCs) were assessed for the expression levels of OCT4 and SOX2, and
Pearson correlation analysis was performed.ResultsOur analysis revealed
that OCT4 is a pivotal gene activated by EVEs in GBM. OCT4 expression
was significantly correlated with mutations in key glioma-associated
genes. Higher OCT4 levels were associated with poorer patient prognosis,
higher tumor grades, and older age. Additionally, GSCs exhibited
elevated expression of both OCT4 and SOX2, with a positive correlation
observed between these two genes in GBM patients.DiscussionThis study
highlights the potential role of EVEs in driving GBM progression through
the activation of OCT4. The findings emphasize the importance of OCT4 in
GBM malignancy and suggest that targeting EVE-mediated pathways may
provide new therapeutic approaches for GBM treatment.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474492"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474492</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
