<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2025-02-01 07:06:27)</p>
<hr />
<h2
id="bts1-knockout-saccharomyces-cerevisiae-with-broad-spectrum-antimicrobial-activity-through-lactic-acid-accumulation">BTS1-knockout
Saccharomyces cerevisiae with broad-spectrum antimicrobial activity
through lactic acid accumulation</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Bacterial infections pose significant threats to human health, and
prudent antibiotic use remains a key strategy for disease treatment and
control. However, a global escalation of drug resistance among
pathogenic bacteria presents a formidable challenge. Probiotics have
emerged as a promising approach to combating pathogenic bacterial
infections. In this study, we investigated the antibacterial activity of
BTS1-knockout (BTS1-KO) Saccharomyces cerevisiae. Our findings
demonstrate its effective inhibition of pathogen growth as evidenced by
Minimum inhibitory concentration (MIC) assays, growth curves,
bacteriostatic spectrum analyses and co-culture experiments.
Additionally, it significantly impedes Escherichia coli and
Staphylococcus aureus biofilm formation. Moreover, BTS1-KO S. cerevisiae
exhibits low haemolytic activity, acid resistance, resistance to high
bile salt concentrations, high auto-aggregation capacity and high
co-aggregation capacities with pathogenic bacteria. Moreover, infected
larvae treated with BTS1-KO S. cerevisiae in Galleria mellonella-E. coli
(in vivo) and G. mellonella-S. aureus (in vivo) infection models showed
significantly prolonged survival times. Mechanistic investigations
revealed that BTS1-KO S. cerevisiae primarily produced lactic acid via
metabolism, thereby lowering the environmental pH and inhibiting
pathogenic bacterial growth. In summary, our study underscores the
probiotic potential of BTS1-KO S. cerevisiae, offering broad-spectrum
antibacterial activity in vitro and in vivo with low toxicity. This
highlights BTS1-KO S. cerevisiae as a promising probiotic candidate for
clinical prevention and control of bacterial infection.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1494149"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1494149</a></p>
<hr />
<h2
id="evaluating-passive-immunity-in-piglets-from-sows-vaccinated-with-a-pedv-s-protein-subunit-vaccine">Evaluating
passive immunity in piglets from sows vaccinated with a PEDV S protein
subunit vaccine</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Porcine epidemic diarrhea virus (PEDV) is a highly contagious virus
that causes severe diarrhea and high mortality in neonatal piglets.
Current control measures, such as inactivated and live-attenuated
vaccines, have limitations in providing complete protection. In this
study, we evaluate the immunogenicity and protective efficacy of a PEDV
S protein subunit vaccine compared to a traditional inactivated vaccine.
Piglets and Sows were immunized with either the subunit vaccine or an
inactivated vaccine, and serum samples were collected to assess IgG and
neutralizing antibody levels. Results demonstrated that the S protein
subunit vaccine induced significantly higher IgG and neutralizing
antibody levels in both piglets and sows compared to the inactivated
vaccine. Piglets born to immunized sows were challenged with a virulent
PEDV strain. Piglets from the subunit vaccine group exhibited lower
viral shedding, reduced clinical symptoms, and minimal intestinal
lesions. These findings suggest that the PEDV S protein subunit vaccine
provides enhanced immunity and protection against PEDV, making it a
promising candidate for preventing PEDV infections in swine.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1498610"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1498610</a></p>
<hr />
<h2
id="temporal-and-spatial-characterization-of-keratinocytes-supporting-orf-virus-replication">Temporal
and spatial characterization of keratinocytes supporting orf virus
replication</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Reflecting their tropism for keratinocytes, most poxviruses that
infect vertebrates replicate to high titers and cause pathology in the
skin. Keratinocytes, the main cells of the epidermis, are found in
different stages of a differentiation program that produces the critical
barrier against environmental damage. While systemic poxviruses
(e.g. smallpox virus, sheeppox virus) also infect other cell types, the
parapoxvirus orf virus (ORFV), which causes localized infections in
sheep and goats, has not been shown to replicate in cells other than
keratinocytes. Notably, ORFV infection only occurs after or concomitant
with epidermal damage and the subsequent healing response and shows
unexplained delayed virus replication in an uncharacterized keratinocyte
subpopulation. Using in situ hybridization, immunohistochemistry,
confocal microscopy, qPCR, and a full-thickness wound/infection model in
sheep, the natural host, we show that during an initial 2-day eclipse
phase viral transcription and viral DNA replication are not detected.
Between days 2 and 3 pi, viral transcription is first detected in
keratinocytes of the stratum granulosum and upper stratum spinosum in
the proliferative zone at the wound margin. These cells are positive for
cytokeratin 10, a suprabasal marker; cytokeratin 6, a protein induced
during early repair responses; stratum granulosum markers filaggrin and
loricrin; and negative for the nuclear proliferation marker Ki-67 and
cytokeratin 14, a basal cell marker. This marker profile suggests that
keratinocytes supportive of viral replication are engaged in advanced
keratinocyte differentiation rather than proliferation.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1486778"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1486778</a></p>
<hr />
<h2
id="targeted-next-generation-sequencing-for-antimicrobial-resistance-detection-in-ventilator-associated-pneumonia">Targeted
next-generation sequencing for antimicrobial resistance detection in
ventilator-associated pneumonia</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundVentilator-associated pneumonia (VAP) carries a high
mortality rate in the intensive care units (ICUs) due to its widespread
drug resistance. Traditional microbial cultures limited by speed and
sensitivity, are often unable to help clinicians make effective
diagnosis and treatment. Therefore, there is an urgent need for a rapid
and accurate test that can identify both pathogens and their
antimicrobial resistance (AMR) to improve the prognosis of patients with
VAP.Study designWe analyzed samples from ICU patients with suspected VAP
using both microbial tests and targeted next-generation sequencing
(tNGS), comparing the results of tNGS pathogen and AMR detection against
microbial culture and antimicrobial susceptibility testing
(AST).ResultsInvolving 199 patients with suspected VAP, tNGS showed a
sensitivity rate of 98.98% for pathogen identification. While the
sensitivity rate of microbial culture was just 66.84%. Additionally,
tNGS performed almost half the turnaround time of microbial culture
(1.66 days vs 3.00 days). For AMR, the overall consistency between AST
and tNGS was 79.31%. The great performance particularly exhibited for
Acinetobacter baumannii carbapenem-penicillin-cephamycin
resistance.ConclusiontNGS excels in identifying pathogens and AMR. Its
rapid workflow makes it ideal for managing critically ill patients,
enhancing treatment precision, and reducing antibiotic misuse.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1526087"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1526087</a></p>
<hr />
<h2
id="genomic-epidemiology-and-antimicrobial-resistance-of-morganella-clinical-isolates-between-2016-and-2023">Genomic
epidemiology and antimicrobial resistance of Morganella clinical
isolates between 2016 and 2023</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Morganella morganii is a Gram-negative, opportunistic pathogen that
is often associated with nosocomial infections. Here, the genomic
characteristics and antimicrobial resistance (AMR) of Morganella
clinical isolates between 2016 and 2023 were determined. A total of 218
clinical isolates were mainly identified from urinary tract (48.2%) and
respiratory tract (16.5%), with 105 isolates randomly selected for whole
genome sequencing. The highest rates of antibiotic resistance were
observed with SAM (68.3%), followed by CIP (39.9%), and SXT (37.2%).
Distance analysis suggested that the 105 newly sequenced isolates could
be divided into two groups: M. morganii subsp. morganii and M. morganii
subsp. sibonii. While, the average nucleotide identity between these
groups showed only 91.5-92.2% similarity, raising the possibility that
they may be distinct species. Phylogenomic analysis revealed that the
102 M. morganii isolates fell into six clades, with clades 4-6 making up
the majority. Core genome multi-locus sequence type analysis indicted
high genomic diversity among different hosts and relatively stability
(&lt; 10 SNPs accumulated over three years) within the same host.
Together with epidemiological data, isolates of four genetic clusters
could be possible nosocomial transmissions. The identified 80 AMR genes
belonged to 15 drug-related classes, with tet(B) gene being the most
prevalent, followed by sul1, catA2, and sul2 genes. This study provided
comprehensive genomic insights and AMR patterns of Morganella isolates
in China, highlighting the necessity for continuous monitoring through
whole genome sequencing.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1464736"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1464736</a></p>
<hr />
<h2
id="evaluation-of-a-new-human-immunodeficiency-virus-antigen-and-antibody-test-using-light-initiated-chemiluminescent-assay">Evaluation
of a new human immunodeficiency virus antigen and antibody test using
light-initiated chemiluminescent assay</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesThe goal of this study is to evaluate the analytical and
clinical performance of a new human immunodeficiency virus antigen and
antibody (HIV Ag/Ab) test using light-initiated chemiluminescent assay
(LiCA®) and compare it with the well-established Architect® HIV Ag/Ab
combo assay in a clinical setting.MethodsWe used banked samples and
national reference controls to identify the ability to detect HIV Ag/Ab
and different viral subtypes. Thirteen seroconversion panels were tested
to evaluate early detection of HIV. A total of 21,042 patient samples
were collected to compare the diagnostic performance of LiCA® with
Architect®. Screening-reactive results were confirmed by Western
blotting and nucleic acid testing.ResultsTotal imprecision was within
2.49%–6.56%. The C5–C95 interval was within −10.20%–7.67% away from C50.
The limit of detection for p24 antigen was &lt;1.00 IU/mL. Using
national reference panels and banked sample pools, LiCA® successfully
detected all negative and positive controls in line with the criteria,
and all HIV-positive specimens containing different viral subtypes. In
13 seroconversion panels, LiCA® detected reactive results on average
5.73 days (95% CI: 3.42–8.04) after the initial RNA test results were
confirmed positive, which was 1.27 days earlier (−3.75 to 1.21) compared
to Architect®. Paired comparisons in 21,042 clinical patient samples
demonstrated that LiCA® detected HIV Ag/Ab with a slightly better
performance in sensitivity (100.00% vs. 99.65%), specificity (99.85%
vs. 99.81%), negative predictive value (NPV, 100.00% vs. 99.99%), and
positive predictive value (PPV, 89.84% vs. 87.85%) than Architect®.
Total agreement between two assays was 99.67% with a kappa value of
0.89.ConclusionLiCA® HIV Ag/Ab is a precise and highly sensitive assay
for measuring HIV-1 p24 antigen and HIV-1/2 antibodies with
differentiated S/Co values of Ag/Ab. The assay is appropriate for use in
the clinical routine test for the early detection of HIV.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1474127"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1474127</a></p>
<hr />
<h2
id="transcriptome-analysis-reveals-the-molecular-mechanism-of-γ-linolenic-acid-eradicating-the-biofilm-of-vancomycin-resistant-enterococcus-faecium">Transcriptome
analysis reveals the molecular mechanism of γ-linolenic acid eradicating
the biofilm of vancomycin-resistant Enterococcus faecium</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionVancomycin-resistant Enterococcus faecium (VRE-fm)
biofilms pose a significant clinical challenge due to the limited
effectiveness of traditional antibiotics. This study investigates the
potential of γ-linolenic acid (GLA) as a novel antibiofilm
agent.MethodsTranscriptome analysis was performed on the V27 isolate,
comparing cells in mature biofilms treated with and without GLA. The
findings were further validated using qRT-PCR on six VRE-fm isolates and
two E. faecalis isolates.ResultsTranscriptome analysis revealed a
significant downregulation in the expression levels of genes associated
with biofilm formation, including fruA, fruB, sgrA, lpxtg-cwa, tfpp,
lafA, lafB, malP, fsrA, and fsrC’, while a significant upregulation was
observed in the expression of fsrBD. Validation by qRT-PCR in six VRE-fm
isolates confirmed the significant changes in the expression levels of
all genes except for lpxtg-cwa, with statistical significance. The
expression of bgsB and bgsA genes, which are the homologs of lafA and
lafB genes, along with the Fsr-regulated genes gelE and sprE in E.
faecalis, were also found to be downregulated by GLA. In addition, KEGG
analysis identified specific metabolic pathways that were significantly
downregulated by GLA.ConclusionGLA effectively targets multiple aspects
of biofilm formation in VRE-fm, including the downregulation of key
biofilm-related genes, the inhibition of quorum sensing systems, and the
modulation of metabolic pathways. GLA emerges as a promising candidate
for eradicating Enterococcus biofilms.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1525581"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1525581</a></p>
<hr />
<h2
id="a-novel-strategy-for-eradication-of-staphylococcal-biofilms-using-blood-clots">A
novel strategy for eradication of staphylococcal biofilms using blood
clots</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionInfections with coagulase negative staphylococcal species
(CoNS) are a major cause of mortality and morbidity in joint and heart
valve replacement procedures, largely due to biofilm formation. Cells
within biofilms have higher rates of antibiotic resistance than their
planktonic counterparts; consequently, novel mechanisms are needed to
combat these infections.MethodsTo enhance antibiotic delivery and
penetration, this innovative study involved treating CoNS biofilms with
murine blood clots impregnated with antibiotics. We then investigated
the impact of this treatment on biofilm density, metabolism, and
architecture.ResultsOur pilot study demonstrates that this method of
antibiotic delivery results in improved biofilm clearance, relative to
conventional exposure methods.DiscussionOur results demonstrate that
blood clot exposure has an intrinsic impact on biofilm density and
potentially reduces colonization, warrenting further investigation into
the mechanism.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1507486"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1507486</a></p>
<hr />
<h2
id="an-updated-overview-on-the-bacterial-phopphoq-two-component-signal-transduction-system">An
updated overview on the bacterial PhoP/PhoQ two-component signal
transduction system</h2>
<p>date: 2025-01-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The PhoP response regulator and the cognate sensor kinase PhoQ form
one of the two-component signal transduction systems that is highly
conserved in bacteria. The PhoP/PhoQ system is a crucial mediator of
signal transduction. It regulates the expression of bacterial
environmental tolerance genes, virulence factors, adhesion, and
invasion-related genes by sensing various environmental signals in the
host, including Mg2+, low pH, antimicrobial peptides, and osmotic
pressure. In this review, we describe the PhoP/PhoQ system-induced
signal composition and its feedback mechanism, and the abundance of PhoP
phosphorylation in the activated state directly or indirectly controls
the transcription and expression of related genes, regulating bacterial
stability. Then, we discuss the relationship between the PhoP/PhoQ
system and other components of the TCS system. Under the same induction
conditions, their interaction relationship determines whether bacteria
can quickly restore their homeostasis and exert virulence effects.
Finally, we investigate the coordinated role of the PhoP/PhoQ system in
acquiring pathogenic virulence.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1509037"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1509037</a></p>
<hr />
<h2
id="distribution-of-sasx-mupa-and-qacab-genes-and-determination-of-genetic-relatedness-of-epidemic-methicillin-resistant-staphylococcus-aureus-strains-associated-with-bloodstream-infections-in-southern-china">Distribution
of sasX, mupA, and qacA/B genes and determination of genetic relatedness
of epidemic methicillin-resistant Staphylococcus aureus strains
associated with bloodstream infections in southern China</h2>
<p>date: 2025-01-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMethicillin-resistant Staphylococcus aureus (MRSA)
exhibits diverse genotypes with varying virulence and resistance
profiles, particularly in the context of bloodstream infections (BSI).
This study investigates the prevalence of the sasX, mupA, and qacA/B
genes among MRSA isolates from bloodstream infections in southern China
and analyzes their genetic relatedness.MethodsA polymerase chain
reaction (PCR) assay was developed to detect the presence of the sasX
gene, which is associated with nasal colonization, immune evasion, and
virulence, the mupirocin resistance gene mupA, and the chlorhexidine
tolerance gene qacA/B in a total of 77 MRSA isolates. Multilocus
sequence typing (MLST) was performed to determine the sequence types
(STs) and assess the genetic relatedness of the isolates. The resistance
of these strains to 16 antibiotics was also analyzed. The distribution
of these genes and their association with epidemic STs were
analyzed.ResultsA total of 26 STs were identified, with notable
prevalence in five epidemic clones: ST59, ST5, and ST764. The prevalence
of the sasX, mupA, and qacA/B genes across all isolates was 23.4%,
33.8%, and 79.2%, respectively. Specifically, the frequency of the sasX
gene was highest in ST59 (29.4%), ST239 (100%), and ST764 (37.5%); mupA
was most prevalent in ST5 (66.7%), ST59 (17.6%), ST764 (37.5%), and ST15
(100%); qacA/B was predominantly found in ST59 (88.2%), ST5 (66.7%),
ST398 (85.7%), ST764 (50.0%), and ST239 (100%). The gene distribution
patterns revealed that sasX+ qacA/B+ mupA+ strains were closely
associated with epidemic clones ST6290 and ST88, whereas sasX+ qacA/B+
mupA- strains were linked to ST59, ST239, and ST764. DiscussionNotably,
forty-seven (61%) MRSA BSI strains were multidrug-resistant, with the
majority exhibiting resistance to penicillin, erythromycin, and
clindamycin. Major MRSA clones in southern China include ST59, ST5,
ST764, and ST398. In this study, sasX, mupA and qacA/B genes were
present in the MRSA isolates, with the mupA gene being the most
prevalent. Variations in the prevalence of virulence and resistance
genes among these epidemic strains underscore the need for targeted
infection control measures. These findings contribute to a better
understanding of the genetic epidemiology of MRSA in the region,
facilitating the development of effective prevention and control
strategies for BSI.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1491658"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1491658</a></p>
<hr />
<h2
id="characterization-of-serum-metabolome-and-respiratory-microbiota-in-children-with-influenza-a-virus-infection">Characterization
of serum metabolome and respiratory microbiota in children with
influenza A virus infection</h2>
<p>date: 2025-01-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The risk of children being infected with Influenza A virus (IAV) is
high, and if not treated promptly, it can lead to serious illness.
Compared with control group, IAV infection decreased the contents of
platelet, white blood cell, lymphocyte, eosinophil, basophil, CD3+ T
cells, CD4+ T cells, CD8+ T cells, and B cells, while increasing the
number of red blood cell. Additionally, IAV infection increased serum
concentrations of total protein, albumin and lipase, while decreasing
the contents of calcium, triglyceride, total bilirubin, direct
bilirubin, indirect bilirubin and gamma-glutamyltransferase. However,
the interactions between the respiratory microbiome and metabolites and
their impact on IAV in children remains unclear. Ultra performance
liquid chromatography quadrupole time of flight mass spectrometry
(UPLC-QTOF/MS) and 16S rRNA gene sequencing were employed to analysis
the respiratory microbiome and serum metabolic characteristics of 85
patients with IAV infection and age-matched 55 controls with respiratory
disease who tested negative for 13 types of respiratory pathogens. The
serum metabolic profile of IAV patients was significantly changed, and
the purine metabolism was destroyed. Purine metabolism was also enriched
in H3N2 patients compared to H1N1, with increased xanthine,
deoxyguanosine, and inosine. The respiratory microbiome structure in
children with IAV, including H1N1 and H3N2, was significantly different
from that of the control, with significantly increased Chao index. The
Mantel test revealed the correlation and consistency in the trends of
Haemophilus, Ureaplasma and Inosine. This study revealed the
characteristics of the respiratory microbiome and serum metabolites in
pediatric patients with IAV, providing a new direction for exploring the
pathogenesis of IAV in children.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478876"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478876</a></p>
<hr />
<h2
id="acinetobacter-baumannii-infection-in-critically-ill-patients-with-covid-19-from-tehran-iran-the-prevalence-antimicrobial-resistance-patterns-and-molecular-characteristics-of-isolates">Acinetobacter
baumannii infection in critically ill patients with COVID-19 from
Tehran, Iran: the prevalence, antimicrobial resistance patterns and
molecular characteristics of isolates</h2>
<p>date: 2025-01-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe COVID-19 pandemic has led to the excessive use of
antimicrobials in critically ill patients. Infections caused by
Acinetobacter baumannii have increased significantly both regionally and
globally during the COVID-19 pandemic, posing dramatic challenges for
intensive care unit (ICU) patients. This study aimed to determine the
prevalence, antimicrobial resistance patterns, presence of selected
antimicrobial resistance genes, and genetic diversity of A. baumannii
isolates obtained from COVID-19 cases admitted to the ICU at the
University Hospital in Iran.Materials and methodsThis was a
cross-sectional and single-center study comprising patients with A.
baumannii infections admitted to the ICU with COVID-19 between April and
November 2021. The demographic and clinical data of the patients were
collected. Antimicrobial susceptibility testing was conducted based on
Clinical Laboratory Standards Institute guidelines. This study used PCR
and multiplex PCR to investigate antibiotic resistance genes (ARGs) and
global clones (GC), respectively. Genetic diversity was investigated by
repetitive element sequence-based PCR (REP-PCR).ResultsThe prevalence of
A. baumannii coinfection in COVID-19 cases was 8.1% (43/528). More than
90% (39/43) of A. baumannii isolates were resistant to cefepime,
ampicillin-sulbactam, gentamicin, trimethoprim-sulfamethoxazole and
amikacin. Furthermore, 44.2% (19/43) of isolates were resistant to
colistin. There were 91% (39/43) isolates that were extensively
drug-resistant (XDR). The most prevalence carbapenem resistance encoding
genes were bla-OXA-23 65.1% (29/43) and blaNDM 41.8% (18/43). The most
common aminoglycoside resistance genes were aac(6’)-Ib 65.1% (28/43) and
ant(2)-Ia 46.5% (20/43). Isolates from the prominent Global clone GCII
comprised 83.7% (36/43) of total isolates. Genetic fingerprinting using
REP-PCR revealed that 39 typeable A. baumannii isolates were categorized
into 12 distinct genotypes, of which 72% (28/39) of isolates belonged to
one genotype.ConclusionThe high prevalence of XDR A. baumannii such as
carbapenem and colistin-resistant strains, poses a significant concern
for the treatment of COVID-19 patients, heightening the risk of
therapeutic failure. The data demonstrate the dissemination of a single
A. baumannii clone carrying multiple ARGs within our hospital. Regarding
the limited therapeutic options, it is crucial to implement effective
prevention and containment policies to curb the spread of these
strains.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1511122"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1511122</a></p>
<hr />
<h2
id="retrospective-analysis-of-300-microbial-cell-free-dna-sequencing-results-in-routine-blood-stream-infection-diagnostics">Retrospective
analysis of 300 microbial cell-free DNA sequencing results in routine
blood stream infection diagnostics</h2>
<p>date: 2025-01-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionBloodstream infections are a critical challenge worldwide
due to the slow turnaround time of conventional microbiological tests
for detecting bacteremia in septic patients. Noscendo GmbH (Duisburg,
Germany) has developed the CE/IVD pipeline DISQVER for clinical
metagenomics testing based on cell-free DNA (cfDNA) from blood samples
to address this issue.MethodsWe conducted a retrospective study to
evaluate the diagnostic utility of this methodological setup in
improving treatment decisions since it was introduced into our clinical
setting. Between January 2021 and June 2022, the first 300 cases in
which DISQVER was applied at our university hospital were collected and
analyzed. The results were compared with routine microbiology test
results, clinical picture, associated treatment decisions, and clinical
course.ResultsOur findings demonstrate that DISQVER results where no
pathogen was reported effectively ruled out bacterial bloodstream
infections, whereas positive results varied in their usefulness. While
the metagenomic approach proved highly valuable for detecting
non-culturable and rare pathogens, its utility was limited in cases
where detected microorganisms were commonly associated with the
microbiota.DiscussionPerforming on-site analysis might mitigate delays
resulting from logistical challenges and might help optimizing
antibiotic stewardship. Once prompt results can be obtained, the
relevance of incorporating molecular resistograms will become more
pronounced. Further, the specific patient subgroups that most benefit
from this analysis must be worked out. Guiding clinicians in identifying
the infection focus based on the detected bacteria would significantly
improve patient care. Lastly, evidence of filamentous fungi must be
diligently followed up.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1504262"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1504262</a></p>
<hr />
<h2
id="challenges-and-limitations-in-using-bacterial-metabolites-as-immunomodulators">Challenges
and limitations in using bacterial metabolites as immunomodulators</h2>
<p>date: 2025-01-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Harnessing the immunomodulatory potential of bacterial metabolites
opens up exciting possibilities for treating various immune-related
disorders. However, turning this potential into a reality presents
significant challenges. This review investigates these challenges,
focusing on discovery, production, characterization, stability,
formulation, safety, and individual variability limitations. The limited
bioavailability of many metabolites, as well as potential improvements
along with the potential for off-target effects and the importance of
precise targeting, are emphasized. Furthermore, the complex interactions
between gut bacterial metabolites and the microbiome are investigated,
highlighting the importance of personalized approaches. We conclude by
discussing promising advances in metagenomics, metabolomics, synthetic
biology, and targeted delivery systems, which hold out hope for
overcoming these limitations and paving the way for the clinical
translation of bacterial metabolites as effective immunomodulators.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535394"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535394</a></p>
<hr />
<h2
id="editorial-the-role-of-gut-microbiota-in-animal-gastrointestinal-diseases">Editorial:
The role of gut microbiota in animal gastrointestinal diseases</h2>
<p>date: 2025-01-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1554277"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1554277</a></p>
<hr />
<h2
id="a-comprehensive-analysis-of-the-uterine-microbiome-in-endometrial-cancer-patients---identification-of-anaerococcus-as-a-potential-biomarker-and-carcinogenic-cofactor">A
comprehensive analysis of the uterine microbiome in endometrial cancer
patients - identification of Anaerococcus as a potential biomarker and
carcinogenic cofactor</h2>
<p>date: 2025-01-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionEndometrial cancer (EC) is a significant gynecological
malignancy with increasing incidence worldwide. Emerging evidence
highlights the role of the uterine microbiome in the pathogenesis of EC.
This study aims to characterize the uterine microbiome in EC patients
and identify potential microbial biomarkers, with a focus on
Anaerococcus as a differentiating taxon.MethodsThe endocervical canal
swabs from patients with EC (n=16) and non-cancerous patients (EM, n=13)
were collected. The V3-V4 region of the 16S rRNA gene was sequenced
using the Illumina platform. Bioinformatic analyses were performed with
QIIME2, and statistical comparisons were conducted to assess differences
in microbial composition and diversity. In vitro experiments were
conducted to assess the functional impact of Anaerococcus on human
uterine fibroblasts, including its ability to adhere to the human cells
and induce oxidative stress.ResultsThe α-diversity metrics, including
Shannon entropy and observed amplicon sequence variants (ASVs), revealed
significantly higher microbial diversity in EC samples compared to EM.
Anaerococcus was identified as a key taxon differentiating EC from EM
groups, showing a higher relative abundance in EC samples. Functional
predictions and in vitro assays indicated that Anaerococcus may
contribute to carcinogenesis by inducing reactive oxygen species (ROS)
production, and has the high ability to adhere to the human endometrial
fibroblasts.DiscussionThe study provides evidence of distinct microbial
signatures in EC, with Anaerococcus emerging as a potential biomarker.
The in vitro findings suggest its role in endometrial carcinogenesis,
underscoring its potential as a target for future diagnostic and
therapeutic applications.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1511625"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1511625</a></p>
<hr />
<h2
id="cryptic-transmission-of-a-sars-cov-2-variant-detected-by-wastewater-surveillance-in-panama">Cryptic
transmission of a SARS-CoV-2 variant detected by wastewater surveillance
in Panama</h2>
<p>date: 2025-01-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The COVID-19 pandemic highlighted the critical role of viral genomic
surveillance, prompting numerous countries to enhance their monitoring
systems for acute respiratory infections (ARIs), especially
influenza-like illnesses (ILIs). Given the significance of asymptomatic
cases in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)
transmission, cases often undetected by the ILI surveillance, a more
comprehensive approach was essential to track the circulation of
SARS-CoV-2 variants in the population. In response, many countries
swiftly adopted wastewater surveillance, which allowed the early
detection of SARS-CoV-2 variants before they were identified through
molecular characterization from confirmed clinical cases. In this
report, we detail the implementation of SARS-CoV-2 wastewater genomic
surveillance in Panama during the first half of 2024. Wastewater samples
were collected monthly in duplicate at two collection points from three
districts of Panama city metropolitan area for testing by SARS-CoV-2
RT-qPCR, and positive samples were analyzed by next-generation
sequencing to identify sublineages. A total of 36 wastewater samples and
822 samples obtained through the clinical surveillance were analyzed for
molecular detection and sequencing. Sublineages detected by wastewater
surveillance were compared to those detected by clinical surveillance
for the same period of time. Wastewater surveillance allowed the
identification of the Omicron sublineage JN.1.16.1 in the capital city
and its surroundings, which was not detected by the clinical
surveillance in the country, despite its global circulation. This
highlights the critical need to sustain both genomic surveillance
programs beyond the pandemic in countries like Panama that serve as
pivotal exchange hubs.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1467484"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1467484</a></p>
<hr />
<h2
id="corynebacteria-from-the-respiratory-microbiota-modulate-inflammatory-responses-and-associate-with-a-reduced-pneumococcal-burden-in-the-lungs">Corynebacteria
from the respiratory microbiota modulate inflammatory responses and
associate with a reduced pneumococcal burden in the lungs</h2>
<p>date: 2025-01-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCertain species from the normal respiratory tract
microbiota have recently been proposed to positively influence human
health. Corynebacterium propinquum and C. pseudodiphtheriticum
(Corynebacteria) are two Gram-positive species that frequently colonize
the upper respiratory tract and strongly associate with a reduced
incidence of respiratory tract infections. The specific role of
Corynebacteria during respiratory health and disease is, however,
largely uncharacterized.MethodRespiratory tract epithelial cells
NCI-H292 and BALB/cByJ mice were inoculated with Corynebacteria (C.
propinquum 2018M3 and 2019M4, and C. pseudodiphtheriticum 2019M8 and
2020M12) alone or with subsequent challenge with Streptococcus
pneumoniae (pneumococci). The inflammatory response and the bacterial
burden of both species over time were determined by Western blot,
luciferase assay, cytokine bead array, flow cytometry and viable plate
counts on blood agar plates.ResultsClinical isolates of Corynebacteria
were well tolerated by human cells and mice. Corynebacteria induced a
transient inflammatory response during healthy conditions in the absence
of known pathogens. Pre-exposure or nasal priming with Corynebacteria
did not affect subsequent acquisition of pneumococci but were associated
with a modulated inflammatory response in vitro and in vivo as well as
with a reduced pneumococcal burden in the respiratory tract of mice.
This indicates that the presence of C. propinquum or C.
pseudodiphtheriticum may protect against severe pneumococcal
infections.ConclusionsIn this study, we delineate the role of
Corynebacteria from the normal microbiota that epidemiologically
associate with respiratory health. We show that the presence of
Corynebacteria modulates the inflammatory response to pneumococci and
associate with faster decrease in pneumococcal burden, primarily in the
lower respiratory tract. Our data indicate that Corynebacteria has
potential to protect against severe pneumococcal infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1530178"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1530178</a></p>
<hr />
<h2
id="development-of-a-one-step-multiplex-rt-qpcr-method-for-rapid-detection-of-bovine-diarrhea-viruses">Development
of a one-step multiplex RT-qPCR method for rapid detection of bovine
diarrhea viruses</h2>
<p>date: 2025-01-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionViral calf diarrhea poses a significant challenge to the
cattle industry worldwide due to its high morbidity and mortality rates,
leading to substantial economic losses. The clinical symptoms associated
with various diarrhea pathogens often overlap, complicating accurate
diagnosis; thus, there is an urgent need for rapid and precise
diagnostic methods to improve prevention and treatment efforts. In this
study, we developed a one-step multiplex reverse-transcription
quantitative real-time polymerase chain reaction (mRT-qPCR) that enables
the simultaneous detection of three key viral pathogens responsible for
calf diarrhea: bovine kobuvirus (BKoV), bovine astrovirus (BoAstV), and
bovine torovirus (BToV). However, development of accurate and rapid
methods to distinguish these three viruses is helpful for the early
detection, disease surveillance, and control of viral calf
diarrhea.MethodsSpecific primers and minor groove binder (MGB)-based
probes were designed targeting the 3D region of BKoV, ORF1 region of
BoAstV, and N region of BToV. The sensitivity, specificity, and
reproducibility ability were evaluated for the mRT-qPCR. Further, 80
bovine fecal samples were subjected to the mRT-qPCR, and the results
were verified using conventional reverse-transcription PCR (RT-PCR) or
PCR methods and sequencing methods.ResultsThis novel method demonstrated
high sensitivity and specificity,achieving a detection limit of 24
copies/mL for each pathogen. Furthermore, the assay exhibited excellent
reproducibility, with coefficients of variation below 1.5%, a strong
linear correlation (R2 &gt; 0.996), and an amplification efficiency
between 90% and 110%. Validation with 80 clinical samples from both
diarrheic and non-diarrheic cattle across four farms in Shanghai showed
a high degree of concordance with RT-PCR, with positive detection rates
for BKoV, BoAstV, and BToV at 28.75%, 8.75%, and 3.75%, respectively,
highlighting the predominance of BKoV and BoAstV. Notably, this study
represents the first identification of BKoV, BoAstV, and BToV in the
Shanghai region.DiscussionThe mRT-qPCR is a robust, rapid, and simple
tool for identifying viral pathogens associated with calf diarrhea,
facilitating the development of effective prevention and control
measures that are vital for the future sustainability of the cattle
industry.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1540710"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1540710</a></p>
<hr />
<h2
id="microbiome-interplays-in-the-gut-liver-axis-implications-for-liver-cancer-pathogenesis-and-therapeutic-insights">Microbiome
interplays in the gut-liver axis: implications for liver cancer
pathogenesis and therapeutic insights</h2>
<p>date: 2025-01-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Globally, primary liver cancer (PLC) ranks the most fatal malignancy.
Most of the patients are in advanced stage of PLC at the very time they
are diagnosed with it, accounting much for its poor prognosis. With the
advancement of modern medical research and care system, the main
etiology of PLC more and more switches from hepatitis viruses such as
HAV, HBV, HCV, HEV to other causes like metabolism-associated
steatohepatitis (MASH) and metabolic-associated fatty liver disease
(MAFLD). As a result, it is of great necessity to find out new ways for
treatment and early diagnosis to cope with this problem. Nowadays, as
the mechanism of the Gut-Liver Axis in the formation of MAFLD, MASH and
PLC has been gradually elucidated. The association between gut
microbiome and the formation of PLC is of great significance to take an
insight into. In this review, we present the concept of Gut-Liver Axis
and its function in the mutual influence between gut microbiota and PLC
from several aspects in which we will focus on the structure of gut
barrier and the functional influences the gut microbiota have on the
immune response and metabolic changes on human liver. Furthermore, we
conclude the potential association of gut microbiota constitution with
the PLC. Eventually, we hope this review can offer novel instructions
for early diagnosis and treatment for liver cancer.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1467197"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1467197</a></p>
<hr />
<h2
id="detection-and-characterization-of-bovine-hepacivirus-in-cattle-and-sheep-from-hulunbuir-northeastern-china">Detection
and characterization of bovine hepacivirus in cattle and sheep from
Hulunbuir, northeastern China</h2>
<p>date: 2025-01-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundBovine hepacivirus (BovHepV) is a recently identified
member of the expanding genus Hepacivirus within the family
Flaviviridae. However, the genetic diversity, geographical distribution,
and host range of the virus remains poorly understood.MethodsIn this
study, serum samples from cattle and sheep were obtained in Hulunbuir
and pooled to establish RNA libraries, which were then analyzed using
transcriptome sequencing. BovHepV-positive samples were confirmed using
semi-nested PCR with primers designed based on the obtained viral
sequences. Comprehensive bioinformatics analyses were employed to assess
sequence similarity, phylogenetic evolution, and recombination of the
obtained viral strains.ResultsA total of 988 serum samples from sheep
(520) and cattle (468) were collected from 12 administrative districts
in Hulunbuir from June to August, 2022. Semi-nested PCR revealed 6
BovHepV-positive districts with prevalence ranging from 2.0% to 35.0% in
cattle, and one BovHepV-positive district with prevalence of 2.5% in
sheep. The nucleotide sequence identities between viral strains from
sheep and cattle ranged from 91.3% to 93.8%, while the amino acid
sequence identities were between 95.4% and 96.7%. Phylogenetic analyses
classified the obtained BovHepV strains within subtype G, genotype 1.
Recombination analysis revealed the intergenerational relationships
among the viral strains obtained from cattle and sheep.ConclusionWe
identified genetic diversity in subtype G strains in cattle and detected
a BovHepV strain in a sheep for the first time in northeastern China,
confirming cross-species transmission and co-circulation between cattle
and sheep, thus expanding the virus’s host range.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540849"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540849</a></p>
<hr />
<h2
id="saussurea-costus-alleviates-ulcerative-colitis-by-regulating-the-gut-microbiota-and-improving-intestinal-barrier-integrity">Saussurea
costus alleviates ulcerative colitis by regulating the gut microbiota
and improving intestinal barrier integrity</h2>
<p>date: 2025-01-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe global health challenge of ulcerative colitis (UC)
has been classified by the WHO as a modern refractory disease, commonly
referred to as green cancer, with limited treatment options still
available, highlighting the urgent need for the development of new
therapeutic strategies. Recent pharmacological research has shown that
traditional Chinese medicine saussurea costus (SC) possesses beneficial
antibacterial and anti-inflammatory properties. Nevertheless, its
underlying mechanism remains elusive.MethodsFirstly, we identified the
main active components of SC through UHPLC-QTOF-MS analysis.
Subsequently, UC mice were induced using DSS and administered different
doses of SC to evaluate its efficacy. Additionally, the impact of SC on
the repair of the intestinal mucosal barrier was evaluated through
immunofluorescence and western blot. Furthermore, 16s rRNA gene
sequencing was conducted to elucidate the contribution of gut microbiota
to UC pathogenesis.ResultsThe primary components of SC include Proline,
Phenylalanine, Isoleucine, Lucidenic acid M, and Pyroglutamic acid. The
efficacy of SC was concurrently assessed, revealing its potential to
ameliorate histological injury in colitis mice. Furthermore, SC was
found to decrease levels of TNF-α, IL-1β, IL-8, and IL-18 while
promoting the expression of IL-10 and IL-22. Similarly, we also found
that the expression of ZO-1 and Occludin was reversed by SC in colitis
mice. In addition, analysis of 16S rRNA gene sequencing indicated that
SC reduced harmful bacterial populations, such as Proteobacteria, while
simultaneously enhancing the levels of beneficial bacteria like
Lactobacillus, thereby contributing to the improvement of UC
pathology.ConclusionThis study highlights the therapeutic potential of
SC in managing UC through its ability to attenuate inflammatory
responses, restore intestinal barrier functionality, and modulate gut
microbiota composition, which findings offer insights into potential
strategies for advancing UC treatment.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1528578"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1528578</a></p>
<hr />
<h2
id="transcriptional-profiles-analysis-of-effects-of-toxoplasma-gondii-rhoptry-protein-16-on-thp-1-macrophages">Transcriptional
profiles analysis of effects of Toxoplasma gondii rhoptry protein 16 on
THP-1 macrophages</h2>
<p>date: 2025-01-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionToxoplasma gondii, an intracellular parasitic protozoan,
is globally recognized for its ability to cause parasitic diseases and
has developed diverse strategies to evade immune-mediated elimination.
The protein ROP16 of T.gondii plays a crucial role in this evasion
process by specifically targeting macrophages and mononuclear phagocytes
in vivo. However, the precise mechanisms underlying the involvement of
type II ROP16 proteins in infection, inflammation, and other processes
remain unknown.MethodsTo investigate the mechanism of action of
gonococcal ROP16 proteins in human macrophages, we constructed a
lentivirus overexpressing ROP16 and established stably transfected cell
lines. We then analyzed the gene transcriptional profiles of ROP16 II in
THP-1 macrophages using transcriptome sequencing. Interaction networks
were constructed by screening differentially expressed genes and
performing gene function enrichment analysis.ResultsAs a result, five
differentially expressed genes were identified: AAMDC, GPR158, RAD9A,
STOML1, and STRA13. Immuno-featured differential analysis showed that
type 17 T helper cells were more strongly correlated with GPR158 and
STRA13, while CD8 T-cell was most strongly correlated with
STOML1.DiscussionTherefore, we conclude that the ROP16 protein plays a
pivotal role in THP-1 macrophage infection and these five differentially
expressed genes may serve as promising molecular targets for the
prevention or control of toxoplasmosis. These findings have significant
implications for the diagnosis and treatment of toxoplasmosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1436712"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1436712</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2025-01-27, from: People’s CDC</p>
<p>This is the People’s CDC bi-weekly update for January 27, 2025! This
Weather Report from the People’s CDC sheds light on the ongoing COVID
situation in the US.</p>
<p><br></p>
<p><a
href="https://peoplescdc.org/2025/01/27/peoples-cdc-covid-19-weather-report-90/"
class="uri">https://peoplescdc.org/2025/01/27/peoples-cdc-covid-19-weather-report-90/</a></p>
<hr />
<h2
id="gastrointestinal-protozoa-in-pet-cats-from-anhui-province-prevalence-and-molecular-characterization">Gastrointestinal
protozoa in pet cats from Anhui province: prevalence and molecular
characterization</h2>
<p>date: 2025-01-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionTo investigate the prevalence of Tritrichomonas foetus,
Pentatrichomonas hominis, Giardia intestinalis, Cryptosporidium,
Microsporidium, and Sarcocystis in domestic cats in Anhui Province,
China, and their potential role as zoonotic hosts for human infection, a
total of 304 fecal samples from two different sources were screened for
the presence of related pathogens.MethodsUsing microscopy, along with
polymerase chain reaction (PCR) or nested PCR amplification, followed by
genotyping through sequence analysis.ResultsThe infection rates of T.
foetus, P. hominis, G. intestinalis, Cryptosporidium, Enterocytozoon
bieneusi, and Sarcocystis were 5.6%, 0%, 1.7%, 0.7%, 2.6%, and 0%,
respectively. The evolutionary relationships and genetic characteristics
of G. intestinalis based on the GDH gene, Cryptosporidium based on the
SSU rRNA gene, and E. bieneusi based on the ITS sequence were assessed:
five cases of G. intestinalis were identified, with four belonging to
assemblage F and one to zoonotic assemblage B, two Cryptosporidium cases
were identified as Cryptosporidium felis, and all eight E. bieneusi
cases were identified as belonging to group 1, with three cases being
genotype D, three EbpA, and two EbpC.DiscussionAge, neutering status,
and deworming were identified as potential risk factors. Further
analysis revealed that diarrhea, as a clinical symptom, could serve as
an indicator for pathogen infection. Although the pathogen infection
rates detected in this study were relatively low, their zoonotic
transmission potential cannot be ignored. Therefore, special attention
should be paid, and it is essential to establish targeted prevention
plans.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1522176"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1522176</a></p>
<hr />
<h2
id="analysis-of-factors-affecting-the-clinical-management-of-infection-in-culture-negative-patients-following-percutaneous-endoscopic-decompression-a-retrospective-study">Analysis
of factors affecting the clinical management of infection in
culture-negative patients following percutaneous endoscopic
decompression: a retrospective study</h2>
<p>date: 2025-01-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCulture-negative spinal infections after prophylactic
antibiotic use in percutaneous endoscopic decompression are rare, and
diagnostic difficulties and the risk of antibiotic-resistant infections
complicate treatment. This study retrospectively analyzed the medical
records of culture-negative patients following percutaneous endoscopic
surgery to identify risk factors influencing antimicrobial therapy and
provide insights for clinical management.MethodsData were
retrospectively collected from patients who underwent lumbar
percutaneous endoscopic decompression at the Affiliated Hospital of
Qingdao University between January 2014 and June 2023. The patients’
medical records were reviewed. Patient demographics, hidden blood loss,
daily blood glucose control, and maximum temperature during treatment
were recorded as potential risk factors. C-reactive protein,
procalcitonin, white blood cells, erythrocyte sedimentation rate, and
the duration of antibiotic treatment were used as indicators of
infection treatment. The impact of these risk factors on infection was
then analyzed.ResultsThe results showed that blood glucose control was
strongly correlated with the severity of infection (Beta = 0.60, P =
0.00), strongly correlated with short-term treatment effectiveness (Beta
= 0.65, P = 0.00), and moderately correlated with the duration of
antibiotic treatment (Beta = 0.41, P = 0.01). Hidden blood loss was
moderately correlated with the severity of infection (Partial-R = 0.49,
P = 0.00) and moderately correlated with the duration of antibiotic
treatment (Partial-R = 0.48, P = 0.00). Hidden blood loss index was
moderately correlated with the duration of antibiotic treatment
(Partial-R = 0.50, P = 0.00). Female was a favorable factor to shorten
the duration of antibiotic treatment (Beta = -0.25, P = 0.01), and
higher maximum temperature during infection may indicate a longer
duration of antibiotic treatment (Beta = 0.28, P = 0.02).ConclusionOur
findings suggest that healthy blood glucose levels, a lower hidden blood
loss and hidden blood loss index might help reduce the duration of
antibiotic use after infection. Effective hemostasis during surgery to
reduce hidden blood loss and good preoperative blood glucose control
indicators are both beneficial measures for infection treatment.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540970"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540970</a></p>
<hr />
<h2
id="the-secreted-host-cell-protein-clusterin-interacts-with-pmpd-and-promotes-chlamydia-trachomatis-infection">The
secreted host-cell protein clusterin interacts with PmpD and promotes
Chlamydia trachomatis infection</h2>
<p>date: 2025-01-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Attachment and uptake into host cells are pivotal steps in the life
cycle of the Chlamydiaceae, a family of obligate intracellular
pathogens. Chlamydia trachomatis (Ctr) possesses a family of nine
polymorphic membrane proteins (Pmps), which have been shown to be
crucial for adhesion and internalization. However, the host-cell
molecules involved have so far remained unknown. Here, we show that a
fragment of Ctr PmpD, which forms high-molecular-weight oligomers in
solution and adheres to epithelial cells, also binds to secreted
clusterin (sCLU), a chaperone-like protein that is secreted into the
extracellular space by the host cell, and forms part of the chaperone-
and receptor-mediated extracellular protein degradation (CRED) pathway.
Using in vitro assays, we demonstrate that sCLU interacts directly with
soluble rPmpD. In infection experiments, depletion of sCLU from the
culture medium leads to a significant decrease in Ctr infection. Thus,
sCLU is the first host-cell interaction partner identified for a Ctr Pmp
and the first case in which sCLU has been shown to be a vital component
for the establishment of a bacterial infection.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1519883"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1519883</a></p>
<hr />
<h2
id="genome-drafting-of-nosocomial-infection-cre-klebsiella-pneumoniae-confirming-resistance-to-colistin-and-eravacycline-carrying-blandm-1-mcr-1-and-blakpc-2-in-neonatology-from-november-to-december-2023">Genome
drafting of nosocomial infection CRE Klebsiella pneumoniae confirming
resistance to colistin and eravacycline, carrying blaNDM-1, mcr-1, and
blaKPC-2, in neonatology from November to December 2023</h2>
<p>date: 2025-01-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCarbapenem-resistant Klebsiella pneumoniae (CRKP) is a
critical pathogen in healthcare settings, associated with high mortality
due to its extensive antibiotic resistance. In this study, we report an
outbreak of CRKP in a neonatal intensive care unit (NICU) within a
200-bed tertiary hospital. The main goal of this study was to
characterize the phenotypic and genomic profiles of the CRKP isolates
involved in the outbreak and to gain insights into their resistance
mechanisms and transmission dynamics within the NICU.MethodsThe study
was conducted between November and December 2023 in a 5-bed NICU.
Monthly surveillance cultures were performed to monitor colonization and
infection with multidrug-resistant organisms. CRKP isolates were
obtained from blood and nasal swabs of affected neonates. Identification
and antimicrobial susceptibility testing were initially conducted using
the Vitek®2 system with an N-395 card and further confirmed by 16S rRNA
sequencing. Whole-genome sequencing (WGS) and antimicrobial resistance
(AMR) profiling were performed to identify resistance genes and
virulence factors. For genetic analysis, both Illumina short-read and
Nanopore long-read sequencing were used, followed by hybrid assembly for
enhanced genome resolution. Plasmid and resistance gene profiles were
determined using AMRFinder and PlasmidFinder databases.ResultsA total of
three CRKP isolates (designated Kp1, Kp2, and Kp3) were identified. Kp1
and Kp2 belonged to sequence type (ST) ST23 and were genetically
near-identical, differing by a single allele, while Kp3 was of a
distinct sequence type, ST2096, with 245 allelic differences from Kp1
and Kp2. All isolates were resistant to colistin and carried resistance
genes, including mcr-1 and blaNDM-1,blaKPC2 confirming carbapenem
resistance. Efflux pump genes and aminoglycoside resistance genes were
also detected, providing a multifaceted defence against antibiotics.
Plasmid analysis identified several incompatibility groups (IncFI,
IncHI, IncFIB, IncX), indicating the potential for horizontal gene
transfer of resistance determinants.ConclusionThis study highlights the
complexity of CRKP outbreaks in neonatal care, with isolates exhibiting
resistance mechanisms that complicate treatment. The plasmid profiles
suggest these strains are reservoirs for multidrug-resistant genes,
emphasizing the need for strict infection control and ongoing genomic
surveillance. For neonatal care, these resistance challenges increase
the risk of treatment failures and mortality, underscoring the
importance of enhanced infection prevention and novel therapeutic
strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1528017"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1528017</a></p>
<hr />
<h2 id="does-gatorade-cause-kidney-stones">Does Gatorade Cause Kidney
Stones?</h2>
<p>date: 2025-01-25, from: The Kidney Dietitian</p>
<p>
It is common knowledge that hydration is important for kidney stones.
But what fluids are the best? Can Gatorade contribute to and cause
kidney stone formation? While Gatorade can have advantages for people
who sweat a lot during intense exercise, gatorade could cause kidney
stones in susceptible people. Read on to learn why hydration is […]
</p>
<p>
The post
<a href="https://www.thekidneydietitian.org/is-gatorade-good-for-kidney-stones/">Does
Gatorade Cause Kidney Stones?</a> appeared first on
<a href="https://www.thekidneydietitian.org">The Kidney Dietitian</a>.
</p>
<p><br></p>
<p><a
href="https://www.thekidneydietitian.org/is-gatorade-good-for-kidney-stones/"
class="uri">https://www.thekidneydietitian.org/is-gatorade-good-for-kidney-stones/</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
