<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2025-04-15 06:08:28)</p>
<hr />
<h2
id="sars-cov-2-causes-significant-abnormalities-in-the-fibrinolysis-system-of-patients-correlation-between-viral-mutations-variants-and-thrombosis">SARS-COV-2
causes significant abnormalities in the fibrinolysis system of patients:
correlation between viral mutations, variants and thrombosis</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCoronavirus disease (COVID-19) is reported as a complex
disorder affecting multiple systems and coagulopathy that can cause
mortality. In this study, we investigated the correlation of SARS-CoV-2
mutations found in blood samples with various changes in the
fibrinolysis system, as well as the severity of the disease based on
outcome and whether or not these patients were admitted into the
ICU.Materials and methodsCOVID-19 patients (n = 446) admitted to our
institute between 2021 and 2022 were recruited. Blood samples were
collected, and a sequence analysis of the SARS-CoV-2 spike gene was
isolated from the blood. Measured several parameters of fibrinolysis and
coagulation, including alpha-2-antiplasmin and plasminogen, thrombin
activatable fibrinolysis inhibitor (TAFI), tissue plasminogen activator
(tPA), plasminogen activator inhibitor-1 (PAI-1), D-dimer, and
fibrinogen levels.ResultsSARS-CoV-2 RNA was found in 123/446 (27.6%) of
the blood samples. The N501Y, D614G, K417N, and P681R mutations among
COVID-19 patients were associated with higher admissions to the ICU (P =
0.0057, P = 0.0068, P = 0.0193, and P = 0.018, respectively). Omicron
(BA.1.1) variant variants are highly associated with thrombosis (P =
0.002) in hospitalized COVID-19 patients that are unvaccinated and have
comorbidity conditions. The plasma levels of tPA, aPTT, and D-dimer were
significantly higher in participants who had the N501Y mutation (P =
0.044, P = 0.024, and P = 0.027, respectively).ConclusionThrombosis was
the most prevalent condition among severe COVID-19 patients. The
correlation between specific SARS-CoV-2 new variants and thrombosis
warrants more investigation.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1531412"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1531412</a></p>
<hr />
<h2
id="enhancing-fever-of-unknown-origin-diagnosis-machine-learning-approaches-to-predict-metagenomic-next-generation-sequencing-positivity">Enhancing
fever of unknown origin diagnosis: machine learning approaches to
predict metagenomic next-generation sequencing positivity</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveMetagenomic next-generation sequencing (mNGS) can
potentially detect various pathogenic microorganisms without bias to
improve the diagnostic rate of fever of unknown origin (FUO), but there
are no effective methods to predict mNGS-positive results. This study
aimed to develop an interpretable machine learning algorithm for the
effective prediction of mNGS results in patients with FUO.MethodsA
clinical dataset from a large medical institution was used to develop
and compare the performance of several predictive models, namely eXtreme
Gradient Boosting (XGBoost), Light Gradient-Boosting Machine (LightGBM),
and Random Forest, and the Shapley additive explanation (SHAP) method
was employed to interpret and analyze the results.ResultsThe
mNGS-positive rate among 284 patients with FUO reached 64.1%. Overall,
the LightGBM-based model exhibited the best comprehensive predictive
performance, with areas under the curve of 0.84 and 0.93 for the
training and validation sets, respectively. Using the SHAP method, the
five most important factors for predicting mNGS-positive results were
albumin, procalcitonin, blood culture, disease type, and sample
type.ConclusionThe validated LightGBM-based predictive model could have
practical clinical value in enhancing the application of mNGS in the
etiological diagnosis of FUO, representing a powerful tool to optimize
the timing of mNGS.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1550933"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1550933</a></p>
<hr />
<h2
id="characteristics-and-outcomes-in-severe-and-critically-ill-children-with-first-wave-sars-cov-2-omicron-infection-in-northeast-china">Characteristics
and outcomes in severe and critically ill children with first wave
SARS-CoV-2 Omicron infection in Northeast China</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>AimTo describe the characteristics of severe and critically ill
children with first-wave SARS-CoV-2 Omicron infection admitted to the
pediatric intensive care unit (PICU) at the National Children’s Regional
Medical Center in Northeast China and to explore factors associated with
poor outcomes.MethodsThis observational cohort study was conducted in a
PICU in northeastern China and included children under 18 years of age
who were severely and critically ill due to SARS-CoV-2 Omicron infection
between December 2022 and February 2023. Patients were categorized into
two groups: the invasive mechanical ventilation (IMV) group and the
non-IMV group. The primary outcome measured was the need for IMV, while
secondary outcomes included death or prolonged PICU stay. Univariate and
multivariate logistic regression analyses were performed to identify
risk factors for poor outcomes.ResultsA total of 38 severe and
critically ill children were included in the study. Of these, 25 (66%)
were diagnosed with respiratory failure, and four (16%) developed acute
respiratory distress syndrome. Additionally, 21 (55%) were diagnosed
with COVID-19-associated neurological disorders, and 18 (47%) received
IMV. Multivariate logistic regression analysis identified the chest
computed tomography (CT) score, based on the COVID-19 Risk Assessment
and Diagnosis System (CO-RADS), was statistically significant as an
independent predictor for IMV in severe and critically ill children
(odds ratio [OR]: 2.781 [95% confidence interval (CI): 1.021–7.571]).
Furthermore, the Pediatric Logistic Organ Dysfunction-2 (PELOD-2) score
and serum aspartate aminotransferase (AST) levels at admission were
found to be independent predictors of death or prolonged PICU
stay.ConclusionsRespiratory failure and COVID-19-associated neurological
disorders were the most common complications among severe and critically
ill children with first-wave SARS-CoV-2 Omicron infection. Chest CT
score, PELOD-2 score, and serum AST levels may serve as important
indicators of poor outcomes in this patient population.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1495783"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1495783</a></p>
<hr />
<h2
id="development-of-the-breastfed-infant-oral-microbiome-over-the-first-two-years-of-life-in-the-blosom-cohort">Development
of the breastfed infant oral microbiome over the first two years of life
in the BLOSOM Cohort</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAcquisition and development of the oral microbiome are
dynamic processes that occur over early life. This study aimed to
characterize the temporal development of the oral microbiome of
predominantly breastfed infants during the first two years of
life.MethodsInfant oral samples (n=667) were collected at ten time
points from the first week to year two of life from 84 infants.
Bacterial DNA profiles were analyzed using full-length 16S rRNA gene
sequencing.ResultsThe oral microbiome was dominated by Streptococcus
mitis, Gemella haemolysans, and Rothia mucilaginosa. Bacterial richness
decreased from 1 to 2 months (P = 0.043) and increased from 12 to 24
months (P = 0.038). Shannon diversity increased from 1 week to 1 month
and again from 6 to 9 months and 9 to 12 months (all P ≤ 0.04). The
composition of the infant oral microbiome was associated with multiple
factors, including early pacifier use, intrapartum antibiotic
prophylaxis, maternal allergy, pre-pregnancy body mass index, siblings,
delivery mode, maternal age, pets at home, and birth season (all P &lt;
0.01). Introduction of solid foods was a significant milestone in oral
microbiome development, triggering an increase in bacterial diversity
(richness P = 0.0004; Shannon diversity P = 0.0007), a shift in the
abundance of seven species, and a change in beta diversity (P =
0.001).ConclusionThese findings underscore the importance of multiple
factors, particularly the introduction of solid foods, in shaping the
oral microbiome of breastfed infants during early life.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534750"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534750</a></p>
<hr />
<h2
id="antifungal-susceptibility-and-clinical-efficacy-of-chlorhexidine-combined-with-topical-ophthalmic-medications-against-fusarium-species-isolated-from-corneal-samples">Antifungal
susceptibility and clinical efficacy of chlorhexidine combined with
topical ophthalmic medications against Fusarium species isolated from
corneal samples</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study investigated the susceptibility of various
Fusarium fungi to five topical antifungal agents: natamycin,
voriconazole, chlorhexidine, natamycin combined with chlorhexidine, and
voriconazole combined with chlorhexidine. And to explore the clinical
feasibility of combination therapy in the treatment of corneal
infections caused by F. solani, with the goal of optimizing the
treatment regimen for fungal keratitis.MethodsA total of 194 strains of
Fusarium were isolated from the corneas between 2013 and 2024 and
identified to the species level using mass spectrometry. The MICs was
determined using a commercial microdilution plate to assess the in vitro
activity of the drugs used alone and in combination
(natamycin/chlorhexidine, voriconazole/chlorhexidine). Additionally, the
clinical efficacy was prospectively observed in 5 patients with corneal
infections caused by F. solani. The treatment regimens included 5%
natamycin combined with 0.04% chlorhexidine, chlorhexidine used alone,
or natamycin used alone, with follow-up lasting up to 90 days.ResultsF.
solani species complex (FSSC, 46.91%) and the F. fujikuroi species
complex (FFSC, 45.88%) were the predominant isolates, with a
geographical distribution concentrated in Northern China. The MICs for
natamycin in FSSC and FFSC were both 2–8 μg /mL, respectively. The MICs
for FSSC and FFSC respectively ranged from 0.25–16 and 1–8 μg/mL for
voriconazole and 2 to &gt; 16 μg/mL and 1 to &gt; 16 μg/mL for
chlorhexidine. The MICs of natamycin were not significantly different
between FSSC and FFSC. However, voriconazole, chlorhexidine, natamycin
combined with chlorhexidine, and voriconazole combined with
chlorhexidine had significantly higher MICs for FSSC compared with FFSC.
Compared with voriconazole, voriconazole combined with chlorhexidine
exhibited enhancement of antifungal activity against 100% of tested
Fusarium strains. Compared with natamycin, enhancement of antifungal
activity of natamycin combined with chlorhexidine was 81.4% for all
Fusarium spp., and the activity were significantly lower for F. solani
(65.9%) than for non-F. solani species (93.6%). Among the 5 patients, 3
patients received treatment with natamycin combined with chlorhexidine,
resulting in clinical cure in 2 patients (in 1-1.5 months), while 1
patient required a corneal transplant due to delayed treatment. One
patient treated with natamycin alone and one treated with chlorhexidine
alone both achieved clinical cure (in 2-3 months).ConclusionNatamycin
combined with chlorhexidine and voriconazole combined with chlorhexidine
exhibited enhancement of antifungal activity against Fusarium spp.
during in vitro sensitivity tests. The findings of this study provide
valuable guidance for establishing the epidemiological cutoff and
clinical MIC values for Fusarium spp. This study paves the way for
future multicenter studies on the treatment of FK with natamycin and
chlorhexidine.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1532289"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1532289</a></p>
<hr />
<h2
id="custom-barcoded-primers-for-influenza-a-nanopore-sequencing-enhanced-performance-with-reduced-preparation-time">Custom
barcoded primers for influenza A nanopore sequencing: enhanced
performance with reduced preparation time</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Highly pathogenic avian influenza is endemic and widespread in wild
birds and is causing major outbreaks in poultry worldwide and in U.S.
dairy cows, with several recent human cases, highlighting the need for
reliable and rapid sequencing to track mutations that may facilitate
viral replication in different hosts. SNP analysis is a useful molecular
epidemiology tool to track outbreaks, but it requires accurate
whole-genome sequencing (WGS) with sufficient read depth across all
eight segments. In outbreak situations, where timely data is critical
for controlling the spread of the virus, reducing sequencing preparation
time while maintaining high-quality standards is particularly important.
In this study, we optimized a custom barcoded primer strategy for
influenza A whole-genome sequencing on the nanopore sequencing platform,
combining the high performance of the Native Barcoding Kit with the
prompt preparation time of the Rapid Barcoding Kit. Custom barcoded
primers were designed to perform barcode attachment during RT-PCR
amplification, eliminating the need for separate barcoding and clean-up
steps, thus reducing library preparation time. We compared the
performance of the custom barcoded primer method with the Native and
Rapid barcoding kits in terms of read quality, read depth, and
sequencing output. The results show that the custom barcoded primers
provided performance comparable to the Native Barcoding Kit while
reducing library preparation time by 2.3X compared to the Native kit and
being only 15 minutes longer than the Rapid kit with better depth of
sequencing. Additionally, the custom barcoded primer method was
evaluated on a variety of clinical sample types. This approach offers a
promising solution for influenza A sequencing, providing both high
throughput and time efficiency, which significantly improves the
time-to-result turnaround, making sequencing more accessible for
real-time surveillance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545032"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545032</a></p>
<hr />
<h2
id="treatment-of-xerostomia-in-sjögrens-syndrome-what-effect-does-it-have-on-the-oral-microbiome">Treatment
of xerostomia in Sjögren’s syndrome – what effect does it have on the
oral microbiome?</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Sjögren’s syndrome is an autoimmune disease characterized by
lymphatic infiltration of secretory tissues. The disease results in
dryness of the eyeball or mouth, which often occur simultaneously.
Agents used to treat Sjögren’s syndrome may improve oral hydration and
the patient’s quality of life. There are several pharmacological and
non-pharmacological agents used to treat significant problem like
xerostomia. The use of appropriate medicines (i.e. pilocarpine and
cevimeline) may cause changes in the local microbiome, which is very
sensitive to quantitative changes in water. As a result of Sjögren’s
syndrome, a new balance of the microbiome is established in the oral
cavity, which, if disturbed by medical measures, may increase the risk
of oral lesions (such as periodontopathies or caries) or reduce this
risk. Overall, the knowledge regarding microbiological aspects and
agents treating oral dryness is still not well described but initial
results indicate some microbial alterations.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1484951"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1484951</a></p>
<hr />
<h2
id="distinct-blood-and-oral-microbiome-profiles-reveal-altered-microbial-composition-and-functional-pathways-in-myocardial-infarction-patients">Distinct
blood and oral microbiome profiles reveal altered microbial composition
and functional pathways in myocardial infarction patients</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe blood microbiome, increasingly recognized as a
distinct microbial niche, may originate partly from oral translocation.
We systematically compared circulating and oral microbiome profiles
between healthy individuals and myocardial infarction (MI) patients to
identify disease-associated signatures.MethodsThe current study
recruited 20 participants, including 10 healthy controls and 10 patients
with MI. Blood and Saliva samples were collected from each participant
to analyze the association between blood and oral microbiome in MI
patients using 16S rRNA gene sequencing.ResultsThe blood microbiome
showed significantly greater alpha diversity than the oral microbiome
(p&lt;0.05), but beta diversity did not differ significantly. The blood
microbiome in MI patients had higher levels of Firmicutes, Bacteroidota,
Actinobacteriota, genus Bacteroides, and lower Proteobacteria, whereas
the oral microbiome was dominated by Firmicutes, Bacteroidota,
Veillonella, and Prevotella_7. LEfSe analysis revealed distinct blood
microbial taxa-Actinobacteria in MI patients and Enterobacterales in
controls. In contrast, the oral microbiota of healthy subjects was
enriched in Rothia, Micrococcaceae, and Micrococcales, while no distinct
taxa were associated with MI. Both blood and oral microbiomes showed
significant functional pathway differences (KEGG) between groups.
Additionally, microbiome signatures significantly correlated with
clinical and demographic markers.DiscussionOur study demonstrates that
the blood harbors a distinct microbiome characterized by specific
microbial taxa and functional pathways rather than merely reflecting
oral bacterial translocation. These findings suggest that the
circulating microbiome may play an active role in the pathology of MI.
Furthermore, we identified significant associations between these
microbial signatures and clinical disease markers. This highlights the
potential importance of the blood microbiome in understanding the
mechanisms underlying MI and its diagnostic or therapeutic
implications.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1506382"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1506382</a></p>
<hr />
<h2
id="vaginal-microbiota-changes-of-persistent-human-papillomavirus-infection-after-cervical-conization">Vaginal
microbiota changes of persistent human papillomavirus infection after
cervical conization</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveWe investigated the changes in vaginal microbiota among
females with persistent human papillomavirus (HPV) infection following
cervical conization in Xinjiang, China.MethodsA total of 108 female
participants were enrolled in the study, including 37 HPV-positive
females without cervical conization (Group P1), 37 HPV-positive females
after cervical conization (Group P2), and 34 HPV-negative females after
cervical conization (Group N). DNA was extracted from vaginal
secretions, and the V3-V4 regions of bacterial 16S rDNA were amplified
and sequenced using NovaSeq technology. The diversity analysis of the
bacterial microbiota was conducted using QIIME2 and R software, while
the phenotypic analysis was performed with Bugbase
software.ResultsLactobacillus was the predominant genus in the vaginal
microbiota of women with persistent HPV infection after cervical
conization in Xinjiang. Following partial cervical resection, the
α-diversity of the vaginal microbiota decreased, particularly among
patients who had cleared HPV. Bacterial vaginosis-associated anaerobes
were common in the vaginal environment, with their relative abundance
increasing in cases of persistent HPV infection. Postoperative
persistent HPV infection was found to be correlated not only with
pathogens linked to bacterial vaginosis but also with those associated
with aerobic vaginitis. Gardnerella and Atopobium, as well as
Bifidobacterium and Streptococcus, demonstrated a symbiotic synergy.
Both Lactobacillus and Gardnerella exhibited negative correlations with
many pathogenic bacteria. Anaerobic and biofilm formation were the most
evident phenotypes in individuals with persistent HPV infection after
conization.ConclusionThe vaginal microbiota of women with persistent HPV
infection following cervical conization is characterized by the
coexistence of Lactobacillus dominance and increased microbial
diversity. Anaerobic bacteria and biofilm formation may play a
significant role in the persistence of HPV infection post-surgery, and
the role of Gardnerella in the vaginal flora under an HPV-infected state
warrants further study.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1544794"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1544794</a></p>
<hr />
<h2
id="ehrlichia-chaffeensis-proteomic-profiling-reveals-distinct-expression-patterns-of-infectious-and-replicating-forms">Ehrlichia
chaffeensis proteomic profiling reveals distinct expression patterns of
infectious and replicating forms</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Ehrlichia chaffeensis is a tick-transmitted rickettsial pathogen
responsible for causing human monocytic ehrlichiosis (HME). The
pathogen’s developmental cycle includes infectious dense-core cells
(DCs) and non-infectious replicating cells (RCs). Defining the proteins
crucial for the two growth forms is of fundamental importance in
understanding the infection and replication process, which also aids in
identifying novel therapeutic targets against HME and other related
rickettsial diseases. E. chaffeensis organisms cultivated in a
macrophage cell line were purified as DC and RC fractions and subjected
to comprehensive quantitative proteome analysis. From triplicate sample
analysis, we identified 195 proteins as commonly expressed in both the
DC and RC forms, while an additional 189 proteins were recognized as
exclusively expressed in the RC form. Equal numbers of commonly
expressed proteins in the RC and DC forms and having substantially more
proteins exclusively expressed in the metabolically active RC form may
reflect specific functional priorities of E. chaffeensis supporting its
replication within a phagosome. The high abundance of metabolic
processes and transport proteins in the RC compared to the DC form may
reflect its higher metabolic requirements and interactions with a host
cell supporting its intraphagosomal replication. This study provides
comprehensive proteome data for E. chaffeensis which will be valuable
for a better understanding of protein expression dynamics during its
infectious and replicating stages.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1463479"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1463479</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
