<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2024-09-10 08:28:32)</p>
<hr />
<h2
id="fine-mapping-based-multi-omics-analysis-interprets-the-gut-lung-axis-function-of-sglt2-inhibitors">Fine
mapping-based multi-omics analysis interprets the gut-lung axis function
of SGLT2 inhibitors</h2>
<p>date: 2024-09-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCurrently, Sodium-glucose cotransporter 2 (SGLT2)
inhibitors demonstrate additional effects beyond glucose control on the
gut microbiota and circulating metabolites. The gut microbiota and
metabolites have been found to be useful in elucidating potential
biological mechanisms of pulmonary diseases. Therefore, our study aims
to investigate the effects of gut microbiota and metabolites mediating
SGLT2 inhibition in 10 pulmonary diseases through Mendelian
randomization (MR) research.MethodsWe conducted a two-sample, two-step
MR study to assess the association between SGLT2 inhibition and 10
pulmonary diseases and to investigate the mediating effects of gut
microbiota and metabolite. Gene-fine mapping and annotation of mediators
by FUMA and Magma analyses were performed, and causal associations of
mapped genes with diseases were assessed by muti-omics MR analyses.
Possible side effects of SGLT2 inhibition were assessed by PheWAS
analysis.ResultsSGLT2 inhibition was linked to a reduced risk of T2DM,
Interstitial lung disease (ILD), Pneumoconiosis, Pulmonary tuberculosis,
and Asthma(OR=0.457, 0.054, 0.002, 0.280, 0.706). The family
Enterobacteriaceae and order Enterobacteriales were associated with
SGLT2 inhibition and ILD(95% CI:0.079–0.138). The family Alcaligenaceae
and X-12719 were linked to pneumoconiosis (95% CI: 0.042–0.120,
0.050–0.099). The genus Phascolarctobacterium was connected to pulmonary
tuberculosis (95% CI: 0.236–0.703).The degree of unsaturation (Fatty
Acids), ratio of docosahexaenoic acid to total fatty acids, and
4-androsten-3beta,17beta-diol disulfate 2, were associated with
asthma(95% CI: 0.042–0.119, 0.039–0.101, 0.181–0.473). Furthermore, Fuma
and Magma analyses identified target genes for the four diseases, and
proteomic MR analysis revealed six overlapping target genes in asthma.
PheWAS analysis also highlighted potential side effects of SGLT2
inhibition.ConclusionsThis comprehensive study strongly supports a
multi-omics association between SGLT2 inhibition and reduced risk of
interstitial lung disease, tuberculosis, pneumoconiosis, and asthma.
Four identified gut microbiota, four metabolites, sixteen metabolic
pathways, and six target genes appear to play a potential role in this
association. The results of the comprehensive phenome-wide association
analysis also identified the full effect of SGLT2 inhibitors.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1447327"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1447327</a></p>
<hr />
<h2
id="expression-of-goat-poxvirus-p32-protein-and-monoclonal-antibody-preparation">Expression
of goat poxvirus P32 protein and monoclonal antibody preparation</h2>
<p>date: 2024-09-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>P32 protein serves as a crucial structural component of Goat pox
virus (GTPV), which causes a highly virulent infectious disease in sheep
and goats. Despite the fact that P32 has been widely expressed in the
previous studies, it is difficult to obtain recombinant P32 efficiently.
This study aimed to achieve soluble expression of P32 recombinant
protein and to develop its specific monoclonal antibody. The gene
fragment of P32Δ (GP32Δ) was synthesized by optimizing the coding
sequence of amino acids 1-246 of the known goatpox P32 protein.
Subsequently, GP32Δ was cloned into a prokaryotic expression vector for
expression and purification, resulting in the successful production of
soluble recombinant protein rP32Δ. Utilizing rP32Δ, an indirect ELISA
method was established by immunizing 6-week-old BALB/c mice with
inactivated GTPV as the antigen. Through hybridoma technology, three
monoclonal antibody hybridoma cell lines secreting anti-goat pox virus
rP32Δ were screened, designated as 2F3, 3E8, and 4H5, respectively.
These monoclonal antibodies, classified as IgG1, IgG2a, and IgG2b,
respectively, with κappa light chains, were characterized following
ascites preparation and purification. Indirect ELISA results
demonstrated that the ELISA potency of the three monoclonal antibodies
exceeded 1:12800. Furthermore, Western blot analysis revealed specific
reactivity of both 3E8 and 4H5 with rP32Δ, while immunofluorescence
assays confirmed 3E8’s ability to specifically recognize GTPV in cells.
The preceding findings demonstrate the successful acquisition of the
soluble expressed recombinant P32 protein and its specific monoclonal
antibody 3E8 in this study, thereby laying a foundational material basis
for the establishment of a GTPV detection method.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1427588"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1427588</a></p>
<hr />
<h2
id="recombinase-aided-amplification-assay-for-rapid-detection-of-imipenem-resistant-pseudomonas-aeruginosa-and-rifampin-resistant-pseudomonas-aeruginosa">Recombinase-aided
amplification assay for rapid detection of imipenem-resistant
Pseudomonas aeruginosa and rifampin-resistant Pseudomonas
aeruginosa</h2>
<p>date: 2024-09-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The indiscriminate use of antibiotics has resulted in a growing
resistance to drugs in Pseudomonas aeruginosa. The identification of
antibiotic resistance genes holds considerable clinical significance for
prompt diagnosis. In this study, we established and optimized a
Recombinase-Aided Amplification (RAA) assay to detect two genes
associated with drug resistance, oprD and arr, in 101 clinically
collected P. aeruginosa isolates. Through screening for the detection or
absence of oprD and arr, the results showed that there were 52
Imipenem-resistant P. aeruginosa (IRPA) strains and 23
Rifampin-resistant P. aeruginosa (RRPA) strains. This method
demonstrated excellent detection performance even when the sample
concentration is 10 copies/μL at isothermal conditions and the results
could be obtained within 20 minutes. The detection results were in
accordance with the results of conventional PCR and Real-time PCR. The
detection outcomes of the arr gene were consistently with the resistance
spectrum. However, the antimicrobial susceptibility results revealed
that 65 strains were resistant to imipenem, while 49 strains sensitive
to imipenem with oprD were identified. This discrepancy could be
attributed to genetic mutations. In summary, the RAA has higher
sensitivity, shorter time, and lower-cost instrument requirements than
traditional detection methods. In addition, to analyze the
epidemiological characteristics of the aforementioned drug-resistant
strains, we conducted Multilocus Sequence Typing (MLST), virulence gene,
and antimicrobial susceptibility testing. MLST analysis showed a strong
correlation between the sequence types ST-1639, ST-639, ST-184 and IRPA,
while ST-261 was the main subtype of RRPA. It was observed that these
drug-resistant strains all possess five or more virulence genes, among
which exoS and exoU do not coexist, and they are all multidrug-resistant
strains. The non-coexistence of exoU and exoS in P.aeruginosa is related
to various factors including bacterial regulatory mechanisms and
pathogenic mechanisms. This indicates that the relationship between the
presence of virulence genes and the severity of patient infection is
worthy of attention. In conclusion, we have developed a rapid and
efficient RAA (Recombinase-Aided Amplification) detection method that
offers significant advantages in terms of speed, simplicity, and
cost-effectiveness (especially in time and equipment aspect). This novel
approach is designed to meet the demands of clinical diagnostics.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1428827"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1428827</a></p>
<hr />
<h2
id="head-to-head-comparison-of-two-human-papillomavirus-vaccines-for-efficacy-against-cervical-intraepithelial-neoplasia-grade-3-and-adenocarcinoma-in-situpopulation-based-follow-up-of-two-cluster-randomized-trials">Head-to-head
comparison of two human papillomavirus vaccines for efficacy against
cervical intraepithelial neoplasia grade 3 and adenocarcinoma in
situ—population-based follow-up of two cluster-randomized trials</h2>
<p>date: 2024-09-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionWe report head-to-head comparison of the bivalent and
quadrivalent HPV vaccine efficacies against immediate precursors of
cervical cancer from 15 years’ country-wide cancer registry follow-up of
phase III trial cohorts and an age-aligned cohort of unvaccinated
women.MethodsThese individually and/or clusterrandomized cohorts of
HPV6/11/16/18- and HPV16/18-vaccinated and unvaccinated women were
enrolled, respectively, in 2002, 2004, and 2003/2005. The trial cohorts
comprised initially 16- to 17-year-old HPV6/11/16/18-vaccinated FUTURE
II (NCT00092534) participants (866) and HPV16/18-vaccinated PATRICIA
(NCT00122681) and 012 trial (NCT00169494) participants (2,465), and
16,526 initially 16- to 19-year-old unvaccinated controls. After active
4-year clinical follow-up, passive, country-wide Finnish Cancer Registry
(FCR) follow-up for cervical intraepithelial neoplasia grade 3 (CIN3)
and adenocarcinoma in situ (AIS) was based on consented use of unique
personal identifiers and started 6 months after the end of the FUTURE II
and PATRICIA trials in 2007 and 2009, and ended at the end of 2019. The
follow-up with altogether 229,020 follow-up years was age-aligned to
ensure that similarly aged cohorts were passively followed up for 15
years post=vaccination for the intention-to-treat analyses of vaccine
efficacy.ResultsOverall, we identified 5 and 16 CIN3 (no AIS) cases in
the HPV6/11/16/18 and HPV16/18 cohorts, respectively, during the
FCR-based follow-up. In the unvaccinated cohort, we identified 281 CIN3
cases, 20 AIS cases, and 13 cases with invasive cervical cancer. Vaccine
efficacies against CIN3+ were 68.4% and 64.5% for the quadrivalent and
the bivalent vaccines, respectively, with overlapping confidence
intervals.DiscussionLong-term follow-up of randomized, initially
adolescent HPV-vaccinated and unvaccinated cohorts shows, in this
head-to-head setting, that the bivalent and quadrivalent HPV vaccines
are equally effective against immediate precursors of cervical
cancer.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1437704"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1437704</a></p>
<hr />
<h2
id="vaginal-microbiota-of-pregnant-women-with-ureaplasma-urealyticum-and-mycoplasma-hominis-infections">Vaginal
microbiota of pregnant women with Ureaplasma urealyticum and Mycoplasma
hominis infections</h2>
<p>date: 2024-09-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe association between preterm birth and Mycoplasma
species such as Mycoplasma hominis and Ureaplasma urealyticum has been
extensively investigated. In a clinical setting, conventional diagnostic
methods for them involve culture methods for Mycoplasma spp. and
Ureaplasma spp., along with PCR tests. However, the clinical utility of
these tests remains controversial, highlighting the necessity for more
robust and reliable methods for identifying and understanding Mycoplasma
infections.ObjectiveThis study aimed to assess the distribution of
microbiota in pregnant women with Mycoplasma hominis and Ureaplasma
urealyticum infection by the comparison of conventional diagnostic
methods with vaginal microbial community analysis.Study DesignThis
prospective case–control study involved 228 Korean pregnant women and
utilized vaginal microbial community analysis, Ureaplasma/Mycoplasma
culture, and 12-multiplex PCR for sexually transmitted diseases.
Cross-correlation analysis in SPSS 27 compared the results of two
conventional methods with vaginal microbial community analysis. R
software generated box plots depicting the relative abundance of
microorganisms. Network analysis was conducted using
Cytoscape.ResultsPositive Ureaplasma urealyticum culture findings were
observed in 60.2% of patients, with 76.4% positive for Ureaplasma parvum
PCR and 13.2% positive for Ureaplasma urealyticum PCR. Mycoplasma
hominis culture was positive only in two patients, while Mycoplasma
hominis PCR was positive in eight women. Vaginal microbial community
analysis identified significant differences in relative abundances of
Gardnerella species type I and Fannyhessea vaginae between the
Ureaplasma urealyticum PCR positive and negative groups. Mycoplasma
hominis PCR positive patients exhibited significant differences in 11
bacterial species, including Gardnerella species I and Fannyhessea
vaginae.ConclusionThis study suggests that STD-PCR may be more accurate
than Ureaplasma/Mycoplasma culture for the diagnosis of Mycoplasma
hominis and Ureaplasma urealyticum infection. Also, the presence of
Gardnerella species I and Fannyhessea vaginae implies their potential
influences on Ureaplasma urealyticum and Mycoplasma hominis infections
based on results of vaginal microbial community analysis. Therefore,
vaginal microbial community analysis may give the more information of
their pathophysiology.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445300"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445300</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
