<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2025-06-06 06:09:06)</p>
<hr />
<h2 id="how-is-the-human-microbiome-linked-to-kidney-stones">How is the
human microbiome linked to kidney stones?</h2>
<p>date: 2025-06-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>In recent years, the incidence of kidney stones has continued to rise
worldwide, and conventional treatments have limited efficacy in treating
stones associated with recurrent or metabolic abnormalities. The
microbiome, as the ‘second genome’ of the host, is involved in the
development of kidney stones through metabolic regulation, immune
homeostasis and inflammatory response. Studies have shown that the
urinary microbiome of healthy people is dominated by commensal bacteria
such as Lactobacillus and Streptococcus, which maintain
microenvironmental homeostasis, whereas patients with renal stones have
a significantly reduced diversity of intestinal and urinary microbiomes,
with a reduced abundance of oxalic acid-degrading bacteria (e.g.,
Bifidobacterium oxalicum, Bifidobacterium bifidum), and a possible
concentration of pathogenic bacteria (e.g., Proteus mirabilis). The
microbiome regulates stone formation through mechanisms such as
metabolites (e.g., short-chain fatty acids), changes in urine
physicochemical properties (e.g., elevated pH), and imbalances in the
inflammatory and immune microenvironments. For example, urease-producing
bacteria promote magnesium ammonium phosphate stone formation through
the breakdown of urea, whereas dysbiosis of the intestinal flora
increases urinary oxalic acid excretion and exacerbates the risk of
calcium oxalate stones. Microbiome-based diagnostic markers (e.g.,
elevated abundance of Aspergillus phylum) and targeted intervention
strategies (e.g., probiotic supplementation, faecal bacteria
transplantation) show potential for clinical application. However,
technical bottlenecks (e.g., sequencing bias in low-biomass samples),
mechanistic complexity (e.g., multistrain synergism), and individual
heterogeneity remain major challenges for future research. Integration
of multi-omics data, development of personalised therapies and
interdisciplinary research will be the core directions to decipher the
relationship between microbiome and kidney stones.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602413"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602413</a></p>
<hr />
<h2
id="a-machine-learning-model-for-robust-prediction-of-sepsis-induced-coagulopathy-in-critically-ill-patients-with-sepsis">A
machine learning model for robust prediction of sepsis-induced
coagulopathy in critically ill patients with sepsis</h2>
<p>date: 2025-06-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionSepsis-induced coagulopathy (SIC) is a common disease in
patients with sepsis. It denotes higher mortality rates and a poorer
prognosis in these patients. This study aimed to develop a practical
machine learning (ML) model for the prediction of the risk of SIC in
critically ill patients with sepsis.MethodsIn this retrospective cohort
study, patients were extracted from the Medical Information Mart for
Intensive Care IV (MIMIC-IV) database and the Inner Mongolia Autonomous
Region People’s Hospital database. Sepsis and SIC were defined based on
the Sepsis-3 criteria and the criteria developed based on the
International Society of Thrombosis and Haemostasis (ISTH),
respectively. We compared nine ML models using the Sequential Organ
Failure Assessment (SOFA) score in terms of SIC prediction ability.
Optimal model selection was based on the superior performance metrics
exhibited by the model on the training dataset, the internal validation
dataset, and the external validation dataset.ResultsOf the 15,479
patients in MIMIC-IV included in the final cohort, a total of 6,036
(38.9%) patients developed SIC during sepsis. We selected 17 features to
construct ML prediction models. The gradient boosting machine (GBM)
model was deemed optimal as it achieved high predictive accuracy and
reliability across the training, internal, and external validation
datasets. The areas under the curve of the GBM model were 0.773 (95%CI =
0.765–0.782) in the training dataset, 0.730 (95%CI = 0.715–0.745) in the
internal validation dataset, and 0.966 (95%CI = 0.938–0.994) in the
external validation dataset. The Shapley Additive Explanations (SHAP)
values illustrated the prediction results, indicating that total
bilirubin, red cell distribution width (RDW), systolic blood pressure
(SBP), heparin, and blood urea nitrogen (BUN) were risk factors for
progression to SIC in patients with sepsis.ConclusionsWe developed an
optimal and operable ML model that was able to predict the risk of SIC
in septic patients better than the SOFA scoring models.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1579558"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1579558</a></p>
<hr />
<h2
id="the-microbiological-features-and-risk-factors-for-liver-abscess-after-thermal-ablation-of-liver-cancer">The
microbiological features and risk factors for liver abscess after
thermal ablation of liver cancer</h2>
<p>date: 2025-06-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesLiver abscess is a severe complication that can occur
following thermal ablation of liver cancer, with an incidence rate
ranging from 0.2% to 2.4%. This study aimed to identify risk factors
associated with the development of post-ablation liver abscesses and to
characterize the microbiological features of the isolated
pathogens.MethodsA matched case-control study was conducted at Beijing
Youan Hospital from January 2018 to December 2023. Cases were defined as
patients who developed liver abscesses within three months following
ablation therapy, while controls comprised patients who did not develop
such abscesses. Clinical and microbiological data were collected and
analyzed. The relevant independent risk factors for the occurrence of
liver abscesses were identified and assessed using multivariate logistic
regression analysis.ResultsPost-ablation liver abscesses predominantly
occurred in male patients aged 60 years or older, typically manifesting
around six days after the procedure. Common symptoms included fever,
chills, abdominal distension, and abdominal pain. Multivariate analysis
identified diabetes mellitus (OR=3.215; 95% CI 1.330–7.771), history of
abdominal surgery (OR=2.810; 95% CI 1.074–7.353), biliary disease
(OR=18.832; 95% CI 2.291–154.795), and elevated ALP levels (OR=1.010;
95% CI 1.002–1.019) as independent risk factors for post-ablation liver
abscesses. Among the 61 patients with liver abscesses, a total of 78
bacterial strains were isolated from the abscess fluid, with
Gram-negative bacteria accounting for 75.6% of the isolates. Escherichia
coli (30.8%) and Klebsiella pneumoniae (20.5%) were the most frequently
identified pathogens. Drug sensitivity testing revealed that both
Escherichia coli and Klebsiella pneumoniae exhibited high susceptibility
to Amikacin, Cefoxitin, Ceftazidime, Imipenem, Meropenem, and
Piperacillin-tazobactam.ConclusionsPost-ablation liver abscesses are
primarily caused by Gram-negative bacteria. A history of diabetes, prior
abdominal surgery, biliary disease, and elevated ALP levels are
significant risk factors influencing the development of post-ablation
liver abscesses.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1567105"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1567105</a></p>
<hr />
<h2
id="retrospective-cohort-analysis-on-predicting-pulmonary-fibrosis-in-elderly-sars-cov-2-infected-patients">Retrospective
cohort analysis on predicting pulmonary fibrosis in elderly
SARS-CoV-2-infected patients</h2>
<p>date: 2025-06-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSARS-CoV-2 exhibits rapid transmission with a high
susceptibility rate, particularly among the elderly. Pulmonary fibrosis
(PF) following SARS-CoV-2 infection is a life-threatening complication.
However, predictive models for PF in older patients are
lacking.MethodsData from patients with COVID-19 aged 60 and above,
collected retrospectively between November 2022 and November 2023 across
two independent hospitals, were analyzed. Patients from Tangdu Hospital
were divided into training and validation cohorts using a 7:3 allocation
ratio, while those from The 940th Hospital of the Joint Logistics
Support Force of the People’s Liberation Army (PLA) served as the test
cohort. Identify the most valuable predictors (MVPs) for PF using Least
Absolute Shrinkage and Selection Operator (LASSO) regression, and
construct a nomogram based on their regression coefficients derived from
logistic regression. The calibration, clinical utility, and
discriminatory ability of the nomogram were evaluated using the
Hosmer-Lemeshow test, decision curve analysis (DCA), and Receiver
Operating Characteristic (ROC) curve, respectively.ResultsNeutrophil
percentage, C-reactive protein (CRP), gender, diagnostic classification,
and time from symptom onset to hospitalization were identified as the
MVPs for PF. The nomogram was developed based on these predictors, In
all the three cohorts, the nomogram showed good calibration, clinical
utility and discriminatory ability, with Area Under the Curve (AUC) of
0.777, 0.735 and 0.753, respectively. Furthermore, based on the
principle of optimizing the balance between sensitivity and specificity,
131.026 was determined as the optimal cutoff value for the nomogram.
Accordingly, patients with a nomogram score of 131.026 or higher were
classified into the high-risk group.ConclusionsThis study presents the
first nomogram for predicting PF in elderly patients following
SARS-CoV-2 infection, which may serve as a clinical tool for risk
assessment and early management in this population.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587321"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587321</a></p>
<hr />
<h2
id="evolution-of-ceftazidime-avibactam-resistance-driven-by-variation-in-blakpc-2-to-blakpc-190-during-treatment-of-st11-k64-hypervirulent-klebsiella-pneumoniae">Evolution
of ceftazidime-avibactam resistance driven by variation in blaKPC-2 to
blaKPC-190 during treatment of ST11-K64 hypervirulent Klebsiella
pneumoniae</h2>
<p>date: 2025-06-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe emergence of Klebsiella pneumoniae carbapenemase
(KPC) variants has significantly compromised the efficacy of
ceftazidime-avibactam (CZA), a critical antibiotic for treating
carbapenem-resistant K. pneumoniae (CRKP) infections. This study
investigates the novel KPC-190 variant, identified in a hypervirulent
ST11-K64 K. pneumoniae strain during CZA therapy, which confers
resistance to CZA while partially restoring carbapenem
susceptibility.MethodsThe K. pneumoniae clinical isolate LX02 harboring
blaKPC-190 was characterized using antimicrobial susceptibility testing,
whole-genome sequencing (Illumina and Nanopore), and plasmid analysis.
Functional studies included plasmid transformation, cloning assays, and
enzyme kinetics (spectrophotometric analysis of purified KPC-190
protein). Genetic context was mapped using bioinformatics tools (RAST,
ResFinder, Proksee), and virulence determinants were
identified.ResultsKPC-190 exhibited a unique resistance profile:
high-level CZA resistance (MIC &gt;64 μg/mL) with reduced carbapenem
MICs (imipenem MIC = 2 μg/mL). Enzyme kinetics revealed decreased
Kcat/Km for carbapenems and ceftazidime, alongside a 9-fold higher IC50
for avibactam (0.13 μM vs. KPC-2’s 0.014 μM). Genomic analysis
identified blaKPC-190 within an IS26 flanked mobile element
(IS26-ISKpn8-blaKPC-ΔISKpn6-ΔtnpR-IS26) on an IncFII plasmid. The strain
also carried hypervirulence markers (rmpA2, iucABCD-iutA, and type 1/3
fimbriae).DiscussionThe KPC-190 variant underscores the adaptive
evolution of blaKPC under antibiotic pressure, combining CZA resistance
via enhanced ceftazidime affinity and avibactam evasion with retained
carbapenem hydrolysis. Its association with hypervirulence plasmids and
IS26-mediated mobility poses a dual threat for dissemination. These
findings highlight the urgent need for genomic surveillance and
alternative therapies (e.g., meropenem-vaborbactam) to address
KPC-190-mediated resistance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1607127"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1607127</a></p>
<hr />
<h2
id="detection-assay-of-polymyxin-resistance-coding-mcr-1-gene-based-on-crisprcas13a-system">Detection
assay of polymyxin resistance coding mcr-1 gene based on CRISPR/Cas13a
system</h2>
<p>date: 2025-06-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionPolymyxins are reserved as an ultimate defense against
multidrug-resistant bacteria. The emergence of the polymyxin resistance
gene mcr-1 poses a potential risk for the treatment of severe infections
caused by Gram-negative bacteria. Timely detection and monitoring the
mcr-1 gene are essential for guiding anti-infective therapy and
controlling the spread of polymyxin resistance. Quantitative real-time
PCR (qPCR) is one of the common methods for detecting resistance genes.
However, qPCR has equipment dependency, and is not feasible in primary
healthcare settings. Currently, there remains a lack of a highly
sensitive and portable method for detecting the mcr-1 gene.MethodsWe
established and optimized detection assays of the mcr-1 gene based on
CRISPR/Cas13a system and lateral flow strips. The detection method was
preliminarily evaluated using clinical isolates from Escherichia coli,
compared with qPCR.ResultsThe method for detecting the mcr-1 gene based
on the CRISPR/Cas13a system and lateral flow strips was established,
with a detection limit of 100 copies/mL. This method demonstrated high
analytical specificity, with no cross-reactivity detected in non-mcr-1
and non-resistant strains. Among 36 clinical isolates, the method
identified 31 strains as positive for the mcr-1 gene, and had a 100%
concordance rate with the results of qPCR.ConclusionsWe established a
detection method for the polymyxin resistance mcr-1 gene based on the
CRISPR/Cas13a system. This method enables visual readouts without
instruments, making it potentially applicable to primary healthcare
settings and field surveillance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1553681"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1553681</a></p>
<hr />
<h2
id="socs3-and-il-6-mrna-levels-in-pbmc-enhance-the-early-prediction-for-patients-with-acute-on-chronic-hepatitis-b-liver-failure-receiving-glucocorticoid-therapy">SOCS3
and IL-6 mRNA levels in PBMC enhance the early prediction for patients
with acute-on-chronic hepatitis B liver failure receiving glucocorticoid
therapy</h2>
<p>date: 2025-06-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionEarly prediction is essential in hepatitis B
virus-related acute-on-chronic liver failure (ACHBLF) patients receiving
glucocorticoid therapy. Interleukin (IL)-6 is the main factor of
cytokine storm that occurs in ACHBLF. IL-6 and suppressors of cytokine
signaling 3 (SOCS3) may be associated with the prognosis of patients.
Peripheral blood mononuclear cells (PBMCs), which are related to the
immune response and inflammation of patients, are often used as
materials for searching for biomarkers to predict ACHBLF. We aimed to
determine serum cytokines levels and to evaluate the prediction value of
SOCS3 and IL-6 mRNA levels in PBMC for ACHBLF patients with
glucocorticoid therapy.MethodsEighty-five patients with ACHBLF, fifty
with pre-ACHBLF, fifty with CHB and thirty healthy controls (HCs) were
enrolled. ACHBLF and pre-ACHBLF patients (n=135) received standard
medical treatments (SMT) and 93 of them treated with glucocorticoids
(SMT+GC). Serum cytokines levels, SOCS3 and IL-6 mRNA levels in
peripheral blood mononuclear cells (PBMCs) were quantified.ResultsSerum
IL-6, IL-1ß, IL-18 and TNF-α levels of non-survivors with glucocorticoid
therapy were significantly higher than that of survivors. Serum IL-10
levels in the survivors with glucocorticoid therapy were significantly
increased. The analysis of variable importance shows that IL-6 ranks
first. SOCS3 and IL-6 mRNA levels in PBMCs were significantly higher in
ACHBLF patients than those in pre-ACHBLF patients (p=0.0158; p=0.0421).
Non-survivors showed significantly lower SOCS3 levels (p=0.0084) and
higher IL-6 levels (p&lt;0 .0001) than survivors in ACHBLF patients.
Non-survivors in patients receiving GC therapy had significantly lower
SOCS3 levels (p=0.0002) and higher IL-6 levels (p&lt;0 .0001) than
survivors.SOCS3 and IL-6 mRNA levels were predictors for 90-day
prognosis of patients receiving GC therapy (p=0.003; p=0.002). AUC of
MELD score+SOCS3+IL-6 was 0.887 (95%CI, 0.818-0.955) and the optimal
cut-off value was 0.29, with sensitivity of 93.3%, specificity of 74.6%.
Patients receiving GC therapy with MELD score+SOCS3+IL-6=0.29 had a
higher risk of poor prognosis (p&lt;0 .0001). Survival rate in the
SMT+GC group was higher than that in the SMT group
(p=0.0350).DiscussionGlucocorticoid treatment could reduce the mortality
of ACHBLF. SOCS3 and IL-6 mRNA combined with MELD score could be used to
predict for ACHBLF patients receiving glucocorticoid therapy.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1571443"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1571443</a></p>
<hr />
<h2
id="exploring-the-evolutionary-journey-of-the-lumpy-skin-disease-virus-through-the-phylogenetic-and-phylo-geo-network-analysis">Exploring
the evolutionary journey of the lumpy skin disease virus through the
phylogenetic and phylo-geo network analysis</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionLumpy Skin Disease Virus (LSDV), an emerging pathogen
from the Capripoxvirus genus, continues to challenge global livestock
health with its expanding host range and genetic adaptability.Materials
and methodsIn this study, we report the first isolation and whole genome
sequencing of LSDV from Bos frontalis, a semi-domesticated bovine
species native to Northeast India, along with the assembly of an isolate
from cattle.ResultsTime to the Most Recent Common Ancestor (TMRCA)
estimates support a relatively recent common origin for Indian strains,
pointing to ongoing virus circulation and regional adaptation. The
maximum likelihood phylogenetic tree of the whole genome and G
protein-coupled chemokine receptor (GPCR) gene further demonstrated the
clustering of global strains, emphasizing the virus’s transboundary
movement and genomic diversity. To strengthen phylogenetic inference, we
identified shared SNPs, synonymous and non-synonymous mutations across
the genome with a total of 2212 variants. Haplotype network and mutation
pattern analyses across global genomes further highlighted the
conservative evolution of Indian isolates within a distinct
haplogroup.DiscussionSeveral mutation events between haplogroups
highlight the virus’s continuous genetic diversification, which
correlates with known patterns of spread.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1575538"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1575538</a></p>
<hr />
<h2
id="metagenomics-reveals-unique-gut-mycobiome-biomarkers-in-major-depressive-disorder---a-non-invasive-method">Metagenomics
reveals unique gut mycobiome biomarkers in major depressive disorder - a
non-invasive method</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAn increasing amount of evidence suggests a potential link
between alterations in the intestinal microbiota and the onset of
various psychiatric disorders, including depression. Nevertheless, the
precise nature of the link between depression and the intestinal
microbiota remains largely unknown. A significant proportion of previous
research has concentrated on the study of gut bacterial communities,
with relatively little attention paid to the link between gut mycobiome
and depression.MethodsIn this research, we analyzed the composition and
differences of intestinal fungal communities between major depressive
disorder (MDD) and healthy controls. Subsequently, we constructed a
machine learning model using support vector machine-recursive feature
elimination to search for potential fungal markers for MDD.ResultsOur
findings indicated that the composition and beta diversity of intestinal
fungal communities were significantly changed in MDD compared to the
healthy controls. A total of 22 specific fungal community markers were
screened out by machine learning, and the predictive model had promising
performance in the prediction of MDD (area under the curve, AUC =
1.000). Additionally, the intestinal fungal communities demonstrated
satisfactory performance in the validation cohort, with an AUC of 0.884
(95% CI: 0.7871-0.9476) in the Russian validation cohort, which
consisted of 36 patients with MDD and 36 healthy individuals. The AUC
for the Wuhan validation cohort was 0.838 (95% CI: 0.7403-0.9102), which
included 40 patients with MDD and 42 healthy individuals.ConclusionTo
summarize, our research revealed the characterization of intestinal
fungal communities in MDD and developed a prediction model based on
specific intestinal fungal communities. Although MDD has
well-established diagnostic criteria, the strategy based on the model of
gut fungal communities may offer predictive biomarkers for MDD.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582522"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582522</a></p>
<hr />
<h2
id="microbiome-in-cancer-metastasis-biological-insights-and-emerging-spatial-omics-methods">Microbiome
in cancer metastasis: biological insights and emerging spatial omics
methods</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The role of the microbiome in cancer metastasis has emerged as a
critical area of research, with growing evidence suggesting that
microbial composition and interactions within the tumour
microenvironment may significantly influence metastatic progression.
This review explores the role of the microbiome in cancer metastasis, as
well as potential key bacteria and their mechanisms through which they
could impact tumour dissemination, seeding and growth. Biological models
used to study metastasis are discussed to provide context for the
further investigation of these interactions. In order to answer
unresolved questions regarding the microbiome’s involvement in
metastatic dissemination, recent advancements in spatial biology
techniques are examined, including spatial genomics, transcriptomics,
proteomics and metabolomics, which enable the spatial mapping of
microbial interactions within the tumour microenvironment. Additionally,
multimodal-omics imaging approaches are highlighted for their potential
to integrate multiple molecular layers, offering comprehensive insights
into the microbiome’s role in cancer metastasis. The review also
addresses the challenges and limitations of these techniques,
underscoring the complexity of studying microbiome-tumour interactions
and offering directions for future research to better explore and target
the microbiological landscape in metastatic cancer.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1559870"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1559870</a></p>
<hr />
<h2
id="salivary-microbiota-and-iga-responses-are-different-in-pre-diabetic-individuals-compared-to-normoglycemic-controls">Salivary
microbiota and IgA responses are different in pre-diabetic individuals
compared to normoglycemic controls</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionIn recent years, changes in the oral microbiota of
patients with type 2 diabetes mellitus (T2DM) have been increasingly
recognized. The salivary microbiota may also be altered in pre-diabetes,
which is the earliest stage of abnormal blood glucose regulation and a
reversible stage preceding T2DM; however, its characteristics are poorly
understood. Salivary immunoglobulin A (IgA) is a host defense factor
central to the oral immune system and may play an important role in
regulating the salivary microbiota. Given that alterations in
immunoreactivity are observed in pre-diabetes, we hypothesized that the
salivary IgA response may also be altered; however, limited knowledge
exists regarding this. Therefore, in the present study, we aimed to
evaluate the characteristics of salivary microbiota and IgA responses
against salivary microbiota in individuals with pre-diabetes, comparing
them to those in individuals with normoglycemia.MethodsSaliva samples
were collected from 101 pre-diabetic individuals (PreDM group) and 101
age- and sex-matched normoglycemic controls (Normal group). Further, 16S
rRNA metagenomic analysis was performed to compare bacterial microbiota
composition. For each of the 19 saliva samples from the PreDM and Normal
groups, IgA-enriched and IgA-nonenriched fractions were separated via
magnetic-activated cell sorting, followed by 16S rRNA metagenomic
analysis. The IgA index was calculated to evaluate the difference in the
IgA response to each bacterium between the PreDM and Normal
groups.ResultsBacterial species richness was significantly lower in the
PreDM group than in the Normal group (observed operational taxonomic
unit index, p = 0.042), and a difference between these groups was noted
in the overall salivary microbiota structure (unweighted UniFrac
distances, p = 0.009). Salivary IgA responses against several bacterial
genera differed between the PreDM and Normal groups. Significantly
higher IgA responses were noted against Haemophilus in the PreDM group,
with lower responses against Capnocytophaga, Corynebacterium, and
Streptococcus relative to those in the Normal group.ConclusionsSalivary
microbiota and IgA responses differ between pre-diabetic individuals and
normoglycemic controls. The current findings advance our understanding
of the interaction between oral bacteria and host immune responses in
patients with a poor glycemic status.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1591285"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1591285</a></p>
<hr />
<h2
id="establishment-and-epidemiological-investigation-of-a-dual-fluorescent-qpcr-assay-for-pasteurella-multocida-and-salmonella-in-yaks-in-the-tibetan-autonomous-prefecture-of-garzê-china">Establishment
and epidemiological investigation of a dual fluorescent qPCR assay for
Pasteurella multocida and Salmonella in yaks in the Tibetan Autonomous
Prefecture of Garzê, China</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionYaks serve as a vital economic and ecological resource in
high-altitude regions, but it faces significant health challenges from
various pathogens. Among these, Pasteurella multocida and Salmonella are
critical pathogens that contribute to severe diseases.MethodsA duplex
real-time fluorescence quantitative PCR assay was developed to
simultaneously detect Pasteurella multocida and Salmonella. The
species-specific genes kmt1 and invA were selected as target regions for
primer and probe design. Following rigorous optimization, a duplex assay
was established. Recombinant plasmids were constructed to serve as
standards for generating standard curves. The detection thresholds were
determined using SPSS statistical analysis and receiver operating
characteristic curve methods. Furthermore, the assay’s sensitivity,
specificity, stability, and clinical applicability were
evaluated.ResultsThe established assay demonstrated high sensitivity,
with detection limits of 100 and 10 copies for pMD-kmt1 and pMD-invA,
respectively. No cross-reactivity was observed with six pathogens,
including Mycoplasma bovis, infectious bovine rhinotracheitis virus and
others. The standard curves showed strong linearity, with coefficients
of determination of 0.995 and 0.998, and amplification efficiencies of
103.37% and 103.47% for pMD-kmt1 and pMD-invA, respectively. No
interference was observed between high- and low-concentration templates
during simultaneous detection. The intra- and inter-assay coefficients
of variation ranged from 0.23% to 1.51%. Detection thresholds were
determined to be cycle threshold values of 41.5 for P. multocida and
40.0 for Salmonella. Clinical evaluation was performed on 226 samples
collected from yaks in seven counties of Ganzi Prefecture, Sichuan
Province, China. The single infection rates of P. multocida and
Salmonella were 20.35% (46/226) and 38.50% (87/226), respectively, while
the co-infection rate was 6.19% (14/226).DiscussionThis study
successfully established a duplex real-time fluorescence PCR assay that
enables the simultaneous detection of P. multocida and Salmonella with
high sensitivity, specificity, and efficiency. The assay offers a
reliable and rapid diagnostic tool that is particularly suited for
clinical and epidemiological investigations in yak populations.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1599817"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1599817</a></p>
<hr />
<h2
id="from-transmission-to-adaptive-evolution-genomic-surveillance-of-getah-virus">From
transmission to adaptive evolution: genomic surveillance of Getah
virus</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Getah virus (GETV) is a member of the Alphavirus of the Togaviridae.
It is a single-stranded positive-RNA virus that is mainly transmitted by
mosquitoes. In recent years, the spread of GETV has become increasingly
serious, causing serious losses to the animal economy and posing a
potential threat to public health. GETV infected animals extend from
traditional domestic animals such as horses and pigs to cattle, foxes
and other animals. Especially in China, the virus has been detected in
many provinces in recent years. In addition, GETV-specific antibodies
were detected in healthy humans. However, the threat posed by GETV in
China has not received enough attention. In this study, we downloaded
all available GETV genome-wide serials (82 serials in total) from the
NCBI as of December 2023. We integrate multiple bioinformatics
approaches to understand the characteristics of GETV from the
perspectives of epidemiology, virus-host co-evolution, and viral
adaptation analysis. The results of this study show that GETV is rapidly
expanding its host range and geographical distribution at high
evolutionary rates due to the lack of commercially available vaccines.
Second, we clearly reveal the cross-species transmission of GETV.
Finally, we identified important adaptive and active selection sites.
GETV and its media are widely distributed in China, and new host
infections continue to appear. Therefore, strengthening surveillance and
prevention to avoid serious losses to the pandemic is an important task
we face today.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1513392"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1513392</a></p>
<hr />
<h2
id="comparison-of-bronchoalveolar-lavage-fluid-mngs-and-sputum-culture-in-the-clinical-application-of-severe-pneumonia">Comparison
of bronchoalveolar lavage fluid mNGS and sputum culture in the clinical
application of severe pneumonia</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe purpose of this study is to compare the clinical
application value of bronchoalveolar lavage fluid (BALF) mNGS and sputum
culture in severe pulmonary infections, and to provide guidance for
clinicians in selecting the appropriate testing method.MethodsThis study
collected 105 patients diagnosed with severe pneumonia, of which 55
patients who underwent BALF mNGS. We compared the pathogen detection
rates, length of stay and mortality rate, treatment, and pathogen
species between BALF mNGS group and sputum culture group.ResultsThe
pathogen detection rate in BALF mNGS group was significantly higher than
that in sputum culture group (P&lt;0.0001). The length of hospital stay
in the BALF mNGS group was shorter than that in the sputum group
(P=0.0093). There was no statistically significant difference in
mortality rate between the two groups (P=0.26). However, BALF mNGS group
had a lower antibiotic usage rate than the sputum culture group
(P=0.0491). According to the mNGS results, initial antipathogenic
treatment was modified in 67.27% of patients. In BALF mNGS group, the
main pathogens detected included Mycoplasma pneumoniae, Mycobacterium
tuberculosis (MTB) and Haemophilus influenzae (HI). The sputum culture
group mainly included: MTB, HI.ConclusionBALF mNGS effectively and
rapidly identifies pathogens, helping doctors quickly diagnose severe
pneumonia pathogens. Combined with the patient’s medical history,
laboratory results and imaging, clinical doctors can adjust the
patient’s treatment plan in time. This has potential advantages in
improving the cure rate of severe pneumonia patients, reducing the
length of hospital stay, and improving the prognosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593818"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593818</a></p>
<hr />
<h2
id="antibiotics-in-periodontal-treatment-an-umbrella-review">Antibiotics
in periodontal treatment: an umbrella review</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>AimAntimicrobial stewardship envisions the appropriate use of
antimicrobials, including antibiotics. Antibiotic therapy in
Periodontology has been widely investigated over the years. This
umbrella review aimed to appraise the methodological quality and
meta-analytical strength and validity of the evidence of systematic
reviews (SRs) on systemic and local antibiotics in periodontal
therapy.Material and methodsAfter registration of the protocol (PROSPERO
CRD42024527222), an extensive search, up to March 2024, for SRs that
have assessed the effect of antibiotics in periodontal therapy, either
nonsurgical and surgical, regardless of the types of patients and type
of antibiotic. The methodological quality of SRs was judged using A
MeaSurement Tool to Assess systematic Reviews 2. Fail-safe number of
Rosenberg explored the number of nonsignificant, unpublished, or missing
studies that would be required to change the direction of that
evidence.ResultsForty-four SRs, consisting of 221 meta-analyses, were
included. The overall methodological quality was low, with only four and
two SRs of high or moderate quality, respectively. Out of 221
meta-analyses, 69 indicated that the effect of systemic or local
antibiotics was statistically not significant. Twenty-nine meta-analyses
from suggestive-to-strong strength from one high and three low
methodological quality SRs indicated that the systemic or local
antibiotics had a beneficial, statistically significant effect on
periodontal health parameters, such as average clinical attachment loss,
bleeding on probing or percentage of pocket closure. Of those, four
strong evidence meta-analyses from a low-quality systematic review
indicated significant and meta-analytically robust but with negligible
effect. About 65.5% of the meta-analyses with suggestive to strong
evidence are unlikely to change with more future studies.ConclusionThere
is no robust evidence to support the use of antibiotics for periodontal
management. Systemic antibiotics have a minimal effect on periodontitis
and additional studies are unlikely to change the level of evidence.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1601464"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1601464</a></p>
<hr />
<h2
id="molecular-and-epidemiological-characterization-of-carbapenem-resistant-hypervirulent-klebsiella-pneumoniae-in-huaian-china-20222024-a-retrospective-study">Molecular
and epidemiological characterization of carbapenem-resistant
hypervirulent Klebsiella pneumoniae in Huaian, China (2022–2024): a
retrospective study</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesCarbapenem-resistant hypervirulent Klebsiella pneumoniae
(CR-hvKP) poses a significant public health challenge. This study
investigated the molecular epidemiology, antimicrobial resistance
patterns, clinical characteristics, and risk factors of CR-hvKP
infection in Huaian, China.MethodsWe retrospectively studied patients
infected with carbapenem-resistant K. pneumoniae (CRKP) between November
2022 and September 2024. Whole-genome sequencing was used to detect
carbapenemase, virulence, capsular serotype-related genes, and plasmid
types in 374 CRKP isolates.ResultsAmong them, 57.49% (215/374) strains
met the criteria for CR-hvKP. The most common type was
blaKPC-2-producing ST11(98.60%, 212/215), whereas K64 (56.74%, 122/215)
and KL25 (39.53%, 85/215) were the main capsular serotypes. The CR-hvKP
strains showed significantly higher resistance to the tested
antibiotics, except for ceftazidime/avibactam and colistin. Resistance
rates of CR-hvKP to the three tested antibiotics (minocycline,
cotrimoxazole, and amikacin) were higher than those of CRnon-hvKP.
Phylogenetic analysis based on whole-genome single-nucleotide
polymorphisms divided the 251 isolates into four independent branches,
with branch 2 being the most prevalent, indicating high clonality among
the strains. Multivariate analysis showed diabetes [odds ratio (OR) =
3.771] and surgery (OR =2.042) to be independent variables associated
with CR-hvKP infection.ConclusionsNotably, the ST11 lineage carrying
blaKPC-2 has emerged as a dominant high-risk clone in Huaian. Given the
wide distribution of these novel CR-hvKP isolates, global monitoring and
stricter control measures should be implemented to prevent their further
spread in hospital settings.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1569004"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1569004</a></p>
<hr />
<h2
id="efficacy-and-safety-of-saccharomyces-boulardii-cncm-i-745-for-the-treatment-of-pediatric-acute-diarrhea-in-china-a-systematic-review-and-meta-analysis">Efficacy
and safety of Saccharomyces boulardii CNCM I-745 for the treatment of
pediatric acute diarrhea in China: a systematic review and
meta-analysis</h2>
<p>date: 2025-06-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPediatric acute gastroenteritis (PAGE) is a common cause of
morbidity and mortality, especially in children under five years old.
Therapeutic strategies including probiotics have been investigated, but
trials from non-English speaking countries may not be easily
accessible.AimTo determine the efficacy of Saccharomyces boulardii
compared to controls for treating PAGE in children receiving standard
rehydration therapy in trials conducted in China.MethodsSystematic
review and meta-analysis using literature search with Google Scholar,
PubMed, Embase, Cochrane Library, China National Knowledge
Infrastructure and China Biology Medicine disc (from inception to June
30, 2024) of randomized, controlled trials comparing S. boulardii CNCM
I-745 to controls for the treatment of PAGE in children conducted in
China. Independent data extraction by two reviewers. Standard
meta-analysis methods were applied and random-effect or fixed-effects
models were used depending upon the degree of heterogeneity using
standardized mean differences for continuous data and relative risk
estimates for dichotomous outcomes. The risk of bias for each study was
determined and heterogeneity was measured by I2.ResultsOf 851 articles
screened, 10 RCTs (1125 participants) met the inclusion criteria, and
none were found in non-Chinese databases. S. boulardii CNCM I-745 was
found to significantly reduce the duration of PAGE (SMD=-1.63 days, 95%
CI -2.08, -1.18), improve the total effectiveness rating (RR=1.22, 95%
CI 1.16, 1.28) and significantly more participants were cured (RR=1.47,
95% CI 1.30, 1.67). The finding that S. boulardii significantly reduced
the levels of two pro-inflammatory cytokines (TNF-α and IL8) has not
been reported in previous meta-analyses of PAGE.ConclusionS. boulardii
CNCM I-745 is an effective treatment for PAGE and was well tolerated in
trials done in China.Systematic review
registrationwww.crd.york.ac.uk/PRSPERO, identifier CRD 42024567537.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587792"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587792</a></p>
<hr />
<h2
id="host-specific-vascular-endothelial-cell-responses-to-angiostrongylus-vasorum-a-comparative-in-vitro-study-in-red-foxes-vulpes-vulpes-and-domestic-dogs">Host-specific
vascular endothelial cell responses to Angiostrongylus vasorum: a
comparative in vitro study in red foxes (Vulpes vulpes) and domestic
dogs</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCanine angiostrongylosis, caused by Angiostrongylus
vasorum, affects dogs and red foxes, with dogs developing
cardiopulmonary and coagulation disorders, while foxes remain mostly
subclinical.MethodsThis study examined aortic endothelial cell responses
from both species to A. vasorum adult full somatic antigen extracts,
first-stage larval (L1) antigen, and adult excretory-secretory products
(ESP). Differential gene expression of interleukins (IL) -6, -10, and
-33, intercellular adhesion molecule 1 (ICAM-1), vascular adhesion
molecule 1 (VCAM-1), endothelial selectin (E-selectin), platelet
selectin (P-selectin), vascular endothelial growth factor (VEGF), and
monocyte chemoattractant protein 1 (MCP-1) was assessed via reverse
transcription quantitative PCR (RT qPCR) after four and 24 hours of
antigen exposure.ResultsFour hours post ESP stimulation, IL-10 increased
in dogs (1.8-fold) but decreased in foxes (0.4-fold). IL-33 declined in
both, (0.9-fold vs. 0.7-fold, respectively). VCAM-1 was upregulated more
in foxes (3.5-fold vs. 1.2 in dogs). Following adult antigen exposure,
P-selectin, ICAM-1, and VCAM-1 increased in fox more than in dog cells
(1.4, 1.7, and 3.1-fold vs. 0.9, 0.5, and 0.7-fold, respectively). L1
antigen downregulated IL-10 and MCP-1 in dogs (0.7 and 0.8-fold) but
upregulated them in foxes (2.1 and 1.1-fold). Twenty-four hours after
ESP stimulation, ICAM-1 decreased in dogs (0.8-fold) but increased in
foxes (1.4-fold). VCAM-1 was downregulated in dogs (0.6-fold) but
upregulated in foxes (12.9-fold). Adult antigen exposure upregulated
P-selectin in both species, more in foxes (4.8-fold) than in dogs
(1.9-fold). ICAM-1 was downregulated in dogs (0.8-fold) but upregulated
7.5-fold in foxes. L1 antigen stimulation caused the most substantial
differences between species: IL-6 was upregulated more in dogs
(4.7-fold) than foxes (1.2-fold). E-Selectin was upregulated in dogs
(12.8-fold) but downregulated in foxes (0.2-fold). P-selectin increased
more in dogs (10.0-fold) than in foxes (1.7-fold). ICAM-1 was
downregulated in dogs (0.6-fold) but upregulated in foxes (2.6-fold), as
was VCAM-1 (0.7-fold and 3.1-fold). VEGF was upregulated 9.5-fold in
dogs after adult antigen exposure, and 7.6-fold after L1 antigen
exposure, while it remained rather unchanged in foxes (0.9-fold and
1.0-fold, respectively).DiscussionThese findings corroborate that foxes
have developed mechanisms for a regulated immune response following A.
vasorum exposure, while dogs exhibit a higher pro-inflammatory reaction,
contributing to severe clinical outcomes. Host-parasite co-evolution may
explain differences in the pathogenesis and clinical presentation of
canid angiostrongylosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584663"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584663</a></p>
<hr />
<h2
id="intestinal-flora-metabolites-indole-3-butyric-acid-and-disodium-succinate-promote-inci2-mcr-1-carrying-plasmid-transfer">Intestinal
flora metabolites indole-3-butyric acid and disodium succinate promote
IncI2 mcr-1-carrying plasmid transfer</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionPlasmid-driven horizontal transfer of resistance genes in
bacterial communities is a major factor in the spread of resistance
worldwide. The gut microbiome, teeming with billions of microorganisms,
serves as a reservoir for resistance genes. The metabolites of gut
microorganisms strongly influence the physiology of their microbial
community, but the role of the metabolites in the transfer of resistance
genes remains unclear.MethodsA dual-fluorescence conjugation model was
established. We assessed the effects of different concentrations of
indole-3-butyric acid (IBA) and disodium succinate (DS) on plasmid
transfer using conjugation assays. The growth of bacteria (donors,
recipients, and transconjugants), the reactive oxygen species (ROS)
levels and membrane permeability were measured under IBA and DS
exposure. The plasmid copy number, and transcriptional levels of
conjugation-related genes (including the related genes of the regulation
of ROS production, the SOS response, cell membrane permeability, pilus
generation, ATP synthesis, and the type IV secretion system (T4SS) )
were evaluated by qPCR.ResultsIn this study, we demonstrated that IBA
and DS at low concentrations, which can also be ingested through diet,
enhance the interspecies transfer ratio of IncI2 mcr-1-carrying plasmid
in Escherichia coli. At 20 mg/L, the transfer ratios in the presence of
IBA or DS increased by 2.5- and 2.7-fold compared to that of the
control, respectively. Exposure to this concentration of IBA or DS
increased the production of reactive oxygen species (ROS), the SOS
response, cell membrane permeability, and plasmid copy number. The
transcription of genes of the related pathways and of pilus, ATP, and
the T4SS was upregulated. DiscussionOur findings revealed that low-dose
gut microbiota metabolites—particularly those with dietary
origins—promote plasmid-mediated resistance gene dissemination through
multifaceted mechanisms involving oxidative stress, SOS activation, and
conjugation machinery enhancement. This highlights potential public
health risks associated with microbiota metabolites, especially those
utilized in food production.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1564810"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1564810</a></p>
<hr />
<h2
id="microbiota-and-gastric-cancer-from-molecular-mechanisms-to-therapeutic-strategies">Microbiota
and gastric cancer: from molecular mechanisms to therapeutic
strategies</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Gastric cancer, a prevalent malignancy globally, is influenced by
various factors. The imbalance in the gut microbiome and the existence
of particular intratumoural microbiota could have a strong connection
with the onset and progression of gastric cancer. High-throughput
sequencing technology and bioinformatics analysis have revealed a close
correlation between abnormal abundance of specific microbial communities
and the risk of gastric cancer. These microbial communities contribute
to gastric cancer progression through mechanisms including increasing
cellular genomic damage, inhibiting DNA repair, activating abnormal
signaling pathways, exacerbating tumor hypoxia, and shaping a tumor
immune-suppressive microenvironment. This significantly impacts the
efficacy of gastric cancer treatments, including chemotherapy and
immunotherapy. Probiotic, prebiotic, antibiotic, carrier-based, dietary
interventions, fecal microbiota transplantation, and traditional Chinese
medicine show potential applications in gastric cancer treatment.
However, the molecular mechanisms regarding dysbiosis of microbiota,
including gut microbiota, and intra-tumoral microbiota during the
progression of gastric cancer, as well as the therapeutic efficacy of
microbiota-related applications, still require extensive exploration
through experiments.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563061"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563061</a></p>
<hr />
<h2
id="pathogen-detection-and-antibiotic-use-in-granulomatous-lobular-mastitis-a-comparison-of-mngs-and-culture">Pathogen
detection and antibiotic use in granulomatous lobular mastitis: a
comparison of mNGS and culture</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study aimed to evaluate the clinical microbial profile
of patients with granulomatous lobular mastitis (GLM) and compare
various detection methods to identify the most effective approach for
pathogen detection, which could help enhance clinical diagnosis and
treatment.MethodsWe retrospectively analyzed data from 84 patients
diagnosed with GLM, assessed the composition of pathogenic
microorganisms in these patients, and compared the effectiveness of
different sampling methods and detection
techniques.ResultsCorynebacterium kroppenstedtii (C. kroppenstedtii) was
identified as the predominant microorganism among GLM patients. The
positivity rate was low in skin swabs (10%) but similar in pus (40%) and
tissue samples (37%). After antibiotic treatment, the pathogen detection
rate of metagenomic next-generation sequencing (mNGS) (54.55%) was found
to be higher than that of culture-based methods (27.27%). Among the GLM
cases with pathogenic infection, although mNGS demonstrated higher
sensitivity (75.0%) than culture tests (50.0%), both methods exhibited
100.0% specificity. However, the time for obtaining results with mNGS
was significantly shorter (1.2 ± 0.41 days) compared to bacterial
culture (5.5 ± 0.64 days) (P &lt; 0.05).ConclusionsOur findings indicate
that pus was the most suitable sample type for microbial evidence
collection in patients with GLM. mNGS demonstrated superior performance
compared to culture in distinguishing infectious from non-infectious
cases, with reduced antibiotic interference, faster turnaround time, and
higher accuracy. Based on our single-center experience, empirical
cephalosporin treatment may be appropriate for these patients.
Additionally, surgical intervention remains the most efficient approach
for rapid and complete resolution.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1570776"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1570776</a></p>
<hr />
<h2
id="real-world-efficacy-and-safety-of-letermovir-versus-ganciclovir-prophylaxis-in-adolescent-patients-undergoing-allogenic-hematopoietic-stem-cell-transplantation-a-single-center-observational-study">Real-world
efficacy and safety of letermovir versus ganciclovir prophylaxis in
adolescent patients undergoing allogenic hematopoietic stem cell
transplantation: a single center observational study</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesTo compare the efficacy and safety of letermovir and
ganciclovir for cytomegalovirus (CMV) prophylaxis in adolescent patients
(aged 14-17 years) undergoing allogenic hematopoietic stem cell
transplantation (allo-HSCT).MethodsThis observational and single-center
study collected data from February 2023 and April 2024.ResultsThe
cumulative incidence of CMV DNAemia following HSCT was 44.4% in the
letermovir group (n=20) and 66.3% in the control group (n=32) receiving
ganciclovir. Notably, the cumulative incidence of clinically significant
CMV infection (csCMVi) was significantly reduced in the letermovir group
compared with control patients (11.0% vs 41.3%, p=0.021). Among patients
diagnosed with grades II-IV acute graft-versus-host disease (aGVHD), a
significantly lower proportion of individuals in the letermovir group
presented CMV DNAemia than in the control group (20.0% vs 73.3%,
p=0.013). The common adverse events observed in the letermovir group
were aGVHD (60.0%), diarrhea (25.0%), and nausea (15.0%). Leukopenia was
reported in only one patient, and did not necessitate an adjustment of
letermovir dosage.ConclusionsIn this single-center real-world study,
letermovir exhibited a favourable efficacy and safety profile for CMV
prophylaxis in adolescent patients undergoing HSCT. However, further
prospective multi-center studies are warranted to validate our
conclusion in adolescent patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1558637"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1558637</a></p>
<hr />
<h2
id="assessment-of-s100a8a9-and-resistin-as-predictive-biomarkers-for-mortality-in-critically-ill-patients-with-sepsis">Assessment
of S100A8/A9 and resistin as predictive biomarkers for mortality in
critically ill patients with sepsis</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionSepsis is associated with high mortality. Early
intervention is crucial to reducing sepsis-related mortality. This study
aims to assess the clinical potential of S100A8/A9 and resistin as novel
biomarkers for predicting mortality risk in sepsis patients.MethodSerum
samples were collected and analyzed from 141 adult sepsis patients
(discovery cohort), 43 non-sepsis intensive care units (ICU) patients,
15 healthy volunteers, and 55 sepsis patients along with 17 non-sepsis
ICU patients (validation cohort). The 28-day mortality and sequential
organ failure assessment (SOFA) scores of the participants were
compared. Additionally, the predictive ability of S100A8/A9 and resistin
for sepsis mortality was evaluated using the area under the receiver
operating characteristic curve at ICU admission.ResultsThe
concentrations of S100A8/A9 and resistin in sepsis patients were
noticeably increased relative to non-sepsis patients and healthy
controls. Serum S100A8/A9 concentrations in surviving sepsis patients
were significantly higher than in non-surviving patients. On the day of
admission, serum resistin concentrations in Gram-negative (G-) sepsis
patients were considerably elevated relative to Gram-positive (G+)
infected sepsis patients. Among sepsis patients admitted to the ICU, the
AUC for S100A8/A9 in predicting 28-day mortality was 0.617 (P = 0.032;
95% confidence bounds 0.513–0.721), and for SOFA was 0.750 (P &lt;
0.0001; 95% confidence bounds 0.660–0.840). Sepsis patients with high
serum S100A8/A9 concentrations (≥ 377.53 ng/mL) had a higher survival
rate relative to those with low concentrations (&lt;377.53 ng/mL). In
the validation cohort, the AUC for S100A8/A9 and 28-day mortality was
0.708 (P = 0.032; 95% confidence bounds 0.563–0.854), and for SOFA was
0.698 (P = 0.025; 95% confidence bounds 0.550–0.845). Additionally,
sepsis patients with high serum S100A8/A9 concentrations (≥ 377.53
ng/mL) also had a higher survival rate relative to those with lower
concentrations (&lt; 377.53 ng/mL). Furthermore, serum resistin levels
in patients with a normal phenotype and mixed phenotype with
hyperinflammation were predictive of mortality, with an AUC of 0.810 (P
= 0.034; 95% confidence bounds 0.605–1.00) and 0.708 (P = 0.015; 95%
confidence bounds 0.571–0.846). In patients with a normal sepsis
phenotype, those with high serum resistin levels (≥ 63.695 ng/mL) had a
lower survival rate compared to those with low resistin levels (&lt;
63.695 ng/mL). In contrast, in patients with a mixed phenotype with
hyperinflammation, those with high serum resistin levels (≥ 107.64
ng/mL) had a higher survival rate compared to those with lower resistin
levels (&lt; 107.64 ng/mL).DiscussionSepsis, the leading cause of death
in intensive care unit patients. Identifying reliable biomarkers is
essential for improving both the diagnosis and treatment of sepsis. We
found that serum S100A8/A9 concentration at ICU admission is a
significant predictor of 28-day mortality risk in sepsis patients.
Additionally, resistin levels at ICU admission play an important role in
predicting 28-day mortality risk in patients with both normal and mixed
phenotypes with hyperinflammation. These findings suggest that S100A8/A9
and resistin could serve as effective biomarkers. Moreover, these
findings could guide early clinical decisions in the treatment of sepsis
patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1555307"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1555307</a></p>
<hr />
<h2
id="detection-of-β-lactam-resistance-genes-in-gram-negative-bacteria-from-positive-blood-cultures-using-a-microchip-based-molecular-assay">Detection
of β-lactam resistance genes in Gram-negative bacteria from positive
blood cultures using a microchip-based molecular assay</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAccurate detection of β-lactam resistance genes in
bloodstream infections is critical for guiding antimicrobial therapy.
This study evaluates the Alifax Gram-negative resistance (GNR) microchip
assay for detecting β-lactam resistance genes directly from positive
blood cultures (PBCs) for Gram-negative (GN) bacteria, including
Enterobacterales, Pseudomonas aeruginosa, and Acinetobacter
baumannii.MethodsSimulated (n=146) and clinical (n=106) GN-PBC samples
were tested for blaKPC, blaVIM, blaNDM, blaIMP, blaOXA-23-like,
blaOXA-48-like, blaSHV-ESBL, blaCTX-M-1/9 group, and blaCMY-2-like genes
using the GNR microchip assay. Whole-genome sequencing (WGS) served as
the reference assay for simulated samples and, selectively, for clinical
samples. The bioMérieux BioFire Blood Culture Identification 2 (BCID2)
panel assay was used as a comparator for clinical samples.ResultsThe GNR
microchip assay correctly identified 203 (99.5%) of 204 β-lactam
resistance genes in simulated samples. One sample tested false negative
for a blaSHV-ESBL gene but true positive for a blaKPC gene. In clinical
samples, GNR results were concordant with BCID2 for 113 (100%) of 113
genes included in both assays. Additionally, the GNR assay detected
blaCMY-2-like (n=6), blaOXA-23-like (n=5), and blaSHV-ESBL (n=2), which
are not targeted by BCID2, all confirmed by WGS. In two
β-lactam-resistant P. aeruginosa samples but negative by the GNR assay,
WGS confirmed the absence of acquired β-lactam resistance genes,
suggesting alternative resistance mechanisms.ConclusionThe GNR microchip
assay demonstrated high concordance and broader β-lactam resistance gene
coverage compared to BCID2, supporting its potential role in routine
diagnostics. Further validation in larger, prospective studies is
warranted.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1597700"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1597700</a></p>
<hr />
<h2
id="based-on-quorum-sensing-reverse-effect-of-traditional-chinese-medicine-on-bacterial-drug-resistance-mechanism">Based
on quorum sensing: reverse effect of traditional Chinese medicine on
bacterial drug resistance mechanism</h2>
<p>date: 2025-06-03, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Antimicrobial resistance has emerged as a critical global health
challenge requiring urgent multidisciplinary interventions. Pathogenic
bacteria utilize six principal resistance mechanisms: (1) Enzymatic
degradation of antibiotics via the production of inactivating enzymes;
(2) Inactivation of antibiotics by changing the drug targets; (3)
Reduction of antibiotics entry by decreasing bacterial permeability; (4)
Enhanced antibiotics efflux through overexpression of efflux pumps; (5)
Acquisition of antibiotics resistance via genetic mutations; (6)
Development of antibiotics resistance through formation of microbial
biofilms. Notably, these resistance determinants demonstrate close
coordination through quorum sensing, collectively establishing
recalcitrant infections that defy conventional therapies. Emerging
evidence confirms the therapeutic potential of traditional Chinese
medicine in combating antimicrobial resistance. Traditional Chinese
medicine can be used as quorum sensing inhibitors to interfere with the
quorum sensing of bacteria, thereby achieving antibacterial effects.
Moreover, traditional Chinese medicine has the characteristics of rich
components, long history, mild action and no drug resistance, which
makes it stand out in the research against drug-resistant bacterial
infections. This paper systematically describes six mechanisms of
bacterial resistance and reviews the antagonistic effects of traditional
Chinese medicine against these mechanisms based on quorum sensing. It
highlights that the active ingredients, extracts and compound
formulations of traditional Chinese medicine have good reversal effects
on bacterial antibiotic resistance, which can effectively treat
drug-resistant bacterial infections. When combined with antibiotics,
traditional Chinese medicine not only reduces antibiotics dosage but
also adverse reactions, holding promise for improving and addressing
clinical challenges posed by bacterial resistance. This article further
discusses the impact of different delivery methods on the anti-bacterial
biofilms efficacy of traditional Chinese medicine. It introduces the
main delivery methods of traditional Chinese medicine at present and the
new delivery methods under research, pointing out the huge development
potential in the research of traditional Chinese medicine dosage forms.
Additionally, the deficiencies and improvement methods of the current
research were pointed out, and prospects for future related research
were put forward.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582003"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582003</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2025-06-02, from: People’s CDC</p>
<p>This is the People’s CDC bi-weekly update for June 2, 2025. This
Weather Report from the People’s CDC shares the latest news on the COVID
situation and other public health crises in the United States.</p>
<p><br></p>
<p><a
href="https://peoplescdc.org/2025/06/02/peoples-cdc-covid-19-weather-report-99/"
class="uri">https://peoplescdc.org/2025/06/02/peoples-cdc-covid-19-weather-report-99/</a></p>
<hr />
<h2
id="synergistic-treatment-of-sodium-propionate-and-sishen-pill-for-diarrhea-mice-with-kidney-yang-deficiency-syndrome">Synergistic
treatment of sodium propionate and Sishen Pill for diarrhea mice with
kidney-yang deficiency syndrome</h2>
<p>date: 2025-06-02, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study aimed to investigate the therapeutic effects of
sodium propionate in combination with Sishen Pill in the treatment of
diarrhea with kidney-yang deficiency syndrome in mice, with a focus on
its influence on intestinal microbiota, enzyme activity, and associated
therapeutic outcomes.MethodsA model of diarrhea with kidney-yang
deficiency syndrome was established via adenine combined with Folium
sennae administration. The model group was randomly assigned to the
following treatment groups: natural recovery group, 100% Sishen Pill
group, 75% Sishen Pill + 60 mg/kg sodium propionate group, 50% Sishen
Pill + 120 mg/kg sodium propionate group, 25% Sishen Pill + 240 mg/kg
sodium propionate group, and 480 mg/kg sodium propionate group. A
variety of parameters, including general symptoms, body weight, rectal
temperature, intestinal microbiota composition, microbial activity, and
enzyme activity, were assessed.ResultsCompared with natural recovery
group, the 480 mg/kg sodium propionate group presented significant
improvements in mental state, anal temperature, fecal water content, the
thymus index, and Bifidobacterium counts (p &lt; 0.01). Compared with
those in normal group, the fecal water content and Escherichia coli
counts in the 100% Sishen Pill group were significantly different (p
&lt; 0.01), but no significant differences were observed compared with
those in natural recovery group (p &gt; 0.05). The 75% Sishen Pill + 60
mg/kg sodium propionate group showed improvements in mental state, food
and water intake, body weight, rectal temperature, fecal water content,
spleen, and thymus index, Bifidobacterium counts, total bacterial count,
E. coli count, microbial activity, and lactase activity, which were
close to normal levels, and significant differences were observed when
compared to the natural recovery group (p &lt; 0.01). The 50% Sishen
Pill + 120 mg/kg sodium propionate group and 25% Sishen Pill + 240 mg/kg
sodium propionate group also exhibited significant differences in mental
status, microbial activity, and Lactobacillus count relative to those of
normal group (p &lt; 0.01). Furthermore, the 50% Sishen Pill + 120 mg/kg
sodium propionate group presented significant changes in fecal water
content (p &lt; 0.01), whereas 25% Sishen Pill + 240 mg/kg sodium
propionate group presented significant differences in the spleen index,
total bacterial count, E. coli count, protease activity, lactase
activity, and xylanase activity compared with those of normal group (p
&lt; 0.01).ConclusionThe 75% Sishen Pill + 60 mg/kg sodium propionate
can improve the symptoms of kidney-yang deficiency syndrome, promote the
growth and development of mice, inhibit excessive bacterial
proliferation, support the growth of beneficial bacteria, and enhance
intestinal enzyme activity. Its effects are superior to the use of
sodium propionate or Sishen Pill alone. These results suggest that this
therapeutic ratio may optimize the efficacy of Sishen Pill in the
treatment of diarrhea with kidney-yang deficiency syndrome. However,
further research is necessary to confirm whether this combination
represents the most effective treatment regimen for this condition in
mice.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1608271"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1608271</a></p>
<hr />
<h2
id="editorial-evolution-molecular-mechanisms-and-the-strategies-to-combat-antimicrobial-resistance-amr-a-one-health-approach">Editorial:
Evolution, molecular mechanisms and the strategies to combat
antimicrobial resistance (AMR): a One Health approach</h2>
<p>date: 2025-06-02, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582613"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582613</a></p>
<hr />
<h2
id="immune-escape-of-staphylococcus-aureus-mediated-by-osteocyte-lacuna-canalicular-network-leads-to-persistent-and-uncured-bone-infection">Immune
escape of Staphylococcus aureus mediated by osteocyte lacuna-canalicular
network leads to persistent and uncured bone infection</h2>
<p>date: 2025-06-02, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Bone infections, specifically chronic osteomyelitis, are
characterized by recurrent episodes. They are considered intractable
clinical diseases as they require protracted and difficult-to-cure
courses. Staphylococcus aureus (S. aureus) is the most common pathogen
responsible for bone infections and has high destruction rates. Previous
literature has indicated that during S. aureus osteomyelitis, immune
evasion mainly involves three mechanisms: biofilm formation,
intracellular infection, and abscess formation. However, recently, it
was observed that S. aureus can enter and persist for a long time in the
Osteocyte lacuno-canalicular network (OLCN), a bone microstructure.
Furthermore, it has been found to successfully evade the host’s immune
system via natural physical barriers, chemical properties, and bone
microstructure’s immune escape mechanisms. Therefore, S. aureus bone
infections are more difficult to cure than soft-tissue infections.
Currently, there are only a few studies on OLCN invasion by S. aureus,
and the clinical evidence is not sufficient. Therefore, this review
aimed to combine relevant published literature on the OLCN-mediated
immune escape of S. aureus to elaborate on the pathological mechanisms
associated with protracted and difficult-to-cure bone infections. The
findings will provide a scientific basis and theoretical foundation for
future comprehensive analysis of how S. aureus invades OLCN and novel
treatment strategies for bone infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1592086"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1592086</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
