<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.12</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="parks.html" title="News about Parks">Parks</a></li>
	<li><a href="tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="2023/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.12">health 2024.12</h1>
<p>(date: 2024-03-21 17:10:36)</p>
<hr />
<h2
id="genomic-insights-into-the-evolution-pathogenicity-and-extensively-drug-resistance-of-emerging-pathogens-kluyvera-and-phytobacter">Genomic
insights into the evolution, pathogenicity, and extensively
drug-resistance of emerging pathogens Kluyvera and Phytobacter</h2>
<p>date: 2024-03-21, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Kluyvera</italic> is a Gram-negative, flagellated, motile
bacillus within the <italic>Enterobacteriaceae</italic>. The case
reports of clinical infections shed light on the importance of this
organism as an emerging opportunistic pathogen. The genus
<italic>Phytobacter</italic>, which often be misidentified with
<italic>Kluyvera</italic>, is also an important clinically relevant
member of the <italic>Enterobacteriaceae</italic>. However, the
identification of <italic>Kluyvera</italic> and
<italic>Phytobacter</italic> is problematic, and their phylogenetic
relationship remains unclear.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Here, 81 strains of <italic>Kluyvera</italic> and 16 strains of
<italic>Phytobacter</italic> were collected. A series of comparative
genomics approaches were applied to the phylogenetic relationship
reconstruction, virulence related genes profiles description, and
antibiotic resistance genes prediction.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Using average nucleotide identity (ANI) and <italic>in silico</italic>
DNA-DNA hybridization (<italic>is</italic>DDH), we offered reliable
species designations of 97 strains, in which 40 (41.24%) strains were
incorrectly labeled. A new <italic>Phytobacter</italic> genomospecies-1
were defined. <italic>Phytobacter</italic> and <italic>Kluyvera</italic>
show great genome plasticity and inclusiveness, which may be related to
their diverse ecological niches. An intergenomic distances threshold of
0.15875 was used for taxonomy reassignments at the phylogenomic-group
level. Further principal coordinates analysis (PCoA) revealed 11 core
genes of <italic>Kluyvera</italic> (<italic>pelX, mdtL, bglC, pcak-1,
uhpB, ddpA-2, pdxY, oppD-1, cptA, yidZ, csbX</italic>) that could be
served as potential identification targets. Meanwhile, the
<italic>Phytobacter</italic> specific virulence genes <italic>clbS,
csgA-C, fliS, hsiB1_vipA and hsiC1_vipB</italic>, were found to
differentiate from <italic>Kluyvera</italic>. We concluded that the
evolution rate of <italic>Kluyvera</italic> was 5.25E-6, approximately
three times higher than that of <italic>Phytobacter</italic>.
Additionally, the co-existence of ESBLs and carbapenem resistance genes
were present in approximately 40% strains, suggesting the potential
development of extensively drug-resistant or even fully drug-resistant
strains.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
This work provided a better understanding of the differences between
closely related species <italic>Kluyvera</italic> and
<italic>Phytobacter</italic>. Their genomes exhibited great genome
plasticity and inclusiveness. They not only possess a potential
pathogenicity threat, but also a risk of multi-drug resistance. The
emerging pathogens <italic>Kluyvera</italic> and
<italic>Phytobacter</italic> warrant close attention.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1376289"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1376289</a></p>
<hr />
<h2
id="inhibition-of-malaria-and-babesiosis-parasites-by-putative-red-blood-cell-targeting-small-molecules">Inhibition
of malaria and babesiosis parasites by putative red blood cell targeting
small molecules</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Chemotherapies for malaria and babesiosis frequently succumb to the
emergence of pathogen-related drug-resistance. Host-targeted therapies
are thought to be less susceptible to resistance but are seldom
considered for treatment of these diseases.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Our overall objective was to systematically assess small molecules for
host cell-targeting activity to restrict proliferation of intracellular
parasites. We carried out a literature survey to identify small
molecules annotated for host factors implicated in <italic>Plasmodium
falciparum</italic> infection. Alongside <italic>P. falciparum</italic>,
we implemented <italic>in vitro</italic> parasite susceptibility assays
also in the zoonotic parasite <italic>Plasmodium knowlesi</italic> and
the veterinary parasite <italic>Babesia divergens</italic>. We
additionally carried out assays to test directly for action on RBCs
apart from the parasites. To distinguish specific host-targeting
antiparasitic activity from erythrotoxicity, we measured
phosphatidylserine exposure and hemolysis stimulated by small molecules
in uninfected RBCs.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We identified diverse RBC target-annotated inhibitors with
<italic>Plasmodium</italic>-specific, <italic>Babesia-</italic>specific,
and broad-spectrum antiparasitic activity. The anticancer MEK-targeting
drug trametinib is shown here to act with submicromolar activity to
block proliferation of <italic>Plasmodium</italic> spp. in RBCs. Some
inhibitors exhibit antimalarial activity with transient exposure to RBCs
prior to infection with parasites, providing evidence for host-targeting
activity distinct from direct inhibition of the parasite.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
We report here characterization of small molecules for antiproliferative
and host cell-targeting activity for malaria and babesiosis parasites.
This resource is relevant for assessment of physiological RBC-parasite
interactions and may inform drug development and repurposing efforts.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1304839"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1304839</a></p>
<hr />
<h2
id="isolation-and-identification-of-a-novel-phage-targeting-clinical-multidrug-resistant-corynebacterium-striatum-isolates">Isolation
and identification of a novel phage targeting clinical
multidrug-resistant Corynebacterium striatum isolates</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Over the past decade, Corynebacterium striatum (C. striatum), an
emerging multidrug-resistant (MDR) pathogen, has significantly
challenged healthcare settings, especially those involving individuals
with weakened immune systems. The rise of these superbugs necessitates
innovative solutions.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This study aimed to isolate and characterize bacteriophages targeting
MDR-C. striatum. Utilizing 54 MDR-C. striatum isolates from a local
hospital as target strains, samples were collected from restroom puddles
for phage screening. Dot Plaque and Double-layer plate Assays were
employed for screening.
</p>
</sec><sec>
<title>
Results
</title>
<p>
A novel temperate bacteriophage, named CSP1, was identified through a
series of procedures, including purification, genome extraction,
sequencing, and one-step growth curves. CSP1 possesses a 39,752 base
pair circular double-stranded DNA genome with HK97-like structural
proteins and potential for site-specific recombination. It represents a
new species within the unclassified Caudoviricetes class, as supported
by transmission electron microscopy, genomic evolutionary analysis, and
collinearity studies. Notably, CSP1 infected and lysed 21 clinical
MDR-C. striatum isolates, demonstrating a wide host range. The phage
remained stable in conditions ranging from -40 to 55°C, pH 4 to 12, and
in 0.9% NaCl buffer, showing no cytotoxicity.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
The identification of CSP1 as the first phage targeting clinical C.
striatum strains opens new possibilities in bacteriophage therapy
research, and the development of diagnostic and therapeutic tools
against pathogenic bacteria.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361045"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361045</a></p>
<hr />
<h2
id="virus-induced-host-genomic-remodeling-dysregulates-gene-expression-triggering-tumorigenesis">Virus-induced
host genomic remodeling dysregulates gene expression, triggering
tumorigenesis</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Virus-induced genomic remodeling and altered gene expression contribute
significantly to cancer development. Some oncogenic viruses such as
Human papillomavirus (HPV) specifically trigger certain cancers by
integrating into the host’s DNA, disrupting gene regulation linked to
cell growth and migration. The effect can be through direct integration
of viral genomes into the host genome or through indirect modulation of
host cell pathways/proteins by viral proteins. Viral proteins also
disrupt key cellular processes like apoptosis and DNA repair by
interacting with host molecules, affecting signaling pathways. These
disruptions lead to mutation accumulation and tumorigenesis. This review
focuses on recent studies exploring virus-mediated genomic structure,
altered gene expression, and epigenetic modifications in tumorigenesis.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1359766"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1359766</a></p>
<hr />
<h2
id="cell-immunity-to-sars-cov-2-after-natural-infection-andor-different-vaccination-regimens">Cell
immunity to SARS-CoV-2 after natural infection and/or different
vaccination regimens</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
The aim of the study was to evaluate the humoral and cellular immunity
after SARS-CoV-2 infection and/or vaccination according to the type of
vaccine, number of doses and combination of vaccines.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Volunteer subjects were sampled between September 2021 and July 2022 in
Hospital Clínico San Carlos, Madrid (Spain). Participants had different
immunological status against SARS-CoV-2: vaccinated and unvaccinated,
with or without previous COVID-19 infection, including healthy and
immunocompromised individuals. Determination of IgG against the spike
protein S1 subunit receptor-binding domain (RBD) was performed by
chemiluminescence microparticle immunoassay (CMIA) using the Architect
i10000sr platform (Abbott). The SARS-CoV-2-specific T-cell responses
were assessed by quantification of interferon gamma release using
QuantiFERON SARS-CoV-2 assay (Qiagen).
</p>
</sec><sec>
<title>
Results
</title>
<p>
A total of 181 samples were collected, 170 were from vaccinated
individuals and 11 from unvaccinated. Among the participants, 41 were
aware of having previously been infected by SARS-CoV-2. Vaccinated
people received one or two doses of the following vaccines against
SARS-CoV-2: ChAdOx1-S (University of Oxford—AstraZeneca) (AZ)
and/orBNT162b2 (Pfizer—BioNTech)(PZ). Subjects immunized with a
third-booster dose received PZ or mRNA-1273 (Moderna—NIAID)(MD)
vaccines. All vaccinees developed a positive humoral response (&gt;7.1
BAU/ml), but the cellular response varied depending on the vaccination
regimen. Only AZ/PZ combination and 3 doses of vaccination elicited a
positive cellular response (median concentration of IFN- γ &gt; 0.3
IU/ml). Regarding a two-dose vaccination regimen, AZ/PZ combination
induced the highest humoral and cellular immunity. A booster with mRNA
vaccine resulted in increases in median levels of IgG-Spike antibodies
and IFN-γ as compared to those of two-dose of any vaccine. Humoral and
cellular immunity levels were significantly higher in participants with
previous infection compared to those without infection.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Heterologous vaccination (AZ/PZ) elicited the strongest immunity among
the two-dose vaccination regimens. The immunity offered by the
third-booster dose of SARS-CoV-2 vaccine depends not only on the type of
vaccine administered but also on previous doses and prior infection.
Previous exposure to SARS-CoV-2 antigens by infection strongly affect
immunity of vaccinated individuals.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1370859"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1370859</a></p>
<hr />
<h2
id="trail-and-ip-10-dynamics-in-pregnant-women-post-covid-19-vaccination-associations-with-neutralizing-antibody-potency">TRAIL
and IP-10 dynamics in pregnant women post COVID-19 vaccination:
associations with neutralizing antibody potency</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
The aim of this study is to investigate changes in TNF-related
apoptosis-inducing ligand (TRAIL) and gamma interferon-induced protein
10 (IP-10) after COVID-19 vaccination in pregnant women and to explore
their association with neutralizing antibody (Nab) inhibition.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
The study evaluated 93 pregnant women who had previously received two
(n=21), three (n=55) or four (n=17) doses of COVID-19 vaccine. Also we
evaluated maternal blood samples that were collected during childbirth.
The levels of TRAIL, IP-10 and Nab inhibition were measured using
enzyme-linked immunosorbent assays (ELISA).
</p>
</sec><sec>
<title>
Results and discussion
</title>
<p>
Our study revealed four-dose group resulted in lower TRAIL levels when
compared to the two-dose and three-dose groups (4.78 vs. 16.07 vs. 21.61
pg/ml, p = 0.014). The two-dose group had reduced IP-10 levels than the
three-dose cohort (111.49 vs. 147.89 pg/ml, p=0.013), with no
significant variation compared to the four-dose group. In addition, the
four-dose group showed stronger Nab inhibition against specific strains
(BA.2 and BA.5) than the three-dose group. A positive correlation was
observed between TRAIL and IP-10 in the two-dose group, while this
relationship was not found in other dose groups or between TRAIL/IP-10
and Nab inhibition. As the doses of the COVID-19 vaccine increase, the
levels of TRAIL and IP-10 generally increase, only by the fourth dose,
the group previously vaccinated with AZD1222 showed lower TRAIL but
higher IP-10. Despite these changes, more doses of the vaccine
consistently reinforced Nab inhibition, apparently without any relation
to TRAIL and IP-10 levels. The variation may indicate the induction of
immunological memory in vaccinated mothers, which justifies further
research in the future.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1358967"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1358967</a></p>
<hr />
<h2
id="a-long-term-cohort-study-the-immune-evasion-and-decreasing-neutralization-dominated-the-sars-cov-2-breakthrough-infection">A
long-term cohort study: the immune evasion and decreasing neutralization
dominated the SARS-CoV-2 breakthrough infection</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Most of vaccinees and COVID-19 convalescents can build effective
anti-SARS-CoV-2 humoral immunity, which helps preventing infection and
alleviating symptoms. However, breakthrough viral infections caused by
emerging SARS-CoV-2 variants, especially Omicron subvariants, still pose
a serious threat to global health. By monitoring the viral infections
and the sera neutralization ability of a long-tracked cohort, we found
out that the immune evasion of emerging Omicron subvariants and the
decreasing neutralization led to the mini-wave of SARS-CoV-2
breakthrough infections. Meanwhile, no significant difference had been
found in the infectivity of tested SARS-CoV-2 variants, even though the
affinity between human angiotensin-converting enzyme 2 (hACE2) and
receptor-binding domain (RBDs) of tested variants showed an increasing
trend. Notably, the immune imprinting of inactivated COVID-19 vaccine
can be relieved by infections of BA.5.2 and XBB.1.5 variants
sequentially. Our data reveal the rising reinfection risk of immune
evasion variants like Omicron JN.1 in China, suggesting the importance
of booster with updated vaccines.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1381877"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1381877</a></p>
<hr />
<h2
id="clinical-distribution-of-carbapenem-genotypes-and-resistance-to-ceftazidime-avibactam-in-enterobacteriaceae-bacteria">Clinical
distribution of carbapenem genotypes and resistance to
ceftazidime-avibactam in Enterobacteriaceae bacteria</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Bacterial resistance is a major threat to public health worldwide. To
gain an understanding of the clinical infection distribution, drug
resistance information, and genotype of CRE in Dongguan, China, as well
as the resistance of relevant genotypes to CAZ-AVI, this research aims
to improve drug resistance monitoring information in Dongguan and
provide a reliable basis for the clinical control and treatment of CRE
infection.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
VITEK-2 Compact automatic analyzer was utilized to identify 516 strains
of CRE collected from January 2017 to June 2023. To determine drug
sensitivity, the K-B method, E-test, and MIC methods were used. From
June 2022 to June 2023, 80 CRE strains were selected, and GeneXpert
Carba-R was used to detect and identify the genotype of the
carbapenemase present in the collected CRE strains. An in-depth analysis
was conducted on the CAZ-AVI <italic>in vitro</italic> drug sensitivity
activity of various genotypes of CRE, and the results were statistically
evaluated using SPSS 23.0 and WHONET 5.6 software.
</p>
</sec><sec>
<title>
Results
</title>
<p>
This study identified 516 CRE strains, with the majority (70.16%) being
<italic>K.pneumoniae</italic>, followed by <italic>E.coli</italic>
(18.99%). Respiratory specimens had highest detection rate with 53.77%
identified, whereas urine specimens had the second highest detection
rate with 17.99%. From June 2022 to June 2023, 95% of the strains tested
using the CRE GeneXpert Carba-R assay possessed carbapenemase genes, of
which 32.5% were <italic>bla<sub>NDM</sub></italic> strains and 61.25%
<italic>bla<sub>KPC</sub></italic> strains. The results showed that CRE
strains containing <italic>bla<sub>KPC</sub></italic> had a
significantly higher rate of resistance to amikacin, cefepime, and
aztreonam than those harboring <italic>bla<sub>NDM</sub></italic>.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
The CRE strains isolated from Dongguan region demonstrated a high
resistance rate to various antibiotics used in clinical practice but a
low resistance rate to tigecycline. These strains produce Class A serine
carbapenemases and Class B metals β-lactamases, with the majority of
them carrying <italic>bla<sub>NDM</sub></italic> and
<italic>bla<sub>KPC</sub></italic>. Notably, CRE strains with
<italic>bla<sub>KPC</sub></italic> and
<italic>bla<sub>NDM</sub></italic> had significantly lower resistance
rates to tigecycline. CAZ-AVI showed a good sensitivity rate with no
resistance to CRE strains carrying <italic>bla<sub>KPC</sub></italic>.
Therefore, CAZ-AVI and tigecycline should be used as a guide for
rational use of antibiotics in clinical practice to effectively treat
CRE.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345935"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1345935</a></p>
<hr />
<h2
id="il-10-and-tgf-β1-may-weaken-the-efficacy-of-preoperative-anti-tuberculosis-therapy-in-older-patients-with-spinal-tuberculosis">IL-10
and TGF-β1 may weaken the efficacy of preoperative anti-tuberculosis
therapy in older patients with spinal tuberculosis</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Spinal tuberculosis is a common extrapulmonary type that is often
secondary to pulmonary or systemic infections. <italic>Mycobacterium
tuberculosis</italic> infection often leads to the balance of immune
control and bacterial persistence. In this study, 64 patients were
enrolled and the clinicopathological and immunological characteristics
of different age groups were analyzed. Anatomically, spinal tuberculosis
in each group mostly occurred in the thoracic and lumbar vertebrae.
Imaging before preoperative anti-tuberculosis therapy showed that the
proportion of abscesses in the older group was significantly lower than
that in the younger and middle-aged groups. However, pathological
examination of surgical specimens showed that the proportion of
abscesses in the older group was significantly higher than that in the
other groups, and there was no difference in the granulomatous
inflammation, caseous necrosis, inflammatory necrosis, acute
inflammation, exudation, granulation tissue formation, and fibrous
tissue hyperplasia. B cell number was significantly lower in the
middle-aged and older groups compared to the younger group, while the
number of T cells, CD4<sup>+</sup> T cells, CD8<sup>+</sup> T cells,
macrophages, lymphocytes, plasma cells, and NK cells did not differ.
Meaningfully, we found that the proportion of IL-10 high expression and
TGF-β1 positive in the older group was significantly higher than that in
the younger group. TNF-α, CD66b, IFN-γ, and IL-6 expressions were not
different among the three groups. In conclusion, there are some
differences in imaging, pathological, and immune features of spinal
tuberculosis in different age groups. The high expression of IL-10 and
TGF-β1 in older patients may weaken their anti-tuberculosis immunity and
treatment effectiveness.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361326"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361326</a></p>
<hr />
<h2
id="helicobacter-pylori-glycan-biosynthesis-modulates-host-immune-cell-recognition-and-response">Helicobacter
pylori glycan biosynthesis modulates host immune cell recognition and
response</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
The pathogenic bacterium <italic>Helicobacter pylori</italic> has
evolved glycan-mediated mechanisms to evade host immune defenses. This
study tests the hypothesis that genetic disruption of <italic>H.
pylori</italic> glycan biosynthesis alters immune recognition and
response by human gastric epithelial cells and monocyte-derived
dendritic cells.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
To test this hypothesis, human cell lines were challenged with wildtype
<italic>H. pylori</italic> alongside an array of <italic>H.
pylori</italic> glycosylation mutants. The relative levels of immune
response were measured via immature dendritic cell maturation and
cytokine secretion.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Our findings indicate that disruption of lipopolysaccharide biosynthesis
diminishes gastric cytokine production, without disrupting dendritic
cell recognition and activation. In contrast, variable immune responses
were observed in protein glycosylation mutants which prompted us to test
the hypothesis that phase variation plays a role in regulating bacterial
cell surface glycosylation and subsequent immune recognition. Lewis
antigen presentation does not correlate with extent of immune response,
while the extent of lipopolysaccharide O-antigen elaboration does.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
The outcomes of this study demonstrate that <italic>H. pylori</italic>
glycans modulate the host immune response. This work provides a
foundation to pursue immune-based tailoring of bacterial glycans towards
modulating immunogenicity of microbial pathogens.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1377077"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1377077</a></p>
<hr />
<h2
id="correlation-of-caecal-microbiome-endotoxins-genes-and-intestinal-immune-cells-in-eimeria-tenella-infection-based-on-bioinformatics">Correlation
of caecal microbiome endotoxins genes and intestinal immune cells in
Eimeria tenella infection based on bioinformatics</h2>
<p>date: 2024-03-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
The infection with <italic>Eimeria tenella</italic> (ET) can elicit
expression of various intestinal immune cells, incite inflammation,
disrupt intestinal homeostasis, and facilitate co-infection with diverse
bacteria. However, the reciprocal interaction between intestinal immune
cells and intestinal flora in the progression of ET-infection remains
unclear.
</p>
</sec><sec>
<title>
Objective
</title>
<p>
The aim of this study was to investigate the correlation between cecal
microbial endotoxin (CME)-related genes and intestinal immunity in
ET-infection, with subsequent identification of hub potential biomarker
and immunotherapy target.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Differential expression genes (DEGs) within ET-infection and hub genes
related to CME were identified through GSE39602 dataset based on
bioinformatic methods and Protein-protein interaction (PPI) network
analysis. Moreover, immune infiltration was analyzed by CIBERSORT
method. Subsequently, comprehensive functional enrichment analyses
employing Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway
analysis along with Gene Ontology (GO), gene set enrichment analysis
(GSEA), and gene set variation analysis (GSVA) were performed.
</p>
</sec><sec>
<title>
Results
</title>
<p>
A total of 1089 DEGs and 25 hub genes were identified and CXCR4 was
ultimately identified as a essential CME related potential biomarker and
immunotherapy target in the ET-infection. Furthermore, activated natural
killer cells, M0 macrophages, M2 macrophages, and T regulatory cells
were identified as expressed intestinal immune cells. The functional
enrichment analysis revealed that both DEGs and hub genes were
significantly enriched in immune-related signaling pathways.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
CXCR4 was identified as a pivotal CME-related potential biomarker and
immunotherapy target for expression of intestinal immune cells during
ET-infection. These findings have significant implications in
elucidating the intricate interplay among ET-infection, CME, and
intestinal immunity.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382160"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382160</a></p>
<hr />
<h2
id="towards-modelling-tick-virus-interactions-using-the-weakly-pathogenic-sindbis-virus-evidence-that-ticks-are-competent-vectors">Towards
modelling tick-virus interactions using the weakly pathogenic Sindbis
virus: Evidence that ticks are competent vectors</h2>
<p>date: 2024-03-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Most tick-borne viruses (TBVs) are highly pathogenic and require high
biosecurity, which severely limits their study. We found that Sindbis
virus (SINV), predominantly transmitted by mosquitoes, can replicate in
ticks and be subsequently transmitted, with the potential to serve as a
model for studying tick-virus interactions. We found that both larval
and nymphal stages of <italic>Rhipicephalus haemaphysaloides</italic>
can be infected with SINV-wild-type (WT) when feeding on infected mice.
SINV replicated in two species of ticks (<italic>R.
haemaphysaloides</italic> and <italic>Hyalomma asiaticum</italic>) after
infecting them by microinjection. Injection of ticks with SINV
expressing enhanced Green Fluorescent Protein (eGFP) revealed that
SINV-eGFP specifically aggregated in the tick midguts for replication.
During blood-feeding, SINV-eGFP migrated from the midguts to the
salivary glands and was transmitted to a new host. SINV infection caused
changes in expression levels of tick genes related to immune responses,
substance transport and metabolism, cell growth and death. SINV mainly
induced autophagy during the early stage of infection; with increasing
time of infection, the level of autophagy decreased, while the level of
apoptosis increased. During the early stages of infection, the
transcript levels of immune-related genes were significantly
upregulated, and then decreased. In addition, SINV induced changes in
the transcription levels of some functional genes that play important
roles in the interactions between ticks and tick-borne pathogens. These
results confirm that the SINV-based transmission model between ticks,
viruses, and mammals can be widely used to unravel the interactions
between ticks and viruses.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334351"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334351</a></p>
<hr />
<h2
id="vaccine-development-obligate-intracellular-bacteria-new-tools-old-pathogens-the-current-state-of-vaccines-against-obligate-intracellular-bacteria">Vaccine
development: obligate intracellular bacteria new tools, old pathogens:
the current state of vaccines against obligate intracellular
bacteria</h2>
<p>date: 2024-03-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Obligate intracellular bacteria have remained those for which effective
vaccines are unavailable, mostly because protection does not solely rely
on an antibody response. Effective antibody-based vaccines, however,
have been developed against extracellular bacteria pathogens or toxins.
Additionally, obligate intracellular bacteria have evolved many
mechanisms to subvert the immune response, making vaccine development
complex. Much of what we know about protective immunity for these
pathogens has been determined using infection-resolved cases and animal
models that mimic disease. These studies have laid the groundwork for
antigen discovery, which, combined with recent advances in vaccinology,
should allow for the development of safe and efficacious vaccines.
Successful vaccines against obligate intracellular bacteria should
elicit potent T cell memory responses, in addition to humoral responses.
Furthermore, they ought to be designed to specifically induce strong
cytotoxic CD8+ T cell responses for protective immunity. This review
will describe what we know about the potentially protective immune
responses to this group of bacteria. Additionally, we will argue that
the novel delivery platforms used during the Sars-CoV-2 pandemic should
be excellent candidates to produce protective immunity once antigens are
discovered. We will then look more specifically into the vaccine
development for <italic>Rickettsiaceae</italic>, <italic>Coxiella
burnetti</italic>, and <italic>Anaplasmataceae</italic> from infancy
until today. We have not included <italic>Chlamydia trachomatis</italic>
in this review because of the many vaccine related reviews that have
been written in recent years.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1282183"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1282183</a></p>
<hr />
<h2
id="combination-of-metagenomic-next-generation-sequencing-and-conventional-tests-unraveled-pathogen-profiles-in-infected-patients-undergoing-allogeneic-hematopoietic-stem-cell-transplantation-in-jilin-province-of-china">Combination
of metagenomic next-generation sequencing and conventional tests
unraveled pathogen profiles in infected patients undergoing allogeneic
hematopoietic stem cell transplantation in Jilin Province of China</h2>
<p>date: 2024-03-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Infection is the main cause of death for patients after allogeneic
hematopoietic stem cell transplantation (HSCT). However, pathogen
profiles still have not been reported in detail due to their
heterogeneity caused by geographic region.
</p>
</sec><sec>
<title>
Objective
</title>
<p>
To evaluate the performance of metagenomic next-generation sequencing
(mNGS) and summarize regional pathogen profiles of infected patients
after HSCT.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
From February 2021 to August 2022, 64 patients, admitted to the
Department of Hematology of The First Hospital of Jilin University for
HSCT and diagnosed as suspected infections, were retrospectively
enrolled.
</p>
</sec><sec>
<title>
Results
</title>
<p>
A total of 38 patients were diagnosed as having infections, including
bloodstream (<italic>n</italic> =17), pulmonary (<italic>n</italic>
=16), central nervous system (CNS) (<italic>n</italic> =4), and chest
(<italic>n</italic> =1) infections. Human betaherpesvirus 5 (CMV) was
the most common pathogen in both bloodstream (<italic>n</italic> =10)
and pulmonary (<italic>n</italic> =8) infections, while CNS
(<italic>n</italic> =2) and chest (<italic>n</italic> =1) infections
were mainly caused by Human gammaherpesvirus 4 (EBV). For bloodstream
infection, <italic>Mycobacterium tuberculosis</italic> complex
(<italic>n</italic> =3), <italic>Staphylococcus epidermidis</italic>
(<italic>n</italic> =1), and <italic>Candida tropicalis</italic>
(<italic>n</italic> =1) were also diagnosed as causative pathogens.
Furthermore, mNGS combined with conventional tests can identify more
causative pathogens with high sensitivity of 82.9% (95% CI 70.4-95.3%),
and the total coincidence rate can reach up to 76.7% (95% CI
64.1-89.4%).
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
Our findings emphasized the importance of mNGS in diagnosing, managing,
and ruling out infections, and an era of more rapid, independent, and
impartial diagnosis of infections after HSCT can be expected.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378112"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378112</a></p>
<hr />
<h2
id="a-case-report-of-vancomycin-in-the-treatment-of-q-fever-endocarditis">A
case report of Vancomycin in the treatment of Q fever endocarditis</h2>
<p>date: 2024-03-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
The patient, a 43-year-old male, was admitted to the hospital with
gradually aggravated exertional palpitations and chest tightness over a
2-day period. Upon hospital admission, a cardiac ultrasound revealed
aortic valve redundancy, however multiple blood culture investigations
came back negative. Blood mNGS was perfected, revealing <italic>Coxiella
burnetii</italic>, and the diagnosis of Q fever (query fever) was
established. The temperature and inflammatory indices of the patient
were all normal with the treatment of vancomycin before cardiac surgery.
But for the potential liver damage of and the <italic>Coxiella
burnetii</italic> was still positive in the anti-phase II IgG titer, the
doxycycline and hydroxychloroquine instead of vancomycin were applied
for the patient. Despite receiving standardized anti-infective therapy
of doxycycline combined with hydroxychloroquine, this patient had fever
and increased leukocytes following surgery. After the addition of
vancomycin as an anti-infective treatment, the temperature and
leukocytes improved quickly. During the treatment of vancomycin, a
discovery of liver injury may have resulted. These findings provide new
therapy options for future professionals.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323054"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323054</a></p>
<hr />
<h2
id="emergence-of-pandrug-resistant-carbapenemase-producing-enterobacterales-in-dogs-and-cats-a-cross-sectional-study-in-egypt">Emergence
of pandrug-resistant carbapenemase-producing Enterobacterales in dogs
and cats: a cross-sectional study in Egypt</h2>
<p>date: 2024-03-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
One of the most important emerging health problems is the increasing
role of animals in the rapid global rise in resistance to last-resort
antibiotics, such as carbapenems. However, there is limited information
on the role of pet animals in harboring and spreading pandrug-resistant
(PDR) carbapenemase-producing <italic>Enterobacterales</italic> (CPE),
especially in Egypt. This cross-sectional study was conducted to screen
for CPE in healthy and diseased pets using phenotypic and molecular
methods and the NG-Test CARBA 5 immunochromatographic assay. Rectal
swabs were collected from 62 dogs and 48 cats, incubated overnight in
tryptic soy broth containing 10 μg of meropenem disc and subsequently
cultured on MacConkey agar supplemented with meropenem (1 mg/L).
Sixty-six isolates (60.6%), including 56 <italic>Klebsiella
pneumoniae</italic>, seven <italic>Escherichia coli</italic>, and three
<italic>K. oxytoca</italic> isolates, were confirmed to be
carbapenem-resistant <italic>Enterobacterales</italic> (CRE) by the disc
diffusion method, broth microdilution test, CNPt-direct, and PCR assay
targeting carbapenemase genes. Forty-three (65.2%) dogs and 23 (34.8%)
cats carried CPE. Of these, 35 (70.0%) were healthy (including 27 dogs
and 8 cats) and 31 (52.5%) were diseased (including 16 dogs and 15
cats). <italic>bla</italic><sub>OXA-181</sub> was the most common gene
detected (42/66, 63.6%), followed by <italic>bla</italic><sub>IMP</sub>
(40/66, 60.6%), <italic>bla</italic><sub>OXA−48−like</sub> (29/66,
43.9%), <italic>bla</italic><sub>KPC</sub> and
<italic>bla</italic><sub>VIM</sub> (20/66, 30.3% each), and
<italic>bla</italic><sub>NDM</sub> (17/66, 25.8%). The identified
genotypes were <italic>bla</italic><sub>KPC-2</sub>,
<italic>bla</italic><sub>IMP-1</sub>,
<italic>bla</italic><sub>VIM-1</sub>,
<italic>bla</italic><sub>NDM-1</sub>, and
<italic>bla</italic><sub>NDM-5</sub>. The CARBA 5 assay showed higher
sensitivity and specificity for the detection of NDM, OXA and KPC than
that for VIM and IMP genes. Antimicrobial resistance profiles of CRE
isolates revealed 20 PDR, 30 extensively drug-resistant (XDR), and 16
multidrug-resistant (MDR) phenotypes. This study provides evidence of
colonization with PDR CPE in dogs and cats. To manage the infection or
colonization of pets in veterinary clinical settings, extended
surveillance systems should be considered, and the use of critical
antibiotics should be strictly controlled.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1318585"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1318585</a></p>
<hr />
<h2
id="modulatory-actions-of-echinococcus-granulosus-antigen-b-on-macrophage-inflammatory-activation">Modulatory
actions of Echinococcus granulosus antigen B on macrophage inflammatory
activation</h2>
<p>date: 2024-03-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Cestodes use own lipid-binding proteins to capture and transport
hydrophobic ligands, including lipids that they cannot synthesise as
fatty acids and cholesterol. In <italic>E. granulosus</italic> s.l., one
of these lipoproteins is antigen B (EgAgB), codified by a multigenic and
polymorphic family that gives rise to five gene products (EgAgB8/1-5
subunits) assembled as a 230 kDa macromolecule. EgAgB has a diagnostic
value for cystic echinococcosis, but its putative role in the
immunobiology of this infection is still poorly understood. Accumulating
research suggests that EgAgB has immunomodulatory properties, but
previous studies employed denatured antigen preparations that might
exert different effects than the native form, thereby limiting data
interpretation. This work analysed the modulatory actions on macrophages
of native EgAgB (nEgAgB) and the recombinant form of EgAg8/1, which is
the most abundant subunit in the larva and was expressed in insect S2
cells (rEgAgB8/1). Both EgAgB preparations were purified to homogeneity
by immunoaffinity chromatography using a novel nanobody anti-EgAgB8/1.
nEgAgB and rEgAgB8/1 exhibited differences in size and lipid
composition. The rEgAgB8/1 generates mildly larger lipoproteins with a
less diverse lipid composition than nEgAgB. Assays using human and
murine macrophages showed that both nEgAgB and rEgAgB8/1 interfered with
<italic>in vitro</italic> LPS-driven macrophage activation, decreasing
cytokine (IL-1β, IL-6, IL-12p40, IFN-β) secretion and ·NO generation.
Furthermore, nEgAgB and rEgAgB8/1 modulated <italic>in vivo</italic>
LPS-induced cytokine production (IL-6, IL-10) and activation of large
(measured as MHC-II level) and small (measured as CD86 and CD40 levels)
macrophages in the peritoneum, although rEgAgB8/1 effects were less
robust. Overall, this work reinforced the notion that EgAgB is an
immunomodulatory component of <italic>E. granulosus</italic> s.l.
Although nEgAgB lipid’s effects cannot be ruled out, our data suggest
that the EgAgB8/1 subunit contributes to EgAgB´s ability to regulate the
inflammatory activation of macrophages.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1362765"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1362765</a></p>
<hr />
<h2
id="detection-and-quantification-of-natural-wolbachia-in-aedes-aegypti-in-metropolitan-manila-philippines-using-locally-designed-primers">Detection
and quantification of natural Wolbachia in Aedes aegypti in Metropolitan
Manila, Philippines using locally designed primers</h2>
<p>date: 2024-03-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
The Philippines bears health and economic burden caused by high dengue
cases annually. Presently, the Philippines still lack an effective and
sustainable vector management. The use of <italic>Wolbachia</italic>, a
maternally transmitted bacterium, that mitigate arbovirus transmission
has been recommended. Cytoplasmic incompatibility and viral blocking,
two characteristics that make <italic>Wolbachia</italic> suitable for
vector control, depend on infection prevalence and density. There are no
current <italic>Wolbachia</italic> release programs in the Philippines,
and studies regarding the safety of this intervention. Here, we screened
for <italic>Wolbachia</italic> in <italic>Aedes aegypti</italic>
collected from Metropolitan Manila, Philippines. We designed
location-specific primers for qPCR to test whether this improved
<italic>Wolbachia</italic> detection in <italic>Ae. aegypti.</italic> We
explored if host sex and <italic>Wolbachia</italic> strain could be
potential factors affecting <italic>Wolbachia</italic> density.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
<italic>Ae. aegypti</italic> mosquitoes (n=429) were screened for
natural <italic>Wolbachia</italic> by taqman qPCR using
location-specific <italic>Wolbachia</italic> surface protein primers
(<italic>wsp</italic>AAML) and known <italic>16S rRNA</italic> primers.
Samples positive for <italic>wsp</italic>AAML (n=267) were processed for
Sanger sequencing. We constructed a phylogenetic tree using IQ-TREE 2 to
further characterize <italic>Wolbachia</italic> present in the
Philippine <italic>Ae. aegypti</italic>. We then compared
<italic>Wolbachia</italic> densities between <italic>Wolbachia</italic>
groups and host sex. Statistical analyses were done using GraphPad Prism
9.0.
</p>
</sec><sec>
<title>
Results
</title>
<p>
<italic>Wolbachia</italic> prevalence for 16S rRNA (40%) and
<italic>wsp</italic>AAML (62%) markers were high.
<italic>Wolbachia</italic> relative densities for 16S rRNA ranged from
−3.84 to 2.71 and <italic>wsp</italic>AAML from −4.02 to 1.81. Densities
were higher in male than female mosquitoes. <italic>Wolbachia</italic>
strains detected in <italic>Ae. aegypti</italic> clustered into
supergroup B. Some 54% (123/226) of these sequences clustered under a
group referred to here as “wAegML,” that belongs to the supergroup B,
which had a significantly lower density than
wAegB/<italic>w</italic>AlbB, and <italic>w</italic>AlbA strains.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Location-specific primers improved detection of natural
<italic>Wolbachia</italic> in <italic>Ae. aegypti</italic> and allowed
for relative quantification. <italic>Wolbachia</italic> density is
relatively low, and differed between host sexes and
<italic>Wolbachia</italic> strains. An economical way of confirming
sporadic or transient <italic>Wolbachia</italic> in <italic>Ae.
aegypti</italic> is necessary while considering host sex and bacterial
strain.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360438"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360438</a></p>
<hr />
<h2
id="thrombocytopenia-in-klebsiella-pneumoniae-liver-abscess-a-retrospective-study-on-its-correlation-with-disease-severity-and-potential-causes">Thrombocytopenia
in Klebsiella pneumoniae liver abscess: a retrospective study on its
correlation with disease severity and potential causes</h2>
<p>date: 2024-03-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Objective
</title>
<p>
Thrombocytopenia is commonly associated with infectious diseases and
serves as an indicator of disease severity. However, reports on its
manifestation in conjunction with <italic>Klebsiella pneumoniae</italic>
liver abscess (KPLA) are scarce. The present study sought to elucidate
the correlation between thrombocytopenia and KPLA severity and delve
into the etiological factors contributing to the incidence of
thrombocytopenia.
</p>
</sec><sec>
<title>
Materials and methods
</title>
<p>
A retrospective analysis of the clinical data from patients with KPLA
admitted between June 2012 and June 2023 was performed. Baseline
characteristics, biochemical assessments, therapeutic interventions,
complications, and clinical outcomes were compared between patients with
and without thrombocytopenia. To investigate the potential etiologies
underlying thrombocytopenia, the association between platelet count
reduction and thrombophlebitis was examined, with a particular focus on
platelet consumption. Furthermore, bone marrow aspiration results were
evaluated to assess platelet production anomalies.
</p>
</sec><sec>
<title>
Results
</title>
<p>
A total of 361 KPLA patients were included in the study, among whom 60
(17%) had concurrent thrombocytopenia. Those in the thrombocytopenia
group exhibited significantly higher rates of thrombophlebitis (p =
0.042), extrahepatic metastatic infection (p = 0.01), septic shock (p =
0.024), admissions to the intensive care unit (p = 0.002), and
in-hospital mortality (p = 0.045). Multivariate analysis revealed that
thrombocytopenia (odds ratio, 2.125; 95% confidence interval,
1.114–4.056; p = 0.022) was independently associated with
thrombophlebitis. Among the thrombocytopenic patients, eight underwent
bone marrow aspiration, and six (75%) had impaired medullar platelet
production. After treatment, 88.6% of thrombocytopenic patients (n = 47)
demonstrated recovery in their platelet counts with a median recovery
time of five days (interquartile range, 3–6 days).
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
Thrombocytopenia in patients with KPLA is indicative of increased
disease severity. The underlying etiologies for thrombocytopenia may
include impaired platelet production within the bone marrow and
augmented peripheral platelet consumption as evidenced by the presence
of thrombophlebitis.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351607"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351607</a></p>
<hr />
<h2
id="association-between-gut-microbiota-and-pan-dermatological-diseases-a-bidirectional-mendelian-randomization-research">Association
between gut microbiota and pan-dermatological diseases: a bidirectional
Mendelian randomization research</h2>
<p>date: 2024-03-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Gut microbiota has been associated with dermatological problems in
earlier observational studies. However, it is unclear whether gut
microbiota has a causal function in dermatological diseases.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Thirteen dermatological diseases were the subject of bidirectional
Mendelian randomization (MR) research aimed at identifying potential
causal links between gut microbiota and these diseases. Summary
statistics for the Genome-Wide Association Study (GWAS) of gut
microbiota and dermatological diseases were obtained from public
datasets. With the goal of evaluating the causal estimates, five
acknowledged MR approaches were utilized along with multiple testing
corrections, with inverse variance weighted (IVW) regression serving as
the main methodology. Regarding the taxa that were causally linked with
dermatological diseases in the forward MR analysis, reverse MR was
performed. A series of sensitivity analyses were conducted to test the
robustness of the causal estimates.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The combined results of the five MR methods and sensitivity analysis
showed 94 suggestive and five significant causal relationships. In
particular, the <italic>genus Eubacterium_fissicatena_group</italic>
increased the risk of developing psoriasis vulgaris (odds ratio [OR] =
1.32, p<sub>FDR</sub> = 4.36 × 10<sup>−3</sup>), <italic>family
Bacteroidaceae</italic> (OR = 2.25, p<sub>FDR</sub> = 4.39 ×
10<sup>−3</sup>), <italic>genus Allisonella</italic> (OR = 1.42,
p<sub>FDR</sub> = 1.29 × 10<sup>−2</sup>), and <italic>genus
Bacteroides</italic> (OR = 2.25, p<sub>FDR</sub> = 1.29 ×
10<sup>−2</sup>) increased the risk of developing acne; and the
<italic>genus Intestinibacter</italic> increased the risk of urticaria
(OR = 1.30, p<sub>FDR</sub> = 9.13 × 10<sup>−3</sup>). A reverse MR
study revealed insufficient evidence for a significant causal
relationship. In addition, there was no discernible horizontal
pleiotropy or heterogeneity.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
This study provides novel insights into the causality of gut microbiota
in dermatological diseases and therapeutic or preventive paradigms for
cutaneous conditions.
</p>
<p></sec></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1327083"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1327083</a></p>
<hr />
<h2
id="deciphering-the-role-of-female-reproductive-tract-microbiome-in-reproductive-health-a-review">Deciphering
the role of female reproductive tract microbiome in reproductive health:
a review</h2>
<p>date: 2024-03-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Relevant studies increasingly indicate that female reproductive health
is confronted with substantial challenges. Emerging research has
revealed that the microbiome interacts with the anatomy, histology, and
immunity of the female reproductive tract, which are the cornerstone of
maintaining female reproductive health and preventing adverse pregnancy
outcomes. Currently, the precise mechanisms underlying their interaction
and impact on physiological functions of the reproductive tract remain
elusive, constituting a prominent area of investigation within the field
of female reproductive tract microecology. From this new perspective, we
explore the mechanisms of interactions between the microbiome and the
anatomy, histology, and immunity of the female reproductive tract,
factors that affect the composition of the microbiome in the female
reproductive tract, as well as personalized medicine approaches in
managing female reproductive tract health based on the microbiome. This
study highlights the pivotal role of the female reproductive tract
microbiome in maintaining reproductive health and influencing the
occurrence of reproductive tract diseases. These findings support the
exploration of innovative approaches for the prevention, monitoring and
treatment of female reproductive tract diseases based on the microbiome.
</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351540"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351540</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
