<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.24</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.24">health 2024.24</h1>
<p>(date: 2024-06-10 09:22:54)</p>
<hr />
<h2
id="clinical-features-of-pediatric-mucormycosis-role-of-metagenomic-next-generation-sequencing-in-diagnosis">Clinical
features of pediatric mucormycosis: role of metagenomic next generation
sequencing in diagnosis</h2>
<p>date: 2024-06-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMucormycosis is an uncommon invasive fungal infection that
has a high mortality rate in patients with severe underlying diseases,
which leads to immunosuppression. Due to its rarity, determining the
incidence and optimal treatment methods for mucormycosis in children is
challenging. Metagenomic next-generation sequencing (mNGS) is a rapid,
precise and sensitive method for pathogen detection, which helps in the
early diagnosis and intervention of mucormycosis in children. In order
to increase pediatricians’ understanding of this disease, we conducted a
study on the clinical features of mucormycosis in children and assessed
the role of mNGS in its diagnosis.MethodsWe retrospectively summarized
the clinical data of 14 children with mucormycosis treated at the First
Affiliated Hospital of Zhengzhou University from January 2020 to
September 2023.ResultsOf the 14 cases, 11 case of mucormycosis were
classified as probable, and 3 cases were proven as mucormycosis. Most
children (85.71%) had high-risk factors for mucormycosis. All 14
children had lung involvement, with 5 cases of extrapulmonary
dissemination. Among the 14 cases, 4 cases underwent histopathological
examination of mediastinum, lung tissue or kidney tissue, in which
fungal pathogens were identified in 3 patients. Fungal hyphae was
identified in 3 cases of mucormycosis, but only 1 case yielded a
positive culture result. All patients underwent mNGS testing with
samples from blood (8/14), bronchoalveolar lavage fluid (6/14), and
tissue (1/14). mNGS detected fungi in all cases: 7 cases had Rhizomucor
pusillus, 4 cases had Rhizopus oryzae, 3 cases had Rhizopus microsporus,
1 case had Lichtheimia ramosa, and 1 case had Rhizomucor miehei.
Coinfections were found with Aspergillus in 3 cases, bacteria in 3
cases, and viruses in 5 cases.ConclusionChildren with mucormycosis
commonly exhibit non-specific symptoms like fever and cough during the
initial stages. Early diagnosis based on clinical symptoms and imaging
is crucial in children suspected of having mucormycosis. mNGS, as a
supplementary diagnostic method, offers greater sensitivity and shorter
detection time compared to traditional mucormycosis culture or
histopathological testing. Additionally, mNGS enables simultaneous
detection of bacteria and viruses, facilitating timely and appropriate
administration of antibiotics and thereby enhancing patient
outcomes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368165"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1368165</a></p>
<hr />
<h2
id="staphylococcus-aureus-persistence-in-osteocytes-weathering-the-storm-of-antibiotics-and-autophagyxenophagy">Staphylococcus
aureus persistence in osteocytes: weathering the storm of antibiotics
and autophagy/xenophagy</h2>
<p>date: 2024-06-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Staphylococcus aureus is a major causative pathogen of osteomyelitis.
Intracellular infections of resident bone cells including osteocytes can
persist despite gold-standard clinical intervention. The mechanisms by
which intracellular S. aureus evades antibiotic therapy are unknown. In
this study, we utilised an in vitro S. aureus infection model of human
osteocytes to investigate whether antibiotic-mediated dysregulation of
autophagy contributes to this phenomenon. Infected or non-infected
osteocyte-like cells were exposed to combinations of rifampicin,
vancomycin, and modulators of autophagy. Intracellular bacterial growth
characteristics were assessed using colony-forming unit (CFU) analysis,
viable bacterial DNA abundance, and the rate of escape into
antibiotic-free medium, together with measures of autophagic flux.
Rifampicin, alone or in combination with vancomycin, caused a rapid
decrease in the culturability of intracellular bacteria, concomitant
with stable or increased absolute bacterial DNA levels. Both antibiotics
significantly inhibited autophagic flux. However, modulation of
autophagic flux did not affect viable bacterial DNA levels. In summary,
autophagy was shown to be a factor in the host–pathogen relationship in
this model, as its modulation affected the growth state of intracellular
S. aureus with respect to both their culturability and propensity to
escape the intracellular niche. While rifampicin and vancomycin
treatments moderately suppressed autophagic flux acutely, this did not
explain the paradoxical response of antibiotic treatment in decreasing
S. aureus culturability whilst failing to clear bacterial DNA and hence
intracellular bacterial load. Thus, off-target effects of rifampicin and
vancomycin on autophagic flux in osteocyte-like cells could not explain
the persistent S. aureus infection in these cells.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1403289"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1403289</a></p>
<hr />
<h2
id="broad-synergistic-antiviral-efficacy-between-a-novel-elite-controller-derived-dipeptide-and-antiretrovirals-against-drug-resistant-hiv-1">Broad
synergistic antiviral efficacy between a novel elite controller-derived
dipeptide and antiretrovirals against drug-resistant HIV-1</h2>
<p>date: 2024-06-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe naturally occurring dipeptide Tryptophylglycine (WG)
is enhanced in human immunodeficiency virus (HIV-1) infected Elite
Controllers (EC). We have shown that this dipeptide has an anti-HIV-1
effect and evaluated now its synergistic antiretroviral activity, in
combination with current antiretrovirals against multi-drug resistant
HIV-1 isolates.MethodsDrug selectivity assay with WG-am and ARVs agains
HIV-1 resistant isolates were carried out. Subsequently, two methods,
Chou-Talalay’s Combination Index (CI) and ZIP synergy score (SS), were
used to quantify the synergism.ResultsWG-am had a moderate/strong
synergism with the four tested antiretrovirals: raltegravir, tenofovir,
efavirenz, darunavir. WG-am:TDF had strong synergism at ED50, ED75, ED90
(CI: &lt;0.2) in isolates resistant to protease inhibitors or integrase
strand inhibitors (INSTI), and a slightly less synergism in isolates
resistant to non-nucleoside or nucleotide reverse transcriptase
inhibitors. WG-am combined with each of the four drugs inhibited all
drug-resistant isolates with over 95% reduction at maximum concentration
tested. The highest selectivity indexes (CC50/ED50) were in
INSTI-resistant isolates.ConclusionOur data suggest that WG, identified
as occurring and enhanced in Elite Controllers has a potential to become
a future treatment option in patients with HIV-1 strains resistant to
any of the four major categories of anti-HIV-1 compounds.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334126"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1334126</a></p>
<hr />
<h2 id="biological-features-of-fowl-adenovirus-serotype-4">Biological
features of fowl adenovirus serotype-4</h2>
<p>date: 2024-06-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Fowl adenovirus serotype 4 (FAdV-4) is highly pathogenic to broilers
aged 3 to 5 weeks and has caused considerable economic loss in the
poultry industry worldwide. FAdV-4 is the causative agent of
hydropericardium-hepatitis syndrome (HHS) or hydropericardium syndrome
(HPS). The virus targets mainly the liver, and HPS symptoms are observed
in infected chickens. This disease was first reported in Pakistan but
has now spread worldwide, and over time, various deletions in the FAdV
genome and mutations in its major structural proteins have been
detected. This review provides detailed information about FAdV-4 genome
organization, physiological features, epidemiology, coinfection with
other viruses, and host immune suppression. Moreover, we investigated
the role and functions of important structural proteins in FAdV-4
pathogenesis. Finally, the potential regulatory effects of FAdV-4
infection on ncRNAs are also discussed.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1370414"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1370414</a></p>
<hr />
<h2
id="fibrinogen-to-prealbumin-and-c-reactive-protein-to-prealbumin-ratios-as-prognostic-indicators-in-severe-fever-with-thrombocytopenia-syndrome">Fibrinogen-to-prealbumin
and C-reactive protein-to-prealbumin ratios as prognostic indicators in
severe fever with thrombocytopenia syndrome</h2>
<p>date: 2024-06-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe primary aim of this study is to investigate the
correlation between serum levels of fibrinogen-to-prealbumin ratio (FPR)
and C-reactive protein-to-prealbumin ratio (CPR) and prognostic outcomes
among patients with severe fever with thrombocytopenia syndrome (SFTS).
SFTS, characterized by elevated mortality rates, represents a
substantial public health challenge as an emerging infectious
disease.MethodsThe study included 159 patients with SFTS. Clinical and
laboratory data were compared between the survival and death groups.
Univariate and multivariate logistic regression analysis were utilized
to identify independent risk factors for mortality. The predictive
efficacy of FPR and CPR was evaluated using receiver operating
characteristic (ROC) curve. Survival analysis was conducted using the
Kaplan–Meier curve and the log-rank test was employed for
comparison.ResultsThe death group exhibited significantly elevated
levels of FPR and CPR compared to the survival group (P &lt; 0.05).
Multivariate logistic regression analysis confirmed that both FPR and
CPR independently correlated with a poorer prognosis among patients with
SFTS. The ROC curve analysis indicated that FPR and CPR had superior
predictive capabilities compared to C-reactive protein and fibrinogen.
Kaplan–Meier survival analysis demonstrated that patients with SFTS who
have FPR &gt; 0.045 (log-rank test; χ2 = 17.370, P &lt; 0.001) or CPR
&gt; 0.05 (log-rank test; χ2 = 19.442, P &lt; 0.001) experienced
significantly lower survival rates within a 30-day follow-up
period.ConclusionElevated levels of FPR and CPR serve as distinct risk
factors for mortality among patients with SFTS, indicating their
potential to predict an unfavorable prognosis in these patients.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397789"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397789</a></p>
<hr />
<h2
id="gut-microbiome-derived-hydrolasesan-underrated-target-of-natural-product-metabolism">Gut
microbiome-derived hydrolases—an underrated target of natural product
metabolism</h2>
<p>date: 2024-06-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>In recent years, there has been increasing interest in studying gut
microbiome-derived hydrolases in relation to oral drug metabolism,
particularly focusing on natural product drugs. Despite the significance
of natural product drugs in the field of oral medications, there is a
lack of research on the regulatory interplay between gut
microbiome-derived hydrolases and these drugs. This review delves into
the interaction between intestinal microbiome-derived hydrolases and
natural product drugs metabolism from three key perspectives. Firstly,
it examines the impact of glycoside hydrolases, amide hydrolases,
carboxylesterase, bile salt hydrolases, and epoxide hydrolase on the
structure of natural products. Secondly, it explores how natural product
drugs influence microbiome-derived hydrolases. Lastly, it analyzes the
impact of interactions between hydrolases and natural products on
disease development and the challenges in developing microbial-derived
enzymes. The overarching goal of this review is to lay a solid
theoretical foundation for the advancement of research and development
in new natural product drugs and personalized treatment.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1392249"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1392249</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
