<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.31</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.31">health 2024.31</h1>
<p>(date: 2024-07-31 08:05:06)</p>
<hr />
<h2
id="pdia-iminosugar-influence-on-subcutaneous-staphylococcus-aureus-and-pseudomonas-aeruginosa-infections-in-mice">PDIA
iminosugar influence on subcutaneous Staphylococcus aureus and
Pseudomonas aeruginosa infections in mice</h2>
<p>date: 2024-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionBiofilm-associated infections persist as a therapeutic
challenge in contemporary medicine. The efficacy of antibiotic therapies
is ineffective in numerous instances, necessitating a heightened focus
on exploring novel anti-biofilm medical strategies. Among these,
iminosugars emerge as a distinctive class of compounds displaying
promising biofilm inhibition properties.MethodsThis study employs an in
vivo wound infection mouse model to evaluate the effectiveness of PDIA
in treating biofilm-associated skin wound infections caused by
Staphylococcus aureus and Pseudomonas aeruginosa. Dermic wounds in mice
were infected with biofilm-forming strains, specifically S. aureus 48
and P. aeruginosa 5, which were isolated from patients with diabetic
foot, and are well-known for their strong biofilm formation. The
subsequent analysis included clinical, microbiological, and
histopathological parameters. Furthermore, an exploration into the
susceptibility of the infectious strains to hydrogen peroxide was
conducted, acknowledging its potential presence during induced
inflammation in mouse dermal wounds within an in vivo model.ResultsThe
findings revealed the efficacy of PDIA iminosugar against the S. aureus
strain, evidenced by a reduction in bacterial numbers within the wound
and the inflammatory focus.DiscussionThis study suggests that PDIA
iminosugar emerges as an active and potentially effective antibiofilm
agent, positioning it as a viable treatment option for staphylococcal
infections.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1395577"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1395577</a></p>
<hr />
<h2
id="corrigendum-molecular-characterization-of-carbapenem-and-ceftazidime-avibactam-resistant-enterobacterales-and-horizontal-spread-of-blandm-5-gene-at-a-lebanese-medical-center">Corrigendum:
Molecular characterization of carbapenem and
ceftazidime-avibactam-resistant Enterobacterales and horizontal spread
of blaNDM-5 gene at a Lebanese medical center</h2>
<p>date: 2024-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1466718"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1466718</a></p>
<hr />
<h2
id="tad-pili-contribute-to-the-virulence-and-biofilm-formation-of-virulent-aeromonas-hydrophila">Tad
pili contribute to the virulence and biofilm formation of virulent
Aeromonas hydrophila</h2>
<p>date: 2024-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Type IV pili (T4P) are versatile proteinaceous protrusions that
mediate diverse bacterial processes, including adhesion, motility, and
biofilm formation. Aeromonas hydrophila, a Gram-negative facultative
anaerobe, causes disease in a wide range of hosts. Previously, we
reported the presence of a unique Type IV class C pilus, known as tight
adherence (Tad), in virulent Aeromonas hydrophila (vAh). In the present
study, we sought to functionalize the role of Tad pili in the
pathogenicity of A. hydrophila ML09-119. Through a comprehensive
comparative genomics analysis of 170 A. hydrophila genomes, the
conserved presence of the Tad operon in vAh isolates was confirmed,
suggesting its potential contribution to pathogenicity. Herein, the
entire Tad operon was knocked out from A. hydrophila ML09-119 to
elucidate its specific role in A. hydrophila virulence. The absence of
the Tad operon did not affect growth kinetics but significantly reduced
virulence in catfish fingerlings, highlighting the essential role of the
Tad operon during infection. Biofilm formation of A. hydrophila ML09-119
was significantly decreased in the Tad operon deletant. Absence of the
Tad operon had no effect on sensitivity to other environmental
stressors, including hydrogen peroxide, osmolarity, alkalinity, and
temperature; however, it was more sensitive to low pH conditions.
Scanning electron microscopy revealed that the Tad mutant had a rougher
surface structure during log phase growth than the wildtype strain,
indicating the absence of Tad impacts the outer surface of vAh during
cell division, of which the biological consequences are unknown. These
findings highlight the role of Tad in vAh pathogenesis and biofilm
formation, signifying the importance of T4P in bacterial infections.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1425624"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1425624</a></p>
<hr />
<h2
id="progranulin-inhibits-autophagy-to-facilitate-intracellular-colonization-of-helicobacter-pylori-through-the-pgrnmtordcn-axis-in-gastric-epithelial-cells">Progranulin
inhibits autophagy to facilitate intracellular colonization of
Helicobacter pylori through the PGRN/mTOR/DCN axis in gastric epithelial
cells</h2>
<p>date: 2024-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Helicobacter pylori (H. pylori) infection is the primary risk factor
for the progress of gastric diseases. The persistent stomach
colonization of H. pylori is closely associated with the development of
gastritis and malignancies. Although the involvement of progranulin
(PGRN) in various cancer types has been well-documented, its functional
role and underlying mechanisms in gastric cancer (GC) associated with H.
pylori infection remain largely unknown. This report demonstrated that
PGRN was up-regulated in GC and associated with poor prognosis, as
determined through local and public database analysis. Additionally, H.
pylori induced the up-regulation of PGRN in gastric epithelial cells
both in vitro and in vivo. Functional studies have shown that PGRN
promoted the intracellular colonization of H. pylori. Mechanistically,
H. pylori infection induced autophagy, while PGRN inhibited autophagy to
promote the intracellular colonization of H. pylori. Furthermore, PGRN
suppressed H. pylori-induced autophagy by down-regulating decorin (DCN)
through the mTOR pathway. In general, PGRN inhibited autophagy to
facilitate intracellular colonization of H. pylori via the PGRN/mTOR/DCN
axis. This study provides new insights into the molecular mechanisms
underlying the progression of gastric diseases, suggesting PGRN as a
potential therapeutic target and prognostic predictor for these
disorders.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1425367"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1425367</a></p>
<hr />
<h2
id="programmed-cell-death-in-helicobacter-pylori-infection-and-related-gastric-cancer">Programmed
cell death in Helicobacter pylori infection and related gastric
cancer</h2>
<p>date: 2024-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Programmed cell death (PCD) plays a crucial role in maintaining the
normal structure and function of the digestive tract in the body.
Infection with Helicobacter pylori (H. pylori) is an important factor
leading to gastric damage, promoting the Correa cascade and accelerating
the transition from gastritis to gastric cancer. Recent research has
shown that several PCD signaling pathways are abnormally activated
during H. pylori infection, and the dysfunction of PCD is thought to
contribute to the development of gastric cancer and interfere with
treatment. With the deepening of studies on H. pylori infection in terms
of PCD, exploring the interaction mechanisms between H. pylori and the
body in different PCD pathways may become an important research
direction for the future treatment of H. pylori infection and H.
pylori-related gastric cancer. In addition, biologically active
compounds that can inhibit or induce PCD may serve as key elements for
the treatment of this disease. In this review, we briefly describe the
process of PCD, discuss the interaction between different PCD signaling
pathways and the mechanisms of H. pylori infection or H. pylori-related
gastric cancer, and summarize the active molecules that may play a
therapeutic role in each PCD pathway during this process, with the
expectation of providing a more comprehensive understanding of the role
of PCD in H. pylori infection.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416819"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416819</a></p>
<hr />
<h2
id="genome-wide-srna-and-mrna-transcriptomic-profiling-insights-into-carbapenem-resistant-acinetobacter-baumannii">Genome-wide
sRNA and mRNA transcriptomic profiling insights into
carbapenem-resistant Acinetobacter baumannii</h2>
<p>date: 2024-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionAcinetobacter baumannii (AB) is rising as a human
pathogen of critical priority worldwide as it is the leading cause of
opportunistic infections in healthcare settings and carbapenem-resistant
AB is listed as a “super bacterium” or “priority pathogen for drug
resistance” by the World Health Organization.MethodsClinical isolates of
A. baumannii were collected and tested for antimicrobial susceptibility.
Among them, carbapenem-resistant and carbapenem-sensitive A. baumannii
were subjected to prokaryotic transcriptome sequencing. The change of
sRNA and mRNA expression was analyzed by bioinformatics and validated by
quantitative reverse transcription-PCR.ResultsA total of 687 clinical
isolates were collected, of which 336 strains of A. baumannii were
resistant to carbapenem. Five hundred and six differentially expressed
genes and nineteen differentially expressed sRNA candidates were
discovered through transcriptomic profile analysis between
carbapenem-resistant isolates and carbapenem-sensitive isolates.
Possible binding sites were predicted through software for sRNA21 and
adeK, sRNA27 and pgaC, sRNA29 and adeB, sRNA36 and katG, indicating a
possible targeting relationship. A negative correlation was shown
between sRNA21 and adeK (r = -0.581, P = 0.007), sRNA27 and pgaC (r =
-0.612, P = 0.004), sRNA29 and adeB (r = -0.516, P =
0.020).DiscussionThis study preliminarily screened differentially
expressed mRNA and sRNA in carbapenem-resistant A. baumannii, and
explored possible targeting relationships, which will help further
reveal the resistance mechanism and provide a theoretical basis for the
development of drugs targeting sRNA for the prevention and treatment of
carbapenem-resistant A. baumannii infection.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1419989"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1419989</a></p>
<hr />
<h2
id="aph3-ie-an-aminoglycoside-modifying-enzyme-discovered-in-a-rabbit-derived-citrobacter-gillenii-isolate">APH(3’)-Ie,
an aminoglycoside-modifying enzyme discovered in a rabbit-derived
Citrobacter gillenii isolate</h2>
<p>date: 2024-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAminoglycoside-modifying enzymes (AMEs) play an essential
role in bacterial resistance to aminoglycoside antimicrobials. With the
development of sequencing techniques, more bacterial genomes have been
sequenced, which has aided in the discovery of an increasing number of
novel resistance mechanisms.MethodsThe bacterial species was identified
by 16S rRNA gene homology and average nucleotide identity (ANI)
analyses. The minimum inhibitory concentration (MIC) of each
antimicrobial was determined by the agar dilution method. The protein
was expressed with the pCold I vector in E. coli BL21, and enzyme
kinetic parameters were examined. The whole-genome sequence of the
bacterium was obtained via the Illumina and PacBio sequencing platforms.
Reconstruction of the phylogenetic tree, identification of conserved
functional residues, and gene context analysis were performed using the
corresponding bioinformatic techniques.ResultsA novel aminoglycoside
resistance gene, designated aph(3’)-Ie, which confers resistance to
ribostamycin, kanamycin, sisomicin and paromomycin, was identified in
the chromosome of the animal bacterium Citrobacter gillenii DW61, which
exhibited a multidrug resistance phenotype. APH(3’)-Ie showed the
highest amino acid identity of 74.90% with the functionally
characterized enzyme APH(3’)-Ia. Enzyme kinetics analysis demonstrated
that it had phosphorylation activity toward four aminoglycoside
substrates, exhibiting the highest affinity (Km, 4.22 ± 0.88 µM) and the
highest catalytic efficiency [kcat/Km, (32.27 ± 8.14) × 104] for
ribomycin. Similar to the other APH(3’) proteins, APH(3’)-Ie contained
all the conserved functional sites of the APH family. The aph(3’)-Ie
homologous genes were present in C. gillenii isolates from different
sources, including some of clinical significance.ConclusionIn this work,
a novel chromosomal aminoglycoside resistance gene, designated
aph(3’)-Ie, conferring resistance to aminoglycoside antimicrobials, was
identified in a rabbit isolate C. gillenii DW61. The elucidation of the
novel resistance mechanism will aid in the effective treatment of
infections caused by pathogens carrying such resistance genes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1435123"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1435123</a></p>
<hr />
<h2
id="investigating-the-association-between-gut-microbiome-and-aortic-aneurysm-diseases-a-bidirectional-two-sample-mendelian-randomization-analysis">Investigating
the association between gut microbiome and aortic aneurysm diseases: a
bidirectional two-sample Mendelian randomization analysis</h2>
<p>date: 2024-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study aims to investigate the associations between
specific bacterial taxa of the gut microbiome and the development of
aortic aneurysm diseases, utilizing Mendelian Randomization (MR) to
explore these associations and overcome the confounding factors commonly
present in observational studies.MethodsEmploying the largest available
gut microbiome and aortic aneurysm Genome-Wide Association Study
databases, including MiBioGen, Dutch Microbiome Project, FinnGen, UK
Biobank, and Michigan Genomics Initiative, this study performs
two-sample bidirectional MR analyses. Instrumental variables, linked to
microbiome taxa at significant levels, were selected for identifying
relationships with abdominal aortic aneurysms (AAA), thoracic aortic
aneurysms (TAA), and aortic dissection (AD). Methods like inverse
variance weighted, MR-PRESSO, MR-Egger, weighted median, simple mode,
and mode-based estimate were used for MR analysis. Heterogeneity was
assessed with the Cochran Q test. MR-Egger regression and MR-PRESSO
addressed potential unbalanced horizontal pleiotropy.ResultsThe analysis
did not find any evidence of statistically significant associations
between the gut microbiome and aortic aneurysm diseases after adjusting
for the false discovery rate (FDR). Specifically, while initial results
suggested correlations between 19 taxa and AAA, 25 taxa and TAA, and 13
taxa with AD, these suggested associations did not hold statistical
significance post-FDR correction. Therefore, the role of individual gut
microbial taxa as independent factors in the development and progression
of aortic aneurysm diseases remains inconclusive. This finding
underscores the necessity for larger sample sizes and more comprehensive
studies to further investigate these potential links.ConclusionThe study
emphasizes the complex relationship between the gut microbiome and
aortic aneurysm diseases. Although no statistically significant
associations were found after FDR correction, the findings provide
valuable insights and highlight the importance of considering gut
microbiota in aortic aneurysm diseases research. Understanding these
interactions may eventually contribute to identifying new therapeutic
and preventive strategies for aortic aneurysm diseases.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1406845"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1406845</a></p>
<hr />
<h2
id="gut-microbiome-a-novel-preventive-and-therapeutic-target-for-prostatic-disease">Gut
microbiome: a novel preventive and therapeutic target for prostatic
disease</h2>
<p>date: 2024-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The human gut microbiome (GM) impacts various physiological processes
and can lead to pathological conditions and even carcinogenesis if
homeostasis is disrupted. Recent studies have indicated a connection
between the GM and prostatic disease. However, the underlying mechanisms
are still unclear. This review aims to provide a summary of the existing
information regarding the connection between the GM and various
prostatic conditions such as chronic prostatitis/chronic pelvic pain
syndrome (CP/CPPS), benign prostatic hyperplasia (BPH), and prostate
cancer (PCa). Furthermore, the review aims to identify possible
pathogenic mechanisms and suggest potential ways of targeting GM to
prevent and treat prostatic disease. Due to the complexity of the
mechanism between GM and prostatic diseases, additional research is
required to comprehend the association between the two. This will lead
to more effective treatment options for prostatic disease.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431088"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431088</a></p>
<hr />
<h2
id="pseudomonas-aeruginosa-mucinous-phenotypes-and-algumucabd-operon-mutant-characteristics-obtained-from-inpatients-with-bronchiectasis-and-their-correlation-with-acute-aggravation">Pseudomonas
aeruginosa mucinous phenotypes and algUmucABD operon mutant
characteristics obtained from inpatients with bronchiectasis and their
correlation with acute aggravation</h2>
<p>date: 2024-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveAlthough the mechanism is unclear, Pseudomonas aeruginosa
(PA) infection directly affects the frequency of acute exacerbations in
patients with bronchiectasis. The aims of this article are to analyze
the genetic mutation characteristics of the algUmucABD operon in PA,
isolated from hospitalized patients with bronchiectasis, and to explore
independent risk factors for frequent acute exacerbations of
bronchiectasis.MethodsBased on the number of acute exacerbations that
occurred in the past year, these patients with bronchiectasis were
divided into those with frequent acute exacerbations (Group A) and those
with non-frequent acute exacerbations (Group B). We identified the
distribution of mucoid phenotypes (MPs) and alginate morphotypes (AMs)
in PA, and classified them into I–IV categories based on their different
AMs; otherwise, the gene mutation types (GMTs) of the algUmucABD operon
were tested. Subsequently, the relationship between GMT, MP, and AM and
the independent risk factors for frequent acute exacerbations in
patients with bronchiectasis were explored.ResultsA total of 93 patients
and 75 PA strains, from January 2019 to August 2023, were included in
this study. The MP and AM distributions of PA were as follows: 64
strains (85.33%) of mucoid (the AMs were 38 strains of type I, 3 strains
of type II, and 23 strains of type IV) and 11 strains of non-mucoid (the
AM was type III only). Mucoid PA with algU, mucA, mucB, and mucD
mutations accounted for 19.61%, 74.51%, 31.37%, and 50.98%,
respectively. GMT was divided into the following: mucA mutations only,
mucA combined with other gene mutations, other gene mutations without
mucA mutations, and without gene mutations. In 91.7% of PA with type I
of AM, only mucA mutations occurred, and in both separate MP and AM, the
GMT differences were statistically significant. Lastly, the number of
lung lobes with bronchiectasis and the number of PA with mucA mutations
only were the independent risk factors for frequent acute
exacerbations.ConclusionThe mucA mutation was primarily responsible for
the mucoid of MP and type I of AM in PA, and it was also an independent
risk factor for frequent exacerbations of bronchiectasis.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1402348"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1402348</a></p>
<hr />
<h2
id="the-value-of-metagenomic-next-generation-sequencing-with-different-nucleic-acid-extracting-methods-of-cell-free-dna-or-whole-cell-dna-in-the-diagnosis-of-non-neutropenic-pulmonary-aspergillosis">The
value of metagenomic next-generation sequencing with different nucleic
acid extracting methods of cell-free DNA or whole-cell DNA in the
diagnosis of non-neutropenic pulmonary aspergillosis</h2>
<p>date: 2024-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>PurposeMetagenomic next-generation sequencing(mNGS) is a novel
molecular diagnostic technique. For nucleic acid extraction methods,
both whole-cell DNA (wcDNA) and cell-free DNA (cfDNA) are widely applied
with the sample of bronchoalveolar lavage fluid (BALF). We aim to
evaluate the clinical value of mNGS with cfDNA and mNGS with wcDNA for
the detection of BALF pathogens in non-neutropenic pulmonary
aspergillosis.MethodsmNGS with BALF-cfDNA, BALF-wcDNA and conventional
microbiological tests (CMTs) were performed in suspected non-neutropenic
pulmonary aspergillosis. The diagnostic value of different assays for
pulmonary aspergillosis was compared.ResultsBALF-mNGS (cfDNA, wcDNA)
outperformed CMTs in terms of microorganisms detection. Receiver
operating characteristic (ROC) analysis indicated BALF-mNGS (cfDNA,
wcDNA) was superior to culture and BALF-GM. Combination diagnosis of
either positive for BALF-mNGS (cfDNA, wcDNA) or CMTs is more sensitive
than CMTs alone in the diagnosis of pulmonary aspergillosis
(BALF-cfDNA+CMTs/BALF-wcDNA+CMTs vs. CMTs: ROC analysis: 0.813 vs.0.66,
P=0.0142/0.796 vs.0.66, P=0.0244; Sensitivity: 89.47% vs. 47.37%,
P=0.008/84.21% vs. 47.37%, P=0.016). BALF-cfDNA showed a significantly
greater reads per million (RPM) than BALF-wcDNA. The area under the ROC
curve (AUC) for RPM of Aspergillus detected by BALF-cfDNA, used to
predict “True positive” pulmonary aspergillosis patients, was 0.779,
with a cut-off value greater than 4.5.ConclusionWe propose that the
incorporation of BALF-mNGS (cfDNA, wcDNA) with CMTs improves diagnostic
precision in the identification of non-neutropenic pulmonary
aspergillosis when compared to CMTs alone. BALF-cfDNA outperforms
BALF-wcDNA in clinical value.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1398190"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1398190</a></p>
<hr />
<h2
id="the-intratumoral-microbiota-a-new-horizon-in-cancer-immunology">The
intratumoral microbiota: a new horizon in cancer immunology</h2>
<p>date: 2024-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Over the past decade, advancements in high-throughput sequencing
technologies have led to a qualitative leap in our understanding of the
role of the microbiota in human diseases, particularly in oncology.
Despite the low biomass of the intratumoral microbiota, it remains a
crucial component of the tumor immune microenvironment, displaying
significant heterogeneity across different tumor tissues and individual
patients. Although immunotherapy has emerged a major strategy for
treating tumors, patient responses to these treatments vary widely.
Increasing evidence suggests that interactions between the intratumoral
microbiota and the immune system can modulate host tumor immune
responses, thereby influencing the effectiveness of immunotherapy.
Therefore, it is critical to gain a deep understanding of how the
intratumoral microbiota shapes and regulates the tumor immune
microenvironment. Here, we summarize the latest advancements on the role
of the intratumoral microbiota in cancer immunity, exploring the
potential mechanisms through which immune functions are influenced by
intratumoral microbiota within and outside the gut barrier. We also
discuss the impact of the intratumoral microbiota on the response to
cancer immunotherapy and its clinical applications, highlighting future
research directions and challenges in this field. We anticipate that the
valuable insights into the interactions between cancer immunity and the
intratumoral microbiota provided in this review will foster the
development of microbiota-based tumor therapies.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1409464"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1409464</a></p>
<hr />
<h2
id="standardization-of-an-antimicrobial-resistance-surveillance-network-through-data-management">Standardization
of an antimicrobial resistance surveillance network through data
management</h2>
<p>date: 2024-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe rapid spread of COVID-19 worldwide within 2 months
demonstrated the vulnerability of the world’s population to infectious
diseases. In 2015, the Global Antimicrobial Resistance and Use
Surveillance System (GLASS) was launched to combat antimicrobial
resistance (AMR). However, there has been no comprehensive assessment of
the decade-long global battle against AMR based on GLASS
data.MethodsSouth Korea established Kor-GLASS (Korean-GLASS) to
proactively monitor data quality and enable international
collaborations. A unique feature of Kor-GLASS is the quality control
center (QCC), which uses network hubs and ensures standardized,
high-quality data through interlaboratory proficiency testing (IPT) and
external quality assessment (EQA). In addition, the QCC multifaceted
endeavors for integrated data quality management.ResultsSince 2020,
high-quality AMR data have indicated fluctuating antibiotic resistance
rates in South Korea. This trend does not align with the decrease in
antibiotic usage seen in humans but coincides with non-human antibiotic
sales, indicating a need for greater monitoring of non-human antibiotic
resistance. Comprehensive and robust management taking account of the
intricate interplay among humans, animals, and the environment is
essential. Kor-GLASS has been expanded into a “One Health” multiagency
collaborative initiative.DiscussionAlthough a standardized solution is
not suitable for all countries, it must align with the local context and
international standards. A centralized top-down management structure
such as that of the QCC is essential to ensure continuous data quality
coordination. Sustained efforts and surveillance systems are crucial for
monitoring and managing AMR and safeguarding human health.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411145"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411145</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
