<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2025-08-03 06:08:26)</p>
<hr />
<h2
id="sars-cov-2-xbb.1.5-infects-wild-type-c57bl6-mice-and-induces-a-protective-cd4-t-cell-response-required-for-viral-clearance">SARS-CoV-2
XBB.1.5 infects wild-type C57BL/6 mice and induces a protective CD4+ T
cell response required for viral clearance</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Mouse models are critical for studying SARS-CoV-2 pathogenesis and
evaluating therapeutic and preventive strategies. Standard C57BL/6 mice
are generally resistant to infection with the ancestral SARS-CoV-2
strain due to inefficient binding of the viral spike protein to the
murine angiotensin-converting enzyme 2 (ACE2) receptor. Although human
ACE2 transgenic mice can support robust pulmonary infection, these
models often develop fatal encephalitis, a pathology not commonly
observed in humans. We and others have previously shown that certain
SARS-CoV-2 variants can infect wild-type C57BL/6 mice and cause
discernible disease. However, the susceptibility of C57BL/6 mice to
recently emerged Omicron subvariants, and the role of T cell-mediated
immunity in controlling these infections, remain incompletely
understood. Herein, we evaluated the susceptibility of wild-type C57BL/6
mice to infection with the SARS-CoV-2 Omicron subvariant XBB.1.5. We
assessed viral burden, innate and adaptive immune responses, and
virus-induced lung pathology. Our findings demonstrate that XBB.1.5
efficiently replicates in both the upper and lower respiratory tracts of
C57BL/6 mice, inducing significant lung inflammation and pathology.
Infection elicited a robust pulmonary CD4+ and CD8+ T cell response.
Through antibody-mediated depletion studies, we further show that CD4+ T
cells are critical for viral clearance, particularly in the nasal
airways, as their depletion resulted in persistent viral RNA in the
upper respiratory tract. These findings underscore the importance of
CD4+ T cell responses in controlling XBB.1.5 infection and provide a
valuable model for studying variant-specific immune responses and
pathogenesis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1621226"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1621226</a></p>
<hr />
<h2
id="the-correlation-between-antibiotic-usage-and-antibiotic-resistance-a-3-year-retrospective-study">The
correlation between antibiotic usage and antibiotic resistance: a 3-year
retrospective study</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMultidrug-resistant (MDR) microorganisms have increased
all over the world, which is considered a public health threat. The
emergence of MDR bacterial pathogens correlates with the increased
antibiotic usage. This study aimed to determine the correlation between
antibiotic usage and antibiotic resistance within 3 years.MethodThis was
a retrospective cross-sectional study reviewing the positive bacterial
culture results and the total antibiotic usage in six hospitals in
Penang for 3 years from January 2021 to December 2023 through a
convenient sampling method.ResultsEvery sample type has experienced a
significant shift over the years. Most microorganisms from all samples
significantly changed in distribution over time, except for
Streptococcus pneumoniae, carbapenem-resistant Enterobacterales (CRE)
Escherichia coli, and CRE Klebsiella pneumoniae. However,
methicillin-resistant Staphylococcus aureus (MRSA), K. pneumoniae, and
Pseudomonas aeruginosa showed significant changes in the number of total
isolates from blood cultures only in the 3 years. In terms of
prevalence, statistically significant differences were observed for most
microorganisms from all samples except for S. pneumoniae, CRE E. coli,
and CRE K. pneumoniae across the years. P. aeruginosa showed significant
prevalence in blood culture over time. Cefoperazone/sulbactam,
amoxicillin/clavulanic acid, ceftriaxone, and ceftazidime showed
significant changes in susceptibility for K. pneumoniae over time. A
statistically significant difference in total antibiotic usage across
the 3 years was observed. Regarding the correlation between antibiotic
usage and antibiotic resistance, Pearson’s correlation was 0.777 (p =
0.433), which is suggestive of a strong positive correlation between
third-generation cephalosporin usage and extended-spectrum
beta-lactamase (ESBL), whereas Pearson’s correlation was 0.762 (p =
0.448), which also suggests strong positive correlation between
carbapenem usage and CREs.ConclusionThe correlation between the use of
third-generation cephalosporins and ESBL rate, as well as the use of
carbapenems and CRE rate, further suggests that controlling certain
antibiotic usage could help mitigate the rise in MDR microorganisms.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1608921"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1608921</a></p>
<hr />
<h2
id="meta-analysis-of-h.-pylori-and-the-gut-microbiome-interactions-and-clinical-outcomes">Meta-analysis
of H. pylori and the gut microbiome interactions and clinical
outcomes</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionHelicobacter pylori is a globally prevalent gastric
pathogen associated with chronic gastritis, peptic ulcers, and gastric
cancer. Its interaction with the gut microbiome (GM), a dynamic
microbial community within the gastrointestinal tract, plays a critical
role in modulating host immune responses and disease progression. This
study aimed to investigate the complex interactions between H. pylori
infection and the GM and to evaluate how microbiome alterations relate
to clinical outcomes such as gastritis, ulcers, and gastric
cancer.MethodsA meta-analysis was conducted using publicly available 16S
rRNA and shotgun metagenomic datasets. Microbiome composition
differences were assessed using differential abundance analysis, alpha-
and beta-diversity metrics, and principal component analysis (PCA).
Random forest models were employed to predict the clinical outcomes
based on microbiome and clinical data. Hyperparameter tuning and
cross-validation were applied to ensure model robustness. ResultsThe
analysis revealed significant microbial shifts associated with H. pylori
infection, including enrichment of Proteobacteria, Fusobacterium spp.,
and Prevotella spp., and depletion of beneficial taxa like Lactobacillus
spp. and Faecalibacterium prausnitzii. Microbial diversity declined
progressively with disease severity. Predictive models demonstrated high
accuracy (89.3%) in classifying the disease states and identifying key
microbial biomarkers such as Fusobacterium spp. and Bacteroides fragilis
with strong predictive power.DiscussionThis study highlights the
critical role of GM dysbiosis in H. pylori-related disease progression.
The identified microbial signatures and predictive models offer
promising tools for early diagnosis, risk stratification, and
personalized treatment of H. pylori-associated gastrointestinal
disorders. Future integration of multi-omics data may further unravel
the microbial mechanisms and support microbiome-based precision
medicine.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1610523"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1610523</a></p>
<hr />
<h2
id="transcriptomic-profiling-reveals-sars-cov-2-infected-humanized-mhc-mice-recapitulate-human-post-vaccination-immune-responses">Transcriptomic
profiling reveals SARS-CoV-2-infected humanized MHC mice recapitulate
human post vaccination immune responses</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe ongoing COVID-19 pandemic caused by SARS-CoV-2 remains
a critical global health priority, with persistent socioeconomic
ramifications despite its reclassification from Public Health Emergency
of International Concern (PHEIC) status. While humanized major
histocompatibility complex (hMHC) murine models have been extensively
utilized in oncological research, their application in virological
studies-particularly for coronavirus pathogenesis-remains underexplored.
MethodsThis study systematically characterized immune responses in
SARS-CoV-2-challenged hMHC mice lung tissues through comparative
transcriptomic profiling, combined with functional enrichment and PPI
network analyses.ResultsKey findings demonstrate that hMHC mice exhibit
enhanced immunological activation relative to wild-type controls,
particularly in IFN-γ signaling pathways and neutrophil mobilization
dynamics that closely parallel human post-vaccination responses.
Comparative analysis with human whole blood RNA-seq datasets revealed
that hMHC mice exhibit both high reproducibility in transcriptomic
profiles and significant similarity to human immune responses across
innate and adaptive immunity. ConclusionsThese results confirm that the
hMHC murine model can serve as an effective platform for vaccine
research, providing a theoretical foundation for the application of
humanized MHC mice and offering new insights into viral infection
mechanisms and the development of novel vaccines.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1634577"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1634577</a></p>
<hr />
<h2
id="assessment-of-knowledge-and-awareness-of-monkeypox-viral-infection-in-palestine-a-community-based-study">Assessment
of knowledge and awareness of monkeypox viral infection in Palestine: a
community-based study</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMonkeypox (MPXV) is a re-emerging global health threat,
particularly in non-endemic areas such as Palestine, where research is
scarce. This study aims to assess public knowledge and awareness of MPXV
in PalestineMethodsA 38-item questionnaire assessed socioeconomic
status, health status, and MPXV knowledge among 1241 participants.
Pearson’s Chi-square test examined the association between high and low
knowledge levels and explanatory variables. A logistic regression model
examined the relationships between knowledge levels and the explanatory
factors of the investigation.ResultsThe study indicated that merely 23%
of the 1241 participants exhibited a high level of knowledge regarding
monkeypox. Approximately 20% of the participants felt that the virus
might be transmitted through contaminated surfaces, whereas 40%
acknowledged close contact with infected individuals as a mode of
transmission. Furthermore, 11.4% of the participants accurately
identified an incubation period of 5 to 21 days. Concerning symptoms,
50% identified rash and high fever as the most severe, whereas 25%
reported fatigue, muscle pain, swollen lymph nodes, and breathing
issues. Regarding prevention and treatment, 38% asserted that masks may
not prevent MPXV, 31% claimed that no treatment is available, and 50%
believed that immunization can be effective. Knowledge was significantly
higher among those with more educational attainment and healthcare
employment (p-value&lt; 0.05). The primary sources of information were
friends, family, and social media.ConclusionThe understanding of
monkeypox infection among the Palestinian population was found to be
somewhat insufficient. These findings highlight the urgent need for
public education on monkeypox to increase awareness and engage the
public prior to any potential future outbreak.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584848"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584848</a></p>
<hr />
<h2
id="correction-prevalence-of-chlamydia-trachomatis-neisseria-gonorrhoeae-and-ureaplasma-urealyticum-infections-in-males-and-females-of-childbearing-age-in-chengdu-china">Correction:
Prevalence of Chlamydia trachomatis, Neisseria gonorrhoeae, and
Ureaplasma urealyticum infections in males and females of childbearing
age in Chengdu, China</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1662536"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1662536</a></p>
<hr />
<h2
id="the-combined-410nm-and-infrared-light-effectively-suppresses-bacterial-survival-under-realistic-conditions">The
combined 410nm and infrared light effectively suppresses bacterial
survival under realistic conditions</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The demand for establishing an effective but inexpensive method to
interfere with the spread of infectious diseases has been higher than
ever before, since the recent pandemic. As a follow-up study, we tested
a few practically applicable lights with a safe 410nm violet light (V)
with infrared (IR, 850nm) under realistic conditions to identify an
optimal light for suppressing pathogens. Our results indicate that 410nm
violet light is as effective as the previously tested 405nm violet light
with infrared (850nm). Therefore, we focused on optimizing combined
lights (3V-1IR or 2.33V-1IR) with lower power level that is below 24
Watt. Using the Multi Drug Resistant (MDR) Escherichia coli (E. coli)
and Staphylococcus aureus (S. aureus) from ATCC, we confirmed that the
combined 20W light effectively suppressed the survival of both MDR
bacterial strains on a smooth surface at the distance of 25cm, 50cm, 1m
or 2m, which mimicked the realistic living spaces. As expected, the
effectiveness was inversely proportional to the exposed distance. For
example, the light exposure suppressed more than 91-97% of E. coli
within 1–2 hours and 96-99% of S. aureus within 2–6 hours at short
distances (25 or 50cm), whereas it took 6–8 hours to reach 92-95% of E.
coli and 91-99% of S. aureus suppression at 1 or 2m. In the mechanistic
studies, we confirmed that the bacterial death was mediated by the
enhanced level of Reactive Oxygen Species (ROS), in addition to reduced
thickness of biofilm from 410nm and 850nm infrared light. Our results
strongly support the possible application of using this combined 410nm
with infrared light as an inexpensive and practical solution to reduce
the potential pathogens, at least from bacterial origins in a variety of
living spaces.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1624160"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1624160</a></p>
<hr />
<h2
id="a-multiplex-targeted-ngs-panel-for-identifying-pathogens-in-canine-neurological-and-reproductive-diseases">A
multiplex targeted NGS panel for identifying pathogens in canine
neurological and reproductive diseases</h2>
<p>date: 2025-08-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMultiple pathogens can infect the canine reproductive and
central nervous systems, including organisms that are zoonotic, such as
Brucella canis, pathogenic Leptospira spp., Anaplasma phagocytophilum,
Histoplasma capsulatum, and Blastomyces dermatitidis. In this study, we
developed a targeted next-generation sequencing (tNGS) panel to identify
common infectious agents related to neurologic and reproductive disease
in canines while incorporating less common zoonotic agents into a single
test.MethodsPrimer pools were developed to detect 34 pathogens and used
in two multiplex PCR assays, which were combined prior to library
preparation and sequencing using Ion Torrent technology. A feasibility
study was performed with known positive clinical samples, bacterial
isolates, or synthetic DNA (gBlocks) spiked into nucleic acids from
negative canine clinical samples.ResultsOf the 34 organisms included in
the panel, 33 were detectable. Some primer sets were not specific for
the intended target organism, based on BLAST analysis (NCBI) of the
obtained sequences. Compared to real-time PCR assays, pathogens could be
detected at Ct values from 33-38, depending on the pathogen, and at
approximately 100–1000 copies based on gBlock testing. A total of 76
samples (39 positive and 40 negative) were tested, representing
neurological and reproductive samples. Diagnostic sensitivity and
specificity for the assay were calculated as 89% and 98%, respectively.
The tNGS assay had the added benefit of strain-typing Canine distemper
virus and Canine parvovirus-2 in the positive samples. For
reproducibility, a blinded panel was tested by our laboratory and
another laboratory using the same tNGS protocol where the assay had an
agreement for 16 out of 18 samples with a Cohen’s kappa value of 0.77,
indicating high reproducibility.DiscussionThe tNGS assay was not as
sensitive as real-time PCR assays, which is a known limitation of this
method, and was evident based on the diagnostic sensitivity testing.
However, the comprehensive nature of this assay is beneficial for
syndromic testing.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1591145"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1591145</a></p>
<hr />
<h2
id="innovative-antifungal-strategies-to-combat-drug-resistant-candida-auris-recent-advances-and-clinical-implications">Innovative
antifungal strategies to combat drug-resistant Candida auris: recent
advances and clinical implications</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Candida auris is an invasive fungal pathogen recognized globally as a
significant health threat due to its marked resistance to multiple
classes of antifungal agents, including azoles, echinocandins, and
polyenes. The associated high morbidity and mortality rates present
considerable public health challenges. Research efforts have largely
focused on understanding the molecular mechanisms of antifungal
resistance and developing alternative therapies to counteract this
issue. This review summarizes current advances in the identification of
natural antifungal compounds, the development of novel synthetic agents,
biological antifungals, nanotechnology-based approaches, combination
therapies, and photodynamic treatments. Notably, several synthetic
compounds such as rezafungin and fosmanogepix are in clinical trials for
C. auris infections. Biological antifungals, including monoclonal
antibodies, vaccines, and peptides, have shown the capacity to enhance
host immune responses and reduce mortality in murine models. Combination
therapies have proven particularly valuable for overcoming resistance by
exploiting synergistic effects and broadening antimicrobial coverage.
Despite these promising developments, majority of studies have been
conducted in vitro, with a relative lack of in vivo or human research.
Therefore, further investigation is needed to validate the efficacy and
safety of these alternative antifungal strategies for the treatment of
drug-resistant C. auris infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1641373"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1641373</a></p>
<hr />
<h2
id="heat-killed-lactobacillus-acidophilus-suppresses-sars-cov-2-infection-in-the-human-intestinal-epithelial-cell-line-caco-2">Heat-killed
Lactobacillus acidophilus suppresses SARS-CoV-2 infection in the human
intestinal epithelial cell line Caco-2</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe gastrointestinal (GI) tract is suspected to be a
possible source for the systemic spread of severe acute respiratory
syndrome coronavirus 2 (SARS-CoV-2), as well as a reservoir of long
coronavirus disease (COVID). Thus, the mucosal epithelial tissue of the
colon is a potential target for probiotics to help control SARS-CoV-2
infection. Recently, the effect of live probiotics on COVID-19 has been
evaluated. However, live probiotics have certain risks, including the
transmission of antibiotic-resistant genes, disturbance of gut
colonization in infants, and systemic infections induced by
translocation. Therefore, there is growing interest in nonviable
microorganisms, particularly heat-killed probiotic bacteria, to mitigate
these risks.MethodsThis study evaluated the antiviral properties of
heat-killed Lactobacillus acidophilus (HK-LA) in the Caco-2 cell line.
Caco-2 cells were infected by SARS-CoV-2 with or without 24-hour
pretreatment of HK-LA and the presence of HK-LA during
infection.ResultsRT-qPCR analysis showed that HK-LA treatment
significantly reduced SARS-CoV-2 genome copies by approximately 30%.
Similarly, flow cytometry revealed a roughly 30% decrease in SARS-CoV-2
spike-positive Caco-2 cells following HK-LA treatment. Additionally,
ELISA demonstrated a significant increase in IFN-λ2 secretion induced by
HK-LA.DiscussionHK-LA reduces viral infection in Caco-2 cells with an
increase in IFN-λ2 secretion. Therefore, heat-killed lactobacilli could
potentially reduce SARS-CoV-2 infection in the GI tract, suggesting a
possible clinical application.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1556344"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1556344</a></p>
<hr />
<h2
id="genomic-characterization-of-multidrug-resistant-clinical-acinetobacter-baumannii-isolates-from-a-hospital-in-paraguay">Genomic
characterization of multidrug-resistant clinical Acinetobacter baumannii
isolates from a hospital in Paraguay</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Acinetobacter baumannii is a clinically important pathogen capable of
causing serious nosocomial infections and acquiring resistance to
antimicrobials, particularly carbapenems, making treatment difficult and
prolonging hospital stays. In Latin America, high carbapenem-resistance
rates have been described among A. baumannii isolates, however, Paraguay
is one of the countries with limited data in this regard. Therefore, we
aimed to investigate resistance rates of A. baumannii isolates from the
National Hospital of Itaugua (NHI), Paraguay, from their database of
2022, and from December 2023 to February 2024, and to study in detail a
representative group of multidrug-resistant clinical isolates. For this
purpose, data were analyzed considering diagnostic, sample type and
antimicrobial susceptibility. Eight A. baumannii isolates recovered from
patients in six separate ICUs in 2024 were then selected and subjected
to susceptibility testing using VITEK® and to short- and long-read
sequencing, and clonality, resistome, virulome and plasmidome of the
isolates were investigated. IC2 (ST2 Pasteur, ST1816/195 Oxford and
ST872 Oxford) was the predominant clone among the Paraguayan isolates,
and a single isolate belonging to clone IC5 (ST79 Pasteur and ST1283
Oxford) was also identified. The carbapenemase gene blaOXA-23 was
located in transposons Tn2006 and Tn2008. Additionally, other antibiotic
resistance genes conferring resistance to aminoglycosides, macrolides,
sulfonamides, chloramphenicol, tetracyclines and trimethoprim were
identified, and were found embedded in genetic environments containing
mobile genetic elements. Multiple virulence genes were also detected,
mainly promoting biofilm formation and immune system modulation. Plasmid
analysis showed the presence of plasmids ranging in size from 2.27 to
10.74 Kb. This work describes the dissemination of the emerging clone
IC2 in Paraguay and offers a detailed analysis of the resistome,
virulome and plasmidome of carbapenem-resistant A. baumannii strains.
The results obtained highlight the importance of correctly
characterizing these multidrug-resistant pathogens to develop infection
prevention and control strategies at hospital level.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1620479"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1620479</a></p>
<hr />
<h2
id="meropenemvaborbactam-activity-against-carbapenem-resistant-klebsiella-pneumoniae-from-catheter-related-bloodstream-infections">Meropenem/vaborbactam
activity against carbapenem-resistant Klebsiella pneumoniae from
catheter-related bloodstream infections</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCarbapenem-resistant Klebsiella pneumoniae (CRKP) poses a
significant threat in oncology settings due to its multidrug resistance
and ability to form biofilms on indwelling medical devices.MethodsThis
study investigated the in vitro and in vivo activity of
meropenem/vaborbactam (MEV) against two CRKP isolates recovered from
catheter-related bloodstream infections in patients undergoing
orthopedic oncologic surgery.ResultsWhole-genome sequencing identified
the isolates as ST101 and ST307, harboring resistance determinants
including blaKPC-3 and blaOXA-1, distributed across IncFII and IncFIB
plasmid replicons. Both isolates exhibited extensive resistance to
β-lactams, aminoglycosides, and fluoroquinolones but remained
susceptible to MEV. Phenotypic assays revealed enhanced biofilm
formation and metabolic activity compared to the reference strain Kp
ATCC 13883 in the absence of hypervirulence-associated genes. MEV
demonstrated bactericidal activity against both planktonic and
biofilm-associated cells, with minimum bactericidal concentration
(MBC90) and minimum biofilm eradication concentration (MBEC90) values of
0.5/8 μg/ml for CRKP ST101, 0.12/8 μg/ml for CRKP ST307, and 0.25/8
μg/ml for the Kp ATCC 13883 strain. In the Galleria mellonella infection
model, MEV significantly improved larval survival following the CRKP
challenge.DiscussionThese findings demonstrate that MEV exhibits
activity against planktonic and biofilm-associated CRKP cells and
highlight the need for further investigation in managing
catheter-related bloodstream infections caused by multidrug-resistant K.
pneumoniae.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616353"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616353</a></p>
<hr />
<h2
id="comparative-evaluation-of-biofilm-forming-capacity-in-uropathogenic-and-commensal-escherichia-coli">Comparative
evaluation of biofilm-forming capacity in uropathogenic and commensal
Escherichia coli</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionEscherichia coli (E. coli) causes most cases of the
urinary tract infections (UTIs) via virulence factors like biofilms.
This study identifies key phenotypic and genotypic virulence attributes
of Uropathogenic Escherichia coli.MethodologyA total of 180
uropathogenic E. coli (UPEC) isolated from patients with different
categories (cystitis, pyelonephritis, recurrent UTI, catheter-associated
UTI, and asymptomatic bacteriuria) of UTI and 30 commensal E. coli
isolated from healthy individuals were evaluated for biofilm production
by phenotypic methods using tissue culture plate, tube adherence, and
Congo red method, and RT-PCR was used to genetically characterize
them.ResultsThis study analyzed 1,600 urine samples from UTI patients,
with 498 showing significant bacterial growth and 180 identifying E.
coli as the pathogen. The female-to-male ratio of UTI cases was 0.74.
Antibiotic susceptibility testing revealed 100% sensitivity to
tigecycline and fosfomycin as well as 89.44%, 86.11%, 81.66%, and 72.22%
sensitivity to nitrofurantoin, amikacin, imipenem, and meropenem,
respectively. Only 64.44% were sensitive to ciprofloxacin, with 10%
being multidrug-resistant (MDR). Moreover, 18.33% of the UPEC isolates
produced mettalo-beta-lactamases (MBL), and 13.33% produced AmpC
beta-lactamases. Biofilm production was observed in 72.22% of UPEC
isolates compared to 16.66% in commensal isolates. The biofilm-forming
UPEC, compared to commensal E. coli, has significantly higher antibiotic
resistance, with a 128-fold reduction in ciprofloxacin susceptibility.
Additionally, the fimH gene was detected in 98.33% of the UPEC
isolates.ConclusionThis study shows that UPEC strains produce specific
virulence determinants like adhesion to uroepithelial cells. Screening
for virulence factors should be integrated into microbiology
laboratories. Specific virulence genes linked to UPEC may serve as
potential targets for prophylactic strategies to prevent recurrent
infections and improve management.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1570422"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1570422</a></p>
<hr />
<h2
id="unveiling-facklamia-detection-of-an-emerging-microbe-in-the-skin-microbiome-of-patients-with-filarial-lymphedema">Unveiling
Facklamia: detection of an emerging microbe in the skin microbiome of
patients with filarial lymphedema</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Facklamia is an emerging pathogen in human beings and only a few
clinical cases were reported in the literature. We detected the presence
of this unusual microbe among the skin flora of three patients with
filarial lymphedema in a 16S rRNA-based metagenomic study on the skin
microbiome. To our knowledge, this is the first report of detection of
this microbe in patients affected with filarial lymphedema. Further
investigations are required to elucidate the role of Facklamia in
secondary skin and soft tissue infection of filarial lymphedema
patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1624288"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1624288</a></p>
<hr />
<h2
id="saccharomyces-boulardii-cncm-i-745-improves-intestinal-damage-in-sepsis-by-remodeling-intestinal-flora-structure">Saccharomyces
boulardii (CNCM I-745) improves intestinal damage in sepsis by
remodeling intestinal flora structure</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSepsis is one of the leading causes of mortality among
intensive care unit (ICU) patients. The intestinal tract is the primary
organ affected by sepsis, resulting in dysbiosis of the gut microbiota
and negatively impacting long-term prognosis. This study investigated
the protective effects of Saccharomyces boulardii (S. boulardii) on
intestinal damage during sepsis.MethodsDistilled water was administered
orally by intragastric for 4 weeks in control group and sepsis group. S.
boulardii (CNCM I-745) suspension (108CFU/ml) was administered orally by
intragastric for 4 weeks in probiotic group and treatment group. Rats in
control group and probiotic group received the intraperitoneal injection
of normal saline (5ml/kg). Rats in sepsis group and treatment group
received the intraperitoneal injection of LPS solution (1mg/ml). Eight
hours after the intraperitoneal injection, samples of serum, colonic
tissue, and colonic contents were collected and stored at - 80°C. Four
weeks later, the samples of colonic contents were taken to observe the
alterations in the intestinal microbiota.ResultsSepsis led to an
increase in the expression of IL-6, TNF-α, and a decrease in the
expression of occludin. After treatment with S. boulardii (CNCM I-745),
the inflammatory damage with sepsis was reduced, and the expression
level of occludin was significantly increased. High-throughput
sequencing analysis revealed that sepsis injury led to a decline in both
the diversity and abundance of the gut microbiota. Simultaneously, the
colonization of beneficial bacteria within the intestine diminished,
whereas the colonization of harmful bacteria surged. However, upon
administration of S. boulardii (CNCM I-745), an increase in the
diversity and abundance of the gut microbiota was evident. Moreover, the
composition of the gut microbiota underwent a discernible
alteration.ConclusionSepsis induces impairment of intestinal barrier
function and exacerbates inflammatory responses. The use of S. boulardii
(CNCM I-745) can modulate the composition of the intestinal flora by
enhancing the colonization of beneficial bacteria while reducing the
presence of harmful bacteria. It helps maintain intestinal mucosal
barrier function, mitigates intestinal damage associated with sepsis,
and potentially influences the long-term growth and development of
pediatric sepsis patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602792"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602792</a></p>
<hr />
<h2
id="integrated-oral-microbiome-and-metabolome-analysis-unveils-key-biomarkers-and-functional-pathway-alterations-in-patients-with-acute-myocardial-infarction">Integrated
oral microbiome and metabolome analysis unveils key biomarkers and
functional pathway alterations in patients with acute myocardial
infarction</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe significance of oral microbiota in acute myocardial
infarction (AMI) has been increasingly appreciated. However, the
association between oral microbiota, metabolites, and AMI parameters, as
well as the key microbiota and metabolites that may play a crucial role
in this process, remains unclear. To investigate the oral microbiome
composition and metabolomic profiles associated with AMI and explore the
roles of specific bacterial species and key metabolites in functional
pathways in the progression of AMI.MethodsA case-control study was
conducted involving 24 AMI patients and 24 matched healthy controls.
Saliva samples were collected for 16S rRNA sequencing and untargeted
LC-MS metabolomics. Correlation analysis was then performed to explore
associations between microbial taxa, metabolomic profiles, and AMI
clinical parameters, with results visualized as heatmaps of correlation
coefficients.ResultsThe component of the oral microbiota of the AMI
group showed significant alterations when compared to the control group.
Particularly, a significant shift in terms of alpha and beta diversity
was observed between the AMI and control groups (p &lt; 0.05). The
Streptococcus and Rothia genera, as well as 9(R)-HODE, 20-HETE
ethanolamide, and 5,6 EET metabolites, were identified as potential
biomarkers, achieving an area under the curve of 0.82–0.88. Functional
pathway analysis demonstrated significant upregulation in key metabolic
pathways, including the Citrate Cycle (TCA cycle), Pyruvate metabolism,
and Glucagon signaling pathways, which exhibited strong correlations
with established clinical markers of AMI.ConclusionThis integrative
analysis underscores the diagnostic potential of oral
microbiome-metabolome interactions in AMI and unveils key mechanistic
pathways for guiding future therapeutic interventions.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1607845"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1607845</a></p>
<hr />
<h2
id="enhancing-prognostic-accuracy-in-sepsis-a-modified-sofa-score-incorporating-lymphocyte-count-as-an-immune-function-marker">Enhancing
prognostic accuracy in sepsis: a modified SOFA score incorporating
lymphocyte count as an immune function marker</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSepsis, characterized by organ dysfunction due to a
dysregulated immune response, is diagnosed using the Sequential Organ
Failure Assessment (SOFA) score, which currently lacks immune function
markers. The objective of this research is to enhance the predictive
precision of SOFA for sepsis by integrating lymphocyte count as an
indicator of immune system functionality.MethodsThis retrospective study
was based on the MIMIC-IV database. The absolute lymphocyte count (ALC)
was assessed as a predictive biomarker through multivariate analysis
utilizing the Cox proportional-hazards model and was integrated into a
modified SOFA score (ALC-SOFA). Associations between ALC-SOFA score and
mortality were assessed using Kaplan-Meier survival analysis. Predictive
performance was evaluated by comparing the area under the receiver
operating characteristic (AUROC) curve for ALC-SOFA and the original
SOFA score. The primary endpoint was the mortality rate at 28 days, with
additional secondary endpoints including the mortality rates at 7 and 90
days.Results10,709 patients with sepsis were included in this study. ALC
was significantly lower in nonsurvivors than in survivors (0.90 ±
0.62×109/L vs. 1.12 ± 0.69×109/L, P&lt;0.001). Patients with lower
absolute lymphocyte counts (ALC) exhibited a significantly higher risk
of 28-day mortality (HR = 0.62, P&lt;0.001). Survival analysis revealed
higher mortality rates with increasing ALC-SOFA scores. The ALC-SOFA
score demonstrated improved prognostic performance for 28-day mortality
(AUROC = 0.680 vs. 0.664, P&lt;0.001) and 90-day mortality (AUROC =
0.666 vs. 0.647, P&lt;0.001) compared to the original SOFA
score.ConclusionIncorporating ALC into the SOFA score significantly
improves its ability to predict sepsis-related outcomes. The ALC-SOFA
score provides a novel tool for prognostic assessment, highlighting the
critical role of immune function in sepsis management.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593589"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593589</a></p>
<hr />
<h2
id="the-strategies-and-mechanisms-of-enteroviruses-to-evade-innate-immunity-and-the-vaccine-progress-of-enteroviruses">The
strategies and mechanisms of enteroviruses to evade innate immunity and
the vaccine progress of enteroviruses</h2>
<p>date: 2025-07-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Enteroviruses (EVs) are a group of highly contagious RNA viruses that
can cause a wide range of diseases, from mild infections to severe
complications like neurological disorders and myocarditis. This review
focuses on the innate immune evasion strategies employed by EVs,
highlighting their mechanisms and consequences. EVs evade host immune
responses through various tactics, including inhibiting pathogen
recognition receptors (PRRs) such as toll-like receptors (TLRs) and
RIG-I-like receptors (RLRs), disrupting key signaling pathways like
nuclear factor kappa-B (NF-κB) and (JAK)-signal transducers and
activators of transcription (STAT), and directly targeting interferon
(IFN) signaling components. Specific viral proteases, such as 2A
protease (2Apro) and 3C protease (3Cpro), play crucial roles in these
evasion strategies by cleaving host proteins involved in immune
signaling. Additionally, EVs manipulate host factors to suppress
antiviral responses, exemplified by the upregulation of proteins like
sex-determining region Y-box 4 (Sox4) and microRNAs (miRNAs) that
inhibit TLR signaling. The review also discusses the development of
vaccines against EVs, emphasizing the importance of prophylactic
measures in controlling infections. Understanding these immune evasion
mechanisms is essential for developing effective antiviral therapies and
vaccines.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1636104"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1636104</a></p>
<hr />
<h2
id="orthopedic-infections-associated-with-distinct-acinetobacter-strains-in-rural-area-of-qingdao-china">Orthopedic
infections associated with distinct Acinetobacter strains in rural area
of Qingdao, China</h2>
<p>date: 2025-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionAcinetobacter baumannii poses a profound global health
threat because of multidrug resistance and its association with
nosocomial infections. However, standard clinical diagnostics often
report it together with other Acinetobacter species as A.
baumannii-calcoaceticus complex (ABC), which unavoidably conceals the
attribution of non-A. baumannii species. This study reported orthopedic
infection cases associated with different Acinetobacter species and
characterized the genomes of the culture isolates to evaluate their
potential impact on the clinical treatment.MethodsNine in-patients with
A. baumannii-calcoaceticus complex identified by culture during
hospitalization were enrolled by the Orthopedics Department from a local
hospital in Qingdao, China. Their clinical data were reviewed. One ABC
isolate from each patient was tested for drug susceptibility and
subjected for whole-genome sequencing, followed by bioinformatic
analyses.ResultsThrough whole-genome analysis, nine ABC isolates were
identified as six A. baumannii, two A. pittii, and one A. soli with
distinct antibiotic resistance profiles and phylogenetic
characteristics, indicating progressing pathogen transmission across
broad geographic regions in One Health perspective. All A. baumannii and
A. pittii strains carried multidrug resistance genes, while A. soli bore
only amvA and rsmA. Phenotypically, eight isolates were susceptible to
almost all the antibiotics tested, with only one A. baumannii being
multidrug resistant. Despite this, eight patients received
cephalosporins following positive reports of A. baumannii-calcoaceticus
complex.ConclusionOur study highlighted the limitation of current
clinical diagnostic approaches for non-A. baumannii cases, which tended
to be overtreated, and suggested that Acinetobacter etiology landscape
should be explored further beyond A. baumannii to avoid antibiotic
misuse.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1601779"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1601779</a></p>
<hr />
<h2
id="research-progress-on-the-application-of-rpa-crisprcas12a-in-the-rapid-visual-detection-of-pathogenic-microorganisms">Research
progress on the application of RPA-CRISPR/Cas12a in the rapid visual
detection of pathogenic microorganisms</h2>
<p>date: 2025-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>In an increasingly complex global public health landscape, the
continuous emergence of novel pathogens and the growing problem of
antibiotic resistance highlight the urgent need for rapid, efficient,
and precise detection technologies for pathogenic microorganisms. The
innovative combination of Recombinase Polymerase Amplification (RPA) and
CRISPR/Cas12a enables the rapid amplification of target gene fragments
under isothermal conditions and the precise recognition and cleavage of
specific nucleic acid sequences. The integration of RPA and
CRISPR/Cas12a significantly enhances the sensitivity and accuracy of
detection simplifies operational procedures, and reduces the dependence
on specialized equipment for testing personnel. This combination
demonstrates great potential for application in clinical diagnostics and
point-of-care testing. This article provides a detailed overview of the
principles of RPA-CRISPR/Cas12a and its latest research progress in the
field of pathogen detection, aiming to promote the widespread
application of RPA-CRISPR/Cas12a technology in clinical medicine and
public health and to offer theoretical support for its further
optimization.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1640938"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1640938</a></p>
<hr />
<h2
id="investigation-of-the-inhibitory-effects-of-total-flavonoids-of-litchi-seed-on-clonorchis-sinensis-induced-liver-damage-and-fibrosis">Investigation
of the inhibitory effects of total flavonoids of litchi seed on
Clonorchis sinensis-induced liver damage and fibrosis</h2>
<p>date: 2025-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundClonorchis sinensis (C. sinensis), which is prevalent in
Asian countries, including China, Korea and Vietnam, is known to cause
liver fibrosis, leading to various liver diseases and potentially fatal
outcomes. Total flavonoids of litchi seed (TFL), a traditional Chinese
medicine abundant in the Southern China, is known for its multiple
pharmacological activities, including anti-fibrotic, anti-oxidative and
hepato-protective properties. The present study explored the inhibitory
effects of TFL on liver damage caused by C. sinensis infection in
rats.MethodsIn the animal experiment, female rats were infected with C.
sinensis and treated with TFL. Serum biochemical indicators were
measured each week. Pathological changes in the rat liver were evaluated
by examining the appearance of liver, and by performing hematoxylin and
eosin (H&amp;E) and Masson’s trichrome staining. Through
immunofluorescence and immunohistochemistry, the number of activated
hepatic stellate cells (HSCs) were counted and the collagen deposition
area in these cells were measured. In the cell experiment, rat hepatic
stellate cells (HSC-T6) were stimulated with TGF-β1 (10 ng/ml) and
treated with different concentrations of TFL. The expression levels of
HSCs activation markers α-smooth muscle actin (α-SMA), collagen I, and
fibronectin were detected using quantitative real-time polymerase chain
reaction (RT-qPCR). The protein levels of α-SMA and collagen I was
detected by Western blot.ResultsTFL improved liver function in C.
sinensis-infected rats, as indicated by reduced levels of transaminases,
bilirubin, and bile acids. TFL also alleviated pathological changes in
liver tissues. TFL also reduced the number of activated HSCs and
downregulated fibrosis-related markers (collagen I/III, α-SMA) in rat
liver. In HSC-T6, TFL inhibited HSCs activation by reducing the mRNA
levels of α-SMA, collagen I, and fibronectin, and reduced the protein
level of α-SMA and collagen I.ConclusionsTFL attenuated C.
sinensis-induced liver fibrosis in rats. Our study provided experimental
evidence for the development of novel anti-liver fibrosis drugs and
offer new insights into the treatment of C. sinensis infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1625004"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1625004</a></p>
<hr />
<h2
id="the-diagnostic-potential-of-extracellular-vesicles-in-virus-related-diseases">The
diagnostic potential of extracellular vesicles in virus-related
diseases</h2>
<p>date: 2025-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Extracellular vesicles (EVs), a heterogeneous population of lipid
bilayer-enclosed membranous particles, are widely involved in cell-cell
communication and pathophysiological regulation. Recent advances reveal
their dual functionality in viral pathogenesis: while facilitating viral
dissemination through transport of pathogenic components, they
simultaneously orchestrate host antiviral defense mechanisms. The unique
molecular cargo loading capacity and high stability of EVs in body
fluids make them ideal biomarkers for early infection diagnosis,
treatment monitoring, and prognostic evaluation of virus-related
disease. Compared to traditional viral detection methods, EV-based
liquid biopsy techniques exhibit distinct advantages, including
non-invasiveness and enhanced sensitivity thresholds. This review
systematically examines the diagnostic potential of EV biomarkers in
viral infections, offering novel perspectives for developing precision
diagnostics and therapeutic interventions.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1641405"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1641405</a></p>
<hr />
<h2
id="the-prognostic-value-of-serum-apolipoprotein-a1-levels-in-elderly-patients-with-de-novo-sars-cov-2-omicron-infection">The
prognostic value of serum apolipoprotein A1 levels in elderly patients
with de novo SARS-CoV-2 omicron infection</h2>
<p>date: 2025-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe coronavirus disease of 2019 (COVID-19) pandemic, caused
by the severe acute respiratory syndrome coronavirus 2, has affected
millions of people worldwide. The omicron variant is currently the
predominant strain circulating worldwide. Serum apolipoprotein A1
(ApoA1) is linked to endothelial cell injury and serves as a valuable
biomarker for monitoring the progression of inflammation in infected
individuals. However, the potential roles of ApoA1 in the context of the
omicron variant remain elusive.MethodsTo investigate the prognostic
value of serum ApoA1 levels at diagnosis, using mortality rate as the
primary evaluation indicator, we performed a 65-day monitoring and
retrospectively analyzed a cohort of 237 individuals diagnosed with
omicron. Patients were categorized into two groups based on their ApoA1
levels, high and low. The Kaplan-Meier method was employed to assess
overall survival (OS), while the log-rank test was utilized for
comparative analysis between the groups. Additionally, both univariate
and multivariate Cox proportional hazards models were applied to
evaluate the prognostic significance of ApoA1 levels.ResultsOur results
indicated that ApoA1 levels were significantly reduced in patients
infected with the omicron variant. Notably, ApoA1 levels in severe cases
were lower than those in mild-to-moderate cases, with this difference
reaching statistical significance. Additionally, we observed a
significant increase in C-reactive protein (CRP) and beta-2
microglobulin (β2-MG) levels in individuals with decreased ApoA1 levels.
Furthermore, patients with reduced ApoA1 levels exhibited a
statistically significant decline in OS (P = 0.001). A decreased ApoA1
level (&lt; 0.87 g/L) was identified as an independent adverse
prognostic factor for OS in omicron patients, as determined by
multivariate cox proportional hazards regression analysis (P =
0.035).ConclusionThe serum ApoA1 level at the initial diagnosis was
significantly correlated with the severity and prognosis of omicron
infections. Therefore, we propose that decreased levels of ApoA1 may
serve as an independent negative prognostic factor in patients infected
with omicron.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1617266"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1617266</a></p>
<hr />
<h2
id="comparison-of-microbial-culture-metagenomic-next-generation-sequencing-and-droplet-digital-polymerase-chain-reaction-methods-for-pathogen-detection-in-patients-with-neurosurgical-central-nervous-system-infection">Comparison
of microbial culture, metagenomic next-generation sequencing and droplet
digital polymerase chain reaction methods for pathogen detection in
patients with neurosurgical central nervous system infection</h2>
<p>date: 2025-07-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundNeurosurgical central nervous system infections (NCNSIs)
are one of the most common complications in neurosurgical patients,
followed by neurosurgery itself, trauma, implants or infection need to
be treated by surgery. However, the diagnosis of NCNSIs continues to
pose a significant challenge. The primary objective of this study was to
comprehensively assess the diagnostic performance of metagenomic
next-generation sequencing (mNGS) and Multiplex Droplet Digital PCR
(ddPCR) in elucidating the microbiological etiologies underlying NCNSIs
in affected patients.MethodsData on 127 enrolled NCNSIs patients were
collected from Emergency Neurosurgical Intensive Care Unit at Qilu
Hospital of Shandong University from June 2022 to October 2024. The
clinical record, cerebrospinal fluid or pus routine, biochemical tests,
microbial smear and culture, mNGS and ddPCR results, time to positive
culture(TTPC), time from sample harvesting to final positive results
(THTR) obtained by the physician, turn-around time for mNGS and ddPCR,
and follow-up data were collected and analyzed.ResultsA total of 127
patients were enrolled in this study. In comparison to the positive rate
achieved by traditional culture method (59.1%) in diagnosing NCNSIs, the
overall pathogen detection rates of mNGS and ddPCR were markedly
elevated (86.6%, p&lt;0.01 and 78.7%, p&lt;0.01, respectively). Notably,
the administration of empiric antibiotics did not significantly
influence the positive detection rates of either mNGS or ddPCR. When
stratified by infection type, mNGS and ddPCR demonstrated notably higher
positive detection rates in three specific categories of
NCNSIs-ventriculitis, intracranial abscess, and implant-associated
infections-compared to meningitis. Among the 127 patients, 37 (29.1%)
tested positive via mNGS but negative via microbial culture, whereas 11
patients were positive via mNGS but negative via ddPCR. The mean TTPC
for microbial culture was 15.1 ± 10.4 hours. Furthermore, the mean THTR
for microbial culture, mNGS and ddPCR were 22.6 ± 9.4 hours, 16.8 ± 2.4
hours and 12.4 ± 3.8 hours, respectively. Importantly, ddPCR exhibited a
significantly shorter THTR compared to mNGS (p&lt;0.01).ConclusionmNGS
and ddPCR hold the potential to substantially augment the diagnostic
efficacy for NCNSIs patients. It is advisable that, in future clinical
practice, mNGS and ddPCR be more extensively employed for the early and
precise identification of pathogens in NCNSIs patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1606283"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1606283</a></p>
<hr />
<h2
id="evaluation-of-molecular-mouse-sepsis-panel-new-portable-and-rapid-tests-for-microorganism-detection-in-suspected-blood-stream-infection">Evaluation
of molecular mouse sepsis panel: new portable and rapid tests for
microorganism detection in suspected blood stream infection</h2>
<p>date: 2025-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionBloodstream infections (BSIs) represent a significant
public health concern, characterized by the presence of pathogens in the
bloodstream, leading to serious conditions. Between 30 and 40% of BSIs
evolve in sepsis, characterized by a life-threatening organ dysfunction
and accompanied by a strong or dysregulated systemic immune or
inflammatory response. A timely and accurate diagnosis of BSIs is
crucial for ensuring effective management and treatment of critically
ill patients. The diagnosis of BSIs can be initiated either from
positive blood culture (BC) or directly from blood specimens using
advanced diagnostic methods, these approaches are increasingly
integrated into traditional diagnostic workflows.MethodsIn the context
of emerging molecular methods, Alifax introduced the Molecular Mouse
(MM) Sepsis Panel in 2022. This study aims to evaluate the performance
of MM cartridges in detecting Gram-positive bacteria and Gram-negative
bacteria, as well as their associated resistance genes, using clinical
leftover positive BCs and contrived samples for low-prevalence targets.
Positive BCs were collected at the Unit of Microbiology, The Great
Romagna Hub Laboratory, Pievesestina, FC, Italy. ResultsOverall, MM
Sepsis Panel showed excellent sensitivity (98.62%) and specificity
(99.81%), confirming the system’s reliability and accuracy.
DiscussionThe high diagnostic accuracy demonstrated by these tests
supports their potential application as rapid diagnostic tools for blood
cultures, facilitating prompt identification of pathogens and
antimicrobial resistance determinants in clinical microbiology
settings.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1579074"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1579074</a></p>
<hr />
<h2
id="detecting-early-kidney-injury-due-to-hostvirus-interaction-response-in-treatment-naive-chba-pilot-study">Detecting
early kidney injury due to host–virus interaction response in
treatment-naive CHB—a pilot study</h2>
<p>date: 2025-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>PurposeClinical evidence suggests that patients with chronic
hepatitis B (CHB) have an increased risk of renal impairment due to
inflammation induced by virus–host interaction. We aimed to evaluate and
validate a set of protein biomarkers singularly and in combination for
the early detection of subclinical kidney injury in patients with CHB
naive to antiretroviral therapy.MethodsThis work is part of a
prospective cross-sectional study for which 69 HBsAg-positive,
treatment-naive patients with CHB with an equal number of age-matched
healthy volunteers were considered. At diagnosis, serum creatinine
(sCr), urea, alanine transaminase, aspartate transaminase, serum
cystatin-C (sCys-C), serum neutrophil gelatinase-associated lipocalin
(sNGAL), serum Fetuin-A (sFet-A), urinary interleukin-18 binding protein
(uIL-18BP), and urinary kidney injury molecule-1 (uKIM-1) levels were
determined.ResultsThere was a significant elevation in the
concentrations of three proteins in our CHB cohort (sCys-C, sNGAL, and
uIL-18BP; p &lt; 0.0001) while sFet-A was down-regulated (p&lt;0.01) as
compared to the control group. A receiver operating characteristic curve
analysis revealed an Area under the curve of 0.935 for sCys-C and 0.811
for sNGAL, which improved to 0.984 when all four indicators were
combined in a panel to discriminate the onset of renal injury incited by
inflammatory response in CHB with 97.1% sensitivity at 91.3%
specificity. Additionally, only sCys-C and sNGAL differed significantly
among the phases of CHB infection (p&lt;0.05). ConclusionsThis novel
noninvasive diagnostic screen is expedient in detecting inflammation and
early kidney injury before a rise in sCr and can aid in predicting renal
outcomes in patients with CHB.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1601678"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1601678</a></p>
<hr />
<h2
id="global-research-trends-on-the-interaction-between-gut-microbiome-and-bile-acids-a-bibliometric-and-visualized-analysis">Global
research trends on the interaction between gut microbiome and bile
acids: a bibliometric and visualized analysis</h2>
<p>date: 2025-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAn increasing number of studies have shown that gut
microbiome-bile acids interactions play a crucial role in host health
and disease. This bibliometric analysis aims to identify the global
scientific output, research hotspots, and frontiers of gut
microbiome-bile acids in the past two decades.MethodsWe searched the
relevant studies of gut microbiome-bile acids published between 2004 and
2024 in the Web of Science Core Collection database. Microsoft Excel
2019, VOSviewer 1.6.18, Tableau Desktop 2024.2.2, Scimago Graphica
1.0.45, and CiteSpace 6.2.R3 were used to analyze the publications,
countries/regions, institutions, journals, authors, references, and
keywords.ResultsA total of 4795 original articles and reviews were
collected. A visual analysis of the results showed that the number of
publications increased rapidly over time. China published the most
papers, the United States had the most citations, and the most
productive institution was Shanghai Jiaotong University. The most
prolific author was Jia Wei, and Jason M. Ridlon was the most frequently
co-cited author. Nutrients was the most productive journal. In the
keyword co-occurrence network, except for gut microbiome and bile acids,
inflammation becomes the keyword with the highest frequency. Keywords
and reference analysis show that metabolic diseases (such as obesity and
type 2 diabetes mellitus), cancer (such as colorectal cancer), and
disease-related mechanisms (such as tgr5 and pathway) are the hot topics
and future research trends in this field.ConclusionIn this study,
bibliometric analysis was utilized to explore the relationship between
gut microbiome and bile acids. The findings can reflect the current
hotspots and new directions of gut microbiome-bile acids, and provide an
objective description and comprehensive guidance for future related
studies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616995"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616995</a></p>
<hr />
<h2
id="characterization-of-key-spike-rbd-residues-influencing-sars-cov-2-variant-adaptation-to-avian-ace2">Characterization
of key spike RBD residues influencing SARS-CoV-2 variant adaptation to
avian ACE2</h2>
<p>date: 2025-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe beta-coronavirus SARS-CoV-2 has been revealed to
infect mammals and other species, which potentially promotes the virus
adaptation to broader species and the emergence of new variants. The
host range of different SARS-CoV-2 variants are mainly determined by the
affinity of the receptor-binding domain (RBD) of the spike protein to
the host receptor angiotensin-converting enzyme 2 (ACE2). Thus, this
study aims to elucidate the detailed mechanisms of such dynamic
adaptation of indicated SARS-CoV-2 variants.MethodsIn this study, flow
cytometry and surface plasmon resonance (SPR) assays were used to assess
the binding affinity between RBDs and avian ACE2. Then, infection assays
with MLV-based SARS-CoV-2 spike pseudovirus or authentic viruses were
performed to verify the avian ACE2 mediated viral entry. Finally,
mutagenesis studies were conducted to identify key amino acids of avian
ACE2 orthologs and RBDs.ResultsOur previous findings revealed that
wild-type SARS-CoV-2 RBD does not bind chicken ACE2. Here, we found that
ACE2 orthologs from chicken and mallard were capable to support binding
to RBDs of the Alpha, Beta, and Gamma variants, which further enabled
the viral entry. On the contrary, the RBD of BA.1 failed to bind avian
ACE2. Whereas, a triple-residue reversal mutant (S446G, S496G, H505Y)
restored ACE2 binding and enabled efficient viral entry. Additionally,
several key residues within RBD were characterized as the determinant of
its affinity to avian ACE2.DiscussionOur findings reveal that higher
mutation rates in emerging variants might lead to future cross-species
receptor usage or even spillover. Understanding such cross-species
transmission mechanisms provides new insights to the virological
features and potential host range of emerging SARS-CoV-2 variants.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1631926"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1631926</a></p>
<hr />
<h2
id="the-predictive-value-of-sofa-and-apsiii-scores-for-28-day-mortality-risk-in-simi-a-cohort-study-based-on-the-mimic-iv-database">The
predictive value of SOFA and APSIII scores for 28-day mortality risk in
SIMI: a cohort study based on the MIMIC-IV database</h2>
<p>date: 2025-07-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe objective of this study was to systematically identify
and evaluate scoring systems that predict the prognosis of patients with
sepsis-induced myocardial injury (SIMI).MethodsData were retrieved from
the Medical Information Mart for Intensive Care IV (MIMIC-IV) database.
Logistic and Cox regression analyses were conducted to identify risk
factors associated with 28-day mortality in patients with sepsis-induced
myocardial injury (SIMI). The prognostic performance of the scoring
systems was comprehensively assessed using receiver operating
characteristic (ROC) curves, Kaplan-Meier survival and decision curve
analysis (DCA).ResultsLogistic regression analysis showed that Acute
Physiology Score III (APSIII) (Odds Ratio [OR] =1.011, 95% Confidence
Interval [CI] 1.002–1.018, P=0.005), Sequential Organ Failure Assessment
(SOFA) (OR =1.097, 95% CI 1.045–1.144, P&lt;0.001), and Charlson
Comorbidity Index (CHARLSON) (OR=1.095, 95% CI 1.048–1.145, P=0.036)
were independent risk factors for 28-day mortality in SIMI patients. Cox
regression analysis confirmed that SOFA (HR=1.082, 95% CI 1.054– 1.111,
P&lt;0.001), APSIII (HR=1.010, 95% CI 1.005–1.015, P&lt;0.001), and
CHARLSON (HR=1.044, 95% CI 1.012–1.077, P=0.007) were independent risk
factors. ROC curve analysis showed that SOFA (AUC=0.685, 95% CI
0.663–0.707) and APSIII (AUC=0.683, 95% CI 0.662–0.705) had
significantly higher AUC values compared to other scoring systems. DCA
results showed that APSIII and SOFA had better net benefit than other
scoring systems.ConclusionsThe SOFA and APSIII scores effectively
identified high-risk patients with SIMI, providing evidence-based
support for early clinical intervention.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1574625"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1574625</a></p>
<hr />
<h2 id="low-sodium-chicken-soup">Low Sodium Chicken Soup</h2>
<p>date: 2025-07-28, from: The Kidney Dietitian</p>
<p>
This article was written by dietetic student Emma Kortmansky &amp;
reviewed by Melanie Betz MS, RD, CSR, FNKF, FAND. I don’t know about
you, but in my opinion, nothing says comfort food quite like […]
</p>
<p>
The post
<a href="https://www.thekidneydietitian.org/low-sodium-chicken-soup/">Low
Sodium Chicken Soup</a> appeared first on
<a href="https://www.thekidneydietitian.org">The Kidney Dietitian</a>.
</p>
<p><br></p>
<p><a href="https://www.thekidneydietitian.org/low-sodium-chicken-soup/"
class="uri">https://www.thekidneydietitian.org/low-sodium-chicken-soup/</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2025-07-28, from: People’s CDC</p>
<p>This is the People’s CDC bi-weekly update for July 28, 2025. This
Weather Report from the People’s CDC shares the latest news on the
ongoing COVID situation and other public health crises in the United
States.</p>
<p><br></p>
<p><a
href="https://peoplescdc.org/peoples-cdc-covid-19-weather-report-103/"
class="uri">https://peoplescdc.org/peoples-cdc-covid-19-weather-report-103/</a></p>
<hr />
<h2
id="blood-microbiome-signatures-in-systemic-diseases-current-insights-methodological-pitfalls-and-future-horizons">Blood
microbiome signatures in systemic diseases: current insights,
methodological pitfalls, and future horizons</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The human-associated microbiome, encompassing diverse microbial
communities across body sites, plays a pivotal role in maintaining host
homeostasis. Disruption of this balance, termed dysbiosis, has been
implicated in a spectrum of pathophysiological conditions.
Traditionally, blood was considered a sterile microenvironment. However,
emerging insights into the blood microbiome challenge the paradigm of
blood sterility, revealing microbial signatures, including cell-free DNA
and viable taxa, with putative implications for host physiology and
disease. The blood taxonomic profile at the phylum level is dominated by
Proteobacteria, with Bacteroidetes, Actinobacteria, and Firmicutes
following in abundance. Dysbiosis in blood microbiome composition may
indicate or contribute to systemic dysregulation, pointing to its
potential role in disease etiology. These findings highlight the blood
microbiome as a possible driver in the pathogenesis of infectious and
non-infectious diseases, neurodegenerative disorders, and
immune-mediated conditions. The detection of specific microbial profiles
in circulation holds promise for biomarker discovery, enhancing disease
stratification, and informing precision therapeutic strategies. However,
advancing this field requires overcoming methodological challenges,
including contamination control, standardization, and reproducibility.
This review aims to present blood microbiome biomarkers across
infectious, non-infectious, neurodegenerative, and immune-mediated
diseases, while critically examining methodological variations,
controversies, limitations, and future research directions. Elucidating
these factors is critical to advancing blood microbiome biomarker
validation and therapeutic targeting, thereby refining mechanistic
insights into systemic disease pathogenesis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616029"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616029</a></p>
<hr />
<h2
id="integrated-oral-gut-microbiota-therapy-a-novel-perspective-on-preventing-bacterial-translocation-for-systemic-disease-management">Integrated
oral-gut microbiota therapy: a novel perspective on preventing bacterial
translocation for systemic disease management</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Oral dysbiosis increases the risk of oral diseases and systemic
diseases, with many related conditions overlapping with systemic
diseases triggered by gut dysbiosis. Studies have shown that the oral
cavity serves as an endogenous reservoir for gut microbial strains,
influencing the homeostasis of both oral and gut microbiota through
interactions involving bacterial translocation, microbial metabolites,
immune cells, and inflammatory factors. In specific disease contexts,
certain microbial communities [e.g., Porphyromonas gingivalis (P.g),
Fusobacterium nucleatum (F.n)], metabolites (e.g., short-chain fatty
acids, gingipains), ligands (e.g., lipopolysaccharides, peptidoglycans),
or host responses may vary. However, substantial evidence has firmly
established the central role of microbiota in oral-gut crosstalk. These
findings position the oral-gut axis as a potential causal mechanism
linking systemic diseases. Compared with healthy non-cancer subjects,
cancer patients exhibit significant differences in oral microbial
abundance and diversity. For instance, F.n is associated with an
increased risk of colorectal cancer(CRC), while Oribacterium and
Fusobacterium may serve as potential biomarkers for hepatocellular
carcinoma. Notably, oral pathogens or their metabolites can translocate
along the oral-gut axis or due to certain oral activities (e.g.,
toothbrushing, tooth extraction), contributing to the initiation and
progression of inflammation and tumorigenesis. For example, P.g can
accumulate in the liver, where its fimbrial protein FimA binds to
Toll-like receptor 2 (TLR2), complement receptor 3 (CR3), and
CXC-chemokine receptor 4 (CXCR4), triggering various immune responses
that promote the development of non-alcoholic fatty liver
disease(NAFLD). This review systematically summarizes recent advances in
understanding the role of the oral microbiota and the oral-gut axis in
systemic diseases, along with their underlying pathological mechanisms.
It particularly highlights the translational value of integrating oral
and gut microbiota research, offering novel insights for the prevention
and precision treatment of systemic disorders. The unique and
heterogeneous microbiota within the oral microbiota and the oral-gut
axis may serve as novel diagnostic biomarkers or therapeutic targets for
diseases associated with oral and gut dysbiosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1641816"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1641816</a></p>
<hr />
<h2
id="exploring-fecal-microbiota-signatures-associated-with-immune-response-and-antibiotic-impact-in-nsclc-insights-from-metagenomic-and-machine-learning-approaches">Exploring
fecal microbiota signatures associated with immune response and
antibiotic impact in NSCLC: insights from metagenomic and machine
learning approaches</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSubstantial interstudy heterogeneity in cancer
immunotherapy-associated biomarkers has hindered their clinical
applicability. To address this challenge, we performed a comprehensive
integration of publicly available global metagenomic datasets. By
leveraging metagenomic profiling and machine learning approaches, this
study aimed to elucidate gut microbial signatures associated with immune
response in lung cancer (LC) and to evaluate the modulatory effects of
antibiotic exposure.MethodsA systematic literature search was conducted
to identify relevant datasets, resulting in the inclusion of 209 fecal
metagenomic samples: 154 baseline samples (45 responders, 37
non-responders, and 72 healthy controls) and 55 longitudinal samples
collected during immunotherapy. We performed taxonomic and functional
characterization of gut microbiota (GM) differentiating responders from
non-responders, delineated microbiome dynamics during treatment, and
assessed the impact of antibiotics on key microbial taxa. Among eight
machine learning algorithms evaluated, the optimal model was selected to
construct a predictive framework for immunotherapy
response.ResultsMicrobial α-diversity was significantly elevated in
responders compared to non-responders, with antibiotic administration
further amplifying this difference—most notably at the species level.
Integrative multi-omics analysis identified two pivotal microbial
biomarkers, s_Bacteroides caccae and s_Prevotella copri, which were
strongly associated with immunotherapy efficacy. A random forest-based
classifier achieved robust predictive performance, with area under the
curve (AUC) values of 0.82 and 0.79 at the species and genus levels,
respectively. Notably, P. copri was further enriched in responders with
poor progression-free survival (PFS &lt;3 months), indicating a
potential deleterious role. Antibiotic exposure significantly influenced
the abundance and functional potential of these key taxa. KEGG-based
functional analysis revealed the enrichment of amino acid metabolism
pathways in responders. Additionally, CARD database annotation
demonstrated that the majority of antibiotic resistance genes were
associated with Bacteroidetes and Proteobacteria, implicating these taxa
in shaping microbial-mediated therapeutic responses.ConclusionsThis
study represents the first large-scale, cross-cohort integration of
metagenomic data to identify reproducible GM signatures predictive of
immune checkpoint inhibitor efficacy in LC. The findings not only
underscore the prognostic relevance of specific taxa but also establish
a foundation for developing microbiome-informed, personalized
immunotherapeutic strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1591076"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1591076</a></p>
<hr />
<h2
id="prediction-of-bacteremia-using-routine-hematological-and-metabolic-parameters-based-on-logistic-regression-and-random-forest-models">Prediction
of bacteremia using routine hematological and metabolic parameters based
on logistic regression and random forest models</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThis study aimed to evaluate the predictive utility of
routine hematological, inflammatory, and metabolic markers for
bacteremia and to compare the classification performance of logistic
regression and random forest models.MethodsA retrospective study was
conducted on 287 inpatients who underwent blood culture testing at
Fuding Hospital, Fujian University of Traditional Chinese Medicine
between March and August 2024. Patients were divided into bacteremia (n
= 137) and non-bacteremia (n = 150) groups based on blood culture
results. Hematological indices, inflammatory markers (e.g., C-reactive
protein (CRP), procalcitonin (PCT)), metabolic indices (e.g., glucose,
cholesterol) and nutritional markers (e.g., albumin) were analyzed.
Univariate and multivariate binary logistic regression analyses were
used to identify independent risk factors. Logistic regression and
random forest models were developed using 33 features with a 70:30
train-test split and evaluated using the receiver operating
characteristic (ROC) curves, confusion matrices and standard
classification.ResultsHemoglobin, cholesterol, and albumin levels were
significantly lower in the bacteremia group, while platelet count, CRP,
PCT, glucose, and triglycerides were significantly elevated (all p &lt;
0.05). Logistic regression identified platelet count (Odds ratios (OR) =
1.003, 95% confidence interval (CI): 1.001–1.006), PCT (OR = 1.032, 95%
CI: 1.004–1.060), triglycerides (OR = 1.740, 95% CI: 1.052–2.879), and
low cholesterol (OR = 0.523, 95% CI: 0.383–0.714) as independent risk
factors. The area under the ROC curve (AUC) was 0.75 for the random
forest model and 0.74 for logistic regression, with recall rates of 0.69
and 0.60, respectively.ConclusionRoutine laboratory markers integrated
into machine learning models demonstrated potential for early bacteremia
prediction. Random forest exhibited superior sensitivity compared to
logistic regression, suggesting its potential utility as a clinical
screening tool.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1605485"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1605485</a></p>
<hr />
<h2
id="drosophila-melanogaster-as-a-model-system-for-studying-the-effects-of-porcine-rotavirus-on-intestinal-immunity">Drosophila
melanogaster as a model system for studying the effects of porcine
rotavirus on intestinal immunity</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionDrosophila melanogaster is a quintessential model
organism that has been used in many scientific studies. The intestinal
immune response of flies is a critical component of their innate immune
system. Given that flies primarily consume decaying organic matter,
harmful microorganisms present in their food can enter the intestine,
leading to frequent infections by exogenous pathogens. When these
pathogens are introduced into the intestinal environment, a cascade of
immune responses is triggered within the intestinal tissue, aimed at
preserving the integrity of the intestinal barrier and ensuring the
proper physiological functions of the gut. Porcine rotavirus (PoRV) is a
key pathogen that causes diarrhea in pigs, and PoRV infection can
significantly reduce piglet survival rates.MethodsIn this study,
wild-type flies were orally administered PoRV to establish an effective
intestinal damage animal model, and a detailed investigation of the
antiviral immune defense mechanism in the fly intestine was
performed.Results and DiscussionOur study revealed that PoRV infection
caused a reduction in the survival rate of flies and an increase in
intestinal epithelial cell death. Concurrently, PoRV infection
significantly promoted the proliferation and differentiation of
intestinal cells, contributing to the maintenance of intestinal
homeostasis. After the activation of JAK/STAT signaling in the
intestines of infected Drosophila, there was an increase in the levels
of reactive oxygen species (ROS). This elevation was concomitant with
the release of antimicrobial peptides (AMPs), which play a crucial role
in pathogen clearance. Additionally, we identified substantial
aggregation of hemocytes in the midgut. The composition of the
intestinal microbiota also underwent changes, potentially playing a role
in intestinal immune defense. Moreover, PoRV can evade clearance via the
RNA interference (RNAi) pathway. In summary, PoRV infection in the fly
intestine activates multiple immune defense mechanisms to eliminate the
pathogen, offering a theoretical basis for PoRV prevention and
control.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1621846"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1621846</a></p>
<hr />
<h2
id="lilra5-macrophages-drive-early-oxidative-stress-surge-in-sepsis-a-single-cell-transcriptomic-landscape-with-therapeutic-implications">LILRA5+
macrophages drive early oxidative stress surge in sepsis: a single-cell
transcriptomic landscape with therapeutic implications</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundIn sepsis, oxidative stress (OS) triggers essential
adaptive responses and emerging OS-related biomarkers show potential for
enhancing sepsis diagnosis and therapy.MethodologyIn this study, we used
single-cell datasets and the OS gene set to identify immune cell types
with the highest oxidative activity across different sepsis states.
Differential expression genes (DEG) between “high state” cells and “low
state” cells were screened. High-dimensional weighted gene co-expression
network analysis (hdWGCNA), combined with multiple machine learning
methods, was used for the selection of hub genes. Expressions of hub
genes were then validated. Cell–cell communication and transcription
factor analysis were performed later. Real-time quantitative reverse
transcription (qRT-PCR) and Western blotting validated expression of
LILRA5 in both the cecal ligation and puncture (CLP) model and the
lipopolysaccharide-induced sepsis model. Reactive oxygen species (ROS)
levels were also detected in THP-1 cells after silencing
LILRA5.ResultsIn the early stages of sepsis, oxidative activity reaches
its peak, with macrophages displaying the highest OS among all cell
types. Through the application of the “Quartile method”, all cells were
clustered into three states based on OS activity (low, medium, and
high). LILRA5, MGST1, PLBD1, and S100A9 were selected as hub genes and
significantly upregulated in sepsis. LILRA5 was predominantly expressed
in macrophages and was highly expressed in the early stage of
macrophage. Specifically, LILRA5+ macrophages exhibit the strongest OS.
LILRA5 showed a higher expression in both mouse sepsis models and the
THP-1 cell after lipopolysaccharide stimulation. Silencing LILRA5
resulted in a significant reduction of ROS in THP-1 cells.ConclusionIn
conclusion, our study has mapped the landscape of OS dynamics in sepsis
and found that LILRA5+ macrophages in the early stage of sepsis exhibit
the highest OS. LILRA5 emerges as a promising gene for modulating
macrophage-mediated OS in sepsis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1606401"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1606401</a></p>
<hr />
<h2
id="advances-in-nanotechnology-for-the-therapy-of-bacterial-pneumonia">Advances
in nanotechnology for the therapy of bacterial pneumonia</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Bacterial pneumonia, a life-threatening infection, is the world’s
sixth deadliest disease and the top cause of mortality in children under
five. Without timely treatment, bacterial pneumonia can escalate to a
30% mortality rate, particularly in high-risk populations. It may also
lead to chronic conditions such as pulmonary fibrosis and chronic
obstructive pulmonary disease(COPD), along with systemic inflammatory
responses that can progress to sepsis and multi-organ failure. Although
antibiotics are generally effective against bacterial pneumonia, current
treatment approaches remain insufficient due to several barriers,
including the lung’s unique mucus barrier, low pH, high oxidative
stress, disruption of alveolar surfactants, and accumulation of
hypertonic fluid on the airway surface. In addition, following the
excessive use of antibiotics, dysbiosis, secondary infections and
resistance occur. Nanomaterials can be an effective way to improve
therapeutic effects owing to their change on drug size, physicochemical
properties, hydrophobicity along with better targeting ability, and
controlled localized release. Organic and inorganic substances and their
composites are the three main types of nanomaterials to treat bacterial
pneumonia. This review presents the latest advancements and constraints
of these nanomaterials from a nanotechnology viewpoint with a view to
developing therapeutic strategies for bacterial pneumonia.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1639783"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1639783</a></p>
<hr />
<h2
id="respiratory-microbiota-diversity-and-composition-in-recurrent-protracted-bacterial-bronchitis-a-cross-sectional-study">Respiratory
microbiota diversity and composition in recurrent protracted bacterial
bronchitis: a cross-sectional study</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionRecurrent protracted bacterial bronchitis (RPBB) is a
significant risk factor for bronchiectasis in children, characterized by
multiple episodes of protracted bacterial bronchitis (PBB) annually.
With an increasing global incidence, a detailed understanding of RPBB’s
pathophysiology is essential, particularly regarding the role of lung
microbiota.MethodsThis cross-sectional study recruited 39 children from
Jinhua Maternal and Child Health Hospital between January 2021 and
December 2022, including 18 with PBB, 11 with RPBB, and 10 as controls.
Bronchoscopy with bronchoalveolar lavage (BAL) was performed to collect
lung microbiota samples, which were analyzed using 16S rDNA sequencing.
Microbial diversity and composition differences among groups were
assessed using alpha and beta diversity metrics, PERMANOVA, and Linear
Discriminant Analysis Effect Size (LEfSe), with statistical significance
set at P &lt; 0.05.ResultsRPBB patients exhibited a distinct lung
microbiota composition compared to controls, characterized by an
increased abundance of pathogens such as Acinetobacter and Mycoplasma,
alongside a reduction in beneficial genera like Streptococcus and
Granulicatella. The RPBB group also demonstrated greater overall
microbiota diversity, indicating dysbiosis that may contribute to
disease severity and persistent respiratory symptoms.ConclusionThis
study revealed significant alterations in the lung microbiota of
children with RPBB, suggesting that microbial imbalance could play a
crucial role in disease pathogenesis. These findings highlight the
importance of targeted prevention and therapeutic strategies aimed at
restoring microbiota balance to improve pediatric respiratory
health.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1524116"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1524116</a></p>
<hr />
<h2
id="respiratory-microbiome-and-metabolome-features-associate-disease-severity-and-the-need-for-doxycycline-treatment-in-children-with-macrolide-resistant-mycoplasma-pneumoniae-mediated-pneumonia">Respiratory
microbiome and metabolome features associate disease severity and the
need for doxycycline treatment in children with macrolide-resistant
Mycoplasma pneumoniae-mediated pneumonia</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCommensal bacterial community along the upper respiratory
tract functions against pathogens. The host determinants of Mycoplasma
pneumoniae severity should be identified against the increasing threat
of macrolide-resistant M. pneumoniae (MRMP) infection. We hypothesized
that respiratory microbiome is involved in the clinical manifestations
of M. pneumoniae infection.MethodsFrom 2017 to 2020, 92 children with
MRMP-mediated pneumonia were enrolled among 845 children with
community-associated pneumonia. Oropharyngeal samplings were collected
within 48 h after admission. We compared respiratory microbiome and
metabolites based on patients’ later development of prolonged fever and
the need for doxycycline treatment (DT, n = 57) and the cured control
without fever or doxycycline treatment (WDT, n = 35) by using 16S
rRNA-based sequencing and untargeted metabolome
analysis.ResultsSignificantly higher diversity and different respiratory
microbiomes were evaluated in WDT patients in contrast to DT patients.
Fusobacterium, Haemophilus, Gemella, Oribacterium, Actinomyces lingnae,
Fusobacterium periodonticum, Gemella sanguinis, and Solobacterium moorei
were inversely correlated with disease severity. We assumed that
metabolites of divergent microbiomes were related to MRMP development.
We identified 15 discriminative amino-acid- and fatty-acid-related
metabolites in two groups. F. periodonticum abundance was negatively
associated with an inflammatory metabolite: a platelet-activating
factor. Fusobacterium and Oribacterium were related to the decrease in
LysoPE(18:1(9Z)/0:0) and LysoPC(18:1(9Z)).ConclusionsMicrobiota
dysbiosis with dysregulated inflammatory glycerolphospholipid-related
metabolites was related to disease severity and the need for doxycycline
treatment in children with MRMP-mediated pneumonia. Anaerobic bacteria
metabolites and metabolic pathway could be beneficial therapeutic
targets against M. pneumoniae infection.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1537182"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1537182</a></p>
<hr />
<h2
id="diagnosis-of-non-puerperal-mastitis-based-on-whole-tongue-features-non-invasive-biomarker-mining-and-diagnostic-model-construction">Diagnosis
of non-puerperal mastitis based on “whole tongue” features: non-invasive
biomarker mining and diagnostic model construction</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundNon-puerperal mastitis (NPM) arises from heterogeneous
factors ranging from autoimmune dysregulation to occult infections. To
establish a diagnosis, biopsy is reliable but invasive. Imaging exhibits
a limited specificity and may cause diagnostic delays, patient
discomfort, and suboptimal management. Inspired by non-invasive tongue
diagnosis in traditional Chinese medicine, this study integrated
tongue-coating microbiota profiling and AI-quantified tongue image
phenotyping to establish an objective, non-invasive diagnostic framework
for NPM.MethodsA total of 100 NPM patients from the Breast Surgery
Department of Longhua Hospital and 100 healthy volunteers were included.
Their clinical characteristics, tongue images, and tongue-coating
microbiota data were collected. Features of tongue images (detection,
segmentation, and classification) were quantitated and extracted via
deep learning. The microbiota composition was assessed using 16S rRNA
gene sequencing (V3–V4 region) and bioinformatic pipelines (QIIME2,
DADA2). Based on clinical, imaging, and microbial features, three
machine learning models—logistic regression (LR), support vector machine
(SVM), and gradient boosting decision tree (GBDT)—were trained to
distinguish NPM.ResultsThe GBDT model achieved a superior diagnostic
performance (AUROC = 0.98, accuracy = 0.95, and specificity = 0.95),
outperforming the LR (AUROC = 0.98, accuracy = 0.95, and specificity =
0.90) and SVM models (AUROC = 0.87, accuracy = 0.80, and specificity =
0.75). Integration of clinical characteristics, tongue image features,
and bacterial profiles (at the genus/family level) yielded the highest
accuracy, whereas models using a single class of features showed a lower
discriminatory ability (AUROC = 0.90–0.91). Key predictors included
Campylobacter (12%), waist–hip ratio (11%), and Alloprevotella
(6%).ConclusionsIntegrating clinical characteristics, tongue image
features, and tongue-coating microbiota profiles, the multimodal GBDT
model demonstrates a high diagnostic accuracy, supporting its utility
for early screening and diagnosis of NPM.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602883"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602883</a></p>
<hr />
<h2
id="a-breakthrough-trichosporon-asahii-infection-in-an-immune-thrombocytopenia-patient-during-caspofungin-and-isavuconazole-combined-therapy-a-case-report">A
breakthrough Trichosporon asahii infection in an immune thrombocytopenia
patient during caspofungin and isavuconazole combined therapy: a case
report</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Trichosporon infection is a rare but highly lethal infectious
disease. Clinically, Trichosporon asahii is the strain most commonly
isolated from patients with Trichosporon infections. T. asahii is an
opportunistic pathogen that can cause local or invasive infections in
immunocompromised patients. In this article, a case of breakthrough T.
asahii infection in an immune thrombocytopenia (ITP) patient during
caspofungin and isavuconazole combined therapy is reported. Metagenomic
next-generation sequencing (mNGS) played a crucial role in the diagnosis
of the breakthrough infection in this patient. Upon receiving combined
therapy with intravenous voriconazole and nebulized amphotericin B, the
patient demonstrated significant clinical improvement and was
subsequently discharged.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1625007"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1625007</a></p>
<hr />
<h2
id="prevalence-and-genomic-insights-into-type-iii-a-crispr-cas-system-acquisition-in-global-staphylococcus-argenteus-strains">Prevalence
and genomic insights into type III-A CRISPR-Cas system acquisition in
global Staphylococcus argenteus strains</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe CRISPR-Cas system serves as a defense mechanism in
bacteria and archaea, protecting them against the invasion of mobile
genetic elements. Staphylococcus argenteus, a Gram-positive bacterium
that diverged from Staphylococcus aureus, is characterized by the rare
presence of the CRISPR-Cas system in only a few isolates. MethodsIn this
study, we analyzed the prevalence of the type III-A CRISPR-Cas system in
368 S. argenteus genome sequences from animals, food sources, and humans
across 26 countries, available in public database. ResultsOur findings
revealed that 44.0% of these strains carry this immune system, with
98.1% of them belonging to the sequence type 2250 (ST2250). Genomic
localization analysis indicated that the CRISPR-Cas is closely
associated with SCCmec (mecA-ΔmecR1-IS1272-ccrB2-ccrA2) or Insertion
sequence 1272 (IS1272) transposase. Further analysis identified a common
IS1272 target inverted repeats (IR) sequence in ST2250 strains,
providing insights into why these strains are more likely to acquire the
CRISPR-Cas system. CRISPR typing identified 41 sequences types,
classifying these strains into two clusters, with Cluster II being the
predominant one. Homology analysis of spacers revealed that all the
identified 15 spacers exhibited homology to sequences from plasmids,
lytic phages, or prophages. ConclusionThis study suggests that the
acquisition of the CRISPR-Cas system in S. argenteus enhances its
resistance to phage attacks and plasmid invasions in environmental
settings, potentially posing significant challenges for clinical
treatment of infections caused by these strains and hindering efforts to
control their spread in food products using phage-based
interventions.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1644286"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1644286</a></p>
<hr />
<h2
id="comparative-evaluation-of-maldi-tof-mass-spectrometry-and-sanger-sequencing-of-the-16s-hsp65-and-rpob-genes-for-non-tuberculous-mycobacteria-species-identification">Comparative
evaluation of MALDI-ToF mass spectrometry and Sanger sequencing of the
16S, hsp65, and rpoB genes for non tuberculous mycobacteria species
identification</h2>
<p>date: 2025-07-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Non tuberculous mycobacteria (NTM) infections are increasing
globally, underscoring the critical importance of accurate species-level
identification for effective clinical management. This study aimed to
evaluate the use of three conserved markers in the mycobacterial family
(16S, hsp65, and rpoB) for NTM identification through Sanger sequencing,
comparing the results to those obtained using MALDI-ToF MS. A total of
59 clinical NTM isolates from plastic surgery patients, previously
characterized by MALDI-ToF MS, were analyzed. These isolates underwent
DNA extraction, PCR amplification, and Sanger sequencing. Species
identification was performed through phylogenetic analyses of each
marker individually and concatenated as a multi locus sequencing
approach. Concordance between MALDI-ToF MS and Sanger sequencing was
assessed using Cohen’s Kappa statistical analysis. Cohen’s Kappa values
indicated moderate concordance of 0.46 for 16S, 0.51 for hsp65, and 0.69
for rpoB. Concatenated phylogenetic analysis yielded improved
concordance values of 0.71 for (16S + hsp65), 0.76 for (16S + rpoB),
0.69 for (rpoB + hsp65), and 0.72 for (16S + hsp65 + rpoB). Our results
show that NTM identification is more accurate when employing a multi
locus sequencing approach. Notably, the combination of 16S + rpoB
outperformed the three-marker concatenation, offering the highest
concordance for species-level identification. NTM identification is
challenging, and concatenated phylogenetic analysis of two or more gene
fragments should be used when MALDI-ToF MS or whole genome sequencing is
not available.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1612459"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1612459</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
