<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.08</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="parks.html" title="News about Parks">Parks</a></li>
	<li><a href="tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="2023/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.08">health 2024.08</h1>
<p>(date: 2024-02-20 09:25:19)</p>
<hr />
<h2
id="covid-19-in-pulmonary-critically-ill-patients-metagenomic-identification-of-fungi-and-characterization-of-pathogenic-microorganisms">COVID-19
in pulmonary critically ill patients: metagenomic identification of
fungi and characterization of pathogenic microorganisms</h2>
<p>date: 2024-02-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Fungal co-infection is prevalent in critically ill patients with
COVID-19. The conventional approach applied to fungal identification has
relatively low sensitivity and is time-consuming. The metagenomic
next-generation sequencing (mNGS) technology can simultaneously detect a
variety of microorganisms, and is increasingly being used for the rapid
detection and diagnosis of pathogens.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
In this single-center retrospective study, we described the clinical
presentation and outcomes of COVID-19 and mNGS positive for fungi in
pulmonary critically ill patients during the outbreak of Omicron
infection from December 2022 to January 2023.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Among 43 COVID-19 patients with acute respiratory distress syndrome
(ARDS) on a single intensive care unit (ICU), 10 were reported to be
fungal positive using the mNGS test. The number of pathogenic
microorganisms detected by mNGS was significantly higher than that via
traditional methods, especially in the detection of fungi and viruses.
<italic>Aspergillus</italic> infection was dominant, and most of these
patients also had concurrent bacterial or viral infections. Probable or
possible COVID-19-associated pulmonary aspergillosis (CAPA) was
diagnosed in all 10 patients, and the prognosis was poor.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Patients with COVID-19 may be at increased risk of developing fungal
infections as well as concurrent bacterial or viral infections, and mNGS
can be a powerful tool in identifying these infections. Clinicians
should be aware of the increased risk of fungal infections in COVID-19
patients, particularly those who have underlying immunocompromising
conditions, and should monitor for early signs of infection.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012">Save
to Pocket</a> </span></p>
<hr />
<h2
id="effects-of-plant-natural-products-on-metabolic-associated-fatty-liver-disease-and-the-underlying-mechanisms-a-narrative-review-with-a-focus-on-the-modulation-of-the-gut-microbiota">Effects
of plant natural products on metabolic-associated fatty liver disease
and the underlying mechanisms: a narrative review with a focus on the
modulation of the gut microbiota</h2>
<p>date: 2024-02-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Metabolic-associated fatty liver disease (MAFLD) is a chronic liver
disease characterized by the excessive accumulation of fat in
hepatocytes. However, due to the complex pathogenesis of MAFLD, there
are no officially approved drugs for treatment. Therefore, there is an
urgent need to find safe and effective anti-MAFLD drugs. Recently, the
relationship between the gut microbiota and MAFLD has been widely
recognized, and treating MAFLD by regulating the gut microbiota may be a
new therapeutic strategy. Natural products, especially plant natural
products, have attracted much attention in the treatment of MAFLD due to
their multiple targets and pathways and few side effects. Moreover, the
structure and function of the gut microbiota can be influenced by
exposure to plant natural products. However, the effects of plant
natural products on MAFLD through targeting of the gut microbiota and
the underlying mechanisms are poorly understood. Based on the above
information and to address the potential therapeutic role of plant
natural products in MAFLD, we systematically summarize the effects and
mechanisms of action of plant natural products in the prevention and
treatment of MAFLD through targeting of the gut microbiota. This
narrative review provides feasible ideas for further exploration of
safer and more effective natural drugs for the prevention and treatment
of MAFLD.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261">Save
to Pocket</a> </span></p>
<hr />
<h2 id="advances-in-application-of-crispr-cas13a-system">Advances in
application of CRISPR-Cas13a system</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) and
CRISPR-associated (Cas) proteins serve as an adaptive immune system that
safeguards prokaryotes and some of the viruses that infect prokaryotes
from foreign nucleic acids (such as viruses and plasmids). The genomes
of the majority of archaea and about half of all bacteria contain
various CRISPR-Cas systems. CRISPR-Cas systems depend on CRISPR RNAs
(crRNAs). They act as a navigation system to specifically cut and
destroy foreign nucleic acids by recognizing invading foreign nucleic
acids and binding Cas proteins. In this review, we provide a brief
overview of the evolution and classification of the CRISPR-Cas system,
focusing on the functions and applications of the CRISPR-Cas13a system.
We describe the CRISPR-Cas13a system and discuss its RNA-directed
ribonuclease function. Meanwhile, we briefly introduce the mechanism of
action of the CRISPR-Cas13a system and summarize the applications of the
CRISPR-Cas13a system in pathogen detection, eukaryotes, agriculture,
biosensors, and human gene therapy. We are right understanding of
CRISPR-Cas13a has been broadened, and the CRISPR-Cas13a system will be
useful for developing new RNA targeting tools. Therefore, understanding
the basic details of the structure, function, and biological
characterization of CRISPR-Cas13a effector proteins is critical for
optimizing RNA targeting tools.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557">Save
to Pocket</a> </span></p>
<hr />
<h2
id="excess-fermentation-and-lactic-acidosis-as-detrimental-functions-of-the-gut-microbes-in-treatment-naive-tb-patients">Excess
fermentation and lactic acidosis as detrimental functions of the gut
microbes in treatment-naive TB patients</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
The link between gut microbiota and host immunity motivated numerous
studies of the gut microbiome in tuberculosis (TB) patients. However,
these studies did not explore the metabolic capacity of the gut
community, which is a key axis of impact on the host’s immunity.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We used deep sequencing of fecal samples from 23 treatment-naive TB
patients and 48 healthy donors to reconstruct the gut microbiome’s
metabolic capacity and strain/species-level content.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We show that the systematic depletion of the commensal flora of the
large intestine, <italic>Bacteroidetes</italic>, and an increase in
<italic>Actinobacteria, Firmicutes</italic>, and
<italic>Proteobacteria</italic> such as <italic>Streptococcaceae,
Erysipelotrichaceae, Lachnospiraceae</italic>, and
<italic>Enterobacteriaceae</italic> explains the strong taxonomic
divergence of the gut community in TB patients. The cumulative expansion
of diverse disease-associated pathobionts in patients reached 1/4 of the
total gut microbiota, suggesting a heavy toll on host immunity along
with MTB infection. Reconstruction of metabolic pathways showed that the
microbial community in patients shifted toward rapid growth using
glycolysis and excess fermentation to produce acetate and lactate.
Higher glucose availability in the intestine likely drives fermentation
to lactate and growth, causing acidosis and endotoxemia.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Excessive fermentation and lactic acidosis likely characterize TB
patients’ disturbed gut microbiomes. Since lactic acidosis strongly
suppresses the normal gut flora, directly interferes with macrophage
function, and is linked to mortality in TB patients, our findings
highlight gut lactate acidosis as a novel research focus. If confirmed,
gut acidosis may be a novel potential host-directed treatment target to
augment traditional TB treatment.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521">Save
to Pocket</a> </span></p>
<hr />
<h2
id="editorial-precision-medicine-in-viral-hepatitis-progress-and-prospects-towards-elimination">Editorial:
Precision medicine in viral hepatitis: progress and prospects towards
elimination</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423">Save
to Pocket</a> </span></p>
<hr />
<h2
id="identification-of-potential-novel-proteomic-markers-of-leishmania-spp.-derived-exosomes">Identification
of potential novel proteomic markers of Leishmania spp.-derived
exosomes</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Extracellular vesicles (EVs) are heterogenous cell-derived
membrane-bound structures which can be subdivided into three distinct
classes according to distinct morphological characteristics, cellular
origins, and functions. Small EVs, or exosomes, can be produced by the
protozoan parasite <italic>Leishmania</italic> through the
evolutionarily conserved ESCRT pathway, and act as effectors of
virulence and drivers of pathogenesis within mammalian hosts. Techniques
for the identification of EVs of non-mammalian origin, however, remain
inaccurate in comparison to their well-characterized mammalian
counterparts. Thus, we still lack reliable and specific markers for
<italic>Leishmania</italic>-derived exosomes, which poses a significant
challenge to the field.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Herein, we utilized serial differential ultracentrifugation to separate
<italic>Leishmania</italic>-derived EV populations into three distinct
fractions. Nanoparticle tracking analysis and transmission electron
microscopy were used to validate their morphological characteristics,
and bioinformatic analysis of LC-MS/MS proteomics corroborated cellular
origins and function.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Proteomic data indicated potential novel proteic markers of
<italic>Leishmania</italic>-derived exosomes, including proteins
involved in endosomal machinery and the ESCRT pathway, as well as the
parasitic phosphatase PRL-1. Further investigation is required to
determine the specificity and sensitivity of these markers.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636">Save
to Pocket</a> </span></p>
<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
