<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.10</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="parks.html" title="News about Parks">Parks</a></li>
	<li><a href="tech_likely.html" title="likely tech news fed but other stuff happens too">Tech Likely</a></li>
	<li><a href="libraries.html" title="Libraries, Archives and Museam related stuff happens too">Libraries</a></li>
	<li><a href="weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="2023/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health-2024.10">health 2024.10</h1>
<p>(date: 2024-03-07 07:33:07)</p>
<hr />
<h2
id="antibiotic-resistance-of-eskape-group-microorganisms-in-health-institutions-from-hermosillo-and-ciudad-obregón-sonora-méxico">Antibiotic
resistance of ESKAPE group-microorganisms in health institutions from
Hermosillo and Ciudad Obregón, Sonora, México</h2>
<p>date: 2024-03-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
<italic>Enterococcus faecium, Staphylococcus aureus</italic>,
<italic>Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas
aeruginosa</italic>, and <italic>Enterobacter</italic> spp. are
microorganisms referred as the ESKAPE group pathogens. These
microorganisms have generated great concern in health institutions
around the world since most of them have resistance to multiple
antibiotics and cause most infections associated with healthcare, as
well as community infections. The aim of this study was the analysis of
antibiotic resistance in microorganisms of the ESKAPE group, recovered
from clinical samples in 11 health institutions from Hermosillo and
Ciudad Obregón in the State of Sonora, México, during the period from
2019 to 2020.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
A cross-sectional, descriptive, observational, and temporality
epidemiological study was carried out. A comparative and statistical
analysis of antibiotic resistance was carried out using the chi-square
test, and small values were analyzed using Fisher’s exact test p ≤ 0.05.
</p>
</sec><sec>
<title>
Results and discussion
</title>
<p>
All the ESKAPE group microorganisms showed significant differences in
antibiotic resistance percentages between both cities. High resistance
percentages for some antibiotics, like cephalosporins and ciprofloxacin
were detected for <italic>Klebsiella pneumoniae</italic> and
<italic>Acinetobacter baumannii</italic>.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348093">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348093</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348093">Save
to Pocket</a> </span></p>
<hr />
<h2
id="pex1-is-essential-for-glycosome-biogenesis-and-trypanosomatid-parasite-survival">PEX1
is essential for glycosome biogenesis and trypanosomatid parasite
survival</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Trypanosomatid parasites are kinetoplastid protists that
compartmentalize glycolytic enzymes in unique peroxisome-related
organelles called glycosomes. The heterohexameric AAA-ATPase complex of
PEX1-PEX6 is anchored to the peroxisomal membrane and functions in the
export of matrix protein import receptor PEX5 from the peroxisomal
membrane. Defects in PEX1, PEX6 or their membrane anchor causes
dysfunction of peroxisomal matrix protein import cycle. In this study,
we functionally characterized a putative <italic>Trypanosoma</italic>
PEX1 orthologue by bioinformatic and experimental approaches and show
that it is a true PEX1 orthologue. Using yeast two-hybrid analysis, we
demonstrate that <italic>Tb</italic>PEX1 can bind to
<italic>Tb</italic>PEX6. Endogenously tagged <italic>Tb</italic>PEX1
localizes to glycosomes in the <italic>T. brucei</italic> parasites.
Depletion of PEX1 gene expression by RNA interference causes lethality
to the bloodstream form trypanosomes, due to a partial mislocalization
of glycosomal enzymes to the cytosol and ATP depletion.
<italic>Tb</italic>PEX1 RNAi leads to a selective proteasomal
degradation of both matrix protein import receptors
<italic>Tb</italic>PEX5 and <italic>Tb</italic>PEX7. Unlike in yeast,
PEX1 depletion did not result in an accumulation of ubiquitinated
<italic>Tb</italic>PEX5 in trypanosomes. As PEX1 turned out to be
essential for trypanosomatid parasites, it could provide a suitable drug
target for parasitic diseases. The results also suggest that these
parasites possess a highly efficient quality control mechanism that
exports the import receptors from glycosomes to the cytosol in the
absence of a functional <italic>Tb</italic>PEX1-<italic>Tb</italic>PEX6
complex.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1274506">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1274506</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1274506">Save
to Pocket</a> </span></p>
<hr />
<h2
id="the-cutaneous-beta-human-papillomavirus-type-8-e6-protein-induces-ccl2-through-the-cebpαmir-203p63-pathway-to-support-an-inflammatory-microenvironment-in-epidermodysplasia-verruciformis-skin-lesions">The
cutaneous beta human papillomavirus type 8 E6 protein induces CCL2
through the CEBPα/miR-203/p63 pathway to support an inflammatory
microenvironment in epidermodysplasia verruciformis skin lesions</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Human papillomavirus type 8 (HPV8), a cutaneous genus beta HPV type, has
co-carcinogenic potential at sun-exposed sites in patients suffering
from the inherited skin disease epidermodysplasia verruciformis (EV). We
had previously shown that Langerhans cells responsible for epithelial
immunosurveillance were strongly reduced at infected sites and that the
HPV8 E7 protein interferes with the CCAAT/enhancer-binding protein
(C/EBP)β to suppress the Langerhans cell chemokine CCL20. At the same
time, however, we observed that EV lesions are heavily infiltrated with
inflammatory immune cells, which is similar to the situation in HPV8 E6
transgenic mice. To identify critical inflammatory factors, we used a
broad multiplex approach and found that the monocyte attracting
chemokine CCL2 was significantly and strongly induced by HPV8 E6 but not
E7-expressing HaCaT cells, which were used as a model for UV-damaged
skin keratinocytes. Conditioned media from HPV8 E6-expressing
keratinocytes enhanced CCL2-receptor (CCR2)-dependent monocyte
recruitment <italic>in vitro</italic>, and macrophages predominated in
the stroma but were also detected in the epidermal compartment of EV
lesions <italic>in vivo</italic>. CCL2 induction by HPV8 E6 was even
stronger than stimulation with the proinflammatory cytokine TNF-α, and
both HPV8 E6 and TNF-α resulted in substantial suppression of the
transcription factor C/EBPα. Using RNAi-mediated knockdown and
overexpression approaches, we demonstrated a mechanistic role of the
recently identified C/EBPα/miR-203/p63 pathway for HPV8 E6-mediated CCL2
induction at protein and transcriptional levels. Epithelial
co-expression of p63 and CCL2 was confirmed in HPV8 E6-expressing
organotypic air–liquid interface cultures and in lesional EV epidermis
<italic>in vivo</italic>. In summary, our data demonstrate that HPV8
oncoproteins actively deregulate epidermal immune homeostasis through
modulation of C/EBP factor-dependent pathways. While HPV8 E7 suppresses
immunosurveillance required for viral persistence, the present study
provides evidence that E6 involves the stemness-promoting factor p63 to
support an inflammatory microenvironment that may fuel carcinogenesis in
EV lesions.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336492">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336492</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336492">Save
to Pocket</a> </span></p>
<hr />
<h2
id="exploring-the-role-of-antioxidants-in-sepsis-associated-oxidative-stress-a-comprehensive-review">Exploring
the role of antioxidants in sepsis-associated oxidative stress: a
comprehensive review</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Sepsis is a potentially fatal condition characterized by organ
dysfunction caused by an imbalanced immune response to infection.
Although an increased inflammatory response significantly contributes to
the pathogenesis of sepsis, several molecular mechanisms underlying the
progression of sepsis are associated with increased cellular reactive
oxygen species (ROS) generation and exhausted antioxidant pathways. This
review article provides a comprehensive overview of the involvement of
ROS in the pathophysiology of sepsis and the potential application of
antioxidants with antimicrobial properties as an adjunct to primary
therapies (fluid and antibiotic therapies) against sepsis. This article
delves into the advantages and disadvantages associated with the
utilization of antioxidants in the therapeutic approach to sepsis, which
has been explored in a variety of animal models and clinical trials.
While the application of antioxidants has been suggested as a potential
therapy to suppress the immune response in cases where an intensified
inflammatory reaction occurs, the use of multiple antioxidant agents can
be beneficial as they can act additively or synergistically on different
pathways, thereby enhancing the antioxidant defense. Furthermore, the
utilization of immunoadjuvant therapy, specifically in septic patients
displaying immunosuppressive tendencies, represents a promising
advancement in sepsis therapy.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348713">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348713</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348713">Save
to Pocket</a> </span></p>
<hr />
<h2
id="validating-the-inactivation-of-viral-pathogens-with-a-focus-on-sars-cov-2-to-safely-transfer-samples-from-high-containment-laboratories">Validating
the inactivation of viral pathogens with a focus on SARS-CoV-2 to safely
transfer samples from high-containment laboratories</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Pathogen leak from a high-containment laboratory seriously threatens
human safety, animal welfare, and environmental security. Transportation
of pathogens from a higher (BSL4 or BSL3) to a lower (BSL2) containment
laboratory for downstream experimentation requires complete pathogen
inactivation. Validation of pathogen inactivation is necessary to ensure
safety during transportation. This study established a validation
strategy for virus inactivation.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
SARS-CoV-2 wild type, delta, and omicron variants underwent heat
treatment at 95°C for 10 minutes using either a hot water bath or a
thermocycler. To validate the inactivation process, heat-treated
viruses, and untreated control samples were incubated with A549-hACE2
and Vero E6-TMPRSS2-T2A-ACE2 cells. The cells were monitored for up to
72 hours for any cytopathic effects, visually and under a microscope,
and for virus genome replication via RT-qPCR. The quality of
post-treated samples was assessed for suitability in downstream
molecular testing applications.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Heat treatment at 95°C for 10 minutes effectively inactivated SARS-CoV-2
variants. The absence of cytopathic effects, coupled with the inability
of virus genome replication, validated the efficacy of the inactivation
process. Furthermore, the heat-treated samples proved to be qualified
for COVID-19 antigen testing, RT-qPCR, and whole-genome sequencing.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
By ensuring the safety of sample transportation for downstream
experimentation, this validation approach enhances biosecurity measures.
Considerations for potential limitations, comparisons with existing
inactivation methods, and broader implications of the findings are
discussed.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1292467">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1292467</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1292467">Save
to Pocket</a> </span></p>
<hr />
<h2
id="unveiling-the-antibacterial-mechanism-of-resveratrol-against-aeromonas-hydrophila-through-proteomics-analysis">Unveiling
the antibacterial mechanism of resveratrol against Aeromonas hydrophila
through proteomics analysis</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
This investigation delves into elucidating the mechanism by which
resveratrol (Res), a natural polyterpenoid renowned for its
antimicrobial properties, exerts its effects on <italic>Aeromonas
hydrophila</italic>, a ubiquitous waterborne pathogen. Our findings
underscore the dose-dependent manifestation of resveratrol in exhibiting
antibacterial and antibiofilm formation activities against <italic>A.
hydrophila</italic>. Employing a Data-independent acquisition (DIA)
based quantitative proteomics methodology, we systematically compared
differentially expressed proteins in <italic>A. hydrophila</italic>
subjected to varying concentrations of Res. Subsequent bioinformatics
analyses revealed key proteins and pathways pivotal in resveratrol’s
antimicrobial action, encompassing oxidative stress, energy metabolism,
and cell membrane integrity. Validation of the proteomics outcomes was
meticulously conducted using the qPCR method at the mRNA level. Dynamic
trend analysis unveiled alterations in biological processes, notably the
correlation between the cell division-related protein ZapC and
resveratrol content. Furthermore, scanning electron microscopy
corroborated a significant elongation of <italic>A. hydrophila</italic>
cells, affirming resveratrol’s capability to inhibit cell division. In
concert, resveratrol emerges as a participant in the cell membrane
integrity pathway, biofilm formation, and potentially, the regulation of
genes associated with cell division, resulting in morphological
elongation. These revelations position resveratrol as a promising
natural alternative to conventional antibiotics for treating <italic>A.
hydrophila</italic> infections.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378094">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378094</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378094">Save
to Pocket</a> </span></p>
<hr />
<h2
id="a-comprehensive-review-of-monkeypox-virus-and-mpox-characteristics">A
comprehensive review of monkeypox virus and mpox characteristics</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Monkeypox virus (MPXV) is the etiological agent of monkeypox (mpox), a
zoonotic disease. MPXV is endemic in the forested regions of West and
Central Africa, but the virus has recently spread globally, causing
outbreaks in multiple non-endemic countries. In this paper, we review
the characteristics of the virus, including its ecology, genomics,
infection biology, and evolution. We estimate by phylogenomic molecular
clock that the B.1 lineage responsible for the 2022 mpox outbreaks has
been in circulation since 2016. We interrogate the host-virus
interactions that modulate the virus infection biology, signal
transduction, pathogenesis, and host immune responses. We highlight the
changing pathophysiology and epidemiology of MPXV and summarize recent
advances in the prevention and treatment of mpox. In addition, this
review identifies knowledge gaps with respect to the virus and the
disease, suggests future research directions to address the knowledge
gaps, and proposes a One Health approach as an effective strategy to
prevent current and future epidemics of mpox.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360586">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360586</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1360586">Save
to Pocket</a> </span></p>
<hr />
<h2
id="signatures-of-disease-outcome-severity-in-the-intestinal-fungal-and-bacterial-microbiome-of-covid-19-patients">Signatures
of disease outcome severity in the intestinal fungal and bacterial
microbiome of COVID-19 patients</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
COVID-19, whose causative pathogen is the Severe Acute Respiratory
Syndrome Coronavirus-2 (SARS-CoV-2), was declared a pandemic in March
2020. The gastrointestinal tract is one of the targets of this virus,
and mounting evidence suggests that gastrointestinal symptoms may
contribute to disease severity. The gut–lung axis is involved in the
immune response to SARS-CoV-2; therefore, we investigated whether
COVID-19 patients’ bacterial and fungal gut microbiome composition was
linked to disease clinical outcome.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
In May 2020, we collected stool samples and patient records from 24
hospitalized patients with laboratory-confirmed SARS-CoV-2 infection.
Fungal and bacterial gut microbiome was characterized by amplicon
sequencing on the MiSeq, Illumina’s integrated next generation
sequencing instrument. A cohort of 201 age- and sex-matched healthy
volunteers from the project PRJNA661289 was used as a control group for
the bacterial gut microbiota analysis.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We observed that female COVID-19 patients had a lower gut bacterial
microbiota richness than male patients, which was consistent with a
different latency in hospital admittance time between the two groups.
Both sexes in the COVID-19 patient study group displayed multiple
positive associations with opportunistic bacterial pathogens such as
<italic>Enterococcus</italic>, <italic>Streptococcus</italic>, and
<italic>Actinomyces</italic>. Of note, the <italic>Candida</italic>
genus dominated the gut mycobiota of COVID-19 patients, and adult
patients showed a higher intestinal fungal diversity than elderly
patients. We found that <italic>Saccharomycetales unassigned</italic>
fungal genera were positively associated with bacterial short-chain
fatty acid (SCFA) producers and negatively associated with the
proinflammatory genus <italic>Bilophila</italic> in COVID-19 patients,
and we observed that none of the patients who harbored it were admitted
to the high-intensity unit.
</p>
</sec><sec>
<title>
Conclusions
</title>
<p>
COVID-19 was associated with opportunistic bacterial pathogens, and
<italic>Candida</italic> was the dominant fungal taxon in the intestine.
Together, we found an association between commensal SCFA-producers and a
fungal genus that was present in the intestines of patients who did not
experience the most severe outcome of the disease. We believe that this
taxon could have played a role in the disease outcome, and that further
studies should be conducted to understand the role of fungi in
gastrointestinal and health protection.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1352202">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1352202</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1352202">Save
to Pocket</a> </span></p>
<hr />
<h2
id="isolation-and-identification-of-wickerhamiella-tropicalis-from-blood-culture-by-maldi-ms">Isolation
and identification of Wickerhamiella tropicalis from blood culture by
MALDI-MS</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Wickerhamiella</italic> is a genus of budding yeast that is
mainly isolated from environmental samples, and 40 species have been
detected. The yeast isolated from human clinical samples usually only
contain three species: <italic>W. infanticola</italic>, <italic>W.
pararugosa</italic> and <italic>W. sorbophila</italic>. In this study,
we isolated <italic>W. tropicalis</italic> from a blood sample of a
six-year-old female with a history of B-cell precursor lymphoblastic
leukemia in Japan in 2022. Though the strain was morphologically
identified as <italic>Candida</italic> species by routine
microbiological examinations, it was subsequently identified as
<italic>W. tropicalis</italic> by sequencing the internal transcribed
spacer (ITS) of ribosomal DNA (rDNA). The isolate had amino acid
substitutions in ERG11 and FKS1 associated with azole and echinocandin
resistance, respectively, in <italic>Candida</italic> species and showed
intermediate-resistant to fluconazole and micafungin. The patient was
successfully treated with micafungin. Furthermore, matrix-assisted laser
desorption/ionization mass spectrometry (MALDI-MS) detected three novel
peaks that are specific for <italic>W. tropicalis</italic>, indicating
that MALDI-MS analysis is useful for rapid detection of
<italic>Wickerhamiella</italic> species in routine microbiological
examinations.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361432">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361432</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361432">Save
to Pocket</a> </span></p>
<hr />
<h2
id="evaluation-of-novel-compounds-as-anti-bacterial-or-anti-virulence-agents">Evaluation
of novel compounds as anti-bacterial or anti-virulence agents</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Antimicrobial resistance is a global threat, leading to an alarming
increase in the prevalence of bacterial infections that can no longer be
treated with available antibiotics. The World Health Organization
estimates that by 2050 up to 10 million deaths per year could be
associated with antimicrobial resistance, which would equal the annual
number of cancer deaths worldwide. To overcome this emerging crisis,
novel anti-bacterial compounds are urgently needed. There are two
possible approaches in the fight against bacterial infections: a)
targeting structures within bacterial cells, similar to existing
antibiotics; and/or b) targeting virulence factors rather than bacterial
growth. Here, for the first time, we provide a comprehensive overview of
the key steps in the evaluation of potential new anti-bacterial and/or
anti-virulence compounds. The methods described in this review include:
a) <italic>in silico</italic> methods for the evaluation of novel
compounds; b) anti-bacterial assays (MIC, MBC, Time-kill); b)
anti-virulence assays (anti-biofilm, anti-quorum sensing,
anti-adhesion); and c) evaluation of safety aspects (cytotoxicity assay
and Ames test). Overall, we provide a detailed description of the
methods that are an essential tool for chemists, computational chemists,
microbiologists, and toxicologists in the evaluation of potential novel
antimicrobial compounds. These methods are cost-effective and have high
predictive value. They are widely used in preclinical studies to
identify new molecular candidates, for further investigation in animal
and human trials.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1370062">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1370062</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1370062">Save
to Pocket</a> </span></p>
<hr />
<h2
id="isoprinosine-as-a-foot-and-mouth-disease-vaccine-adjuvant-elicits-robust-host-defense-against-viral-infection-through-immunomodulation">Isoprinosine
as a foot-and-mouth disease vaccine adjuvant elicits robust host defense
against viral infection through immunomodulation</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Commercial foot-and-mouth disease (FMD) vaccines have limitations, such
as local side effects, periodic vaccinations, and weak host defenses. To
overcome these limitations, we developed a novel FMD vaccine by
combining an inactivated FMD viral antigen with the small molecule
isoprinosine, which served as an adjuvant (immunomodulator).
</p>
</sec><sec>
<title>
Method
</title>
<p>
We evaluated the innate and adaptive immune responses elicited by the
novel FMD vaccine involved both in vitro and in vivo using mice and
pigs.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We demonstrated isoprinosine-mediated early, mid-term, and long-term
immunity through in vitro and in vivo studies and complete host defense
against FMD virus (FMDV) infection through challenge experiments in mice
and pigs. We also elucidated that isoprinosine induces innate and
adaptive (cellular and humoral) immunity via promoting the expression of
immunoregulatory gene such as pattern recognition receptors [PRRs;
retinoic acid-inducible gene (RIG)-I and toll like receptor (TLR)9],
transcription factors [T-box transcription factor (TBX)21, eomesodermin
(EOMES), and nuclear factor kappa B (NF-kB)], cytokines [interleukin
(IL)-12p40, IL-23p19, IL-23R, and IL-17A)], and immune cell core
receptors [cluster of differentiation (CD)80, CD86, CD28, CD19, CD21,
and CD81] in pigs.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
These findings present an attractive strategy for constructing novel FMD
vaccines and other difficult-to-control livestock virus vaccine
formulations based on isoprinosine induced immunomodulatory functions.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331779">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331779</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331779">Save
to Pocket</a> </span></p>
<hr />
<h2
id="factor-h-related-protein-1-promotes-complement-mediated-opsonization-of-pseudomonas-aeruginosa">Factor
H-related protein 1 promotes complement-mediated opsonization of
Pseudomonas aeruginosa</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Pseudomonas aeruginosa</italic> is an important human
opportunistic pathogen responsible for a wide range of infections. The
complement system is the main early host defense mechanism to control
these infections. <italic>P. aeruginosa</italic> counteracts complement
attack by binding Factor H (FH), a complement regulator that inactivates
C3b, preventing the formation of the C3-convertase and complement
amplification on the bacterial surface. Factor H-related proteins (FHRs)
are a group of plasma proteins evolutionarily related to FH that have
been postulated to interfere in this bacterial mechanism of resisting
complement. Here, we show that FHR-1 binds to <italic>P.
aeruginosa</italic> via the outer membrane protein OprG in a
lipopolysaccharide (LPS) O antigen-dependent manner. Binding assays with
purified components or with FHR-1-deficient serum supplemented with
FHR-1 show that FHR-1 competes with FH for binding to <italic>P.
aeruginosa.</italic> Blockage of FH binding to C3b deposited on the
bacteria reduces FH-mediated cofactor activity of C3b degradation,
increasing the opsonization of the bacteria and the formation of the
potent chemoattractant C5a. Overall, our findings indicate that FHR-1 is
a host factor that promotes complement activation, facilitating
clearance of <italic>P. aeruginosa</italic> by opsonophagocytosis.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1328185">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1328185</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1328185">Save
to Pocket</a> </span></p>
<hr />
<h2
id="editorial-epidemiology-of-antimicrobial-resistance-and-virulence-factors-of-emerging-and-re-emerging-bacteria">Editorial:
Epidemiology of antimicrobial resistance and virulence factors of
emerging and re-emerging bacteria</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1387087">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1387087</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1387087">Save
to Pocket</a> </span></p>
<hr />
<h2
id="erratum-resistance-against-two-lytic-phage-variants-attenuates-virulence-and-antibiotic-resistance-in-pseudomonas-aeruginosa">Erratum:
Resistance against two lytic phage variants attenuates virulence and
antibiotic resistance in Pseudomonas aeruginosa</h2>
<p>date: 2024-03-06, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391783">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391783</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391783">Save
to Pocket</a> </span></p>
<hr />
<h2
id="ursodeoxycholic-acid-does-not-reduce-sars-cov-2-infection-in-newly-allogeneic-hematopoietic-stem-cell-transplantation-recipients-a-prospective-niche-cohort">Ursodeoxycholic
acid does not reduce SARS-CoV-2 infection in newly allogeneic
hematopoietic stem cell transplantation recipients: a prospective NICHE
cohort</h2>
<p>date: 2024-03-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Retrospective studies have suggested that Ursodeoxycholic Acid (UDCA)
provide a protective effect against SARS-CoV-2 infection, particularly
in patients with liver disease. However, it is uncertain whether this
finding can be extended to the allogeneic hematopoietic stem cell
transplantation (allo-HSCT) cohort. Therefore, we aim to examine the
protective potential of UDCA against SARS-CoV-2 infection in recently
received allo-HSCT patients.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
During the initial Omicron variant wave in China (December 2022 to
February 2023), we conducted a prospective observational study involving
91 hospitalized patients who had undergone allo-HSCT within the previous
6 months as part of the National Longitudinal Cohort of Hematological
Diseases (NICHE). Throughout hospitalization, we continuously monitored
the status of COVID-19 using SARS-CoV-2 PCR kits or SARS-CoV-2 Antigen
Rapid Tests.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Among these patients, 67.0% (n = 61) were confirmed to have contracted
SARS-CoV-2 infection. For the 52 patients evaluated, 23.1% experienced a
severe or critical clinical course. There was no difference in the
infection rate or severity of COVID-19 between the UDCA group and the
non-UDCA group. We found that only patients transplanted between 3 and 6
months ago demonstrated a higher risk of SARS-CoV-2 infection compared
to those who received allo-HSCT within 3 months (Odds Ratio [OR]: 3.241,
95% Confidence Interval [CI]: 1.287-8.814, <italic>P</italic> = 0.016).
But other clinical factors, such as administration of UDCA, showed no
difference. Notably, only age ≥38 years old remained as an independent
risk factor for a severe clinical course of SARS-CoV-2 infection (OR:
3.664, 95% CI: 1.129-13.007, <italic>P</italic> = 0.035).
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The effectiveness of UDCA in protecting newly allo-HSCT recipients
against SARS-CoV-2 infection remains unconfirmed. Presently, the most
effective strategy appears to be minimizing exposure to SARS-CoV-2.
</p>
</sec><sec>
<title>
Clinical trial registration
</title>
<p>
<ext-link ext-link-type="uri" xlink:href="https://clinicaltrials.gov/ct2/show/NCT04645199" xmlns:xlink="http://www.w3.org/1999/xlink">https://clinicaltrials.gov/ct2/show/NCT04645199</ext-link>,
identifier NCT04645199.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324019">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324019</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324019">Save
to Pocket</a> </span></p>
<hr />
<h2
id="genome-and-evolution-of-tibet-orbivirus-tibov-genus-orbivirus-family-reoviridae">Genome
and evolution of Tibet orbivirus, TIBOV (genus Orbivirus, family
Reoviridae)</h2>
<p>date: 2024-03-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Tibet orbivirus (TIBOV) was first isolated from <italic>Anopheles
maculatus</italic> mosquitoes in Xizang, China, in 2009. In recent
years, more TIBOV strains have been isolated in several provinces across
China, Japan, East Asia, and Nepal, South Asia. Furthermore, TIBOVs have
also been isolated from <italic>Culex</italic> mosquitoes, and several
midge species. Additionally, TIBOV neutralizing antibodies have been
detected in serum specimens from several mammals, including cattle,
sheep, and pigs. All of the evidence suggests that the geographical
distribution of TIBOVs has significantly expanded in recent years, with
an increased number of vector species involved in its transmission.
Moreover, the virus demonstrated infectivity towards a variety of
animals. Although TIBOV is considered an emerging orbivirus, detailed
reports on its genome and molecular evolution are currently lacking.
Thus, this study performed the whole-genome nucleotide sequencing of
three TIBOV isolates from mosquitoes and midges collected in China in
2009, 2011, and 2019. Furthermore, the genome and molecular genetic
evolution of TIBOVs isolated from different countries, periods, and
hosts (mosquitoes, midges, and cattle) was systematically analyzed. The
results revealed no molecular specificity among TIBOVs isolated from
different countries, periods, and vectors. Meanwhile, the time-scaled
phylogenetic analysis demonstrated that the most recent common ancestor
(TMRCA) of TIBOV appeared approximately 797 years ago (95% HPD: 16-2347)
and subsequently differentiated at least three times, resulting in three
distinct genotypes. The evolutionary rate of TIBOVs was about 2.12 ×
10<sup>-3</sup> nucleotide substitutions per site per year (s/s/y) (95%
HPD: 3.07 × 10<sup>-5</sup>, 9.63 × 10<sup>-3</sup>), which is similar
to that of the bluetongue virus (BTV), also in the
<italic>Orbivirus</italic> genus. Structural analyses of the viral
proteins revealed that the three-dimensional structures of the outer
capsid proteins of TIBOV and BTV were similar. These results suggest
that TIBOV is a newly discovered and rapidly evolving virus transmitted
by various blood-sucking insects. Given the potential public health
burden of this virus and its high infectious rate in a wide range of
animals, it is significant to strengthen research on the genetic
variation of TIBOVs in blood-feeding insects and mammals in the natural
environment and the infection status in animals.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1327780">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1327780</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1327780">Save
to Pocket</a> </span></p>
<hr />
<h2
id="dynamic-modulation-of-the-non-canonical-nf-κb-signaling-pathway-for-hiv-shock-and-kill">Dynamic
modulation of the non-canonical NF-κB signaling pathway for HIV shock
and kill</h2>
<p>date: 2024-03-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
HIV cure still remains an elusive target. The “Shock and Kill” strategy
which aims to reactivate HIV from latently infected cells and
subsequently kill them through virally induced apoptosis or immune
mediated clearance, is the subject of widespread investigation. NF-κB is
a ubiquitous transcription factor which serves as a point of confluence
for a number of intracellular signaling pathways and is also a crucial
regulator of HIV transcription. Due to its relatively lower side effect
profile and proven role in HIV transcription, the non-canonical NF-κB
pathway has emerged as an attractive target for HIV reactivation, as a
first step towards eradication. A comprehensive review examining this
pathway in the setting of HIV and its potential utility to cure efforts
is currently lacking. This review aims to summarize non-canonical NF-κB
signaling and the importance of this pathway in HIV shock-and-kill
efforts.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354502">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354502</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354502">Save
to Pocket</a> </span></p>
<hr />
<h2
id="the-impact-of-gut-microbiota-on-autoimmune-thyroiditis-and-relationship-with-pregnancy-outcomes-a-review">The
impact of gut microbiota on autoimmune thyroiditis and relationship with
pregnancy outcomes: a review</h2>
<p>date: 2024-03-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Autoimmune thyroiditis (AITD) is a T-cell-mediated, organ- specific
autoimmune disease caused by interactions between genetic and
environmental factors. Patients with AITD show thyroid lymphocyte
infiltration and an increase in the titer of thyroid autoimmune
antibodies, thereby altering the integrity of thyroid follicle
epithelial cells and dysregulating their metabolism and immune function,
leading to a decrease in multi-tissue metabolic activity. Research has
shown that patients with AITD have a significantly higher risk of
adverse pregnancy outcomes, such as infertility and miscarriage.
Levothyroxine(LT<sub>4</sub>) treatment can improve the pregnancy
outcomes of normal pregnant women with thyroid peroxidase
antibodies(TPOAb) positivity, but it is not effective for invitro
fertilization embryo transfer (IVF-ET) in women with normal thyroid
function and positive TPOAb. Other factors may also influence pregnancy
outcomes of patients with AITD. Recent studies have revealed that the
gut microbiota participates in the occurrence and development of AITD by
influencing the gut-thyroid axis. The bacterial abundance and diversity
of patients with Hashimoto thyroiditis (HT) were significantly reduced,
and the relative abundances of <italic>Bacteroides</italic>,
<italic>fecal Bacillus</italic>, <italic>Prevotella</italic>, and
<italic>Lactobacillus</italic> also decreased. The confirmation of
whether adjusting the composition of the gut microbiota can improve
pregnancy outcomes in patients with AITD is still pending. This article
reviews the characteristics of the gut microbiota in patients with AITD
and the current research on its impact in pregnancy.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361660">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361660</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1361660">Save
to Pocket</a> </span></p>
<hr />
<h2
id="subversion-of-a-family-of-antimicrobial-proteins-by-salmonella-enterica">Subversion
of a family of antimicrobial proteins by Salmonella enterica</h2>
<p>date: 2024-03-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Salmonella enterica</italic> is a food-borne pathogen able to
cause a wide spectrum of diseases ranging from mild gastroenteritis to
systemic infections. During almost all stages of the infection process
<italic>Salmonella</italic> is likely to be exposed to a wide variety of
host-derived antimicrobial peptides (AMPs). AMPs are important
components of the innate immune response which integrate within the
bacterial membrane, thus forming pores which lead ultimately to
bacterial killing. In contrast to other AMPs
Bactericidal/Permeability-increasing Protein (BPI) displayed only weak
bacteriostatic or bactericidal effects towards <italic>Salmonella
enterica</italic> sv. Typhimurium (STM) cultures. Surprisingly, we found
that sub-antimicrobial concentrations of BPI fold-containing (BPIF)
superfamily members mediated adhesion of STM depending on pre-formed
type 1 fimbriae. BPIF proteins directly bind to type 1 fimbriae through
mannose-containing oligosaccharide modifications. Fimbriae decorated
with BPIF proteins exhibit extended binding specificity, allowing for
bacterial adhesion on a greater variety of abiotic and biotic surfaces
likely promoting host colonization. Further, fimbriae significantly
contributed to the resistance against BPI, probably through
sequestration of the AMP before membrane interaction. In conclusion,
functional subversion of innate immune proteins of the BPIF family
through binding to fimbriae promotes <italic>Salmonella</italic>
virulence by survival of host defense and promotion of host
colonization.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1375887">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1375887</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1375887">Save
to Pocket</a> </span></p>
<hr />
<h2
id="diagnosis-and-management-of-invasive-fungal-diseases-in-non-neutropenic-icu-patients-with-focus-on-candidiasis-and-aspergillosis-a-comprehensive-review">Diagnosis
and management of invasive fungal diseases in non-neutropenic ICU
patients, with focus on candidiasis and aspergillosis: a comprehensive
review</h2>
<p>date: 2024-03-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Invasive fungal diseases pose a significant threat to non-neutropenic
ICU patients, with <italic>Candida</italic> and
<italic>Aspergillus</italic> infections being the most common. However,
diagnosing these infections in the ICU population remains challenging
due to overlapping clinical features, poor sensitivity of blood
cultures, and invasive sampling requirements. The classical host
criteria for defining invasive fungal disease do not fully apply to ICU
patients, leading to missed or delayed diagnoses. Recent advancements
have improved our understanding of invasive fungal diseases, leading to
revised definitions and diagnostic criteria. However, the diagnostic
difficulties in ICU patients remain unresolved, highlighting the need
for further research and evidence generation. Invasive candidiasis is
the most prevalent form of invasive fungal disease in non-neutropenic
ICU patients, presenting as candidemia and deep-seated candidiasis.
Diagnosis relies on positive blood cultures or histopathology, while
non-culture-based techniques such as beta-D-glucan assay and PCR-based
tests show promise. Invasive aspergillosis predominantly manifests as
invasive pulmonary aspergillosis in ICU patients, often associated with
comorbidities and respiratory deterioration in viral pneumonia.
Diagnosis remains challenging due to poor sensitivity of blood cultures
and difficulties in performing lung biopsies. Various diagnostic
criteria have been proposed, including mycological evidence,
clinical/radiological factors and expanded list of host factors.
Non-culture-based techniques such as galactomannan assay and PCR-based
tests can aid in diagnosis. Antifungal management involves tailored
therapy based on guidelines and individual patient factors. The
complexity of diagnosing and managing invasive fungal diseases in ICU
patients underscore the importance of ongoing research and the need for
updated diagnostic criteria and treatment approaches. Invasive fungal
disease, Invasive fungal infection, Invasive candidiasis, Invasive
aspergillosis, Antifungal drugs.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1256158">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1256158</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1256158">Save
to Pocket</a> </span></p>
<hr />
<h2
id="integrins-regulation-of-wound-healing-processes-insights-for-chronic-skin-wound-therapeutics">Integrins
regulation of wound healing processes: insights for chronic skin wound
therapeutics</h2>
<p>date: 2024-03-05, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Integrins are heterodimers composed of non-covalently associated alpha
and beta subunits that mediate the dynamic linkage between extracellular
adhesion molecules and the intracellular actin cytoskeleton. Integrins
are present in various tissues and organs and are involved in different
physiological and pathological molecular responses <italic>in
vivo</italic>. Wound healing is an important process in the recovery
from traumatic diseases and consists of three overlapping phases:
inflammation, proliferation, and remodeling. Integrin regulation acts
throughout the wound healing process to promote wound healing. Prolonged
inflammation may lead to failure of wound healing, such as wound
chronicity. One of the main causes of chronic wound formation is
bacterial colonization of the wound. In this review, we review the role
of integrins in the regulation of wound healing processes such as
angiogenesis and re-epithelialization, as well as the role of integrins
in mediating bacterial infections during wound chronicity, and the
challenges and prospects of integrins as therapeutic targets for
infected wound healing.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324441">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324441</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324441">Save
to Pocket</a> </span></p>
<hr />
<h2
id="causal-relationship-between-the-gut-microbiota-and-insomnia-a-two-sample-mendelian-randomization-study">Causal
relationship between the gut microbiota and insomnia: a two-sample
Mendelian randomization study</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Changes in the gut microbiota are closely related to insomnia, but the
causal relationship between them is not yet clear.
</p>
</sec><sec>
<title>
Objective
</title>
<p>
To clarify the relationship between the gut microbiota and insomnia and
provide genetic evidence for them, we conducted a two-sample Mendelian
randomization study.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We used a Mendelian randomized two-way validation method to discuss the
causal relationship. First, we downloaded the data of 462,341
participants relating to insomnia, and the data of 18,340 participants
relating to the gut microbiota from a genome-wide association study
(GWAS). Then, we used two regression models, inverse-variance weighted
(IVW) and MR-Egger regression, to evaluate the relationship between
exposure factors and outcomes. Finally, we took a reverse MR analysis to
assess the possibility of reverse causality.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The combined results show 19 gut microbiotas to have a causal
relationship with insomnia (odds ratio (OR): 1.03; 95% confidence
interval (CI): 1.01, 1.05; p=0.000 for class. Negativicutes; OR: 1.03;
95% CI: 1.01, 1.05; p=0.000 for order.Selenomonadales; OR: 1.01; 95% CI:
1.00, 1.02; p=0.003 for genus.RikenellaceaeRC9gutgroup). The results
were consistent with sensitivity analyses for these bacterial traits. In
reverse MR analysis, we found no statistical difference between insomnia
and these gut microbiotas.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
This study can provide a new direction for the causal relationship
between the gut microbiota (class.Negativicutes, order.Selenomonadales,
genus.Lactococcus) and insomnia and the treatment or prevention
strategies of insomnia.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1279218">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1279218</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1279218">Save
to Pocket</a> </span></p>
<hr />
<h2
id="influenza-and-covid-19-co-infection-and-vaccine-effectiveness-against-severe-cases-a-mathematical-modeling-study">Influenza
and COVID-19 co-infection and vaccine effectiveness against severe
cases: a mathematical modeling study</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Influenza A virus have a distinctive ability to exacerbate SARS-CoV-2
infection proven by in vitro studies. Furthermore, clinical evidence
suggests that co-infection with COVID-19 and influenza not only
increases mortality but also prolongs the hospitalization of patients.
COVID-19 is in a small-scale recurrent epidemic, increasing the
likelihood of co-epidemic with seasonal influenza. The impact of
co-infection with influenza virus and SARS-CoV-2 on the population
remains unstudied.
</p>
</sec><sec>
<title>
Method
</title>
<p>
Here, we developed an age-specific compartmental model to simulate the
co-circulation of COVID-19 and influenza and estimate the number of
co-infected patients under different scenarios of prevalent virus type
and vaccine coverage. To decrease the risk of the population developing
severity, we investigated the minimum coverage required for the COVID-19
vaccine in conjunction with the influenza vaccine, particularly during
co-epidemic seasons.
</p>
</sec><sec>
<title>
Result
</title>
<p>
Compared to the single epidemic, the transmission of the SARS-CoV-2
exhibits a lower trend and a delayed peak when co-epidemic with
influenza. Number of co-infection cases is higher when SARS-CoV-2
co-epidemic with Influenza A virus than that with Influenza B virus. The
number of co-infected cases increases as SARS-CoV-2 becomes more
transmissible. As the proportion of individuals vaccinated with the
COVID-19 vaccine and influenza vaccines increases, the peak number of
co-infected severe illnesses and the number of severe illness cases
decreases and the peak time is delayed, especially for those &gt;60
years old.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
To minimize the number of severe illnesses arising from co-infection of
influenza and COVID-19, in conjunction vaccinations in the population
are important, especially priority for the elderly.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347710">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347710</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347710">Save
to Pocket</a> </span></p>
<hr />
<h2
id="detection-of-infectious-sars-cov-2-in-ocular-samples-is-linked-to-viral-load-in-the-nasopharynx">Detection
of infectious SARS-CoV-2 in ocular samples is linked to viral load in
the nasopharynx</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
SARS-CoV-2 is known to infect respiratory tissue cells. However, less is
known about infection of ocular tissue and potential infectivity of
lacrimal fluid. With this study, we want to compare viral loads in eye
and nasopharyngeal swabs and analyze these for infectious virus.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Between May 2020 and April 2021 ocular and nasopharyngeal swabs were
collected from 28 SARS-CoV-2 infected patients treated on the corona
virus disease 2019 (COVID-19)-ward of the University Hospital of
Innsbruck, Austria. Samples with PCR detectable SARS-CoV-2 were analyzed
via whole genome sequencing and an attempt was made to isolate
infectious virus.
</p>
</sec><sec>
<title>
Results
</title>
<p>
At the time point of sample collection, 22 individuals were still PCR
positive in nasopharyngeal samples and in 6 of these patients one or
both ocular samples were additionally positive. CT-values in eyes were
generally higher compared to corresponding nasopharyngeal samples and we
observed a tendency for lower CT-values, i.e. increased viral load, in
nasopharyngeal swabs of individuals with at least one infected eye,
compared to those where ocular samples were PCR negative. Ocular and
nasopharyngeal sequences from the same patient were assigned to the same
variant, either the D614G or the Alpha variant. Infectious virus was
successfully isolated from 9 nasopharyngeal swabs, however only from one
of the seven PCR positive ocular samples.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
We could detect SARS-CoV-2 in eyes of some of the infected patients
albeit at lower levels compared to nasopharyngeal swabs. However, our
results also indicate that lacrimal fluid might be infectious in
patients with high viral load.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1332157">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1332157</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1332157">Save
to Pocket</a> </span></p>
<hr />
<h2
id="development-and-validation-a-nomogram-prediction-model-for-early-diagnosis-of-bloodstream-infections-in-the-intensive-care-unit">Development
and validation a nomogram prediction model for early diagnosis of
bloodstream infections in the intensive care unit</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Purpose
</title>
<p>
This study aims to develop and validate a nomogram for predicting the
risk of bloodstream infections (BSI) in critically ill patients based on
their admission status to the Intensive Care Unit (ICU).
</p>
</sec><sec>
<title>
Patients and methods
</title>
<p>
Patients’ data were extracted from the Medical Information Mart for
Intensive Care−IV (MIMIC−IV) database (training set), the Beijing
Friendship Hospital (BFH) database (validation set) and the eICU
Collaborative Research Database (eICU−CRD) (validation set). Univariate
logistic regression analyses were used to analyze the influencing
factors, and lasso regression was used to select the predictive factors.
Model performance was assessed using area under receiver operating
characteristic curve (AUROC) and Presented as a Nomogram. Various
aspects of the established predictive nomogram were evaluated, including
discrimination, calibration, and clinical utility.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The model dataset consisted of 14930 patients (1444 BSI patients) from
the MIMIC-IV database, divided into the training and internal validation
datasets in a 7:3 ratio. The eICU dataset included 2100 patients (100
with BSI) as the eICU validation dataset, and the BFH dataset included
419 patients (21 with BSI) as the BFH validation dataset. The nomogram
was constructed based on Glasgow Coma Scale (GCS), sepsis related organ
failure assessment (SOFA) score, temperature, heart rate, respiratory
rate, white blood cell (WBC), red width of distribution (RDW), renal
replacement therapy and presence of liver disease on their admission
status to the ICU. The AUROCs were 0.83 (CI 95%:0.81-0.84) in the
training dataset, 0.88 (CI 95%:0.88-0.96) in the BFH validation dataset,
and 0.75 (95%CI 0.70-0.79) in the eICU validation dataset. The clinical
effect curve and decision curve showed that most areas of the decision
curve of this model were greater than 0, indicating that this model has
a certain clinical effectiveness.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The nomogram developed in this study provides a valuable tool for
clinicians and nurses to assess individual risk, enabling them to
identify patients at a high risk of bloodstream infections in the ICU.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348896">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348896</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1348896">Save
to Pocket</a> </span></p>
<hr />
<h2
id="phage-p2-71-against-multi-drug-resistant-proteus-mirabilis-isolation-characterization-and-non-antibiotic-antimicrobial-potential">Phage
P2-71 against multi-drug resistant Proteus mirabilis: isolation,
characterization, and non-antibiotic antimicrobial potential</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Proteus mirabilis</italic>, a prevalent urinary tract pathogen
and formidable biofilm producer, especially in Catheter-Associated
Urinary Tract Infection, has seen a worrying rise in multidrug-resistant
(MDR) strains. This upsurge calls for innovative approaches in infection
control, beyond traditional antibiotics. Our research introduces
bacteriophage (phage) therapy as a novel non-antibiotic strategy to
combat these drug-resistant infections. We isolated P2-71, a lytic phage
derived from canine feces, demonstrating potent activity against MDR
<italic>P. mirabilis</italic> strains. P2-71 showcases a notably brief
10-minute latent period and a significant burst size of 228 particles
per infected bacterium, ensuring rapid bacterial clearance. The phage
maintains stability over a broad temperature range of 30-50°C and within
a pH spectrum of 4-11, highlighting its resilience in various
environmental conditions. Our host range assessment solidifies its
potential against diverse MDR <italic>P. mirabilis</italic> strains.
Through killing curve analysis, P2-71’s effectiveness was validated at
various MOI levels against <italic>P. mirabilis</italic> 37,
highlighting its versatility. We extended our research to examine
P2-71’s stability and bactericidal kinetics in artificial urine,
affirming its potential for clinical application. A detailed genomic
analysis reveals P2-71’s complex genetic makeup, including genes
essential for morphogenesis, lysis, and DNA modification, which are
crucial for its therapeutic action. This study not only furthers the
understanding of phage therapy as a promising non-antibiotic
antimicrobial but also underscores its critical role in combating
emerging MDR infections in both veterinary and public health contexts.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347173">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347173</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347173">Save
to Pocket</a> </span></p>
<hr />
<h2
id="airway-necrosis-and-granulation-tissue-formation-caused-by-rhizopus-oryzae-leading-to-severe-upper-airway-obstruction-a-case-report">Airway
necrosis and granulation tissue formation caused by Rhizopus oryzae
leading to severe upper airway obstruction: a case report</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Pulmonary Mucormycosis is a fatal infectious disease with high mortality
rate. The occurrence of Mucormycosis is commonly related to the fungal
virulence and the host’s immunological defenses against pathogens.
Mucormycosis infection and granulation tissue formation occurred in the
upper airway was rarely reported. This patient was a 60-year-old male
with diabetes mellitus, who was admitted to hospital due to progressive
cough, sputum and dyspnea. High-resolution computed tomography (HRCT)
and bronchoscopy revealed extensive tracheal mucosal necrosis,
granulation tissue proliferation, and severe airway stenosis. The
mucosal necrotic tissue was induced by the infection of <italic>Rhizopus
Oryzae</italic>, confirmed by metagenomic next-generation sequencing
(mNGS) in tissue biopsy. This patient was treated with the placement of
a covered stent and local instillation of amphotericin B via
bronchoscope. The tracheal mucosal necrosis was markedly alleviated, the
symptoms of cough, shortness of breath, as well as exercise tolerance
were significantly improved. The placement of airway stent and
transbronchial microtube drip of amphotericin B could conduce to rapidly
relieve the severe airway obstruction due to Mucormycosis infection.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366472">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366472</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1366472">Save
to Pocket</a> </span></p>
<hr />
<h2
id="optimal-selection-of-specimens-for-metagenomic-next-generation-sequencing-in-diagnosing-periprosthetic-joint-infections">Optimal
selection of specimens for metagenomic next-generation sequencing in
diagnosing periprosthetic joint infections</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Objective
</title>
<p>
This study aimed to assess the diagnostic value of metagenomic
next-generation sequencing (mNGS) across synovial fluid, prosthetic
sonicate fluid, and periprosthetic tissues among patients with
periprosthetic joint infection (PJI), intending to optimize specimen
selection for mNGS in these patients.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This prospective study involved 61 patients undergoing revision
arthroplasty between September 2021 and September 2022 at the First
Affiliated Hospital of Zhengzhou University. Among them, 43 cases were
diagnosed as PJI, and 18 as aseptic loosening (AL) based on the American
Musculoskeletal Infection Society (MSIS) criteria. Preoperative or
intraoperative synovial fluid, periprosthetic tissues, and prosthetic
sonicate fluid were collected, each divided into two portions for mNGS
and culture. Comparative analyses were conducted between the
microbiological results and diagnostic efficacy derived from mNGS and
culture tests. Furthermore, the variability in mNGS diagnostic efficacy
for PJI across different specimen types was assessed.
</p>
</sec><sec>
<title>
Results
</title>
<p>
The sensitivity and specificity of mNGS diagnosis was 93% and 94.4% for
all types of PJI specimens; the sensitivity and specificity of culture
diagnosis was 72.1% and 100%, respectively. The diagnostic sensitivity
of mNGS was significantly higher than that of culture (X<sup>2</sup> =
6.541, <italic>P</italic>=0.011), with no statistically significant
difference in specificity (X<sup>2</sup> = 1.029,
<italic>P</italic>=0.310). The sensitivity of the synovial fluid was
83.7% and the specificity was 94.4%; the sensitivity of the prosthetic
sonicate fluid was 90.7% and the specificity was 94.4%; and the
sensitivity of the periprosthetic tissue was 81.4% and the specificity
was 100%. Notably, the mNGS of prosthetic sonicate fluid displayed a
superior pathogen detection rate compared to other specimen types.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
mNGS can function as a precise diagnostic tool for identifying pathogens
in PJI patients using three types of specimens. Due to its superior
ability in pathogen identification, prosthetic sonicate fluid can
replace synovial fluid and periprosthetic tissue as the optimal sample
choice for mNGS.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356804">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356804</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356804">Save
to Pocket</a> </span></p>
<hr />
<h2
id="identifying-potential-monkeypox-virus-inhibitors-an-in-silico-study-targeting-the-a42r-protein">Identifying
potential monkeypox virus inhibitors: an in silico study targeting the
A42R protein</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Monkeypox (now Mpox), a zoonotic disease caused by the monkeypox virus
(MPXV) is an emerging threat to global health. In the time span of only
six months, from May to October 2022, the number of MPXV cases breached
80,000 and many of the outbreaks occurred in locations that had never
previously reported MPXV. Currently there are no FDA-approved
MPXV-specific vaccines or treatments, therefore, finding drugs to combat
MPXV is of utmost importance. The A42R profilin-like protein of the MPXV
is involved in cell development and motility making it a critical drug
target. A42R protein is highly conserved across orthopoxviruses, thus
A42R inhibitors may work for other family members. This study sought to
identify potential A42R inhibitors for MPXV treatment using
computational approaches. The energy minimized 3D structure of the A42R
profilin-like protein (PDB ID: 4QWO) underwent virtual screening using a
library of 36,366 compounds from Traditional Chinese Medicine (TCM),
AfroDb, and PubChem databases as well as known inhibitor tecovirimat via
AutoDock Vina. A total of seven compounds comprising PubChem CID:
11371962, ZINC000000899909, ZINC000001632866, ZINC000015151344,
ZINC000013378519, ZINC000000086470, and ZINC000095486204, predicted to
have favorable binding were shortlisted. Molecular docking suggested
that all seven proposed compounds have higher binding affinities to A42R
(–7.2 to –8.3 kcal/mol) than tecovirimat (–6.7 kcal/mol). This was
corroborated by MM/PBSA calculations, with tecovirimat demonstrating the
highest binding free energy of –68.694 kJ/mol (lowest binding affinity)
compared to the seven shortlisted compounds that ranged from –73.252 to
–97.140 kJ/mol. Furthermore, the 7 compounds in complex with A42R
demonstrated higher stability than the A42R-tecovirimat complex when
subjected to 100 ns molecular dynamics simulations. The protein-ligand
interaction maps generated using LigPlot+ suggested that residues Met1,
Glu3, Trp4, Ile7, Arg127, Val128, Thr131, and Asn133 are important for
binding. These seven compounds were adequately profiled to be potential
antivirals via PASS predictions and structural similarity searches. All
seven potential lead compounds were scored Pa &gt; Pi for antiviral
activity while ZINC000001632866 and ZINC000015151344 were predicted as
poxvirus inhibitors with Pa values of 0.315 and 0.215, and Pi values of
0.052 and 0.136, respectively. Further experimental validations of the
identified lead compounds are required to corroborate their predicted
activity. These seven identified compounds represent solid footing for
development of antivirals against MPXV and other orthopoxviruses.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351737">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351737</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1351737">Save
to Pocket</a> </span></p>
<hr />
<h2
id="functional-characterization-of-francisella-tularensis-subspecies-holarctica-genotypes-during-tick-cell-and-macrophage-infections-using-a-proteogenomic-approach">Functional
characterization of Francisella tularensis subspecies holarctica
genotypes during tick cell and macrophage infections using a
proteogenomic approach</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Tularemia is a vector-borne disease caused by the Gram-negative
bacterium <italic>Francisella tularensis</italic>. Known hosts and
vectors in Europe are hare and ticks. <italic>F. tularensis</italic> is
transmitted from ticks and animals, but also from the hydrotelluric
environment and the consumption of contaminated water or food. A
changing climate expands the range in which ticks can live and
consequently might contribute to increasing case numbers of tularemia.
Two subspecies of <italic>F. tularensis</italic> are human pathogenic.
<italic>Francisella tularensis tularensis (Ftt)</italic> is endemic in
North America, while <italic>Francisella tularensis holarctica
(Fth)</italic> is the only subspecies causing tularemia in Europe.
<italic>Ft</italic> is classified as a category A bioterrorism agent due
to its low infectious dose, multiple modes of transmission, high
infectivity and potential for airborne transmission and has become a
global public health concern. In line with the European survey and
previous phylogenetic studies, Switzerland shows the co-distribution of
B.6 and B.12 strains with different geographical distribution and
prevalence within the country. To establish itself in different host
environments of ticks and mammals, <italic>F. tularensis</italic>
presumably undergoes substantial changes on the transcriptomics and
proteomic level. Here we investigate the transcriptomic and proteomic
differences of five strains of <italic>Fth</italic> upon infection of
rabbit macrophages and tick cells.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1355113">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1355113</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1355113">Save
to Pocket</a> </span></p>
<hr />
<h2
id="bumble-bee-microbiota-shows-temporal-succession-and-increase-of-lactic-acid-bacteria-when-exposed-to-outdoor-environments">Bumble
bee microbiota shows temporal succession and increase of lactic acid
bacteria when exposed to outdoor environments</h2>
<p>date: 2024-03-04, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Question
</title>
<p>
The large earth bumble bee (<italic>Bombus terrestris</italic>)
maintains a social core gut-microbiota, similar as known from the honey
bee, which plays an important role for host health and resistance.
Experiments under laboratory conditions with commercial hives are
limited to vertically transmitted microbes and neglect influences of
environmental factors or external acquisition of microbes. Various
environmental and landscape-level factors may have an impact on the
gut-microbiota of pollinating insects, with consequences for pollinator
health and fitness in agroecosystems. Still, it is not fully clear
whether access to different flower diversities will have a significant
influence on the bumble bee microbiota. Here, we tested in a semi-field
experiment if the bumble bee microbiota changes over time when exposed
to different flower diversities within outdoor flight cages. We used
commercial hives to distinguish between vertically and horizontally
transmitted bacteria, respectively from the nest environment or the
exposed outside environment.
</p>
</sec><sec>
<title>
Result
</title>
<p>
The sequential sampling of foraging workers over a period of 35 days
indicated a temporal progression of the bumble bee microbiota when
placed outside. The microbiota increased in diversity and changed in
composition and variability over time. We observed a major increase in
relative abundance of the families <italic>Lactobacillaceae</italic>,
<italic>Bifidobacteriaceae</italic> and <italic>Weeksellaceae</italic>.
In contrast, major core-taxa like <italic>Snodgrassella</italic> and
<italic>Gilliamella</italic> declined in their relative abundance over
time. The genus <italic>Lactobacillus</italic> showed a high diversity
and strain specific turnover, so that only specific ASVs showed an
increase over time, while others had a more erratic occurrence pattern.
Exposure to different flower diversities had no significant influence on
the progression of the bumble bee microbiota.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
The bumble bee microbiota showed a dynamic temporal succession with
distinct compositional changes and diversification over time when placed
outdoor. The exposure of bumble bees to environmental conditions, or
environmental microbes, increases dissimilarity and changes the
gut-community composition. This shows the importance of environmental
influences on the temporal dynamic and progression of the bumble bee
microbiota.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342781">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342781</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1342781">Save
to Pocket</a> </span></p>
<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
