<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
		src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
		alt="line art showing an antenna"
		height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="./">The Antenna</a></li>
	<li><a href="snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="archives/">Archives</a></li>
	<li><a href="search.html">Search</a></li>
	<li><a href="about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
Items collected from feeds in <a href="health.txt">health.txt</a>
</div>
<h1 id="health">health</h1>
<p>(date: 2024-10-07 07:42:39)</p>
<hr />
<h2
id="bacteriophage-mediated-approaches-for-biofilm-control">Bacteriophage-mediated
approaches for biofilm control</h2>
<p>date: 2024-10-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Biofilms are complex microbial communities in which planktonic and
dormant bacteria are enveloped in extracellular polymeric substances
(EPS) such as exopolysaccharides, proteins, lipids, and DNA. These
multicellular structures present resistance to conventional
antimicrobial treatments, including antibiotics. The formation of
biofilms raises considerable concern in healthcare settings, biofilms
can exacerbate infections in patients and compromise the integrity of
medical devices employed during treatment. Similarly, certain bacterial
species contribute to bulking, foaming, and biofilm development in water
environments such as wastewater treatment plants, water reservoirs, and
aquaculture facilities. Additionally, food production facilities provide
ideal conditions for establishing bacterial biofilms, which can serve as
reservoirs for foodborne pathogens. Efforts to combat antibiotic
resistance involve exploring various strategies, including bacteriophage
therapy. Research has been conducted on the effects of phages and their
individual proteins to assess their potential for biofilm removal.
However, challenges persist, prompting the examination of refined
approaches such as drug-phage combination therapies, phage cocktails,
and genetically modified phages for clinical applications. This review
aims to highlight the progress regarding bacteriophage-based approaches
for biofilm eradication in different settings.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1428637"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1428637</a></p>
<hr />
<h2
id="synergistic-effects-of-antimicrobial-components-of-the-human-derived-composite-amnion-chorion-membrane-on-bacterial-growth">Synergistic
effects of antimicrobial components of the human-derived composite
amnion-chorion membrane on bacterial growth</h2>
<p>date: 2024-10-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe human-derived amnion-chorion membrane (ACM) has
endogenous antimicrobial properties, which are important for preventing
the colonization and survival of oral bacteria on exposed membranes.
This project aimed to decipher the underlying mechanism by identifying
the components of ACM that confer antibacterial properties. In addition,
the antimicrobial efficacy of these identified components on oral
bacteria was assessed.MethodsFour antimicrobial proteins, histone
H2A/H2B, cathelicidin LL-37, lactoferrin, and lysozyme, were identified
via mass spectrometry in ACM. These proteins were then assessed for
their efficacy in killing Streptococcus gordonii Challis. Log-phased
bacterial cells were cultured with the commercially available proteins
that were identified in ACM, either individually or in combination, at
different concentrations. After incubation for 8 or 24 hours, the
bacteria were stained with a live/dead viability kit and analyzed via
confocal microscopy.ResultsThe combination of these proteins effectively
killed S. gordonii in a dose-dependent fashion after 8 or 24 hours of
incubation. When each protein was tested individually, it killed S.
gordonii at a much lower efficacy relative to the combinations. The
synergistic effects of the antimicrobial protein combinations were also
observed in both the viable cell count recovery and minimum inhibitory
concentration assays.DiscussionBy shedding light on the mechanisms in
the ACM’s antimicrobial property, this study may raise more awareness of
the potential benefit of utilization of a membrane with endogenous
antimicrobial properties in regeneration surgeries.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1472737"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1472737</a></p>
<hr />
<h2
id="the-rna-m5c-methyltransferase-nsun1-modulates-human-malaria-gene-expression-during-intraerythrocytic-development">The
RNA m5C methyltransferase NSUN1 modulates human malaria gene expression
during intraerythrocytic development</h2>
<p>date: 2024-10-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionPlasmodium falciparum is the most damaging malaria
pathogen and brings a heavy burden to global health. Host switching and
morphological changes in P. falciparum are dependent on an effective
gene expression regulatory system. C5 methylation of cytosines is a
common RNA modification in eukaryotes, and the NSUN family are essential
m5C modification executors. Currently, little is known about this family
in Plasmodium spp. In this study, we focus on exploring the function of
PfNSUN1 protein.MethodsAn efficient CRISPR/Cas9 gene editing technique
was applied to construct the PfNSUN1 knockdown strain. The knockdown
efficiency was confirmed by growth curves and western blot experiments.
The knockdown transcriptome data was acquired to find differentially
expressed genes, and target genes of PfNSUN1 protein were identified by
RNA immunoprecipitation and high-throughput sequencing
experiments.ResultsThe efficiency of PfNSUN1 protein down-regulated was
about 34%. RNA-seq data revealed that differentially expressed genes
were mainly down-regulated. And there were 224, 278, 556 genes that were
down-regulated with more than 2-fold changes and p-adj&lt;0.05 at ring,
trophozoite and schizont stages, respectively. PfNSUN1 protein was
significantly enriched on 154 target genes, including 28S ribosomal RNA
and pfap2-g5 transcription factor.DiscussionPfNSUN1 is a crucial RNA
post-transcriptional modification protein in P. falciparum. It plays a
pivotal role in regulating gene expression and parasite growth by
targeting 28S ribosomal RNA and pfap2-g5 transcription factor.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474229"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474229</a></p>
<hr />
<h2
id="antifungal-susceptibility-molecular-epidemiology-and-clinical-risk-factors-of-candida-glabrata-in-intensive-care-unit-in-a-chinese-tertiary-hospital">Antifungal
susceptibility, molecular epidemiology, and clinical risk factors of
Candida glabrata in intensive care unit in a Chinese Tertiary
Hospital</h2>
<p>date: 2024-10-07, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe increasing incidence and high mortality rate of Candida
glabrata infection in ICU patients is an important issue. Therefore, it
is imperative to investigate the antifungal susceptibility profiles and
epidemiological characteristics in local regions.MethodsHerein,
antifungal susceptibility testing was conducted to determine the minimum
inhibitory concentrations (MICs) of eight antifungal drugs. Multilocus
sequence typing (MLST) was used to study the strain genotype,
geographical distribution, and susceptibility to antifungal agents among
C. glabrata isolates. The mechanism of echinocandin resistance was
explored by sequencing the FKS1 and FKS2 genes (encoding 1,3-β-D-glucan
synthases) of echinocandin-resistant C. glabrata strains. Moreover, we
further investigated the clinical manifestations and the various risk
factors of patients infected with C. glabrata in the ICU.ResultsWe
selected 234 C. glabrata isolates from 234 patients in the ICU randomly
for the follow-up study. Cross-resistance was found among the ICU C.
glabrata isolates. Analysis using MLST showed that the genetic diversity
among the C. glabrata isolates was low. Furthermore, sequence type
showed no correlation with the antifungal resistance profiles, but was
associated with geographical distribution. We also revealed novel
mutations in FKS1 (S629P) and FKS2 (W1497stop) that mediated high-level
echinocandin resistance (MIC &gt;8 µg/mL). More than 14 days’ stay in
ICU (P=0.007), Acute Physiology and Chronic Health Evaluation II
(APACHE-II) score (P=0.024), prior antifungal exposure (P=0.039) and
lung disease (P=0.036) were significantly associated with antifungal
resistant/non-wild-type C. glabrata infection.ConclusionOur study shed
light on the antifungal susceptibility, molecular epidemiology, and
clinical risk factors of C. glabrata in the ICU of a Chinese Tertiary
Hospital. Importantly, we revealed the molecular mechanism of
echinocandin resistance. These results highlight the significance of
continued surveillance in ICUs and provide data support for the
treatment of C. glabrata in clinics.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1455145"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1455145</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
