<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2025-04-20 06:08:16)</p>
<hr />
<h2
id="early-prediction-of-sepsis-associated-encephalopathy-in-elderly-icu-patients-using-machine-learning-models-a-retrospective-study-based-on-the-mimic-iv-database">Early
prediction of sepsis associated encephalopathy in elderly ICU patients
using machine learning models: a retrospective study based on the
MIMIC-IV database</h2>
<p>date: 2025-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSepsis associated encephalopathy (SAE) is prevalent among
elderly patients in the ICU and significantly affects patient prognosis.
Due to the symptom similarity with other neurological disorders and the
absence of specific biomarkers, early clinical diagnosis remains
challenging. This study aimed to develop a predictive model for SAE in
elderly ICU patients.MethodsThe data of elderly sepsis patients were
extracted from the MIMIC IV database (version 3.1) and divided into
training and test sets in a 7:3 ratio. Feature variables were selected
using the LASSO-Boruta combined algorithm, and five machine learning
(ML) models, including Extreme Gradient Boosting (XGBoost), Categorical
Boosting (CatBoost),Light Gradient Boosting Machine(LGBM), Multilayer
Perceptron (MLP), and Support Vector Machines (SVM), were subsequently
developed using these variables. A comprehensive set of performance
metrics was used to assess the predictive accuracy, calibration, and
clinical applicability of these models. For the machine learning model
with the best performance, we employed the SHapley Additive
Explanations(SHAP) method to visualize the model.ResultsBased on strict
inclusion and exclusion criteria, a total of 3,156 elderly sepsis
patients were enrolled in the study, with an SAE incidence rate of
48.7%. The mortality rate of elderly sepsis patients who developed SAE
was significantly higher than that of patients in the non-SAE group
(28.78% vs. 12.59%, P &lt; 0.001). A total of 18 feature variables were
selected for the construction of the ML model using the LASSO-Boruta
combined algorithm. Compared to the other four models and traditional
scoring systems, the XGBoost model demonstrated the best overall
predictive performance, with Area Under the Curve(AUC)=0.898,
accuracy=0.830, recall=0.819, F1-Score=0.820, specificity=0.840, and
Precision=0.821. Furthermore, the results from the Decision Curve
Analysis (DCA) and calibration curves demonstrated that the XGBoost
model has significant clinical value and stable predictive performance.
The ten-fold cross-validation method further confirmed the robustness
and generalizability of the model. In addition, we simplified the model
based on the SHAP feature importance ranking, and the results indicated
that the simplified XGBoost model retains excellent predictive ability
(AUC=0.858).ConclusionsThe XGBoost model effectively predicts SAE in
elderly ICU patients and may serve as a reliable tool for clinicians to
identify high-risk patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545979"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545979</a></p>
<hr />
<h2
id="predictive-value-of-the-preoperative-c-reactive-protein-to-albumin-ratio-for-surgical-site-infection-after-percutaneous-kyphoplasty-a-single-center-retrospective-study">Predictive
value of the preoperative C-reactive protein-to-albumin ratio for
surgical site infection after percutaneous kyphoplasty: a single-center
retrospective study</h2>
<p>date: 2025-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study aimed to investigate the risk factors for
surgical site infection (SSI) after percutaneous kyphoplasty (PKP) and
evaluate the application value of the preoperative C-reactive protein
(CRP)-to-albumin ratio (CAR) in predicting SSI.MethodsThis study
retrospectively enrolled 329 patients with thoracolumbar compression
fractures who underwent PKP in the Affiliated Hospital of Qingdao
University from January 2019 to June 2024. The demographic information,
surgery-related data and laboratory examination results of the patients
were collected. According to these results, the patients were divided
into SSI and non-SSI groups, and the results were compared and analyzed.
The receiver operating characteristic curve was used to determine the
optimal cutoff value of preoperative CAR for predicting SSI, and binary
logistic regression analysis was employed to evaluate the predictive
value of CAR for SSI. The risk factors of SSI in the thoracolumbar
subgroup were further explored.ResultsThe study enrolled a total of 329
patients, and SSI occurred in 29 (8.81%). The optimal cut-off value of
CAR was 0.1213, and the area under the curve was 0.808 (P &lt; 0. 001).
The results showed that SSI rates were related to the surgical site, and
the SSI rate in the lumbar spine was higher than that in the thoracic
spine. The SSI group had a longer surgical duration and more operated
segments. The levels of preoperative CRP, CAR, procalcitonin and
erythrocyte sedimentation rate (ESR) were higher; however, serum albumin
levels were lower. More patients had CAR ≥0.1213 (75.86% vs 25.33%) and
white blood cell (WBC) &gt;10*109 (27.59% vs 10.00%). In addition, no
significant differences were found by the other demographic data and
laboratory examinations between the two groups. In the binary logistic
regression analysis, preoperative CAR was an independent risk factor for
post-PKP SSI, and the SSI risk increased by 7.464 times in patients with
CAR ≥0.1213. The number of operated segments, surgical duration, and ESR
were also independent risk factors for SSI, whereas serum albumin is a
protective factor.ConclusionPreoperative CAR is an effective predictor
of post-PKP SSI, which can be used for clinical prevention and reduction
of SSI risk.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565468"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565468</a></p>
<hr />
<h2
id="comprehensive-analysis-of-transcriptome-and-microbiome-in-colorectal-cancer-with-synchronous-polyp-patients">Comprehensive
analysis of transcriptome and microbiome in colorectal cancer with
synchronous polyp patients</h2>
<p>date: 2025-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundColorectal cancer (CRC) is a prevalent and lethal
malignancy, with the role of gut microbiota in its development still
unclear. This study examines differences in gut microbiota between CRC
patients and healthy controls and explores their association with host
gene expression to identify potential diagnostic and therapeutic
targets.MethodsFecal samples from 10 CRC patients and 13 healthy
controls were subjected to 16S rRNA sequencing. Transcriptome sequencing
of tumor tissues, normal mucosa, and colorectal polyps from same 10 CRC
patients was performed to identify differentially expressed genes
(DEGs). Pearson correlation analysis was employed to associate
operational taxonomic units (OTUs) with host gene
expression.Resultsβ-diversity analysis showed significant differences in
microbiota between CRC patients and controls (P &lt; 0.01). LEfSe
identified 38 distinct bacterial taxa, with genera such as Bacteroides,
Peptostreptococcus, and Parabacteroides being enriched in CRC patients.
Transcriptome analysis uncovered 1,026 DEGs. Notably, TIMP1 and BCAT1
were positively correlated (r &gt; 0.76, P &lt; 0.01) with pathogenic
bacteria like Fusobacterium nucleatum and Peptostreptococcus stomatis.
Tumor-related genes TRPM4, MYBL2, and CDKN2A were significantly
upregulated and correlated with specific bacterial taxa.ConclusionThis
study underscores the significant alterations in gut microbiota
associated with CRC and reveals novel correlations between specific
microbes and host gene expression, offering potential diagnostic markers
and therapeutic targets for CRC.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547057"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547057</a></p>
<hr />
<h2
id="different-patterns-of-leukocyte-immune-responses-to-infection-of-ancestral-sars-cov-2-and-its-variants">Different
patterns of leukocyte immune responses to infection of ancestral
SARS-CoV-2 and its variants</h2>
<p>date: 2025-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundContributions of leukocytes to severe acute respiratory
syndrome coronavirus 2 (SARS-CoV-2) defense have been reported
extensively. However, it remains unclear whether there are different
leukocyte responses to ancestral SARS-CoV-2 and its variants.MethodsWe
analyzed peripheral blood leukocyte and subtype concentrations from 575
COVID-19 patients and 950 non-COVID-19 subjects registered at the
University of Connecticut John Dempsey Hospital between 2020 and 2022,
which covers the ancestral strain, Delta, and Omicron variants.ResultsWe
found that neutrophils, immature granulocytes, and monocytes were
elevated, and lymphocytes were reduced after infection. These
hyperactive neutrophils/immature granulocytes and suppressed
lymphocytes/monocytes were associated with poorer prognosis in ancestral
strain infection. Different from the ancestral strain, hyperactive
immature granulocytes were not shown in the decedents of Delta
infection, and immature granulocyte concentration was not observed to be
associated with mortality. In Omicron infection, suppressed lymphocytes
and monocytes were not shown in the decedents, and lymphocyte/monocyte
concentrations were not associated with mortality.ConclusionsOur
findings provided insights into different leukocyte immune responses to
ancestral SARS-CoV-2, Delta, and Omicron variants.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1508120"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1508120</a></p>
<hr />
<h2
id="linking-peri-implantitis-to-microbiome-changes-in-affected-implants-healthy-implants-and-saliva-a-cross-sectional-pilot-study">Linking
peri-implantitis to microbiome changes in affected implants, healthy
implants, and saliva: a cross-sectional pilot study</h2>
<p>date: 2025-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe rising use of dental implants is accompanied by an
expected increase in peri-implant diseases, particularly
peri-implantitis (PI), which poses a significant threat to implant
success and necessitates a thorough understanding of its pathogenesis
for effective management.MethodsTo gain deeper insights into the role
and impact of the peri-implant microbiome in the pathogenesis and
progression of PI, we analyzed 100 samples of saliva and subgingival
biofilm from 40 participants with healthy implants (HI group) or with
co-occurrence of diagnosed PI-affected implants and healthy implants (PI
group) using shotgun metagenomic sequencing. We identified the most
discriminative species distinguishing healthy from diseased study groups
through log ratios and differential ranking analyses.Results and
discussionMogibacterium timidum, Schaalia cardiffensis, Parvimonas
micra, Filifactor alocis, Porphyromonas endodontalis, Porphyromonas
gingivalis and Olsenella uli were associated with the subgingival
peri-implant biofilm. In contrast, Neisseria sp oral taxon 014,
Haemophilus parainfluenzae, Actinomyces naeslundii, Rothia mucilaginosa
and Rothia aeria were more prevalent in the healthy peri-implant
biofilm. Functional pathways such as arginine and polyamine
biosynthesis, including putrescine and citrulline biosynthesis, showed
stronger correlations with PI-affected implants. In contrast,
peri-implant health was characterized by the predominance of pathways
involved in purine and pyrimidine deoxyribonucleotide de novo
biosynthesis, glucose and glucose-1-phosphate degradation, and
tetrapyrrole biosynthesis. Our findings reveal that healthy implants in
PI-free oral cavities differ significantly in microbial composition and
functional pathways compared to healthy implants co-occurring with
PI-affected implants, which more closely resemble PI-associated
profiles. This pattern extended to salivary samples, where microbial and
functional biomarkers follow similar trends.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1543100"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1543100</a></p>
<hr />
<h2
id="cytokine-concentration-and-t-cell-subsets-in-the-female-genital-tract-in-the-presence-of-bacterial-vaginosis-and-trichomonas-vaginalis">Cytokine
concentration and T cell subsets in the female genital tract in the
presence of bacterial vaginosis and Trichomonas vaginalis</h2>
<p>date: 2025-04-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Trichomonas vaginalis (TV) and bacterial vaginosis (BV) are highly
prevalent vaginal infections. Both are associated with pelvic
inflammatory disease and HIV acquisition and transmission, though the
underlying mechanisms are incompletely understood. We characterized the
effect of TV and BV infection on inflammatory markers in the vagina
among reproductive-aged women in Atlanta, Georgia. Cervicovaginal lavage
specimens were collected from HIV seronegative women at a baseline visit
and again three months later. Eighteen individual cytokines, 17 T cell
subsets, BV, and TV were measured at both timepoints. After natural log
transformation, the median cytokine concentration and number of T cells
were compared by infection status statistically using the Kruskal-Wallis
test. A cytokine inflammation score and a T cell score were created
using principal components analysis. The scores were then used as
outcomes in separate linear mixed regression models with a random
intercept. Sixty women had baseline data and 43 were seen for follow-up.
The median age was 30 years, 78% self-reported Black race. TV and BV
prevalence at the baseline visit was 15% and 37%, respectively. The
concentration of 16 out of 18 cytokines differed by infection status. In
multivariable modeling, neither TV nor BV were associated with cytokine
score. Most CD4+ T cell subsets (7 out of 9) differed by infection
status. In a multivariable model, TV infection was associated with a
higher T cell score (1.54; 95%CI 0.00, 3.08). BV was not associated with
a higher T cell score. Increased concentration of vaginal mucosal T
cells may explain the observed association between TV infection and HIV
risk.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1539086"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1539086</a></p>
<hr />
<h2
id="camp-independent-crp-homolog-adds-to-the-multi-layer-regulatory-network-in-porphyromonas-gingivalis">cAMP-independent
Crp homolog adds to the multi-layer regulatory network in Porphyromonas
gingivalis</h2>
<p>date: 2025-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionPorphyromonas gingivalis encodes three CRP/FNR
superfamily proteins: HcpR, PgRsp, and CrpPg, with CrpPg similar to
cAMP-sensing proteins but not classified into known families. This study
investigates the role of CrpPg in regulating the expression of factors
essential for P. gingivalis virulence in A7436 and ATCC 33277
strains.MethodsThe role of CrpPg protein in P. gingivalis was determined
using the ΔcrpPg mutant strains to characterize their phenotype and to
assess the impact of crpPg inactivation on gene expression using RNA-seq
and RT-qPCR. Additionally, the CrpPg protein was purified and
characterized. ResultsKey findings in the ΔcrpPg mutant strain include
up-regulated mfa1-5 and rgpA genes and down-regulated trxA, soxR, and
ustA genes. While crpPg inactivation does not affect growth in liquid
culture media, it impairs biofilm formation and enhances adhesion to and
invasion of gingival keratinocytes. CrpPg binds directly to its own and
mfa promoters without interacting with cyclic nucleotides or
di-nucleotides. Its three-dimensional structure, resembling E. coli Crp
in complex with cAMP and DNA, suggests that CrpPg functions as a global
regulator independently of cAMP binding. The highest crpPg expression in
the early exponential growth phase declines as cell density and
metabolic conditions change over time, suggesting a regulatory function
depending on the CrpPg protein amount.ConclusionsBy controlling the
shift from planktonic to biofilm lifestyle, CrpPg may play a role in
pathogenicity. Regulating the expression of virulence factors required
for host cell invasion and intracellular replication, CrpPg may help P.
gingivalis evade immune responses.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535009"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535009</a></p>
<hr />
<h2
id="the-application-prospect-of-metagenomic-next-generation-sequencing-technology-in-diagnosing-suspected-lower-respiratory-tract-infections">The
application prospect of metagenomic next-generation sequencing
technology in diagnosing suspected lower respiratory tract
infections</h2>
<p>date: 2025-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveLower respiratory tract infections present substantial
diagnostic and therapeutic challenges, negatively impacting individual
health. This study aims to utilize metagenomic next-generation
sequencing (mNGS) technology to comprehensively explore the spectrum of
pathogens, the detection of antibiotic resistance genes, and
contributing factors associated with lung infections.MethodThe mNGS data
of 217 patients with suspected lung infections attending the Respiratory
Department of Nanjing Lishui People’s Hospital and Gaochun People’s
Hospital from September 2022 to September 2023 were retrospectively
analyzed. The study assessed the pathogenic spectrum of lung infections
and compared the performance of patients with mNGS results from
conventional microbiological techniques (CMT).ResultsThe overall
positivity rate of mNGS was 95.20%, demonstrating superior sensitivity
(97.01% vs. 41.79%) and accuracy (75.56% vs. 56.67%) compared to CMT.
Bacterial infections were the most prevalent, accounting for 60.76% of
cases. And the most prevalent bacteria, fungus and virus were
Mycobacterium tuberculosis (14.41%), Candida albicans (15.72%), and EB
virus (14.85%), respectively. The primary resistance genes detected were
tetM (17, 8.29%), mel (6, 2.93%), and PC1 beta-lactamase (blaZ) (3,
1.46%). Notably, TEM-183, PDC-5 and PDC-3 were exclusively detected in
the Chronic Obstructive Pulmonary Disease (COPD) group. The multivariate
binary logistic regression analysis revealed that there was no
significant association between gender, presence of hypertension, or
COPD with the type of infection in patients (p=0.679, p=0.229, p=0.345).
However, the immune status was found to be statistically significant
(p=0.009).ConclusionWith the guidance of mNGS, patients with suspected
respiratory tract infections can rapidly and accurately establish a
pathogenic basis for their conditions. mNGS effectively identify mixed
infections, enrich the pathogen spectrum of lung infections, and provide
a large and reliable information base for the clinical realization of
targeted medication.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1494638"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1494638</a></p>
<hr />
<h2
id="the-immune-mechanisms-of-the-urinary-tract-against-infections">The
immune mechanisms of the urinary tract against infections</h2>
<p>date: 2025-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Urinary tract infection (UTI), a common clinical infectious disease,
is marked by high incidence and frequent recurrence. Recurrent UTIs can
cause severe complications, negatively affecting health. The emergence
and spread of drug-resistant bacteria present significant challenges to
UTI treatment. This article systematically reviews the key immune
mechanisms in the body’s defense against UTI pathogens. It discusses
various immune response components, such as the urinary tract mucosal
epithelium, neutrophils, macrophages, dendritic cells, mast cells,
innate lymphocytes, T cells, and B cells, with the aim of providing
insights for future UTI research.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540149"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540149</a></p>
<hr />
<h2
id="the-effects-of-cyclic-peptide-r4w4-in-combination-with-first-line-therapy-on-the-survival-of-mycobacterium-avium">The
effects of cyclic peptide [R4W4] in combination with first-line therapy
on the survival of Mycobacterium avium</h2>
<p>date: 2025-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMycobacterium avium (M. avium) is a nontuberculous
mycobacterium (NTM) that can cause pulmonary and extrapulmonary
infections mostly in immunocompromised individuals, such as those with
HIV and diabetes. Traditionally, rifampicin (RIF) and azithromycin (AZ)
have been used for a 12-month duration as first-line antibiotics against
M. avium. Due to the increased multidrug resistance, novel ways, such as
enhancement of macrophages response, are needed to provide adequate
immune response required to clear M. avium infection.Methods and
findingsIn this study, we aim to study the effects of using THP-1 cells,
which are monocyte-like cells, to induce a macrophage response and
control M. avium infection when used in combination with traditional
treatments such as RIF and AZ in free and liposomal forms. Traditional
treatments’ effects are studied when used alone and in combination
therapy with cyclic peptide [R4W4] (liposomal encapsulated and liposomal
combination). Colony-forming units (CFU) counts were assessed for all
samples 3 hours, 4 days, and 8 days post-treatment. A significant
reduction in the intracellular viability of M. avium was observed when
THP-1 cells were treated with liposomal combination [R4W4]+RIF and
liposomal combination [R4W4]+AZ compared to when treated with liposomal
RIF or liposomal AZ alone, respectively.ConclusionOur findings show that
liposomal combination [R4W4] is a promising adjuvant therapy to increase
M. avium susceptibility to known antibiotics.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547376"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1547376</a></p>
<hr />
<h2
id="uncovering-the-role-of-mitochondrial-genome-in-pathogenicity-and-drug-resistance-in-pathogenic-fungi">Uncovering
the role of mitochondrial genome in pathogenicity and drug resistance in
pathogenic fungi</h2>
<p>date: 2025-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Fungal infections are becoming more prevalent globally, particularly
affecting immunocompromised populations, such as people living with HIV,
organ transplant recipients and those on immunomodulatory therapy.
Globally, approximately 6.55 million people are affected by invasive
fungal infections annually, leading to serious health consequences and
death. Mitochondria are membrane-bound organelles found in almost all
eukaryotic cells and play an important role in cellular metabolism and
energy production, including pathogenic fungi. These organelles possess
their own genome, the mitochondrial genome, which is usually circular
and encodes proteins essential for energy production. Variation and
evolutionary adaptation within and between species’ mitochondrial
genomes can affect mitochondrial function, and consequently cellular
energy production and metabolic activity, which may contribute to
pathogenicity and drug resistance in certain fungal species. This review
explores the link between the mitochondrial genome and mechanisms of
fungal pathogenicity and drug resistance, with a particular focus on
Cryptococcus neoformans and Candida albicans. These insights deepen our
understanding of fungal biology and may provide new avenues for
developing innovative therapeutic strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1576485"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1576485</a></p>
<hr />
<h2
id="editorial-detection-and-drug-treatment-of-emerging-viral-diseases">Editorial:
Detection and drug treatment of emerging viral diseases</h2>
<p>date: 2025-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1606637"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1606637</a></p>
<hr />
<h2
id="aptamer-functionalized-graphene-quantum-dots-combined-with-artificial-intelligence-detect-bacteria-for-urinary-tract-infections">Aptamer-functionalized
graphene quantum dots combined with artificial intelligence detect
bacteria for urinary tract infections</h2>
<p>date: 2025-04-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesUrinary tract infection is one of the most prevalent forms
of bacterial infection, and prompt and efficient identification of
pathogenic bacteria plays a pivotal role in the management of urinary
tract infections. In this study, we propose a novel approach utilizing
aptamer-functionalized graphene quantum dots integrated with an
artificial intelligence detection system (AG-AI detection system) for
rapid and highly sensitive detection of Escherichia coli (E.
coli).MethodsFirstly, graphene quantum dots were modified with the
aptamer that can specifically recognize and bind to E. coli. Therefore,
the fluorescence intensity of graphene quantum dots was positively
correlated with the concentration of E. coli. Finally, the fluorescence
images were processed by artificial intelligence system to obtain the
result of bacterial concentration.ResultsThe AG-AI detection system,
with wide linearity (103-109 CFU/mL) and a low detection limit (3.38×102
CFU/mL), can effectively differentiate between E. coli and other urinary
tract infection bacteria. And the result of detection system is in good
agreement with MALDI-TOF MS.ConclusionsThe detection system is an
accurate and effective way to detect bacteria in urinary tract
infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1555617"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1555617</a></p>
<hr />
<h2
id="sars-cov-2-causes-significant-abnormalities-in-the-fibrinolysis-system-of-patients-correlation-between-viral-mutations-variants-and-thrombosis">SARS-COV-2
causes significant abnormalities in the fibrinolysis system of patients:
correlation between viral mutations, variants and thrombosis</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCoronavirus disease (COVID-19) is reported as a complex
disorder affecting multiple systems and coagulopathy that can cause
mortality. In this study, we investigated the correlation of SARS-CoV-2
mutations found in blood samples with various changes in the
fibrinolysis system, as well as the severity of the disease based on
outcome and whether or not these patients were admitted into the
ICU.Materials and methodsCOVID-19 patients (n = 446) admitted to our
institute between 2021 and 2022 were recruited. Blood samples were
collected, and a sequence analysis of the SARS-CoV-2 spike gene was
isolated from the blood. Measured several parameters of fibrinolysis and
coagulation, including alpha-2-antiplasmin and plasminogen, thrombin
activatable fibrinolysis inhibitor (TAFI), tissue plasminogen activator
(tPA), plasminogen activator inhibitor-1 (PAI-1), D-dimer, and
fibrinogen levels.ResultsSARS-CoV-2 RNA was found in 123/446 (27.6%) of
the blood samples. The N501Y, D614G, K417N, and P681R mutations among
COVID-19 patients were associated with higher admissions to the ICU (P =
0.0057, P = 0.0068, P = 0.0193, and P = 0.018, respectively). Omicron
(BA.1.1) variant variants are highly associated with thrombosis (P =
0.002) in hospitalized COVID-19 patients that are unvaccinated and have
comorbidity conditions. The plasma levels of tPA, aPTT, and D-dimer were
significantly higher in participants who had the N501Y mutation (P =
0.044, P = 0.024, and P = 0.027, respectively).ConclusionThrombosis was
the most prevalent condition among severe COVID-19 patients. The
correlation between specific SARS-CoV-2 new variants and thrombosis
warrants more investigation.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1531412"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1531412</a></p>
<hr />
<h2
id="enhancing-fever-of-unknown-origin-diagnosis-machine-learning-approaches-to-predict-metagenomic-next-generation-sequencing-positivity">Enhancing
fever of unknown origin diagnosis: machine learning approaches to
predict metagenomic next-generation sequencing positivity</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveMetagenomic next-generation sequencing (mNGS) can
potentially detect various pathogenic microorganisms without bias to
improve the diagnostic rate of fever of unknown origin (FUO), but there
are no effective methods to predict mNGS-positive results. This study
aimed to develop an interpretable machine learning algorithm for the
effective prediction of mNGS results in patients with FUO.MethodsA
clinical dataset from a large medical institution was used to develop
and compare the performance of several predictive models, namely eXtreme
Gradient Boosting (XGBoost), Light Gradient-Boosting Machine (LightGBM),
and Random Forest, and the Shapley additive explanation (SHAP) method
was employed to interpret and analyze the results.ResultsThe
mNGS-positive rate among 284 patients with FUO reached 64.1%. Overall,
the LightGBM-based model exhibited the best comprehensive predictive
performance, with areas under the curve of 0.84 and 0.93 for the
training and validation sets, respectively. Using the SHAP method, the
five most important factors for predicting mNGS-positive results were
albumin, procalcitonin, blood culture, disease type, and sample
type.ConclusionThe validated LightGBM-based predictive model could have
practical clinical value in enhancing the application of mNGS in the
etiological diagnosis of FUO, representing a powerful tool to optimize
the timing of mNGS.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1550933"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1550933</a></p>
<hr />
<h2
id="characteristics-and-outcomes-in-severe-and-critically-ill-children-with-first-wave-sars-cov-2-omicron-infection-in-northeast-china">Characteristics
and outcomes in severe and critically ill children with first wave
SARS-CoV-2 Omicron infection in Northeast China</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>AimTo describe the characteristics of severe and critically ill
children with first-wave SARS-CoV-2 Omicron infection admitted to the
pediatric intensive care unit (PICU) at the National Children’s Regional
Medical Center in Northeast China and to explore factors associated with
poor outcomes.MethodsThis observational cohort study was conducted in a
PICU in northeastern China and included children under 18 years of age
who were severely and critically ill due to SARS-CoV-2 Omicron infection
between December 2022 and February 2023. Patients were categorized into
two groups: the invasive mechanical ventilation (IMV) group and the
non-IMV group. The primary outcome measured was the need for IMV, while
secondary outcomes included death or prolonged PICU stay. Univariate and
multivariate logistic regression analyses were performed to identify
risk factors for poor outcomes.ResultsA total of 38 severe and
critically ill children were included in the study. Of these, 25 (66%)
were diagnosed with respiratory failure, and four (16%) developed acute
respiratory distress syndrome. Additionally, 21 (55%) were diagnosed
with COVID-19-associated neurological disorders, and 18 (47%) received
IMV. Multivariate logistic regression analysis identified the chest
computed tomography (CT) score, based on the COVID-19 Risk Assessment
and Diagnosis System (CO-RADS), was statistically significant as an
independent predictor for IMV in severe and critically ill children
(odds ratio [OR]: 2.781 [95% confidence interval (CI): 1.021–7.571]).
Furthermore, the Pediatric Logistic Organ Dysfunction-2 (PELOD-2) score
and serum aspartate aminotransferase (AST) levels at admission were
found to be independent predictors of death or prolonged PICU
stay.ConclusionsRespiratory failure and COVID-19-associated neurological
disorders were the most common complications among severe and critically
ill children with first-wave SARS-CoV-2 Omicron infection. Chest CT
score, PELOD-2 score, and serum AST levels may serve as important
indicators of poor outcomes in this patient population.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1495783"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1495783</a></p>
<hr />
<h2
id="development-of-the-breastfed-infant-oral-microbiome-over-the-first-two-years-of-life-in-the-blosom-cohort">Development
of the breastfed infant oral microbiome over the first two years of life
in the BLOSOM Cohort</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAcquisition and development of the oral microbiome are
dynamic processes that occur over early life. This study aimed to
characterize the temporal development of the oral microbiome of
predominantly breastfed infants during the first two years of
life.MethodsInfant oral samples (n=667) were collected at ten time
points from the first week to year two of life from 84 infants.
Bacterial DNA profiles were analyzed using full-length 16S rRNA gene
sequencing.ResultsThe oral microbiome was dominated by Streptococcus
mitis, Gemella haemolysans, and Rothia mucilaginosa. Bacterial richness
decreased from 1 to 2 months (P = 0.043) and increased from 12 to 24
months (P = 0.038). Shannon diversity increased from 1 week to 1 month
and again from 6 to 9 months and 9 to 12 months (all P ≤ 0.04). The
composition of the infant oral microbiome was associated with multiple
factors, including early pacifier use, intrapartum antibiotic
prophylaxis, maternal allergy, pre-pregnancy body mass index, siblings,
delivery mode, maternal age, pets at home, and birth season (all P &lt;
0.01). Introduction of solid foods was a significant milestone in oral
microbiome development, triggering an increase in bacterial diversity
(richness P = 0.0004; Shannon diversity P = 0.0007), a shift in the
abundance of seven species, and a change in beta diversity (P =
0.001).ConclusionThese findings underscore the importance of multiple
factors, particularly the introduction of solid foods, in shaping the
oral microbiome of breastfed infants during early life.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534750"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534750</a></p>
<hr />
<h2
id="antifungal-susceptibility-and-clinical-efficacy-of-chlorhexidine-combined-with-topical-ophthalmic-medications-against-fusarium-species-isolated-from-corneal-samples">Antifungal
susceptibility and clinical efficacy of chlorhexidine combined with
topical ophthalmic medications against Fusarium species isolated from
corneal samples</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study investigated the susceptibility of various
Fusarium fungi to five topical antifungal agents: natamycin,
voriconazole, chlorhexidine, natamycin combined with chlorhexidine, and
voriconazole combined with chlorhexidine. And to explore the clinical
feasibility of combination therapy in the treatment of corneal
infections caused by F. solani, with the goal of optimizing the
treatment regimen for fungal keratitis.MethodsA total of 194 strains of
Fusarium were isolated from the corneas between 2013 and 2024 and
identified to the species level using mass spectrometry. The MICs was
determined using a commercial microdilution plate to assess the in vitro
activity of the drugs used alone and in combination
(natamycin/chlorhexidine, voriconazole/chlorhexidine). Additionally, the
clinical efficacy was prospectively observed in 5 patients with corneal
infections caused by F. solani. The treatment regimens included 5%
natamycin combined with 0.04% chlorhexidine, chlorhexidine used alone,
or natamycin used alone, with follow-up lasting up to 90 days.ResultsF.
solani species complex (FSSC, 46.91%) and the F. fujikuroi species
complex (FFSC, 45.88%) were the predominant isolates, with a
geographical distribution concentrated in Northern China. The MICs for
natamycin in FSSC and FFSC were both 2–8 μg /mL, respectively. The MICs
for FSSC and FFSC respectively ranged from 0.25–16 and 1–8 μg/mL for
voriconazole and 2 to &gt; 16 μg/mL and 1 to &gt; 16 μg/mL for
chlorhexidine. The MICs of natamycin were not significantly different
between FSSC and FFSC. However, voriconazole, chlorhexidine, natamycin
combined with chlorhexidine, and voriconazole combined with
chlorhexidine had significantly higher MICs for FSSC compared with FFSC.
Compared with voriconazole, voriconazole combined with chlorhexidine
exhibited enhancement of antifungal activity against 100% of tested
Fusarium strains. Compared with natamycin, enhancement of antifungal
activity of natamycin combined with chlorhexidine was 81.4% for all
Fusarium spp., and the activity were significantly lower for F. solani
(65.9%) than for non-F. solani species (93.6%). Among the 5 patients, 3
patients received treatment with natamycin combined with chlorhexidine,
resulting in clinical cure in 2 patients (in 1-1.5 months), while 1
patient required a corneal transplant due to delayed treatment. One
patient treated with natamycin alone and one treated with chlorhexidine
alone both achieved clinical cure (in 2-3 months).ConclusionNatamycin
combined with chlorhexidine and voriconazole combined with chlorhexidine
exhibited enhancement of antifungal activity against Fusarium spp.
during in vitro sensitivity tests. The findings of this study provide
valuable guidance for establishing the epidemiological cutoff and
clinical MIC values for Fusarium spp. This study paves the way for
future multicenter studies on the treatment of FK with natamycin and
chlorhexidine.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1532289"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1532289</a></p>
<hr />
<h2
id="custom-barcoded-primers-for-influenza-a-nanopore-sequencing-enhanced-performance-with-reduced-preparation-time">Custom
barcoded primers for influenza A nanopore sequencing: enhanced
performance with reduced preparation time</h2>
<p>date: 2025-04-15, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Highly pathogenic avian influenza is endemic and widespread in wild
birds and is causing major outbreaks in poultry worldwide and in U.S.
dairy cows, with several recent human cases, highlighting the need for
reliable and rapid sequencing to track mutations that may facilitate
viral replication in different hosts. SNP analysis is a useful molecular
epidemiology tool to track outbreaks, but it requires accurate
whole-genome sequencing (WGS) with sufficient read depth across all
eight segments. In outbreak situations, where timely data is critical
for controlling the spread of the virus, reducing sequencing preparation
time while maintaining high-quality standards is particularly important.
In this study, we optimized a custom barcoded primer strategy for
influenza A whole-genome sequencing on the nanopore sequencing platform,
combining the high performance of the Native Barcoding Kit with the
prompt preparation time of the Rapid Barcoding Kit. Custom barcoded
primers were designed to perform barcode attachment during RT-PCR
amplification, eliminating the need for separate barcoding and clean-up
steps, thus reducing library preparation time. We compared the
performance of the custom barcoded primer method with the Native and
Rapid barcoding kits in terms of read quality, read depth, and
sequencing output. The results show that the custom barcoded primers
provided performance comparable to the Native Barcoding Kit while
reducing library preparation time by 2.3X compared to the Native kit and
being only 15 minutes longer than the Rapid kit with better depth of
sequencing. Additionally, the custom barcoded primer method was
evaluated on a variety of clinical sample types. This approach offers a
promising solution for influenza A sequencing, providing both high
throughput and time efficiency, which significantly improves the
time-to-result turnaround, making sequencing more accessible for
real-time surveillance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545032"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1545032</a></p>
<hr />
<h2
id="treatment-of-xerostomia-in-sjögrens-syndrome-what-effect-does-it-have-on-the-oral-microbiome">Treatment
of xerostomia in Sjögren’s syndrome – what effect does it have on the
oral microbiome?</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Sjögren’s syndrome is an autoimmune disease characterized by
lymphatic infiltration of secretory tissues. The disease results in
dryness of the eyeball or mouth, which often occur simultaneously.
Agents used to treat Sjögren’s syndrome may improve oral hydration and
the patient’s quality of life. There are several pharmacological and
non-pharmacological agents used to treat significant problem like
xerostomia. The use of appropriate medicines (i.e. pilocarpine and
cevimeline) may cause changes in the local microbiome, which is very
sensitive to quantitative changes in water. As a result of Sjögren’s
syndrome, a new balance of the microbiome is established in the oral
cavity, which, if disturbed by medical measures, may increase the risk
of oral lesions (such as periodontopathies or caries) or reduce this
risk. Overall, the knowledge regarding microbiological aspects and
agents treating oral dryness is still not well described but initial
results indicate some microbial alterations.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1484951"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1484951</a></p>
<hr />
<h2
id="distinct-blood-and-oral-microbiome-profiles-reveal-altered-microbial-composition-and-functional-pathways-in-myocardial-infarction-patients">Distinct
blood and oral microbiome profiles reveal altered microbial composition
and functional pathways in myocardial infarction patients</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe blood microbiome, increasingly recognized as a
distinct microbial niche, may originate partly from oral translocation.
We systematically compared circulating and oral microbiome profiles
between healthy individuals and myocardial infarction (MI) patients to
identify disease-associated signatures.MethodsThe current study
recruited 20 participants, including 10 healthy controls and 10 patients
with MI. Blood and Saliva samples were collected from each participant
to analyze the association between blood and oral microbiome in MI
patients using 16S rRNA gene sequencing.ResultsThe blood microbiome
showed significantly greater alpha diversity than the oral microbiome
(p&lt;0.05), but beta diversity did not differ significantly. The blood
microbiome in MI patients had higher levels of Firmicutes, Bacteroidota,
Actinobacteriota, genus Bacteroides, and lower Proteobacteria, whereas
the oral microbiome was dominated by Firmicutes, Bacteroidota,
Veillonella, and Prevotella_7. LEfSe analysis revealed distinct blood
microbial taxa-Actinobacteria in MI patients and Enterobacterales in
controls. In contrast, the oral microbiota of healthy subjects was
enriched in Rothia, Micrococcaceae, and Micrococcales, while no distinct
taxa were associated with MI. Both blood and oral microbiomes showed
significant functional pathway differences (KEGG) between groups.
Additionally, microbiome signatures significantly correlated with
clinical and demographic markers.DiscussionOur study demonstrates that
the blood harbors a distinct microbiome characterized by specific
microbial taxa and functional pathways rather than merely reflecting
oral bacterial translocation. These findings suggest that the
circulating microbiome may play an active role in the pathology of MI.
Furthermore, we identified significant associations between these
microbial signatures and clinical disease markers. This highlights the
potential importance of the blood microbiome in understanding the
mechanisms underlying MI and its diagnostic or therapeutic
implications.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1506382"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1506382</a></p>
<hr />
<h2
id="vaginal-microbiota-changes-of-persistent-human-papillomavirus-infection-after-cervical-conization">Vaginal
microbiota changes of persistent human papillomavirus infection after
cervical conization</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveWe investigated the changes in vaginal microbiota among
females with persistent human papillomavirus (HPV) infection following
cervical conization in Xinjiang, China.MethodsA total of 108 female
participants were enrolled in the study, including 37 HPV-positive
females without cervical conization (Group P1), 37 HPV-positive females
after cervical conization (Group P2), and 34 HPV-negative females after
cervical conization (Group N). DNA was extracted from vaginal
secretions, and the V3-V4 regions of bacterial 16S rDNA were amplified
and sequenced using NovaSeq technology. The diversity analysis of the
bacterial microbiota was conducted using QIIME2 and R software, while
the phenotypic analysis was performed with Bugbase
software.ResultsLactobacillus was the predominant genus in the vaginal
microbiota of women with persistent HPV infection after cervical
conization in Xinjiang. Following partial cervical resection, the
α-diversity of the vaginal microbiota decreased, particularly among
patients who had cleared HPV. Bacterial vaginosis-associated anaerobes
were common in the vaginal environment, with their relative abundance
increasing in cases of persistent HPV infection. Postoperative
persistent HPV infection was found to be correlated not only with
pathogens linked to bacterial vaginosis but also with those associated
with aerobic vaginitis. Gardnerella and Atopobium, as well as
Bifidobacterium and Streptococcus, demonstrated a symbiotic synergy.
Both Lactobacillus and Gardnerella exhibited negative correlations with
many pathogenic bacteria. Anaerobic and biofilm formation were the most
evident phenotypes in individuals with persistent HPV infection after
conization.ConclusionThe vaginal microbiota of women with persistent HPV
infection following cervical conization is characterized by the
coexistence of Lactobacillus dominance and increased microbial
diversity. Anaerobic bacteria and biofilm formation may play a
significant role in the persistence of HPV infection post-surgery, and
the role of Gardnerella in the vaginal flora under an HPV-infected state
warrants further study.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1544794"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1544794</a></p>
<hr />
<h2
id="ehrlichia-chaffeensis-proteomic-profiling-reveals-distinct-expression-patterns-of-infectious-and-replicating-forms">Ehrlichia
chaffeensis proteomic profiling reveals distinct expression patterns of
infectious and replicating forms</h2>
<p>date: 2025-04-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Ehrlichia chaffeensis is a tick-transmitted rickettsial pathogen
responsible for causing human monocytic ehrlichiosis (HME). The
pathogen’s developmental cycle includes infectious dense-core cells
(DCs) and non-infectious replicating cells (RCs). Defining the proteins
crucial for the two growth forms is of fundamental importance in
understanding the infection and replication process, which also aids in
identifying novel therapeutic targets against HME and other related
rickettsial diseases. E. chaffeensis organisms cultivated in a
macrophage cell line were purified as DC and RC fractions and subjected
to comprehensive quantitative proteome analysis. From triplicate sample
analysis, we identified 195 proteins as commonly expressed in both the
DC and RC forms, while an additional 189 proteins were recognized as
exclusively expressed in the RC form. Equal numbers of commonly
expressed proteins in the RC and DC forms and having substantially more
proteins exclusively expressed in the metabolically active RC form may
reflect specific functional priorities of E. chaffeensis supporting its
replication within a phagosome. The high abundance of metabolic
processes and transport proteins in the RC compared to the DC form may
reflect its higher metabolic requirements and interactions with a host
cell supporting its intraphagosomal replication. This study provides
comprehensive proteome data for E. chaffeensis which will be valuable
for a better understanding of protein expression dynamics during its
infectious and replicating stages.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1463479"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1463479</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
