<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2025-04-29 06:12:40)</p>
<hr />
<h2
id="egg-driven-immunosuppression-and-granuloma-zonation-in-peyers-patches-of-mice-with-schistosoma-japonicum-infection">Egg-driven
immunosuppression and granuloma zonation in Peyer’s patches of mice with
Schistosoma japonicum infection</h2>
<p>date: 2025-04-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Egg granulomas caused by Schistosoma japonicum (S. japonicum) are
important causes of morbidity and mortality in schistosomiasis. The
intestine plays a crucial role in the complete life cycle of S.
japonicum; eggs are transported through the intestine and excreted with
feces. During this process, the interaction between the eggs and the
intestine can trigger a strong intestinal immune system response and
cause inflammation. Eggs in the intestine preferentially accumulate in
Peyer’s patches (PPs). However, the cellular composition of intestinal
granulomas and the impacts of egg deposition on the immune function of
PPs remain poorly understood. Using a mouse model of S. japonicum
infection, we revealed that the deposition of eggs disrupted the
structure of PPs, resulting in immunosuppression. We further
characterized the cellular composition of intestinal granulomas,
revealing a layered distribution of neutrophils, macrophages, T cells,
and B cells, with marked neutrophil accumulation. Single-cell RNA
sequencing revealed that egg deposition drives B-cell apoptosis, T-cell
exhaustion, and activation of fibrotic pathways in myeloid cells,
collectively impairing PP function. In conclusion, the layered cellular
architecture of intestinal granulomas in PPs suggests a unique immune
microenvironment of egg-driven immunosuppression and fibrotic
remodeling, and the identification of fibrotic pathways in myeloid cells
provides a potential therapeutic target to alleviate fibrosis in
patients with S. japonicum infection.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587166"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587166</a></p>
<hr />
<h2
id="editorial-interaction-of-microbiota-and-metabolomic-disorders">Editorial:
Interaction of microbiota and metabolomic disorders</h2>
<p>date: 2025-04-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1597214"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1597214</a></p>
<hr />
<h2
id="screening-of-neurotransmitter-receptor-modulators-reveals-novel-inhibitors-of-influenza-virus-replication">Screening
of neurotransmitter receptor modulators reveals novel inhibitors of
influenza virus replication</h2>
<p>date: 2025-04-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Influenza presents a significant public health threat, as severe
cases can lead to excessive inflammation and complications such as
pneumonia or acute respiratory distress syndrome. Current antiviral
agents targeting viral proteins may lead to the development of
resistance, highlighting the need for new agents targeting host factors.
Neurotransmitter receptors are vital for cellular signaling and cell
cycle modulation, making them promising antiviral therapeutic targets.
Recent research has demonstrated that screening libraries of compounds
aimed at these receptors can help identify inhibitors that prevent the
replication of various viruses, including filoviruses and SARS-CoV-2. We
screened a neurotransmitter receptor modulator library in
influenza-infected U937 cells and found that many adrenergic, histamine,
dopamine, and serotonin receptor agonists and antagonists exhibit
antiviral activity. We identified 20 candidate compounds with IC50
values below 20 μM, suggesting a critical role for these receptors in
influenza replication. Three representative compounds (isoxsuprine,
ciproxifan, and rotigotine) inhibited H1N1 replication in a
dose-dependent manner in multiple cell lines, and were effective against
H1N1, oseltamivir-resistant H1N1, H3N2, and influenza B strains.
Mechanistic studies indicated that these compounds affect virus
internalization during the early infection stages. In a mouse model of
lethal influenza, isoxsuprine significantly decreased lung viral titers,
mitigated pulmonary inflammation, and enhanced survival rates. These
findings highlight neurotransmitter receptors as potential targets for
developing novel anti-influenza agents, providing a foundation for
further optimization of the identified compounds as potential
therapeutic agents.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562650"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562650</a></p>
<hr />
<h2
id="escherichia-coli-aggravates-inflammatory-response-in-mice-oral-mucositis-through-regulating-th17treg-imbalance">Escherichia
coli aggravates inflammatory response in mice oral mucositis through
regulating Th17/Treg imbalance</h2>
<p>date: 2025-04-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMicrobial dysbiosis links to mucosal immune
dysregulation, but the specific bacterial contributions to oral mucosal
inflammation remain unclear. Escherichia coli (E. coli), a pathogen
well-characterized in mucosal immunity and immune regulation studies,
has been observed to be enriched in chronic oral inflammatory lesions
and was reported to modulate T helper 17 cells (Th17)/T regulatory cells
(Treg) homeostasis. Here, we developed an oral mucositis mouse model via
tongue scratch and E. coli topical application to investigate its role
in Th17/Treg imbalance.MethodsThe inflammatory infiltration was
evaluated by macroscopic photography and HE staining. The expression of
inflammatory factors in tongue tissue and peripheral blood of mice were
detected by immunohistochemical staining and enzyme-linked immunosorbent
assay. The number of Th17 and Treg in mice spleen lymphocytes were
evaluated with flow cytometry. Differential gene expression analysis,
functional enrichment analysis and immune infiltration analysis were
performed using RNA-seq data from oral lichen planus (OLP).ResultsE.
coli stimulation aggravated inflammatory responses induced by scratching
in lingual mucosa of mice, including increased local and systemic
expression of interleukin 6 (IL6), interleukin 17 (IL17), chemokine
receptor 6 (CCR6) and chemokine C-C motif ligand 20 (CCL20), increased
proportions of Th17 cells and increased Th17/Treg ratio in spleen
lymphocytes. Analysis of RNA-seq data from OLP revealed alterations in
antimicrobial responses and inflammatory factors associated with
upregulation of Th17/Treg balance.ConclusionThis study supports the role
of E. coli in promoting oral mucosal inflammation and provides an
experimental basis for in vivo study of OLP from the perspective of
microorganisms.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1585020"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1585020</a></p>
<hr />
<h2
id="clinicopathological-characteristics-and-diagnostic-performance-of-metagenomic-pathogen-detection-technology-in-mycobacterial-infections-among-hiv-patients">Clinicopathological
characteristics and diagnostic performance of metagenomic pathogen
detection technology in mycobacterial infections among HIV patients</h2>
<p>date: 2025-04-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMycobacterial infections represent a major cause of
morbidity and mortality in HIV-infected individuals. This study
evaluated diagnostic techniques for mycobacterial identification and
compared clinicopathological features between HIV-positive and
HIV-negative patients.MethodsWe analyzed 88 tissue samples (with 41
matched blood and 28 sputum samples) using histopathology (HE and
acid-fast staining), bacterial culture, MTB-PCR (sputum/biopsy),
PCR-reverse dot blot hybridization (RDBH), and metagenomic pathogen
detection technology (MetaPath™). Logistic regression analyses were
performed to identify factors affecting detection
rates.ResultsMycobacterial infection was detected in 95.5% (84/88) of
patients. Among HIV-positive patients (n=63), 46% (29/63) had
Mycobacterium tuberculosis (MTB) infections, and 44% (28/63) had
non-tuberculous mycobacteria (NTM) infections, significantly higher than
the 20% (5/25) NTM rate in HIV-negative patients. Univariate analysis
identified HIV-positive status (p=0.009), lymph node involvement
(p=0.020), and positive MetaPath™ results (p=0.002) as significant
predictors of detection, while multivariate analysis confirmed these as
independent factors (p=0.036; p=0.042; p=0.006). Lymph nodes were the
most common infection site in HIV-positive patients (42.9%, 27/63),
while lung tissue predominated in HIV-negative patients (48%, 12/25).
MetaPath™ demonstrated superior sensitivity and specificity for
detecting both MTB and NTM. Biopsy samples provided higher diagnostic
accuracy than sputum or blood for lung and lymph node infections, but
not for brain. In HIV-positive patients, NTM infections showed
significantly more granuloma formation (p=0.032) and foam cells
(p=0.005), but less necrosis (p=0.0005) compared to MTB infections. No
significant differences were observed in HIV-negative
patients.ConclusionsMetaPath™ is a highly effective diagnostic tool for
mycobacterial infections, particularly in tissue biopsies. HIV-positive
status, lymph node involvement, and MetaPath™ positivity independently
predict mycobacterial detection. HIV-positive patients exhibit distinct
clinicopathological features, emphasizing the need for tailored
diagnostic and therapeutic approaches based on immune status.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584189"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584189</a></p>
<hr />
<h2
id="comparison-of-the-gut-microbiota-in-older-people-with-and-without-sarcopenia-a-systematic-review-and-meta-analysis">Comparison
of the gut microbiota in older people with and without sarcopenia: a
systematic review and meta-analysis</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionSarcopenia, an age-related disorder marked by decreased
skeletal muscle mass, strength, and function, is associated with
negative health impacts in individuals and financial burdens on families
and society. Studies have suggested that age-related alterations in gut
microbiota may contribute to the development of sarcopenia in older
people through the gut-muscle axis, thus modulation of gut microbiota
may be a promising approach for sarcopenia treatment. However, the
characteristic gut microbiota for sarcopenia has not been consistent
across studies. Therefore, the aim of this study was to compare the
diversity and compositional differences in the gut microbiota of older
people with and without sarcopenia, and to identify gut microbiota
biomarkers with therapeutic potential for sarcopenia.MethodsThe PubMed,
Embase, Web of Science, Cochrane Library, China National Knowledge
Infrastructure, and Wanfang Database were searched studies about the gut
microbiota characteristics in older people with sarcopenia. The quality
of included articles was assessed by the Newcastle-Ottawa Scale (NOS).
Weighted standardized mean differences (SMDs) and 95% confidence
intervals (CIs) for α-diversity index were estimated using a random
effects model. Qualitative synthesis was conducted for β-diversity and
the correlation between gut microbiota and muscle parameters. The
relative abundance of the gut microbiota was analyzed quantitatively and
qualitatively, respectively.ResultsPooled estimates showed that
α-diversity was significantly lower in older people with sarcopenia
(SMD: -0.41, 95% CI: -0.57 to -0.26, I²: 71%, P &lt; 0.00001). The
findings of β-diversity varied across included studies. In addition, our
study identified gut microbiota showing a potential and negative
correlation with sarcopenia, such as Prevotella, Slackia, Agathobacter,
Alloprevotella, Prevotella copri, Prevotellaceae sp., Bacteroides
coprophilus, Mitsuokella multacida, Bacteroides massiliensis,
Bacteroides coprocola Conversely, a potential and positive correlation
was observed with opportunistic pathogens like Escherichia-Shigella,
Eggerthella, Eggerthella lenta and Collinsella
aerofaciens.DiscussionThis study showed that α-diversity is decreased in
sarcopenia, probably predominantly due to diminished richness rather
than evenness. In addition, although findings of β-diversity varied
across included studies, the overall trend toward a decrease in
SCFAs-producing bacteria and an increase in conditionally pathogenic
bacteria. This study provides new ideas for targeting the gut microbiota
for the prevention and treatment of sarcopenia.Systematic review
registrationhttps://www.crd.york.ac.uk/PROSPERO/view/CRD42024573090,
identifier CRD42024573090.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1480293"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1480293</a></p>
<hr />
<h2
id="wound-repair-and-immune-function-in-the-pseudomonas-infected-cf-lung-before-and-after-highly-effective-modulator-therapy">Wound
repair and immune function in the Pseudomonas infected CF lung: before
and after highly effective modulator therapy</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The leading cause of death for people with cystic fibrosis (pwCF)
continues to be due to respiratory-related illnesses. Both wound repair
and immune cell responses are dysregulated in the CF airways, creating a
cycle of unresolved injury and perpetuating inflammation. PwCF are
predisposed to colonization and infections with opportunistic bacteria
like Pseudomonas aeruginosa (Pa), the most common adult pathogen in CF.
Pa possesses key virulence factors that can exacerbate chronic
inflammation and lung injury. With the approval of highly effective
modulator therapies like elexacaftor/tezacaftor/ivacaftor (ETI), pwCF
eligible for ETI have seen drastic improvements in lung function and
clinical outcomes, including an increased life expectancy. While
modulator therapies are improving bronchial epithelial cellular
processes in wound repair and some areas of immunity, many of these
processes do not reach a non-CF baseline state or have not been
thoroughly studied. The effect of modulator therapy on Pa may lead to a
reduction in infection, but in more longitudinal studies, there is not
always eradication of Pa, and colonization and infection frequency can
return to pre-modulator levels over time. Finally, in this review we
explore the current state of additional treatments for CF lung disease,
independent of CFTR genotype, including anti-inflammatories,
phage-therapies, and Pa vaccines.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1566495"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1566495</a></p>
<hr />
<h2
id="genomic-epidemiology-of-a-novel-pandoraea-pneumonica-group-caused-severe-bloodstream-infection-in-hainan-china-2021-2024">Genomic
epidemiology of a novel Pandoraea pneumonica group caused severe
bloodstream infection in Hainan, China, 2021-2024</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionRarely does Pandoraea occur in bloodstream infections
(BSI), although it’s typically found in cystic fibrosis. This study aims
to decipher the genetic map and obtain insights of clinical symptoms
into Pandoraea from BSI patients. Methods30 suspected BSI patients’
diagnostic records and medical histories were recorded. Pandoraea spp.
isolates were collected and subjected to antimicrobial susceptibility
testing, Sanger sequencing and Whole-genome sequencing (WGS). ResultsOf
the 30 clinical cases, five (16.67%) ultimately died, whereas 25
(83.33%) are alive. 30 purified Pandoraea isolates showed high degree of
MIC values to Meropenem, Amoxicillin and Potassium Clavulanate,
Gentamicin, and Ceftazidime. Then, all isolates were identified as P.
pneumonica based on the 16S rRNA-based phylogenetic analysis. Among 28
genomes of them, the average genome size and average GC contents were
5,397,568 bp, and 62.43%, respectively. However, WP1 displayed high
similarity (90.6%) to reference Pandoraea sp. LMG 31114. Genetic
differences between the tested isolates and LMG 31114 suggested that the
outbreak’s causative pathogen could be a novel cluster of P. pneumonica.
The genomes accumulated mutations at an estimated rate of 1.3 × 10-7
mutations/year/site. Moreover, 26 clinical isolates within the P.
pneumonica cluster were formed in July 2014, revealing a tendency to
develop regional endemic patterns. ConclusionBSI caused by this novel
cluster of P. pneumonica is linked to significant morbidity and
mortality. Such cluster remains a critical public health challenge due
to their regional epidemiological patterns and antibiotic treatment
risk. This study contributed to the basis on pathogen identification,
disease diagnosis, and BSI treatment.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560634"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560634</a></p>
<hr />
<h2
id="unveiling-wolbachia-transcriptomic-signature-in-the-arboviral-vector-aedes-aegypti">Unveiling
Wolbachia transcriptomic signature in the arboviral vector Aedes
aegypti</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe mosquito Aedes aegypti is the main vector of
arboviral diseases such as dengue and imposes a global health burden. A
promising control strategy is to infect A. aegypti populations with
Wolbachia, a genus of intracellular bacteria capable of blocking
arboviral infections. Enhancing and preserving the efficacy of this
method will depend on a solid mechanistic knowledge of the A.
aegypti-Wolbachia symbiosis. By identifying differences between
Wolbachia-infected and uninfected A. aegypti, previous transcriptomic
studies proposed a wide range of symbiotic interactions, but a
systematic identification of consistent effects across datasets is still
missing.MethodsTo identify A. aegypti genes and functions consistently
affected by Wolbachia, we performed differential expression and
functional enrichment analysis on published transcriptomic datasets,
followed by a meta-analysis of the obtained p-values using the maxP
method. Six datasets were retrieved from Gene Expression Omnibus,
Sequence Read Archive and ArrayExpress (last searched in July 2024,
considering lack of replication as the exclusion criteria). After
discarding one dataset from wAlbB-infected cell line due to poor mapping
to the A. aegypti genome, the data comprised adult female A. aegypti
heads, muscles, carcasses, midguts and bodies, and Wolbachia strains
wMel and wMelPop.Results and DiscussionMeta-analysis revealed 10 and 21
consistently down- and upregulated host genes, some of which have
escaped the focus of previous research, including the consistently
downregulated exonuclease AAEL009650 which has a pro-dengue virus
homolog in Drosophila. At the function level, we found consistent
upregulation of electron transport chain (ETC), carbohydrate transport
and serine-type peptidase activity and inhibition, and downregulation of
DNA replication. ETC upregulation suggests an alternative mechanism for
Wolbachia’s induction of antiviral oxidative stress, previously
attributed to dual- and NADPH-oxidases which here showed downregulation
or no regulation. Through analysis of previously published datasets,
this work identifies promising molecular and functional targets for
future studies aimed at elucidating the most fundamental mechanisms of
the A. aegypti–Wolbachia symbiosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1538459"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1538459</a></p>
<hr />
<h2
id="plasma-thrombomodulin-is-a-valuable-biomarker-to-predict-the-severity-of-hemorrhagic-fever-with-renal-syndrome-caused-by-the-hantaan-virus">Plasma
thrombomodulin is a valuable biomarker to predict the severity of
hemorrhagic fever with renal syndrome caused by the Hantaan virus</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesThis study aimed to investigate the Thrombomodulin (TM)
levels in patients who suffered hemorrhagic fever with renal syndrome
(HFRS) of varying severities, and to evaluate the predictive properties
of TM for the seriousness of HFRS, thereby providing a clue for the
monitoring and management of this patients in the
future.MethodsChemiluminescence was used to determine the concentrations
of TM in 196 patients with HFRS and 49 healthy controls. Conventional
testing techniques were used to test the basic clinical reference values
for leukocytes, platelets (PLT), C-reactive protein (CRP), creatine
(Cr), uric acid (UA), and urea, and the values for activated partial
thromboplastin time, prothrombin time, and fibrinogen. The colloidal
gold method was used to measure HFRS antibody levels in the patients.
The correlation of TM with conventional parameters was assessed using
Spearman correlation analysis, and ordinal logistic regression analysis
was used to analyze the severity risk factors. The predictive potency of
TM for HFRS patients’ severity was evaluated by receiver operating
characteristics (ROC) curve analysis.ResultsThe concentrations of TM
increased with disease severity and peaked in the critical type
patients. In addition, plasma levels of TM were proportionally
correlated with the levels of leukocytes (r= 0.4218; p&lt;0.01),
creatine (r= 0.3797; p&lt;0.01), urea (r= 0.3763; p&lt;0.01), uric acid
(r= 0.3624; p&lt;0.01), and C-reactive protein (r= 0.2767; p&lt;0.01).
Conversely, there was an inverse correlation between TM, platelet counts
(r= -0.4509; p&lt;0.01), and fibrinogen levels (r= -0.2431; p&lt;0.01).
Furthermore, TM demonstrated significant predictive value for the
severity of HFRS with an area under the ROC curve (AUC) of 0.872(95% CI:
0.822-0.921, P&lt;0.001).ConclusionsTM levels are associated with HFRS
severity, suggesting that TM detection might be beneficial for
monitoring the status and effective management of HFRS patients.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563621"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563621</a></p>
<hr />
<h2
id="integrative-multi-omics-analysis-of-the-microbiome-and-metabolome-in-bronchoalveolar-lavage-fluid-from-patients-with-early-stage-lung-cancer">Integrative
multi-omics analysis of the microbiome and metabolome in bronchoalveolar
lavage fluid from patients with early-stage lung cancer</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Lung cancer is a significant health concern that poses a considerable
threat to human health and quality of life. In order to enhance the
prognosis of patients with lung cancer, we conducted a combined analysis
of 16S rDNA gene sequencing of alveolar lavage fluid and LC-MS
metabolomics research, with the objective of identifying biomarkers in
patients with early-stage lung cancer presenting as SPN. A comparison of
the benign nodule group and the early-stage lung cancer patients
revealed that the phylum-level Bacteroidetes and the genus-level
Chryseobacterium and Delftia were more abundant in the latter group.
Additionally, the Fusobacteriales might serve as a predictive marker for
the diagnosis of early-stage lung cancer. In the context of
metabolomics, the early-stage lung cancer was found to be characterised
by elevated levels of specific metabolites, including Alternariol, dTMP,
Oxymatrine, Gedunin, PC 36:4. Conversely, reductions in other
metabolites, such as LPC O-24:0, PC 18:2_18:3, PC 19:2_19:2,
Cholecalciferol and T-2 Triol, were also observed. Correlation analyses
demonstrated that alveolar lavage microorganisms were closely associated
with differential metabolites. Specifically, reductions in
Cholecalciferol were associated with a variety of high-abundance flora
and involved in vitamin digestion and absorption pathways. Furthermore,
reductions in cholecalciferol may serve as a robust predictor of
early-stage lung cancer. These findings provide new predictive
biomarkers for early-stage lung cancer manifested by SPN, which is
clinically important and requires further study of the potential
mechanisms of action and function of the targets.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1513270"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1513270</a></p>
<hr />
<h2
id="genomic-analysis-of-prophages-in-44-clinical-strains-of-pseudomonas-aeruginosa-isolated-in-saudi-arabia">Genomic
analysis of prophages in 44 clinical strains of Pseudomonas aeruginosa
isolated in Saudi Arabia</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Prophages are bacteriophages that integrate their genomes into the
bacterial chromosome. This research aimed to analyze and characterize
prophages integrated into 44 Pseudomonas aeruginosa strains isolated
from tertiary hospitals in Saudi Arabia. A total of 97 intact prophages
were identified among clinical strains, with 16 prophages found present
in more than one strain simultaneously. All prophages were found to have
lengths ranging from 7.7 kb to 74.1 kb, and their GC content was found
to be between 49.91% and 64.9%. Our findings show that prophages are
present in the majority of the isolated P. aeruginosa strains (41 out of
44). Additionally, several proteins related to viral defense
(toxin/antitoxin modules and proteins against restriction-modification
enzymes) were identified, supporting the idea that prophages influence
bacterial pathogenesis and anti-phage defenses.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563781"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563781</a></p>
<hr />
<h2
id="aneuploidy-enables-adaptation-to-brefeldin-a-in-candida-albicans">Aneuploidy
enables adaptation to brefeldin A in Candida albicans</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Candida albicans is the most prevalent opportunistic fungal pathogen.
Both in vivo and in vitro studies have demonstrated that genome
plasticity is a hallmark of C. albicans. While aneuploidy formation is a
well-documented adaptive mechanism under various stress conditions, the
response to brefeldin A—a compound that induces endoplasmic reticulum
stress—remains poorly understood. In this study, we demonstrate that C.
albicans adapts to subinhibitory and inhibitory concentrations of
brefeldin A, primarily through the formation of chromosome 3 trisomy.
These aneuploid strains were found to be unstable, reverting to euploidy
in the absence of stress, accompanied by a loss of brefeldin A
tolerance. We identified at least two genes on chromosome 3, SEC7 and
CDR1, that contribute to this adaptive response. Notably, higher
concentrations of brefeldin A selected for strains with increasingly
complex aneuploidies. Our findings underscore the remarkable genomic
plasticity of C. albicans and reveal aneuploidy as a reversible
mechanism for adapting to brefeldin A stress. This study provides new
insights into the role of aneuploidy in fungal adaptation and offers
potential implications for understanding drug resistance mechanisms in
pathogenic fungi.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562726"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562726</a></p>
<hr />
<h2
id="prevalence-of-chlamydia-trachomatis-neisseria-gonorrhoeae-and-ureaplasma-urealyticum-infections-in-males-and-females-of-childbearing-age-in-chengdu-china">Prevalence
of Chlamydia trachomatis, Neisseria gonorrhoeae, and Ureaplasma
urealyticum infections in males and females of childbearing age in
Chengdu, China</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSexually transmitted infections (STIs) are a global public
health issue, due to their high prevalence and potential impact on
pregnancy outcome and fetal health. The objective of this study is to
investigate the prevalence of STI-causative pathogens including
Chlamydia trachomatis (CT), Neisseria gonorrhoeae (NG) and Ureaplasma
urealyticum (UU) as well as the epidemiological characteristics of STIs
among males and females of childbearing age in Chengdu.MethodsThis
retrospective cross-sectional study involved 15,055 participants of
childbearing age including 7,235 males and 7,820 females. All specimens
of participants were tested for CT, NG, and UU by nucleic acid
amplification tests (NAATs) methods.Results27.80% of the participants
were infected with at least one of the three pathogens, with
significantly higher overall prevalence in females (45.22%) than males
(8.98%, p&lt;0.001). Multivariate logistic regression analysis
demonstrated that gender was independently associated with both CT
positivity (females vs males, OR 2.276, 95% CI 1.724-3.005, p&lt;0.001)
and UU positivity (females vs males, OR 8.079, 95% CI 7.183-9.086,
p&lt;0.001). Single infections predominated in both males and females
(males: 98.15%; females: 93.16%), while mixed infections were more
frequent in females (6.84%) than in males (1.85%). CT prevalence was
highest in both males and females aged 18-24, followed by aged 25–30 and
31-35. Among aged 18-24, the prevalence of UU in males and NG and UU in
females were also higher. Compared to other age groups, mixed infections
(CT+NG, CT+UU, NG+UU, CT+NG+UU) were also highest in females aged 18-24.
Compared to other clinical diagnostic groups, The prevalence of CT, NG
and mixed infection of CT+NG was highest in both males and females in
the urogenital inflammation group (p&lt;0.05).ConclusionsAmong the
population of childbearing age in Chengdu, China, the prevalence among
females was significantly higher than that among males. Single
infections predominated in both males and females, while mixed
infections occurred more frequently in females. STIs were more prevalent
in sexually active young people aged 18-35, especially in the 18–24 age
group. CT and NG infections in both males and females may cause
urogenital inflammation, and mixed infections of CT+NG further elevate
the risk of inflammatory responses.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1566163"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1566163</a></p>
<hr />
<h2
id="advances-in-whole-genome-sequencing-for-foodborne-pathogens-implications-for-clinical-infectious-disease-surveillance-and-public-health">Advances
in whole genome sequencing for foodborne pathogens: implications for
clinical infectious disease surveillance and public health</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Foodborne outbreaks affecting millions of people worldwide are a
significant and growing global health threat, exacerbated by the
emergence of new and increasingly virulent foodborne pathogens.
Traditional methods of detecting these outbreaks, including
culture-based techniques, serotyping and molecular methods such as
real-time PCR, are still widely used. However, these approaches often
lack the precision and resolution required to definitively trace the
source of an outbreak and distinguish between closely related strains of
pathogens. Whole genome sequencing (WGS) has emerged as a revolutionary
tool in outbreak investigations, providing high-resolution,
comprehensive genetic data that allows accurate species identification
and strain differentiation. WGS also facilitates the detection of
virulence and antimicrobial resistance (AMR) genes, providing critical
insight into the potential pathogenicity, treatment/control options and
risks of spreading foodborne pathogens. This capability enhances
outbreak surveillance, source tracing and risk assessment, making WGS an
increasingly integrated component of public health surveillance systems.
Despite its advantages, the widespread implementation of WGS faces
several pressing challenges, including high sequencing costs, the need
for specialized bioinformatics expertise, limited computational
infrastructure in resource-constrained settings, and the standardization
of data-sharing frameworks across regulatory and public health agencies.
Addressing these barriers is crucial to maximizing the impact of WGS on
foodborne disease surveillance. Even so, WGS is emerging as a vital tool
in food safety and public health, and its potential to become the gold
standard in outbreak detection has been recognized by public health
authorities in the USA, the European Union, Australia and China, for
example. This review highlights the role of WGS in foodborne outbreak
investigations, its implementation challenges, and its impact on public
health surveillance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593219"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1593219</a></p>
<hr />
<h2
id="clinical-application-of-metagenomic-next-generation-sequencing-in-etiologic-diagnosis-of-severe-pneumonia-in-adults">Clinical
application of metagenomic next-generation sequencing in etiologic
diagnosis of severe pneumonia in adults</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveTo analyze the clinical characteristics and risk factors for
death of severe pneumonia (SP) in adults and explore the application
value of metagenomic next-generation sequencing in the detection of
pathogens.MethodsA total of 132 adult patients with SP admitted from May
2021 to October 2023 were selected. Data on gender, age, smoking,
underlying diseases, laboratory tests and prognosis were collected. BALF
samples were sent for mNGS, smear-stained microscopy and culture.
Meanwhile, conventional methods were used for pathogen detection of
blood, urine and throat swab specimens. The detection efficiencies of
different methods were compared and the associated pathogen profiles
were analyzed.ResultsAmong the 132 patients, there were 92 males and 40
females, with a total of 52 deaths. Age≥65 years, heart failure, renal
insufficiency, positive of COVID-19, use of vasoactive drugs, use of
mechanical ventilation and use of CRRT were statistically different
between the survivors and non-survivors. Heart failure (OR=4.751) and
use of mechanical ventilation (OR=11.914) were risk factors of SP
mortality. The bacteria detected were mainly Klebsiella pneumoniae,
Acinetobacter baumannii and Pseudomonas aeruginosa. The fungi detected
were mainly Candida and Aspergillus. The viruses detected were mainly
COVID-19 and influenza virus. The positive rate of mNGS was higher than
conventional methods in both bacteria, fungus and virus (82.58% vs
63.64%, 50.76% vs 37.88% and 67.42% vs 37.88%, respectively)
(P&lt;0.05). The sensitivity and accuracy of mNGS in bacterial detection
were significantly higher than traditional methods (P&lt;0.05). Compared
to culture, mNGS detected more Staphylococcus aureus, Streptococcus
pneumoniae, Haemophilus influenzae and Escherichia coli, and had a
significant advantage in the detection of Mycobacterium tuberculosis
complex, Nontuberculous mycobacterial, Legionella pneumophila, Chlamydia
psittaci, Pneumocystis jirovecii and Aspergillus. Moreover, mNGS can
better indicate mixed infections of bacteria, viruses, or
fungi.ConclusionElderly people with chronic diseases were the main group
of severe pneumonia in adults. The pathogenic microorganisms that caused
SP are complex, and mixed infection is common. mNGS enhanced the
effectiveness of pathogen detection, makes up for the shortcomings of
conventional methods, especially in identifying unexpected pathogens,
and provides a new means for early targeted anti-infection
treatment.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1561468"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1561468</a></p>
<hr />
<h2
id="clinical-evaluation-of-personalized-helicobacter-pylori-treatment-guided-by-pcr-detection-from-fecal-samples-a-real-world-study">Clinical
evaluation of personalized Helicobacter pylori treatment guided by PCR
detection from fecal samples: a real-world study</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundGrowing antibiotic resistance in Helicobacter pylori (H.
pylori) diminishes eradication therapy effectiveness, emphasizing the
need for accurate, non-invasive diagnostic techniques. This study aims
to assess the clinical utility of PCR analysis of fecal samples for
detecting antibiotic resistance in guiding personalized treatment for H.
pylori infection.MethodsA retrospective, observational study was
conducted across 13 hospitals within Jiangsu Province. Fecal samples
were analyzed using fluorescence PCR for the 23S rRNA and gyrA genes,
indicating clarithromycin and levofloxacin resistance. Then,
individualized eradication recommendations were proposed for the H.
pylori-positive patients. A follow-up was performed one year later to
evaluate the eradication outcomes in a routine medical environment, with
participants having provided informed consent.ResultsA total of 387
participants completed the eradication treatment, with an overall
success rate of 91.0% (352/387, 95% CI: 87.6%-93.6%). Among them, 310
individuals received a 14-day course of bismuth quadruple therapy (BQT),
achieving an eradication rate of 90.0% (279/310, 95% CI: 86.1%-93.1%).
77 participants were treated with high-dose dual therapy (HDDT) for the
same duration, resulting in a slightly higher eradication rate of 94.8%
(73/77, 95% CI: 87.2%-98.6%), although this difference was not
statistically significant (P=0.188). No significant differences in
eradication rates were observed among various BQT antibiotic
combinations (P=0.208). The eradication rates for HDDT, based on either
vonoprazan or esomeprazole, were 96.8% (61/63, 95% CI: 89.0%-99.6%) and
85.7% (12/14, 95% CI: 57.2%-98.2%), respectively, without a significant
difference (P=0.304).ConclusionPCR detection from fecal samples
targeting the resistance genes of H. pylori is effective in guiding
personalized treatments, highlighting its clinical utility and potential
for broader application.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1519804"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1519804</a></p>
<hr />
<h2
id="unlocking-the-power-of-antimicrobial-peptides-advances-in-production-optimization-and-therapeutics">Unlocking
the power of antimicrobial peptides: advances in production,
optimization, and therapeutics</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Antimicrobial peptides (AMPs) are critical effectors of innate
immunity, presenting a compelling alternative to conventional
antibiotics amidst escalating antimicrobial resistance. Their
broad-spectrum efficacy and inherent low resistance development are
countered by production challenges, including limited yields and
proteolytic degradation, which restrict their clinical translation.
While chemical synthesis offers precise structural control, it is often
prohibitively expensive and complex for large-scale production.
Heterologous expression systems provide a scalable, cost-effective
platform, but necessitate optimization. This review comprehensively
examines established and emerging AMP production strategies,
encompassing fusion protein technologies, molecular engineering
approaches, rational peptide design, and post-translational
modifications, with an emphasis on maximizing yield, bioactivity,
stability, and safety. Furthermore, we underscore the transformative
role of artificial intelligence, particularly machine learning
algorithms, in accelerating AMP discovery and optimization, thereby
propelling their expanded therapeutic application and contributing to
the global fight against drug-resistant infections.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1528583"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1528583</a></p>
<hr />
<h2
id="progress-on-the-mechanism-of-intestinal-microbiota-against-colorectal-cancer">Progress
on the mechanism of intestinal microbiota against colorectal cancer</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The intestinal microbiota plays a crucial role in the occurrence and
development of colorectal cancer, and its anti - colorectal cancer
mechanism has become a research hotspot. This article comprehensively
expounds on the molecular mechanisms of the intestinal microbiota in
anti - colorectal cancer, including aspects such as immune regulation,
activation of carcinogenic signaling pathways (it should be noted that
it is more reasonable to be “inhibition of carcinogenic signaling
pathways”), metabolite - mediated effects, and maintenance of intestinal
barrier function. At the same time, it explores the roles and potential
mechanisms of intervention methods such as probiotic supplementation
therapy, immunotherapy, and fecal microbiota transplantation. In
addition, it analyzes the impact of the intestinal flora on the
therapeutic efficacy of colorectal cancer. The existing research results
are summarized, and the future research directions are prospected, with
the aim of providing new theoretical bases and treatment ideas for the
prevention and treatment of colorectal cancer.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565103"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565103</a></p>
<hr />
<h2
id="extracellular-vesicles-in-arbovirus-infections-from-basic-biology-to-potential-clinical-applications">Extracellular
vesicles in arbovirus infections: from basic biology to potential
clinical applications</h2>
<p>date: 2025-04-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Arthropod-borne viruses, or arboviruses, are currently considered a
global health threat responsible for potentially severe human diseases.
The increased population density, changes in land use and climate change
are some of the factors that are contributing to the spread of these
infections over the last years. The pathogenesis of these diseases and
the mechanisms of interaction with the host, especially those leading to
the development of severe forms, are yet to be fully understood. In
recent years extracellular vesicles (EVs) have emerged as important
players in the inter-cellular and host-pathogen interaction arising a
lot of interest also in the field of vector-borne viruses. In this
context, EVs seem to play a dual role, by either promoting, thus
facilitating, or preventing infection. Many studies are showing how
viruses can hijack the vesiculation machinery to escape the host immune
response and exploit EVs to sustain their replication and propagation,
even though EVs shed by immune cells seem essential to promote antiviral
responses. In this manuscript we reviewed the current knowledge
regarding the association between EVs and vector-borne viruses, paying
particular attention to their possible role in disease transmission and
dissemination, as well as to their potential as novel tools for clinical
applications, spanning from biomarkers of clinical utility to novel
therapeutic options.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1558520"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1558520</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
