---
title: health
updated: 2025-04-07 06:08:23
---

# health

(date: 2025-04-07 06:08:23)

---

## Commentary: Propionimicrobium lymphophilum in urine of children with monosymptomatic nocturnal enuresis

date: 2025-04-07, from: Frontiers in Cellular and Infection Microbiology

 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1553911>

---

## Changes in children respiratory infections pre and post COVID-19 pandemic

date: 2025-04-07, from: Frontiers in Cellular and Infection Microbiology

BackgroundNon-pharmaceutical interventions (NPIs) implemented during the COVID-19 pandemic had a significant impact on the prevalence of various acute respiratory infections (ARIs) pathogens.MethodsWe collected 337,310 real-time PCR results for 13 pathogens from clinical samples between January 2018 and January 2024 to assess the changes of ARIs among children before and after the COVID-19 pandemic.ResultsA variety of ARIs pathogens, including Influenza A (Flu A), Influenza B (Flu B), Adenovirus (ADV), Rhinovirus (RhV), and Respiratory Syncytial Virus (RSV), as well as co-infecting bacterial such as Klebsiella pneumoniae (KPN), Pseudomonas aeruginosa (PAE), Streptococcus pneumoniae (SP), Haemophilus influenzae (HI), and Listeria monocytogenes (LP), reached a peak positive rate at the age of 3. The susceptible age of Mycoplasma pneumoniae (MP) was from 3 to 7 years old. Compared to the pre-COVID pandemic period, the positive rates of Flu A, MP, ADV, SP, HI, Staphylococcus aureus (SA) and KPN decreased during the COVID-19 pandemic. And the positive rates of Flu B and PAE increased. Compared to the period during the COVID-19 pandemic, the positive rates of Flu A, ADV, RSV, RhV, SP, HI, KPN, PAE and SA were increased after the pandemic. Conversely, the positive rates of MP, Flu B, and Parainfluenza virus (PIV) decreased.ConclusionsThe implementation of NPIs interrupted the circulation of ARIs pathogens. However, release of NPIs and the reduced baseline of population immunity, may contribute to a resurgence of ARIs pathogens among children. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1549497>

---

## Oral Fusobacterium nucleatum exacerbates ulcerative colitis via the oral-gut axis: mechanisms and therapeutic implications

date: 2025-04-07, from: Frontiers in Cellular and Infection Microbiology

BackgroundFusobacterium nucleatum (F. nucleatum) is an anaerobic bacterium known for its association with periodontal disease and oral infections. It has been implicated in the development of gastrointestinal diseases such as inflammatory bowel disease and colorectal cancer. Ulcerative colitis (UC), which is characterized by chronic inflammation of the colon, is a condition of unknown etiology with a rising incidence rate, significantly affecting the quality of life for patients. The increased intestinal permeability during UC may facilitate the adherence or invasion of F. nucleatum into the damaged intestinal barrier, leading to exacerbated inflammation.MethodsThis article introduces the concept of the oral-gut axis, reviewing existing literature to analyze the role of F. nucleatum in the pathogenesis of UC and exploring its potential pathogenic mechanisms. It also summarizes the latest advances in treating patients with UC who have F. nucleatum and looks forward to prospective therapeutic strategies and the translational prospects of F. nucleatum within the oral-gut axis.ResultsF. nucleatum may be a key player in the pathogenesis of UC, likely due to its invasiveness during periods of increased intestinal permeability. The paper also discusses innovative approaches for the prevention and management of UC exacerbated by F. nucleatum, paving the way for more effective treatment of UC.ConclusionThe review offers new insights into the complex relationship between the oral microbiome and intestinal diseases, enhancing our understanding of their dynamic interactions. There is a paucity of literature on therapeutic approaches, indicating a need for further clinical research. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1564169>

---

## Alcohol exposure alters the diversity and composition of oral microbiome

date: 2025-04-07, from: Frontiers in Cellular and Infection Microbiology

IntroductionAlcohol exposure has been shown to have complex, and sometimes paradoxical, associations with various serious diseases. Currently, there is no knowledge about the effects of alcohol exposure on the dynamics of oral microbial communities.ObjectiveThe study aims to investigate the effects of chronic alcohol consumption on the diversity and composition dynamics of the rat oral microbiota using 16S rRNA gene amplicon sequencing.MethodsIn our study, there were 2 groups, including a control group (C group) and an alcohol group (A group), with 10 rats in every group. For ten weeks, rats in the A group were treated with alcohol intragastrically every day, whereas rats in the C group got water. After 10 weeks, serum levels of alanine aminotransferase (ALT) and aspartate aminotransferase (AST) were measured. Oral swabs were taken from both groups, and total DNA was extracted for high-throughput sequencing of the 16S rRNA gene.ResultsAccording to the results obtained from our study, significant differences were observed in the relative abundances of microbial communities. Alpha diversity measures were statistically significantly higher (P < 0.05) in the A group compared to the C group. At the genus level, alcohol exposure altered the relative abundance of several microbes, including increased relative abundance of unidentified_Chloroplast, Acinetobacter, Vibrio, Romboutsia, Pseudoalteromonas, Aeromonas, Ralstonia, Turicibacter, Shewanella, and Bacteroides. Conversely, Haemophilus and Streptococcus were significantly less abundant in the A group.ConclusionAlcohol exposure was associated with the diversity and composition of the oral microbiome. These findings contribute to our understanding of the potential role of oral bacteria in alcohol-related oral and systemic diseases, providing foundational work for future prevention and intervention studies. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1561055>

---

## Targeted and untargeted metabolomics uncovering the effects of Jiawei Ermiao Granule on patients with persistent HPV infection

date: 2025-04-07, from: Frontiers in Cellular and Infection Microbiology

Jiawei Ermiao Granule (JWEMG) is a traditional Chinese herbal formulation widely used in China for the treatment of human papilloma virus (HPV) infections. However, the mechanisms underlying their efficacy in clearing HPV infections remain unclear. This study aimed to elucidate the mechanisms by which JWEMG clears persistent HPV infections from a metabolomics perspective using modern analytical techniques. Untargeted and targeted metabolomics analyses were performed on vaginal lavage samples from 33 patients using liquid chromatograph mass spectrometer (LC-MS) and hydrophilic interaction liquid chromatography-mass spectrometry (HILIC-MS). Untargeted metabolomics identified 47 potential biomarkers through volcano plot analysis, among which 30 exhibited a reversal trend following JWEMG intervention. Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis suggested that JWEMG may exert therapeutic effects on patients with persistent HPV infections via pathways related to starch and sucrose metabolism, galactose metabolism, fructose and mannose metabolism, and amino sugar and nucleotide sugar metabolism. Targeted metabolomics revealed a significant increase in tyrosine levels in the vaginal/cervical microenvironment following JWEMG treatment. By integrating targeted and untargeted metabolomics, this study provides a comprehensive exploration of the holistic effects of JWEMG on HPV-infected patients, addressing the challenges of scientifically explaining the pharmacological mechanisms of multi-component, multi-target traditional Chinese medicines. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1550908>

---

## Resistance phenotypes and genomic features of Mycobacterium seoulense isolates

date: 2025-04-07, from: Frontiers in Cellular and Infection Microbiology

BackgroundMycobacterium seoulense (M. seoulense) is an emerging pathogen increasingly associated with infections; however, its resistance phenotypes and genomic characteristics remain largely unknown.MethodsSeven M. seoulense isolates were collected from clinical samples. Drug susceptibility testing was conducted using Sensititre™ SLOMYCO2 susceptibility plates. Whole genome sequencing and supporting bioinformatics analyses were performed to analyze the genomic features.ResultsAll M. seoulense isolates (n=7) exhibited growth on 7H10 agar medium containing thiophenecarboxylic acid hydrazide or p-Nitrobenzoic acid, with marked diversity in growth rates in liquid culture. All strains exhibited high minimum inhibitor concentrations (MICs) for minocycline (>8 μg/mL), doxycycline (>8 μg/mL), and amikacin (16-32 μg/mL). The MICs for linezolid, rifabutin, moxifloxacin, ciprofloxacin, streptomycin, clarithromycin, and rifampicin varied among the isolates. High levels of genomic diversity were noted among these strains concerning genome-called single nucleotide polymorphisms and average nucleotide identity. In total, 4,282 genes were shared across all genomes, while 315 unique genes were restricted to one strain. Comparative genomic analysis identified two unique virulence genes encoding a catalase enzyme and a protein involved in capsule biosynthesis and transport. Additionally, all M. seoulense strains demonstrated the ability to survive within macrophages.ConclusionThe clinical M. seoulense isolates analyzed in this study exhibited varying levels of antibiotic susceptibility, suggesting the potential need for susceptibility testing to guide clinical treatment. Genomic features of these isolates indicated that they are likely pathogenic non-tuberculous mycobacterium, highlighting a need for closer epidemiological monitoring. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1553591>

---

## Identification and characterization of bile microbiota in patients with biliary obstructive diseases using next-generation sequencing of 16S rRNA and ITS

date: 2025-04-07, from: Frontiers in Cellular and Infection Microbiology

BackgroundComparative studies of the bile microbiota in different biliary obstructive infections remain limited. This study aims to characterize bile microbiota and investigate differences in microbial profiles across various biliary obstructive diseases.MethodsThis study included patients with biliary obstructive diseases admitted to Jinling Hospital and Suqian First Hospital. The cohort consisted of individuals with benign biliary disorders, malignant biliary obstruction, and biliary obstruction secondary to severe acute pancreatitis (SAP) or intestinal fistulas. A total of 133 bile samples were collected from 118 patients and analyzed using next-generation sequencing (NGS) targeting the bacterial 16S rRNA gene and the fungal internal transcribed spacer (ITS) gene. Clinical data, including routine culture results, were extracted from electronic medical records.ResultsNGS targeting the 16S rRNA and ITS revealed a positive rate of 68.42% for bile samples, which was higher than the culture positivity rate of 60.15%, indicating a significant difference (Chi-square test, p < 0.05). The predominant bacteria identified in the bile samples through NGS were Klebsiella pneumoniae, Acinetobacter baumannii, and Escherichia coli. Bacterial species varied among benign biliary diseases, malignant obstructive diseases, and biliary obstruction caused by SAP or intestinal fistulas. Fungi were detected in 7.52% (10/133) of the samples, with 4 samples obtained from patients with biliary obstructive diseases due to SAP. Microbial diversity and clustering analysis showed no significant differences among various biliary disorders. Based on the culture results, the sensitivity and specificity of NGS were 81.82% and 69.64%, respectively.ConclusionThe composition of bile microbes may be related to the etiology of biliary obstruction. Klebsiella pneumoniae, Acinetobacter baumannii, and Escherichia coli are the predominant bacteria found in bile. NGS can be effectively applied for the identification and characterization of bile microbes associated with various biliary obstruction diseases. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1575824>

