---
title: health
updated: 2025-03-31 06:08:01
---

# health

(date: 2025-03-31 06:08:01)

---

## Emergence and evolution of rare ST592 blaNDM-1-positive carbapenem-resistant hypervirulent Klebsiella pneumoniae in China

date: 2025-03-31, from: Frontiers in Cellular and Infection Microbiology

ObjectivesThis study aimed to characterize the genomes of two rare ST592 Klebsiella pneumoniae isolates and to explore their evolution into carbapenem-resistant hypervirulent K. pneumoniae (CR-hvKp).MethodsThe minimum inhibitory concentrations (MICs) were determined using a VITEK 2 compact system. Conjugation experiments were conducted using film matings. Whole-genome sequencing (WGS) was performed using the Illumina and Nanopore platforms. The antimicrobial resistance determinants were identified using the ABRicate program in the ResFinder database. Insertion sequences were identified using ISFinder and the bacterial virulence factors identified using the Virulence Factor Database (VFDB). The K and O loci were examined using Kleborate. Multilocus sequence typing (MLST) and replicon type identification were performed by the Center for Genomic Epidemiology. Conjugation-related elements were predicted using oriTfinder. The plasmid structure was visualized using Circos, and a possible evolutionary model was constructed using BioRender.ResultsIsolates KPZM6 and KPZM16 were identified as ST592 and KL57, respectively, and were collected from the same department. The antimicrobial susceptibility testing data revealed that KPZM16 possesses an extensively drug-resistant (XDR) profile, whereas KPZM6 is a susceptible K. pneumoniae. The hybrid assembly showed that both KPZM6 and KPZM16 have one pLVPK-like virulence plasmid carrying the rmpA, rmpA2, and iucABCD-iutA gene clusters. However, strain KPZM16 harbors one IncN plasmid carrying the carbapenem resistance genes blaNDM-1, dfrA14, and qnrS1. The results of the conjugation experiments demonstrated that the plasmid could be transferred to the recipient strain. It is possible that the NDM-1-producing plasmid was transferred from KPZM6 to KPZM16 via conjugation, leading to the formation of CR-hvKp.ConclusionsThis is the first study in which complete genomic characterization of the rare NDM-1-producing ST592 K. pneumoniae clinical isolate was performed. This study provides a possible evolutionary hypothesis for the formation of CR-hvKp via conjugation. Early detection is recommended to avoid the extensive spread of this clone. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565980>

---

## Immunogenicity of PvVir14-derived peptides to improve the serological diagnosis of Plasmodium vivax infection

date: 2025-03-31, from: Frontiers in Cellular and Infection Microbiology

IntroductionMalaria, caused by Plasmodium vivax, remains a major global health problem, particularly in tropical and subtropical regions. This study aimed to investigate the potential of PvVir14-derived peptides to improve diagnostic accuracy for P. vivax infections.MethodsThe reactivity of specific PvVir14-derived peptides against sera from acutely infected individuals from endemic and non-endemic areas experiencing a P. vivax malaria outbreak was assessed, as well as their sensitivity, specificity, and immunodominance.ResultsAmong the eight tested peptides (H08, K09, I03, G08, F08, H04, L06, and N04), H08, G08, and L06 showed significantly higher IgG reactivity in sera from individuals living in endemic areas, mainly within those with multiple malaria episodes. After a P. vivax outbreak in a non-endemic area, H08 and G08 had the highest IgM frequencies, however, sensitivity and specificity analyses indicated that L06 was the most effective peptide for identifying infected individuals. Depletion ELISA confirmed the immunodominance of L06, G08, and H08 and showed a significant reduction in IgG reactivity to PvVir14. The peptides L06, G08, and H08, showed high sensitivity and specificity as diagnostic tools for P. vivax malaria.ConclusionsThese peptides can improve diagnostic accuracy, especially in endemic areas, providing better support to malaria control and management programs. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1484863>

---

## Partial normalization of microbiota dysbiosis in condyloma acuminatum patients following treatment

date: 2025-03-31, from: Frontiers in Cellular and Infection Microbiology

IntroductionCondyloma acuminatum (CA) is the most common sexually transmitted disease and the presence of microbiota dysbiosis has been observed to promote the progress of the disease. However, the explicit characteristics of microbiota dysbiosis in CA patients have not been well elucidated yet.MethodsWe recruited 40 CA patients who received QYXJ (an in-hospital prescription that has been used to treat CA for many years) treatment and 40 healthy controls (HC) in the current study. Before and after two weeks QYXJ administration, the skin microbiome of each patient was assessed using 16S rRNA gene sequencing. ResultsHere, we found increased relative abundances of Staphylococcus and Lactobacillus, whereas a decreased Escherichia in CA patients relative to healthy controls (HC). Moreover, we also observed significant alpha and beta diversity differences between the CA and HC groups, and QYXJ treatment effectivity attenuated these alterations of genus level and microbial diversity in patients with CA. Importantly, further microbial interaction and function analyses revealed the significantly enriched relative abundance of Caldivirga and Streptococcus in microbial community, decreased complexity of microbial interactions and downregulated metabolic pathways in CA patients, including membrane transport, lipid metabolism and carbohydrate metabolism. Remarkably, QYXJ administration partially restored these microbiota dysbiosis, which subsequently shifts microbiomes of patients with CA towards healthy-like microbiota. ConclusionThis study further confirmed the changes of skin microbiome in CA pathogenesis and firstly revealed the protective effects of QYXJ in microbiota dysbiosis resolution, suggesting its potential role as a novel method for CA treatment. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1558469>

---

## Editorial: Advances in the microbiome, immunity and cancer interplay

date: 2025-03-31, from: Frontiers in Cellular and Infection Microbiology

 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1568679>

