<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2025-03-15 06:07:29)</p>
<hr />
<h2
id="polyclonal-carbapenemase-producing-escherichia-coli-in-northern-italy-the-emergence-of-ndm-7">Polyclonal
carbapenemase-producing Escherichia coli in Northern Italy: the
emergence of NDM-7</h2>
<p>date: 2025-03-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The spread of extended-spectrum beta-lactamase (ESBL)- and
carbapenemase-producing Escherichia coli clones in humans, animals, and
the environment is of great concern worldwide. In this study, we
characterized four carbapenemase-producing E. coli (CP-Ec) isolated from
human samples. Two isolates of ST401, rarely associated with
carbapenemase and/or ESBL resistance genes, harbored blaKPC-3 and
blaVIM-1 genes, respectively, and were genetically distant from each
other. One CP-Ec isolate belonging to ST355, typically found in poultry
and environmental sources and not associated with carbapenemases, was
blaKPC-3 positive and showed a wide range of virulence genes. The last
CP-Ec strain belonged to ST3564, previously described in livestock with
a large virulome but no carbapenemase. The CP-Ec ST3564 isolate
co-harbored blaVIM-1 and blaNDM-7 genes, which, to our knowledge, have
not been previously reported in Italy. These results emphasize the
crucial role of a genomic-based surveillance program to intercept the
emergence of critical E. coli clones.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1519827"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1519827</a></p>
<hr />
<h2
id="cyclic-di-amp-alleviates-periodontitis-by-activating-pi3kaktnrf2-pathways">Cyclic
di-AMP alleviates periodontitis by activating PI3K/Akt/Nrf2
pathways</h2>
<p>date: 2025-03-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Emerging research demonstrates the regulatory effects of c-di-AMP, a
bacterial-derived small molecule secondary messenger, on host immune
responses and promoting resistance against infection-related diseases.
This study aims to elucidate the role of c-di-AMP in the occurrence and
development of periodontitis. Using model of ligation-induced
periodontitis, we observed that c-di-AMP effectively alleviated alveolar
bone resorption. Transcriptomic sequencing in mice gingival tissues
demonstrated that treatment with c-di-AMP led to a significant
upregulation of the PI3K/Akt signaling pathway and its key components,
including Akt3. Concurrently, we observed an upregulation of the
cGMP/PKG signaling pathway. To validate our findings, we treated
gingival epithelial cells with c-di-AMP and confirmed the activation of
the PI3K/Akt pathway by c-di-AMP in gingival epithelial cells. Under
LPS-induced inflammation, c-di-AMP significantly suppressed the release
of inflammatory factors (such as IL-6 and TNF-α) from gingival
epithelial cells. Moreover, key components of the PI3K/Akt pathway,
including Akt, and downstream inflammation regulatory gene Nrf2, were
upregulated, which were also confirmed at the protein level.
Collectively, this study demonstrates that c-di-AMP definitely plays a
role in alleviating periodontitis. Our findings highlight the mechanisms
by which c-di-AMP modulates periodontitis, including activating the
PI3K/Akt pathway and potentially involving the cGMP/PKG pathway,
ultimately contributing to improved immune defense and maintenance of
bone homeostasis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560155"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560155</a></p>
<hr />
<h2
id="intestinal-overexpression-of-pla2g10-alters-the-composition-diversity-and-function-of-gut-microbiota-in-mice">Intestinal
overexpression of Pla2g10 alters the composition, diversity and function
of gut microbiota in mice</h2>
<p>date: 2025-03-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The intestinal microbiota is important for the health of the host and
recent studies have shown that some genes of the host regulated the
composition of the intestinal microbiota. Group 10 phospholipase A2
(PLA2G10) is a member of the lipolytic enzyme family PLA2, which
hydrolyze the ester bond at the sn-2 position of phospholipids to
produce free fatty acids and lysophospholipids. PLA2G10 is secreted into
the intestinal lumen, but its impact on the gut microbiota remains
unclear. In this study, we generated intestine-specific Pla2g10 knock-in
mice, and used 16S RNA sequencing to compare their gut microbiota with
that of their wild-type (WT) littermates. Results showed that
gut-specific Pla2g10 knock-in induced both PLA2G10 mRNA and protein
levels in the colon. Moreover, intestinal Pla2g10 overexpression reduced
the α-diversity of the gut microbiota relative to that of WT mice. The
abundance of Bacteroidetes was lower in the Pla2g10 knock-in mice than
that in the control mice, while the ratio of Firmicutes/Bacteroidetes
was higher. Furthermore, the abundance of the genus Allobaculum was
reduced, whereas the abundance of beneficial bacteria genera, including
Enterorhabdus, Dubosiella, and Lactobacillus, was increased by host
intestinal Pla2g10 overexpression. In summary, intestinal Pla2g10
overexpression increased the proportions of beneficial bacterial in the
colonic chyme of mice, providing a potential therapeutic target for
future improvement of the gut microbiota.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535204"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535204</a></p>
<hr />
<h2
id="an-explainable-web-application-based-on-machine-learning-for-predicting-fragility-fracture-in-people-living-with-hiv-data-from-beijing-ditan-hospital-china">An
explainable web application based on machine learning for predicting
fragility fracture in people living with HIV: data from Beijing Ditan
Hospital, China</h2>
<p>date: 2025-03-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>PurposeThis study aimed to develop and validate a novel web-based
calculator using machine learning algorithms to predict fragility
fracture risk in People living with HIV (PLWH), who face increased
morbidity and mortality from such fractures.MethodWe retrospectively
analyzed clinical data from Beijing Ditan Hospital orthopedic department
between 2015 and September 2023. The dataset included 1045 patients
(2015-2021) for training and 450 patients (2021-September 2023) for
external testing. Feature selection was performed using multivariable
logistic regression, LASSO, Boruta, and RFE-RF. Six machine learning
models (logistic regression, decision trees, SVM, KNN, random forest,
and XGBoost) were trained with 10-fold cross-validation and
hyperparameter tuning. Model performance was assessed with ROC curves,
Decision Curve Analysis, and other metrics. The optimal model was
integrated into an online risk assessment calculator.ResultsThe XGBoost
model showed the highest predictive performance, with key features
including age, smoking, fall history, TDF use, HIV viral load, vitamin
D, hemoglobin, albumin, CD4 count, and lumbar spine BMD. It achieved an
ROC-AUC of 0.984 (95% CI: 0.977-0.99) in the training set and 0.979 (95%
CI: 0.965-0.992) in the external test set. Decision Curve Analysis
indicated clinical utility across various threshold probabilities, with
calibration curves showing high concordance between predicted and
observed risks. SHAP values explained individual risk profiles. The
XGBoostpowered web calculator
(https://sydtliubo.shinyapps.io/cls2shiny/) enables clinicians and
patients to assess fragility fracture risk in PLWH.ConclusionWe
developed a web-based risk assessment tool using the XGBoost algorithm
for predicting fragility fractures in HIV-positive patients. This tool,
with its high accuracy and interpretability, aids in fracture risk
stratification and management, potentially reducing the burden of
fragility fractures in the HIV population.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1461740"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1461740</a></p>
<hr />
<h2
id="clinical-application-of-combined-crp-and-pct-detection-in-diagnosing-and-prognosing-fracture-related-infections">Clinical
application of combined CRP and PCT detection in diagnosing and
prognosing fracture-related infections</h2>
<p>date: 2025-03-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveTo investigate the clinical significance of combined
procalcitonin (PCT) and C-reactive protein (CRP) testing in the
diagnosis of fracture-related infections (FRIs).MethodsA retrospective
analysis was performed on 252 patients with bone infections admitted
between January 2018 and September 2024. CRP and PCT levels were
measured in all patients both at diagnosis and postoperatively. The
sensitivity, specificity, positive predictive value (PPV), and negative
predictive value (NPV) of combined CRP and PCT for diagnosing FRIs were
calculated. Based on clinical follow-up data, patients were divided into
low-level and high-level groups according to the changes in CRP and PCT
levels, and clinical outcomes, including length of hospital stay and
infection control, were analyzed.ResultsThe postoperative levels of CRP
and PCT in the infection group were significantly higher than in the
control group (P &lt; 0.05). The sensitivity, specificity, PPV, and NPV
of combined CRP and PCT detection were 90.48%, 96.83%, 96.61%, and
91.04%, respectively. Patients in the high CRP and PCT group had a
significantly longer hospital stay compared to the low-level group, and
infection control rates were lower. Combined CRP and PCT detection was
more effective in diagnosing and predicting clinical
outcomes.ConclusionCombined detection of CRP and PCT has high clinical
application value in the diagnosis and prognosis evaluation of
fracture-related infections, providing more accurate guidance,
particularly in predicting recovery and infection control.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563170"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563170</a></p>
<hr />
<h2
id="pa-and-pa-x-two-key-proteins-from-segment-3-of-the-influenza-viruses">PA
and PA-X: two key proteins from segment 3 of the influenza viruses</h2>
<p>date: 2025-03-14, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>In recent years, the influenza viruses have posed an increasingly
severe threat to public health. It is essential to analyze the virulence
and pathogenesis of influenza viruses to prevent and control them, as
well as create antiviral drugs. Previous studies have revealed that
influenza virus segment 3 codes for not only the PA protein but also a
novel protein, PA-X. PA protein is one subunit of the polymerase of
influenza viruses and plays a critical role in its life cycle. PA
presented endonuclease activity, the transcription and replication of
the viral genome, viral virulence, protein degradation, and host immune
response by interacting with viral proteins, including PB2, PB1, and
host factors, including ANP32A, CHD6, HAX1, hCLE, HDAC6, MCM complex. PA
mutations were involved in the viral replication, pathogenicity, and
transmission of influenza viruses in poultry, mammals, and humans. PA-X
is an open reading frame generated by +1 ribosomal code shift at the
N-terminal amino acids of segment 3 and possesses the shutoff activity
of host gene expression, regulating the host immune response, viral
virulence and transmission. Therefore, PA is one ideal target for the
development of antiviral drugs against influenza viruses. Baloxavir
marboxil (BXM) and Favipiravir are two very effective anti-influenza
virus drugs targeting the PA endonuclease domain of influenza A viruses.
In this review, we summarized the structures, viral replication,
virulent determinants and transmission, host factors, innate immunity,
and antiviral drugs involved in PA and PA-X. The information is of great
value for underlying the mechanism of viral replication and developing
novel effective strategies to prevent and control influenza infection
and the pandemic.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560250"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560250</a></p>
<hr />
<h2 id="roles-of-circrnas-in-viral-pathogenesis">Roles of circRNAs in
viral pathogenesis</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Circular RNAs (circRNAs) are a class of non-coding RNAs with a
covalently closed circular structure, lacking 5’-caps or 3’-poly(A)
tails. They are relatively conserved, highly stable, and often exhibit
tissue- or cell-specific production in eukaryotic cells. Based on the
advances in sequencing technologies and bioinformatics, multiple reports
have suggested that viruses and other microorganisms may encode
circRNA-like molecules, providing new insights into the physiological
and pathological roles of circRNAs. The innate immune system functions
as the body’s primary defense mechanism against viral infections. It
detects pathogen-associated molecular patterns (PAMPs) and activates
signaling pathways to suppress viral replication and limit their spread.
CircRNAs are involved in regulation of the host innate immune signaling
pathways and play essential roles in viral pathogenesis. It has been
shown that circRNAs can regulate gene expression by acting as miRNA
sponges or protein sponges, or encoding small proteins in specific
cases. For example, previous studies have revealed that circRNAs
participate in the host antiviral immune response through the
competitive endogenous RNA (ceRNA) network by acting as miRNA sponges.
This review highlights research progress in the regulation and functions
of host- and virus-encoded circRNAs in host-virus interactions, as well
as their potential as diagnostic biomarkers and therapeutic targets in
clinical applications.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1564258"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1564258</a></p>
<hr />
<h2
id="rapid-and-sensitive-identification-of-candida-in-blood-based-on-m1-beads-enrichment-combined-with-multiple-recombinase-aided-pcr-a-culture-independent-approach">Rapid
and sensitive identification of Candida in blood based on M1 beads
enrichment combined with multiple recombinase-aided PCR: a
culture-independent approach</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionClinically, timely diagnosis and effective treatment of
Candida bloodstream infections rely on rapid and sensitive detection
methods. However, the long turn-around time and low detection rate of
blood culture (the gold standard) make rapid diagnosis of Candida
challenging. This study develops a novel molecular assay (M1-mRAP)
designed for the rapid and sensitive detection of three Candida species
in blood samples: Candida albicans(CA), Candida tropicalis(CT), and
Candida glabrata(CG).MethodsWe used the M1-mRAP method aimed at
detecting Candida DNA in blood samples, in which we developed a novel
multiplex recombinase-aided PCR (mRAP) assay for sensitive amplification
of Candida DNA and used a self-developed recombinant human
mannan-binding lectin beads (M1 beads)method for enrichment of Candida
in blood. The analytical sensitivity of mRAP was evaluated using Candida
recombinant plasmids. The analytical sensitivity of the M1-mRAP method
for blood sample detection was assessed using quantitative Candida
simulated blood samples. The clinical performance of the mRAP and
M1-mRAP methods was evaluated in 120 non-blood samples and 9 blood
samples and compared with conventional qPCR methods.ResultsThe limit of
detection(LOD) for CA, CT, and CG by the mRAP method were 4, 4, and 3
copies/μL, respectively. The LOD for CA, CT, and CG simulated blood
samples by the M1-mRAP were 2, 2, and 1 CFU/mL, and the overall
detection time was about 3.5 h. Clinical assays of mRAP and M1-mRAP
showed that these two methods were consistent with qPCR (P&lt;0.05), but
had better clinical detection ability than qPCR. Specifically, the mRAP
method identified 5 (4.2%) qPCR-negative samples, while M1-mRAP detected
1 (11.1%) classified as the qPCR grey zone sample.ConclusionThe M1-mRAP
method provides rapid and sensitive detection of low concentrations of
CA, CT, and CG blood samples and has the potential to emerge as an
important tool for the early detection of Candida bloodstream infections
in clinical settings.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1552529"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1552529</a></p>
<hr />
<h2
id="editorial-the-microbiome-in-the-development-of-gastrointestinal-diseases">Editorial:
The microbiome in the development of gastrointestinal diseases</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1561143"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1561143</a></p>
<hr />
<h2
id="the-gut-microbiota-inflammation-hfpef-axis-deciphering-the-role-of-gut-microbiota-dysregulation-in-the-pathogenesis-and-management-of-hfpef">The
gut microbiota-inflammation-HFpEF axis: deciphering the role of gut
microbiota dysregulation in the pathogenesis and management of
HFpEF</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Heart failure with preserved left ventricular ejection fraction
(HFpEF) is a disease that affects multiple organs throughout the body,
accounting for over 50% of heart failure cases. HFpEF has a significant
impact on individuals’ life expectancy and quality of life, but the
exact pathogenesis remains unclear. Emerging evidence implicates
low-grade systemic inflammation as a crucial role in the onset and
progression of HFpEF. Gut microbiota dysregulation and associated
metabolites alteration, including short-chain fatty acids,
trimethylamine N-oxides, amino acids, and bile acids can exacerbate
chronic systemic inflammatory responses and potentially contribute to
HFpEF. In light of these findings, we propose the hypothesis of a “gut
microbiota-inflammation-HFpEF axis”, positing that the interplay within
this axis could be a crucial factor in the development and progression
of HFpEF. This review focuses on the role of gut microbiota
dysregulation-induced inflammation in HFpEF’s etiology. It explores the
potential mechanisms linking dysregulation of the gut microbiota to
cardiac dysfunction and evaluates the therapeutic potential of restoring
gut microbiota balance in mitigating HFpEF severity. The objective is to
offer novel insights and strategies for the management of HFpEF.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1537576"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1537576</a></p>
<hr />
<h2
id="regulatory-effects-on-virulence-and-phage-susceptibility-revealed-by-sdia-mutation-in-klebsiella-pneumoniae">Regulatory
effects on virulence and phage susceptibility revealed by sdiA mutation
in Klebsiella pneumoniae</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe World Health Organization has identified multi-drug
resistant Klebsiella pneumoniae strains as the highest priority in 2024.
Understanding the regulatory routes of virulence features is crucial for
the development of novel anti-virulence strategies. SdiA, a LuxR-like
quorum sensing (QS) receptor that responds to N-acyl-homoserine lactones
(AHLs), is involved in the regulation of virulence traits in some
Gram-negative bacteria. The function of this receptor in the virulence
of K. pneumoniae remains uncertain. The objective of the present study
was to elucidate the function of SdiA in K. pneumoniae biofilm formation
and virulence.MethodsTo this end, a genetic knockout of sdiA was
conducted, and virulence-related phenotypic studies were performed
following AHL provision.Results and DiscussionThe results demonstrate
that sdiA deficiency increases susceptibility to phage infection and
human serum resistance, and promotes biofilm maturation and cell
filamentation, although no effect on virulence was observed in vivo in
the Galleria mellonella infection model. On the other hand, C6-HSL
promoted sdiA-dependent biofilm maturation, capsule production and serum
resistance while reducing virulence against G. mellonella in the absence
of sdiA. The addition of C6-HSL did not affect phage susceptibility. The
results of this study demonstrate that AHLs and SdiA exert a dual
influence on virulence phenotypes, operating both independently and
hierarchically. These findings provide new insights into the virulence
of K. pneumoniae and its regulation by SdiA.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562402"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562402</a></p>
<hr />
<h2
id="identification-of-causative-agents-of-infective-endocarditis-by-metagenomic-next-generation-sequencing-of-resected-valves">Identification
of causative agents of infective endocarditis by metagenomic
next-generation sequencing of resected valves</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundInfective endocarditis (IE) is a rare and life-threatening
condition with considerable mortality rates. Diagnosis is often
complicated by negative blood culture results, limiting the accurate
identification of causative pathogens. This study aimed to evaluate the
effectiveness of metagenomic next-generation sequencing (mNGS) of
cardiac valve specimens compared to conventional clinical laboratory
methods for identifying pathogens in IE.MethodsNineteen patients with
suspected IE who were scheduled for surgical valve removal were
prospectively enrolled. The metagenomic workflow included bacterial DNA
enrichment from resected valves using the Molzym Ultra-Deep Microbiome
Prep, sequencing of metagenomic libraries using the Illumina MiSeq
platform, and Kraken 2 taxonomic assignments based on read
data.ResultsValve mNGS achieved a sensitivity of 82.4% and a specificity
of 100% relative to the final adjudicated pathogen diagnosis. Blood
culture, considered the reference standard, exhibited slightly higher
sensitivity (88.2%) with comparable specificity (100%). In comparison,
valve culture (sensitivity: 29.4%, specificity: 50.0%) and microscopy
(sensitivity: 35.3%, specificity: 100%) showed lower diagnostic
performance. Delays between blood culture negativization and valve
resection impacted mNGS sensitivity, likely due to pathogen clearance.
Notably, valves resected within 12 days from blood culture
negativization achieved 100% diagnostic accuracy, emphasizing the
importance of timing for optimal mNGS results.ConclusionThis study
underscores mNGS as a valuable diagnostic tool for detecting IE
pathogens, complementing traditional diagnostic methods. The detection
of antibiotic resistance genes and multi-locus sequence typing profiles
in some samples further demonstrated its utility.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1532257"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1532257</a></p>
<hr />
<h2
id="computational-analysis-of-ayurvedic-metabolites-for-potential-treatment-of-drug-resistant-candida-auris">Computational
analysis of Ayurvedic metabolites for potential treatment of
drug-resistant Candida auris</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>This study explored the effectiveness of secondary metabolites of
referred traditional Ayurvedic plants in treating fungal infections,
particularly targeting Candida auris. Recognized as a global health
threat, this fungus is notorious for its resistance to several
antifungal treatments. The inhibition of lanosterol 14α-demethylase
causes the depletion of ergosterol, ultimately resulting in the
inhibition of fungal cell growth. A total of 469 metabolites, including
alkaloids, flavonoids, and tannins from Ayurvedic plants, were screened
against CYP51 (PDB ID: 4UYL) using molecular docking. Key active site
residues, namely HIS461, CYS463, and TYR122, were targeted to inhibit
the ergosterol synthesis, with VNI employed to benchmark the findings.
Shortlisted metabolites underwent physicochemical analysis, ADMET
analyses, and the principles of medicinal chemistry, which were
confirmed through pharmacokinetic simulations. Further, this study
investigated the molecular dynamics (MD) of co-crystalized VNI,
trans-p-coumaric acid, and MCPHB
[(r)-n-(1’-methoxycarbonyl-2’-phenylethyl)-4-hydroxybenzamide] to
evaluate RMSD, RMSF, Rg, SASA, cross-correlation of residue motions,
PCA, and free energy decomposition. The top compounds demonstrated
favorable drug-like criteria. They exhibited good absorption potential
with high gastrointestinal uptake. Distribution and metabolism were
manageable with low risks of drug-drug interactions. Excretion profiles
indicated proper clearance, and toxicity assessments showed low
potential for cardiovascular issues. The results showed stable
interactions for trans-p-coumaric acid and MCPHB, suggesting that all
the ligands maintain stable binding interactions with the protein, which
preserves structural integrity across all systems. This comprehensive
approach suggests that these natural metabolites from Ayurvedic medicine
could potentially serve as primary agents against fungal diseases,
pending further validation through controlled in vitro and in vivo
clinical trials.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1537872"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1537872</a></p>
<hr />
<h2
id="the-investigation-of-molecular-epidemiological-characteristics-and-resistance-mechanism-of-tigecycline-resistant-klebsiella-pneumoniae-from-a-large-teaching-hospital-in-southwest-china-chongqing">The
investigation of molecular epidemiological characteristics and
resistance mechanism of tigecycline resistant Klebsiella pneumoniae from
a large teaching hospital in southwest China, Chongqing</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundKlebsiella pneumoniae is one of the main pathogens of
nosocomial infection, among which carbapenems can be used for
multidrug-resistant Klebsiella pneumoniae. However, in the past decade,
the resistance rate of carbapenem-resistant Klebsiella pneumoniae has
increased yearly. Tigecycline has good antibacterial activity in
treating severe bacterial infections, but the reports of tigecycline
resistance are increasing. This study aimed to investigate the mechanism
of drug resistance and epidemiological characteristics of
tigecycline-resistant Klebsiella pneumoniae (TRKP) in a large teaching
hospital in southwest China, Chongqing.MethodsWe isolated 30 TRKP
strains from this hospital between August 2021 and December 2023. By PCR
and sequencing, we examined the presence and mutation rates of genes
associated with tigecycline resistance, including acrR, oqxR, ramR,
tmexC, tet(x), tet(A), tet(L), and rpsj, and performed efflux pump
inhibition experiments to verify efflux pump activity. At the same time,
real-time RT-PCR was used to detect the expression levels of efflux pump
genes (acrB and oqxB) and ramA. To investigate the prevalence trend of
TRKP in our hospital, we performed multi-site sequence typing (MLST)
analysis.ResultsThe mutation rates of ramR (73.3%) and tet(A) (63.3%)
were significant. In efflux pump inhibition experiments, PaβN could
reverse the resistance of 29 TRKP strains (96.7%) to tigecycline.
Real-time RT-PCR results showed that acrB and ramA genes were
up-regulated in 22 strains, while oqxB genes were overexpressed in only
4 strains. MLST analysis showed that these strains could be divided into
25 different ST subtypes, indicating that no outbreak of TRKP occurred
in our hospital. In addition, two tmexCD-torpj positive strains, ST661
and ST1561, were identified for the first time.ConclusionThe efflux pump
acrB and tet(A) mutations are the primary mechanisms of resistance to
tigecycline-resistant Klebsiella pneumoniae at our hospital. The ramR
mutation can mediate efflux pump activity of acrB by up-regulating ramA
overexpression.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540967"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1540967</a></p>
<hr />
<h2
id="infection-of-human-induced-pluripotent-stem-cells-by-an-oncogenic-herpesvirus">Infection
of human induced pluripotent stem cells by an oncogenic herpesvirus</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveAs one of the major human oncogenic viruses, Kaposi’s
Sarcoma-associated Herpesvirus (KSHV) is closely related to several
cancers such as Kaposi’s sarcoma (KS) and primary effusion lymphoma
(PEL). KSHV can infect a broad tropism of human primary cells in vitro
and in vivo. Embryonic stem cell-like pluripotent stem cells can be
generated by the simultaneous introduction of several factors, into
somatic cells, yielding induced pluripotent stem (iPS) cells. However,
it remains unclear whether human induced pluripotent stem cells (hiPSCs)
are permissive to KSHV and how this oncogenic virus infection may affect
cellular gene profile.MethodsIn the current study, we examined whether
hiPSCs were permissive to KSHV infection. The flow cytometry was used to
assess the impacts of KSHV infection on hiPSCs viability and apoptosis.
The Illumina RNA-Sequencing was used to determine cellular gene profile
changed in KSHV-infected hiPSCs and lytically induced cells.ResultsWe
report that KSHV successfully establishes latent infection in hiPSCs,
which can be completely induced to lytic reactivation and release
infectious virions. KSHV de novo infection arrests the growth of hiPSCs
through inducing cell apoptosis. Transcriptomic analysis revealed
significant changes in global cellular gene expression in KSHV-infected
hiPSCs as well as lytically induced cells.ConclusionOur findings
demonstrate hiPSCs as a powerful tool to explore the potential impacts
of KSHV infection on stem cell functions and virus pathogenesis in stem
cell differentiated cells.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563440"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563440</a></p>
<hr />
<h2 id="editorial-cancer-in-people-living-with-hivaids">Editorial:
Cancer in people living with HIV/AIDS</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1575386"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1575386</a></p>
<hr />
<h2
id="stool-carriage-of-ctx-mcmy-producing-salmonella-enterica-in-a-chinese-tertiary-hospital-in-shenzhen-china">Stool
carriage of CTX-M/CMY-producing Salmonella enterica in a Chinese
tertiary hospital in Shenzhen, China</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Salmonellosis, caused by non-typhoidal Salmonella, is a common
foodborne gastrointestinal infection. Third-generation cephalosporins
are recommended as the first-line treatment for Salmonella infections.
Our study aimed to investigate the molecular epidemiology, antimicrobial
resistance, and the transmission of extended-spectrum β-lactamases
(ESBL) genes in 96 clinical Salmonella isolates collected between 2020
and 2022 at a tertiary hospital in Shenzhen, China. We performed
antimicrobial susceptibility testing and whole-genome sequencing to
identify serotypes, multilocus sequence typing, antimicrobial resistance
genes in these isolates, and the genetic structures of the
blaCTX-M/blaCMY genes. Seventeen Salmonella serotypes were identified,
with S. 4,[5],12:i:- (37.5%) being the most common, followed by S.
Enteritidis (15.63%), S. Typhimurium (14.58%), S. London (7.29%), and S.
Rissen (5.21%). MLST analysis revealed 19 distinct sequence types (STs),
with ST34 being the most prevalent (36.46%), followed by ST11 (15.63%)
and ST19 (13.54%). Antimicrobial resistance testing showed those
isolates had high levels of resistance to ampicillin (72.92%) and
tetracycline (71.88%), with 70.83% of isolates as multidrug-resistant
(MDR). Three blaCTX-M genes (blaCTX-M-14,blaCTX-M-55, and blaCTX-M-65)
and blaCMY-2 were identified among 18 cefotaxime-resistant strains, of
which one and 12 isolates successfully transferred blaCMY or blaCTX-M to
E. coli C600 via conjugation, respectively. The
blaCTX-M/blaCMY-2-carrying contigs in nine Salmonella isolates ranged
from 2,156 to 164,862 bp, were located either on the chromosome (n=1) or
plasmids (IncI1, IncK1, IncA/C) (n=9), and the blaCTX-M/blaCMY-2 genes
were associated with ISEcp1. Our study demonstrates the diversity of MDR
Salmonella serotypes in clinical isolates, and highlights the role of
plasmids and mobile genetic elements in the horizontal transfer of
blaCTX-M/blaCMY, emphasizing the need for continuous surveillance of
Salmonella in clinical samples.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1544757"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1544757</a></p>
<hr />
<h2
id="in-depth-characterization-of-klebsiella-pneumoniae-carbapenemase-kpc-encoding-plasmids-points-at-transposon-related-transmission-of-resistance-genes">In-depth
characterization of Klebsiella pneumoniae carbapenemase (KPC)-encoding
plasmids points at transposon-related transmission of resistance
genes</h2>
<p>date: 2025-03-13, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Antimicrobial resistance (AMR) is a growing threat in healthcare
systems, particularly in the management of infections in critically ill
patients. This study highlights how to identify clusters and putative
sharing of mobile genetic elements, such as transposons, in the hospital
setting using long-read whole genome sequencing (lrWGS). The approach
described here can be employed to investigate the transmission dynamics
of KPC-3-positive Klebsiella pneumoniae at multiple levels, from the
entire isolate down to individual plasmids and transposons. Here, a
blaKPC-3 harboring transposon cluster was identified by using a
Mash-based distance calculation for plasmids. This approach was used to
investigate a local accumulation of KPC-3-positive Klebsiella pneumoniae
on surgical and infectious disease wards of a tertiary care center in
Germany over a time of six months. In total, seven patients were
affected. Core genome multi-locus sequence typing analysis (cgMLST)
identified two distinct genetic clusters: a sequence type (ST) 307
cluster (n = 5) and a ST101 cluster (n = 2). All isolates carried a
blaKPC-3 carbapenemase. Further Mash distance-based plasmid analysis was
not consistent with plasmid transfer due to genetic heterogeneity, but
identified a transposon cluster across all isolates. Infection control
evaluation of patient movements within their hospital admission supports
a possible clonal transmission. Subsequent infection control measures,
including point prevalence screening and enhanced contact precautions,
successfully contained further transmissions. The study illustrates the
value of in-depth plasmid analysis in understanding the transmission
dynamics and epidemiology of AMR, particularly in hospital
environments.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1542828"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1542828</a></p>
<hr />
<h2
id="tick-hrf-dependent-ferroptosis-pathway-to-promote-tick-acquisition-of-babesia-microti">Tick
HRF-dependent ferroptosis pathway to promote tick acquisition of Babesia
microti</h2>
<p>date: 2025-03-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>B. microti is a tick-transmitted zoonotic erythrocytic intracellular
parasite. Ferroptosis is an iron-dependent form of programmed cell death
that affects pathogen replication in the host. Currently, there is
limited research concerning the effect of tick ferroptosis on Babesia
infection and the underlying mechanism of action. The present study used
a B. microti -mouse- Haemaphysalis longicornis infection model in which
nymphs fed on the blood of B. microti-infected mice. The midgut divalent
iron (p&lt;0.01) and reactive oxygen species (ROS) (p&lt;0.05) levels
were significantly elevated in infected ticks, and transmission electron
microscopy (TEM) showed that mitochondrial ridges were absent or
decreased in size. Downregulation of ferritin 1 and glutathione
peroxidase 4 (GPX4) in ticks infected with B. microti suggests that
these changes promote ferroptosis. In vivo studies demonstrated that the
ferroptosis promoter Erastin increased B. microti load (p&lt;0.05),
while the inhibitor Ferrostatin-1 effectively decreased load
(p&lt;0.01). Tick histamine-releasing factor (HRF), a protein related to
the antioxidant system, was downregulated in infected nymphs compared
with uninfected nymphs (p&lt;0.05), and interference with HRF promoted
tick acquisition of B. microti (p&lt;0.001). Transcriptomic analyses
showed that HRF interference promotes tick ferroptosis by downregulating
ferritin 1 and GPX4. Meanwhile, interference with tick HRF molecules
showed increased divalent iron and ROS and decreased mitochondrial
ridges compared with controls. These findings highlight the critical
role of tick HRF molecules in regulating ferroptosis and acquisition of
B. microti, thereby providing important insights for a deeper
understanding of the tick-Babesia interaction.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560152"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560152</a></p>
<hr />
<h2
id="the-role-of-intestinal-homeostasis-in-sevoflurane-induced-myelin-development-and-cognitive-impairment-in-neonatal-mice">The
role of intestinal homeostasis in sevoflurane-induced myelin development
and cognitive impairment in neonatal mice</h2>
<p>date: 2025-03-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundInhalational anesthetic sevoflurane is commonly used in
pediatric anesthesia. Multiple exposures to sevoflurane in early
postnatal life have been associated with long-term abnormalities in
myelin development and cognitive and memory impairments, although the
underlying mechanisms remain incompletely elucidated. Disruption of gut
microbiota is recognized as an important contributor to neurological
diseases. Here, we explore the potential mechanisms underlying the
abnormal myelin development induced by multiple sevoflurane exposures in
neonatal rats by analyzing gut homeostasis.MethodsSix-day-old (P6)
C57BL/6 mice were exposed to 3% sevoflurane for 2 hours per day for
three consecutive days. Mice exposed to a mixture of 60% nitrogen and
oxygen under the same conditions and duration served as controls.
Behavioral tests were conducted between P32 and P42. At P9 (24 hours
after the last sevoflurane exposure) and P42 (after the completion of
behavioral tests), intestinal and brain examinations were performed to
investigate the effects of sevoflurane exposure during the lactation and
adolescent periods on gut homeostasis and myelin development in mice.
Subsequently, the ameliorative effects of butyrate supplementation on
sevoflurane-induced abnormalities in myelin development and cognitive
and memory impairments were observed.ResultsAfter repeated exposure to
sevoflurane, neonatal mice developed persistent gut microbiota imbalance
accompanied by a decrease in short-chain fatty acids. Short-term
intestinal inflammation emerged, with damage to the mucus layer and
barrier function. In the hippocampus and prefrontal cortex, the
expression of genes and transcription factors related to oligodendrocyte
differentiation and myelin development was significantly affected, and
these changes persisted even after the exposure ended. There was a
reduction in proteins associated with oligodendrocytes and myelin
formation, which had a certain impact on memory and cognitive behavior.
This study also explored the potential connections between microbiota,
metabolism, the gut, the brain, and behavior. Timely supplementation
with butyrate could effectively reverse these changes, indicating that
gut homeostasis is crucial for brain neurodevelopment.ConclusionMultiple
exposures to sevoflurane in neonatal mice disrupt gut homeostasis and
affect oligodendrocyte differentiation and myelin development in the
hippocampus and prefrontal cortex, inducing cognitive and memory
impairments. Supplementation with butyrate can alleviate these
changes.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1541757"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1541757</a></p>
<hr />
<h2
id="corrigendum-clinical-characteristics-and-the-role-of-il-6-in-acute-on-chronic-liver-failure-patients-with-or-without-covid-19-a-multicenter-paired-cohort-study">Corrigendum:
Clinical characteristics and the role of IL-6 in acute-on-chronic liver
failure patients with or without COVID-19: a multicenter paired cohort
study</h2>
<p>date: 2025-03-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584105"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584105</a></p>
<hr />
<h2
id="exploration-of-the-primary-antibiofilm-substance-and-mechanism-employed-by-lactobacillus-salivarius-atcc-11741-to-inhibit-biofilm-of-streptococcus-mutans">Exploration
of the primary antibiofilm substance and mechanism employed by
Lactobacillus salivarius ATCC 11741 to inhibit biofilm of Streptococcus
mutans</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionLactobacillus salivarius serves as a probiotic
potentially capable of preventing dental caries both in vitro and in
vivo. This study focused on understanding the key antibiofilm agents and
the mechanisms of action of the Lactobacilli supernatant against
Streptococcus mutans.MethodsStreptococcus mutans biofilm was constructed
and the cell-free supernatant of Lactobacillus salivarius was added.
After the biofilm was collected, RNA-seq and qRT-PCR were then performed
to get gene information. The influence of temperature, pH and other
factors on the supernatant were measured and non-targeted metabolome
analysis was performed to analyze the effective components.ResultsThe
findings indicated that the supernatant derived from Lactobacillus
salivarius could inhibit the biofilm formation of Streptococcus mutans
at different times. Through transcriptome analysis, we discovered that
the cell-free supernatant reduced biofilm formation, by suppressing
phosphoenolpyruvate-dependent phosphotransferase systems along with two
ATP-binding cassette transporters, rather than directly affecting the
genes that code for glucosyltransferases; additionally, the supernatant
was observed to diminish the expression of genes linked to two-component
systems, polyketides/non-ribosomal peptides, acid stress response,
quorum sensing, and exopolysaccharide formation. Non-targeted LC-MS/MS
analysis was employed to discover a variety of potential active
compounds present in the cellular filtrate of Lactobacillus salivarius
that hinder the growth of S. mutans, including phenyllactic acid,
sorbitol, and honokiol.DiscussionIn summary, our findings support the
evaluation of Lactobacillus salivarius as a promising oral probiotic
aimed at hindering the formation of biofilms by cariogenic pathogens and
the development of dental caries.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535539"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1535539</a></p>
<hr />
<h2
id="aac6-iaq-a-novel-aminoglycoside-acetyltransferase-gene-identified-from-an-animal-isolate-brucella-intermedia-dw0551">aac(6’)-Iaq,
a novel aminoglycoside acetyltransferase gene identified from an animal
isolate Brucella intermedia DW0551</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundBacterial resistance to aminoglycoside antimicrobials is
becoming increasingly severe due to their use as commonly prescribed
antibiotics. The discovery of new molecular mechanisms of aminoglycoside
resistance is critical for the effective treatment of bacterial
infections.MethodsBacteria in goose feces were isolated by plate
streaking. The identification and characterization of a novel resistance
gene from the bacterial genome involved various techniques, including
molecular cloning, drug susceptibility testing, protein expression and
purification, and enzyme kinetic analysis. Additionally, whole-genome
sequencing and phylogenetic studies were performed.ResultsBrucella
intermedia DW0551, isolated from goose feces, was resistant to 35
antibiotics, and the minimum inhibitory concentration (MIC) was
particularly high for most aminoglycoside antibiotics. The novel
aminoglycoside resistance gene aac(6’)-Iaq encoded by B. intermedia
DW0551 conferred resistance to netilmicin, sisomicin, amikacin,
kanamycin, gentamicin, tobramycin, and ribostamycin. The amino acid
sequence of AAC(6’)-Iaq shared the highest identity (52.63%) with the
functionally characterized aminoglycoside acetyltransferase AAC(6’)-If.
AAC(6’)-Iaq contained all the conserved sites of the acetyltransferase
family NAT_SF. The enzyme exhibited strong affinity and catalytic
activity toward netilmicin and sisomicin. The mobile genetic element
(MGE) was not found in the flanking regions of the aac(6’)-Iaq and
aac(6’)-Iaq-like genes.ConclusionIn this work, a novel aminoglycoside
acetyltransferase gene, designated aac(6’)-Iaq, which conferred
resistance to a variety of aminoglycoside antimicrobials, was identified
in an animal Brucella intermedia isolate. Identification of new
antibiotic resistance mechanisms in bacteria isolated from animals could
aid in the treatment of animal and human infectious diseases caused by
related bacterial species.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1551240"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1551240</a></p>
<hr />
<h2
id="gut-microbiota-an-emerging-target-connecting-polycystic-ovarian-syndrome-and-insulin-resistance">Gut
microbiota: an emerging target connecting polycystic ovarian syndrome
and insulin resistance</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Polycystic ovary syndrome (PCOS) is a highly heterogeneous metabolic
disorder, with oligomenorrhea and hirsutism as patients’ primary
complaints. Hyperinsulinemia is a crucial pathophysiological mechanism
in the development of PCOS, with 50-70% of patients exhibiting insulin
resistance (IR). This condition not only exacerbates ovulatory
dysfunction but also leads to various adverse metabolic outcomes, such
as dyslipidemia and diabetes, and increases the risk of cardiovascular
events both before and after menopause. Gut microbiota is a microbial
community within the host that possesses significant metabolic potential
and is shaped by external environmental factors, the neuro-immune
network, and metabolism. Recent studies have shown that gut microbiota
dysbiosis is closely related to the development and progression of PCOS.
Despite the growing recognition of the potential role of gut microbiota
in the pathogenesis and treatment of PCOS, its clinical application
remains in its infancy. Currently, most clinical guidelines and expert
consensus still emphasize traditional therapeutic approaches, such as
hormonal treatments, lifestyle modifications, and insulin sensitizers.
However, accumulating evidence suggests that gut microbiota may
influence the metabolic and reproductive health of PCOS patients through
various mechanisms. Therefore, understanding the role of gut microbiota
between PCOS and IR is essential. This review describes the changes in
the gut microbiota of IR-PCOS patients, examines the potential
mechanisms by which the gut microbiota contributes to IR in PCOS
patients, and updates the evidence supporting the gut microbiota as a
potential metabolic regulatory target in IR-PCOS. In summary, gut
microbiota dysbiosis may be involved in the development and progression
of IR in PCOS patients, and improving gut microbiota may offer metabolic
stability benefits.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1508893"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1508893</a></p>
<hr />
<h2
id="predicting-the-immunological-nonresponse-to-antiretroviral-therapy-in-people-living-with-hiv-a-machine-learning-based-multicenter-large-scale-study">Predicting
the immunological nonresponse to antiretroviral therapy in people living
with HIV: a machine learning-based multicenter large-scale study</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAlthough highly active antiretroviral therapy (HAART) has
greatly enhanced the prognosis for people living with HIV (PLWH), some
individuals fail to achieve adequate immune reconstitution, known as
immunological nonresponse (INR), which is linked to poor prognosis and
higher mortality. However, the early prediction and intervention of INR
remains challenging in South China.MethodsThis study included 1,577 PLWH
who underwent at least two years of HAART and clinical follow-up between
2017 and 2022 at two major tertiary hospitals in South China. We
utilized logistic multivariate regression to identify independent
predictors of INR and employed restricted cubic splines (RCS) for
nonlinear analysis. We also developed several machine-learning models,
assessing their performance using internal and external datasets to
generate receiver operating characteristic (ROC) curves, calibration
curves, and decision curve analysis (DCA). The best-performing model was
further interpreted using Shapley additive explanations (SHAP)
values.ResultsIndependent predictors of INR included baseline, 6-month
and 12-month CD4+ T cell counts, baseline hemoglobin, and 6-month
hemoglobin levels. RCS analysis highlighted significant nonlinear
relationships between baseline CD4+ T cells, 12-month CD4+ T cells and
baseline hemoglobin with INR. The Random Forest model demonstrated
superior predictive accuracy, with ROC areas of 0.866, 0.943, and 0.897
across the datasets. Calibration was robust, with Brier scores of 0.136,
0.102, and 0.126. SHAP values indicated that early CD4+T cell counts and
CD4/CD8 ratio were crucial in predicting INR.ConclusionsThis study
introduces the random forest model to predict incomplete immune
reconstitution in PLWH, which can significantly assist clinicians in the
early prediction and intervention of INR among PLWH.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1466655"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1466655</a></p>
<hr />
<h2
id="characterization-of-fosa13-a-novel-fosfomycin-glutathione-transferase-identified-in-a-morganella-morganii-isolate-from-poultry">Characterization
of FosA13, a novel fosfomycin glutathione transferase identified in a
Morganella morganii isolate from poultry</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundM. morganii is a species of the genus Morganella in the
family Enterobacteriaceae. This species primarily causes infections of
postoperative wounds and the urinary tract. Some isolates of M. morganii
exhibit resistance to multiple antibiotics due to multidrug resistance
traits, complicating clinical treatment; thus, there is a growing need
to elucidate the resistance mechanisms of this pathogen.MethodsA total
of 658 bacterial strains were isolated from anal fecal swabs from
poultry and livestock and from the surrounding environment in Wenzhou,
China, via plate streaking. The full genome sequences of the bacteria
were obtained via next-generation sequencing platforms. The standard
agar dilution method was employed to determine the minimum inhibitory
concentrations (MICs) of various antimicrobial agents. The resistance
gene (fosA13) of the isolate was identified using the Comprehensive
Antibiotic Resistance Database (CARD) and confirmed via molecular
cloning. The FosA13 protein encoded by the novel resistance gene fosA13
was expressed with the vector pCold I, and its enzyme kinetics
parameters were characterized. The genetic background and evolutionary
process of the sequence of this novel resistance gene were analyzed by
means of bioinformatics methods.ResultsIn this study, we identified a
new chromosomally encoded fosfomycin resistance gene, designated fosA13,
from the M. morganii isolate DW0548, which was isolated from poultry on
a farm in Wenzhou, China. Compared with the control strain
(pUCP19/DH5α), the recombinant strain carrying fosA13
(pUCP19-fosA13/DH5α) presented a fourfold increase in the MIC value for
fosfomycin. The enzyme kinetics data of FosA13 revealed effective
inactivation of fosfomycin, with a kcat/Km of (1.50 ± 0.02)×104 M-1·s-1.
Among functionally characterized resistance proteins, FosA13 presented
the highest amino acid (aa) homology (55.6%) with FosA. FosA13 also
contained essential functional residues of FosA proteins. The isolate M.
morganii DW0548 presented high MIC values (≥ 8 μg/mL) for 5 classes of
antimicrobials, namely, aminoglycosides, β-lactams, quinolones,
tetracycline, and chloramphenicol, but only two functionally
characteristic antimicrobial resistance genes (ARGs) have been
identified in the complete genome: a β-lactam resistance gene
(blaDHA-16) and a phenol resistance gene (catII). These findings
indicate that in addition to the novel resistance gene identified in
this work, other uncharacterized resistance mechanisms might exist in M.
morganii DW0548.ConclusionA novel chromosomal fosfomycin resistance
gene, fosA13, was identified in an animal M. morganii isolate, and its
enzymatic parameters were characterized. This protein shares the highest
aa identity of 55.6% with the functionally characterized protein FosA
and has all the essential functional residues of FosA proteins.
Exploring more antimicrobial resistance mechanisms of this pathogen
would help clinicians choose effective drugs to treat infectious
diseases in animal husbandry and clinical practice and facilitate the
development of methods to prevent the spread of resistance between
bacteria of different species.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534084"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534084</a></p>
<hr />
<h2
id="fungi-from-malus-in-qujing-china-two-new-species-three-new-records-and-insights-into-potential-host-jumping-and-lifestyle-switching">Fungi
from Malus in Qujing, China: two new species, three new records, and
insights into potential host jumping and lifestyle switching</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Apple trees [Malus domestica Borkh. (Rosaceae)] are one of the
important temperate fruit crops in China. In comparison to other
temperate fruits, such as grapes and pears, fungal studies (in Yunnan)
associated with M. domestica are fewer in number. In the present study,
we investigated fungi associated with M. domestica in Qujing City,
Yunnan Province, China. Samples were collected from apple gardens in
different locations. Single spore isolation was carried out to isolate
saprobic fungi, while the surface sterilization method was carried out
to isolate endophytic fungi. Molecular analyses were carried out to
determine the phylogenetic placement of the new collections. Based on
the combined methods of morphology and phylogeny, Cytospora qujingensis
sp. nov. and Hypoxylon malongense sp. nov. are introduced as novel
saprobic and endophytic taxa, respectively. Moreover, Aureobasidium
pullulans and Cytospora schulzeri are reported as new geological records
from southwestern China. Allocryptovalsa castaneae is reported on M.
domestica in China for the first time. The checklist of fungi associated
with M. domestica in China is presented.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1517908"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1517908</a></p>
<hr />
<h2
id="adaptive-attenuation-of-virulence-mediated-by-wzc-mutation-in-st11-kl47-carbapenem-resistant-klebsiella-pneumonia">Adaptive
attenuation of virulence mediated by Wzc mutation in ST11-KL47
Carbapenem-resistant Klebsiella pneumonia</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe impact of the hypermucoviscosity (HMV) phenotype in
ST11-KL47 carbapenem-resistant Klebsiella pneumoniae (CRKp)
pathogenicity warrants investigation for public health risk
assessment.MethodsWe analyzed 230 clinical ST11-KL47 CRKp to identify
the key factor in mucoviscosity acquisition via comparative genomic
analysis. Sedimentation value served as the objective index to quantify
HMV. The virulence in vivo was assessed using Galleria mellonella and
mouse infection models. We employed genome engineering, capsular
polysaccharides (CPS) quantification, and visualization to explore the
role of Wzc mutation in CPS biosynthesis and HMV. The biological impact
of Wzc-mediated HMV was investigated through competitive growth
analysis, biofilm formation, serum resistance, anti-phagocytic ability,
and adhesion assays. Transcriptomic analysis and scanning electron
microscopy (SEM) were utilized to explore the relationship between
polysaccharide composition, physical distribution, and changes in
virulence.ResultsThe Wzc mutations are identified as the key to
mucoviscosity acquisition. Unexpectedly, Wzc-mediated HMV CRKp exhibits
reduced pathogenicity versus non-mucoviscosity (NMV) strains in
different animal models, with competitive disadvantage, decreased
biofilm formation, serum resistance, and adhesion, yet higher
anti-phagocytic ability in vitro. CPS extraction and visualization of
genome-engineered strains verify the Wzc mutations mediate HMV by
standardizing CPS chain length and overproducing cell-free extracellular
polysaccharides (cell-free EPS). Transcriptomic results,
lipopolysaccharides (LPS) quantification, and SEM collectively indicate
a downregulation of LPS synthesis and the masking of LPS in HMV
strains.DiscussionThese findings demonstrate that the Wzc-induced HMV
attenuates ST11-KL47 CRKp virulence by modifying the exopolysaccharide
composition and physical distribution.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1561631"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1561631</a></p>
<hr />
<h2
id="the-microbial-metabolite-trimethylamine-n-oxide-and-the-kidney-diseases">The
microbial metabolite trimethylamine N-oxide and the kidney diseases</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Trimethylamine N-oxide (TMAO), a metabolite, is a co-metabolite
produced by both gut microbiota and livers, originating from foods rich
in choline or carnitine. Emerging evidence suggests that TMAO may play a
role in the pathogenesis of various kidney diseases, including acute
kidney injury and chronic kidney disease. Research has demonstrated that
heightened levels of TMAO are correlated with a heightened likelihood of
kidney disease advancement and cardiovascular incidents among
individuals with chronic kidney disease. Furthermore, TMAO has been
observed to stimulate inflammation, oxidative stress, and fibrosis in
animal models of kidney disease. Mechanistically, TMAO may contribute to
kidney disease pathogenesis by inhibiting autophagy, activating the
NLRP3 inflammasome, and inducing mitochondrial dysfunction. Therefore,
targeting TMAO may represent a promising therapeutic strategy for the
treatment of kidney diseases. Future studies are needed to further
investigate the role of TMAO in kidney disease pathogenesis and to
develop TMAO-targeted therapies for the prevention and treatment of
kidney diseases.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1488264"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1488264</a></p>
<hr />
<h2
id="genomic-epidemiology-and-public-health-implications-of-zoonotic-monophasic-salmonella-typhimurium-st34">Genomic
epidemiology and public health implications of zoonotic monophasic
Salmonella Typhimurium ST34</h2>
<p>date: 2025-03-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMonophasic Salmonella Typhimurium sequence type 34 (mSTM
ST34) has emerged as a significant global health threat, but our
understanding of its genomic epidemiology and potential public health
implications in international and regional contexts remains limited.
This study aims to fill this crucial gap by assessing the genomic
epidemiology of multidrug resistance (MDR) mSTM ST34, as well as its
clinical characteristics and virulence.MethodsTo achieve the objectives
of this study, we conducted a comprehensive genomic analysis of mSTM
ST34 isolates. We obtained a global dataset comprising 13,844 strains
from public databases, along with 339 strains from a regional
surveillance collection in Zhejiang Province, China. This dataset aims
to provide in-depth insights into antimicrobial resistance, mobile
genetic elements, and pathogenicity. Additionally, we meticulously
assessed the association between phenotypic profiles and clinical
presentations.ResultsOur findings revealed that the prevalence of mSTM
ST34 has surpassed that of the previously dominant ST19. In addition, we
observed an increase in the detection of the IncQ1 plasmid, which is
responsible for disseminating MDR. The prevalence of mSTM ST34 carriage
was exceptionally high among children (≤12 years old) and elderly
individuals (≥65 years old), with 92.6% of the isolates exhibiting MDR,
including resistance to frontline antimicrobials such as
third-generation cephalosporins and ciprofloxacin. Additionally, the
human mSTM ST34 strain demonstrates a remarkable capacity for biofilm
formation, which increases its virulence in animal models and
complicates therapeutic interventions.ConclusionsmSTM ST34 has surpassed
the previously dominant ST19, and its ability to transmit across
multi-species increases its potential for further human transmission.
This study addresses critical gaps in our understanding of mSTM ST34
prevalence, highlighting the importance of whole genome sequencing in
surveilling zoonotic pathogens.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1490183"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1490183</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2025-03-10, from: People’s CDC</p>
<p>This is the People’s CDC bi-weekly update for March 10, 2025! This
Weather Report from the People’s CDC sheds light on the ongoing COVID
pandemic and other public health crises in the US.</p>
<p><br></p>
<p><a
href="https://peoplescdc.org/2025/03/10/peoples-cdc-covid-19-weather-report-93/"
class="uri">https://peoplescdc.org/2025/03/10/peoples-cdc-covid-19-weather-report-93/</a></p>
<hr />
<h2
id="urinary-microbiota-changes-among-nmibc-patients-during-bcg-therapy-comparing-bcg-responders-and-non-responders">Urinary
microbiota changes among NMIBC patients during BCG therapy: comparing
BCG responders and non-responders</h2>
<p>date: 2025-03-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The gold standard for treating high-risk non-muscle-invasive bladder
cancer involves the transurethral removal of cancerous tissue followed
by BCG immunotherapy. So far, there is no reliable biomarker for
predicting BCG efficacy and identifying patients who will or will not
respond to BCG treatment. Emerging evidence suggests that urinary
microbiota may play a crucial role in BCG efficacy. This study aimed to
explore (i) changes in urinary microbiota during the six induction
cycles of BCG and (ii) its potential predictive role in determining the
outcome of BCG treatment. To this end, catheterized urine samples were
collected before each of the six BCG doses and bacterial composition was
analyzed using 16S rRNA gene sequencing. Patient inclusion criteria were
male gender, no previous history of urothelial cancer, no other
malignancies, no active infection, and no antibiotic usage for at least
20 days before the first BCG dose. We observed a significant decrease in
biodiversity, measured by the Shannon Index, during the first week of
therapy in 10 out of 12 patients (p=0.021). Additionally, differences in
microbiota composition before the start of BCG therapy were noted
between responders and non-responders to BCG therapy. Non-responders
exhibited a 12 times higher abundance of genus Aureispira (p&lt;0.001),
and, at the species level, a 27-fold lower abundance of Negativicoccus
succinivorans (p&lt;0.001). Throughout the treatment, the abundance of
the genus Aureispira decreased, showing an eightfold reduction by the
end of therapy among non-responders (p&lt;0.001). Our findings suggest
that urinary microbiota plays an active role before and during the
course of BCG therapy. However, this is a preliminary study, and further
research involving larger patient cohorts is needed.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1479795"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1479795</a></p>
<hr />
<h2
id="distinct-gut-microbiome-features-characterize-fasciola-hepatica-infection-and-predict-triclabendazole-treatment-outcomes-in-peruvian-patients">Distinct
gut microbiome features characterize Fasciola hepatica infection and
predict triclabendazole treatment outcomes in Peruvian patients</h2>
<p>date: 2025-03-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundFasciola hepatica, a globally distributed helminth, causes
fasciolosis, a disease with significant health and economic impacts.
Variability in triclabendazole (TCBZ) efficacy and emerging resistance
are remaining challenges. Evidence suggests that the gut microbiome
influences host-helminth interactions and is associated with
anthelmintic effects, but its association with human F. hepatica
infection and TCBZ efficacy is not well understood.MethodsIn this study,
we investigated the relationship between Fasciola hepatica infection and
the gut microbiome through metagenomic shotgun sequencing of 30 infected
and 60 age- and sex-matched uninfected individuals from Peru.
Additionally, we performed a longitudinal analysis to evaluate
microbiome dynamics in relation to TCBZ treatment response.Results and
discussionInfection was associated with specific microbial taxonomic and
functional features, including higher abundance of Negativibacillus
sp900547015, Blautia A sp000285855, and Prevotella sp002299635 species,
and enrichment of microbial pathways linked to survival under stress and
depletion of pathways for microbial growth. Unexpectedly, we identified
that responders to TCBZ treatment (who cleared infection) harbored many
microbiome features significantly different relative to non-responders,
both before and after treatment. Specifically, the microbiomes of
responders had a higher abundance Firmicutes A and Bacteroides species
as well as phospholipid synthesis and glucuronidation pathways, while
non-responders had higher abundance of Actinobacteria species including
several from the Parolsenella and Bifidobacterium genera, and
Bifidobacterium shunt and amino acid biosynthesis
pathways.ConclusionsOur findings underscore the impact of helminth
infection on gut microbiome and suggest a potential role of gut
microbiota in modulating TCBZ efficacy, offering novel insights into F.
hepatica-microbiome interactions and paving the way for
microbiome-informed treatment approaches.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1555171"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1555171</a></p>
<hr />
<h2
id="gut-microbiome-is-associated-with-radiotherapy-response-in-lung-cancer-patients-with-brain-metastases">Gut
microbiome is associated with radiotherapy response in lung cancer
patients with brain metastases</h2>
<p>date: 2025-03-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>PurposeTo investigate the gut microbiome of lung cancer patients with
brain metastases undergoing radiotherapy, identify key microorganisms
associated with radiotherapy response, and evaluate their potential as
biomarkers.Methods and materialsThis study enrolled 55 newly diagnosed
lung cancer patients with brain metastases. Fecal samples were collected
before radiotherapy and analyzed by 16S rRNA sequencing to assess the
gut microbiome’s composition and function. Patients were categorized
into response (n=28) and non-response (n=27) groups based on treatment
efficacy, and α-diversity, β-diversity, and functional pathways were
compared between them. Linear Discriminant Analysis Effect Size was used
to identify microbial features associated with treatment efficacy.
Logistic regression analyses were performed to evaluate the predictive
capacity of clinical and microbial factors for treatment
outcomes.ResultsNo significant difference in α-diversity was observed
between the groups (P &gt; 0.05), but β-diversity differed significantly
(P = 0.036). Twelve characteristic microorganisms were identified in the
response group, including g_ Oscillibacter and g_ Blautia, and nine in
the non-response group, such as f_ Desulfovibrionaceae and g_ Megamonas.
Metabolic pathways associated with treatment response included ketone
body metabolism and pathways related to amyotrophic lateral sclerosis.
Multivariate analysis identified g_Flavonifractor (odds ratio [OR] =
6.680, P = 0.004), g_Negativibacillus (OR = 3.862, P = 0.014),
C-reactive protein (OR = 1.054, P = 0.017), and systemic inflammation
response index (OR = 1.367, P = 0.043) as independent predictors of
radiotherapy response. The nomogram and microbiome models achieved area
under the curve (AUC) values of 0.935 and 0.866, respectively,
demonstrating excellent predictive performance. Decision curve analysis
further confirmed these models provided significant net benefits across
risk thresholds.ConclusionsThe composition and functional
characteristics of the gut microbiome in lung cancer patients with brain
metastases prior to radiotherapy are associated with therapeutic
response and possess potential as predictive biomarkers. Further studies
are warranted to validate these findings.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562831"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1562831</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
