<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2025-05-29 14:08:08)</p>
<hr />
<h2
id="concentrations-of-uremic-bacterial-metabolites-in-patients-with-post-covid-19-syndrome">Concentrations
of uremic bacterial metabolites in patients with post-COVID-19
syndrome</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Post-COVID-19 syndrome (PCS) is characterized by persistent symptoms
and reduced mental and physical performance following the acute phase of
COVID-19. The underlying mechanisms remain unclear but may involve gut
microbiota dysbiosis and immune-related changes in amino acid
metabolism. This pilot study aimed to investigate whether specific
bacterial uremic metabolites (BUM) are altered in patients with
post-infectious syndromes and whether these alterations are associated
with PCS symptoms. We examined BUM in 25 PCS patients, 8 Myalgic
Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS) patients, and 8
healthy controls (Ctrls). Concentrations of BUM were determined in
second morning urine samples using mass spectrometry (Biovis Diagnostik,
Limburg, Germany). Standardized questionnaires assesed physical,
cognitive, psychological, and somatic symptoms and mental health status.
PCS and ME/CFS patients exhibited significantly higher scores for
post-exertional malaise (PEM) and somatic symptom severity compared to
healthy controls (p&lt;0.001). Elevated BUM concentrations were found in
64% of PCS patients, compared to 37.5% of both healthy controls and
ME/CFS patients. While overall BUM levels did not significantly differ
between groups, heatmap clustering revealed distinct metabolic patterns.
Elevated tryptamine and 4-hydroxyphenylpropionic acid (HPHPA) and higher
hippuric acid and trimethylamine concentrations, were exclusively
analysed in patients with post-infectious syndromes. Our pilot study
suggests that urine metabolomic analysis may be a useful approach for
investigating the role of gut dysbiosis and BUM in patients with
PCS.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582972"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1582972</a></p>
<hr />
<h2
id="influence-of-helicobacter-pylori-infection-on-chinese-adult-males-body-muscle-mass-a-cross-sectional-and-cohort-analysis">Influence
of helicobacter pylori infection on Chinese adult males’ body muscle
mass: a cross-sectional and cohort analysis</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>AimHelicobacter pylori (HP) infection may cause many disorders
outside the digestive system, although research on the association
between HP infection and muscular atrophy among males is limited. This
study aimed to examine the influence of HP infection on adult Chinese
males’ body muscle mass using a cross-sectional and cohort
analysis.MethodsA total of 8110 Chinese adult males were enrolled in the
study. HP infection was assessed using the C13 breath test. Using
bioelectrical impedance analysis (BIA), body muscle mass was detected
and total muscle mass, total skeletal muscle mass, and appendicular
skeletal muscle mass index (ASMI) were recorded. Univariate and
multivariate linear regression analyses were conducted to identify the
factors associated with body muscle mass.ResultsSubjects with HP
infection had a lower total muscle mass, total skeletal muscle mass, and
ASMI compared with those without HP infection (P &lt; 0.01). Univariate
and multivariate linear regression analyses demonstrated that HP
infection was an independent risk factor of total muscle mass, total
skeletal muscle mass, and ASMI (P &lt; 0.01); this association persisted
when subjects were divided into young, middle-aged, and elderly
according to age (P &lt; 0.01). A subsequent cohort study confirmed that
persistent HP infection accelerated the pathological process of muscle
decline (P &lt; 0.01).ConclusionHP infection is an independent risk
factor for muscle decline in adult Chinese males, long-term HP infection
may accelerate this pathological progression.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1575108"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1575108</a></p>
<hr />
<h2
id="clinical-efficacy-of-non-pharmacological-treatment-of-functional-constipation-a-systematic-review-and-network-meta-analysis">Clinical
efficacy of non-pharmacological treatment of functional constipation: a
systematic review and network meta-analysis</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe purpose of this study is to compare the relative
effectiveness and safety of non-pharmacological interventions for the
treatment of functional constipation (FC).MethodsWe searched Pubmed,
Embase, Cochrane, and Web of Science databases for randomized controlled
trials published from 2010 to November 2024. The quality of the included
studies was evaluated using the Cochrane bias risk tool and Review
Manager 5.4, and the evidence was graded using GRADEPro. A network
meta-analysis (NMA) was conducted using R Studio, and the surface under
the cumulative ranking curve (SUCRA) was used to rank the included drugs
for each outcome measure to compare the clinical efficacy of different
treatment methods for chronic functional constipation.ResultsA total of
29 RCT studies were included, with a total of 4389 patients with
functional constipation who were randomly assigned to receive placebo or
one of the nine different non-pharmacological treatment methods. The
assessment of bias risk showed that the bias risk of most included
studies was low. The results showed that the first-ranked treatment
method for clinical efficacy was acupuncture; the first-ranked treatment
method for changes in spontaneous bowel movement (SBM) and complete
spontaneous bowel movement (CSBM) was fecal microbiota transplantation
(FMT); the first-ranked treatment method for changes in the Bristol
Stool Form Scale (BSFS) score was FMT; the first-ranked treatment method
for changes in the Patient Assessment of Constipation Quality of Life
(PAC-QOL) score after treatment was the Vibration capsule; the
first-ranked treatment method for changes in the Patient Assessment of
Constipation Symptoms (PAC-SYM) score after treatment was percutaneous
electrical stimulation; and the treatment method with the lowest
incidence of adverse reactions was probiotics.ConclusionBased on the
SUCRA values and NMA results, we found that FMT showed better effects
and higher safety on BSFS scores, SBM, and CSBM. In addition,
acupuncture showed a good clinical efficacy. We hypothesize that the
combination of FMT and acupuncture may be an effective and safe
treatment option for functional constipation, but further high-quality
clinical studies are needed to confirm this.Systematic review
registrationhttps://www.crd.york.ac.uk/prospero/, identifier
CRD42024625747.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565801"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565801</a></p>
<hr />
<h2
id="editorial-improve-immunity-against-infection-and-tumor-by-regulating-gut-microbiota">Editorial:
Improve immunity against infection and tumor by regulating gut
microbiota</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1546123"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1546123</a></p>
<hr />
<h2
id="innovative-nucleic-acid-detection-of-clostridioides-difficile-utilizing-the-pam-unconventional-one-step-lampcrispr-cas12b-detection-platforms">Innovative
nucleic acid detection of Clostridioides difficile utilizing the
PAM-unconventional, one-step LAMP/CRISPR-Cas12b detection platforms</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionClostridioides difficile (C. difficile), a human pathogen
that causes diarrhea and colon lesions, has garnered widespread
attention. Rapid and accurate detection of bacterial virulence factors
is essential for the diagnosis of C. difficile infection (CDI). To date,
numerous laboratory tests have been developed; however, none fully meet
the combined requirements of speed, cost-effectiveness, portability,
sensitivity, and specificity. Molecular diagnostic technologies based on
CRISPR-Cas systems have provided a promising solution to this challenge.
Nonetheless, the limited compatibility between pre-amplification and
CRISPR cleavage, coupled with the inherent selectivity of CRISPR systems
for protospacer adjacent motif (PAM) sequences near the target site,
poses additional constraints on the broader adoption of this
approach.MethodsHere, we developed PAM-unconventional, One-step
LAMP/CRISPR-Cas12b (POLC) detection platforms for the toxin-encoding
genes tcdA and tcdB of C. difficile.ResultsThe POLC platforms operated
at 60 °C, enabling result interpretation either through fluorescence
intensity measurements or direct visualization under UV light. The
limits of detection (LoDs) ranged from 3 to 14 copies/μL using a
fluorescence reader and from 6 to 18 copies/μL via direct observation.
Compared to qPCR, which typically requires over an hour, the POLC
platforms reduced the detection time to approximately 40 minutes. Each
reaction cost approximately USD 6.5, offering a substantial cost saving
compared to qPCR-based commercial kits (over USD 10 per test). In
clinical validation with 55 fecal samples, the tcdA POLC assay achieved
86.4% sensitivity and 84.8% specificity, while the tcdB POLC assay
demonstrated 96.6% sensitivity and 100% specificity, using qPCR as the
reference standard.DiscussionOur research presents innovative
CRISPR-based one-step nucleic acid detection platforms that eliminate
canonical PAM sequence requirements. These platforms exhibit high
sensitivity and specificity while achieving rapid detection under simple
conditions, making them promising candidates for clinical diagnostics
and point-of-care testing (POCT).</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1594271"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1594271</a></p>
<hr />
<h2
id="micrornas-as-regulators-of-nlrp3-inflammasome-activation-in-herpes-simplex-virus-type-2-infection">MicroRNAs
as regulators of NLRP3 inflammasome activation in herpes simplex virus
type 2 infection</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionHerpes Simplex Virus Type 2 is a prevalent sexually
transmitted pathogen that causes genital herpes and severe neurological
complications, including meningitis and encephalitis. A major challenge
in HSV-2 infection is the uncontrolled inflammatory response mediated by
NLRP3 inflammasome activation, leading to pyroptosis and excessive
cytokine secretion. Despite its significant clinical burden, the
molecular mechanisms underlying HSV-2-induced inflammation remain poorly
understood. Recent evidence suggests that microRNAs play a crucial role
in regulating host immune responses and inflammasome activation. In this
study, we investigate the regulatory role of miR-141 and miR-211 in
modulating inflammasome activation and viral replication during HSV-2
infection.MethodsTHP-1-derived macrophages were transfected with miR-141
or miR-211 mimics or scrambled controls before infection with HSV-2.
Quantitative PCR and Western blot analysis were performed to assess the
expression of NLRP3, CASP1, IL-1β, IL-18, and GSDM-D. Luciferase
reporter assays were conducted to validate miRNA–target interactions,
and ELISA was used to quantify cytokine levels in culture
supernatants.ResultsOur results demonstrate that HSV-2 infection
significantly downregulates miR-141 and miR-211, leading to enhanced
NLRP3 inflammasome activation, increased caspase-1 cleavage, and
excessive secretion of IL-1β and IL-18, ultimately causing pyroptotic
cell death. Transfection with miR-141 and miR-211 mimics restored miRNA
expression, resulting in a marked suppression of inflammasome activation
and inflammatory cytokine release, as well as significant inhibition of
HSV-2 viral gene expression. Luciferase assays confirmed that miR-141
directly targets NLRP3, while miR-211 regulates CASP1, validating their
roles as post-transcriptional repressors of inflammasome
components.DiscussionThese findings establish miR-141 and miR-211 as
critical modulators of HSV-2-induced inflammasome activation,
highlighting a novel miRNA-based regulatory mechanism. Restoring these
miRNAs significantly reduces viral replication and inflammation,
underscoring their potential as therapeutic targets for managing
HSV-2-induced immunopathology. Future research should focus on in vivo
validation and therapeutic optimization to develop miRNA-based
interventions.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602965"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602965</a></p>
<hr />
<h2
id="unveiling-the-gut-heart-potential-connection-microbiotas-role-in-kawasaki-disease-and-coronary-artery-lesions">Unveiling
the gut-heart potential connection: microbiota’s role in kawasaki
disease and coronary artery lesions</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundKawasaki disease (KD) is an acute systemic immune
vasculitis predominantly affecting medium and small arteries, commonly
observed in pediatric patients. It represents the most common form of
acquired heart disease in this population. Emerging evidence suggests
that gut microbiota can modulate the gut-vascular axis, influencing
coronary artery lesions (CALs). This study aims to elucidate the
potential association between gut microbiota, KD, and CALs, as well as
identify bacterial biomarkers for CALs.MethodsWe analyzed the gut
microbiota composition of 60 children with KD (15 in the acute phase, 45
in the nonacute phase) and 30 healthy controls using alpha diversity
indices and t-tests. Microbial biomarkers were identified through LEfSe
to analyze the interplay between gut microbiota and CALs in KD during
acute and non-acute phases.ResultsIn the acute phase, KD children
exhibited decreased richness and diversity of gut microbiota,
characterized by dysbiosis, particularly a reduction in short-chain
fatty acid-producing bacteria and overgrowth of opportunistic pathogens.
Thirteen genera showed statistically significant changes, including
Enterococcus, Bacteroides, Faecalibacterium, Agathobacter,
Lachnospiraceae_NK4A136_group, Roseburia, Monoglobus, etc. LEfSe
analysis revealed enrichment of
Burkholderia-Caballeronia-Paraburkholderia, Hungatella, and
Clostridium_innocuum group in patients with concurrent CALs, while
Halomonas was depleted. In the nonacute phase, gut microbiota diversity
was similar to healthy controls, but Streptococcus was upregulated,
while Eubacterium_eligens_group and Erysipelotrichaceae_UCG_003 were
downregulated. CALs were associated with reduced Anaerostipes,
Subdoligranulum, Roseburia, and Lachnospira. LEfSe analysis also showed
enrichment of Coprobacillus, Paludicola, Lautropia, UCG-009,
Acetanaerobacterium, Methylobacterium-Methylorubrum, and Alistipes in
patients with CALs.ConclusionsOur findings provide evidence of gut
microbiota dysbiosis in KD children, suggesting its involvement in CALs
development. It indicates that under the premise of standardized
treatment during the acute phase of KD, it is plausible that
microbiota-targeted strategies may alleviate CALs, thereby improving
patient prognosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560083"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560083</a></p>
<hr />
<h2
id="studies-on-the-antibacterial-activity-of-the-antimicrobial-peptide-mastoparan-x-against-methicillin-resistant-staphylococcus-aureus">Studies
on the antibacterial activity of the antimicrobial peptide Mastoparan X
against methicillin-resistant Staphylococcus aureus</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundMethicillin-resistant Staphylococcus aureus (MRSA) poses a
serious threat to the public health system due to its multi-drug
resistance and strong biofilm-forming ability. Here, we explored the
possible inhibitory mechanism of an antimicrobial peptide, Mastoparan X,
against MRSAMethodsMinimum Inhibitory Concentration (MIC) and Minimum
Bactericidal Concentration (MBC) of Mastoparan X against MRSA USA300
were determined by microbroth dilution method. The antibacterial
activity of Mastoparan X against USA300 was then evaluated by
time-growth curves, membrane fluidity, reactive oxygen species(ROS),
flow cytometry, scanning electron microscopy (SEM) and confocal laser
scanning microscopy (CLSM). In addition, the inhibitory and scavenging
effects of Mastoparan X on USA300 biofilm were evaluated using crystal
violet staining. Finally, gene expression changes in USA300 after
treatment with Mastoparan X were analyzed by transcriptomics and
verified by RT-qPCR.ResultsThe MIC and MBC of Mastoparan X on USA300
were 32 μg/mL and 64 μg/mL, respectively. SEM observation showed
significant changes in cell morphology after Mastoparan X treatment.
Flow cytometry confirmed that Mastoparan X promoted the apoptosis of
MRSA cells. In addition, Mastoparan X inhibited the formation of MRSA
biofilm while destroying the mature bioepithelia already formed.
Transcriptomic analysis showed that 851 genes were significantly altered
and ABC transport protein, amino acid biosynthesis, glycolysis and
tricarboxylic acid (TCA) cycle were inhibited after 16μg/mL Mastoparan X
treatment.ConclusionOur study demonstrated that Mastoparan X has potent
bactericidal activity against MRSA and is expected to provide new
potential peptides for the clinical treatment of MRSA.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1552872"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1552872</a></p>
<hr />
<h2
id="epidemiological-characteristics-of-adenovirus-in-children-in-yancheng-china-2023-2024">Epidemiological
characteristics of adenovirus in children in Yancheng, China,
2023-2024</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveTo understand the epidemiology and evolutionary
characteristics of human adenovirus (HAdV) infecting children in
Yancheng, China, during the winter of 2023-2024.MethodsHAdV-positive
throat swab samples were collected from pediatric patients in Yancheng.
The Hexon, Penton base, and Fiber genes were amplified and sequenced for
typing and phylogenetic analysis of HAdV.Results170 HAdV-positive
samples were collected and identified from children with acute
respiratory infection (ARI) in Yancheng. A total of 133 samples were
successfully amplified and sequenced for Hexon, Penton base, and Fiber
genes. Phylogenetic analysis showed that HAdV-B3 was the dominant
circulating strain, and two strains of B21, one strain of C1 and one
strain of C5 were also identified.ConclusionDuring the winter of
2023-2024, HAdV infecting children in Yancheng was mainly the B3
type.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587257"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1587257</a></p>
<hr />
<h2
id="characterization-of-the-salivary-microbiome-in-healthy-individuals-under-fatigue-status">Characterization
of the salivary microbiome in healthy individuals under fatigue
status</h2>
<p>date: 2025-05-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Background &amp; AimsLimited understanding exists regarding the
characteristics and biological significance of the salivary microbiome
in healthy individuals experiencing physiological fatigue. This study
aimed to delineate the structural and functional alterations in the
salivary microbiome of healthy individuals undergoing physiological
fatigue compared to energetic controls, and to explore its potential as
a biomarker for fatigue status.MethodsA cohort of 7 healthy individuals
experiencing acute physiological fatigue (induced by prolonged study and
confirmed via electroencephalography; Fatigue group, FTG) and 63
energetic healthy controls (Energetic group, ENG) were enrolled. Saliva
samples were collected, from which microbial DNA was extracted. The
V3–V4 hypervariable region of the 16S rRNA gene was subsequently
sequenced using high-throughput technology. Bioinformatics analyses
encompassed assessment of alpha and beta diversity, identification of
differential taxa using Linear discriminant analysis Effect Size (LEfSe)
with multi-method cross-validation, construction of microbial
co-occurrence networks, and screening of fatigue-associated biomarker
genera via the Boruta-SHAP algorithm. Microbial community phenotypes and
potential functional pathways were predicted using BugBase and PICRUSt2,
respectively.ResultsThe FTG group exhibited significantly diminished
alpha diversity (Simpson index, p=0.01071) relative to the ENG group.
Beta diversity analysis demonstrated significant dissimilarities in
microbial community structure between the groups (p&lt;0.05). Taxonomic
profiling revealed a significant enrichment in the relative abundance of
potential periodontopathogenic genera, including Streptococcus and
Filifactor, within the FTG group, concomitantly with a significant
depletion of health-associated genera such as Rothia and Neisseria. A
predictive model constructed using the Boruta-SHAP algorithm, based on
15 key genera, effectively discriminated between fatigue and non-fatigue
states, achieving an area under the receiver operating characteristic
curve (AUC) of 0.948. Phenotypic predictions indicated a significant
increase in the proportion of bacteria harboring Mobile Genetic Elements
(MGEs) (p=0.048), alongside significant reductions in the proportion of
aerobic bacteria (p=0.006) and biofilm-forming capacity (p=0.002) in the
FTG group. Functional pathway analysis (PICRUSt2) revealed an enrichment
of pathways such as “Neuroactive ligand-receptor interaction” in the FTG
group, whereas pathways pertinent to energy metabolism (e.g., Citrate
cycle (TCA cycle), Oxidative phosphorylation) and amino acid metabolism
(e.g., Phenylalanine metabolism, Histidine metabolism) were
significantly enriched in the ENG group.ConclusionThis study provides
novel evidence that physiological fatigue induces significant structural
and functional alterations in the salivary microbiome of healthy
individuals. These perturbations include diminished microbial diversity,
disrupted community architecture, enrichment of potential opportunistic
pathogens, and marked shifts in key metabolic pathways, particularly
those governing neuroactivity and energy metabolism. These findings
suggest that the salivary microbiome may be implicated in the
physiological regulation of fatigue, potentially via an
“oral-microbiome-brain axis,” and underscore its potential as a source
of non-invasive biomarkers for assessing fatigue status. Further
mechanistic investigations are warranted to elucidate these
interactions.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1506723"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1506723</a></p>
<hr />
<h2
id="editorial-global-excellence-in-virology-latin-america">Editorial:
Global excellence in virology: Latin America</h2>
<p>date: 2025-05-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1623374"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1623374</a></p>
<hr />
<h2 id="editorial-advancement-in-malaria-treatment">Editorial:
Advancement in malaria treatment</h2>
<p>date: 2025-05-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1628923"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1628923</a></p>
<hr />
<h2
id="optimizing-phage-therapy-with-artificial-intelligence-a-perspective">Optimizing
phage therapy with artificial intelligence: a perspective</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Phage therapy is emerging as a promising strategy against the growing
threat of antimicrobial resistance, yet phage and bacteria are
incredibly diverse and idiosyncratic in their interactions with one
another. Clinical applications of phage therapy often rely on a process
of manually screening collections of naturally occurring phages for
activity against a specific clinical isolate of bacteria, a
labor-intensive task that is not guaranteed to yield a phage with
optimal activity against a particular isolate. Herein, we review recent
advances in artificial intelligence (AI) approaches that are advancing
the study of phage-host interactions in ways that might enable the
design of more effective phage therapeutics. In light of concurrent
advances in synthetic biology enabling rapid genetic manipulation of
phages, we envision how these AI-derived insights could inform the
genetic optimization of the next generation of synthetic phages.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1611857"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1611857</a></p>
<hr />
<h2
id="immune-pre-stimulation-by-injected-yeast-beta-glucan-as-a-strategy-to-prevent-calf-diarrhea-and-bovine-respiratory-disease-during-the-first-74-days-of-age">Immune
pre-stimulation by injected yeast beta-glucan as a strategy to prevent
calf diarrhea and bovine respiratory disease during the first 74 days of
age</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe complexity of intestinal pathogens poses a great
challenge to the prevention of infectious diarrhea in calves. Studies
have shown that immune stimulants such as yeast beta-glucan can induce
the innate immune system to acquire memory and improve their
non-specific defense functions. This trial was conducted to evaluate the
prophylactic effect of intraperitoneal injection of yeast β-glucan after
birth on diarrhea in Holstein calves during the first 74 days of
age.MethodsA total of 52 healthy newborn Holstein calves (body weight
39.3 ± 0.82 kg) were enrolled and randomly assigned into two groups (n =
26 in per group): 1) placebo group (CON), and 2) intraperitoneal
injection with yeast β-glucan solution (0.1 g/mL, 50 mg/kg body weight)
at 3 and 6 days of age (IP). The CON group received an equal volume of
sterile saline at the same time. Body weight was measured monthly, and
health checks and fecal consistency were evaluated daily for every calf.
Jugular blood and rectal feces were collected at 7 and 30 days of
age.ResultsIP induced inflammation in calves, which was manifested as
obvious increased levels of serum cytokines (IL-1β, IL-6, and TNF-a),
immunoglobulin (IgG and IgM), and oxidative stress after 24 h, and the
antimicrobial substance (defensin and secreted immunoglobulin A) in
feces also significantly increased, but stimulation didn’t lead to a
higher level of serum diamine oxidase (DAO). The pre-stimulation had no
positive effect on growth performance or feed efficiency, but reduced
the frequencies of diarrhea and bovine respiratory disease, especially
during 31-60 d. Furthermore, the pre-stimulation increased the levels of
serum IL-6, fecal defensin and secreted immunoglobulin A, while
decreased the levels of serum DAO and malonaldehyde at 30 d. In
addition, compared with the ones in the CON group, calves in the IP
group showed a better rectal bacterial structure at 30 d, with a more
enrichment of beneficial bacteria such as Bifidobacterium.DiscussionOur
findings suggested that early stimulation with yeast β-glucan could be a
promising strategy for reducing the frequencies of both diarrhea and BRD
in calves.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1586121"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1586121</a></p>
<hr />
<h2
id="biomaterials-for-bone-infections-antibacterial-mechanisms-and-immunomodulatory-effects">Biomaterials
for bone infections: antibacterial mechanisms and immunomodulatory
effects</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>As the global population continues to age, an increasing number of
individuals suffer from osteoporosis, fractures, bone infections, and
bone tumors. Among these, bone infection is considered one of the most
challenging clinical infections due to its high recurrence rate,
bacterial resistance, high incidence, and substantial treatment costs.
However, these challenges underscore the urgent need for clinicians to
develop novel therapeutic strategies to improve the current cure rate
and reduce the mortality associated with bone infections. Current
scientific research on bone infections primarily focuses on developing
new antibacterial targets and infection-resistant biomaterials. In
recent years, remarkable advancements have been made in anti-infective
biomaterials, offering promising solutions to overcome bone infections.
By optimizing the biological properties of biomaterials or integrating
them with other materials, researchers aim to achieve maximum
antibacterial efficacy and biocompatibility. Such advancements enhance
the integration of biomaterials with soft tissues, improve interactions
between bone cells and biomaterials, promote osteogenesis, and mitigate
inflammatory responses. This review primarily focuses on exploring the
antibacterial mechanisms of infection-resistant biomaterials and their
regulatory effects on the immune system, with particular emphasis on
nanoscale carriers, scaffolds, and particulate materials.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1589653"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1589653</a></p>
<hr />
<h2
id="sars-cov-2-exploits-steroidogenic-machinery-triggers-lipid-metabolism-for-viral-replication-and-induces-immune-response-in-leydig-cells-of-k18-hace2-mice">SARS-CoV-2
exploits steroidogenic machinery, triggers lipid metabolism for viral
replication and induces immune response in Leydig cells of K18-hACE2
mice</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundDuring COVID-19 pandemic, men had reduced serum
testosterone and higher mortality rate than women. Variations in high
density lipoprotein (HDL) levels were detected in severe COVID-19
individuals. We evaluated the response of testicular macrophages,
steroidogenic activity and lipid metabolism of Leydig cells in
SARS-CoV-2-infected K18-hACE2 mice.MethodsTestes were analyzed under
light and electron microscope. Immunolocalization of human angiotensin
converting enzyme (hACE2) and viral proteins (spike and nucleocapsid)
were evaluated in association with the expression of viral recognition
receptor, Rigi. Steroidogenesis was evaluated by the expression of
steroidogenic factor-1 (Sf1), and the immunolocalization of
steroidogenic proteins and testosterone. Pro-inflammatory (TNF-α, IL-1β,
IL-6), anti-inflammatory (IL-10) cytokines, macrophages (CD68 and CD163)
and macrophage inhibitory factor (MIF) were detected by
immunolocalization and Western blot. The expression of lipid metabolism
genes (Srebp, Dgat1 and Scarb1) were investigated by RT-qPCR.ResultsIn
the infected animals, the Leydig cells showed enhanced
immunolocalization of hACE2, spike and nucleocapsid. The expression of
Rigi, pro-inflammatory cytokines and number of macrophages increased,
confirming viral infection. Sf1 expression, steroidogenic proteins and
testosterone were reduced whereas the expression of Dgat1, Srebp and
Scarb1 increased. Lipid droplets-enriched Leydig cells and viral
particles in lipids were observed. The infected Leydig cells also showed
enhanced pro-inflammatory cytokines immunolabeling.ConclusionSARS-CoV-2
infects Leydig cells, activates its immune response and impairs
steroidogenesis. The virus uses the steroidogenic machinery and induces
lipid metabolism pathways for its survival and replication in these
cells. These findings support the low testosterone and HDL levels in men
with severe COVID-19.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1538461"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1538461</a></p>
<hr />
<h2
id="periodontitis-aggravates-pulmonary-fibrosis-by-porphyromonas-gingivalis-promoted-infiltration-of-neutrophils-and-th17-cells">Periodontitis
aggravates pulmonary fibrosis by Porphyromonas gingivalis-promoted
infiltration of neutrophils and Th17 cells</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionIdiopathic pulmonary fibrosis (IPF) is a fatal
interstitial lung disease. However, the pathogeny of IPF is poorly
understood, and therapeutic options are very limited. Periodontitis (PD)
is a chronic inflammatory disease that leads to dysbiosis of both the
oral microbiome and host immune responses. While previous studies have
suggested a PD-IPF association, insights into the mechanisms remain
limited.MethodsThe PD mouse model was established by the ligation of
molars and oral inoculation of subgingival plaques from PD patients and
subsequently incorporated with a bleomycin-induced pulmonary fibrosis
model. The effect of PD on pulmonary fibrosis was determined. Changes of
immune cells were analysed using flow cytometry. Moreover, the
microbiome changes of the lungs and oral cavity were assessed by 16S
rRNA gene sequencing and fluorescence in situ hybridization. Finally,
the effect and mechanism of the specific PD pathogen on pulmonary
fibrosis were determined.ResultsPD significantly aggravated pulmonary
fibrosis in mice by increasing the infiltration of neutrophils and Th17
cells. Neutrophils and Th17 cells are critical in PD-induced aggravation
of pulmonary fibrosis, and Th17 cells regulate neutrophils via IL-17A.
The PD pathogen Porphyromonas gingivalis (Pg) was detected enriched in
both the oral cavity and lungs. Pg was further determined to exacerbate
pulmonary fibrosis by increasing the expansion of neutrophils and Th17
cells in mice.ConclusionPD aggravates pulmonary fibrosis in mice, which
is likely induced by Pg-promoted infiltration of neutrophils and Th17
cells. Treatment targeting PD or Pg might be a promising strategy to
clinically ameliorate IPF.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1595500"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1595500</a></p>
<hr />
<h2
id="influenza-induced-microrna-155-expression-is-altered-in-extracellular-vesicles-derived-from-the-copd-epithelium">Influenza-induced
microRNA-155 expression is altered in extracellular vesicles derived
from the COPD epithelium</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundInfluenza virus particularly affects those with chronic
lung conditions such as Chronic Obstructive Pulmonary Disease (COPD).
Airway epithelial cells are the first line of defense and primary target
of influenza infection and release extracellular vesicles (EVs). EVs can
transfer of biological molecules such as microRNAs (miRNAs) that can
modulate the immune response to viruses through control of the innate
and adaptive immune systems. The aim of this work was to profile the EV
miRNAs released from bronchial epithelial cells in response to influenza
infection and discover if EV miRNA expression was altered in
COPD.MethodsInfluenza infection of air-liquid interface (ALI)
differentiated BCi-NS1.1 epithelial cells were characterized by
analyzing the expression of antiviral genes, cell barrier permeability
and cell death. EVs were isolated by filtration and size exclusion
chromatography from the apical surface wash of ALI cultured bronchial
epithelial cells. The EV miRNA cargo was sequenced and reads mapped to
miRBase. The BCi sequencing results were further investigated by RT-qPCR
and by using healthy and COPD primary epithelial cells.ResultsInfection
of ALI cultured BCi cells with IAV at 3.6 x 106 IU/ml for 24 h led to
significant upregulation of anti-viral genes without high levels of cell
death. EV release from ALI-cultured BCi cells was confirmed using
electron microscopy and detection of known tetraspanin EV markers using
western blot and the ExoView R100 platform. Differential expression
analyses identified 5 miRNA that had a fold change of &gt;0.6:
miR-155-5p, miR-122-5p, miR-378a-3p, miR-7-5p and miR-146a-5p
(FDR&lt;0.05). Differences between EV, non-EV and cellular levels of
these miRNA were detected. Primary epithelial cell release of EV and
their miRNA cargo was similar to that observed for BCi. Intriguingly,
miR-155 expression was decreased in EVs derived from COPD patients
compared to EVs from control samples.ConclusionEpithelial EV miRNA
release may be a key mechanism in modulating the response to IAV in the
lungs. Furthermore, changes in EV miRNA expression may play a
dysfunctional role in influenza-induced exacerbations of COPD. However,
further work to fully characterize the function of EV miRNA in response
to IAV in both health and COPD is required.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560700"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1560700</a></p>
<hr />
<h2
id="activation-of-the-wnt-signaling-pathway-and-its-role-in-epithelial-mesenchymal-transition-and-hepatic-fibrosis-in-alveolar-echinococcosis">Activation
of the Wnt signaling pathway and its role in epithelial-mesenchymal
transition and hepatic fibrosis in alveolar echinococcosis</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study examined the effects of Echinococcus
multilocularis infection on the activation of the Wnt signaling pathway
in hepatocytes, its association with epithelial-mesenchymal transition
(EMT), and its role in E. multilocularis-induced hepatic
fibrosis.MethodsHepatic lesion tissues were obtained from 20 patients
with clinically diagnosed alveolar echinococcosis (AE). These tissues
were categorized into near-lesion and distant-from-lesion groups.
Additionally, a murine model of AE infection was developed through the
injection of E. multilocularis protoscoleces. The mice were divided into
control, Wnt pathway enhancement (Wnt3a), and Wnt pathway inhibition
(DKK1) groups. Four weeks post-infection, AAV-EGFP, AAV-Wnt3a-EGFP, or
AAV-DKK1-EGFP vectors were administered, followed by tissue collection
four weeks later. Both human and murine liver tissues were analyzed
using Masson’s trichrome, hematoxylin and eosin (H&amp;E), and Sirius
red staining, as well as immunohistochemical and western blot analyses
to assess protein expression levels associated with EMT and
fibrosis.ResultsElevated expression levels of Wnt3a, β-catenin,
N-cadherin, Col1a1, α-SMA, Vimentin, CTGF, and TGF-β were observed in
tissues adjacent to human AE lesions and in the Wnt3a-treated mouse
group. Conversely, E-cadherin expression was low. Immunohistochemical
analysis demonstrated lower expression of Wnt3a, β-catenin, and other
EMT- and fibrosis-related proteins in perilesional areas in human
tissues and in the DKK1-treated mouse group, while increased E-cadherin
expression was elevated. Inflammatory cell infiltration and fibrosis
were observed near human lesions, whereas the DKK1-treated mouse group
exhibited significantly reduced fibrosis.ConclusionThe Wnt signaling
pathway plays a key role in the development of hepatic fibrosis
associated with AE infection. Its activation is positively correlated
with EMT and the increased expression of fibrogenic markers, including
Collagen I, CTGF, and TGF-β, thereby contributing to the progression of
hepatic fibrosis in hepatic AE.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1583802"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1583802</a></p>
<hr />
<h2
id="methodology-and-application-of-multiplex-pcr-dipstick-dna-chromatography-for-the-detection-of-eight-respiratory-bacterial-pathogens">Methodology
and application of multiplex PCR-dipstick DNA chromatography for the
detection of eight respiratory bacterial pathogens</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCommunity-acquired pneumonia is primarily caused by
Acinetobacter baumannii, Pseudomonas aeruginosa, Klebsiella pneumoniae,
Streptococcus pneumoniae, Haemophilus influenzae, Staphylococcus aureus,
Mycoplasma pneumoniae, and Chlamydia pneumoniae, leading to severe
illness and death in developing countries.MethodsA rapid,
straightforward, sensitive, high-throughput, and precise multiplex
PCR-dipstick DNA chromatography assay was devised. This innovative
technique was specifically engineered for the immediate and efficient
detection of the aforementioned eight respiratory pathogens, with
particular emphasis on scenarios involving co-infections.
Custom-designed specific primers were employed, wherein the 5′ end of
the forward primers was tagged with oligonucleotide tags (Tag) and the
5′ end of the reverse primers was conjugated with biotin. A C3 spacer
was incorporated to bridge the Tag and the forward primer. Complementary
oligonucleotides (cTag) corresponding to each of the eight pathogens
were immobilized within the test area of the test strip. Meanwhile,
biotin was strategically utilized to create an internal control line at
the distal end of the test strip. The biotin moiety at the 5′ end of the
reverse primer was engineered to interact with blue latex microspheres
coated with streptavidin, thereby triggering a detectable signal.
Following the PCR amplification of the target DNA fragments, during the
membrane strip chromatography hybridization process, the Tag- and
biotin-labeled target DNA engaged in a dual interaction. First, it bound
to the blue latex microspheres via streptavidin–biotin binding, and
second, it hybridized with the cTag on the membrane strip. This led to
the accumulation of captured blue latex microspheres at both the test
line and the internal control line, manifesting as visible blue bands. A
total of 186 respiratory sputum or bronchoalveolar lavage fluid
specimens were collected and analyzed. The multiplex PCR-dipstick DNA
chromatography assay was deployed for detection, while traditional
bacterial culture was also carried out in parallel for comparative
purposes. To rigorously validate the accuracy of the multiplex
PCR-dipstick DNA chromatography assay in identifying PCR products, DNA
sequencing was performed on all PCR products derived from the clinical
samples.ResultsThe multiplex PCR-dipstick DNA chromatography assay
demonstrated remarkable efficacy, being capable of specifically
discriminating among the eight pathogens within a remarkably short
timeframe of 40 minutes. The detection limit for individual bacterial
species ranged from 10 to 102 CFU/mL. Notably, no cross-reactions were
observed among the eight target bacteria, nor with other representative
respiratory bacteria, including Acinetobacter junii, Enterobacter
cloacae, Klebsiella oxytoca, Haemophilus parainfluenzae, Pseudomonas
fluorescens, Aeromonas hydrophila, and Staphylococcus epidermidis. The
concordance between the results obtained from the multiplex PCR-dipstick
DNA chromatography assay and those from DNA sequencing was absolute,
with a kappa value of 1.00.ConclusionA successful multiplex PCR-dipstick
DNA chromatography assay was established for the simultaneous detection
of eight respiratory bacterial pathogens and was effectively applied in
clinical sample analysis. This indicates that this single-use device has
promising potential for analyzing the microbial composition related to
respiratory infections and is also suitable for small laboratories and
field diagnostics.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1558612"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1558612</a></p>
<hr />
<h2
id="decoding-respiratory-syncytial-virus-morphology-distinct-structural-and-molecular-signatures-of-spherical-and-filamentous-particles">Decoding
respiratory syncytial virus morphology: distinct structural and
molecular signatures of spherical and filamentous particles</h2>
<p>date: 2025-05-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Respiratory syncytial virus (RSV) is a pleomorphic enveloped virus
that buds as both spherical and filamentous particles. The determinants
of RSV particle morphology and the roles of these forms in transmission
and pathogenicity are not clearly defined, owing to a complex interplay
of viral proteins and host factors that remains poorly understood. To
further characterize RSV morphology, we developed a sucrose gradient
velocity sedimentation method to separate spherical and filamentous
virions. Fluorescence microscopy and electron microscopy (EM) confirmed
two distinct peaks containing predominantly spherical or filamentous
particles, respectively. Notably, EM images revealed a distinctive
“honeycomb” pattern on the RSV envelope, suggesting an ordered lattice
of glycoproteins on the virion surface. Biochemical analyses of viral
protein and lipid content showed that filamentous particles contained
higher levels of uncleaved fusion protein F0 and exhibited distinct
phospholipid profiles compared to spherical particles. Both forms were
enriched in cholesterol and phospholipids characteristic of lipid rafts,
consistent with the idea that RSV buds from lipid raft microdomains.
This enrichment in raft lipids is linked to cell-to-cell fusion
(syncytium) formation and virion assembly. Quantitative real-time PCR
analysis indicated a high particle-to-PFU ratio (~4:1), meaning only
about one in four RSV virions was infectious. Spherical particles
contained on average ~3 genomic RNA copies per virion, whereas
filamentous particles contained ~2 copies. These data reveal several
structural and compositional differences between RSV particle
morphologies that may influence viral pathogenesis, and they provide a
foundation for new antiviral approaches targeting virion assembly and
morphology.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1597279"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1597279</a></p>
<hr />
<h2
id="a-novel-loss-of-function-variant-in-stat1-causes-mendelian-susceptibility-to-mycobacterial-disease">A
novel loss-of-function variant in STAT1 causes Mendelian susceptibility
to mycobacterial disease</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMendelian Susceptibility to mycobacterial disease (MSMD)
is a rare inherited immunodeficiency disorder characterized by increased
susceptibility to atypical mycobacterial infections induced by defective
IFN-γ pathway.MethodsWe report three patients from a family presenting
with multiple osteolytic lesions and cutaneous granulomas due to
Mycobacterium marinum infections. Functional studies, including Western
blotting and immunofluorescence, assessed phosphorylation and nuclear
translocation of the mutant STAT1-Ile707Thr in eukaryotic overexpression
systems. A luciferase reporter assay evaluated its transcriptional
activity. Additionally, structural analysis using AlphaFold3 predicted
the variant’s functional impact.ResultsA novel STAT1 variant
(c.2120T&gt;C, p.Ile707Thr) was identified. The STAT1-Ile707Thr mutant
exhibited reduced phosphorylation and impaired nuclear translocation
compared to wild-type STAT1. The luciferase assay confirmed decreased
transcriptional activity. AlphaFold3-based cluster analysis supported a
loss-of-function effect of the mutant.DiscussionThis study expands the
spectrum of STAT1 variants and microbial pathogens associated with
MSMD.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1595389"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1595389</a></p>
<hr />
<h2
id="the-relationship-between-virulence-and-drug-resistance-genes-in-pseudomonas-aeruginosa-and-antibiotic-resistance-a-targeted-next-generation-sequencing-approach">The
relationship between virulence and drug resistance genes in Pseudomonas
aeruginosa and antibiotic resistance: a targeted next-generation
sequencing approach</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivePseudomonas aeruginosa is an opportunistic pathogen
responsible for nosocomial infections in critically ill and
immunocompromised patients. To elucidate genomic-phenotypic-clinical
correlations in P. aeruginosa infections, this study integrates targeted
next-generation sequencing (tNGS) with conventional diagnostics.MethodsA
retrospective study was conducted between September, 2023 and December,
2024 at The Second Affiliated Hospital of Guangxi Medical University on
patient specimens that were subjected to both conventional culture and
tNGS testing. Only a single Gram-negative bacterium, P. aeruginosa, was
detected by tNGS, and these isolates were subsequently analyzed in order
to compare their genomic profiling (virulence/resistance genes),
antimicrobial susceptibility testing (AST), and clinical
characteristics.ResultsA total of 105 samples were included in the
study. Exoenzyme Y gene (exoY) and O-antigen polymerase gene (wzy) were
detected in 77.1% and 12.4% of the samples, respectively. Meanwhile,
aminoglycoside acetyltransferase gene (aac(6’)aac(3’)), aminoglycoside
resistance methyltransferase gene (armA), and chloram phenicol
resistance genes (cmlA) were detected in 6.7%, 4.8%, and 12.4% of the
samples, respectively. P. aeruginosa isolates detected the exoY gene
exhibit a higher level of drug resistance, particularly to cefepime
(32.1% vs 4.2%, p&lt;0.05) and piperacillin-tazobactam (33.3% vs 8.3%,
p&lt;0.05). aac(6’)aac(3’) and armA genes were statistically associated
with tobramycin resistance (p&lt;0.05). The use of antibiotics before
hospitalization (within 90 days), hospital-acquired infections, ICU
admission, as well as pre-P. aeruginosa detection interventions
including invasive procedures, catheterization, mechanical ventilation,
duration of antibiotic use time≥14 days, and antibiotic use type ≥3
types (p&lt;0.05) is associated with multidrug-resistant (MDR) and
carbapenem-resistant P. aeruginosa (CRPA) infections (p&lt;0.05).
Furthermore, the detection of MDR and CRPA strains appears to lead to an
increase in the duration and variety of antibiotic use, as well as
prolonged of hospital stay (p&lt;0.05).ConclusionOur study highlights
the importance of integrating tNGS results, which provide insights into
pathogen identification, resistance, and virulence genes, with
phenotypic and clinical data in order to enhance the accuracy of
diagnosis and guide treatment strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563741"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1563741</a></p>
<hr />
<h2
id="chloroquine-induces-eryptosis-in-p.-falciparum-infected-red-blood-cells-and-the-release-of-extracellular-vesicles-with-a-unique-protein-profile">Chloroquine
induces eryptosis in P. falciparum-infected red blood cells and the
release of extracellular vesicles with a unique protein profile</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMalaria is a vector-borne parasitic disease that affects
millions worldwide. To achieve the objective set by the World Health
Organization of reducing malaria cases by 2030, antimalarial drugs with
novel modes of action are required. Previously, a novel mechanism of
action of chloroquine (CQ) was reported involving features of programmed
cell death in the parasite, mainly characterized by calcium efflux from
digestive vacuole permeabilization. Increased intracellular calcium
induces suicidal death of erythrocytes, a process known as eryptosis.
This study aimed to identify the hallmarks of eryptosis due to calcium
redistribution and examine the downstream cellular effects during CQ
treatment in infected red blood cells (iRBCs).MethodsSynchronized
Plasmodium falciparum 3D7 cultures at mid-late trophozoites were treated
with CQ and other antimalarial compounds for 10 hours. Eryptotic
markers, including phosphatidylserine (PS) exposure, cell shrinkage and
membrane blebbing, were assessed by flow cytometry, scanning electron
microscopy and western blot, respectively. Extracellular vesicles (EVs)
were isolated from 3D7 malaria culture supernatants using differential
ultracentrifugation, followed by their physical and proteomic
characterization. THP-1-derived macrophages were stimulated with EVs to
determine parasite-host interactions, as indicated by cytokine levels
and transcriptomic analysis.ResultsIncreased PS exposure, cell
shrinkage, and membrane blebbing were observed, delineating an eryptotic
phenotype in the host RBCs. Notably, the outward budding and blebbing of
the iRBC plasma membrane led to the formation of EVs, which are complex
structures with unique functional properties. Proteomic characterization
of EVs from CQ-treated iRBCs revealed a high enrichment of proteasome
and ribosome protein clusters. This unique EV cargo did not influence
the parasite growth rate but might activate IFN signaling pathways
mediated by IL-6 in THP-1-derived macrophages.ConclusionThese findings
provide new insights into a novel drug-induced cell death mechanism that
targets the parasite and specific components of the infected host
RBC.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1553123"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1553123</a></p>
<hr />
<h2
id="long-term-alterations-in-gut-microbiota-following-mild-covid-19-recovery-bacterial-and-fungal-community-shifts">Long-term
alterations in gut microbiota following mild COVID-19 recovery:
bacterial and fungal community shifts</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveCOVID-19 has had a profound impact on public health
globally. However, most studies have focused on patients with long COVID
or those in the acute phase of infection, with limited research on the
health of individuals who have recovered from mild COVID-19. This study
investigates the long-term changes in bacterial and fungal communities
in individuals recovering from mild COVID-19 and their clinical
relevance.MethodsHealthy individuals from Hainan Province were enrolled
before the COVID-19 outbreak, along with individuals recovering from
COVID-19 at 3 months and 6 months post-recovery. Stool, blood samples,
and metadata were collected. Metagenomic sequencing and Internal
Transcribed Spacer (ITS) analysis characterized bacterial and fungal
communities, while bacterial-fungal co-occurrence networks were
constructed. A random forest model evaluated the predictive capacity of
key taxa.ResultsThe gut microbiota of COVID-19 recoverees differed
significantly from that of healthy individuals. At 3 months
post-recovery, probiotics (e.g., Blautia massiliensis and Kluyveromyces
spp.) were enriched, linked to improved metabolism, while at 6 months,
partial recovery of probiotics (e.g., Acidaminococcus massiliensis and
Asterotremella spp.) was observed alongside persistent pathogens (e.g.,
Streptococcus equinus and Gibberella spp.). Dynamic changes were
observed, with Acidaminococcus massiliensis enriched at both baseline
and 6 months but absent at 3 months. Co-occurrence network analysis
revealed synergies between bacterial (Rothia spp.) and fungal
(Coprinopsis spp.) taxa, suggesting their potential roles in gut
restoration. The bacterial random forest model (10 taxa) outperformed
the fungal model (8 taxa) in predicting recovery status (AUC = 0.99
vs. 0.80).ConclusionThese findings highlight the significant long-term
impacts of mild COVID-19 recovery on gut microbiota, with key taxa
influencing metabolism and immune regulation, supporting
microbiome-based strategies for recovery management.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565887"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1565887</a></p>
<hr />
<h2
id="stage-specific-mcm-protein-expression-in-trypanosoma-cruzi-insights-into-metacyclogenesis-and-g1-arrested-epimastigotes">Stage-specific
MCM protein expression in Trypanosoma cruzi: insights into
metacyclogenesis and G1 arrested epimastigotes</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Trypanosoma cruzi is a protozoan parasite that is the etiological
agent of Chagas disease, which is endemic to Latin America with reported
cases in non-endemic regions such as Europe, Asia, and Oceania due to
migration. During its lifecycle, T. cruzi alternates between replicative
and non-replicative infective lifeforms. Metacyclogenesis is the most
studied transition of the T. cruzi life cycle, where replicative
epimastigotes differentiate into infective metacyclic trypomastigotes
inside the gut of the triatomine vector. This early-branching organism
expresses a divergent pre-replication complex (pre-RC) where the only
conserved component is the MCM2–7 protein family. Given the role of
pre-RC components in cell cycle regulation, we investigated whether MCM
expression and location could be involved in proliferation control in
epimastigotes and during metacyclogenesis. Using CRISPR/Cas9, we tagged
MCM subunits and tracked their expression and subcellular localization.
Our findings reveal that MCM subunits are consistently expressed and
localized to the nucleus throughout the epimastigote cell cycle,
including in G1/G0-arrested cells. However, MCM subunits are degraded
during metacyclogenesis as cells enter the G0 state, marking the
transition to replication arrest. Therefore, epimastigotes arrested in
G1/G0 can either maintain MCM complex expression and resume the cell
cycle when conditions become favorable, or they can undergo
metacyclogenesis, exiting the cell cycle and entering a G0 state, where
MCM subunits are degraded as part of the replication repression
mechanism.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584812"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1584812</a></p>
<hr />
<h2
id="editorial-exploring-sars-cov-2-inflammatory-responses-and-potential-targets-for-treatment">Editorial:
Exploring SARS-CoV-2 inflammatory responses and potential targets for
treatment</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1625563"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1625563</a></p>
<hr />
<h2
id="comparative-of-metagenomic-and-targeted-next-generation-sequencing-in-lower-respiratory-tract-fungal-infections">Comparative
of metagenomic and targeted next-generation sequencing in lower
respiratory tract fungal infections</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesThis study aims to compare the diagnostic efficiency and
consistency of metagenomic next-generation sequencing (mNGS) and
targeted next-generation sequencing (tNGS) in patients with lower
respiratory tract fungal infections.MethodsA total of 115 patients with
probable pulmonary infection between September 2022 and April 2023 were
enrolled at the Second Xiangya Hospital, Changsha, China, of which 61
were clinically diagnosed with invasive pulmonary fungal infection
(IPFI) and 54 were non-IPFI cases. All patients received bronchoalveolar
lavage, with mNGS, tNGS, and cultures being conducted paralleled.
Diagnostic effectiveness and consistency in detecting microorganisms
were compared.ResultsBoth mNGS and tNGS showed high sensitivity rates of
95.08% each, with specificity of 90.74% and 85.19%, respectively. They
also demonstrated positive predictive values (PPVs) of 92.1% and 87.9%
and negative predictive values (NPVs) of 94.2% and 93.9%, respectively,
in diagnosing IPFI. The sensitivity and NPV of mNGS and tNGS were
superior to that of any individual or combined conventional
microbiological tests (CMTs) (P &lt; 0.05). The consistency of culture
with mNGS and tNGS was 48.70% and 50.43%, respectively. For fungal
detection, Pneumocystis jirovecii (26/61, 42.6%; and 28/61, 45.9%),
Candida albicans (19/61, 31.1%; and 21/61, 34.4%), and Aspergillus
fumigatus (16/61, 26.2%; and 15/61, 24.6%) are most prevalent for mNGS
and tNGS in enrolled cases, and the detection rate was greatly higher
than that of culture. Furthermore, mNGS and tNGS were capable of
diagnosing mixed infections in 65 and 55 out of the 115 cases, whereas
only nine cases of bacterial-fungal infection were detected by
culture.ConclusionThe diagnostic efficacy of mNGS and tNGS was
comparable to that of identified IPFI. NGS-based methodologies present a
promising tool for detecting IPFI, which can be a good supplement to
CMT.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534519"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1534519</a></p>
<hr />
<h2
id="fecal-16s-rrna-sequencing-and-metabolomics-reveal-abnormal-metabolism-activity-in-preterm-infants-with-different-gestational-ages">Fecal
16S rRNA sequencing and metabolomics reveal abnormal metabolism activity
in preterm infants with different gestational ages</h2>
<p>date: 2025-05-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study aims to conduct a comprehensive analysis of the
differences in gut microbiota and metabolomics in preterm infants
stratified by gestational age.MethodsFresh fecal samples were collected
from neonates within the first 3 days after birth. The gut microbiota
composition and the changes in specific taxa abundance were analyzed
using 16S rRNA sequencing. Metabolomic profiling was performed using
liquid chromatography-tandem mass spectrometry (LC-MS/MS). Participants
were categorized into four groups based on gestational age at birth:
PreA group (34–36 weeks), PreB group (32–33 weeks), PreC group (28–31
weeks), and control group (37–42 weeks). Metabolic pathways were
identified through metabolomics analysis, referencing the Kyoto
Encyclopedia of Genes and Genomes (KEGG) database.ResultsNotably,
Principal Coordinates Analysis (PCoA) showed clear separation among
samples from all groups, with significant differences noted in the PreC
group when compared with the other three. We found a strong association
between Escherichia-Shigella and Ureaplasma genera with infants born
before 32 weeks of gestation, suggesting a higher risk of opportunistic
infections for preterm infants under this gestational threshold. As
gestational age increases, Megamonas and Prevotella gradually emerged,
while Escherichia-Shigella and Ureaplasma progressively diminished. KEGG
enrichment analysis indicated that Pyrimidine metabolism was a
differentially regulated pathway between the PreA group and the control
group. Interestingly, the only major differential metabolic pathway
between the PreB group and the control group was Arachidonic acid
metabolism. The bubble diagram revealed significant enrichment of
differential metabolites in Pyrimidine and beta-Alanine metabolism
pathways when comparing the PreC group with the control
group.ConclusionSignificant differences were observed in the fecal
microbiome and metabolome between preterm and full-term infants,
particularly in those born before 32 weeks of gestation. These findings
suggested that the gut microbial system in preterm infants undergone
progressive maturation, approaching a “healthy” state characteristic of
full-term infants as gestational age increases.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2025.1530653"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2025.1530653</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
