---
title: health
updated: 2025-07-29 06:09:19
---

# health

(date: 2025-07-29 06:09:19)

---

## Evaluation of molecular mouse sepsis panel: new portable and rapid tests for microorganism detection in suspected blood stream infection

date: 2025-07-29, from: Frontiers in Cellular and Infection Microbiology

IntroductionBloodstream infections (BSIs) represent a significant public health concern, characterized by the presence of pathogens in the bloodstream, leading to serious conditions. Between 30 and 40% of BSIs evolve in sepsis, characterized by a life-threatening organ dysfunction and accompanied by a strong or dysregulated systemic immune or inflammatory response. A timely and accurate diagnosis of BSIs is crucial for ensuring effective management and treatment of critically ill patients. The diagnosis of BSIs can be initiated either from positive blood culture (BC) or directly from blood specimens using advanced diagnostic methods, these approaches are increasingly integrated into traditional diagnostic workflows.MethodsIn the context of emerging molecular methods, Alifax introduced the Molecular Mouse (MM) Sepsis Panel in 2022. This study aims to evaluate the performance of MM cartridges in detecting Gram-positive bacteria and Gram-negative bacteria, as well as their associated resistance genes, using clinical leftover positive BCs and contrived samples for low-prevalence targets. Positive BCs were collected at the Unit of Microbiology, The Great Romagna Hub Laboratory, Pievesestina, FC, Italy. ResultsOverall, MM Sepsis Panel showed excellent sensitivity (98.62%) and specificity (99.81%), confirming the system’s reliability and accuracy. DiscussionThe high diagnostic accuracy demonstrated by these tests supports their potential application as rapid diagnostic tools for blood cultures, facilitating prompt identification of pathogens and antimicrobial resistance determinants in clinical microbiology settings.  

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1579074>

---

## Detecting early kidney injury due to host–virus interaction response in treatment-naive CHB—a pilot study

date: 2025-07-29, from: Frontiers in Cellular and Infection Microbiology

PurposeClinical evidence suggests that patients with chronic hepatitis B (CHB) have an increased risk of renal impairment due to inflammation induced by virus–host interaction. We aimed to evaluate and validate a set of protein biomarkers singularly and in combination for the early detection of subclinical kidney injury in patients with CHB naive to antiretroviral therapy.MethodsThis work is part of a prospective cross-sectional study for which 69 HBsAg-positive, treatment-naive patients with CHB with an equal number of age-matched healthy volunteers were considered. At diagnosis, serum creatinine (sCr), urea, alanine transaminase, aspartate transaminase, serum cystatin-C (sCys-C), serum neutrophil gelatinase-associated lipocalin (sNGAL), serum Fetuin-A (sFet-A), urinary interleukin-18 binding protein (uIL-18BP), and urinary kidney injury molecule-1 (uKIM-1) levels were determined.ResultsThere was a significant elevation in the concentrations of three proteins in our CHB cohort (sCys-C, sNGAL, and uIL-18BP; p < 0.0001) while sFet-A was down-regulated (p<0.01) as compared to the control group. A receiver operating characteristic curve analysis revealed an Area under the curve of 0.935 for sCys-C and 0.811 for sNGAL, which improved to 0.984 when all four indicators were combined in a panel to discriminate the onset of renal injury incited by inflammatory response in CHB with 97.1% sensitivity at 91.3% specificity. Additionally, only sCys-C and sNGAL differed significantly among the phases of CHB infection (p<0.05). ConclusionsThis novel noninvasive diagnostic screen is expedient in detecting inflammation and early kidney injury before a rise in sCr and can aid in predicting renal outcomes in patients with CHB. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1601678>

---

## Global research trends on the interaction between gut microbiome and bile acids: a bibliometric and visualized analysis

date: 2025-07-29, from: Frontiers in Cellular and Infection Microbiology

BackgroundAn increasing number of studies have shown that gut microbiome-bile acids interactions play a crucial role in host health and disease. This bibliometric analysis aims to identify the global scientific output, research hotspots, and frontiers of gut microbiome-bile acids in the past two decades.MethodsWe searched the relevant studies of gut microbiome-bile acids published between 2004 and 2024 in the Web of Science Core Collection database. Microsoft Excel 2019, VOSviewer 1.6.18, Tableau Desktop 2024.2.2, Scimago Graphica 1.0.45, and CiteSpace 6.2.R3 were used to analyze the publications, countries/regions, institutions, journals, authors, references, and keywords.ResultsA total of 4795 original articles and reviews were collected. A visual analysis of the results showed that the number of publications increased rapidly over time. China published the most papers, the United States had the most citations, and the most productive institution was Shanghai Jiaotong University. The most prolific author was Jia Wei, and Jason M. Ridlon was the most frequently co-cited author. Nutrients was the most productive journal. In the keyword co-occurrence network, except for gut microbiome and bile acids, inflammation becomes the keyword with the highest frequency. Keywords and reference analysis show that metabolic diseases (such as obesity and type 2 diabetes mellitus), cancer (such as colorectal cancer), and disease-related mechanisms (such as tgr5 and pathway) are the hot topics and future research trends in this field.ConclusionIn this study, bibliometric analysis was utilized to explore the relationship between gut microbiome and bile acids. The findings can reflect the current hotspots and new directions of gut microbiome-bile acids, and provide an objective description and comprehensive guidance for future related studies. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616995>

---

## Characterization of key spike RBD residues influencing SARS-CoV-2 variant adaptation to avian ACE2

date: 2025-07-29, from: Frontiers in Cellular and Infection Microbiology

IntroductionThe beta-coronavirus SARS-CoV-2 has been revealed to infect mammals and other species, which potentially promotes the virus adaptation to broader species and the emergence of new variants. The host range of different SARS-CoV-2 variants are mainly determined by the affinity of the receptor-binding domain (RBD) of the spike protein to the host receptor angiotensin-converting enzyme 2 (ACE2). Thus, this study aims to elucidate the detailed mechanisms of such dynamic adaptation of indicated SARS-CoV-2 variants.MethodsIn this study, flow cytometry and surface plasmon resonance (SPR) assays were used to assess the binding affinity between RBDs and avian ACE2. Then, infection assays with MLV-based SARS-CoV-2 spike pseudovirus or authentic viruses were performed to verify the avian ACE2 mediated viral entry. Finally, mutagenesis studies were conducted to identify key amino acids of avian ACE2 orthologs and RBDs.ResultsOur previous findings revealed that wild-type SARS-CoV-2 RBD does not bind chicken ACE2. Here, we found that ACE2 orthologs from chicken and mallard were capable to support binding to RBDs of the Alpha, Beta, and Gamma variants, which further enabled the viral entry. On the contrary, the RBD of BA.1 failed to bind avian ACE2. Whereas, a triple-residue reversal mutant (S446G, S496G, H505Y) restored ACE2 binding and enabled efficient viral entry. Additionally, several key residues within RBD were characterized as the determinant of its affinity to avian ACE2.DiscussionOur findings reveal that higher mutation rates in emerging variants might lead to future cross-species receptor usage or even spillover. Understanding such cross-species transmission mechanisms provides new insights to the virological features and potential host range of emerging SARS-CoV-2 variants. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1631926>

---

## The predictive value of SOFA and APSIII scores for 28-day mortality risk in SIMI: a cohort study based on the MIMIC-IV database

date: 2025-07-29, from: Frontiers in Cellular and Infection Microbiology

ObjectiveThe objective of this study was to systematically identify and evaluate scoring systems that predict the prognosis of patients with sepsis-induced myocardial injury (SIMI).MethodsData were retrieved from the Medical Information Mart for Intensive Care IV (MIMIC-IV) database. Logistic and Cox regression analyses were conducted to identify risk factors associated with 28-day mortality in patients with sepsis-induced myocardial injury (SIMI). The prognostic performance of the scoring systems was comprehensively assessed using receiver operating characteristic (ROC) curves, Kaplan-Meier survival and decision curve analysis (DCA).ResultsLogistic regression analysis showed that Acute Physiology Score III (APSIII) (Odds Ratio [OR] =1.011, 95% Confidence Interval [CI] 1.002–1.018, P=0.005), Sequential Organ Failure Assessment (SOFA) (OR =1.097, 95% CI 1.045–1.144, P<0.001), and Charlson Comorbidity Index (CHARLSON) (OR=1.095, 95% CI 1.048–1.145, P=0.036) were independent risk factors for 28-day mortality in SIMI patients. Cox regression analysis confirmed that SOFA (HR=1.082, 95% CI 1.054– 1.111, P<0.001), APSIII (HR=1.010, 95% CI 1.005–1.015, P<0.001), and CHARLSON (HR=1.044, 95% CI 1.012–1.077, P=0.007) were independent risk factors. ROC curve analysis showed that SOFA (AUC=0.685, 95% CI 0.663–0.707) and APSIII (AUC=0.683, 95% CI 0.662–0.705) had significantly higher AUC values compared to other scoring systems. DCA results showed that APSIII and SOFA had better net benefit than other scoring systems.ConclusionsThe SOFA and APSIII scores effectively identified high-risk patients with SIMI, providing evidence-based support for early clinical intervention. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1574625>

---

## Low Sodium Chicken Soup

date: 2025-07-28, from: The Kidney Dietitian

<p>This article was written by dietetic student Emma Kortmansky &#38; reviewed by Melanie Betz MS, RD, CSR, FNKF, FAND. I don’t know about you, but in my opinion, nothing says comfort food quite like [&#8230;]</p>
<p>The post <a href="https://www.thekidneydietitian.org/low-sodium-chicken-soup/">Low Sodium Chicken Soup</a> appeared first on <a href="https://www.thekidneydietitian.org">The Kidney Dietitian</a>.</p>
 

<br> 

<https://www.thekidneydietitian.org/low-sodium-chicken-soup/>

---

## Blood microbiome signatures in systemic diseases: current insights, methodological pitfalls, and future horizons

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

The human-associated microbiome, encompassing diverse microbial communities across body sites, plays a pivotal role in maintaining host homeostasis. Disruption of this balance, termed dysbiosis, has been implicated in a spectrum of pathophysiological conditions. Traditionally, blood was considered a sterile microenvironment. However, emerging insights into the blood microbiome challenge the paradigm of blood sterility, revealing microbial signatures, including cell-free DNA and viable taxa, with putative implications for host physiology and disease. The blood taxonomic profile at the phylum level is dominated by Proteobacteria, with Bacteroidetes, Actinobacteria, and Firmicutes following in abundance. Dysbiosis in blood microbiome composition may indicate or contribute to systemic dysregulation, pointing to its potential role in disease etiology. These findings highlight the blood microbiome as a possible driver in the pathogenesis of infectious and non-infectious diseases, neurodegenerative disorders, and immune-mediated conditions. The detection of specific microbial profiles in circulation holds promise for biomarker discovery, enhancing disease stratification, and informing precision therapeutic strategies. However, advancing this field requires overcoming methodological challenges, including contamination control, standardization, and reproducibility. This review aims to present blood microbiome biomarkers across infectious, non-infectious, neurodegenerative, and immune-mediated diseases, while critically examining methodological variations, controversies, limitations, and future research directions. Elucidating these factors is critical to advancing blood microbiome biomarker validation and therapeutic targeting, thereby refining mechanistic insights into systemic disease pathogenesis. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1616029>

---

## Integrated oral-gut microbiota therapy: a novel perspective on preventing bacterial translocation for systemic disease management

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

Oral dysbiosis increases the risk of oral diseases and systemic diseases, with many related conditions overlapping with systemic diseases triggered by gut dysbiosis. Studies have shown that the oral cavity serves as an endogenous reservoir for gut microbial strains, influencing the homeostasis of both oral and gut microbiota through interactions involving bacterial translocation, microbial metabolites, immune cells, and inflammatory factors. In specific disease contexts, certain microbial communities [e.g., Porphyromonas gingivalis (P.g), Fusobacterium nucleatum (F.n)], metabolites (e.g., short-chain fatty acids, gingipains), ligands (e.g., lipopolysaccharides, peptidoglycans), or host responses may vary. However, substantial evidence has firmly established the central role of microbiota in oral-gut crosstalk. These findings position the oral-gut axis as a potential causal mechanism linking systemic diseases. Compared with healthy non-cancer subjects, cancer patients exhibit significant differences in oral microbial abundance and diversity. For instance, F.n is associated with an increased risk of colorectal cancer(CRC), while Oribacterium and Fusobacterium may serve as potential biomarkers for hepatocellular carcinoma. Notably, oral pathogens or their metabolites can translocate along the oral-gut axis or due to certain oral activities (e.g., toothbrushing, tooth extraction), contributing to the initiation and progression of inflammation and tumorigenesis. For example, P.g can accumulate in the liver, where its fimbrial protein FimA binds to Toll-like receptor 2 (TLR2), complement receptor 3 (CR3), and CXC-chemokine receptor 4 (CXCR4), triggering various immune responses that promote the development of non-alcoholic fatty liver disease(NAFLD). This review systematically summarizes recent advances in understanding the role of the oral microbiota and the oral-gut axis in systemic diseases, along with their underlying pathological mechanisms. It particularly highlights the translational value of integrating oral and gut microbiota research, offering novel insights for the prevention and precision treatment of systemic disorders. The unique and heterogeneous microbiota within the oral microbiota and the oral-gut axis may serve as novel diagnostic biomarkers or therapeutic targets for diseases associated with oral and gut dysbiosis. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1641816>

---

## Exploring fecal microbiota signatures associated with immune response and antibiotic impact in NSCLC: insights from metagenomic and machine learning approaches

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

BackgroundSubstantial interstudy heterogeneity in cancer immunotherapy-associated biomarkers has hindered their clinical applicability. To address this challenge, we performed a comprehensive integration of publicly available global metagenomic datasets. By leveraging metagenomic profiling and machine learning approaches, this study aimed to elucidate gut microbial signatures associated with immune response in lung cancer (LC) and to evaluate the modulatory effects of antibiotic exposure.MethodsA systematic literature search was conducted to identify relevant datasets, resulting in the inclusion of 209 fecal metagenomic samples: 154 baseline samples (45 responders, 37 non-responders, and 72 healthy controls) and 55 longitudinal samples collected during immunotherapy. We performed taxonomic and functional characterization of gut microbiota (GM) differentiating responders from non-responders, delineated microbiome dynamics during treatment, and assessed the impact of antibiotics on key microbial taxa. Among eight machine learning algorithms evaluated, the optimal model was selected to construct a predictive framework for immunotherapy response.ResultsMicrobial α-diversity was significantly elevated in responders compared to non-responders, with antibiotic administration further amplifying this difference—most notably at the species level. Integrative multi-omics analysis identified two pivotal microbial biomarkers, s_Bacteroides caccae and s_Prevotella copri, which were strongly associated with immunotherapy efficacy. A random forest-based classifier achieved robust predictive performance, with area under the curve (AUC) values of 0.82 and 0.79 at the species and genus levels, respectively. Notably, P. copri was further enriched in responders with poor progression-free survival (PFS <3 months), indicating a potential deleterious role. Antibiotic exposure significantly influenced the abundance and functional potential of these key taxa. KEGG-based functional analysis revealed the enrichment of amino acid metabolism pathways in responders. Additionally, CARD database annotation demonstrated that the majority of antibiotic resistance genes were associated with Bacteroidetes and Proteobacteria, implicating these taxa in shaping microbial-mediated therapeutic responses.ConclusionsThis study represents the first large-scale, cross-cohort integration of metagenomic data to identify reproducible GM signatures predictive of immune checkpoint inhibitor efficacy in LC. The findings not only underscore the prognostic relevance of specific taxa but also establish a foundation for developing microbiome-informed, personalized immunotherapeutic strategies. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1591076>

---

## Prediction of bacteremia using routine hematological and metabolic parameters based on logistic regression and random forest models

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

BackgroundThis study aimed to evaluate the predictive utility of routine hematological, inflammatory, and metabolic markers for bacteremia and to compare the classification performance of logistic regression and random forest models.MethodsA retrospective study was conducted on 287 inpatients who underwent blood culture testing at Fuding Hospital, Fujian University of Traditional Chinese Medicine between March and August 2024. Patients were divided into bacteremia (n = 137) and non-bacteremia (n = 150) groups based on blood culture results. Hematological indices, inflammatory markers (e.g., C-reactive protein (CRP), procalcitonin (PCT)), metabolic indices (e.g., glucose, cholesterol) and nutritional markers (e.g., albumin) were analyzed. Univariate and multivariate binary logistic regression analyses were used to identify independent risk factors. Logistic regression and random forest models were developed using 33 features with a 70:30 train-test split and evaluated using the receiver operating characteristic (ROC) curves, confusion matrices and standard classification.ResultsHemoglobin, cholesterol, and albumin levels were significantly lower in the bacteremia group, while platelet count, CRP, PCT, glucose, and triglycerides were significantly elevated (all p < 0.05). Logistic regression identified platelet count (Odds ratios (OR) = 1.003, 95% confidence interval (CI): 1.001–1.006), PCT (OR = 1.032, 95% CI: 1.004–1.060), triglycerides (OR = 1.740, 95% CI: 1.052–2.879), and low cholesterol (OR = 0.523, 95% CI: 0.383–0.714) as independent risk factors. The area under the ROC curve (AUC) was 0.75 for the random forest model and 0.74 for logistic regression, with recall rates of 0.69 and 0.60, respectively.ConclusionRoutine laboratory markers integrated into machine learning models demonstrated potential for early bacteremia prediction. Random forest exhibited superior sensitivity compared to logistic regression, suggesting its potential utility as a clinical screening tool. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1605485>

---

## Drosophila melanogaster as a model system for studying the effects of porcine rotavirus on intestinal immunity

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

IntroductionDrosophila melanogaster is a quintessential model organism that has been used in many scientific studies. The intestinal immune response of flies is a critical component of their innate immune system. Given that flies primarily consume decaying organic matter, harmful microorganisms present in their food can enter the intestine, leading to frequent infections by exogenous pathogens. When these pathogens are introduced into the intestinal environment, a cascade of immune responses is triggered within the intestinal tissue, aimed at preserving the integrity of the intestinal barrier and ensuring the proper physiological functions of the gut. Porcine rotavirus (PoRV) is a key pathogen that causes diarrhea in pigs, and PoRV infection can significantly reduce piglet survival rates.MethodsIn this study, wild-type flies were orally administered PoRV to establish an effective intestinal damage animal model, and a detailed investigation of the antiviral immune defense mechanism in the fly intestine was performed.Results and DiscussionOur study revealed that PoRV infection caused a reduction in the survival rate of flies and an increase in intestinal epithelial cell death. Concurrently, PoRV infection significantly promoted the proliferation and differentiation of intestinal cells, contributing to the maintenance of intestinal homeostasis. After the activation of JAK/STAT signaling in the intestines of infected Drosophila, there was an increase in the levels of reactive oxygen species (ROS). This elevation was concomitant with the release of antimicrobial peptides (AMPs), which play a crucial role in pathogen clearance. Additionally, we identified substantial aggregation of hemocytes in the midgut. The composition of the intestinal microbiota also underwent changes, potentially playing a role in intestinal immune defense. Moreover, PoRV can evade clearance via the RNA interference (RNAi) pathway. In summary, PoRV infection in the fly intestine activates multiple immune defense mechanisms to eliminate the pathogen, offering a theoretical basis for PoRV prevention and control. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1621846>

---

## LILRA5+ macrophages drive early oxidative stress surge in sepsis: a single-cell transcriptomic landscape with therapeutic implications

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

BackgroundIn sepsis, oxidative stress (OS) triggers essential adaptive responses and emerging OS-related biomarkers show potential for enhancing sepsis diagnosis and therapy.MethodologyIn this study, we used single-cell datasets and the OS gene set to identify immune cell types with the highest oxidative activity across different sepsis states. Differential expression genes (DEG) between “high state” cells and “low state” cells were screened. High-dimensional weighted gene co-expression network analysis (hdWGCNA), combined with multiple machine learning methods, was used for the selection of hub genes. Expressions of hub genes were then validated. Cell–cell communication and transcription factor analysis were performed later. Real-time quantitative reverse transcription (qRT-PCR) and Western blotting validated expression of LILRA5 in both the cecal ligation and puncture (CLP) model and the lipopolysaccharide-induced sepsis model. Reactive oxygen species (ROS) levels were also detected in THP-1 cells after silencing LILRA5.ResultsIn the early stages of sepsis, oxidative activity reaches its peak, with macrophages displaying the highest OS among all cell types. Through the application of the “Quartile method”, all cells were clustered into three states based on OS activity (low, medium, and high). LILRA5, MGST1, PLBD1, and S100A9 were selected as hub genes and significantly upregulated in sepsis. LILRA5 was predominantly expressed in macrophages and was highly expressed in the early stage of macrophage. Specifically, LILRA5+ macrophages exhibit the strongest OS. LILRA5 showed a higher expression in both mouse sepsis models and the THP-1 cell after lipopolysaccharide stimulation. Silencing LILRA5 resulted in a significant reduction of ROS in THP-1 cells.ConclusionIn conclusion, our study has mapped the landscape of OS dynamics in sepsis and found that LILRA5+ macrophages in the early stage of sepsis exhibit the highest OS. LILRA5 emerges as a promising gene for modulating macrophage-mediated OS in sepsis. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1606401>

---

## Advances in nanotechnology for the therapy of bacterial pneumonia

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

Bacterial pneumonia, a life-threatening infection, is the world’s sixth deadliest disease and the top cause of mortality in children under five. Without timely treatment, bacterial pneumonia can escalate to a 30% mortality rate, particularly in high-risk populations. It may also lead to chronic conditions such as pulmonary fibrosis and chronic obstructive pulmonary disease(COPD), along with systemic inflammatory responses that can progress to sepsis and multi-organ failure. Although antibiotics are generally effective against bacterial pneumonia, current treatment approaches remain insufficient due to several barriers, including the lung’s unique mucus barrier, low pH, high oxidative stress, disruption of alveolar surfactants, and accumulation of hypertonic fluid on the airway surface. In addition, following the excessive use of antibiotics, dysbiosis, secondary infections and resistance occur. Nanomaterials can be an effective way to improve therapeutic effects owing to their change on drug size, physicochemical properties, hydrophobicity along with better targeting ability, and controlled localized release. Organic and inorganic substances and their composites are the three main types of nanomaterials to treat bacterial pneumonia. This review presents the latest advancements and constraints of these nanomaterials from a nanotechnology viewpoint with a view to developing therapeutic strategies for bacterial pneumonia. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1639783>

---

## Respiratory microbiota diversity and composition in recurrent protracted bacterial bronchitis: a cross-sectional study

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

IntroductionRecurrent protracted bacterial bronchitis (RPBB) is a significant risk factor for bronchiectasis in children, characterized by multiple episodes of protracted bacterial bronchitis (PBB) annually. With an increasing global incidence, a detailed understanding of RPBB’s pathophysiology is essential, particularly regarding the role of lung microbiota.MethodsThis cross-sectional study recruited 39 children from Jinhua Maternal and Child Health Hospital between January 2021 and December 2022, including 18 with PBB, 11 with RPBB, and 10 as controls. Bronchoscopy with bronchoalveolar lavage (BAL) was performed to collect lung microbiota samples, which were analyzed using 16S rDNA sequencing. Microbial diversity and composition differences among groups were assessed using alpha and beta diversity metrics, PERMANOVA, and Linear Discriminant Analysis Effect Size (LEfSe), with statistical significance set at P < 0.05.ResultsRPBB patients exhibited a distinct lung microbiota composition compared to controls, characterized by an increased abundance of pathogens such as Acinetobacter and Mycoplasma, alongside a reduction in beneficial genera like Streptococcus and Granulicatella. The RPBB group also demonstrated greater overall microbiota diversity, indicating dysbiosis that may contribute to disease severity and persistent respiratory symptoms.ConclusionThis study revealed significant alterations in the lung microbiota of children with RPBB, suggesting that microbial imbalance could play a crucial role in disease pathogenesis. These findings highlight the importance of targeted prevention and therapeutic strategies aimed at restoring microbiota balance to improve pediatric respiratory health. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1524116>

---

## Respiratory microbiome and metabolome features associate disease severity and the need for doxycycline treatment in children with macrolide-resistant Mycoplasma pneumoniae-mediated pneumonia

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

IntroductionCommensal bacterial community along the upper respiratory tract functions against pathogens. The host determinants of Mycoplasma pneumoniae severity should be identified against the increasing threat of macrolide-resistant M. pneumoniae (MRMP) infection. We hypothesized that respiratory microbiome is involved in the clinical manifestations of M. pneumoniae infection.MethodsFrom 2017 to 2020, 92 children with MRMP-mediated pneumonia were enrolled among 845 children with community-associated pneumonia. Oropharyngeal samplings were collected within 48 h after admission. We compared respiratory microbiome and metabolites based on patients’ later development of prolonged fever and the need for doxycycline treatment (DT, n = 57) and the cured control without fever or doxycycline treatment (WDT, n = 35) by using 16S rRNA-based sequencing and untargeted metabolome analysis.ResultsSignificantly higher diversity and different respiratory microbiomes were evaluated in WDT patients in contrast to DT patients. Fusobacterium, Haemophilus, Gemella, Oribacterium, Actinomyces lingnae, Fusobacterium periodonticum, Gemella sanguinis, and Solobacterium moorei were inversely correlated with disease severity. We assumed that metabolites of divergent microbiomes were related to MRMP development. We identified 15 discriminative amino-acid- and fatty-acid-related metabolites in two groups. F. periodonticum abundance was negatively associated with an inflammatory metabolite: a platelet-activating factor. Fusobacterium and Oribacterium were related to the decrease in LysoPE(18:1(9Z)/0:0) and LysoPC(18:1(9Z)).ConclusionsMicrobiota dysbiosis with dysregulated inflammatory glycerolphospholipid-related metabolites was related to disease severity and the need for doxycycline treatment in children with MRMP-mediated pneumonia. Anaerobic bacteria metabolites and metabolic pathway could be beneficial therapeutic targets against M. pneumoniae infection. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1537182>

---

## Diagnosis of non-puerperal mastitis based on “whole tongue” features: non-invasive biomarker mining and diagnostic model construction

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

BackgroundNon-puerperal mastitis (NPM) arises from heterogeneous factors ranging from autoimmune dysregulation to occult infections. To establish a diagnosis, biopsy is reliable but invasive. Imaging exhibits a limited specificity and may cause diagnostic delays, patient discomfort, and suboptimal management. Inspired by non-invasive tongue diagnosis in traditional Chinese medicine, this study integrated tongue-coating microbiota profiling and AI-quantified tongue image phenotyping to establish an objective, non-invasive diagnostic framework for NPM.MethodsA total of 100 NPM patients from the Breast Surgery Department of Longhua Hospital and 100 healthy volunteers were included. Their clinical characteristics, tongue images, and tongue-coating microbiota data were collected. Features of tongue images (detection, segmentation, and classification) were quantitated and extracted via deep learning. The microbiota composition was assessed using 16S rRNA gene sequencing (V3–V4 region) and bioinformatic pipelines (QIIME2, DADA2). Based on clinical, imaging, and microbial features, three machine learning models—logistic regression (LR), support vector machine (SVM), and gradient boosting decision tree (GBDT)—were trained to distinguish NPM.ResultsThe GBDT model achieved a superior diagnostic performance (AUROC = 0.98, accuracy = 0.95, and specificity = 0.95), outperforming the LR (AUROC = 0.98, accuracy = 0.95, and specificity = 0.90) and SVM models (AUROC = 0.87, accuracy = 0.80, and specificity = 0.75). Integration of clinical characteristics, tongue image features, and bacterial profiles (at the genus/family level) yielded the highest accuracy, whereas models using a single class of features showed a lower discriminatory ability (AUROC = 0.90–0.91). Key predictors included Campylobacter (12%), waist–hip ratio (11%), and Alloprevotella (6%).ConclusionsIntegrating clinical characteristics, tongue image features, and tongue-coating microbiota profiles, the multimodal GBDT model demonstrates a high diagnostic accuracy, supporting its utility for early screening and diagnosis of NPM. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1602883>

---

## A breakthrough Trichosporon asahii infection in an immune thrombocytopenia patient during caspofungin and isavuconazole combined therapy: a case report

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

Trichosporon infection is a rare but highly lethal infectious disease. Clinically, Trichosporon asahii is the strain most commonly isolated from patients with Trichosporon infections. T. asahii is an opportunistic pathogen that can cause local or invasive infections in immunocompromised patients. In this article, a case of breakthrough T. asahii infection in an immune thrombocytopenia (ITP) patient during caspofungin and isavuconazole combined therapy is reported. Metagenomic next-generation sequencing (mNGS) played a crucial role in the diagnosis of the breakthrough infection in this patient. Upon receiving combined therapy with intravenous voriconazole and nebulized amphotericin B, the patient demonstrated significant clinical improvement and was subsequently discharged. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1625007>

---

## Prevalence and genomic insights into type III-A CRISPR-Cas system acquisition in global Staphylococcus argenteus strains

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

IntroductionThe CRISPR-Cas system serves as a defense mechanism in bacteria and archaea, protecting them against the invasion of mobile genetic elements. Staphylococcus argenteus, a Gram-positive bacterium that diverged from Staphylococcus aureus, is characterized by the rare presence of the CRISPR-Cas system in only a few isolates. MethodsIn this study, we analyzed the prevalence of the type III-A CRISPR-Cas system in 368 S. argenteus genome sequences from animals, food sources, and humans across 26 countries, available in public database. ResultsOur findings revealed that 44.0% of these strains carry this immune system, with 98.1% of them belonging to the sequence type 2250 (ST2250). Genomic localization analysis indicated that the CRISPR-Cas is closely associated with SCCmec (mecA-ΔmecR1-IS1272-ccrB2-ccrA2) or Insertion sequence 1272 (IS1272) transposase. Further analysis identified a common IS1272 target inverted repeats (IR) sequence in ST2250 strains, providing insights into why these strains are more likely to acquire the CRISPR-Cas system. CRISPR typing identified 41 sequences types, classifying these strains into two clusters, with Cluster II being the predominant one. Homology analysis of spacers revealed that all the identified 15 spacers exhibited homology to sequences from plasmids, lytic phages, or prophages. ConclusionThis study suggests that the acquisition of the CRISPR-Cas system in S. argenteus enhances its resistance to phage attacks and plasmid invasions in environmental settings, potentially posing significant challenges for clinical treatment of infections caused by these strains and hindering efforts to control their spread in food products using phage-based interventions. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1644286>

---

## Comparative evaluation of MALDI-ToF mass spectrometry and Sanger sequencing of the 16S, hsp65, and rpoB genes for non tuberculous mycobacteria species identification

date: 2025-07-28, from: Frontiers in Cellular and Infection Microbiology

Non tuberculous mycobacteria (NTM) infections are increasing globally, underscoring the critical importance of accurate species-level identification for effective clinical management. This study aimed to evaluate the use of three conserved markers in the mycobacterial family (16S, hsp65, and rpoB) for NTM identification through Sanger sequencing, comparing the results to those obtained using MALDI-ToF MS. A total of 59 clinical NTM isolates from plastic surgery patients, previously characterized by MALDI-ToF MS, were analyzed. These isolates underwent DNA extraction, PCR amplification, and Sanger sequencing. Species identification was performed through phylogenetic analyses of each marker individually and concatenated as a multi locus sequencing approach. Concordance between MALDI-ToF MS and Sanger sequencing was assessed using Cohen’s Kappa statistical analysis. Cohen’s Kappa values indicated moderate concordance of 0.46 for 16S, 0.51 for hsp65, and 0.69 for rpoB. Concatenated phylogenetic analysis yielded improved concordance values of 0.71 for (16S + hsp65), 0.76 for (16S + rpoB), 0.69 for (rpoB + hsp65), and 0.72 for (16S + hsp65 + rpoB). Our results show that NTM identification is more accurate when employing a multi locus sequencing approach. Notably, the combination of 16S + rpoB outperformed the three-marker concatenation, offering the highest concordance for species-level identification. NTM identification is challenging, and concatenated phylogenetic analysis of two or more gene fragments should be used when MALDI-ToF MS or whole genome sequencing is not available. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2025.1612459>

