---
title: health 2024.23
updated: 2024-06-03 08:51:08
---

# health 2024.23

(date: 2024-06-03 08:51:08)

---

## Clinical differences between Raoultella spp. and Klebsiella oxytoca

date: 2024-06-03, from: Frontiers in Cellular and Infection Microbiology

PurposeRaoultella spp. is a genus of bacteria that is known to be closely related to Klebsiella. It has been debated whether Raoultella should be reclassified as a subgroup of Klebsiella. The aim of this study is to compare clinical aspects of Raoultella and Klebsiella oxytoca, a species of Klebsiella that is known to be bacteriologically similar to Raoultella spp.MethodsUsing data collected at a tertiary care hospital in the United States, we identified 43 patients with Raoultella infection and 1173 patients with Klebsiella oxytoca infection. We compared patient demographics (age and sex), hospitalization status, isolation sites and antibiotic resistance profiles between the two species.ResultsThere was no significant difference in patient demographics between the two bacteria species. The proportions of intensive care unit (ICU) admission were higher among patients with Raoultella infection (p=0.008). The most common site of isolation was urine for both species (39.5% of all patients with Raoultella spp. vs. 59.3% for K. oxytoca). The second most common site of isolation was blood stream for Raoultella spp. (23.3%) and respiratory tract for K. oxytoca (10.8%). Except for the high proportion of resistant isolates of Raoultella spp. for Trimethoprim/sulfamethoxazole, the antibiotic susceptibility profiles were similar between the two bacteria species. Both were susceptible to ciprofloxacin and meropenem.ConclusionWhile there are no significant differences in the patient demographics and antibiotic susceptibility profiles between Raoultella spp. and K. oxytoca, Raoultella may cause more serious infection requiring ICU admissions. Also, Raoultella may cause blood stream infection more frequently than K. oxytoca. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1260212>

---

## Investigating gut microbiota–blood and urine metabolite correlations in early sepsis-induced acute kidney injury: insights from targeted KEGG analyses

date: 2024-06-03, from: Frontiers in Cellular and Infection Microbiology

BackgroundThe interplay between gut microbiota and metabolites in the early stages of sepsis-induced acute kidney injury (SA-AKI) is not yet clearly understood. This study explores the characteristics and interactions of gut microbiota, and blood and urinary metabolites in patients with SA-AKI.MethodsUtilizing a prospective observational approach, we conducted comparative analyses of gut microbiota and metabolites via metabolomics and metagenomics in individuals diagnosed with SA-AKI compared to those without AKI (NCT06197828). Pearson correlations were used to identify associations between microbiota, metabolites, and clinical indicators. The Comprehensive Antibiotic Resistance Database was employed to detect antibiotic resistance genes (ARGs), while Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways informed on metabolic processes and microbial resistance patterns.ResultsOur study included analysis of four patients with SA-AKI and five without AKI. Significant disparities in bacterial composition were observed, illustrated by diversity indices (Shannon index: 2.0 ± 0.4 vs. 1.4 ± 0.6, P = 0.230; Simpson index: 0.8 ± 0.1 vs. 0.6 ± 0.2, P = 0.494) between the SA-AKI group and the non-AKI group. N6, N6, N6-Trimethyl-L-lysine was detected in both blood and urine metabolites, and also showed significant correlations with specific gut microbiota (Campylobacter hominis and Bacteroides caccae, R > 0, P < 0.05). Both blood and urine metabolites were enriched in the lysine degradation pathway. We also identified the citrate cycle (TCA cycle) as a KEGG pathway enriched in sets of differentially expressed ARGs in the gut microbiota, which exhibits an association with lysine degradation.ConclusionsSignificant differences in gut microbiota and metabolites were observed between the SA-AKI and non-AKI groups, uncovering potential biomarkers and metabolic changes linked to SA-AKI. The lysine degradation pathway may serve as a crucial link connecting gut microbiota and metabolites. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1375874>

---

## Presence of Group A streptococcus frequently assayed virulence genes in invasive disease: a systematic review and meta-analysis

date: 2024-06-03, from: Frontiers in Cellular and Infection Microbiology

IntroductionIt is currently unclear what the role of Group A streptococcus (GAS) virulence factors (VFs) is in contributing to the invasive potential of GAS. This work investigated the evidence for the association of GAS VFs with invasive disease.MethodsWe employed a broad search strategy for studies reporting the presence of GAS VFs in invasive and non-invasive GAS disease. Data were independently extracted by two reviewers, quality assessed, and meta-analyzed using Stata®.ResultsA total of 32 studies reported on 45 putative virulence factors [invasive (n = 3,236); non-invasive (n = 5,218)], characterized by polymerase chain reaction (PCR) (n = 30) and whole-genome sequencing (WGS) (n = 2). The risk of bias was rated as low and moderate, in 23 and 9 studies, respectively. Meta-,analyses of high-quality studies (n = 23) revealed a significant association of speM [OR, 1.64 (95%CI, 1.06; 2.52)] with invasive infection. Meta-analysis of WGS studies demonstrated a significant association of hasA [OR, 1.91 (95%CI, 1.36; 2.67)] and speG [OR, 2.83 (95%CI, 1.63; 4.92)] with invasive GAS (iGAS). Meta-analysis of PCR studies indicated a significant association of speA [OR, 1.59 (95%CI, 1.10; 2.30)] and speK [OR, 2.95 (95%CI, 1.81; 4.80)] with invasive infection. A significant inverse association was observed between prtf1 [OR, 0.42 (95%CI, 0.20; 0.87)] and invasive infection.ConclusionThis systematic review and genomic meta-analysis provides evidence of a statistically significant association with invasive infection for the hasA gene, while smeZ, ssa, pnga3, sda1, sic, and NaDase show statistically significantly inverse associations with invasive infection. SpeA, speK, and speG are associated with GAS virulence; however, it is unclear if they are markers of invasive infection. This work could possibly aid in developing preventative strategies. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1337861>

---

## Exploring the role of brain-derived extracellular vesicles in viral infections: from pathological insights to biomarker potential

date: 2024-06-03, from: Frontiers in Cellular and Infection Microbiology

Extracellular vesicles (EVs) are membrane-bound vesicles secreted by all cell types that play a central role in cell-to-cell communication. Since these vesicles serve as vehicles of cellular content (nucleic acids, proteins and lipids) with the potential to cross biological barriers, they represent a novel attractive window into an otherwise inaccessible organ, such as the brain. The composition of EVs is cell-type specific and mirrors the physiological condition of the cell-of-origin. Consequently, during viral infection, EVs undergo significant changes in their content and morphology, thereby reflecting alterations in the cellular state. Here, we briefly summarize the potential of brain-derived EVs as a lens into viral infection in the central nervous system, thereby: 1) uncovering underlying pathophysiological processes at play and 2) serving as liquid biopsies of the brain, representing a non-invasive source of biomarkers for monitoring disease activity. Although translating the potential of EVs from research to diagnosis poses complexities, characterizing brain-derived EVs in the context of viral infections holds promise to enhance diagnostic and therapeutic strategies, offering new avenues for managing infectious neurological diseases. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1423394>

