<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2024-11-02 18:01:00)</p>
<hr />
<h2
id="era-crisprcas12a-based-fast-and-specific-diagnostic-detection-for-chlamydia-pneumoniae">ERA-CRISPR/Cas12a-based,
fast and specific diagnostic detection for Chlamydia pneumoniae</h2>
<p>date: 2024-11-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Chlamydia pneumoniae (C. pneumoniae) is a specialized intracellular
parasitic pathogen capable of causing pneumonia, sinusitis, bronchitis,
and other respiratory diseases, which pose significant public health
challenges. Therefore, rapid, accurate, and sensitive diagnosis is
crucial for the prevention and treatment of respiratory diseases caused
by C. pneumoniae. In this study, we combined enzymatic recombination
amplification (ERA) with the clustered regularly interspaced short
palindromic repeats (CRISPR)/CRISPR-associated protein (Cas) 12a system
(CRISPR/Cas12a) to develop a dual detection platform termed the
Cpn-ERA-CRISPR/Cas12a dual system. This system integrates both the
ERA-CRISPR/Cas12a fluorescence system and the ERA-CRISPR/Cas12a lateral
flow system. Detection results can be measured using a fluorescence
detector or observed with the naked eye on lateral flow strips. The
fluorescence system and the lateral flow system detect C. pneumoniae in
30 minutes and 15 minutes, respectively. This dual system exhibits no
cross-reactivity with the other seven pathogens, demonstrating high
specificity, and achieves a sensitivity of 100 copies/µL. Additionally,
the Cpn-ERA-CRISPR/Cas12a dual system was employed to analyze 39
clinical samples, comprising 19 positive and 20 negative samples. The
detection rate for positive samples was 100%, with no positive results
in the negative samples, indicating a high level of concordance with
qPCR results. In summary, the Cpn-ERA-CRISPR/Cas12a dual system
represents a novel tool for diagnosing C. pneumoniae and holds promising
application potential in grassroots community hospitals.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477422"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477422</a></p>
<hr />
<h2
id="phyllospheric-microbial-community-structure-and-carbon-source-metabolism-function-in-tobacco-wildfire-disease">Phyllospheric
microbial community structure and carbon source metabolism function in
tobacco wildfire disease</h2>
<p>date: 2024-11-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The phyllospheric microbial composition of tobacco plants is
influenced by multiple factors. Disease severity level is one of the
main influencing factors. This study was designed to understand the
microbial community in tobacco wildfire disease with different disease
severity levels. Tobacco leaves at disease severity level of 1, 5, 7,
and 9 (L1, L5, L7, and L9) were collected; both healthy and diseased
leaf tissues for each level were collected. The community structure and
diversity in tobacco leaves with different disease severity levels were
compared using high-throughput technique and Biolog Eco. The results
showed that in all healthy and diseased tobacco leaves, the most
dominant bacterial phylum was Proteobacteria with a high prevalence of
genus Pseudomonas; the relative abundance of Pseudomonas was most found
at B9 diseased samples. Ascomycota represents the most prominent fungal
phylum, with Blastobotrys as the predominant genus. In bacterial
communities, the Alpha diversity of healthy samples was higher than that
of diseased samples. In fungal community, the difference in Alpha
diversity between healthy and diseased was not significant. LEfSe
analysis showed that the most enriched bacterial biomarker was
unclassified_Gammaproteobacteria in diseased samples;
unclassified_Alcaligenaceae were the most enrich bacterial biomarker in
healthy samples. FUNGuild analysis showed that saprotroph was the
dominated mode in health and lower diseased samples, The abundance of
pathotroph–saprotroph and pathotroph–saprotroph–symbiotroph increases at
high disease levels. PICRUSt analysis showed that the predominant
pathway was metabolism function, and most bacterial gene sequences seem
to be independent of the disease severity level. The Biolog Eco results
showed that the utilization rates of carbon sources decrease with
increasing disease severity level. The current study revealed the
microbial community’s characteristic of tobacco wildfire disease with
different disease severity levels, providing scientific references for
the control of tobacco wildfire disease.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1458253"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1458253</a></p>
<hr />
<h2
id="enhancing-ascitic-fungal-infection-diagnosis-through-next-generation-sequencing-a-pilot-study-in-surgical-icu-patients">Enhancing
ascitic fungal infection diagnosis through next-generation sequencing: a
pilot study in surgical ICU patients</h2>
<p>date: 2024-11-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesAscites, often associated with critical pathologies such as
liver cirrhosis or bowel perforation, can be complicated by fungal
infection, increasing mortality especially in intensive care settings
and demanding rapid diagnosis and adequate treatment. Traditional
microbiological diagnostic methods have limited sensitivity in
accurately identifying fungal pathogens in ascitic fluid. Alternative
diagnostic methods may offer important insights to enable guiding of
antifungal therapy and refining empirical treatment strategies. The
objective of this study was to evaluate the potential of next-generation
sequencing methods to identify specific fungal pathogens responsible for
ascitic fluid infections.MethodsWe prospectively collected 50 ascitic
fluid samples from ICU patients with suspected ascites infection. In
addition to standard culture-based microbiological testing, an ascitic
fluid aliquot underwent fungal DNA isolation and was analyzed by
next-generation sequencing (NGS) methods for identification of fungal
species.ResultsOf 50 ascitic samples collected, five samples showed
growth of Candida spp. in culture. After DNA isolation and ITS2 PCR,
detectable amplification was achieved in 10 samples. Sequencing of the
50 patients’ samples identified facultative pathogenic fungi in 19
patients. In 15 cases, culture alone would not have permitted the
identification of all facultative pathogenic fungi. The identification
of fungal DNA by sequencing was significantly associated with poor
patient outcome and a number of clinical parameters.ConclusionsOur
results show a higher sensitivity for NGS-based diagnostic methods in
the identification of ascitic fluid fungal infections compared to
culture-based diagnostics. This may be beneficial especially for
patients in a critical care setting, who have an increased prevalence of
comorbidities and high mortality. The implementation of such methods in
standard diagnosis will require increased standardization of the
workflows and interpretation of the sequencing results with respect to
patients’ clinical picture.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1441805"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1441805</a></p>
<hr />
<h2
id="integrating-enzyme-nanoparticles-bring-new-prospects-for-the-diagnosis-and-treatment-of-immune-dysregulation-in-periodontitis">Integrating
enzyme-nanoparticles bring new prospects for the diagnosis and treatment
of immune dysregulation in periodontitis</h2>
<p>date: 2024-11-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Enzymes play a significant role in mediating inflammatory and immune
responses in periodontitis. Effective diagnosis, timely treatment, and
continuous management of periodontal enzymes are essential to prevent
undesirable consequences; however, this remains a significant challenge.
Nanoparticles (NPs) have attracted significant attention in biomedicine
because of their advantageous nanosized effects. NPs are conjugated with
specific enzyme substrates at responsive sites that are triggered by
periodontitis enzyme biomarkers, leading to functional or characteristic
changes. In contrast, NPs with enzyme-mimetic activities exhibit
catalytic activity, effectively destroying pathogenic biofilms and
modulating the immune response in periodontitis. The unique properties
of enzyme-targeting NPs have enabled the development of biosensors and
fluorescent probes capable of identifying enzyme biomarkers associated
with periodontitis. Enzyme-responsive and enzyme-mimetic NPs both exert
therapeutic applications in the treatment of periodontitis. In this
review, we provide a comprehensive overview of the enzymes associated
with periodontitis, the mechanisms of enzyme-responsive and
enzyme-mimetic NPs, recent advancements in the use of NPs for detecting
these enzymes, and the therapeutic applications of NPs in targeting or
mimicking enzyme functions. We also discuss the challenges and prospects
of using NPs in the diagnosis and treatment of periodontitis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494651"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494651</a></p>
<hr />
<h2
id="artificial-intelligence-in-predicting-pathogenic-microorganisms-antimicrobial-resistance-challenges-progress-and-prospects">Artificial
intelligence in predicting pathogenic microorganisms’ antimicrobial
resistance: challenges, progress, and prospects</h2>
<p>date: 2024-11-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The issue of antimicrobial resistance (AMR) in pathogenic
microorganisms has emerged as a global public health crisis, posing a
significant threat to the modern healthcare system. The advent of
Artificial Intelligence (AI) and Machine Learning (ML) technologies has
brought about revolutionary changes in this field. These advanced
computational methods are capable of processing and analyzing
large-scale biomedical data, thereby uncovering complex patterns and
mechanisms behind the development of resistance. AI technologies are
increasingly applied to predict the resistance of pathogens to various
antibiotics based on gene content and genomic composition. This article
reviews the latest advancements in AI and ML for predicting
antimicrobial resistance in pathogenic microorganisms. We begin with an
overview of the biological foundations of microbial resistance and its
epidemiological research. Subsequently, we highlight the main AI and ML
models used in resistance prediction, including but not limited to
Support Vector Machines, Random Forests, and Deep Learning networks.
Furthermore, we explore the major challenges in the field, such as data
availability, model interpretability, and cross-species resistance
prediction. Finally, we discuss new perspectives and solutions for
research into microbial resistance through algorithm optimization,
dataset expansion, and interdisciplinary collaboration. With the
continuous advancement of AI technology, we will have the most powerful
weapon in the fight against pathogenic microbial resistance in the
future.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1482186"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1482186</a></p>
<hr />
<h2
id="degs-regulates-the-aerobic-metabolism-of-vibrio-cholerae-via-the-arca-isocitrate-dehydrogenase-pathway-for-growth-and-intestinal-colonization">DegS
regulates the aerobic metabolism of Vibrio cholerae via the
ArcA-isocitrate dehydrogenase pathway for growth and intestinal
colonization</h2>
<p>date: 2024-11-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Aerobic respiration is the key driver of Vibrio cholerae
proliferation and infection. Our previous transcriptome results
suggested that degS knockout downregulates a few genes involved in NADH
and ATP synthesis in the aerobic respiratory pathway. In this study,
non-targeted metabolomics results showed that the differential
metabolites affected by degS knockout were associated with aerobic
respiration. Further results suggested that the key products of aerobic
respiration, NADH and ATP, were reduced upon degS deletion and were not
dependent on the classical σE pathway. The two-component system response
factor aerobic respiration control A (ArcA) is involved in regulating
NADH and ATP levels. qRT-PCR demonstrated that DegS negatively regulates
the transcription of the arcA gene, which negatively regulates the
expression of isocitrate dehydrogenase (ICDH), a key rate-limiting
enzyme of the tricarboxylic acid cycle. NADH and ATP levels were
partially restored with the knockout of the arcA gene in the ΔdegS
strain, while levels were partially restored with overexpression of ICDH
in the ΔdegS strain. In a growth experiment, compared to the ΔdegS
strain, the growth rates of ΔdegSΔarcA and ΔdegS-overexpressed icdh
strains (ΔdegS+icdh) were partially restored during the logarithmic
growth period. Colonization of the intestines of suckling mice showed a
significant reduction in the colonizing ability of the ΔdegS strain,
similar colonizing ability of the ΔdegS::degS strain and the wild-type
strain, and a partial recovery of the colonizing ability of the
ΔdegS+icdh strain. Overall, these findings suggest that the DegS
protease regulates the expression of ICDH through ArcA, thereby
affecting the NADH and ATP levels of V. cholerae and its growth and
intestinal colonization ability.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1482919"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1482919</a></p>
<hr />
<h2
id="enhancing-icu-candida-spp.-surveillance-a-cost-effective-approach-focused-on-candida-auris-detection">Enhancing
ICU Candida spp. surveillance: a cost-effective approach focused on
Candida auris detection</h2>
<p>date: 2024-11-01, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCandida auris is an emerging pathogen that represents a
worldwide health problem due to its global expansion, multidrug
resistance, and difficult laboratory identification. Among the risk
factors for colonization/infection by C. auris, a stay in an intensive
care unit (ICU) stands out. This prospective multicenter study aimed to
monitor the trend of the local epidemiology of Candida spp. and unveil
the prevalence of C. auris.MethodsFrom 2020 to 2022, axillar/inguinal
swabs were collected from adult patients at three points: upon admission
(D1) and on the fifth (D5) and eighth (D8) days of their ICU stay. We
employed culture-based screening methods combined with molecular
techniques to identify Candida spp. down to the species level. Specific
screening for Candida auris was conducted using a real-time PCR assay in
combination with an improved selective culture medium, mannitol salt
agar auris (MSAA). To validate the effectiveness of MSAA, a collection
of reference C. auris strains representing the four major geographical
clades was used.ResultsWe enrolled 675 patients, and 355 Candida
isolates were retrieved from the 988 swab samples collected. From those,
185/355 (52.1%) were identified as C. albicans and 170/355 (47.9%) as
non-albicans Candida (NAC). MSAA medium showed a specificity of 94.8%,
albeit C. auris was not detected in this cohort. The dynamics of Candida
spp. colonization by ICU were significant at the three collection
points. Upon admission, C. albicans was associated with the Beatriz
Ângelo Hospital ICU (p=0.003) and C. tropicalis with the general
Hospital Professor Doutor Fernando Fonseca (FFH) ICU (p=0.006). C.
parapsilosis and C. lusitaniae were associated with FFH ICUs, with the
general ICU at D5 (p=0.047) and surgical ICU at D8 (p=0.012). The
dynamics of NAC colonization by ICU were significantly different at D1
(p=0.011), D5 (p=0.047), and D8 (p=0.012).ConclusionWe developed and
implemented a screening protocol for C. auris while uncovering the
colonization patterns of Candida in the ICU. Our findings contribute to
the optimization of overall patient management, ensuring that ICU
protocols are resilient and adaptive to emerging fungal threats.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1463456"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1463456</a></p>
<hr />
<h2 id="high-oxalate-foods-list-your-common-questions-answered">High
Oxalate Foods List: Your Common Questions Answered!</h2>
<p>date: 2024-10-31, from: The Kidney Dietitian</p>
<p>
For many people with kidney stones, navigating a high oxalate foods list
can be frustrating. Oxalate is a foreign concept to many people and it
can seem like there are endless lists you “should” follow. Not to
mention information on the high oxalate foods list can be different from
various sources! A high oxalate foods […]
</p>
<p>
The post
<a href="https://www.thekidneydietitian.org/high-oxalate-foods-list/">High
Oxalate Foods List: Your Common Questions Answered!</a> appeared first
on <a href="https://www.thekidneydietitian.org">The Kidney
Dietitian</a>.
</p>
<p><br></p>
<p><a href="https://www.thekidneydietitian.org/high-oxalate-foods-list/"
class="uri">https://www.thekidneydietitian.org/high-oxalate-foods-list/</a></p>
<hr />
<h2
id="pathogen-spectrum-and-microbiome-in-lower-respiratory-tract-of-patients-with-different-pulmonary-diseases-based-on-metagenomic-next-generation-sequencing">Pathogen
spectrum and microbiome in lower respiratory tract of patients with
different pulmonary diseases based on metagenomic next-generation
sequencing</h2>
<p>date: 2024-10-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe homeostasis of the microbiome in lower respiratory
tract is crucial in sustaining normal physiological functions of the
lung. Different pulmonary diseases display varying degrees of microbiome
imbalance; however, the specific variability and clinical significance
of their microbiomes remain largely unexplored.MethodsIn this study, we
delineated the pathogen spectrum and commensal microorganisms in the
lower respiratory tract of various pulmonary diseases using metagenomic
sequencing. We analyzed the disparities and commonalities of the
microbial features and examined their correlation with disease
characteristics.ResultsWe observed distinct pathogen profiles and a
diversity in lower airway microbiome in patients diagnosed with cancer,
interstitial lung disease, bronchiectasis, common pneumonia,
Nontuberculous mycobacteria (NTM) pneumonia, and severe
pneumonia.DiscussionThis study illustrates the utility of Metagenomic
Next-generation Sequencing (mNGS) in identifying pathogens and analyzing
the lower respiratory microbiome, which is important for understanding
the microbiological aspect of pulmonary diseases and essential for their
early and precise diagnosis.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1320831"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1320831</a></p>
<hr />
<h2
id="influence-of-gender-age-and-body-mass-index-on-the-gut-microbiota-of-individuals-from-south-china">Influence
of gender, age, and body mass index on the gut microbiota of individuals
from South China</h2>
<p>date: 2024-10-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe symbiotic gut microbiota is pivotal for human health,
with its composition linked to various diseases and metabolic disorders.
Despite its significance, there remains a gap in systematically
evaluating how host phenotypes, such as gender, age, and body mass index
(BMI), influence gut microbiota.Methodology/principal findingsWe
conducted an analysis of the gut microbiota of 185 Chinese adults based
on whole-metagenome shotgun sequencing of fecal samples. Our
investigation focused on assessing the effects of gender, age, and BMI
on gut microbiota across three levels: diversity, gene/phylogenetic
composition, and functional composition. Our findings suggest that these
phenotypes have a minor impact on shaping the gut microbiome compared to
enterotypes, they do not correlate significantly within- or
between-sample diversity. We identified a substantial number of
phenotype-associated genes and metagenomic linkage groups (MLGs),
indicating variations in gut microflora composition. Specifically, we
observed a decline in beneficial Firmicutes microbes, such as
Eubacterium, Roseburia, Faecalibacterium and Ruminococcus spp., in both
older individuals and those with higher BMI, while potentially harmful
microbes like Erysipelotrichaceae, Subdoligranulum and Streptococcus
spp. increased with age. Additionally, Blautia and Dorea spp. were found
to increase with BMI, aligning with prior research. Surprisingly,
individuals who were older or overweight exhibited a lack of
Bacteroidetes, a dominant phylum in the human gut microbiota that
includes opportunistic pathogens, while certain species of the
well-known probiotics Bifidobacterium were enriched in these groups,
suggesting a complex interplay of these bacteria warranting further
investigation. Regarding gender, several gender-associated MLGs from
Bacteroides, Parabacteroides, Clostridium and Akkermansia were enriched
in females. Functional analysis revealed a multitude of
phenotype-associated KEGG orthologs (KOs).Conclusions/significanceOur
study underscores the influence of gender, age, and BMI on gut
metagenomes, affecting both phylogenetic and functional composition.
However, further investigation is needed to elucidate the precise roles
of these bacteria, including both pathogens and probiotics.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1419884"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1419884</a></p>
<hr />
<h2
id="the-potential-role-of-gut-microbiota-in-the-occurrence-and-development-of-endometriosis">The
potential role of gut microbiota in the occurrence and development of
endometriosis</h2>
<p>date: 2024-10-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Endometriosis (EMT) has a significant impact on women’s physical and
mental health. In this study, high-throughput sequencing technology was
employed to detect differences in gut microbiota between EMT patients
and healthy individuals (CTL). Additionally, Spearman correlation
analysis was utilized to analyze the correlation between different
bacterial genera and EMT biomarkers (CA125 and CA199). The results
demonstrated that at the phylum level, the relative abundances of
Proteobacteria and Desulfobacterota_G_459546 in the EMT group were
significantly higher than those in the CTL group, while the relative
abundances of Bacteroidota and Firmicutes_A in the EMT group were
significantly lower than those in the CTL group. At the genus level, the
relative abundances of Burkholderiales and Sphingomonadales in the EMT
group were significantly higher than those in the CTL group, while the
relative abundances of Bacteroidales and Roseburia in the EMT group were
significantly lower than those in the CTL group. The correlation
analysis results show that CA125 and CA199 are significantly positively
correlated with Burkholderiales and Sphingomonadales, and significantly
negatively correlated with Bacteroidales, Oscillospirales, and
Roseburia. The PICRUSt2 results show that the relative abundance in the
cell motility and xenobiotics biodegradation and metabolism pathways in
the EMT group was higher than that in the CTL group, while the relative
abundance in the translation, replication and repair, folding, sorting
and degradation, metabolism of terpenoids and polyketides and metabolism
of cofactors and vitamins pathways in the EMT group was lower than that
in the CTL group. In brief, there is a close correlation between the
imbalance of gut microbiota and the onset of EMT. The intestinal
microbiota has great significance broad prospects for the prevention,
diagnosis and treatment of EMT.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1454931"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1454931</a></p>
<hr />
<h2
id="external-damp-environment-aggravates-diarrhea-in-spleen-deficiency-and-dampness-syndrome-in-mice-involvement-of-small-intestinal-contents-microbiota-energy-metabolism-gastrointestinal-and-fluid-functions">External
damp environment aggravates diarrhea in spleen deficiency and dampness
syndrome in mice: involvement of small intestinal contents microbiota,
energy metabolism, gastrointestinal and fluid functions</h2>
<p>date: 2024-10-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectivesRecent studies have increasingly demonstrated that a
multiplatform water environment combined with lard gavage is an
effective method for establishing a mouse model of diarrhea. However,
the interactions between intestinal microorganisms and diarrhea, as well
as the relationships among energy metabolism, fluid balance, and
gastrointestinal function in this model, remain poorly
understood.MethodsBuilding on previous research, this study aimed to
optimiz and replicate a multiplatform water environment combined with a
lard gavage model. Male Kunming mice, free of specific pathogens, were
randomly divided into four groups: a normal control group (ZC), a
standing group (ZL), a standing combined with lard group (ZLZ), and a
standing combined with internal and external wet conditions group
(ZLZS). The mice in the ZL, ZLZ, and ZLZS groups were subjected to 4
hours of daily standing in a custom-designed multiplatform water
environment. Starting on day 8, mice in the ZLZ and ZLZS groups were
gavaged with lard (0.4 mL per session, twice daily) for 7 consecutive
days, while those in the ZLZS group were additionally exposed to a wet
litter environment (50 g/100 mL). The ZC and ZL groups received equal
volumes of sterile water via gavage. The microbiota in the small
intestine, as well as serum levels of cAMP, cGMP, VIP, Gas, and
D-xylose, were analyzed.ResultsCompared with the ZLZ group, the ZLZS
group showed significantly lower serum levels of cAMP/cGMP (p&lt;0.01)
and Gas (p&lt;0.01). D-xylose levels were lower in the ZL, ZLZ, and ZLZS
groups compared to the ZC group, while VIP levels were significantly
higher in the ZL and ZLZS groups (p&lt;0.01). Moverover,
Corynebacterium, Empedobacter, and Pseudochrobactrum were identified as
characteristic bacterial genera in the ZLZS group. The mechanism by
which the small intestinal microbiota induces diarrhea was linked to the
biosynthesis of secondary bile acids.ConclusionA multiplatform water
environment combined with lard gavage can effectively induce diarrhea,
and the addition of an external wet environment exacerbates this
condition by affecting small intestinal contents microbiota and other
functions.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1495311"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1495311</a></p>
<hr />
<h2
id="the-microbiome-biomarkers-of-pregnant-womens-vaginal-area-predict-preterm-prelabor-rupture-in-western-china">The
microbiome biomarkers of pregnant women’s vaginal area predict preterm
prelabor rupture in Western China</h2>
<p>date: 2024-10-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionIntraamniotic infection is crucial in preterm prelabor
rupture of membranes(PPROM), a clinical condition resulting from the
invasion of vaginal opportunistic microbes into the amniotic cavity.
Although previous studies have suggested potential associations between
infection and PPROM, the role of vaginalopportunistic bacteria in PPROM
has received limited attention.MethodsThis study aimed to confirm the
vaginal bacterial etiology of PPROM. We investigated vaginal microbiotas
using automatic analysis of vaginal discharge, microbiological tests,
and 16s rRNA genehigh-throughput sequencing.ResultsThe research findings
revealed that the proportion of parabasal epitheliocytes, leukocytes,
toxic leukocytes, and bacteria with diameters smaller than 1.5 um was
significantly higher in the PPROM group than that in the normal
full-term labor (TL) group. The top three vaginal opportunistic
bacterial isolates in all participants were 9.47% Escherichia coli,
5.99% Streptococcus agalactiae, and 3.57% Enterococcus faecalis. The
bacterial resistance differed, but all the isolates were sensitive to
nitrofurantoin. Compared with the vaginal microbiota dysbiosis (VMD) TL
(C) group, the VMD PPROM (P) group demonstrated more operational
taxonomic units, a high richness of bacterial taxa, and a different
beta-diversity index. Indicator species analysis revealed that
Lactobacillus jensenii, Lactobacillus crispatus, and Veillonellaceae
bacterium DNF00626 were strongly associated with the C group. Unlike the
C group, the indicator bacteria in the P group were Enterococcus
faecalis, Escherichia coli, and Streptococcus agalactiae.DiscussionThese
findings provide solidevidence that an abnormal vaginal microbiome is a
very crucial risk factorclosely related to PPROM. There were no unique
bacteria in the vaginalmicrobiota of the PPROM group; however, the
relative abundance of bacteria inthe abnormal vaginal flora of PPROM
pregnancies differed. Antibiotics should bereasonably selected based on
drug sensitivity testing. The findings presented in this paper enhance
our understanding of Streptococcus agalactiae, Enterococcus faecalis,
and Escherichia coli vaginal bacterial etiology of PPROM in Western
China.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1471027"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1471027</a></p>
<hr />
<h2
id="oliguria-as-a-diagnostic-marker-of-severe-leptospirosis-a-study-from-the-transcarpathian-region-of-ukraine">Oliguria
as a diagnostic marker of severe leptospirosis: a study from the
Transcarpathian region of Ukraine</h2>
<p>date: 2024-10-31, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Leptospirosis is an emerging illness presenting a broad range of
clinical manifestations, ranging from asymptomatic or mild cases to
severe and fatal outcomes. Early detection is crucial for effective
treatment; however, similar clinical presentations in various febrile
illnesses or co-infections, along with challenges in laboratory
diagnostics, can lead to misdiagnosis and severe consequences.
Identifying clinical predictors for severe forms of the disease is
essential in mitigating complications and reducing mortality.
Consequently, we conducted a retrospective case-control study to
identify clinical markers indicative of severe disease in leptospirosis
patients from the Transcarpathian region. The study focused on patients
admitted with clinically suspected leptospirosis, involving a total of
51 diagnosed cases, with 13 resulting in severe outcomes and death.
Categorical variables were analyzed using χ2, revealing a mean patient
age of 50 years, predominantly male (n = 36, 70.5%). Oliguria emerged as
a significant independent factor associated with mortality (odds ratio
[OR], 13.5; 95% confidence interval [CI], 2.56–71.12; p = 0.001).
Additionally, our analysis uncovered a noteworthy increase in
leptospirosis notification rates in Transcarpathian compared to Ukraine,
with 150 cases out of the total 433 in Ukraine. The highest notification
rates were observed in Mukachevo District and Perechyn District. These
findings highlight the importance of early recognition of key clinical
markers, such as oliguria, which are critical for predicting severe
outcomes in leptospirosis patients. The higher notification rates in
Transcarpathian regions also underscore the need for enhanced
surveillance, targeted public health interventions, and timely treatment
to reduce mortality in endemic areas.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1467915"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1467915</a></p>
<hr />
<h2
id="multiple-brucella-melitensis-lineages-are-driving-the-human-brucellosis-epidemic-in-shaanxi-province-china-evidence-from-whole-genome-sequencing-based-analysis">Multiple
Brucella melitensis lineages are driving the human brucellosis epidemic
in Shaanxi Province, China: evidence from whole genome sequencing-based
analysis</h2>
<p>date: 2024-10-30, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionHuman brucellosis is a severe public concern in Shaanxi
Province, China, and investigating the epidemiological relationship and
transmission pattern of B. melitensis is necessary to devise control
strategies.MethodsIn this study, a conventional bio-typing approach and
whole genome sequencing of single-nucleotide polymorphisms (SNPs) were
employed to identify 189 strains.ResultsBased on the conventional
bio-typing, 189 Brucella strains were identified as B. melitensis, of
which 14 were in bv. 1, 145 were in bv. 3, and 30 were variant, and the
Brucella strains were distributed in all ten cities in Shaanxi Province.
SNP analysis was used to identify genetic variation in 189 B. melitensis
genomes, and maximum-likelihood was used to generate a phylogeny that
identified two clades (A and B) and 19 sequence types (STs). The two
clades were highly diverse and exclusively of Eastern Mediterranean
origin. Clade B contained 18 STs (2-19), with most isolates originating
from a broad swath, implying that multiple B. melitensis lineages
circulated in Shaanxi. The 19 STs were composed of 3 to 46 strains
isolated from different counties and years, suggesting that multiple
cross-county brucellosis outbreak events are driven by multiple B.
melitensis lineages. Global phylogenetic analysis revealed that clade A
was close to GTIIb, and clade B was placed in the GTIIh lineage,
expanding the known diversity of B. melitensis from China.ConclusionThe
human brucellosis epidemic in Shaanxi is driven by multiple indigenous
circulating B. melitensis lineages, the knowledge of which will
contribute to devising a control strategy and providing the foundation
for a comprehensive regional phylogeny of this important zoonotic
pathogen.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1452143"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1452143</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2024-10-29, from: People’s CDC</p>
<p>This is the People’s CDC bi-weekly update for October 28, 2024! This
Weather Report from the People’s CDC sheds light on the ongoing COVID
situation in the US.</p>
<p><br></p>
<p><a
href="https://peoplescdc.org/2024/10/28/peoples-cdc-covid-19-weather-report-85/"
class="uri">https://peoplescdc.org/2024/10/28/peoples-cdc-covid-19-weather-report-85/</a></p>
<hr />
<h2
id="chemokine-profile-in-the-serum-of-patients-with-leptospirosis">Chemokine
profile in the serum of patients with leptospirosis</h2>
<p>date: 2024-10-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionLeptospirosis is a global zoonosis that affects more than
one million people per year, with a lethality rate of approximately 15%.
Chemokines are crucial in the immune response against Leptospira,
recruiting leukocytes to the site of infection and regulating immune
activity. In previous studies, we have shown that CCL2, CXCL5, and CCL8
are involved in the leptospirosis process, although the mechanisms are
not understood. MethodsIn this study, we present the frequency of
Leptospira serovars in human samples. We then evaluated the profile of
various chemokines in sera from patients diagnosed with leptospirosis,
assessing the possible correlation between them. Moreover, we evaluated
the changes in the chemokine profile on different days after the first
symptoms. The frequency of the Leptospira serovars in human samples is
presented. Results and discussionThe main findings were that CCL5,
CXCL5, and CXCL9 are highly expressed during leptospirosis, indicating a
special role of these molecules in the immunity and pathogenesis of the
disease. The correlation analysis of detected chemokines CXCL11, CXCL9,
CCL3, and CCL2 helps to clarify the role of each cytokine in
leptospirosis. The possible use of CCL5 as a biomarker for complementary
diagnosis of the disease is suggested.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1484291"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1484291</a></p>
<hr />
<h2
id="bronchoalveolar-lavage-fluid-and-lung-biopsy-tissue-metagenomic-next-generation-sequencing-in-the-diagnosis-of-pulmonary-cryptococcosis">Bronchoalveolar
lavage fluid and lung biopsy tissue metagenomic next-generation
sequencing in the diagnosis of pulmonary cryptococcosis</h2>
<p>date: 2024-10-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveTo evaluate the diagnostic value of metagenomic
next-generation sequencing (mNGS) in pulmonary cryptococcosis (PC) using
bronchoalveolar lavage fluid (BALF) and lung biopsy tissue
specimens.MethodsIn this retrospective study, 321 patients diagnosed
with lower respiratory tract diseases who underwent mNGS using BALF and
LBT samples, between January 2021 and December 2023 were included.
Individuals were classified into PC and non-PC groups according to the
diagnostic criteria for PC, and conventional fungal cultures were
performed. A serum/BALF cryptococcal antigen (CrAg) test was performed
in some patients with PC. The diagnostic efficiencies of three methods
for PC (mNGS, conventional culture, and CrAg) were compared.
Additionally, two mNGS methods were used in this study: original mNGS
(OmNGS, testing time from January 2021 to December 2022) and modified
mNGS (MmNGS, testing time from January to December 2023). The diagnostic
efficiency of the two mNGS methods on PC was simultaneously
compared.ResultsAmong the 321 patients, 23 (7.2%) had PC and 298 (92.8%)
did not. Compared with the composite reference standard for PC
diagnosis, the sensitivity, specificity, and accuracy of mNGS for PC
were 78.3% (95% confidence interval [CI], 55.8%–91.7%), 98.7% (95% CI,
96.4%–99.6%), and 97.2% (95% CI, 94.7%–98.7%), respectively. The
sensitivity of mNGS was similar to that of CrAg (80.0%, 12/15) (P &gt;
0.05). The diagnostic sensitivity of both mNGS and CrAg was higher than
that of conventional culture (35.0%, 7/20) (P = 0.006, P = 0.016), and
the combined detection of mNGS and CrAg further improved the diagnostic
sensitivity of PC (93.3%, 14/15). The area under the receiver operating
characteristic curve of mNGS was superior to that of conventional
culture (0.885 vs. 0.675). In addition, the diagnostic sensitivity of PC
was higher than that of OmNGS (P = 0.046).ConclusionThe sensitivity of
mNGS is better than that of conventional culture. The combination of
mNGS and CrAg improves the testing sensitivity of Cryptococcus. MmNGS
could further improve the detection of Cryptococcus. Conventional PC
detection methods are indispensable and mNGS can be used as a rapid and
accurate auxiliary diagnostic method for PC.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446814"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446814</a></p>
<hr />
<h2
id="development-and-preliminary-assessment-of-the-ifind-tbr-all-in--one-molecular-diagnostic-assay-for-rapid-detection-of-mycobacterium-tuberculosis-and-rifampicin-resistance">Development
and preliminary assessment of the iFIND TBR: all-in- one molecular
diagnostic assay for rapid detection of Mycobacterium tuberculosis and
rifampicin resistance</h2>
<p>date: 2024-10-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionEarly and accurate diagnosis of tuberculosis (TB) is
crucial for initiating timely treatment and preventing new infections.
In this study, we introduced the iFIND TBR assay, an automated
all-in-one tuberculosis detection approach that simultaneously detect
Mycobacterium tuberculosis (MTB) and rifampicin (RIF)
resistance.MethodsThe limits of detection (LOD), sensitivity,
specificity, and RIF-R rpoB mutation detection of the iFIND TBR were
tested on Mycobacterium tuberculosis DNA or sputum samples spiked with
known numbers of M.tuberculosis H37Rv. Frozen clinical samples from
patients suspected of having TB were also tested.ResultsThe LOD of the
iFIND TBR for MTB detection were 13.34 CFU/ml (95% CI, 11.71-16.47), and
for RIF resistance was 109.79CFU/mL (95% CI, 95-138.19). The iFIND TBR
assay accurately distinguish MTB strains from non-tuberculous
mycobacteria (NTM) without any cross reactivity. Testing on 157 clinical
sputum samples, compared with the bacteriologically TB standard, the
overall sensitivity and specificity of the iFIND TBR was 100% (95%CI,
94.64, 100) and 85.29% (95% CI, 74.61, 92.72), respectively. When
assessing RIF susceptibility, the iFIND TBR achieved a sensitivity of
98.15% (95% CI, 90.11–99.95) and a specificity of 85.71% (95% CI,
67.33–95.97), compared with phenotypic drug susceptibility testing.
Discordant RIF susceptibility results were more frequently observed in
samples exhibiting heteroresistance.DiscussionThese findings demonstrate
that iFIND TBR assay performs well in detecting TB and RIF resistance,
and shows promise as a point-of-care tool in resource-limited areas.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1439099"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1439099</a></p>
<hr />
<h2
id="scalable-solutions-for-global-health-the-salivadirect-model">Scalable
solutions for global health: the SalivaDirect model</h2>
<p>date: 2024-10-29, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The COVID-19 pandemic caught the world unprepared. Large-scale
testing efforts were urgently needed, and diagnostic strategies had to
rapidly evolve in response to unprecedented worldwide demand. However,
the rollout of diagnostic testing and screening for SARS-CoV-2 was often
impeded by logistical challenges, including regulatory delays, workforce
shortages, laboratory bottlenecks, and supply chain disruptions.
Recognizing these hurdles early on, we developed a testing approach that
supported frequent, repeat testing, particularly as communities
reopened. We hypothesized and experimentally demonstrated that saliva
was a suitable specimen for the detection of SARS-CoV-2. This finding
was advanced into the development of open-source, extraction-free
reverse transcription polymerase chain reaction protocols using readily
available, “off-the-shelf” reagents and equipment for the direct
detection of SARS-CoV-2 in saliva (“SalivaDirect’’). Working with the US
Food and Drug Administration (FDA), we established a novel regulatory
framework wherein the FDA granted Emergency Use Authorization to Yale
University to offer the SalivaDirect test protocol to high-complexity
diagnostic laboratories (as designated by the Clinical Laboratory
Improvement Amendments) with quality oversight provided by Yale. This
grew into a network of more than 200 labs across the United States that,
as of May 2024, resulted in over 6.5 million SARS-CoV-2 tests. By making
the protocol flexible and open-source, laboratories were able to rapidly
and economically scale testing using a simple, self-collected saliva
specimen. Additionally, fostering a national network of laboratories
enabled real-time exchanges, problem solving, and the development of
community best practices. Preparing for the next pandemic, or simply the
next seasonal epidemic, the SalivaDirect model of deploying a readily
available, expandable solution and accompanying network provides a
proven method for the successful implementation of pathogen testing in
the United States and globally.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446514"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446514</a></p>
<hr />
<h2
id="nitric-oxide-producing-monocyte-myeloid-suppressor-cells-expand-and-accumulate-in-the-spleen-and-mesenteric-lymph-nodes-of-yersinia-enterocolitica-infected-mice">Nitric
oxide-producing monocyte-myeloid suppressor cells expand and accumulate
in the spleen and mesenteric lymph nodes of Yersinia
enterocolitica-infected mice</h2>
<p>date: 2024-10-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionYersinia enterocolitica (Ye) is a Gram-negative bacterium
that causes gastrointestinal infections. The myeloid-derived suppressor
cells (MDSCs) constitute a cellular population with the capacity of
inducing the specific suppression of T cells. Although there is evidence
supporting the role of MDSCs in controlling the immune responses in
several bacterial infections, its role during Ye infection has not yet
been reported. Therefore, the purpose of the present work was to analyze
MDSCs after oral Ye infection.MethodsC57BL/6 wild-type mice were
infected with Ye WAP-314 serotype O:8. The proliferation of splenocytes
and mesenteric lymph nodes (MLN) cells was measured as well as the
levels of cytokines and nitric oxide (NO) in culture supernatants. The
frequency and subsets of MDSCs were analyzed in the intestinal mucosa
and spleen by flow cytometry. Furthermore, monocytic-MDSCs (Mo-MDSCs)
and polymorphonuclear-MDSCs (PMN-MDSCs) were purified from the spleen of
infected mice and their suppressor activity was evaluated in co-cultures
with purified T cells.Resultswe observed a marked expansion of
CD11b+Gr-1+ cells, a phenotype consistent with MDSCs, in the spleen and
intestinal mucosa of Ye-infected mice. Interestingly, a robust
proliferation of splenocytes and MLN cells was observed only when the
MDSCs were depleted or the NO production was blocked. In addition, we
determined that only Mo-MDSCs had the ability to suppress T-cell
proliferation.ConclusionOur results highlight a mechanism by which Ye
may induce suppression of the immune responses. We suggest that
NO-producing Mo-MDSCs expand and accumulate in MLN and spleen of
Ye-infected mice. These cells can then suppress the T-cell function
without interfering with the anti-bacterial effector response. Instead,
these immature myeloid cells may perform an important function in
regulating the inflammatory response and protecting affected
tissues.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1440514"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1440514</a></p>
<hr />
<h2
id="co-frequency-or-contrary-the-effects-of-qiwei-baizhu-powder-and-its-bioactive-compounds-on-mucosa-associated-microbiota-of-mice-with-antibiotic-associated-diarrhea">Co-frequency
or contrary? The effects of Qiwei Baizhu Powder and its bioactive
compounds on mucosa-associated microbiota of mice with
antibiotic-associated diarrhea</h2>
<p>date: 2024-10-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Qiwei Baizhu Powder (QWBZP) has been proven effective in treating
antibiotic-associated diarrhea (AAD), and the mechanism is associated
with regulating the gut microbiota. However, the role of the bioactive
compounds of QWBZP in regulating the gut microbiota is still unclear. In
this study, 24 mice were divided into a normal control group (N), a
model group (R), a QWBZP decoction group (TW), and a QWBZP-TG group
(TG). AAD mouse models were established by mixed antibiotic
administration. After modeling, mice in the TW group and TG group were
treated with QWBZP decoction and QWBZP-TG, respectively. Mice in the N
group and R group were gavaged with sterile water. 16S rRNA gene
sequencing was used to investigate the changes of mucosa-associated
microbiota (MAM) in the small intestine of mice. Moreover, the levels of
diamine oxidase (DAO), D-Lactate, secretory immunoglobulin A (sIgA),
interleukin 6 (IL-6), IL-10, and tumor necrosis factor-α (TNF-α) were
detected using enzyme-linked immunosorbent assay (ELISA) kits. The
results showed that QWBZP-TG significantly altered the diversity,
structure, and abundance of MAM in the AAD mice. QWBZP-TG exerted a
stronger suppression effect on Escherichia and Clostridium compared with
QWBZP decoction. Meanwhile, QWBZP-TG downregulated the abundance of
Lactobacillus, which elicited an opposite effect to QWBZP decoction.
Prevotella was the signature bacteria that responded to the QWBZP-TG
intervention. Furthermore, both QWBZP decoction and QWBZP-TG decreased
the levels of DAO, D-Lactate, sIgA, IL-6, and TNF-α in the AAD mice. The
role of glycosides is to help QWBZP ameliorate diarrhea symptoms by
inhibiting the proliferation of diarrhea-associated bacteria, reducing
inflammation and regulating immunity.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1483048"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1483048</a></p>
<hr />
<h2
id="ifn-treated-macrophage-derived-exosomes-prevents-hbv-hcc-migration-and-invasion-via-regulating-mir-106b-3ppcgf3pi3kakt-signaling-axis">IFN-treated
macrophage-derived exosomes prevents HBV-HCC migration and invasion via
regulating miR-106b-3p/PCGF3/PI3K/AKT signaling axis</h2>
<p>date: 2024-10-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundStudies revealed that exosomes from IFN-α-treated liver
non-parenchymal cells (IFN-exo) mediate antiviral activity. MiR-106b-3p
has been shown to play a paradoxical role in disease progressing from
different studies. However, its specific role in HBV-related
hepatocellular carcinoma (HBV-HCC) and the underlying mechanism remains
unclear.MethodHuh7 cells transient transfected with plasmids of HBV-C2
and B3 were co-cultured with IFN-exo. Cell supernatants were collected
to detect miR-106b-3p, HBsAg, HBeAg and HBV DNA levels. Cell
proliferation, apoptosis, migration and invasion were analyzed. The
putative targets of miR-106b-3p were identified by a dual-luciferase
reporter system. The expression of PCGF3, migratory proteins(MMP2/9),
and the PI3K/AKT signaling pathway-related proteins were assessed by
western blot. The expression of PCGF3 mRNA was quantitative analyzed by
using 52 pairs of paraffin-embedded tissues from HCC patients.
siRNAs-PCGF3 were used to knocked-down PCGF3 expression.ResultsThe
expression of miR-106b-3p was significantly higher in THP-1 cells and
supernatants treated with IFN-exo than those untreated. Significantly
increased expression of miR-106b-3p and decreased expression of HBsAg
and HBV DNA were observed in Huh7-C2/B3 cells treated with IFN-exo. In
addition, miR-106b-3p was directly target to PCGF3. Scratch healing
assay and transwell assay showed that either IFN-exo or miRNA-106-3p
over-expression, or siRNAs-PCGF3 inhibited migration and invasion of
Huh7-C2/B3 cells, and subsequently resulted in suppression of p-AKT/AKT
and p-PI3K/PI3K. Notably, the expression level of PCGF3 was
significantly lower in HBeAg (+)-HCC tumor tissues than HBeAg (-)-HCC
tumor.ConclusionIFN-α-induced macrophage-derived miR-106b-3p inhibits
HBV replication, HBV- Huh7 cells migration and invasion via regulating
PCGF3/PI3K/AKT signaling axis. miR-106b-3p and PCGF3 were potential
biomarkers in the prevention and treatment of HBV-HCC.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1421195"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1421195</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
