---
title: health 2024.30
updated: 2024-07-22 08:22:26
---

# health 2024.30

(date: 2024-07-22 08:22:26)

---

## Elevation of IL-8 secretion induced by PEDV infection via NF-κB signaling pathway

date: 2024-07-22, from: Frontiers in Cellular and Infection Microbiology

Porcine epidemic diarrhea virus (PEDV) is associated with severe enteritis, which contributes to high mortality in piglets. The aim of this study was to describe molecular mechanisms associated with proinflammatory cytokine(s) production during PEDV infection. We showed that infection of porcine intestine epithelial cell clone J2 (IPEC-J2) with PEDV induces a gradual increase in interleukin 8 (IL-8) production at different time points, as well as infection of Vero E6 with PEDV. The secretion of IL-8 in these two cell lines infected with PEDV is related to the activation of NF-κB. Furthermore, the cells expressing PEDV M or E protein can induce the upregulation of IL-8. These findings suggest that the IL-8 production can be the initiator of inflammatory response by the host cells upon PEDV infection. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1422560>

---

## Biliary atresia: the role of gut microbiome, and microbial metabolites

date: 2024-07-22, from: Frontiers in Cellular and Infection Microbiology

Biliary atresia (BA) is a progressive fibroinflammatory disease affecting both the extrahepatic and intrahepatic bile ducts, potentially leading to chronic cholestasis and biliary cirrhosis. Despite its prevalence, the exact mechanisms behind BA development remain incompletely understood. Recent research suggests that the gut microbiota and its metabolites may play significant roles in BA development. This paper offers a comprehensive review of the changing characteristics of gut microbiota and their metabolites at different stages of BA in children. It discusses their influence on the host’s inflammatory response, immune system, and bile acid metabolism. The review also explores the potential of gut microbiota and metabolites as a therapeutic target for BA, with interventions like butyrate and gut microbiota preparations showing promise in alleviating BA symptoms. While progress has been made, further research is necessary to untangle the complex interactions between gut microbiota and BA, paving the way for more effective prevention and treatment strategies for this challenging condition. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411843>

---

## Molecular and phenotypic characterization of Streptococcus pneumoniae isolates in a Japanese tertiary care hospital

date: 2024-07-22, from: Frontiers in Cellular and Infection Microbiology

This study aimed to investigate the bacterial characteristics of pneumococcal isolates obtained from a tertiary care hospital in Japan. We analyzed the antimicrobial susceptibility, possession of macrolide resistance genes, pneumococcal serogroup/serotype, and sequence type (ST) of pneumococcal isolates from patients aged 15 years or older between 2011 and 2020 at Nagasaki University Hospital. Of the 73 isolates analyzed, 86.3% showed resistance to macrolides, and 28.8%, 46.6%, and 11.0% harbored mefA, ermB, and both, respectively. Of the isolates possessing ermB, 97.6% showed high levels of macrolide resistance [minimal inhibitory concentration (MIC) range, > 16 µg/mL]. Solithromycin (MIC range, 0.03–0.25 µg/mL), regardless of the presence of macrolide resistance genes, and lascufloxacin (MIC range, 0.06–0.5 µg/mL) showed potent in vitro activity against pneumococci. Serotype 19A was the most prevalent (six isolates), followed by serotypes 10A, 15A, and 15B/C (five isolates each). Four serotypes (11A, 19A, 22F, and 23B) and five STs (36, 99, 433, 558, and 3111) were significantly correlated with the presence of macrolide resistance genes. All four isolates with serotype 11A/ST99 and three isolates with serotype 19A/ST3111 harbored both mefA and ermB. No macrolide resistance genes were detected in either of the two isolates with serotype 22F/ST433, while all ten isolates with serogroup 15 (serotypes 15A and 15B/C, five isolates each) possessed ermB alone. Our study revealed the bacterial characteristics of the pneumococcal isolates obtained from our hospital. In vitro activity of solithromycin and lascufloxacin against these isolates was confirmed. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391879>

---

## Genome composition-based deep learning predicts oncogenic potential of HPVs

date: 2024-07-22, from: Frontiers in Cellular and Infection Microbiology

Human papillomaviruses (HPVs) account for more than 30% of cancer cases, with definite identification of the oncogenic role of viral E6 and E7 genes. However, the identification of high-risk HPV genotypes has largely relied on lagged biological exploration and clinical observation, with types unclassified and oncogenicity unknown for many HPVs. In the present study, we retrieved and cleaned HPV sequence records with high quality and analyzed their genomic compositional traits of dinucleotide (DNT) and DNT representation (DCR) to overview the distribution difference among various types of HPVs. Then, a deep learning model was built to predict the oncogenic potential of all HPVs based on E6 and E7 genes. Our results showed that the main three groups of Alpha, Beta, and Gamma HPVs were clearly separated between/among types in the DCR trait for either E6 or E7 coding sequence (CDS) and were clustered within the same group. Moreover, the DCR data of either E6 or E7 were learnable with a convolutional neural network (CNN) model. Either CNN classifier predicted accurately the oncogenicity label of high and low oncogenic HPVs. In summary, the compositional traits of HPV oncogenicity-related genes E6 and E7 were much different between the high and low oncogenic HPVs, and the compositional trait of the DCR-based deep learning classifier predicted the oncogenic phenotype accurately of HPVs. The trained predictor in this study will facilitate the identification of HPV oncogenicity, particularly for those HPVs without clear genotype or phenotype. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1430424>

