---
title: health
updated: 2024-10-25 08:11:27
---

# health

(date: 2024-10-25 08:11:27)

---

## Corrigendum: Development and validation of next-generation sequencing panel for personalized Helicobacter pylori eradication treatment targeting multiple species

date: 2024-10-25, from: Frontiers in Cellular and Infection Microbiology

 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1502072>

---

## Identification and characterization of LuxR solo homolog PplR in pathogenic Pseudomonas plecoglossicida NB2011

date: 2024-10-25, from: Frontiers in Cellular and Infection Microbiology

Pseudomonas plecoglossicida is a causative agent of visceral granulomas in large yellow croaker (Larimichthys crocea). Quorum sensing (QS) is widely involved in imparting virulence to pathogenic bacteria; however, it has not been studied in P. plecoglossicida. In this study, we annotated a LuxR family transcriptional regulator in P. plecoglossicida NB2011 and designated as PplR. We aligned the protein sequence by BlastP and Clustal X2, monitored the N-acyl-homoserine lactone (AHL) signal production through cross-feeding bioassay and HC-MS/MS; investigated exogenous AHL signal binding by recombinant expression and thin layer chromatography; constructed a deletion mutant of the target gene by method of double homologous recombination; sequenced the transcript RNA and analyzed the data; additionally, characterized phenotypes of wild type and mutant strain. The LuxR homolog PplR was found to share high similarity with PpoR—the LuxR solo of Pseudomonas putida—without a cognate LuxI. The wild-type strain did not produce any AHL signals and the recombinant LuxR protein was found to bind C6-L-homoserine lactone (C6-HSL), C8-HSL, 3-oxo-C10-HSL, and 3-oxo-C12-HSL. RNA-seq analysis indicated 84 differentially expressed genes—5 upregulated and 79 downregulated—mainly enriched in gene ontology terms, such as flagella-dependent motility, integral component of membrane, DNA binding and transcription, and metal ion binding, suggesting that PplR is a master transcription regulator. The mutant strain showed attenuated biofilm-forming ability and stress resistance, and the data support a role for PplR in the regulation of these traits in P. plecoglossicida NB2011 independent of the presence of AHL signals. This is the first study to provide QS-related information on P. plecoglossicida. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1458976>

---

## Phage-encoded depolymerases as a strategy for combating multidrug-resistant Acinetobacter baumannii

date: 2024-10-24, from: Frontiers in Cellular and Infection Microbiology

Acinetobacter baumannii, a predominant nosocomial pathogen, represents a grave threat to public health due to its multiple antimicrobial resistance. Managing patients afflicted with severe infections caused by multiple drug-resistant A. baumannii is particularly challenging, given the associated high mortality rates and unfavorable prognoses. The diminishing efficacy of antibiotics against this superbug underscores the urgent necessity for novel treatments or strategies to address this formidable issue. Bacteriophage-derived polysaccharide depolymerase enzymes present a potential approach to combating this pathogen. These enzymes target and degrade the bacterial cell’s exopolysaccharide, capsular polysaccharide, and lipopolysaccharide, thereby disrupting biofilm formation and impairing the bacteria’s defense mechanisms. Nonetheless, the narrow host range of phage depolymerases limits their therapeutic efficacy. Despite the benefits of these enzymes, phage-resistant strains have been identified, highlighting the complexity of phage-host interactions and the need for further investigation. While preliminary findings are encouraging, current investigations are limited, and clinical trials are imperative to advance this treatment approach for broader clinical applications. This review explores the potential of phage-derived depolymerase enzymes against A. baumannii infections. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1462620>

---

## Virome analysis provides new insights into the pathogenesis mechanism and treatment of SLE disease

date: 2024-10-24, from: Frontiers in Cellular and Infection Microbiology

IntroductionThis study aimed to investigate the virome diversity of the SLE disease and the association between viral infections and the disease.MethodsSLE-related RNA-Seq data were retrieved from public databases. A rigorous computational workflow was employed to identify the human viruses. Differential expression analysis and functional enrichment analysis were conducted in R.ResultsWe identified ten human virus species from 826 RNA-Seq samples of human blood, comprising 688 SLE patients and 138 healthy controls. Eight of the ten virus species exhibited higher positive rates in SLE patients compared to healthy controls, with Human betaherpesvirus 5 (HHV5) having the highest positive rate (4.1%) and being exclusively detected in SLE samples. The virus abundances were low and comparable in both SLE patients and healthy controls. Analysis of the antiviral interferon-stimulated genes (ISGs) in samples showed higher ISG expression levels in HHV4 and HHV5-positive samples compared to virus-negative samples. Several genes that were up-regulated in SLE patients were further up-regulated after HHV5 infection, and they were mainly enriched in immune response-related biological processes. Additionally, the expression levels of several marker genes of SLE severity were compared between HHV5-positive and virus-negative SLE patients, suggesting that HHV5 infection may be associated with aggravated SLE disease.DiscussionWe found that SLE patients are more susceptible to viral infections than healthy individuals. Viral infections, such as HHV5, may be associated with aggravated SLE disease. This study deepens our understanding of the association between viruses and SLE and provides new insights into prevention and control of the disease. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1484529>

---

## Editorial: Immune response to gram-negative bacteria in the lungs

date: 2024-10-24, from: Frontiers in Cellular and Infection Microbiology

 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1503892>

---

## Whole genome sequencing distinguishes skin colonizing from infection-associated Cutibacterium acnes isolates

date: 2024-10-24, from: Frontiers in Cellular and Infection Microbiology

IntroductionCutibacterium acnes can both be a helpful colonizer of the human skin as well as the causative agent of acne and purulent infections. Until today, it is a moot point whether there are C. acnes strains exclusively devoted to be part of the skin microbiome and others, that carry special features enabling them to cause disease. So far, the search for the molecular background of such diverse behavior has led to inconsistent results.MethodsIn the present study, we prospectively collected C. acnes strains from 27 infected persons and 18 healthy controls employing rigid selection criteria to ensure their role as infectious agent or colonizer. The genome sequences from these strains were obtained and carefully controlled for quality.ResultsDeduced traditional phylotyping assigned almost all superficial isolates to type IA1, while the clinical strains were evenly distributed between types IA1, IB, and II. Single locus sequence typing (SLST) showed a predominance of A1 type for the control strains, whereas 56% of the clinical isolates belonged to types A1, H1 and K8. Pangenome analysis from all the present strains and 30 published genomes indicated the presence of an open pangenome. Except for three isolates, the colonizing strains clustered in clades separate from the majority of clinical strains, while 4 clinical strains clustered with the control strains. Identical results were obtained by a single nucleotide polymorphism (SNP) analysis. However, there were no significant differences in virulence gene contents in both groups.DiscussionGenome-wide association studies (GWAS) from both the pangenome and SNP data consistently showed genomic differences between both groups located in metabolic pathway and DNA repair genes. Thus, the different behavior of colonizing and infectious C. acnes strains could be due to special metabolic capacities or flexibilities rather than specific virulence traits 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1433783>

---

## Species and genetic diversity of nontuberculous mycobacteria in suspected tuberculosis cases in East Azerbaijan, Iran: a cross-sectional analysis

date: 2024-10-24, from: Frontiers in Cellular and Infection Microbiology

IntroductionThe incidence of nontuberculous mycobacterial (NTM) infections has increased worldwide, attracting attention in routine diagnostic settings, particularly among patients with suspected tuberculosis. This study aimed to acquire knowledge of NTM infections in patients with suspected tuberculosis and to evaluate the genetic diversity of the strains.MethodsIn this study, 230 clinical specimens were collected from suspected tuberculosis patients. Following decontamination with N-Acetyl-L-cysteine–sodium hydroxide (NALC-NaOH), the sediments of specimens were inoculated onto Löwenstein–Jensen medium and then incubated at 37°C for 8 weeks. The samples that yielded positive cultures were evaluated through the sequencing of conserved fragments of IS6110 and hsp65. For those samples that were not identified as part of the M. tuberculosis complex (MTC) by IS6110 PCR, further analysis was conducted via PCR to detect fragments of the hsp65 gene.ResultsTwenty-one NTM species were isolated from 230 clinical specimens (14 NTM from pulmonary specimens and 7 from extrapulmonary specimens). Among these, 12 (57.14%) were rapid-growing mycobacteria (RGM), and 9 (42.85%) were slow-growing mycobacteria (SGM). No M. avium complex (MAC) was identified in any of the specimens. Notably, the M. kansasii, M. gordonae, and M. abscessus strains presented significant genetic diversity.ConclusionsThe prevalence of infections attributed to nontuberculous species surpasses that attributed to tuberculosis. These findings underscore the importance of exploring NTM species in individuals suspected of having TB. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477015>

---

## Phenotypic and genetic characterization of daptomycin non-susceptible Staphylococcus aureus strains selected by adaptive laboratory evolution

date: 2024-10-24, from: Frontiers in Cellular and Infection Microbiology

BackgroundDaptomycin non-susceptible Staphylococcus aureus (DNS) strains pose a serious clinical threat, yet their characteristics remain poorly understood.MethodsDNS derivatives were generated by exposing S. aureus strains to subinhibitory concentrations of daptomycin. Competition experiment and growth kinetics experiment were used to observe the growth of bacteria. Galleria mellonella larvae and mouse skin abscess models were used to observe the virulence of bacteria. Transmission electron microscopy (TEM), cytochrome C experiment and biofilm formation experiment were used to observe the drug resistance phenotype. And homologous recombination was used to study the role of mutations.ResultsPhenotypic profiling of DNS strains revealed impaired growth, increased cell wall thickness, enhanced biofilm formation, reduced negative surface charge, and attenuated virulence compared to their wild-type strains. Whole genome sequencing identified mutations in mprF, cls2, and saeR in DNS strains. Allelic replacement experiments validated the roles of MprF L341F and Cls2 F60S substitutions in augmenting daptomycin non-susceptibility in Newman. Deletion of saeR in the NewmanMprFL341F strain and complementation of saeR in the Newman-DNS strain did not directly alter daptomycin susceptibility. However, the deletion of saeR was found to enhance competitive fitness under daptomycin pressure.ConclusionThis work validates adaptive laboratory evolution (ALE) for modeling clinical DNS strains and uncovers contributions of mprF, cls2, and saeR mutations to the adaptation and resistance mechanisms of S. aureus against daptomycin. These findings enrich our understanding of how S. aureus acquired resistance to daptomycin, thus paving the way for the development of more effective treatment strategies and offering potential molecular markers for resistance surveillance. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1453233>

---

## Prevalence and clinical significance of the genotypic carriage among ESBL phenotype-negative Escherichia coli and Klebsiella pneumoniae clinical isolates in bacteremia: a study in a Malaysian tertiary center

date: 2024-10-24, from: Frontiers in Cellular and Infection Microbiology

BackgroundAntimicrobial resistance (AMR) can lead to fatal consequences. AMR genes carriage by phenotypically susceptible bacteria, such as Extended-Spectrum β-Lactamases (ESBL)s in Enterobacteriaceae, have potential implications for AMR spread and therapeutic outcomes. This phenomenon should be investigated.MethodsPositive blood cultures from hospitalized patients in a Malaysian tertiary center between April 2022 and March 2023 were reviewed. A total of 137 clinical isolates of Escherichia coli (E.coli), Klebsiella pneumoniae (K.pneumoniae), and Klebsiella oxytoca were included. The antibiotic susceptibility and ESBL phenotypes were determined by disk diffusion method and the identification of genotypes by multiplex polymerase chain reaction. The clinical characteristics and outcome information were extracted by reviewing patients’ medical records to evaluate the clinical significance of the ESBL genotype-positive but phenotype-negative isolates in bacteremia.ResultsAll 137 isolates were positive for at least one genotype (blaCTX-M, n = 71, 51.8%; blaSHV, n = 87, 63.5%; blaTEM, n = 95, 69.3%; blaOXA-1, n = 38, 27.7%). While blaCTX-M was proportionately higher in the ESBL phenotype-positive isolates than ESBL phenotype-negative isolates (33/37, 89.2% vs 38/100, 38%; p < 0.001), more than half of those harboring blaCTX-M remained susceptible to third-generation cephalosporins (3GC). The sensitivity (Sen) of blaCTX-M for ESBL phenotypes prediction was 89.19% (95% confidence interval [CI], 74.58 - 96.97%); however, specificity (Sp) was low (46.47%; 95% CI 39.75 - 53.32). The patient characteristics were similar among 98 ESBL phenotype-negative cases, except that the non-blaCTX-M carrier group had significantly more renal impairment (0/37 vs 7/61, p = 0.043) and gastrointestinal sources of bacteremia (9/37 vs 27/61, p = 0.047). No differences were observed in infection severity, in-hospital mortality, and length of stay (LOS) between the blaCTX-M and non-blaCTX-M carrier groups.ConclusionThe current study provides insight into the gene carriage in E.coli and Klebsiella species clinical isolates, including blaCTX-M genotypes in antibiotic-susceptible strains from a Malaysian hospital. The ESBL encoding genotypes such as blaCTX-M presented substantially beyond one-third of the ESBL phenotype-negative or 3GC susceptible E.coli and K.pneumoniae isolated from bloodstream infection. Although clinical outcomes were not worsened with blaCTX-M genotype-positive but ESBL phenotype-negative isolates in bacteremia, the potential implications for AMR spread deserve further investigation. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1429830>

---

## Non-classical monocyte levels correlate negatively with HIV-associated cerebral small vessel disease and cognitive performance

date: 2024-10-23, from: Frontiers in Cellular and Infection Microbiology

BackgroundDespite antiretroviral treatment (cART), aging people living with HIV (PWH) are more susceptible to neurocognitive impairment (NCI) probably due to synergistic/additive contribution of traditional cerebrovascular risk factors. Specifically, transmigration of inflammatory CD16+ monocytes through the altered blood brain barrier (BBB) may exacerbate cerebral small vessel disease (CSVD), a known cause of vascular cognitive impairment.MethodsPWH on cART (n=108) and age, sex, and Reynold’s cardiovascular risk score-matched uninfected individuals (PWoH, n=111) were enrolled. This is a longitudinal observational study but only cross-sectional data from entry visit are reported. Neuropsychological testing and brain magnetic resonance imaging (MRI) were performed. CSVD was diagnosed by Fazekas score ≥1. Flow cytometric analyses of fresh whole blood were conducted to evaluate circulating levels of monocyte subsets (classical, intermediate, and non-classical) and markers of monocyte activation (CCR2, CD40, PSGL-1, TNFR2 and tissue factor). ELISAs were used to measure sCD14, ICAM, and Osteoprotegerin. Two-way analysis of variance (ANOVA), and linear regression models were performed to study the effects of HIV status, CSVD status, and their interaction to outcome variables such as cognitive score. Two-sample t-tests and correlation analyses were performed between and within PWoH with CSVD and PWH with CSVD participants.ResultsPWH with CSVD (n=81) had significantly lower total cognitive scores, higher levels of NCMs and soluble CD14 and intracellular adhesion molecule 1 (ICAM-1) as compared to PWoH with CSVD group (n=68). sCD14 and ICAM1 were positively correlated with each other indicating that monocyte and endothelial activation are associated with each other. Cognition was negatively correlated with NCMs, especially in the PWH with CSVD group. Among other blood biomarkers measured, osteoprotegerin levels showed mild negative correlation with cognitive performance in individuals with CSVD irrespective of HIV status.ConclusionsElevated levels of NCMs may contribute to neuroinflammation, CSVD and subsequent cognitive impairment. This finding is of particular relevance in aging PWH as both HIV and aging are associated with increased levels of NCMs. NCMs may serve as a potential biomarker to address these comorbidities. Further longitudinal studies are needed to evaluate whether changes in NCM levels are associated with changes in CSVD burden and cognitive impairment. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1405431>

---

## Changes functional prediction of ear canal flora in chronic bacterial otitis externa

date: 2024-10-23, from: Frontiers in Cellular and Infection Microbiology

ObjectiveTo investigate ear canal microflora’s structure, composition and function in patients with chronic bacterial otitis externa.MethodsA case-control study design method was used to collect the ear canal secretions from 14 patients with chronic bacterial external otitis (CB group) and 14 healthy controls (H group) treated in the Department of Otolaryngology and Head and Neck Surgery in the First Affiliated Hospital of Hainan Medical University. 16S rRNA high-throughput sequencing technology was used to sequence the ear canal microflora’s V3 ~ V4 region gene amplification products in the participating population. The α diversity of ear canal microflora in 2 groups was analyzed. Based on the weighted Unifrac distance, principal coordinate analysis was performed to compare the β diversity of ear tract microflora between the two groups. The differences in ear microflora at phylum and genus levels were analyzed. PICRUSt2 function prediction and BugBase phenotype prediction were also performed.Resultsα diversity analysis showed that the diversity and richness of auricular microflora in the CB group were significantly lower than those in the H group. β diversity analysis showed that there were some differences between the two groups. At the phylum level, the relative abundance of Bdellovibrionota, Campylobacterota, and WPS-2 in the microbiota of patients in the CB group was significantly lower than that in the H group, and the differences were statistically significant. At the genus level, the relative abundance of Pseudomonas, Acinetobacter, Pelomonas, Sphingomonas, Bradyrhizobium, Brevundimonas, Enhydrobacter, Actinomyces, Paracoccus and Chryseobacterium in the ear canal of Group H is significantly higher than that of Group CB. Functional prediction of PICRUSt2 suggests that amino acid biosynthesis and bacterial microbiota may be related. In BugBase phenotypic prediction, the contribution of aerobic phenotype in group CB was significantly lower than that in group H.ConclusionThe diversity and abundance of the ear canal flora of patients with chronic bacterial otitis externa were significantly lower than those of the healthy population, and their bacterial colony structure was significantly altered. Dysbiosis of the ear canal flora may be an important cause of chronic bacterial otitis externa. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1434754>

---

## Target antigen screening and development of a multi-component subunit vaccine against Mycoplasma synoviae in chickens

date: 2024-10-23, from: Frontiers in Cellular and Infection Microbiology

IntroductionMycoplasma synoviae (MS) is a globally important avian pathogen causing infectious synovitis and respiratory diseases in poultry, leading to significant economic losses. Despite advances in vaccine development, a commercially viable subunit vaccine against MS remains elusive.MethodsWe sequenced whole genomes of six clinical MS strains isolated from different Chinese provinces. Common genes were analyzed using Biopython software, identifying those with high copy numbers in virulent strains and shared among all strains. Vaxign2 and IEDB Antibody Epitope Prediction were used to analyze protein properties. We assessed immune protective effects of candidate proteins and developed a multivalent subunit vaccine.ResultsTen candidate vaccine proteins were initially selected. A multivalent subunit vaccine composed of MSPB, Ppht, Cfba, and EF-G displayed the best protective effect. The optimal immunization dosage was 20μg, with each protein accounting for 25%. The immune production period was determined to be 28 days post-first immunization, lasting 180 days. The immune protection rate against highly virulent strains reached 90%∼100%.DiscussionThis study provides a new approach for screening vaccine antigens and develops an effective candidate vaccine for MS prevention. The multivalent subunit vaccine shows promising results in protecting against MS infections, potentially offering a solution to reduce economic losses in the poultry industry. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1458865>

---

## Brucella mediates autophagy, inflammation, and apoptosis to escape host killing

date: 2024-10-23, from: Frontiers in Cellular and Infection Microbiology

Brucellosis is a serious zoonosis caused by Brucella spp. infection, which not only seriously jeopardizes the health of humans and mammals, but also causes huge economic losses to the livestock industry. Brucella is a Gram-negative intracellular bacterium that relies primarily on its virulence factors and a variety of evolved survival strategies to replicate and proliferate within cells. Currently, the mechanisms of autophagy, inflammation, and apoptosis in Brucella-infected hosts are not fully understood and require further research and discussion. This review focuses on the relationship between Brucella and autophagy, inflammation, and apoptosis to provide the scientific basis for revealing the pathogenesis of Brucella. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1408407>

---

## Risk factors for identifying pneumocystis pneumonia in pediatric patients

date: 2024-10-23, from: Frontiers in Cellular and Infection Microbiology

ObjectivesThis study aimed to identify the risk factors and construct the diagnostic model associated with pneumocystis pneumonia (PCP) in pediatric patients.MethodsThis retrospective observational study analyzed 34 cases of PCP and 51 cases of other types of pneumonia treated at Children’s Hospital Affiliated to Shandong University between January 2021 and August 2023. Multivariate binary logistic regression was used to identify the risk factors associated with PCP. Receiver operating characteristic curves and calibration plots were constructed to evaluate the diagnostic model.ResultsTwenty clinical variables significantly differed between the PCP and non-PCP groups. Multivariate binary logistic regression analysis revealed that dyspnea, body temperature>36.5°C, and age<1.46 years old were risk factors for PCP. The area under the curve of the diagnostic model was 0.958, the P-value of Hosmer‐Lemeshow calibration test was 0.346, the R2 of the calibration plot for the actual and predicted probability of PCP was 0.9555 (P<0.001), and the mean Brier score was 0.069. In addition, metagenomic next-generation sequencing revealed 79.41% (27/34) and 52.93% (28/53) mixed infections in the PCP and non-PCP groups, respectively. There was significantly more co-infection with cytomegalovirus and Streptococcus pneumoniae in the PCP group than that in the non-PCP group (p<0.05).ConclusionsDyspnea, body temperature>36.5°C, and age<1.46 years old were found to be independent risk factors for PCP in pediatric patients. The probability of co-infection with cytomegalovirus and S. pneumoniae in the PCP group was significantly higher than that in the non-PCP group. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1398152>

---

## A rapid and visual detection assay for Senecavirus A based on recombinase-aided amplification and lateral flow dipstick

date: 2024-10-23, from: Frontiers in Cellular and Infection Microbiology

BackgroundSenecavirus A (SVA) is a newly pathogenic virus correlated with the acute death of piglets and vesicular lesions in pigs. The further prevalence of SVA will cause considerable economic damage to the global pig farming industry. Therefore, rapid and accurate diagnostic tools for SVA are crucial for preventing and controlling the disease.MethodsWe designed multiple primer pairs targeting the most conserved region of the SVA 3D gene and selected those with the highest specificity. Nfo-probes were subsequently developed based on the highest specificity primer pairs. Subsequently, the recombinase-assisted amplification (RAA) reaction was completed, and the reaction temperature and duration were optimized. The RAA amplicons were detected using a lateral flow device (LFD). Finally, a rapid and intuitive RAA-LFD assay was established against SVA.ResultsThe SVA RAA-LFD assay can be performed under reaction conditions of 35°C within 17 minutes, with results observable to the naked eye. We then evaluated the performance of this method. It exhibited high specificity and no cross-reaction with the other common swine pathogens. The lowest detectable limits of this method for the plasmid of pMD18-SVA-3D, DNA amplification product, and viral were 3.86×101 copies/µL, 8.76×10-7 ng/µL, and 1×100.25 TCID50/mL, respectively. A total of 44 clinical samples were then tested using the RAA-LFD, PCR, and RT-qPCR methods. The results demonstrated a consistent detection rate between the RAA-LFD and RT-qPCR assays.ConclusionThe SVA RAA-LFD assay developed in our study exhibits excellent specificity, sensitivity, and time-saving attributes, making it ideally suited for utilization in lack-instrumented laboratory and field settings. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474676>

---

## Genomic epidemiology and evolutionary dynamics of the Omicron variant of SARS-CoV-2 during the fifth wave of COVID-19 in Pakistan

date: 2024-10-22, from: Frontiers in Cellular and Infection Microbiology

IntroductionThe coronavirus disease 2019 (COVID-19) pandemic, caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has posed extraordinary challenges to global health systems and economies. The virus’s rapid evolution has resulted in several variants of concern (VOCs), including the highly transmissible Omicron variant, characterized by extensive mutations. In this study, we investigated the genetic diversity, population differentiation, and evolutionary dynamics of the Omicron VOC during the fifth wave of COVID-19 in Pakistan.MethodsA total of 954 Omicron genomes sequenced during the fifth wave of COVID-19 in Pakistan were analyzed. A Bayesian framework was employed for phylogenetic reconstructions, molecular dating, and population dynamics analysis.ResultsUsing a population genomics approach, we analyzed Pakistani Omicron samples, revealing low within-population genetic diversity and significant structural variation in the spike (S) protein. Phylogenetic analysis showed that the Omicron variant in Pakistan originated from two distinct lineages, BA.1 and BA.2, which were introduced from South Africa, Thailand, Spain, and Belgium. Omicron-specific mutations, including those in the receptor-binding domain, were identified. The estimated molecular evolutionary rate was 2.562E-3 mutations per site per year (95% HPD interval: 8.8067E-4 to 4.1462E-3). Bayesian skyline plot analysis indicated a significant population expansion at the end of 2021, coinciding with the global Omicron outbreak. Comparative analysis with other VOCs showed Omicron as a highly divergent, monophyletic group, suggesting a unique evolutionary pathway.ConclusionsThis study provides a comprehensive overview of Omicron’s genetic diversity, genomic epidemiology, and evolutionary dynamics in Pakistan, emphasizing the need for global collaboration in monitoring variants and enhancing pandemic preparedness. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1484637>

---

## Novel antimicrobial coating for hernia meshes

date: 2024-10-22, from: Frontiers in Cellular and Infection Microbiology

PurposeAntibiotic coating for several medical devices has been carried out; however, there are only few studies about coating hernia meshes with antimicrobial substances. In this study we checked the capacity of different commercially available hernia meshes to act as drug carrier.MethodsThe meshes were coated with gentamicin palmitate, chlorhexidine palmitic acid and chlorhexidine palmitate. The coating mass and subsequent in vitro delivery rate were evaluated for gentamicin palmitate by fluorescence polarization. For Chlorhexidine coated devices the coating mass was determined by weighing. The in vitro delivery rate was determined by UV absorption (255 nm). The interaction of each mesh to the different coating substances was observed by scanning electron microscopy.Results1. Certain uniformity was observed on the quantity of chlorhexidine coating the surface of each mesh used when compared with gentamicin palmitate coating. 2.We did not detect significant difference between the amounts of gentamicin palmitate released from each mesh. 3. The release of chlorhexidine palmitate and chlorhexidine palmitic acid from UltraPro™ and Mersilene™ were significantly higher (p<0.05) in comparison with the other two meshes. 4. The coating substances covered the surface of the fibers without damaging its structure. 5. The coating substances were distributed all along the fibers in all samples.ConclusionsWe suggest the use of chlorhexidine palmitate and chlorhexidine palmitic acid, as well as gentamicin palmitate, for coating of hernia meshes aiming prevention of infections. Further investigation of the bactericidal effect of coated hernia meshes against biofilm form of S. aureus and other device-related infections is suggested. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1383680>

---

## Alternative splicing in the genome of HPV and its regulation

date: 2024-10-22, from: Frontiers in Cellular and Infection Microbiology

Persistent infection with high-risk human papillomavirus (HR-HPV) is the main cause of cervical cancer. These chronic infections are characterized by high expression of the HPV E6 and E7 oncogenes and the absence of the L1 and L2 capsid proteins. The regulation of HPV gene expression plays a crucial role in both the viral life cycle and rare oncogenic events. Alternative splicing of HPV mRNA is a key mechanism in post-transcriptional regulation. Through alternative splicing, HPV mRNA is diversified into various splice isoforms with distinct coding potentials, encoding multiple proteins and influencing the expression of HPV genes. The spliced mRNAs derived from a donor splicing site within the E6 ORF and one of the different acceptor sites located in the early mRNA contain E6 truncated mRNAs, named E6*. E6* is one of the extensively studied splicing isoforms. However, the role of E6* proteins in cancer progression remains controversial. Here, we reviewed and compared the alternative splicing events occurring in the genomes of HR-HPV and LR-HPV. Recently, new HPV alternative splicing regulatory proteins have been continuously discovered, and we have updated the regulation of HPV alternative splicing. In addition, we summarized the functions of known splice isoforms from three aspects: anti-tumorigenic, tumorigenic, and other cancer-related functions, including not only E6*, but also E6^E7, E8^E2, and so on. Comprehending their contributions to cancer development enhances insights into the carcinogenic mechanisms of HPV and explores the potential utility of alternative splicing in the diagnosis and treatment of cervical cancer. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1443868>

---

## Apoptosis, inflammatory and innate immune responses induced by infection with a novel goose astrovirus in goose embryonic kidney cells

date: 2024-10-22, from: Frontiers in Cellular and Infection Microbiology

IntroductionSince 2016, a highly lethal visceral gout induced by infection with the novel goose astrovirus (GoAstV) resulted in an ongoing outbreak in goslings in China, with a mortality rate ranging from 10% to 50%, and causing considerable economic losses in the goose industry. However, the pathogenesis of GoAstV and the molecular mechanism by which kidney lesions are induced by GoAstV infection are unclear.MethodsIn the present study, a GEK cell infection model for GoAstV was established, and the apoptosis, inflammatory and innate immune responses induced by GoAstV were investigated in GEK cells.ResultsThe results shown that the expression of proapoptotic proteins, including Bax, caspase-3, caspase-9 and cytochrome c, increased in the infection group; however, the expression of the antiapoptotic protein Bcl-2 decreased, indicating that apoptosis was induced by GoAstV infection in GEK cells. Besides, the activation of the RIG-I/MDA5 pathway and the downstream upregulation of proinflammatory cytokines, including the adapter proteins MAVS, IRF7 and NF-κB and the proinflammatory cytokines IL-6, IL-8 and TNF-α, were detected in GEK cells infected with GoAstV. In addition, GoAstV infection induces the activation of the NLPR3 pathway and further stimulates the increased production of IL-1β. In summary, the present study revealed that GoAstV infection could induce apoptosis and the activation of the RIG-I/MDA5 and NLRP3 pathways in GEK cells, as well as the massive release of proinflammatory cytokines.DiscussionThese results are helpful for elucidating the molecular mechanism of pathological lesions in the kidney in gout goslings infected with GoAstV and the interaction between GoAstV and the innate immune system of the host. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1452158>

---

## Pathogenetic identification in ticks and yaks from Zoige County, China

date: 2024-10-21, from: Frontiers in Cellular and Infection Microbiology

Background:Ticks represent a significant vector for the transmission of infectious diseases, with the prevalence of tick-borne diseases becoming a prominent global health concern in recent decades. Anaplasma spp., Rickettsia spp., and Piroplasma have been identified as significant pathogens with the potential to impact human and animal health. However, there is a dearth of data concerning the prevalence of these pathogens in the eastern Tibetan Plateau, China.MethodsIn this study, a total of 643 Dermacentor silvarum and 314 Haemaphysalis longicornis were identified through the application of morphological and molecular identification techniques on 957 ticks collected from yaks in Zoige County. The assessed of Anaplasma spp., Rickettsia spp., Theileria spp., and Babesia spp. was assessed in 957 ticks and 96 blood samples collected from yaks.ResultsSignificant discrepancies were observed in the positivity rates for the four pathogens among the tick species and sampling sites. The identification of different species within the four pathogens was based on the analysis of the 16S rRNA of Anaplasma spp., the ompA and ompB genes of Rickettsia spp., and the 18S rRNA of Theileria spp. and Babesia spp. The prevalence ranges of the four pathogens are 9.9-50.2%, 29.5-100%, 16.2-46.4%, and 14.5-58.4%, respectively.ConclusionIn view of the growing zoonotic risks, further investigations into the prevalence of additional pathogens in ticks and animals, including livestock, in the eastern Tibetan Plateau, China, are essential. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474519>

---

## The ovarian cancer-associated microbiome contributes to the tumor’s inflammatory microenvironment

date: 2024-10-21, from: Frontiers in Cellular and Infection Microbiology

A growing body of research has established a correlation between tumors and persistent chronic inflammatory infiltration. As a primary instigator of inflammation, the majority of microbiomes naturally residing within our bodies engage in a mutually beneficial symbiotic relationship. Nevertheless, alterations in the microbiome's composition or breaches in the normal barrier function can disrupt the internal environment's homeostasis, potentially leading to the development and progression of various diseases, including tumors. The investigation of tumor-related microbiomes has contributed to a deeper understanding of their role in tumorigenesis. This review offers a comprehensive overview of the microbiome alterations and the associated inflammatory changes in ovarian cancer. It may aid in advancing research to elucidate the mechanisms underlying the ovarian cancer-associated microbiome, providing potential theoretical support for the future development of microbiome-targeted antitumor therapies and early screening through convenient methods. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1440742>

---

## Non-tuberculous mycobacteria enhance the tryptophan-kynurenine pathway to induce immunosuppression and facilitate pulmonary colonization

date: 2024-10-21, from: Frontiers in Cellular and Infection Microbiology

The increasing prevalence of non-tuberculous mycobacterium (NTM) infections alongside tuberculosis (TB) underscores a pressing public health challenge. Yet, the mechanisms governing their infection within the lung remain poorly understood. Here, we integrate metagenomic sequencing, metabolomic sequencing, machine learning classifiers, SparCC, and MetOrigin methods to profile bronchoalveolar lavage fluid (BALF) samples from NTM/TB patients. Our aim is to unravel the intricate interplay between lung microbial communities and NTM/Mycobacterium tuberculosis infections. Our investigation reveals a discernible reduction in the compositional diversity of the lung microbiota and a diminished degree of mutual interaction concomitant with NTM/TB infections. Notably, NTM patients exhibit a distinct microbial community characterized by marked specialization and notable enrichment of Pseudomonas aeruginosa and Staphylococcus aureus, driving pronounced niche specialization for NTM infection. Simultaneously, these microbial shifts significantly disrupt tryptophan metabolism in NTM infection, leading to an elevation of kynurenine. Mycobacterium intracellulare, Mycobacterium paraintracellulare, Mycobacterium abscessus, and Pseudomonas aeruginosa have been implicated in the metabolic pathways associated with the conversion of indole to tryptophan via tryptophan synthase within NTM patients. Additionally, indoleamine-2,3-dioxygenase converts tryptophan into kynurenine, fostering an immunosuppressive milieu during NTM infection. This strategic modulation supports microbial persistence, enabling evasion from immune surveillance and perpetuating a protracted state of NTM infection. The elucidation of these nuanced microbial and metabolic dynamics provides a profound understanding of the intricate processes underlying NTM and TB infections, offering potential avenues for therapeutic intervention and management. 

<br> 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1455605>

