<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.08</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="../../snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../search.html">Search</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.08</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.08">health 2024.08</h1>
<p>(date: 2024-02-24 18:31:22)</p>
<hr />
<h2
id="establishment-and-application-of-a-rapid-molecular-diagnostic-platform-for-the-isothermal-visual-amplification-of-group-b-streptococcus-based-on-recombinase-polymerase">Establishment
and application of a rapid molecular diagnostic platform for the
isothermal visual amplification of group B Streptococcus based on
recombinase polymerase</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
With growing concerns about Group B streptococcal (GBS) infections and
their adverse effects on perinatal pregnancies, including infection,
premature delivery, neonatal septicemia, and meningitis, it is urgent to
promote GBS screening at all pregnancy stages. The purpose of this study
is to establish a device-independent, fast, sensitive, and visual GBS
detection method. Taking advantage of the characteristics of the
recombinase polymerase isothermal amplification (RPA), the activity of
the nfo nuclease cleavage base analog (tetrahydrofuran, THF) site, and
the advantages of visual reading of the lateral flow chromatography
strip (LFS), a GBS detection method was developed. This method focused
on the conservative region of the Christie–Atkins–Munch–Petersen factor
encoded by the cfb gene, a virulence gene specific to GBS. Two forward
primers, two biotin-labeled reverse primers, and one fluorescein
isothiocyanate (FITC)-labeled and C3spacer-blocked probe were designed.
The study involved optimizing the primer pair and probe combination,
determining the optimal reaction temperature and time, evaluating
specificity, analyzing detection limits, and testing the method on 87
vaginal swabs from perinatal pregnant women. The results showed that the
visual detection method of GBS-RPA-LFS, using the cfb-F1/R2/P1 primer
probe, could detect GBS within 15 min at the temperature ranging from
39°C to 42°C. Furthermore, the method specifically amplified only GBS,
without cross-reacting with pathogens like <italic>Lactobacillus
iners</italic>, <italic>Lactobacillus crispatus</italic>,
<italic>Candida albicans</italic>, <italic>Listeria
monocytogenes</italic>, <italic>Yersinia enterocolitica</italic>,
<italic>Klebsiella Pneumoniae</italic>, <italic>Enterobacter
cloacae</italic>, <italic>Citrobacter freundii</italic>, <italic>Vibrio
alginolyticus</italic>, <italic>Vibrio parahaemolyticus</italic>,
<italic>Salmonella typhimurium</italic>, <italic>Staphylococcus
aureus</italic>, <italic>Pseudomonas aeruginosa</italic>, or
<italic>Trichomonas vaginalis</italic>. It could detect a minimum of 100
copies per reaction. In clinical 98 samples of vaginal swabs from
pregnant women, the agreement rate between the GBS-RPA-LFS method and
TaqMan real-time fluorescence quantification method was 95.92%. In
conclusion, this study successfully established a combined RPA and LFS
GBS <italic>in situ</italic> detection platform, with short reaction
time, high sensitivity, high specificity, portability, and device
independence, providing a feasible strategy for clinical GBS screening.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1281827">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1281827</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1281827">Save
to Pocket</a> </span></p>
<hr />
<h2 id="marmosets-as-models-of-infectious-diseases">Marmosets as models
of infectious diseases</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Animal models of infectious disease often serve a crucial purpose in
obtaining licensure of therapeutics and medical countermeasures,
particularly in situations where human trials are not feasible, i.e.,
for those diseases that occur infrequently in the human population. The
common marmoset (<italic>Callithrix jacchus</italic>), a Neotropical
new-world (platyrrhines) non-human primate, has gained increasing
attention as an animal model for a number of diseases given its small
size, availability and evolutionary proximity to humans. This review
aims to (i) discuss the pros and cons of the common marmoset as an
animal model by providing a brief snapshot of how marmosets are
currently utilized in biomedical research, (ii) summarize and evaluate
relevant aspects of the marmoset immune system to the study of
infectious diseases, (iii) provide a historical backdrop, outlining the
significance of infectious diseases and the importance of developing
reliable animal models to test novel therapeutics, and (iv) provide a
summary of infectious diseases for which a marmoset model exists,
followed by an in-depth discussion of the marmoset models of two studied
bacterial infectious diseases (tularemia and melioidosis) and one viral
infectious disease (viral hepatitis C).
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1340017">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1340017</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1340017">Save
to Pocket</a> </span></p>
<hr />
<h2
id="corrigendum-exploring-the-antibacterial-and-dermatitis-mitigating-properties-of-chicken-egg-white-synthesized-zinc-oxide-nano-whiskers">Corrigendum:
Exploring the antibacterial and dermatitis-mitigating properties of
chicken egg white-synthesized zinc oxide nano whiskers</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1376621">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1376621</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1376621">Save
to Pocket</a> </span></p>
<hr />
<h2
id="placebo-resistant-gut-bacteria-akkermansia-muciniphila-spp.-and-familial-mediterranean-fever-disease">Placebo-resistant
gut bacteria: Akkermansia muciniphila spp. and Familial Mediterranean
fever disease</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Despite numerous investigations into the impact of drugs/probiotics on
the gut microbiota composition in Familial Mediterranean Fever (FMF)
patients, the question as to whether there exists a significant
bacterial diversity(ies) independent of the placebo effect that can be
reliably considered in clinical and nutritional trials remains
unresolved.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
This study represents the in augural analysis of the placebo’s influence
on the gut microbiota of both healthy individuals and FMF afflicted men,
utilizing previously collected data from PhyloChip™ DNA microarray
experiments. A total of 15 healthy and 15 FMF male volunteers, aged 18
to 50, participated in this partially randomized placebo trial, which is
accessible through the GEO Series accession number GSE111835.
</p>
</sec><sec>
<title>
Results and Discussion
</title>
<p>
Key findings from current investigations include <italic>i.</italic> the
anticipated divergence in gut bacteria resistance to placebo between
healthy and FMF individuals, <italic>ii.</italic> the minor impact of
placebo on gut bacterial diversities in healthy individuals, with
<italic>Enterobacteriaceae</italic> diversities identified as
placebo-resistant among “healthy” gut bacteria, and
<italic>iii.</italic> the comprehensive influence of placebo on all
bacterial phyla in the gut microbiome of FMF patients, extending to
nearly all bacterial genera, except for the resilience of gut
<italic>Akkermansia muciniphila</italic> spp. to placebo in FMF
patients. This study underscores the susceptibility of
<italic>Faecalibacterium</italic>, <italic>Blautia</italic>, and
<italic>Clostridium</italic> genera to placebo. Consequently, this
investigation holds significance for the proper design of
placebo-controlled trials and establishes a foundation for further
exploration of the gut-brain axis. Furthermore, it contributes valuable
insights to discussions regarding proposals for probiotic therapies,
particularly focusing on <italic>Faecalibacterium</italic> spp.,
<italic>Blautia</italic> spp., and <italic>Clostridium</italic> spp.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336752">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336752</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1336752">Save
to Pocket</a> </span></p>
<hr />
<h2
id="relationship-between-biofilm-formation-and-antibiotic-resistance-of-klebsiella-pneumoniae-and-updates-on-antibiofilm-therapeutic-strategies">Relationship
between biofilm formation and antibiotic resistance of Klebsiella
pneumoniae and updates on antibiofilm therapeutic strategies</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Klebsiella pneumoniae</italic> is a Gram-negative bacterium
within the <italic>Enterobacteriaceae</italic> family that can cause
multiple systemic infections, such as respiratory, blood, liver
abscesses and urinary systems. Antibiotic resistance is a global health
threat and <italic>K. pneumoniae</italic> warrants special attention due
to its resistance to most modern day antibiotics. Biofilm formation is a
critical obstruction that enhances the antibiotic resistance of
<italic>K. pneumoniae</italic>. However, knowledge on the molecular
mechanisms of biofilm formation and its relation with antibiotic
resistance in <italic>K. pneumoniae</italic> is limited. Understanding
the molecular mechanisms of biofilm formation and its correlation with
antibiotic resistance is crucial for providing insight for the design of
new drugs to control and treat biofilm-related infections. In this
review, we summarize recent advances in genes contributing to the
biofilm formation of <italic>K. pneumoniae</italic>, new progress on the
relationship between biofilm formation and antibiotic resistance, and
new therapeutic strategies targeting biofilms. Finally, we discuss
future research directions that target biofilm formation and antibiotic
resistance of this priority pathogen.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324895">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324895</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1324895">Save
to Pocket</a> </span></p>
<hr />
<h2
id="the-construction-and-evaluation-of-secretory-expression-engineering-bacteria-for-the-trans-cry3aa-t-hasa-fusion-protein-against-the-monochamus-alternatus-vector">The
construction and evaluation of secretory expression engineering bacteria
for the trans-Cry3Aa-T-HasA fusion protein against the Monochamus
alternatus vector</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Pine wood nematode disease is currently the most deadly forest disease
in China, and the <italic>Monochamus alternatus</italic> is its primary
vector. Controlling the <italic>M. alternatus</italic> is crucial for
managing pine wood nematode disease. This study, based on the selected
HasA (pGHKW4) secretory expression vector, used electroporation to
combine the genetically modified high-toxicity toxin Cry3Aa-T with the
entomopathogenic bacterium <italic>Yersinia entomophaga</italic>
isolated from the gut of the <italic>M. alternatus</italic>. The
SDS-PAGE and Western blotting techniques were employed to confirm the
toxin protein’s secretion capability. The engineered bacteria’s genetic
stability and effectiveness in controlling <italic>M.
alternatus</italic> were assessed for their insecticidal activity. The
results of the SDS-PAGE and Western blotting analyses indicate that the
HasA system effectively expresses toxin protein secretion, demonstrates
certain genetic stability, and exhibits high insecticidal activity
against <italic>M. alternatus.</italic> This study constructed a highly
toxic entomopathogenic engineered bacterial strain against <italic>M.
alternatus</italic> larvae, which holds significant implications for
controlling <italic>M. alternatus</italic>, laying the foundation for
subsequent research and application of this strain.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1362961">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1362961</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1362961">Save
to Pocket</a> </span></p>
<hr />
<h2
id="picornavirus-2c-proteins-structure-function-relationships-and-interactions-with-host-factors">Picornavirus
2C proteins: structure-function relationships and interactions with host
factors</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Picornaviruses, which are positive-stranded, non-enveloped RNA viruses,
are known to infect people and animals with a broad spectrum of
diseases. Among the nonstructural proteins in picornaviruses, 2C
proteins are highly conserved and exhibit multiple structural domains,
including amphipathic α-helices, an ATPase structural domain, and a zinc
finger structural domain. This review offers a comprehensive overview of
the functional structures of picornaviruses’ 2C protein. We summarize
the mechanisms by which the 2C protein enhances viral replication. 2C
protein interacts with various host factors to form the replication
complex, ultimately promoting viral replication. We review the
mechanisms through which picornaviruses’ 2C proteins interact with the
NF-κB, RIG-I, MDA5, NOD2, and IFN pathways, contributing to the evasion
of the antiviral innate immune response. Additionally, we provide an
overview of broad-spectrum antiviral drugs for treating various
enterovirus infections, such as guanidine hydrochloride, fluoxetine, and
dibucaine derivatives. These drugs may exert their inhibitory effects on
viral infections by targeting interactions with 2C proteins. The review
underscores the need for further research to elucidate the precise
mechanisms of action of 2C proteins and to identify additional host
factors for potential therapeutic intervention. Overall, this review
contributes to a deeper understanding of picornaviruses and offers
insights into the antiviral strategies against these significant viral
pathogens.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347615">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347615</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1347615">Save
to Pocket</a> </span></p>
<hr />
<h2
id="characterization-of-merozoite-specific-thrombospondin-related-anonymous-protein-mtrap-in-plasmodium-vivax-and-p.-knowlesi-parasites">Characterization
of merozoite-specific thrombospondin-related anonymous protein (MTRAP)
in Plasmodium vivax and P. knowlesi parasites</h2>
<p>date: 2024-02-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Plasmodium vivax</italic>, the most widespread human malaria
parasite, and <italic>P. knowlesi</italic>, an emerging
<italic>Plasmodium</italic> that infects humans, are the
phylogenetically closest malarial species that infect humans, which may
induce cross-species reactivity across most co-endemic areas in
Southeast Asia. The thrombospondin-related anonymous protein (TRAP)
family is indispensable for motility and host cell invasion in the
growth and development of <italic>Plasmodium</italic> parasites. The
merozoite-specific TRAP (MTRAP), expressed in blood-stage merozoites, is
supposed to be essential for human erythrocyte invasion. We aimed to
characterize MTRAPs in blood-stage <italic>P. vivax</italic> and
<italic>P. knowlesi</italic> parasites and ascertain their cross-species
immunoreactivity. Recombinant <italic>P. vivax</italic> and <italic>P.
knowlesi</italic> MTRAPs of full-length ectodomains were expressed in a
mammalian expression system. The MTRAP-specific immunoglobulin G,
obtained from immune animals, was used in an immunofluorescence assay
for subcellular localization and invasion inhibitory activity in
blood-stage parasites was determined. The cross-species humoral immune
responses were analyzed in the sera of patients with <italic>P.
vivax</italic> or <italic>P. knowlesi</italic> infections. The MTRAPs of
<italic>P. vivax</italic> (PvMTRAP) and <italic>P. knowlesi</italic>
(PkMTRAP) were localized on the rhoptry body of merozoites in
blood-stage parasites. Both anti-PvMTRAP and anti-PkMTRAP antibodies
inhibited erythrocyte invasion of blood-stage <italic>P.
knowlesi</italic> parasites. The humoral immune response to PvMTRAP
showed high immunogenicity, longevity, and cross-species
immunoreactivity with <italic>P. knowlesi</italic>. MTRAPs are promising
candidates for development of vaccines and therapeutics against vivax
and knowlesi malaria.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354880">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354880</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354880">Save
to Pocket</a> </span></p>
<hr />
<h2
id="oral-infectivity-through-carnivorism-in-murine-model-of-trypanosoma-cruzi-infection">Oral
infectivity through carnivorism in murine model of Trypanosoma cruzi
infection</h2>
<p>date: 2024-02-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Oral transmission of <italic>T. cruzi</italic> is probably the most
frequent transmission mechanism in wild animals. This observation led to
the hypothesis that consuming raw or undercooked meat from animals
infected with <italic>T. cruzi</italic> may be responsible for
transmitting the infection. Therefore, the general objective of this
study was to investigate host-pathogen interactions between the parasite
and gastric mucosa and the role of meat consumption from infected
animals in the oral transmission of <italic>T. cruzi.</italic>
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Cell infectivity assays were performed on AGS cells in the presence or
absence of mucin, and the roles of pepsin and acidic pH were determined.
Moreover, groups of five female Balb/c mice were fed with muscle tissue
obtained from mice in the acute phase of infection by the clone H510
C8C3<bold><italic>hvir</italic></bold> of <italic>T. cruzi</italic>, and
the infection of the fed mice was monitored by a parasitemia curve.
Similarly, we assessed the infective capacity of <italic>T.
cruzi</italic> trypomastigotes and amastigotes by infecting groups of
five mice Balb/c females, which were infected orally using a nasogastric
probe, and the infection was monitored by a parasitemia curve. Finally,
different trypomastigote and amastigote inoculums were used to determine
their infective capacities. Adhesion assays of <italic>T. cruzi</italic>
proteins to AGS stomach cells were performed, and the adhered proteins
were detected by western blotting using monoclonal or polyclonal
antibodies and by LC-MS/MS and bioinformatics analysis.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Trypomastigote migration in the presence of mucin was reduced by
approximately 30%, whereas in the presence of mucin and pepsin at pH
3.5, only a small proportion of parasites were able to migrate (∼6%).
Similarly, the ability of TCTs to infect AGS cells in the presence of
mucin is reduced by approximately 20%. In all cases, 60–100% of the
animals were fed meat from mice infected in the acute phase or infected
with trypomastigotes or amastigotes developed high parasitemia, and 80%
died around day 40 post-infection. The adhesion assay showed that
cruzipain is a molecule of trypomastigotes and amastigotes that binds to
AGS cells. LC-MS/MS and bioinformatics analysis, also confirmed that
transialidase, cysteine proteinases, and gp63 may be involved in TCTs
attachment or invasion of human stomach cells because they can
potentially interact with different proteins in the human stomach
mucosa. In addition, several human gastric mucins have cysteine protease
cleavage sites.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Then, under our experimental conditions, consuming meat from infected
animals in the acute phase allows the <italic>T. cruzi</italic>
infection. Similarly, trypomastigotes and amastigotes could infect mice
when administered orally, whereas cysteinyl proteinases and
trans-sialidase appear to be relevant molecules in this infective
process. 
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297099">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297099</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1297099">Save
to Pocket</a> </span></p>
<hr />
<h2 id="oral-streptococci-modulators-of-health-and-disease">Oral
streptococci: modulators of health and disease</h2>
<p>date: 2024-02-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Streptococci are primary colonizers of the oral cavity where they are
ubiquitously present and an integral part of the commensal oral biofilm
microflora. The role oral streptococci play in the interaction with the
host is ambivalent. On the one hand, they function as gatekeepers of
homeostasis and are a prerequisite for the maintenance of oral health -
they shape the oral microbiota, modulate the immune system to enable
bacterial survival, and antagonize pathogenic species. On the other
hand, also recognized pathogens, such as oral <italic>Streptococcus
mutans</italic> and <italic>Streptococcus sobrinus</italic>, which
trigger the onset of dental caries belong to the genus
<italic>Streptococcus</italic>. In the context of periodontitis, oral
streptococci as excellent initial biofilm formers have an accessory
function, enabling late biofilm colonizers to inhabit gingival pockets
and cause disease. The pathogenic potential of oral streptococci fully
unfolds when their dissemination into the bloodstream occurs;
streptococcal infection can cause extra-oral diseases, such as infective
endocarditis and hemorrhagic stroke. In this review, the taxonomic
diversity of oral streptococci, their role and prevalence in the oral
cavity and their contribution to oral health and disease will be
discussed, focusing on the virulence factors these species employ for
interactions at the host interface.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1357631">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1357631</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1357631">Save
to Pocket</a> </span></p>
<hr />
<h2
id="comprehensive-characterization-of-erv-k-hml-8-in-the-chimpanzee-genome-revealed-less-genomic-activity-than-humans">Comprehensive
characterization of ERV-K (HML-8) in the chimpanzee genome revealed less
genomic activity than humans</h2>
<p>date: 2024-02-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Endogenous retroviruses (ERVs) originate from ancestral germline
infections caused by exogenous retroviruses. Throughout evolution, they
have become fixed within the genome of the animals into which they were
integrated. As ERV elements coevolve with the host, they are normally
epigenetically silenced and can become upregulated in a series of
physiological and pathological processes. Generally, a detailed ERV
profile in the host genome is critical for understanding the
evolutionary history and functional performance of the host genome. We
previously characterized and cataloged all the ERV-K subtype HML-8 loci
in the human genome; however, this has not been done for the chimpanzee,
the nearest living relative of humans. In this study, we aimed to
catalog and characterize the integration of HML-8 in the chimpanzee
genome and compare it with the integration of HML-8 in the human genome.
We analyzed the integration of HML-8 and found that HML-8 pervasively
invaded the chimpanzee genome. A total of 76 proviral elements were
characterized on 23/24 chromosomes, including detailed elements
distribution, structure, phylogeny, integration time, and their
potential to regulate adjacent genes. The incomplete structure of HML-8
proviral LTRs will undoubtedly affect their activity. Moreover, the
results indicated that HML-8 integration occurred before the divergence
between humans and chimpanzees. Furthermore, chimpanzees include more
HML-8 proviral elements (76 vs. 40) and fewer solo long terminal repeats
(LTR) (0 vs. 5) than humans. These results suggested that chimpanzee
genome activity is less than the human genome and that humans may have a
better ability to shape and screen integrated proviral elements. Our
work is informative in both an evolutionary and a functional context for
ERVs.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349046">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349046</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1349046">Save
to Pocket</a> </span></p>
<hr />
<h2
id="identification-of-plasminogen-binding-sites-in-streptococcus-suis-enolase-that-contribute-to-bacterial-translocation-across-the-blood-brain-barrier">Identification
of plasminogen-binding sites in Streptococcus suis enolase that
contribute to bacterial translocation across the blood-brain
barrier</h2>
<p>date: 2024-02-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
<italic>Streptococcus suis</italic> is an emerging zoonotic pathogen
that can cause invasive disease commonly associated with meningitis in
pigs and humans. To cause meningitis, <italic>S. suis</italic> must
cross the blood-brain barrier (BBB) comprising blood vessels that
vascularize the central nervous system (CNS). The BBB is highly
selective due to interactions with other cell types in the brain and the
composition of the extracellular matrix (ECM). Purified streptococcal
surface enolase, an essential enzyme participating in glycolysis, can
bind human plasminogen (Plg) and plasmin (Pln). Plg has been proposed to
increase bacterial traversal across the BBB via conversion to Pln, a
protease which cleaves host proteins in the ECM and monocyte
chemoattractant protein 1 (MCP1) to disrupt tight junctions. The
essentiality of enolase has made it challenging to unequivocally
demonstrate its role in binding Plg/Pln on the bacterial surface and
confirm its predicted role in facilitating translocation of the BBB.
Here, we report on the CRISPR/Cas9 engineering of <italic>S.
suis</italic> enolase mutants <italic>eno<sup>261</sup></italic>,
<italic>eno<sup>252/253/255</sup></italic>,
<italic>eno<sup>252/261</sup></italic>, and
<italic>eno<sup>434/435</sup></italic> possessing amino acid
substitutions at <italic>in silico</italic> predicted binding sites for
Plg. As expected, amino acid substitutions in the predicted Plg binding
sites reduced Plg and Pln binding to <italic>S. suis</italic> but did
not affect bacterial growth <italic>in vitro</italic> compared to the
wild-type strain. The binding of Plg to wild-type <italic>S.
suis</italic> enhanced translocation across the human cerebral
microvascular endothelial cell line hCMEC/D3 but not for the
<italic>eno</italic> mutant strains tested. To our knowledge, this is
the first study where predicted Plg-binding sites of enolase have been
mutated to show altered Plg and Pln binding to the surface of <italic>S.
suis</italic> and attenuation of translocation across an endothelial
cell monolayer <italic>in vitro</italic>.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356628">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356628</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1356628">Save
to Pocket</a> </span></p>
<hr />
<h2
id="natural-products-as-a-source-of-coronavirus-entry-inhibitors">Natural
products as a source of Coronavirus entry inhibitors</h2>
<p>date: 2024-02-21, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
The COVID-19 pandemic has had a significant and lasting impact on the
world. Four years on, despite the existence of effective vaccines, the
continuous emergence of new SARS-CoV-2 variants remains a challenge for
long-term immunity. Additionally, there remain few purpose-built
antivirals to protect individuals at risk of severe disease in the event
of future coronavirus outbreaks. A promising mechanism of action for
novel coronavirus antivirals is the inhibition of viral entry. To
facilitate entry, the coronavirus spike glycoprotein interacts with
angiotensin converting enzyme 2 (ACE2) on respiratory epithelial cells.
Blocking this interaction and consequently viral replication may be an
effective strategy for treating infection, however further research is
needed to better characterize candidate molecules with antiviral
activity before progressing to animal studies and clinical trials. In
general, antiviral drugs are developed from purely synthetic compounds
or synthetic derivatives of natural products such as plant secondary
metabolites. While the former is often favored due to the higher
specificity afforded by rational drug design, natural products offer
several unique advantages that make them worthy of further study
including diverse bioactivity and the ability to work synergistically
with other drugs. Accordingly, there has recently been a renewed
interest in natural product-derived antivirals in the wake of the
COVID-19 pandemic. This review provides a summary of recent research
into coronavirus entry inhibitors, with a focus on natural compounds
derived from plants, honey, and marine sponges.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1353971">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1353971</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1353971">Save
to Pocket</a> </span></p>
<hr />
<h2
id="covid-19-in-pulmonary-critically-ill-patients-metagenomic-identification-of-fungi-and-characterization-of-pathogenic-microorganisms">COVID-19
in pulmonary critically ill patients: metagenomic identification of
fungi and characterization of pathogenic microorganisms</h2>
<p>date: 2024-02-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Background
</title>
<p>
Fungal co-infection is prevalent in critically ill patients with
COVID-19. The conventional approach applied to fungal identification has
relatively low sensitivity and is time-consuming. The metagenomic
next-generation sequencing (mNGS) technology can simultaneously detect a
variety of microorganisms, and is increasingly being used for the rapid
detection and diagnosis of pathogens.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
In this single-center retrospective study, we described the clinical
presentation and outcomes of COVID-19 and mNGS positive for fungi in
pulmonary critically ill patients during the outbreak of Omicron
infection from December 2022 to January 2023.
</p>
</sec><sec>
<title>
Results
</title>
<p>
Among 43 COVID-19 patients with acute respiratory distress syndrome
(ARDS) on a single intensive care unit (ICU), 10 were reported to be
fungal positive using the mNGS test. The number of pathogenic
microorganisms detected by mNGS was significantly higher than that via
traditional methods, especially in the detection of fungi and viruses.
<italic>Aspergillus</italic> infection was dominant, and most of these
patients also had concurrent bacterial or viral infections. Probable or
possible COVID-19-associated pulmonary aspergillosis (CAPA) was
diagnosed in all 10 patients, and the prognosis was poor.
</p>
</sec><sec>
<title>
Conclusion
</title>
<p>
Patients with COVID-19 may be at increased risk of developing fungal
infections as well as concurrent bacterial or viral infections, and mNGS
can be a powerful tool in identifying these infections. Clinicians
should be aware of the increased risk of fungal infections in COVID-19
patients, particularly those who have underlying immunocompromising
conditions, and should monitor for early signs of infection.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012">https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2023.1220012">Save
to Pocket</a> </span></p>
<hr />
<h2
id="effects-of-plant-natural-products-on-metabolic-associated-fatty-liver-disease-and-the-underlying-mechanisms-a-narrative-review-with-a-focus-on-the-modulation-of-the-gut-microbiota">Effects
of plant natural products on metabolic-associated fatty liver disease
and the underlying mechanisms: a narrative review with a focus on the
modulation of the gut microbiota</h2>
<p>date: 2024-02-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Metabolic-associated fatty liver disease (MAFLD) is a chronic liver
disease characterized by the excessive accumulation of fat in
hepatocytes. However, due to the complex pathogenesis of MAFLD, there
are no officially approved drugs for treatment. Therefore, there is an
urgent need to find safe and effective anti-MAFLD drugs. Recently, the
relationship between the gut microbiota and MAFLD has been widely
recognized, and treating MAFLD by regulating the gut microbiota may be a
new therapeutic strategy. Natural products, especially plant natural
products, have attracted much attention in the treatment of MAFLD due to
their multiple targets and pathways and few side effects. Moreover, the
structure and function of the gut microbiota can be influenced by
exposure to plant natural products. However, the effects of plant
natural products on MAFLD through targeting of the gut microbiota and
the underlying mechanisms are poorly understood. Based on the above
information and to address the potential therapeutic role of plant
natural products in MAFLD, we systematically summarize the effects and
mechanisms of action of plant natural products in the prevention and
treatment of MAFLD through targeting of the gut microbiota. This
narrative review provides feasible ideas for further exploration of
safer and more effective natural drugs for the prevention and treatment
of MAFLD.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1323261">Save
to Pocket</a> </span></p>
<hr />
<h2 id="advances-in-application-of-crispr-cas13a-system">Advances in
application of CRISPR-Cas13a system</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>
Clustered Regularly Interspaced Short Palindromic Repeats (CRISPRs) and
CRISPR-associated (Cas) proteins serve as an adaptive immune system that
safeguards prokaryotes and some of the viruses that infect prokaryotes
from foreign nucleic acids (such as viruses and plasmids). The genomes
of the majority of archaea and about half of all bacteria contain
various CRISPR-Cas systems. CRISPR-Cas systems depend on CRISPR RNAs
(crRNAs). They act as a navigation system to specifically cut and
destroy foreign nucleic acids by recognizing invading foreign nucleic
acids and binding Cas proteins. In this review, we provide a brief
overview of the evolution and classification of the CRISPR-Cas system,
focusing on the functions and applications of the CRISPR-Cas13a system.
We describe the CRISPR-Cas13a system and discuss its RNA-directed
ribonuclease function. Meanwhile, we briefly introduce the mechanism of
action of the CRISPR-Cas13a system and summarize the applications of the
CRISPR-Cas13a system in pathogen detection, eukaryotes, agriculture,
biosensors, and human gene therapy. We are right understanding of
CRISPR-Cas13a has been broadened, and the CRISPR-Cas13a system will be
useful for developing new RNA targeting tools. Therefore, understanding
the basic details of the structure, function, and biological
characterization of CRISPR-Cas13a effector proteins is critical for
optimizing RNA targeting tools.
</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1291557">Save
to Pocket</a> </span></p>
<hr />
<h2
id="excess-fermentation-and-lactic-acidosis-as-detrimental-functions-of-the-gut-microbes-in-treatment-naive-tb-patients">Excess
fermentation and lactic acidosis as detrimental functions of the gut
microbes in treatment-naive TB patients</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
The link between gut microbiota and host immunity motivated numerous
studies of the gut microbiome in tuberculosis (TB) patients. However,
these studies did not explore the metabolic capacity of the gut
community, which is a key axis of impact on the host’s immunity.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
We used deep sequencing of fecal samples from 23 treatment-naive TB
patients and 48 healthy donors to reconstruct the gut microbiome’s
metabolic capacity and strain/species-level content.
</p>
</sec><sec>
<title>
Results
</title>
<p>
We show that the systematic depletion of the commensal flora of the
large intestine, <italic>Bacteroidetes</italic>, and an increase in
<italic>Actinobacteria, Firmicutes</italic>, and
<italic>Proteobacteria</italic> such as <italic>Streptococcaceae,
Erysipelotrichaceae, Lachnospiraceae</italic>, and
<italic>Enterobacteriaceae</italic> explains the strong taxonomic
divergence of the gut community in TB patients. The cumulative expansion
of diverse disease-associated pathobionts in patients reached 1/4 of the
total gut microbiota, suggesting a heavy toll on host immunity along
with MTB infection. Reconstruction of metabolic pathways showed that the
microbial community in patients shifted toward rapid growth using
glycolysis and excess fermentation to produce acetate and lactate.
Higher glucose availability in the intestine likely drives fermentation
to lactate and growth, causing acidosis and endotoxemia.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Excessive fermentation and lactic acidosis likely characterize TB
patients’ disturbed gut microbiomes. Since lactic acidosis strongly
suppresses the normal gut flora, directly interferes with macrophage
function, and is linked to mortality in TB patients, our findings
highlight gut lactate acidosis as a novel research focus. If confirmed,
gut acidosis may be a novel potential host-directed treatment target to
augment traditional TB treatment.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1331521">Save
to Pocket</a> </span></p>
<hr />
<h2
id="editorial-precision-medicine-in-viral-hepatitis-progress-and-prospects-towards-elimination">Editorial:
Precision medicine in viral hepatitis: progress and prospects towards
elimination</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1378423">Save
to Pocket</a> </span></p>
<hr />
<h2
id="identification-of-potential-novel-proteomic-markers-of-leishmania-spp.-derived-exosomes">Identification
of potential novel proteomic markers of Leishmania spp.-derived
exosomes</h2>
<p>date: 2024-02-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<sec>
<title>
Introduction
</title>
<p>
Extracellular vesicles (EVs) are heterogenous cell-derived
membrane-bound structures which can be subdivided into three distinct
classes according to distinct morphological characteristics, cellular
origins, and functions. Small EVs, or exosomes, can be produced by the
protozoan parasite <italic>Leishmania</italic> through the
evolutionarily conserved ESCRT pathway, and act as effectors of
virulence and drivers of pathogenesis within mammalian hosts. Techniques
for the identification of EVs of non-mammalian origin, however, remain
inaccurate in comparison to their well-characterized mammalian
counterparts. Thus, we still lack reliable and specific markers for
<italic>Leishmania</italic>-derived exosomes, which poses a significant
challenge to the field.
</p>
</sec><sec>
<title>
Methods
</title>
<p>
Herein, we utilized serial differential ultracentrifugation to separate
<italic>Leishmania</italic>-derived EV populations into three distinct
fractions. Nanoparticle tracking analysis and transmission electron
microscopy were used to validate their morphological characteristics,
and bioinformatic analysis of LC-MS/MS proteomics corroborated cellular
origins and function.
</p>
</sec><sec>
<title>
Discussion
</title>
<p>
Proteomic data indicated potential novel proteic markers of
<italic>Leishmania</italic>-derived exosomes, including proteins
involved in endosomal machinery and the ESCRT pathway, as well as the
parasitic phosphatase PRL-1. Further investigation is required to
determine the specificity and sensitivity of these markers.
</p>
<p></sec></p>
<p><span class="feed-item-link">
<a href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636</a>
<a href="https://getpocket.com/save" class="pocket-btn" data-lang="en" data-save-url="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354636">Save
to Pocket</a> </span></p>
<script type="text/javascript">!function(d,i){if(!d.getElementById(i)){var j=d.createElement("script");j.id=i;j.src="https://widgets.getpocket.com/v1/j/btn.js?v=1";var w=d.getElementById(i);d.body.appendChild(j);}}(document,"pocket-btn-js");</script>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
