<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.34</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="../../snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../search.html">Search</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.34</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.34">health 2024.34</h1>
<p>(date: 2024-08-24 08:51:07)</p>
<hr />
<h2
id="investigating-cutaneous-tuberculosis-and-nontuberculous-mycobacterial-infections-a-department-of-dermatology-beijing-china-a-comprehensive-clinicopathological-analysis">Investigating
cutaneous tuberculosis and nontuberculous mycobacterial infections a
Department of Dermatology, Beijing, China: a comprehensive
clinicopathological analysis</h2>
<p>date: 2024-08-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCutaneous tuberculosis (CTB) and nontuberculous
mycobacteria (NTM) infections present considerable diagnostic and
therapeutic challenges. This study aims to provide a comprehensive
clinicopathological analysis of CTB and NTM infections.MethodsWe
conducted a retrospective analysis of 103 patients diagnosed with
cutaneous tuberculosis (CTB) and nontuberculous mycobacteria (NTM)
infections at a Beijing dermatology department from January 2000 to
January 2024. Demographic, clinical, histological, and laboratory
finding data were collected. Diagnostic methods and histopathological
examination were recorded. Treatment regimens and outcomes were
reviewed. Descriptive statistics were used to summarize demographic and
clinical data, and continuous variables expressed as means and standard
deviations (SD), and categorical variables as frequencies and
percentages. Statistical analyses were conducted using SPSS version
25.0.ResultsThe cohort included 103 patients (40.8% males and 59.2%
females), with a mean age of 51.86 years. Common clinical manifestations
included nodules (97.1%), erythema (74.8%), and plaques (68.9%).
Histological examination revealed hyperkeratosis (68.9%), parakeratosis
(23.3%), and extensive neutrophil infiltration (95.1%) were observed.
Acid fast bacteria (AFB) stains and nucleic acid tests exhibited
respective positivity rates of 39.6% and 52.3%, respectively. Most
patients were treated with a combination of three drugs; 77.1% of
patients showed improvement, with the cure rate for CTB being
20.0%.DiscussionThis study highlights the diverse clinical and
histological presentations of CTB and NTM infections, emphasizing the
need for comprehensive diagnostic approaches. The variability in
treatment regimens reflects the complex management of these
infections.ConclusionThe implementation of advanced molecular techniques
and standardized treatment protocols is imperative for enhancing
diagnostic precision and therapeutic outcomes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451602"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451602</a></p>
<hr />
<h2
id="men-with-a-history-of-commercial-heterosexual-contact-play-essential-roles-in-the-transmission-of-hiv-1-crf55_01b-from-men-who-have-sex-with-men-to-the-general-population-in-guangxi-china">Men
with a history of commercial heterosexual contact play essential roles
in the transmission of HIV-1 CRF55_01B from men who have sex with men to
the general population in Guangxi, China</h2>
<p>date: 2024-08-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThere is increasing focus on HIV-1 CRF55_01B in China.
However, there is limited information regarding the dissemination of
CRF55_01B across different regions and populations in Guangxi. This
study was performed to elucidate the evolutionary history of the
introduction and dissemination of CRF55_01B in Guangxi.MethodsMolecular
network and phylogenetic analyses were used to investigate the
transmission characteristics of CRF55_01B in China. The analyses
particularly focused on the cross-provincial spatial and temporal
transmission patterns between Guangdong Province and Guangxi, as well as
the transmission dynamics among different regions and populations within
Guangxi.ResultsIn total, 2226 partial pol sequences of CRF55_01B strains
sampled from 2007 to 2022 were collected, including 1895 (85.09%)
sequences from Guangdong, 199 (8.94%) sequences from Guangxi, and 172
(7.59%) sequences from other provinces of China. Most people living with
HIV in Guangxi were infected with HIV-1 through heterosexuals (52.76%).
Among these, 19.10% had a history of commercial heterosexual contact
(CHC) and 15.58% had a history of non-marital non-commercial
heterosexual contact (NMNCHC). Overall, 1418 sequences were identified
in the molecular network. Notably, the sequences from Guangdong Province
were most closely linked to those from Guangxi. Phylogenetic analysis
showed that CRF55_01B was first introduced from Shenzhen City to Nanning
City around 2007. Subsequently, CRF55_01B established local transmission
within Guangxi, with Nanning City serving as the transmission center
from 2008 to 2017. After 2017, the CRF55_01B strain spread to other
regions of Guangxi. Men who have sex with men (MSM) and men with a
history of CHC have played a significant role in the transmission of
CRF55_01B among different populations in Guangxi.ConclusionsThis study
provides evidence on the transmission trajectory of CRF55_01B among
different regions and populations in Guangxi. Given the bridging role of
men with a history of CHC in the dissemination of CRF55_01B from MSM to
the general population, it is imperative to enhance surveillance among
key populations to mitigate the secondary transmission of HIV-1.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391215"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391215</a></p>
<hr />
<h2
id="investigating-the-regulatory-effect-of-shen-qi-bu-qi-powder-on-the-gastrointestinal-flora-and-serum-metabolites-in-calves">Investigating
the regulatory effect of Shen Qi Bu Qi powder on the gastrointestinal
flora and serum metabolites in calves</h2>
<p>date: 2024-08-23, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectTo investigate the effects of Shen Qi Bu Qi Powder (SQBQP) on
the average daily gain, blood indexes, gastrointestinal microflora, and
serum metabolites of calves.MethodsA total of 105 calves were randomly
assigned to three groups (n = 35 per group): the control group (C, fed
with a basal diet for 21 days) and two treatment groups (SQBQP-L and
SQBQP-H, fed with the basal diet supplemented with 15 and 30 g/kg of
SQBQP), respectively for 21 days. The active components of SQBQP were
identified using LC-MS/MS. Serum digestive enzymes and antioxidant
indices were determined by ELISA kits and biochemical kits,
respectively. Serum differential metabolites were analyzed by liquid
chromatography-mass spectrometry/mass spectrometry (LC-MS/MS), while
flora in rumen fluid and fecal were analyzed by 16S rDNA sequencing.
Further correlation analysis of gastrointestinal flora and serum
metabolites of SQBQP-H and C groups were performed with Spearman’s
correlation.ResultsThe principal active components of SQBQP mainly
includes polysaccharides, flavonoids, and organic acids. Compared to the
control group (C), calves in the SQBQP-H (high dose) and SQBQP-L (low
dose) groups showed a significant increase in serum amylase (AMS) levels
(P&lt;0.001), while lipase content significantly decreased (P&lt;0.05).
Additionally, the average daily gain, T-AOC, and cellulase content of
calves in the SQBQP-H group significantly increased (P&lt;0.05).
Proteobacteria and Succinivibrio in the rumen flora of the SQBQP-H group
was significantly lower than that of the C group (P&lt;0.05). The
relative abundance of Proteobacteria, Actinobacteria,
Candidatus_Saccharibacteria, Deinococcus_Thermus, Cyanobacteria, and
Succinivibrio in the SQBQP-H group was significantly increased
(P&lt;0.05), while the relative abundance of Tenericutes and
Oscillibacter was significantly decreased (P&lt;0.05). Serum
metabolomics analysis revealed 20 differential metabolites, mainly
enriched in amino acid biosynthesis, β-alanine metabolism, tyrosine, and
tryptophan biosynthesis metabolic pathways (P&lt;0.05). Correlation
analysis results showed that Butyrivibrio in rumen flora and
Oscillibacter_valericigenes in intestinal flora were significantly
positively correlated with average daily gain, serum biochemical
indexes, and differential metabolite (-)-Epigallocatechin (R&gt;0.58,
P&lt;0.05).ConclusionSQBQP can promote calves weight gain and enhance
health by modulating gastrointestinal flora and metabolic processes in
the body.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1443712"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1443712</a></p>
<hr />
<h2
id="broadening-horizons-intestinal-microbiota-as-a-novel-biomarker-and-potential-treatment-for-hypertensive-disorders-of-pregnancy">Broadening
horizons: intestinal microbiota as a novel biomarker and potential
treatment for hypertensive disorders of pregnancy</h2>
<p>date: 2024-08-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Hypertensive disorders of pregnancy (HDP) are severe complications of
pregnancy with high morbidity and are a major cause of increased
maternal and infant morbidity and mortality. Currently, there is a lack
of effective early diagnostic indicators and safe and effective
preventive strategies for HDP in clinical practice, except for
monitoring maternal blood pressure levels, the degree of proteinuria,
organ involvement and fetal conditions. The intestinal microbiota
consists of the gut flora and intestinal environment, which is the
largest microecosystem of the human body and participates in material
and energy metabolism, gene expression regulation, immunity regulation,
and other functions. During pregnancy, due to changes in hormone levels
and altered immune function, the intestinal microecological balance is
affected, triggering HDP. A dysregulated intestinal microenvironment
influences the composition and distribution of the gut flora and changes
the intestinal barrier, driving beneficial or harmful bacterial
metabolites and inflammatory responses to participate in the development
of HDP and promote its malignant development. When the gut flora is
dysbiotic and affects blood pressure, supplementation with probiotics
and dietary fiber can be used to intervene. In this review, the
interaction between the intestinal microbiota and HDP was investigated
to explore the feasibility of the gut flora as a novel biomarker of HDP
and to provide a new strategy and basis for the prevention and treatment
of clinical HDP.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446580"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446580</a></p>
<hr />
<h2
id="mucin-adhesion-of-serial-cystic-fibrosis-airways-pseudomonas-aeruginosa-isolates">Mucin
adhesion of serial cystic fibrosis airways Pseudomonas aeruginosa
isolates</h2>
<p>date: 2024-08-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The chronic airway infections with Pseudomonas aeruginosa are the
major co-morbidity in people with cystic fibrosis (CF). Within CF lungs,
P. aeruginosa persists in the conducting airways together with human
mucins as the most abundant structural component of its
microenvironment. We investigated the adhesion of 41 serial CF airway P.
aeruginosa isolates to airway mucin preparations from CF sputa. Mucins
and bacteria were retrieved from five modulator-naïve patients with
advanced CF lung disease. The P. aeruginosa isolates from CF airways and
non-CF reference strains showed a strain-specific signature in their
adhesion to ovine, porcine and bovine submaxillary mucins and CF airway
mucins ranging from no or low to moderate and strong binding. Serial CF
clonal isolates and colony morphotypes from the same sputum sample were
as heterogeneous in their affinity to mucin as representatives of other
clones thus making ‘mucin binding’ one of the most variable intraclonal
phenotypic traits of P. aeruginosa known to date. Most P. aeruginosa CF
airway isolates did not adhere more strongly to CF airway mucins than to
plastic surfaces. The strong binders, however, exhibited a
strain-specific affinity gradient to O-glycans, CF airway and mammalian
submaxillary mucins.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1448104"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1448104</a></p>
<hr />
<h2
id="analysis-of-mirnas-involved-in-mouse-brain-injury-upon-coxsackievirus-a6-infection">Analysis
of miRNAs involved in mouse brain injury upon Coxsackievirus A6
infection</h2>
<p>date: 2024-08-22, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionCoxsackievirus A6 (CV-A6) has emerged as the predominant
epidemic strain responsible for hand, foot and mouth disease (HFMD).
CV-A6 infection can result in severe clinical manifestations, including
encephalitis, meningitis, and potentially life-threatening central
nervous system disorders. Our previous research findings demonstrated
that neonatal mice infected with CV-A6 exhibited limb weakness,
paralysis, and ultimately succumbed to death. However, the underlying
mechanism of CV-A6-induced nervous system injury remains elusive.
Numerous reports have highlighted the pivotal role of miRNAs in various
viral infections.MethodsSeparately established infection and control
groups of mice were used to create miRNA profiles of the brain tissues
before and after CV-A6 transfection, followed by experimental
verification, prediction, and analysis of the results.ResultsAt 2 days
post-infection (dpi), 4 dpi, and 2dpi vs 4dpi, we identified 175, 198
and 78 significantly differentially expressed miRNAs respectively using
qRT-PCR for validation purposes. Subsequently, we predicted target genes
of these differentially expressed miRNAs and determined their potential
targets through GO (Gene Ontology) enrichment analysis and KEGG (Kyoto
Encyclopedia of Genes and Genomes) enrichment analysis. Finally, we
verified the miRNA-mRNA pairing via double luciferase experiments while
confirming functional enrichment of target genes through Western
Blotting analyses.DiscussionThe results from this study suggest that
transcriptional regulation, neuronal necrosis, pro-inflammatory cytokine
release, and antiviral immunity are all implicated in the pathogenesis
of central nervous system injury in mice infected with CV-A6. Brain
injury resulting from CV-A6 infection may involve multiple pathways,
including glial cell activation, neuronal necrosis, synaptic
destruction, degenerative diseases of the nervous system. It can even
encompass destruction of the blood-brain barrier, leading to central
nervous system injury. The dysregulated miRNAs and signaling pathways
discovered in this study provide valuable insights for further
investigations into the pathogenesis of CV-A6.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1405689"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1405689</a></p>
<hr />
<h2
id="performance-of-metagenomic-next-generation-sequencing-and-metagenomic-nanopore-sequencing-for-the-diagnosis-of-tuberculosis-in-hiv-positive-patients">Performance
of metagenomic Next-Generation Sequencing and metagenomic Nanopore
Sequencing for the diagnosis of tuberculosis in HIV-positive
patients</h2>
<p>date: 2024-08-21, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPatients who were infected by the Human Immunodeficiency
Virus (HIV) could have weakened immunity that is complicated by
opportunistic infections, especially for Mycobacterium tuberculosis
(MTB). Notably, the HIV-MTB co-infection will accelerate the course of
disease progress and greatly increase the mortality of patients. Since
the traditional diagnostic methods are time-consuming and have low
sensitivity, we aim to investigate the performance of mNGS (metagenomic
Next-Generation Sequencing) and mNPS (metagenomic NanoPore Sequencing)
for the rapid diagnosis of tuberculosis in HIV-infected
patients.MethodsThe 122 HIV-infected patients were enrolled for the
retrospective analysis. All of the patients underwent traditional
microbiological tests, mNGS, and (or) mNPS tests. The clinical
comprehensive diagnosis was used as the reference standard to compare
the diagnostic performance of culture, mNGS, and mNPS on tuberculosis.
We also investigate the diagnostic value of mNGS and mNPS on
mixed-infection. Furthermore, the treatment adjustment directed by mNGS
and mNPS was analyzed.ResultsCompared with the composite reference
standard, the culture showed 42.6% clinical sensitivity and 100%
specificity, and the OMT(other microbiological testing) had 38.9%
sensitivity and 100% specificity. The mNGS had 58.6% clinical
sensitivity and 96.8% specificity, and the mNPS had 68.0% clinical
sensitivity and 100% specificity. The proportion of mixed-infection
cases (88.9%) in the TB group was higher than those in the non-TB group
(54.8%) and the mNGS and mNPS are more competitive on mixed-infection
diagnosis compared with the traditional methods. Furthermore, there are
63 patients (69.2%) and 36 patients (63.2%) achieved effective treatment
after receiving the detection of mNPS and mNGS,
respectively.ConclusionOur study indicated that mNPS and mNGS have high
sensitivity and specificity for TB diagnosis compared with the
traditional methods, and mNPS seems to have better diagnostic
performance than mNGS. Moreover, mNGS and mNPS showed apparent
advantages in detecting mixed infection. The mNPS and mNGS-directed
medication adjustment have effective treatment outcomes for HIV-infected
patients who have lower immunity.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1423541"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1423541</a></p>
<hr />
<h2
id="era-crisprcas12a-system-a-rapid-highly-sensitive-and-specific-assay-for-mycobacterium-tuberculosis">ERA-CRISPR/Cas12a
system: a rapid, highly sensitive and specific assay for Mycobacterium
tuberculosis</h2>
<p>date: 2024-08-21, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMycobacterium tuberculosis, the causative agent of human
tuberculosis, poses a significant threat to global public health and
imposes a considerable burden on the economy. However, existing
laboratory diagnostic methods for M. tuberculosis are time-consuming and
have limited sensitivity levels.MethodsThe CRISPR/Cas system, commonly
known as the “gene scissors”, demonstrates remarkable specificity and
efficient signal amplification capabilities. Enzymatic recombinase
amplification (ERA) was utilized to rapidly amplify trace DNA fragments
at a consistent temperature without relying on thermal cyclers. By
integrating of CRISPR/Cas12a with ERA, we successfully developed an
ERA-CRISPR/Cas12a detection system that enables rapid identification of
M. tuberculosis.ResultsThe sensitivity of the ERA-CRISPR/Cas12a
fluorescence and lateral flow systems was 9 copies/μL and 90 copies/μL,
respectively. Simultaneously, the detection system exhibited no
cross-reactivity with various of respiratory pathogens and
non-tuberculosis mycobacteria, demonstrating a specificity of 100%. The
positive concordance rate between the ERA-CRISPR/Cas12a fluorescence
system and commercial qPCR was 100% in 60 clinical samples. Meanwhile,
the lateral flow system showed a positive concordance rate of 93.8% when
compared to commercial qPCR. Both methods demonstrated a negative
concordance rate of 100%, and the test results can be obtained in 50 min
at the earliest.DiscussionThe ERA-CRISPR/Cas12a system offers a rapid,
sensitive, and specific method that presents a novel approach to
laboratory diagnosis of M. tuberculosis.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1454076"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1454076</a></p>
<hr />
<h2
id="melatonin-inhibits-bovine-viral-diarrhea-virus-replication-by-er-stress-mediated-nf-κb-signal-pathway-and-autophagy-in-mdbk-cells">Melatonin
inhibits bovine viral diarrhea virus replication by ER stress-mediated
NF-κB signal pathway and autophagy in MDBK cells</h2>
<p>date: 2024-08-21, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Bovine viral diarrhea-mucosal disease (BVD-MD) is a contagious
disease in cattle, caused by the bovine viral diarrhea virus (BVDV).
This virus continues to spread globally, exerting pressure on both
public health and the economy. Despite its impact, there are currently
no effective drugs for treating BVDV. This study utilized Madin-Darby
bovine kidney (MDBK) cells as a model to investigate the antiviral
effects of melatonin against Bovine Viral Diarrhea Virus (BVDV) and its
connection with endoplasmic reticulum (ER) stress. Our results show that
melatonin can suppress BVDV proliferation in MDBK cells by modulating
the endoplasmic reticulum (ER) stress-mediated NF-κB pathway and
autophagy. Specifically, melatonin alleviated ER stress, inhibited the
activation of IκBα and p65, regulated autophagy, and reduced the
expression levels of pro-inflammatory cytokines. Further, when we
treated BVDV-infected cells with the ER stress inducer thapsigargin, it
led to significant activation of the NF-κB pathway and autophagy.
Conversely, treating the cells with the ER stress inhibitor
4-phenylbutyric acid reversed these effects. These findings suggest that
melatonin exerts its antiviral effects primarily through the
PERK-eIF2α-ATF4 of ER stress-mediated NF-κB pathway and autophagy.
Overall, our study underscores the potential of melatonin as an
effective protective and therapeutic option against BVDV, offering
insights into its anti-infective mechanisms.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431836"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431836</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2024-08-20, from: People’s CDC</p>
<p>This is the <span class="citation"
data-cites="PeoplesCDC">@PeoplesCDC</span> bi-weekly update for August
19, 2024! This Weather Report from the People’s CDC sheds light on the
ongoing COVID situation in the US.</p>
<p><a
href="https://peoplescdc.org/2024/08/20/peoples-cdc-covid-19-weather-report-81/"
class="uri">https://peoplescdc.org/2024/08/20/peoples-cdc-covid-19-weather-report-81/</a></p>
<hr />
<h2
id="development-of-a-novel-sandwich-immunoassay-based-on-targeting-recombinant-francisella-outer-membrane-protein-a-for-the-diagnosis-of-tularemia">Development
of a novel sandwich immunoassay based on targeting recombinant
Francisella outer membrane protein A for the diagnosis of tularemia</h2>
<p>date: 2024-08-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionTularemia, caused by the bacterium Francisella
tularensis, poses health risks to humans and can spread through a
variety of routes. It has also been classified as a Tier 1 Select agent
by the CDC, highlighting its potential as a bioterrorism agent.
Moreover, it is difficult to diagnose in a timely fashion, owing to the
non-specific nature of tularemia infections. Rapid, sensitive, and
accurate detection methods are required to reduce mortality rates. We
aimed to develop antibodies directed against the outer membrane protein
A of F. tularensis (FopA) for rapid and accurate diagnosis of
tularemia.MethodsWe used a baculovirus insect cell expression vector
system to produce the FopA antigen and generate anti-FopA antibodies
through immunization of BALB/c mice. We then employed hybridoma and
phage display technologies to screen for antibodies that could recognize
unique epitopes on FopA.ResultTwo monoclonal antibodies, 6B12 and 3C1,
identified through phage display screening specifically bound to
recombinant FopA in a dose-dependent manner. The binding affinity of the
anti-FopA 6B12 and 3C1 antibodies was observed to have an equilibrium
dissociation constant of 1.76 × 10-10 M and 1.32 × 10-9 M, respectively.
These antibodies were used to develop a sandwich ELISA system for the
diagnosis of tularemia. This assay was found to be highly specific and
sensitive, with detection limits ranging from 0.062 ng/mL in PBS to
0.064 ng/mL in skim milk matrices.DiscussionOur findings demonstrate the
feasibility of a novel diagnostic approach for detecting F. tularensis
based on targeting FopA, as opposed to existing tests that target the
bacterial lipopolysaccharide.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1455259"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1455259</a></p>
<hr />
<h2
id="global-transcriptomic-network-analysis-of-the-crosstalk-between-microbiota-and-cancer-related-cells-in-the-oral-gut-lung-axis">Global
transcriptomic network analysis of the crosstalk between microbiota and
cancer-related cells in the oral-gut-lung axis</h2>
<p>date: 2024-08-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe diagnosis and treatment of lung, colon, and gastric
cancer through the histologic characteristics and genomic biomarkers
have not had a strong impact on the mortality rates of the top three
global causes of death by cancer.MethodsTwenty-five transcriptomic
analyses (10 lung cancer, 10 gastric cancer, and 5 colon cancer
datasets) followed our own bioinformatic pipeline based on the
utilization of specialized libraries from the R language and DAVID´s
gene enrichment analyses to identify a regulatory metafirm network of
transcription factors and target genes common in every type of cancer,
with experimental evidence that supports its relationship with the
unlocking of cell phenotypic plasticity for the acquisition of the
hallmarks of cancer during the tumoral process. The network’s regulatory
functional and signaling pathways might depend on the constant crosstalk
with the microbiome network established in the oral-gut-lung
axis.ResultsThe global transcriptomic network analysis highlighted the
impact of transcription factors (SOX4, TCF3, TEAD4, ETV4, and FOXM1)
that might be related to stem cell programming and cancer progression
through the regulation of the expression of genes, such as cancer-cell
membrane receptors, that interact with several microorganisms, including
human T-cell leukemia virus 1 (HTLV-1), the human papilloma virus (HPV),
the Epstein–Barr virus (EBV), and SARS−CoV−2. These interactions can
trigger the MAPK, non-canonical WNT, and IFN signaling pathways, which
regulate key transcription factor overexpression during the
establishment and progression of lung, colon, and gastric cancer,
respectively, along with the formation of the microbiome
network.ConclusionThe global transcriptomic network analysis highlights
the important interaction between key transcription factors in lung,
colon, and gastric cancer, which regulates the expression of cancer-cell
membrane receptors for the interaction with the microbiome network
during the tumorigenic process.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1425388"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1425388</a></p>
<hr />
<h2
id="covid-19-epidemic-investigation-study-of-a-follow-up-cohort-of-patients-with-diabetic-kidney-disease">COVID-19
epidemic investigation study of a follow-up cohort of patients with
diabetic kidney disease</h2>
<p>date: 2024-08-20, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe impact of coronavirus disease 2019 (COVID-19) on
diabetic kidney disease (DKD) patients in China is not fully understood.
This study aimed to investigate infection status in a DKD cohort
post-renal biopsy and analyze vaccination and infection rates, as well
as symptom severity, across various renal pathologies in DKD
patients.MethodsThis epidemiological survey, centered on COVID-19,
employed a Chinese DKD and renal puncture follow-up cohort. A customized
questionnaire enabled standardized data gathering. It collected data on
clinical characteristics, vaccination and infection statuses, and
diverse pathological types. The study analyzed the relationship between
vaccination and infection statuses across various pathological types,
evaluating characteristics and treatment outcomes in patients with
infections.ResultsIn total, 437 patients with DKD from 26 Chinese
provinces were followed up for a median of 44.6 ± 20 months. COVID-19
infection, vaccination, and novel coronavirus pneumonia (NCP) rates were
73.68%, 59.3%, and 6.63%, respectively. Ten patients with NCP had severe
pneumonia or died of COVID-19. Renal pathology revealed that 167
(38.22%) patients had diabetic nephropathy (DN), 171 (39.13%) had
non-diabetic renal disease (NDRD), and 99 had DN and NDRD (22.65%). The
DN group had the lowest vaccination (54.5%), highest all-cause mortality
(3.6%), and highest endpoint rates (34.10%). Compared to patients who
were not vaccinated pre-infection (117 cases), vaccinated patients (198
cases) had reduced NCP (6.6% vs. 13.7%), severity (1.0% vs. 3.4%), and
endpoint (9.10% vs. 31.60%) rates.ConclusionVaccination can prevent
infection and diminish COVID-19 severity in patients with DKD;
therefore, increasing vaccination rates is particularly
important.Clinical Trial registrationClinicalTrails.gov,
NCT05888909.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1388260"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1388260</a></p>
<hr />
<h2 id="low-sodium-dips-recommendations-from-a-registered-dietitian">Low
Sodium Dips [Recommendations from a Registered Dietitian]</h2>
<p>date: 2024-08-19, from: The Kidney Dietitian</p>
<p>
This article was written by dietetic student Cattie Clifft &amp;
reviewed by Melanie Betz MS, RD, CSR, FNKF, FAND. A sodium diet does not
have to be bland and boring. Low sodium dips DO exist! Low sodium dips
are a great option for those looking to reduce their salt intake while
still enjoying flavorful snacks. In […]
</p>
<p>
The post
<a href="https://www.thekidneydietitian.org/low-sodium-dips/">Low Sodium
Dips [Recommendations from a Registered Dietitian]</a> appeared first on
<a href="https://www.thekidneydietitian.org">The Kidney Dietitian</a>.
</p>
<p><a href="https://www.thekidneydietitian.org/low-sodium-dips/"
class="uri">https://www.thekidneydietitian.org/low-sodium-dips/</a></p>
<hr />
<h2
id="evaluation-of-phage-based-decontamination-in-respiratory-intensive-care-unit-environments-using-ddpcr-and-16s-rrna-targeted-sequencing-techniques">Evaluation
of phage-based decontamination in respiratory intensive care unit
environments using ddPCR and 16S rRNA targeted sequencing
techniques</h2>
<p>date: 2024-08-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundKlebsiella pneumoniae is a major cause of hospital-acquired
infections (HAIs), primarily spread through environmental contamination
in hospitals. The effectiveness of current chemical disinfectants is
waning due to emerging resistance, which poses environmental hazards and
fosters new resistance in pathogens. Developing environmentally friendly
and effective disinfectants against multidrug-resistant organisms is
increasingly important.MethodsThis study developed a bacteriophage
cocktail targeting two common carbapenem-resistant Klebsiella pneumoniae
(CRKP) strains, ST11 KL47 and ST11 KL64. The cocktail was used as an
adjunctive disinfectant in a hospital’s respiratory intensive care unit
(RICU) via ultrasonic nebulization. Digital PCR was used to quantify
CRKP levels post-intervention. The microbial community composition was
analyzed via 16S rRNA sequencing to assess the intervention’s impact on
overall diversity.ResultsThe phage cocktail significantly reduced CRKP
levels within the first 24 hours post-treatment. While a slight increase
in pathogen levels was observed after 24 hours, they remained
significantly lower than those treated with conventional disinfectants.
16S rRNA sequencing showed a decrease in the target pathogens’ relative
abundance, while overall species diversity remained stable, confirming
that phages selectively target CRKP without disrupting ecological
balance.DiscussionThe findings highlight the efficacy and safety of
phage-based biocleaners as a sustainable alternative to conventional
disinfectants. Phages selectively reduce multidrug-resistant pathogens
while preserving microbial diversity, making them a promising tool for
infection control.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1442062"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1442062</a></p>
<hr />
<h2
id="identifying-the-plasma-metabolome-responsible-for-mediating-immune-cell-action-in-severe-covid-19-a-mendelian-randomization-investigation">Identifying
the plasma metabolome responsible for mediating immune cell action in
severe COVID-19: a Mendelian randomization investigation</h2>
<p>date: 2024-08-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe immune response regulates the severity of COVID-19
(sCOVID-19). This study examined the cause-and-effect relationship
between immune cell traits (ICTs) and the risk of severe COVID-19.
Additionally, we discovered the potential role of plasma metabolome in
modulating this risk.MethodsEmploying data from a genome-wide
association study (GWAS), we conducted a two-sample Mendelian
randomization (MR) assessment of 731 genetic ICTs and sCOVID-19 (5,101
cases, 1,383,241 controls) incidence. The MR analysis was utilized to
further quantitate the degree of plasma metabolome-mediated regulation
of immune traits in sCOVID-19.ResultsThe inverse variance weighted
method recognized 2 plasma metabolites (PMs) responsible for casual
associations between immune cells and sCOVID-19 risk. These included
Tridecenedioate (C13:1-DC) which regulated the association between CD27
on IgD- CD38br (OR 0.804, 95% CI 0.699–0.925, p = 0.002) and sCOVID-19
risk (mediated proportion: 18.7%); arginine to citrulline ratio which
controlled the relationship of CD39 on monocyte (OR 1.053, 95% CI
1.013–1.094, p = 0.009) with sCOVID-19 risk (mediated proportion:
-7.11%). No strong evidence that genetically predicted sCOVID-19
influenced the aforementioned immune traits.ConclusionIn this study, we
have successfully identified a cause-and-effect relationship between
certain ICTs, PMs, and the likelihood of contracting severe COVID-19.
Our findings can potentially improve the accuracy of COVID-19 prognostic
evaluation and provide valuable insights into the underlying mechanisms
of the disease.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1393432"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1393432</a></p>
<hr />
<h2
id="molecular-mechanism-of-autophagy-in-porcine-reproductive-and-respiratory-syndrome-virus-infection">Molecular
mechanism of autophagy in porcine reproductive and respiratory syndrome
virus infection</h2>
<p>date: 2024-08-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Porcine reproductive and respiratory syndrome virus (PRRSV), a
significant pathogen affecting the swine industry globally, has been
shown to manipulate host cell processes, including autophagy, to
facilitate its replication and survival within the host. Autophagy, an
intracellular degradation process crucial for maintaining cellular
homeostasis, can be hijacked by viruses for their own benefit. During
PRRSV infection, autophagy plays a complex role, both as a defense
mechanism of the host and as a tool exploited by the virus. This review
explores the current understanding of the molecular mechanisms
underlying autophagy induction under PRRSV infection, its impact on
virus replication, and the potential implications for viral pathogenesis
and antiviral strategies. By synthesizing the latest research findings,
this article aims to enhance our understanding of the intricate
relationship between autophagy and PRRSV, paving the way for novel
therapeutic approaches against this swine pathogen.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1434775"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1434775</a></p>
<hr />
<h2
id="development-and-application-of-quadruplex-real-time-quantitative-pcr-method-for-differentiation-of-muscovy-duck-parvovirus-goose-parvovirus-duck-circovirus-and-duck-adenovirus-3">Development
and application of quadruplex real time quantitative PCR method for
differentiation of Muscovy duck parvovirus, Goose parvovirus, Duck
circovirus, and Duck adenovirus 3</h2>
<p>date: 2024-08-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMuscovy duck parvovirus (MDPV), Goose parvovirus (GPV),
Duck circovirus, (DuCV) and Duck adenovirus 3 (DAdV-3) are important
pathogens that cause high morbidity and mortality in ducks, causing huge
economic loss for the duck industry.MethodsThe present study, a
quadruplex one-step real time quantitative PCR method for the detection
of MDPV, GPV, DuCV, and DAdV-3 was developed. ResultsThe results showed
that assay had no cross-reactivity with other poultry pathogens [Duck
plague virus (DPV), Duck tembusu virus (DTMUV), H6 avian influenza virus
(H6 AIV), New duck reovirus (NDRV), Newcastle disease virus (NDV), H4
avian influenza virus (H4 AIV), Escherichia coli (E. coli), Muscovy duck
reovirus (MDRV), Egg drop syndrome virus (EDSV), Pasteurella multocida
(P. multocida)]. The sensitivity result showed that the limits of
detection for MDPV, GPV, DuCV, and DAdV-3 were 10, 10, 1 and 10
copies/µl, respectively; The coefficients of variation intra- and
inter-method was 1-2%; The range of linear (109 to 103 copies/µL)
demonstrated the R2 values for MDPV, GPV, DuCV, and DAdV-3 as 0.9975,
0.998, 0.9964, and 0.996, respectively. The quadruplex real time
quantitative PCR method efficiency was 90.30%, 101.10%, 90.72%, and
90.57% for MDPV, GPV, DuCV, and DAdV-3, respectively. 396 clinical
specimens collected in some duck sausages from June 2022 to July 2023
were simultaneously detected using the established quadruplex real time
quantitative PCR method and the reported assays. The detection rates for
MDPV, GPV, DuCV, and DAdV-3 were 8.33% (33/396), 17.93% (71/396), 33.58%
(133/396), and 29.04% (115/396), respectively. The agreement between
these assays was greater than 99.56%.DiscussionThe developed quadruplex
real-time quantitative PCR assay can accurately detect these four
viruses infecting ducks, providing a rapid, sensitive, specific and
accurate technique for clinical testing.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1448480"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1448480</a></p>
<hr />
<h2
id="metagenomic-next-generation-sequencing-targeted-and-metagenomic-next-generation-sequencing-for-pulmonary-infection-in-hiv-infected-and-non-hiv-infected-individuals">Metagenomic
next-generation sequencing targeted and metagenomic next-generation
sequencing for pulmonary infection in HIV-infected and non-HIV-infected
individuals</h2>
<p>date: 2024-08-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundWhen individuals infected with human immunodeficiency virus
(HIV) experience pulmonary infections, they often exhibit severe
symptoms and face a grim prognosis. Consequently, early, rapid, and
accurate pathogen diagnosis is vital for informing effective treatment
strategies. This study aimed to use metagenomic next-generation
sequencing (mNGS) and targeted mNGS (tNGS) to elucidate the
characteristics of pulmonary infections in HIV and non-HIV
individuals.MethodsThis study enrolled 90 patients with pulmonary
infection at the Department of Infectious Diseases of The First Hospital
of Jilin University from June 2022 to May 2023, and they were divided
into HIV (n=46) and non-HIV (n=44) infection groups. Their
bronchoalveolar lavage fluid (BALF) was collected for mNGS analysis to
evaluate the differences in pulmonary infection pathogens, and tNGS
detection was performed on BALF samples from 15 HIV-infected
patients.ResultsA total of 37 pathogens were identified in this study,
including 21 bacteria, 5 fungi, 5 viruses, 5 mycobacteria, and 1
mycoplasma. The sensitivity of mNGS was 78.9% (71/90), which is
significantly higher than that of conventional methods (CTM) (39/90,
P=1.5E-8). The combination of mNGS with CTM can greatly enhance the
sensitivity of pathogen detection. The prevalence of Pneumocystis
jirovecii (82.6% vs. 9.1%), cytomegalovirus (CMV) (58.7% vs. 0%), and
Epstein-Barr virus (EBV) (17.4% vs. 2.3%) was significantly higher in
the HIV infection group than in the non-HIV infection group (P&lt;0.05).
Although no statistically significant difference was observed, the
detection rate of Mycobacteria was higher in HIV-infected patients
(17.4%) than in the non-HIV group (6.8%). Furthermore, the tNGS results
of BALF from 15 HIV-infected patients were not entirely consistent with
the mNGS results., and the concordance rate of tNGS for the detection of
main pathogens reached 86.7% (13/15).ConclusionNext-generation
sequencing (NGS) can accurately detect pathogens in the BALF of patients
with pulmonary infection. The sensitivity of tNGS is comparable to that
of mNGS. Therefore, this technique should be promoted in the clinic for
better patient outcomes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1438982"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1438982</a></p>
<hr />
<h2
id="phosphatidylserine-liposomes-induce-a-phagosome-acidification-dependent-and-ros-mediated-intracellular-killing-of-mycobacterium-abscessus-in-human-macrophages">Phosphatidylserine
liposomes induce a phagosome acidification-dependent and ROS-mediated
intracellular killing of Mycobacterium abscessus in human
macrophages</h2>
<p>date: 2024-08-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Mycobacterium abscessus (Mab) is an opportunistic nontuberculous
mycobacterium responsible of difficult-to-treat pulmonary infections in
vulnerable patients, such as those suffering from Cystic Fibrosis (CF),
where it represents a major cause of morbidity and mortality.
Additionally, due to the intrinsic extensive antimicrobial resistance
spectrum displayed by this species and the side effects reported for
some available antibiotics, the therapeutic management of such
infections remains extremely difficult. In the present study, we show
that phosphatidylserine liposomes (PS-L) enhance intracellular
mycobacterial killing of Mab infected human macrophages with functional
or pharmacologically inhibited cystic fibrosis conductance regulator
(CFTR), by a mechanism involving phagosome acidification and reactive
oxygen species (ROS) production. Additionally, PS-L significantly reduce
proinflammatory response of Mab infected macrophages in terms of NF-kB
activation and TNF-α production, irrespective of CFTR inhibition.
Altogether, these results represent the proof of concept for a possible
future development of PS-L as a therapeutic strategy against
difficult-to-treat Mab infection.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1443719"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1443719</a></p>
<hr />
<h2
id="lung-microbiota-implications-and-interactions-in-chronic-pulmonary-diseases">Lung
microbiota: implications and interactions in chronic pulmonary
diseases</h2>
<p>date: 2024-08-19, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The lungs, as vital organs in the human body, continuously engage in
gas exchange with the external environment. The lung microbiota, a
critical component in maintaining internal homeostasis, significantly
influences the onset and progression of diseases. Beneficial
interactions between the host and its microbial community are essential
for preserving the host’s health, whereas disease development is often
linked to dysbiosis or alterations in the microbial community. Evidence
has demonstrated that changes in lung microbiota contribute to the
development of major chronic lung diseases, including chronic
obstructive pulmonary disease (COPD), idiopathic pulmonary fibrosis
(IPF), asthma, and lung cancer. However, in-depth mechanistic studies
are constrained by the small scale of the lung microbiota and its
susceptibility to environmental pollutants and other factors, leaving
many questions unanswered. This review examines recent research on the
lung microbiota and lung diseases, as well as methodological
advancements in studying lung microbiota, summarizing the ways in which
lung microbiota impacts lung diseases and introducing research methods
for investigating lung microbiota.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1401448"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1401448</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
