<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2024-12-12 07:05:26)</p>
<hr />
<h2
id="diagnosis-value-of-targeted-and-metagenomic-sequencing-in-respiratory-tract-infection">Diagnosis
value of targeted and metagenomic sequencing in respiratory tract
infection</h2>
<p>date: 2024-12-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundTargeted next-generation sequencing (tNGS) has become a
trending tool in the field of infection diagnosis, but concerns are also
raising about its performance compared with metagenomic next-generation
sequencing (mNGS). This study aims to explore the clinical feasibility
of a tNGS panel for respiratory tract infection diagnosis and compare it
with mNGS in the same cohort of inpatients.Methods180 bronchoalveolar
lavage fluid samples were collected and sent to two centers for mNGS and
tNGS blinded tests, respectively. The concordance between pathogen
reports of both methods and the clinical significance among samples
with/without known etiology was further evaluated.ResultsOverall, both
methods displayed high agreement on pathogen reports, as the average
percent agreement reached 95.29%. But tNGS presented a slightly higher
detection rate per species than mNGS (PWilcoxon=1.212e-05; standard mean
difference = 0.2887091), as detection rates for 32 out of 48 species
were higher than those of mNGS. Due to limitations of panel coverage,
tNGS identified 28 fewer species than mNGS, among which only 3 were
considered clinically relevant. In reference to composite reference
standard, accuracy, sensitivity, and specificity combining both tNGS and
mNGS reached 95.61%, 96.71%, and 95.68%, respectively, while positive
prediction value (PPV) was low at 48.13%, which was caused by low
agreement regarding opportunistic pathogens. tNGS and mNGS improved the
etiology identification in 30.6% (55/180) and 33.9% (61/180) cases,
respectively.ConclusionCollectively, tNGS presented a similar overall
performance in pathogen identification compared to mNGS, but
outperformed in some pathogens. This study also demonstrated that
deployment of tNGS significantly improves etiology identification in
routine practice and provides hints for clinical decisions. The low
agreement between clinical diagnosis and NGS reports towards
opportunistic pathogens implies that adjudication is essential for
report interpretation. Finally, We proposed tNGS as a diagnosis option
in clinical practice due to its cost-efficiency.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1498512"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1498512</a></p>
<hr />
<h2
id="a-simple-predictive-model-for-puerperal-infections-emphasizing-risk-factors-and-pathogen-analysis">A
simple predictive model for puerperal infections: emphasizing risk
factors and pathogen analysis</h2>
<p>date: 2024-12-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPuerperal infection (PI) accounting for approximately 11%
of maternal deaths globally is an important preventable cause of
maternal morbidity and mortality. This study aims to analyze the
high-risk factors and pathogenic bacteria of PI, design a nomogram to
predict the risk of PI occurrence, and provide clinical guidance for
prevention and treatment to improve maternal outcomes.MethodsA total of
525 pregnant women were included in the study. The mothers were randomly
divided into a training cohort (n=367) and a test cohort (n=158). The
performance of our model was assessed using the area under the receiver
operating characteristic (ROC) curve, calibration curve, and decision
curve analyses. All the women in the group of PI underwent blood culture
tests, if the bacteria were detected, drug sensitivity tests were
performed. The drug sensitivity spectrum was recorded and
analyzed.ResultsUnivariate analysis showed that 12 indicators were
significantly different (P &lt; 0.05). Logistic regression analysis
showed 6 factors, such as parity, number of vaginal examinations, amount
of postpartum bleeding, antibiotics administered in one week before
admission, induced labor, and indwelling catheter were significantly
different between the PI group and control group (P &lt; 0.05). The area
under the ROC curve was 0.904 (95% CI: 0.871-0.936) in the training set
and 0.890 (95% CI: 0.837-0.942) in the test set. The calibration curve
of the nomogram showed good agreement between prediction and
observation. The analysis of the clinical decision curve showed that the
nomogram is of practical significance. There were 100 patients with
positive blood cultures in the PI group, and Escherichia.coli was the
main pathogenic bacteria, accounting for 89%. The sensitivity to
Meropenem and Imipenem was 100%, to Piperacillin tazobactam 97.75%, to
Ceftazidime 95.51%, and to Amoxicillin/Clavulanat (AMC) was
93.26%.ConclusionThe risk of PI will be significantly reduced by
controlling the number of vaginal examinations less than 4 times,
postpartum hemorrhage less than 414ml, and reducing the time of urethral
catheter indwelling. If PI was clinically diagnosed or highly suspected,
it was recommended to use antibiotics that were sensitive to
Escherichia. coli, such as Piperacillin tazobactam, Ceftazidime, and
AMC</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1464485"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1464485</a></p>
<hr />
<h2
id="deciphering-the-gut-microbiotas-role-in-diverticular-disease-insights-from-a-mendelian-randomization-study">Deciphering
the gut microbiota’s role in diverticular disease: insights from a
Mendelian randomization study</h2>
<p>date: 2024-12-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPrevious studies have indicated a potential association
between gut microbiota and diverticular disease. However, the precise
nature of this relationship remains unclear. In light of this, we
decided to use a bidirectional two-sample Mendelian randomization (MR)
study to investigate the causal relationship between gut microbiota and
intestinal diverticular disease in greater depth.MethodsTo investigate
the potential causal relationship between gut microbiota and intestinal
diverticular disease, we conducted a two-sample MR study in a European
ancestry. Genetic instrumental variables for gut microbiota were
obtained from a genome-wide association study (GWAS) involving 5,959
participants. Summary statistics for intestinal diverticular disease
were sourced from the IEU Open GWAS project, which included data from
5,193 cases and 457,740 controls. The analysis was primarily conducted
using the inverse variance weighted method, with additional sensitivity
analyses to assess the robustness of the findings.ResultsWith regard to
the findings, 11 microbial taxa were identified as having a potential
causal relationship with intestinal diverticular disease. Specifically,
the microbial taxa Caryophanales, Paenibacillaceae, Herbinix,
Turicibacter, Turicibacteraceae, and Staphylococcus fleurettii were
found to be positively associated with the risk of developing intestinal
diverticular disease, while Chromatiales, Arcobacter, Herbidospora,
Ligilactobacillus ruminis, and Megamonas funiformis were found to be
negatively associated with the risk. Further reverse MR analysis did not
reveal a reverse causal effect between these microbial taxa and
intestinal diverticular disease.ConclusionOur MR analyses revealed a
potential causal relationship between certain gut microbiota and
intestinal diverticular disease, which may provide new directions for
future intestinal diverticular disease prevention and treatment
strategies.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1460504"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1460504</a></p>
<hr />
<h2
id="mmp-3-and-timp-1-as-prognostic-biomarkers-in-vzv-induced-retinal-necrosis">MMP-3
and TIMP-1 as prognostic biomarkers in VZV-induced retinal necrosis</h2>
<p>date: 2024-12-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveAcute retinal necrosis (ARN) caused by varicella-zoster
virus (VZV) is associated with changes in specific proteins in the eye’s
fluid, particularly matrix metalloproteinase-3 (MMP-3), an enzyme that
breaks down tissue structures, and tissue inhibitor of
metalloproteinase-1 (TIMP-1), which regulates MMP activity. This study
aims to investigate how these proteins correlate with the progression of
ARN.MethodsWe analyzed aqueous humor samples from 33 patients with ARN
and 23 control patients with virus-negative uveitis. MMP-3 levels were
measured using immunoturbidimetry, and TIMP-1 levels were determined
using an enzyme-linked immunosorbent assay. We examined the
relationships between these protein levels and clinical findings using
statistical correlation methods.ResultsMMP-3, TIMP-1 were significantly
higher in the aqueous humor of ARN patients compared to the controls
(P&lt;0.0001). Correlation analysis revealed a significant correlation
between MMP-3 levels and TIMP-1 (r = 0.460, P = 0.007). The upregulation
of MMP-3 and TIMP-1 was found to parallel VZV DNA load and IL-6 levels.
Additionally, they exhibited negative correlation with best corrected
visual acuity (BCVA) and positive correlation with the percentage of
active retinal necrosis area.MMP-3 was markedly enhanced in all 14 cases
of retinal detachment (RD), whereas TIMP-1 levels were significantly
reduced in the same cohort of eyes. Patients with initial higher TIMP-1
levels have a significantly increased risk of developing RD, with a
hazard ratio (HR) of 3.152 (95% CI, 1.082-9.18).ConclusionThe imbalance
between MMP-3 and TIMP-1 may play a critical role in the development and
severity of ARN. Measuring these proteins in the eye’s aqueous humor
could be valuable for assessing disease progression and guiding
treatment strategies, potentially improving outcomes for patients with
virus-induced retinal diseases.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1502912"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1502912</a></p>
<hr />
<h2
id="editorial-applications-of-next-generation-sequencing-ngs-technologies-to-decipher-the-oral-microbiome-in-systemic-health-and-disease-volume-ii">Editorial:
Applications of next generation sequencing (NGS) technologies to
decipher the oral microbiome in systemic health and disease, volume
II</h2>
<p>date: 2024-12-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1532762"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1532762</a></p>
<hr />
<h2
id="five-new-species-two-new-sexual-morph-reports-and-one-new-geographical-record-of-apiospora-amphisphaeriales-sordariomycetes-isolated-from-bamboo-in-yunnan-china">Five
new species, two new sexual morph reports, and one new geographical
record of Apiospora (Amphisphaeriales, Sordariomycetes) isolated from
bamboo in Yunnan, China</h2>
<p>date: 2024-12-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Apiospora is an important genus in the Apiosporaceae family with a
worldwide distribution. They exhibit different lifestyles including
pathogenic, saprophytic, and endophytic. In this study, we aimed to
explore the Apiospora associated with bamboo and collected 14
apiospora-like taxa from the forests of Yunnan Province, China.
Morphological and phylogenetic analyses (combined ITS, LSU, tef1-α, and
tub2 sequence data) confirmed that these collections belong to Apiospora
s. str. and reports five new species (viz., Ap. dehongensis, Ap.
jinghongensis, Ap. shangrilaensis, Ap. zhaotongensis, and Ap.
zhenxiongensis). New sexual morphs of asexually typified Ap. globose and
Ap. guangdongensis species, and a new geographical record of Ap.
subglobosa are also reported. The findings of this study not only
enhance the diversity of bambusicolous fungi in the region of Yunnan,
but also geographical distribution of some known Apiospora species.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1476066"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1476066</a></p>
<hr />
<h2
id="functional-role-of-the-biofilm-regulator-csgd-in-salmonella-enterica-sv.-typhi">Functional
role of the biofilm regulator CsgD in Salmonella enterica sv. Typhi</h2>
<p>date: 2024-12-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionTyphoid fever is an infectious disease primarily caused
by Salmonella enterica sv. Typhi (S. Typhi), a bacterium that causes as
many as 20 million infections and 600,000 deaths annually. Asymptomatic
chronic carriers of S. Typhi play a major role in the transmission of
typhoid fever, as they intermittently shed the bacteria and can
unknowingly infect humans in close proximity. An estimated 90% of
chronic carriers have gallstones; biofilm formation on gallstones is a
primary factor in the establishment and maintenance of gallbladder
carriage. CsgD is a central biofilm regulator in Salmonella, but the S.
Typhi csgD gene has a mutation that introduces an early stop codon,
resulting in a protein truncated by 8 amino acids at the C-terminus. In
this study, we investigate the role of role of CsgD in S. Typhi.
MethodsWe introduced a fully functional copy of the csgD gene from S.
Typhimurium into S. Typhi under both a native and a constitutive
promoter and tested for red, dry, and rough (Rdar) colony morphology,
curli fimbriae, cellulose, and biofilm formation. Results and
discussionWe demonstrate that although CsgD-regulated curli and
cellulose production were partially restored, the introduction of the S.
Typhimurium csgD did not induce the Rdar colony morphology.
Interestingly, we show that CsgD does not have a significant role in S.
Typhi biofilm formation, as biofilm-forming capacities depend more on
the isolate than the CsgD regulator. This data suggests the presence of
an alternative biofilm regulatory process in this human-restricted
pathogen.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478488"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478488</a></p>
<hr />
<h2
id="advances-in-diagnostic-assays-for-clostridioides-difficile-infection-in-adults">Advances
in diagnostic assays for Clostridioides difficile infection in
adults</h2>
<p>date: 2024-12-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Clostridioides difficile (C. difficile) was a gram-positive anaerobic
bacterium in the gut, exhibiting clinical manifestations ranging from
mild diarrhoea to fatal pseudomembranous colitis. C. difficile infection
(CDI) remains a serious public health problem and accounted for an
estimated 360,075 cases in the United States in 2021. It has attracted
the utmost attention of the world health organization (WHO). Since
publication of a review of the diagnosis of CDI in adults, new clinical
diagnostic assays have become available and clinical practice guidelines
were updated. This paper presents a comprehensive review of contemporary
laboratory diagnostic approaches for CDI in adult patients, with a focus
on the utilisation and potential advancements of five sophisticated
methodologies, CRISPR in conjunction with nucleic acid amplification
tests (NAATs), gene sequencing technology, ultra-high performance liquid
chromatography-mass spectrometry, Raman spectroscopy, and real-time cell
analysis (RTCA). It can provide new perspectives and ideas for the early
diagnosis of CDI in clinical settings.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1492511"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1492511</a></p>
<hr />
<h2
id="memory-stem-cd8t-cells-in-hivmtb-mono--and-co-infection-characteristics-implications-and-clinical-significance">Memory
stem CD8+T cells in HIV/Mtb mono- and co-infection: characteristics,
implications, and clinical significance</h2>
<p>date: 2024-12-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Human immunodeficiency Virus (HIV) and Mycobacterium tuberculosis
(Mtb) co-infection presents a significant public health challenge
worldwide. Comprehensive assessment of the immune response in HIV/Mtb
co-infection is complex and challenging. CD8+T cells play a pivotal role
in the adaptive immune response to both HIV and Mtb. The differentiation
of CD8+T cells follow a hierarchical pattern, with varying degrees of
exhaustion throughout the process. Memory stem T cells (TSCM cells) is
at the apex of the memory T lymphocyte system, which has recently
emerged as a promising target in immunotherapy. In this context, we
discuss the alterations of CD8+TSCM cells in HIV/Mtb mono- and
co-infection, their implications and clinical significance, and
potential for improving immunotherapy.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1485825"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1485825</a></p>
<hr />
<h2
id="integrated-analysis-of-proteome-and-transcriptome-revealed-changes-in-multiple-signaling-pathways-involved-in-immunity-in-the-northern-snakehead-channa-argus-during-nocardia-seriolae-infection">Integrated
analysis of proteome and transcriptome revealed changes in multiple
signaling pathways involved in immunity in the northern snakehead
(Channa argus) during Nocardia seriolae infection</h2>
<p>date: 2024-12-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The northern snakehead (Channa argus) is a valuable aquaculture
species across certain Asian countries, contributing significantly to
economic prosperity and dietary needs. However, its productivity faces
significant challenges, particularly from diseases such as nocardiosis,
caused by Nocardia seriolae. To date, the majority of research efforts
have focused on describing the observed phenomena related to N. seriolae
infection. However, there remains a notable gap in knowledge concerning
the infectivity of N. seriolae and the immune response it elicits. To
better understand the modulation of the immune responses to N. seriolae
infection in snakeheads, we investigated the splenic proteome profiles.
Specifically, we compared the profiles between uninfected northern
snakehead specimens and those infected with N. seriolae at 96 h using
the label-free data-independent acquisition methodology. A total of 700
differentially expressed proteins (DEPs) were obtained. Of these, 353
proteins exhibited upregulation, whereas 347 proteins displayed
downregulation after the infection. The DEPs were mapped to the
reference canonical pathways in Kyoto Encyclopedia of Genes and Genomes
database, revealing several crucial pathways that were activated
following N. seriolae infection. Noteworthy, among these were pathways
such as ferroptosis, complement and coagulation cascades, chemokine
signaling, tuberculosis, natural killer cell-mediated cytotoxicity, and
Th17 cell differentiation. Furthermore, protein–protein interaction
networks were constructed to elucidate the interplay between
immune-related DEPs. These results revealed expression changes in
multiple signaling pathways during the initial colonization phase of N.
seriolae. This discovery offers novel insights into the infection
mechanisms and host interaction dynamics associated with N.
seriolae.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1482901"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1482901</a></p>
<hr />
<h2
id="the-value-of-metagenomic-next-generation-sequencing-with-blood-samples-for-the-diagnosis-of-disseminated-tuberculosis">The
value of metagenomic next-generation sequencing with blood samples for
the diagnosis of disseminated tuberculosis</h2>
<p>date: 2024-12-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe aim of this study was to assess the clinical value of
metagenomic next-generation sequencing (mNGS) of blood samples for the
identification of disseminated tuberculosis (DTB).MethodsA total of 48
individuals suspected of DTB were enrolled. All patients underwent mNGS
of peripheral blood and conventional microbiological tests. Patient
characteristics were collected from their medical records.ResultsA total
of 28 patients were diagnosed with DTB, whereas 20 patients were
confirmed as non-DTB cases. In the DTB groups, 19 (67.9%) contained TB
sequences, with specific reads of TB ranging from 1 to 219. The TB
sequence was more detectable by mNGS in male patients, those with
elevated PCT levels, those who are HIV positive, and those with a
decreased CD4 T-cell count. The HIV-positive group shows higher TB mNGS
reads (p = 0.012) and TB mNGS sensitivity (p = 0.05). The sensitivity of
TB mNGS in blood samples was 80% for HIV-infected patients and 44.4% for
non-HIV-infected individuals (p = 0.05). The non-HIV group had a higher
prevalence of miliary tuberculosis (p = 0.018), and extrapulmonary
tuberculosis was more prevalent in the HIV-positive group.ConclusionOur
research has shown that the mNGS of blood samples has excellent
sensitivity for the diagnosis of DTB. The TB sequence was more
detectable by mNGS in patients with elevated PCT levels, those who are
HIV positive, and those with a decreased CD4 T-cell count.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1456119"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1456119</a></p>
<hr />
<h2
id="baicalin-promotes-β-13-glucan-exposure-in-candida-albicans-and-enhances-macrophage-response">Baicalin
promotes β-1,3-glucan exposure in Candida albicans and enhances
macrophage response</h2>
<p>date: 2024-12-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Among the diverse fungal opportunistic pathogens, Candida albicans
garners significant attention due to its wide range of infections and
high frequency of occurrence. The emergence of resistance and the
limited number of antifungals drives the need to develop novel
antifungal drugs. Although the natural product baicalin has been shown
to trigger apoptosis in C. albicans in previous experiments, its
influence on cell wall (CW) structure along with immune recognition
remains elusive. In this work, baicalin showed a significant killing
effect against C. albicans SC5314. Moreover, CW destruction,
characterized by β-1,3-glucan unmasking and chitin deposition, was
observed as a consequence of the treatment with baicalin. The RNA
sequencing analysis revealed that treatment with baicalin resulted in
eight hundred forty-two differentially expressed genes (DEGs).
Sixty-five genes, such as GSC1, ENG1, CHS3, GWT1, and MKC1, were
associated with CW organization or biogenesis. Baicalin-pretreated C.
albicans SC5314 was phagocytosed more efficiently by RAW264.7
macrophages, accompanied by increased TNF-α and IL-1β production.
Accordingly, it is hypothesized that baicalin could stimulate
β-1,3-glucan unmasking by governing CW-associated gene expression in C.
albicans SC5314, which contributes to macrophage recognition and
clearance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1487173"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1487173</a></p>
<hr />
<h2
id="dysfunction-and-regulatory-interplay-of-t-and-b-cells-in-chronic-hepatitis-b-immunotherapy-and-emerging-antiviral-strategies">Dysfunction
and regulatory interplay of T and B cells in chronic hepatitis B:
immunotherapy and emerging antiviral strategies</h2>
<p>date: 2024-12-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>In the context of chronic hepatitis B virus (HBV) infection, the
continuous replication of HBV within host hepatocytes is a
characteristic feature. Rather than directly causing hepatocyte
destruction, this replication leads to immune dysfunction and
establishes a state of T-B immune tolerance. Successful clearance of the
HBV virus is dependent on the close collaboration between humoral and
cellular immunity. Humoral immunity, mediated by B-cell subpopulations,
and cellular immunity, dominated by T-cell subpopulations show varying
degrees of dysfunction during chronic hepatitis B (CHB). Notably, not
all T- and B-cells produce positive immune responses. This review
examine the most recent developments in the mutual regulation of T-B
cells during chronic HBV infection. Our focus is on the prevailing
immunotherapeutic strategies, such as T cell engineering, HBV-related
vaccines, PD-1 inhibitors, and Toll-like receptor agonists. While
nucleos(t)ide analogues (NUCs) and interferons have notable limitations,
including inadequate viral suppression, drug resistance, and adverse
reactions, several HBV entry inhibitors have shown promising clinical
efficacy. To overcome the challenges posed by NUCs or monotherapy, the
combination of immunotherapy and novel antiviral agents presents a
promising avenue for future CHB treatment and potential cure.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1488527"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1488527</a></p>
<hr />
<h2
id="development-of-a-prediction-nomogram-for-igg-levels-among-asymptomatic-or-mild-patients-with-covid-19">Development
of a prediction nomogram for IgG levels among asymptomatic or mild
patients with COVID-19</h2>
<p>date: 2024-12-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveCOVID-19 has evolved into a seasonal coronavirus disease,
characterized by prolonged infection duration and repeated infections,
significantly increasing the risk of patients developing long COVID. Our
research focused on the immune responses in asymptomatic and mild cases,
particularly the critical factors influencing serum immunoglobulin G
(IgG) levels and their predictive value.MethodsWe conducted a
retrospective analysis on data from 1939 asymptomatic or mildly
symptomatic COVID-19 patients hospitalized between September 2022 and
June 2023. Spearman methods were used to test the correlation between
serum IgG and age, immunoglobulin M (IgM), procalcitonin (PCT),
interleukin-6 (IL-6), nucleic acid conversion time, and BMI. Univariate
and multivariate logistic regression analyses identified independent key
factors influencing serum IgG levels, which were integrated and
visualized in a nomogram. Finally, receiver operating characteristic
(ROC) curves were plotted to predict the model’s diagnostic performance
by calculating the AUC.ResultsMild patients showed higher levels of IgG,
IgM, and longer nucleic acid conversion times than asymptomatic
patients, and a lower proportion of them had received ≥ 3 COVID-19
vaccine doses. Serum IgG was positively correlated with serum IgM and
negatively correlated with age, PCT, IL-6, and BMI. Notably, it showed a
moderate negative correlation with nucleic acid conversion time (r =
-0.578, P &lt; 0.001). Logistic regression results showed that younger
age, lower IL-6 levels, ≥ 3 doses of vaccine, and no comorbidities were
independent predictors of serum IgG levels ≥ 21.08 g/L. We used age,
IL-6 levels, vaccine doses, and comorbidities to create a nomogram for
predicting serum IgG levels, with the area under the ROC curve reaching
0.772.ConclusionAge, IL-6 levels, vaccination status, and comorbidities
were independent predictors of serum IgG levels in asymptomatic or mild
COVID-19 patients, facilitating risk stratification and clinical
decision-making. Notably, receiving ≥3 doses of the COVID-19 vaccine was
the most beneficial factor for elevated serum IgG levels.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477585"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477585</a></p>
<hr />
<h2
id="epidemiological-molecular-and-evolutionary-characteristics-of-g1p8-rotavirus-in-china-on-the-eve-of-rotateq-application">Epidemiological,
molecular, and evolutionary characteristics of G1P[8] rotavirus in China
on the eve of RotaTeq application</h2>
<p>date: 2024-12-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThis study, conducted in China prior to RotaTeq’s launch,
examined the epidemiological, molecular, and evolutionary features of
the G1P[8] genotype RVA in children admitted with diarrhea, to aid in
evaluating its efficacy and impact on G1P[8] RVA in China.MethodsData
from the Chinese viral diarrhea surveillance network were collected from
January 2016 to December 2018. RVA strains identified as the G1P[8]
genotype were subjected to whole-genome sequencing. Neutralizing
epitope, amino acid selection pressure, and evolution dynamics analyses
on VP7 and VP4 were performed using BioEdit v.7.0.9.0 and PyMOL v.2.5.2,
four algorithms (MEME, SLAC, FEL, and FUBAR) in the Datamonkey online
software, and the MCMC model in BEAST v. 1.10.4, respectively.
Phylogenetic and identity features of 11 genes were assessed by DNAStar
and MEGA v.7.ResultsResults showed that the detection rate of G1P[8] in
China from 2016 to 2018 was generally low with significant seasonality.
The whole genome of G1P[8] of four 2016 childhood diarrhea specimens was
successfully sequenced. Phylogenetic and neutralizing epitope analysis
showed that Rotavin-M1 might have better protection on G1P[8] prevalent
in China than Rotarix and RotaTeq. Two conserved N-glycosylation sites
on VP7 of Chinese G1P[8] might affect the protective effect of the
vaccine. Evolution rate and selection pressure analysis identified the
possibility of rapidly evolving and adapting to the new environment
introduced by vaccines of G1P[8], whereas positive selection specific to
VP4 indicated the potential tendency to select for dominant traits.
Identity and phylogeny analysis showed that Chinese G1P[8] from before
2018 was generally stable with possible genetic recombination among
local strains.DiscussionThese findings not only are of great
significance for predicting the prevalence of G1P [8] in China, but also
provide data reference for evaluating rotavirus vaccine efficacy.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1453862"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1453862</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
