<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2024-12-16 07:06:21)</p>
<hr />
<h2
id="new-tools-to-monitor-pseudomonas-aeruginosa-infection-and-biofilms-in-vivo-in-c.-elegans">New
tools to monitor Pseudomonas aeruginosa infection and biofilms in vivo
in C. elegans</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionAntimicrobial resistance is a growing health problem.
Pseudomonas aeruginosa is a pathogen of major concern because of its
multidrug resistance and global threat, especially in health-care
settings. The pathogenesis and drug resistance of P. aeruginosa depends
on its ability to form biofilms, making infections chronic and
untreatable as the biofilm protects against antibiotics and host
immunity. A major barrier to developing new antimicrobials is the lack
of in vivo biofilm models. Standard microbiological testing is usually
performed in vitro using planktonic bacteria, without representation of
biofilms, reducing translatability. Here we develop tools to study both
infection and biofilm formation by P. aeruginosa in vivo to accelerate
development of strategies targeting infection and pathogenic
biofilms.MethodsBiofilms were quantified in vitro using Crystal Violet
staining and fluorescence biofilm assays. For in vivo assays, C. elegans
were infected with P. aeruginosa strains. Pathogenicity was quantified
by measuring healthspan, survival and GFP fluorescence. Healthspan
assays were performed using the WormGazerTM automated imaging
technology.ResultsUsing the nematode Caenorhabditis elegans and P.
aeruginosa reporters combined with in vivo imaging we show that
fluorescent P. aeruginosa reporters that form biofilms in vitro can be
used to visualize tissue infection. Using automated tracking of C.
elegans movement, we find that that the timing of this infection
corresponds with a decline in health endpoints. In a mutant strain of P.
aeruginosa lacking RhlR, a transcription factor that controls quorum
sensing and biofilm formation, we find reduced capacity of P. aeruginosa
to form biofilms, invade host tissues and negatively impact healthspan
and survival.DiscussionOur findings suggest that RhlR could be a new
antimicrobial target to reduce P. aeruginosa biofilms and virulence in
vivo and C. elegans could be used to more effectively screen for new
drugs to combat antimicrobial resistance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478881"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478881</a></p>
<hr />
<h2
id="diagnostic-and-prognostic-values-of-nsclc-patients-with-or-without-obstructive-pneumonia-after-sleeve-lobectomy">Diagnostic
and prognostic values of NSCLC patients with or without obstructive
pneumonia after sleeve lobectomy</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveWe aimed to identify the diagnostic value of next-generation
sequencing (NGS) of bronchoalveolar lavage fluid (BALF) from patients
with non-small-cell lung cancer (NSCLC).MethodsForty patients who were
initially diagnosed with pulmonary nodules were enrolled. Frozen section
histology was used to identify the NSCLC cell types. NGS of collected
BALF samples was used for microbial identification. We compared the
bacterial and viral distributions in BALF samples from patients with
NSCLC with and without obstructive pneumonia as well as their NSCLC
drainage times following surgery.ResultsOf the 29 patients with NSCLC,
eight had obstructive pneumonia. Streptococcus pneumoniae, Streptococcus
pseudopneumoniae, and Haemophilus parainfluenzae were the top three
bacteria present in almost 50% of patients, both with and without
obstructive pneumonia. The viral detection rate was higher in the BALF
of patients with NSCLC who did not have obstructive pneumonia. However,
in patients with NSCLC and drain times of &gt;5 days, the human herpes
virus type 7 detection rate was higher following surgery than it was in
patients with NSCLC who had drain times of ≤5 days.ConclusionViral
imbalance in NSCLC is closely related to the occurrence of obstructive
pneumonia and postoperative drainage time.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474998"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474998</a></p>
<hr />
<h2
id="two-genotypes-of-h3n2-swine-influenza-viruses-identified-in-pigs-from-shandong-province-china">Two
genotypes of H3N2 swine influenza viruses identified in pigs from
Shandong Province, China</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Swine influenza virus (SIV) is a highly contagious pathogen that
poses significant economic challenges to the swine industry and carries
zoonotic potential, underscoring the need for vigilant surveillance. In
this study, we performed a comprehensive genetic and molecular analysis
of H3N2 SIV isolates obtained from 372 swine samples collected in
Shandong Province, China. Phylogenetic analysis revealed two distinct
genotypes. The surface genes of both genotypes clustered with the
human-like H3N2 lineage, while the internal genes of one genotype
clustered with the 2009 pandemic H1N1 (pdm/09) lineage. In the second
genotype, the NS gene clustered with the classical swine (CS) H1N1
lineage, while the remaining internal genes clustered with pdm/09,
suggesting stable integration of pdm/09 gene segments into H3N2 SIV.
Homology analysis showed over 96% genetic similarity between the
isolates and reference strains from China and Brazil, suggesting
potential transmission through swine trade or human movement. Molecular
characterization identified amino acid substitutions in the HA protein
(190D, 226I, and 228S), potentially enhancing the virus’s affinity for
human-like receptors, thereby increasing the zoonotic risk. Key
mutations in the PB2 (271A, 591R), PA (336M, 356R, 409N), and M2 (S31N)
proteins, along with novel drug resistance mutations, indicate the
potential for enhanced virulence and drug resistance in these isolates.
Moreover, glycosylation site analysis revealed four differences, and
antigenic site analysis showed 13 differences between the HA proteins of
the isolates and the WHO-recommended vaccine strain
A/Cambodia/E0826360/2020 for the 2021–2022 season, which may reduce
vaccine efficacy. Serological analysis revealed that 11 out of the
tested serum samples were positive for H3N2 antibodies, resulting in an
overall positivity rate of 0.42%. These findings emphasize the urgent
need for strengthened SIV surveillance in China to monitor the risk of
human transmission and ensure better preparedness for future influenza
outbreaks.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1517023"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1517023</a></p>
<hr />
<h2
id="identification-and-characterization-of-a-relish-type-nf-κb-dvrelish-in-dermacentor-variabilis-in-response-to-rickettsia-rickettsii-infection">Identification
and characterization of a Relish-type NF-κB, DvRelish, in Dermacentor
variabilis in response to Rickettsia rickettsii infection</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Ixodid ticks serve as hosts and transmission vectors for several
obligate intracellular bacteria, including members of the spotted fever
group (SFG) of Rickettsia. Although ticks generate an immune response to
bacterial insults, many of the signaling molecules associated with the
response and how they may contribute to vector competence for Rickettsia
are undefined. In this study, we isolated a full-length dvrelish
transcript from Dermacentor variabilis, which encoded a Relish-type
NF-κB. The presence of a canonical Rel homology domain (RHD) consistent
with NF-κB proteins suggested a role in tick immune response for
DvRelish. The expression of DvRelish was confirmed in tick tissues and
fluorescent microscopy of tick hemocytes indicated increased expression
following infection with Rickettsia as compared to a non-tick-borne
bacterial pathogen. To further determine the effect of dvRelish gene
knockdown on rickettsial infection, we used RNA interference-mediated
gene knockdown in D. variabilis and demonstrated that transcription of
dvRelish was decreased after 24 h post-injection of siRNA. We then
assessed the response of D. variabilis when exposed to Rickettsia
rickettsii and determined that transcription of dvRelish was inversely
associated with rickettsial loads at 48 h post-exposure. Further studies
are required to broaden the understanding of differential immune
responses in ticks to SFG Rickettsia infection and elucidate the role
played by the arthropod immune system in vector competence.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494450"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494450</a></p>
<hr />
<h2
id="transcriptomic-profiling-of-brain-eating-amoeba-naegleria-fowleri-infection-in-mice-the-host-and-the-protozoa-perspectives">Transcriptomic
profiling of “brain-eating amoeba” Naegleria fowleri infection in mice:
the host and the protozoa perspectives</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The free-living amoeba Naegleria fowleri (NF) causes a rare but
lethal parasitic meningoencephalitis (PAM) in humans. Currently, this
disease lacks effective treatments and the specific molecular mechanisms
that govern NF pathogenesis and host brain response remain unknown. To
address some of these issues, we sought to explore naturally existing
virulence diversity within environmental NF isolates. Herein, we
purified two new NF environmental isolates (NF45 and NF1) and tested
their in vivo virulence using experimental infection in mice. We found
that NF45 was highly virulent (NF45_HV) compared with NF1 (low
virulence, NF1_LV), based on in vivo amoeba growth kinetics and mouse
survival. To identify underlying differences, we conducted RNA-seq and
bioinformatics analyses from the infected mouse brains. Our results
showed that NF1_LV and NF45_HV modulated the expression of their genes
during mouse brain infection. Differentially expressed genes (DEGs) in
NF1_LV were mostly involved in Translational protein, Protein-binding
activity modulator, Protein modifying enzyme, while DEGs in NF45_HV were
related to DNA metabolism, Cytoskeletal protein, Protein-binding
activity modulator. Proteases (namely the virulence factor Cathepsin B)
were upregulated in NF1_LV, while downregulated in NF45_HV. When
analyzing the host response against infection by these two NF strains,
enrichment analyses uncovered genes and mechanisms related to the host
immune responses and nervous systems. We detected more DEGs in NF1_LV
infected mice compared to NF45_HV, related to blood brain barrier
leakage, immune cell recruitment, cytokine production (including IL-6,
IFN-Ɣ and TNFα), inflammation of astrocytes and microglia, and
oligodendrocyte and neurons degeneration. Increased expression of
neuromotor-related genes such as Adam22, Cacnb4 and Zic1 (activated by
NF1_LV infection) and ChAt (activated by NF45_LV infection) could
explain PAM symptoms such as muscle weakness and seizures. Globally, our
results showed that NF isolated from the environment can have different
levels of virulence and differentially modulate their gene expression
during brain infection. We also provided, for the first time, a
comprehensive information for the molecular mechanisms of neuro-immune
and host–pathogen interactions during PAM disease. As the host and the
protozoa are strongly implicated in PAM lethality, new therapies
targeting both the parasite, and the host should be considered to treat
PAM infection.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1490280"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1490280</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
