<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.28</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="../../snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../search.html">Search</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.28</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.28">health 2024.28</h1>
<p>(date: 2024-07-12 09:21:05)</p>
<hr />
<h2
id="the-salivary-microbiome-as-a-diagnostic-biomarker-of-periodontitis-a-16s-multi-batch-study-before-and-after-the-removal-of-batch-effects">The
salivary microbiome as a diagnostic biomarker of periodontitis: a 16S
multi-batch study before and after the removal of batch effects</h2>
<p>date: 2024-07-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionMicrobiome-based clinical applications that improve
diagnosis related to oral health are of great interest to precision
dentistry. Predictive studies on the salivary microbiome are scarce and
of low methodological quality (low sample sizes, lack of biological
heterogeneity, and absence of a validation process). None of them
evaluates the impact of confounding factors as batch effects (BEs). This
is the first 16S multi-batch study to analyze the salivary microbiome at
the amplicon sequence variant (ASV) level in terms of differential
abundance and machine learning models. This is done in periodontally
healthy and periodontitis patients before and after removing
BEs.MethodsSaliva was collected from 124 patients (50 healthy, 74
periodontitis) in our setting. Sequencing of the V3-V4 16S rRNA gene
region was performed in Illumina MiSeq. In parallel, searches were
conducted on four databases to identify previous Illumina V3-V4
sequencing studies on the salivary microbiome. Investigations that met
predefined criteria were included in the analysis, and the own and
external sequences were processed using the same bioinformatics
protocol. The statistical analysis was performed in the R-Bioconductor
environment.ResultsThe elimination of BEs reduced the number of ASVs
with differential abundance between the groups by approximately
one-third (Before=265; After=190). Before removing BEs, the model
constructed using all study samples (796) comprised 16 ASVs (0.16%) and
had an area under the curve (AUC) of 0.944, sensitivity of 90.73%, and
specificity of 87.16%. The model built using two-thirds of the specimens
(training=531) comprised 35 ASVs (0.36%) and had an AUC of 0.955,
sensitivity of 86.54%, and specificity of 90.06% after being validated
in the remaining one-third (test=265). After removing BEs, the models
required more ASVs (all samples=200–2.03%; training=100–1.01%) to obtain
slightly lower AUC (all=0.935; test=0.947), lower sensitivity
(all=81.79%; test=78.85%), and similar specificity (all=91.51%;
test=90.68%).ConclusionsThe removal of BEs controls false positive ASVs
in the differential abundance analysis. However, their elimination
implies a significantly larger number of predictor taxa to achieve
optimal performance, creating less robust classifiers. As all the
provided models can accurately discriminate health from periodontitis,
implying good/excellent sensitivities/specificities, the salivary
microbiome demonstrates potential clinical applicability as a precision
diagnostic tool for periodontitis.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1405699"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1405699</a></p>
<hr />
<h2
id="screening-for-anti-influenza-virus-compounds-from-traditional-mongolian-medicine-by-gfp-based-reporter-virus">Screening
for anti-influenza virus compounds from traditional Mongolian medicine
by GFP-based reporter virus</h2>
<p>date: 2024-07-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionScreening for effective antiviral compounds from
traditional Mongolian medicine not only aids in the research of
antiviral mechanisms of traditional medicines, but is also of
significant importance for the development of new antiviral drugs
targeting influenza A virus. Our study aimed to establish
high-throughput, rapid screening methods for antiviral compounds against
influenza A virus from abundant resources of Mongolian
medicine.MethodsThe use of GFP-based reporter viruses plays a pivotal
role in antiviral drugs screening by enabling rapid and precise
identification of compounds that inhibit viral replication. Herein, a
GFP-based reporter influenza A virus was used to identify potent
anti-influenza compounds within traditional Mongolian
medicine.ResultsOur study led to the discovery of three active
compounds: Cardamonin, Curcumin, and Kaempferide, all of which exhibited
significant antiviral properties in vitro. Subsequent analysis confirmed
that their effectiveness was largely due to the stimulation of the
antiviral signaling pathways of host cells, rather than direct
interference with the viral components, such as the viral
polymerase.DiscussionThis study showcased the use of GFP-based reporter
viruses in high-throughput screening to unearth antiviral agents from
traditional Mongolian medicine, which contains rich antiviral compounds
and deserves further exploration. Despite certain limitations,
fluorescent reporter viruses present substantial potential for antiviral
drug screening research due to their high throughput and efficiency.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431979"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1431979</a></p>
<hr />
<h2
id="clinical-utility-of-metagenomic-next-generation-sequencing-in-the-diagnosis-of-invasive-pulmonary-aspergillosis-in-acute-exacerbation-of-chronic-obstructive-pulmonary-disease-patients-in-the-intensive-care-unit">Clinical
utility of metagenomic next-generation sequencing in the diagnosis of
invasive pulmonary aspergillosis in acute exacerbation of chronic
obstructive pulmonary disease patients in the intensive care unit</h2>
<p>date: 2024-07-12, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveTo explore the clinical utility of metagenomic
next-generation sequencing (mNGS) in diagnosing invasive pulmonary
aspergillosis (IPA) among patients with acute exacerbation of chronic
obstructive pulmonary disease (AECOPD) in the intensive care unit
(ICU).MethodsA retrospective analysis was conducted on patients with
AECOPD admitted to the ICU of Xinxiang Central Hospital in Henan
Province, China, between March 2020 and September 2023, suspected of
having IPA. Bronchoalveolar lavage fluid (BALF) samples were collected
for fungal culture, the galactomannan (GM) test, and mNGS. Based on host
factors, clinical features, and microbiological test results, patients
were categorized into 62 cases of IPA and 64 cases of non-IPA.
Statistical analysis was performed to compare the diagnostic efficacy of
fungal culture, the serum and BALF GM test, and mNGS detection for IPA
in patients with AECOPD.Results1. The sensitivity and specificity of
mNGS in diagnosing IPA were 70.9% and 71.8% respectively, with the
sensitivity of mNGS surpassing that of fungal culture (29.0%,
P&lt;0.01), serum GM test (35.4%, P&lt;0.01), and BALF GM test (41.9%,
P&lt;0.05), albeit with slightly lower specificity compared to fungal
culture (90.6%, P &gt;0.05), serum GM test (87.5%, P &gt;0.05), and BALF
GM test (85.9%, P &gt;0.05).Combining fungal culture with the GM test
and mNGS resulted in a sensitivity of 80.6% and a specificity of 92.2%,
underscoring a superior diagnostic rate compared to any single detection
method. 2.mNGS accurately distinguished strains of the Aspergillus
genus. 3.The area under the ROC curves of mNGS was 0.73, indicating good
diagnostic performance. 4.The detection duration for mNGS is shorter
than that of traditional fungal culture and GM testing.ConclusionmNGS
presents a pragmatic and highly sensitive approach, serving as a
valuable complementary tool to conventional microbiological tests (CMT).
Our research demonstrated that, compared to fungal culture and GM
testing, mNGS exhibits superior diagnostic capability for IPA among
patients with AECOPD. Integration of mNGS with established conventional
methods holds promise for improving the diagnosis rate of IPA.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397733"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397733</a></p>
<hr />
<h2
id="target-next-generation-sequencing-for-the-accurate-diagnosis-of-parvimonas-micra-lung-abscess-a-case-series-and-literature-review">Target
next-generation sequencing for the accurate diagnosis of Parvimonas
micra lung abscess: a case series and literature review</h2>
<p>date: 2024-07-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundParvimonas micra (P. micra) has been identified as a
pathogen capable of causing lung abscesses; however, its identification
poses challenges due to the specialized culture conditions for anaerobic
bacterial isolation. Only a few cases of lung abscesses caused by P.
micra infection have been reported. Therefore, we describe the clinical
characteristics of lung abscesses due to P. micra based on our case
series.MethodsA retrospective analysis was conducted on eight patients
who were diagnosed with lung abscesses attributed to P. micra. Detection
of P. micra was accomplished through target next-generation sequencing
(tNGS). A systematic search of the PubMed database using keywords “lung
abscess” and “Parvimonas micra/Peptostreptococcus micros” was performed
to review published literature pertaining to similar cases.ResultsAmong
the eight patients reviewed, all exhibited poor oral hygiene, with four
presenting with comorbid diabetes. Chest computed tomography (CT) showed
high-density mass shadows with necrosis and small cavities in the
middle. Bronchoscopic examination revealed purulent sputum and bronchial
mucosal inflammation. Thick secretions obstructed the airway, leading to
the poor drainage of pus, and the formation of local abscesses leading
to irresponsive to antibiotic therapy, which finally protracted recovery
time. P. micra was successfully identified in bronchoalveolar lavage
fluid (BALF) samples from all eight patients using tNGS; in contrast,
sputum and BALF bacterial cultures yielded negative results, with P.
micra cultured from only one empyema sample. Following appropriate
antibiotic therapy, seven patients recovered. In previously documented
cases, favorable outcomes were observed in 77.8% of individuals treated
with antibiotics and 22.2% were cured after surgical interventions for
P. micra lung abscesses.ConclusionsThis study enriches our understanding
of the clinical characteristics associated with lung abscesses
attributed to P. micra. Importantly, tNGS has emerged as a rapid and
effective diagnostic test in scenarios where traditional sputum cultures
are negative. Encouragingly, patients with lung abscesses caused by P.
micra infection exhibit a favorable prognosis with effective airway
clearance and judicious anti-infective management.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416884"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416884</a></p>
<hr />
<h2
id="association-between-gut-microbiota-and-adrenal-disease-a-two-sample-mendelian-randomized-study">Association
between gut microbiota and adrenal disease: a two-sample Mendelian
randomized study</h2>
<p>date: 2024-07-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSome observational studies and clinical experiments suggest
a close association between gut microbiota and metabolic diseases.
However, the causal effects of gut microbiota on adrenal diseases,
including Adrenocortical insufficiency, Cushing syndrome, and
Hyperaldosteronism, remain unclear.MethodsThis study conducted a
two-sample Mendelian randomization analysis using summary statistics
data of gut microbiota from a large-scale genome-wide association study
conducted by the MiBioGen Consortium. Summary statistics data for the
three adrenal diseases were obtained from the FinnGen study. The study
employed Inverse variance weighting, MR-Egger, and MR-PRESSO methods to
assess the causal relationship between gut microbiota and these three
adrenal diseases. Additionally, a reverse Mendelian randomization
analysis was performed for bacteria found to have a causal relationship
with these three adrenal diseases in the forward Mendelian randomization
analysis. Cochran’s Q statistic was used to test for heterogeneity of
instrumental variables.ResultsThe IVW test results demonstrate that
class Deltaproteobacteria, Family Desulfovibrionaceae, and Order
Desulfovibrionales exhibit protective effects against adrenocortical
insufficiency. Conversely, Family Porphyromonadaceae, Genus
Lachnoclostridium, and Order MollicutesRF9 are associated with an
increased risk of adrenocortical insufficiency. Additionally, Family
Acidaminococcaceae confers a certain level of protection against Cushing
syndrome. In contrast, Class Methanobacteria, Family Lactobacillaceae,
Family Methanobacteriaceae, Genus. Lactobacillus and Order
Methanobacteriales are protective against Hyperaldosteronism.
Conversely, Genus Parasutterella, Genus Peptococcus, and Genus
Veillonella are identified as risk factors for
Hyperaldosteronism.ConclusionsThis two-sample Mendelian randomization
analysis revealed a causal relationship between microbial taxa such as
Deltaproteobacteria and Desulfovibrionaceae and Adrenocortical
insufficiency, Cushing syndrome, and Hyperaldosteronism. These findings
offer new avenues for comprehending the development of adrenal diseases
mediated by gut microbiota.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1421128"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1421128</a></p>
<hr />
<h2
id="oral-microbiota-distinguishes-patients-with-osteosarcoma-from-healthy-controls">Oral
microbiota distinguishes patients with osteosarcoma from healthy
controls</h2>
<p>date: 2024-07-11, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThe human microbiota plays a key role in cancer diagnosis,
pathogenesis, and treatment. However, osteosarcoma-associated oral
microbiota alterations have not yet been unraveled. The aim of this
study was to explore the characteristics of oral microbiota in
osteosarcoma patients compared to healthy controls, and to identify
potential microbiota as a diagnostic tool for osteosarcoma.MethodsThe
oral microbiota was analyzed in osteosarcoma patients (n = 45) and
matched healthy controls (n = 90) using 16S rRNA MiSeq sequencing
technology.ResultsThe microbial richness and diversity of the tongue
coat were increased in osteosarcoma patients as estimated by the
abundance-based coverage estimator indices, the Chao, and observed
operational taxonomy units (OTUs). Principal component analysis
delineated that the oral microbial community was significant differences
between osteosarcoma patients and healthy controls. 14 genera including
Rothia, Halomonas, Rhodococcus, and Granulicatella were remarkably
reduced, whereas Alloprevotella, Prevotella, Selenomonas, and
Campylobacter were enriched in osteosarcoma. Eventually, the optimal
four OTUs were identified to construct a microbial classifier by the
random forest model via a fivefold cross-validation, which achieved an
area under the curve of 99.44% in the training group (30 osteosarcoma
patients versus 60 healthy controls) and 87.33% in the test group (15
osteosarcoma patients versus 30 healthy controls), respectively.
Notably, oral microbial markers validated strong diagnostic potential
distinguishing osteosarcoma patients from healthy
controls.ConclusionThis study comprehensively characterizes the oral
microbiota in osteosarcoma and reveals the potential efficacy of oral
microbiota-targeted biomarkers as a noninvasive biological diagnostic
tool for osteosarcoma.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1383878"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1383878</a></p>
<hr />
<h2
id="clinical-course-and-prognostic-factors-of-pneumocystis-pneumonia-with-respiratory-failure-in-non-hiv-patients">Clinical
course and prognostic factors of Pneumocystis pneumonia with respiratory
failure in non-HIV patients</h2>
<p>date: 2024-07-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundCompared with Human Immunodeficiency Virus (HIV) patients,
non-HIV patients with Pneumocystis pneumonia (PCP) have more rapid
onset, more rapid progression, and higher mortality.ObjectivesTo
investigate the predictive value of variables obtained upon hospital
admission for in-hospital death and 90-day outcomes in non-HIV-PCP
patients with respiratory failure (RF).MethodsThis was a single center
retrospective study in a tertiary care institution over 15 years. It
included all adults inpatients (≥18 years old) with laboratory confirmed
non-HIV-PCP with RF who were discharged or died from Peking University
First Hospital between April 1st, 2007 and November 1st, 2022.
Epidemiological, clinical, laboratory, imaging and outcome data were
collected from patient records.ResultsIn this study, a total of 146
non-HIV-PCP patients with RF were included. There were 57 patients (39%)
died during hospitalization, 44 patients (53%) died in Intensive care
unit (ICU). A total of 137 patients completed 90 days of follow-up, of
which 58 (42.3%) died. The multivariable regression analysis revealed
that a CD8+ T cell count &lt;115/μl (P=0.009), bronchoalveolar lavage
fluid (BALF)-neutrophil percentage ≥50% (P=0.047), the time from
corticosteroids withdrawal to symptom onset ≤5 days (P=0.012), and the
time from visit to initiation of sulfonamides ≥2 days (P=0.011) were
independent risk factors for in-hospital death. Furthermore, a CD8+ T
cell count &lt; 115/μl (P=0.001) and the time from visit to initiation
of sulfonamides therapy ≥2 days (P=0.033) was independently associated
with 90-day all-cause death.ConclusionsA low CD8+ T cell count in
peripheral blood, a high percentage of BALF-neutrophils, a short time
from corticosteroids withdrawal to symptom onset, and a long time from
visit to initiation of sulfonamides are associated with poor prognosis
in non-HIV-PCP patients with RF.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1380494"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1380494</a></p>
<hr />
<h2
id="the-role-of-gut-microbiota-and-blood-metabolites-in-postpartum-depression-a-mendelian-randomization-analysis">The
role of gut microbiota and blood metabolites in postpartum depression: a
Mendelian randomization analysis</h2>
<p>date: 2024-07-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPostpartum depression (PPD) is a common complication of
pregnancy that imposes a heavy health and economic burden on
individuals, families and society. The etiology of PPD is complex and
incompletely defined, and recent studies have identified an important
role for gut microbiota (GM) and their metabolites in neurological
disorders. However, fewer studies on GM and PPD are available and have
not yielded uniform results.MethodsInstrumental variables for GM and
blood metabolites were obtained from the MiBioGen consortium and
metabolomics GWAS server. Single nucleotide polymorphisms (SNPs)
associated with PPD phenotypes were obtained from the FinnGen
consortium. Inverse variance weighted (IVW), weighted median, weighted
mode, and MR-Egger methods were used to assess causal effects. Inverse
MR analysis and sensitivity analysis were also utilized to improve the
stability of the results.ResultsIn this study, 5 intestinal species and
24 blood metabolites causally associated with PPD were identified using
MR analysis. In addition, MR analysis showed that Prevotellaceae and
Bifidobacteria may reduce the risk of PPD by elevating Xanthine and
1-arachidonoylglycerophosphoinositol (LysoPI) levels.ConclusionsThis
study identified GM and blood metabolites causally associated with PPD.
The results of this study may provide a theoretical basis for the
discovery of PPD-related biomarkers and the treatment of the disease by
regulating the gut microenvironment.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416298"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416298</a></p>
<hr />
<h2
id="mapping-knowledge-landscapes-and-research-frontiers-of-gastrointestinal-microbiota-and-bone-metabolism-a-text-mining-study">Mapping
knowledge landscapes and research frontiers of gastrointestinal
microbiota and bone metabolism: a text-mining study</h2>
<p>date: 2024-07-10, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionExtensive research efforts have been dedicated to
elucidating the intricate pathways by which gastrointestinal microbiota
and their metabolites exert influence on the processes of bone
formation. Nonetheless, a notable gap exists in the literature
concerning a bibliometric analysis of research trends at the nexus of
gastrointestinal microbiota and bone metabolism.MethodsTo address this
scholarly void, the present study employs a suite of bibliometric tools
including online platforms, CiteSpace and VOSviewer to scrutinize the
pertinent literature in the realm of gastrointestinal microbiota and
bone metabolism.Results and discussionExamination of the temporal
distribution of publications spanning from 2000 to 2023 reveals a
discernible upward trajectory in research output, characterized by an
average annual growth rate of 19.2%. Notably, China and the United
States emerge as primary contributors. Predominant among contributing
institutions are Emory University, Harvard University, and the
University of California. Pacifici R from Emory University contributed
the most research with 15 publications. In the realm of academic
journals, Nutrients emerges as the foremost publisher, followed closely
by Frontiers in Microbiology and PLOS One. And PLOS One attains the
highest average citations of 32.48. Analysis of highly cited papers
underscores a burgeoning interest in the therapeutic potential of
probiotics or probiotic blends in modulating bone metabolism by
augmenting host immune responses. Notably, significant research
attention has coalesced around the therapeutic interventions of
probiotics, particularly Lactobacillus reuteri, in osteoporosis, as well
as the role of gastrointestinal microbiota in the etiology and
progression of osteoarthritis. Keyword analysis reveals prevalent terms
including gut microbiota, osteoporosis, bone density, probiotics,
inflammation, SCFAs, metabolism, osteoarthritis, calcium absorption,
obesity, double-blind, prebiotics, mechanisms, postmenopausal women,
supplementation, risk factors, oxidative stress, and immune system.
Future research endeavors warrant a nuanced exploration of topics such
as inflammation, obesity, SCFAs, postmenopausal osteoporosis, skeletal
muscle, oxidative stress, double-blind trials, and pathogenic
mechanisms. In summary, this study presents a comprehensive bibliometric
analysis of global research on the interplay between gastrointestinal
microbiota and bone metabolism, offering valuable insights for scholars,
particularly nascent researchers, embarking on analogous investigations
within this domain.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1407180"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1407180</a></p>
<hr />
<h2
id="an-insight-into-the-vaginal-microbiome-of-infertile-women-in-bangladesh-using-metagenomic-approach">An
insight into the vaginal microbiome of infertile women in Bangladesh
using metagenomic approach</h2>
<p>date: 2024-07-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThe dysbiosis of vaginal microbiota is recognized as a
potential underlying factor contributing to infertility in women. This
study aimed to compare the vaginal microbiomes of infertile and fertile
women to investigate their relationship with infertility.
MethodsMetagenomic analysis was conducted on samples from 5 infertile
and 5 fertile individuals using both amplicon 16S and metagenomics
shotgun sequencing methods.Results and discussionIn the infertile group,
the bacterial community was primarily represented by three major
bacterial genera: Lactobacillus (79.42%), Gardnerella (12.56%) and
Prevotella (3.33%), whereas, the fertile group exhibited a more diverse
composition with over 8 major bacterial genera, accompanied by
significantly reduced abundance of Lactobacillus (48.79%) and
Gardnerella (6.98%). At the species level, higher abundances of L.
iners, L. gasseri and G. vaginalis were observed in the infertile group.
Regarding the microbiome composition, only one fertile and two infertile
subjects exhibited the healthiest Community State Types, CST-1, while
CST-3 was observed among two infertile and one fertile subject, and
CST-4 in three other fertile and one infertile subject. Overall, alpha
diversity metrics indicated greater diversity and lower species richness
in the control (fertile) group, while the infertile group displayed the
opposite trend. However, beta-diversity analysis did not show distinct
clustering of samples associated with any specific group; instead, it
demonstrated CST-type specific clustering. Shotgun metagenomics further
confirmed the dominance of Firmicutes, with a greater abundance of
Lactobacillus species in the infertile group. Specifically, L. iners and
G. vaginalis were identified as the most dominant and highly abundant in
the infertile group. Fungi were only identified in the control group,
dominated by Penicillium citrinum (62.5%). Metagenome-assembled genomes
(MAGs) corroborated read-based taxonomic profiling, with the taxon L.
johnsonii identified exclusively in disease samples. MAG identities
shared by both groups include Shamonda orthobunyavirus, L. crispatus,
Human endogenous retrovirus K113, L. iners, and G. vaginalis.
Interestingly, the healthy microbiomes sequenced in this study contained
two clusters, Penicillium and Staphylococcus haemolyticus, not found in
the public dataset. In conclusion, this study suggests that lower
species diversity with a higher abundance of L. iners, L. gasseri and G.
vaginalis, may contribute to female infertility in our study datasets.
However, larger sample sizes are necessary to further evaluate such
association.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1390088"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1390088</a></p>
<hr />
<h2
id="bacterial-aggregation-facilitates-internalin-mediated-invasion-of-listeria-monocytogenes">Bacterial
aggregation facilitates internalin-mediated invasion of Listeria
monocytogenes</h2>
<p>date: 2024-07-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Dissemination of food-borne L. monocytogenes in the host relies on
internalin-mediated invasion, but the underlying invasion strategies
remain elusive. Here we use live-cell microscopy to follow single cell
interactions between individual human cells and L. monocytogenes and
elucidate mechanisms associated with internalin B (InlB)-mediated
invasion. We demonstrate that whilst a replicative invasion of
nonphagocytic cells is a rare event even at high multiplicities of
invasion, L. monocytogenes overcomes this by utilising a strategy
relaying on PrfA-mediated ActA-based aggregation. We show that L.
monocytogenes forms aggregates in extracellular host cell environment,
which promote approximately 5-fold more host cell adhesions than the
non-aggregating actA-ΔC mutant (which lacks the C-terminus coding
region), with the adhering bacteria inducing 3-fold more intracellular
invasions. Aggregation is associated with robust MET tyrosine kinase
receptor clustering in the host cells, a hallmark of InlB-mediated
invasion, something not observed with the actA-ΔC mutant. Finally, we
show via RNA-seq analyses that aggregation involves a global adaptive
response to host cell environment (including iron depletion), resulting
in metabolic changes in L. monocytogenes and upregulation of the PrfA
virulence regulon. Overall, our analyses provide new mechanistic
insights into internalin-mediated host-pathogen interactions of L.
monocytogenes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411124"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411124</a></p>
<hr />
<h2
id="editorial-reducing-healthcare-associated-infections-through-antimicrobial-materials">Editorial:
Reducing healthcare-associated infections through antimicrobial
materials</h2>
<p>date: 2024-07-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446870"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1446870</a></p>
<hr />
<h2
id="pathoimmunological-analyses-of-fatal-e11-infection-in-premature-infants">Pathoimmunological
analyses of fatal E11 infection in premature infants</h2>
<p>date: 2024-07-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>E11 causes acute fulminant hepatitis in newborns. We investigated the
pathological changes of different tissues from premature male twins who
died due to E11 infection. The E11 expression level was higher in the
liver than in other tissues. IP10 was upregulated in liver tissue in the
patient group, and might be regulated by IFNAR and IRF7, whereas IFNα
was regulated by IFNAR or IRF5.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391824"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1391824</a></p>
<hr />
<h2
id="assessment-of-bacillus-species-capacity-to-protect-nile-tilapia-from-a.-hydrophila-infection-and-improve-growth-performance">Assessment
of Bacillus species capacity to protect Nile tilapia from A. hydrophila
infection and improve growth performance</h2>
<p>date: 2024-07-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The present study evaluated the capacity of three Bacillus species to
improve health status and growth performance of Nile Tilapia fed with
high levels of soybean meal and challenged with Aeromonas hydrophila. In
vitro experiments showed that β-hemolysin and metalloprotease enzymes
were produced by A. hydrophila throughout the exponential growth phase.
In vivo experiments showed that 107 colony-forming units (CFUs)/ml of
this pathogen killed 50% of control group fishes in 13 days. To evaluate
the influence of Bacillus strains on health status and growth
performance in Nile Tilapia, 180 fishes (33.44 + 0.05 g) were
distributed in 12 tanks of 200 L each, and animals were fed twice per
day until satiety. 1) Control group without Bacillus, 2) Bacillus sp1,
3) Bacillus sp2, and 4) Bacillus sp3 groups were formulated containing
106 CFU/g. After 40 days of feeding, the fishes were intraperitoneally
injected with 1 ml of A. hydrophila at 2 × 107 CFU/ml, and mortality was
recorded. The results showed that cumulative mortality rate was
significantly (p&lt; 0.05) lower in the Bacillus sp1 (25%), sp2 (5%),
and sp3 (15%) groups, than the control group (50%). Weight gain was also
significantly better (p&lt; 0.05) in the Bacillus sp1 (36%), sp2 (67%),
and sp3 (55%) groups with respect to the control group (30%). In
conclusion, functional diet formulated with high levels of soybean meal
and supplemented with Bacillus sp2 could be an alternative to protect
Nile tilapia cultures from A. hydrophila infections and improve fish
growth performance.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354736"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1354736</a></p>
<hr />
<h2
id="clinical-characteristics-and-risk-factors-for-mortality-in-pneumonia-associated-acute-respiratory-distress-syndrome-patients-a-single-center-retrospective-cohort-study">Clinical
characteristics and risk factors for mortality in pneumonia-associated
acute respiratory distress syndrome patients: a single center
retrospective cohort study</h2>
<p>date: 2024-07-09, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundPathogenic diversity may have contributed to the high
mortality of pneumonia-associated acute respiratory distress syndrome
(p-ARDS). Metagenomics next-generation sequencing (mNGS) serves as a
valuable diagnostic tool for early pathogen identification. However, its
clinical utility in p-ARDS remains understudied. There are still limited
researches on the etiology, clinical characteristics and risk factors
for 28-day mortality in p-ARDS patients.MethodsA single center
retrospective cohort study of 75 p-ARDS patients was conducted. Patients
were categorized into survival and deceased groups based on their 28-day
outcomes. A comprehensive clinical evaluation was conducted, including
baseline characteristics, laboratory indicators, outcomes and pathogen
identification by mNGS and traditional microbiological testing. We then
evaluated the diagnostic value of mNGS and identified clinical
characteristics and risk factors for 28-day mortality in
p-ARDS.ResultThe overall ICU mortality was 26.67%, and the 28-day
mortality was 57.33%, with 32 cases (42.67%) in the survival group, and
43 cases (57.33%) in the deceased group. Patients in the deceased group
were older than those in the survival group (68(59,73) years
vs. 59(44,67) years, P=0.04). The average lengths of ICU and hospital
stay were 9(5,13) days and 14(7,21) days, respectively. The survival
group had longer lengths of ICU and hospital stay (ICU: 11(7,17) days
and hospital: 17(9,27) days) compared to the deceased group (ICU:
8(4,11) days and hospital: 12(6,19) days) (P&lt;0.05). Survival patients
exhibited lower Acute Physiology and Chronic Health Evaluation (APACHE)
II score on the 3rd and 7th days, higher lymphocyte counts, higher CD3+
and CD8+ T cell counts compared to deceased patients (P&lt;0.05).
Multivariate logistic regression analysis identified age, APACHE II
scores on 3rd and 7th days, CD8+ T cell count and length of ICU as
independent risk factors for 28-day mortality in p-ARDS patients. mNGS
demonstrated a significantly higher overall pathogen detection rate
(70/75, 93.33%) compared to the traditional method (50/75, 66.67%,
P=0.022). The average turnaround time (TAT) for mNGS was significantly
shorter at 1(1,1) day compared to 4(3,5) days for the traditional method
(P&lt;0.001).ConclusionMetagenome next-generation sequencing can be used
as a valuable tool for identifying pathogens in p-ARDS, reducing
diagnostic time and improving accuracy. Early application of mNGS
alongside traditional methods is recommended for p-ARDS. Furthermore,
older age, higher APACHE II scores, lower lymphocyte counts and
lymphocyte subset counts were associated with increased mortality in
p-ARDS patients, highlighting the importance of timely assessment of
immune status and disease severity, especially in elderly.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1396088"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1396088</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2024-07-08, from: People’s CDC</p>
<p>This is the <span class="citation"
data-cites="PeoplesCDC">@PeoplesCDC</span> bi-weekly update for July 8,
2024! This Weather Report from the People’s CDC sheds light on the
ongoing COVID situation in the US.</p>
<p><a
href="https://peoplescdc.org/2024/07/08/peoples-cdc-covid-19-weather-report-78/"
class="uri">https://peoplescdc.org/2024/07/08/peoples-cdc-covid-19-weather-report-78/</a></p>
<hr />
<h2
id="acupuncture-influences-multiple-diseases-by-regulating-gut-microbiota">Acupuncture
influences multiple diseases by regulating gut microbiota</h2>
<p>date: 2024-07-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Acupuncture, an important green and side effect-free therapy in
traditional Chinese medicine, is widely use both domestically and
internationally. Acupuncture can interact with the gut microbiota and
influence various diseases, including metabolic diseases,
gastrointestinal diseases, mental disorders, nervous system diseases,
and other diseases. This review presents a thorough analysis of these
interactions and their impacts and examines the alterations in the gut
microbiota and the potential clinical outcomes following acupuncture
intervention to establish a basis for the future utilization of
acupuncture in clinical treatments.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1371543"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1371543</a></p>
<hr />
<h2
id="differential-role-of-m-cells-in-enteroid-infection-by-mycobacterium-avium-subsp.-paratuberculosis-and-salmonella-enterica-serovar-typhimurium">Differential
role of M cells in enteroid infection by Mycobacterium avium subsp.
paratuberculosis and Salmonella enterica serovar Typhimurium</h2>
<p>date: 2024-07-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Infection of ruminants such as cattle with Mycobacterium avium subsp.
paratuberculosis (MAP) causes Johne’s disease, a disease characterized
by chronic inflammation of the small intestine and diarrhoea. Infection
with MAP is acquired via the faecal-to-oral route and the pathogen
initially invades the epithelial lining of the small intestine. In this
study we used an in vitro 3D mouse enteroid model to determine the
influence of M cells in infection of the gut epithelia by MAP, in
comparison with another bacterial intestinal pathogen of veterinary
importance, Salmonella enterica serovar Typhimurium. The differentiation
of M cells in the enteroid cultures was induced by stimulation with the
cytokine receptor activator of nuclear factor-κB ligand (RANKL), and the
effects on MAP and Salmonella uptake and intracellular survival were
determined. The presence of M cells in the cultures correlated with
increased uptake and intracellular survival of Salmonella, but had no
effect on MAP. Interestingly neither pathogen was observed to
preferentially accumulate within GP2-positive M cells.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416537"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1416537</a></p>
<hr />
<h2 id="editorial-reviews-in-virus-and-host">Editorial: Reviews in virus
and host</h2>
<p>date: 2024-07-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445721"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445721</a></p>
<hr />
<h2
id="a-prediction-model-for-secondary-invasive-fungal-infection-among-severe-sars-cov-2-positive-patients-in-icu">A
prediction model for secondary invasive fungal infection among severe
SARS-CoV-2 positive patients in ICU</h2>
<p>date: 2024-07-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundThe global COVID-19 pandemic has resulted in over seven
million deaths, and IFI can further complicate the clinical course of
COVID-19. Coinfection of COVID-19 and IFI (secondary IFI) pose
significant threats not only to healthcare systems but also to patient
lives. After the control measures for COVID-19 were lifted in China, we
observed a substantial number of ICU patients developing
COVID-19-associated IFI. This creates an urgent need for predictive
assessment of COVID-19 patients in the ICU environment for early
detection of suspected fungal infection cases.MethodsThis study is a
single-center, retrospective research endeavor. We conducted a
case-control study on severe acute respiratory syndrome coronavirus 2
(SARS-CoV-2) positive patients. The cases consisted of patients who
developed any secondary IFI during their ICU stay at Jilin University
China-Japan Union Hospital in Changchun, Jilin Province, China, from
December 1st, 2022, to August 31st, 2023. The control group consisted of
SARS-CoV-2 positive patients without secondary IFI. Descriptive and
comparative analyses were performed, and a logistic regression
prediction model for secondary IFI in COVID-19 patients was established.
Additionally, we observed an increased incidence of COVID-19-associated
pulmonary aspergillosis (CAPA) during this pandemic. Therefore, we
conducted a univariate subgroup analysis on top of IFI, using non-CAPA
patients as the control subgroup.ResultsFrom multivariate analysis, the
prediction model identified 6 factors that are significantly associated
with IFI, including the use of broad-spectrum antibiotics for more than
2 weeks (aOR=4.14, 95% CI 2.03-8.67), fever (aOR=2.3, 95%CI 1.16-4.55),
elevated log IL-6 levels (aOR=1.22, 95% CI 1.04-1.43) and prone position
ventilation (aOR=2.38, 95%CI 1.15-4.97) as independent risk factors for
COVID-19 secondary IFI. High BMI (BMI ≥ 28 kg/m2) (aOR=0.85, 95% CI
0.75-0.94) and the use of COVID-19 immunoglobulin (aOR=0.45, 95% CI
0.2-0.97) were identified as independent protective factors against
COVID-19 secondary IFI. The Receiver Operating Curve (ROC) area under
the curve (AUC) of this model was 0.81, indicating good
classification.ConclusionWe recommend paying special attention for the
occurrence of secondary IFI in COVID-19 patients with low BMI (BMI &lt;
28 kg/m2), elevated log IL-6 levels and fever. Additionally, during the
treatment of COVID-19 patients, we emphasize the importance of
minimizing the duration of broad-spectrum antibiotic use and highlight
the potential of immunoglobulin application in reducing the incidence of
IFI.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382720"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1382720</a></p>
<hr />
<h2
id="editorial-the-evolution-characterization-and-role-of-human-endogenous-retroviruses-in-health-and-diseases">Editorial:
The evolution, characterization, and role of human endogenous
retroviruses in health and diseases</h2>
<p>date: 2024-07-08, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1449864"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1449864</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
