<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health">health</h1>
<p>(date: 2024-12-18 15:46:50)</p>
<hr />
<h2
id="amoebae-beyond-pathogens--exploring-their-benefits-and-future-potential">Amoebae:
beyond pathogens- exploring their benefits and future potential</h2>
<p>date: 2024-12-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Amoebae, fascinatingly diverse protists, showcase a dual nature that
positions them as both friends and foes in our world. These organisms,
defined by their distinctive pseudopodia, span a spectrum from harmful
to helpful. On the darker side, species like Entamoeba histolytica pose
serious health risks, causing intestinal and liver diseases, while the
infamous “brain-eating” Naegleria fowleri leads to fatal primary amoebic
meningoencephalitis (PAM), with a daunting 97% mortality rate. Other
free-living amoebae, including Acanthamoeba castellanii and Balamuthia
mandrillaris, also threaten the human central nervous system. Yet,
beyond these dangers, amoebae play critical ecological roles. They
function as nature’s recyclers, decomposing organic material and
nourishing aquatic ecosystems, while also serving as food for various
organisms. Moreover, certain amoebae help control plant pathogens and
offer insight into human disease, proving valuable as model organisms in
biomedical research. This review sheds light on the complex,
multifaceted world of amoebae, highlighting their dual role as pathogens
and as key contributors to vital ecological processes, as well as their
significant impact on research and their promising potential for
enhancing human well-being.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1518925"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1518925</a></p>
<hr />
<h2
id="characterization-of-glycogen-related-glycoside-hydrolase-glgx-and-glgb-from-klebsiella-pneumoniae-and-their-roles-in-biofilm-formation-and-virulence">Characterization
of glycogen-related glycoside hydrolase glgX and glgB from Klebsiella
pneumoniae and their roles in biofilm formation and virulence</h2>
<p>date: 2024-12-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Glycogen is a polymer used by bacteria to store excess glucose,
playing a crucial role in bacterial growth, stress resistance, biofilm
formation, and virulence. In bacteria, the glycoside hydrolase family 13
protein are involved in the synthesis and metabolism of glycogen,
respectively. The absence of these enzymes leads to changes in bacterial
glycogen content, thereby affecting the growth metabolism of the strain.
To date, research on the roles of these glycogen-related glycoside
hydrolase genes in the synthesis metabolism and bacterial phenotypes of
Klebsiella pneumoniae has been limited. In this study, we characterized
the glycogen-related glycoside hydrolase genes glgB and glgX of K.
pneumoniae. We found that both enzymes exhibited significant degradation
activity against glycogen substrates and were capable of degrading
amylopectin, amylose, and pullulan. The optimal temperatures for GlgB
and GlgX were both in the range of 35-40°C, with optimal pH values of
7.5 and 7.0, respectively, and they exhibited high stability at 37°C.
Subsequently, we deleted the glgB and glgX genes in K. pneumoniae. The
deletion of the glgB gene resulted in a decrease in the growth rate of
the bacteria and defected glycogen synthesis. In contrast, the deletion
of the glgX gene slightly accelerated the growth rate and led to
continuous glycogen accumulation. In terms of biofilm formation and
virulence, defects in glycogen synthesis impeded biofilm formation and
virulence, while continuous glycogen accumulation did not affect biofilm
formation but slightly increased virulence. In conclusion, the glgB and
glgX genes are essential for the glycogen synthesis and metabolism in K.
pneumoniae and further influence the biofilm formation capacity and
virulence.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1507332"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1507332</a></p>
<hr />
<h2
id="prolonged-fecal-shedding-of-replication-competent-virus-lasting-immune-activation-and-intestinal-inflammation-in-a-rhesus-macaque-after-experimental-sars-cov-2-infection">Prolonged
fecal shedding of replication-competent virus, lasting immune
activation, and intestinal inflammation in a rhesus macaque after
experimental SARS-CoV-2 infection</h2>
<p>date: 2024-12-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Infection of an adult rhesus macaque with SARS-CoV-2 led to viral
RNAemia in nose, throat, and lungs. The animal also presented extended
fecal shedding of viral genomic and subgenomic messenger RNA and
replication-competent virus for more than 3 weeks after infection.
Positron emission tomography revealed increased intestinal glucose
metabolism which was histologically related to inflammation of the
ileum. These findings highlight the potential of the virus to cause
gastrointestinal infections in macaques like this is also regularly
observed in COVID-19 patients and substantiates the probability of virus
transmission via the fecal-oral route. This study further adds the
importance of nonhuman primates as a valuable animal model to study
SARS-CoV-2 infection in humans.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1505720"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1505720</a></p>
<hr />
<h2
id="preparation-characterization-and-protective-effects-of-gardenia-fructus-carbon-dots-against-oxidative-damage-induced-by-lps-in-ipec-j2-cells">Preparation,
characterization, and protective effects of Gardenia fructus carbon dots
against oxidative damage induced by LPS in IPEC-J2 cells</h2>
<p>date: 2024-12-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>This study aimed to prepare Gardenia fructus carbon dots (GF-CDs) and
examine their efficacy in mitigating oxidative stress and apoptosis in
intestinal porcine epithelial cells from the jejunum (IPEC-J2 cells)
induced by lipopolysaccharide (LPS). The GF-CDs were synthesized using a
one-step hydrothermal method. The oxidative damage model of IPEC-J2
cells was induced through LPS treatment. The potential mechanism by
which GF-CDs affect cellular oxidative damage was examined through the
perspectives of apoptosis, reactive oxygen species level,
antioxidant-related enzyme index, mRNA transcription of
antioxidant-related genes, and the expression of antioxidant proteins.
The results revealed that GF-CDs, characterized by particle sizes&lt;7
nm, abundant functional groups, and good water solubility, were
synthesized using a one-step hydrothermal method. The carbon spots of
Gardenia fructus at concentrations of 50, 100, and 200 μg/mL exhibited
protective effects, as evidenced by their ability to enhance viability
(P&lt;0.01) and restore cellular morphology after oxidative damage. The
GF-CDs decreased oxidative damage and reduced the apoptosis rate of
cells by upregulating AKT1 expression and downregulating the expression
of Caspase 3, STAT3, TNF-α, and JNK. These results indicate that GF-CDs
have the characteristic physicochemical properties of CDs, exhibit
biological activities related to antioxidation and cellular damage
mitigation, and may serve as a potential healthcare product in swine
raising.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1423760"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1423760</a></p>
<hr />
<h2
id="gut-microbiome-and-clinical-and-lifestyle-host-factors-associated-with-recurrent-positive-rt-pcr-for-sars-cov-2">Gut
microbiome and clinical and lifestyle host factors associated with
recurrent positive RT-PCR for SARS-CoV-2</h2>
<p>date: 2024-12-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundSARS-CoV-2 and COVID-19 are still active in the population.
Some patients remained PCR-positive for more than 4 weeks, called
“persistently PCR-positive”. Recent evidence suggests a link between the
gut microbiota and susceptibility to COVID-19, although no studies have
explored persistent PCR conditions. We aimed to evaluate the
relationship between persistent positive SARS-CoV-2 RT-PCR, the gut
microbiome, and individual host determinants.MethodsA shotgun
metagenomic analysis was conducted on fecal samples from 28 individuals
affected by COVID-19. Patients were divided into two groups: those who
had cleared the virus within 30 days (designated as the control group)
(n = 15), and those who remained PCR-positive beyond 30 days (called the
PCR+ group) (n = 13). We also investigated the correlation between
prolonged viral clearance and several additional factors, including
clinical parameters, immune responses, microbial metabolites, and
dietary habits.ResultsThe composition and functionality of the
microbiome varied based on the duration of positivity as determined by
PCR. Compared to the control group, the persistent PCR+ group exhibited
elevated pathogen levels and augmented diversity in functional gene
families (p-value &lt; 0.05). A multi-omics analysis integrating
metagenomics, metabolites, and metadata also revealed the specific
contribution of certain blood markers in this group, including
basophils, IgM, IgG (both general and specific for SARS-CoV-2), and
markers of liver damage. Unhealthy diet was identified as a significant
factor influencing the duration of PCR positivity.ConclusionsThese
findings indicate that the gut microbiome may play a role in delayed
viral clearance and persistent positive RT-PCR results. Our study also
contributes to the understanding of the role of host factors as
mediators linking the gut microbiota and disease outcomes. Further
large-scale studies must confirm these data; however, they suggest the
relevance of monitoring microbiome changes in the early post-viral years
to control SARS-CoV-2 and providing individual healthcare support.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494193"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494193</a></p>
<hr />
<h2
id="immunogenicity-and-safety-of-the-fourth-dose-of-quadrivalent-human-papillomavirus-hpv-vaccine-in-immunosuppressed-women-who-did-not-seroconvert-after-three-doses">Immunogenicity
and safety of the fourth dose of quadrivalent human papillomavirus (HPV)
vaccine in immunosuppressed women who did not seroconvert after three
doses</h2>
<p>date: 2024-12-18, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionImmunocompromised persons have high risk of persistent
human papillomavirus (HPV) infection and HPV-related diseases, and lower
immune response to vaccines. This study evaluated the immunogenicity and
safety of administering a fourth dose of quadrivalent (4v)HPV vaccine in
immunosuppressed women who did not seroconvert after three
doses.MethodsAn open-label, not-controlled trial included
immunosuppressed women (solid organ transplant patients and women
receiving treatment for SLE) who did not seroconvert to at least one of
the four HPV vaccine types after three 4vHPV vaccine doses. All
participants received a fourth 4vHPV vaccine dose (median 27 months
after third dose). Immunogenicity was evaluated a month after the fourth
dose, by measuring seroconversion rates and antibody geometric mean
concentration (GMC).ResultsTwenty-three women were included. Among women
who did not seroconvert for each vaccine type after three doses, 2/10
seroconverted to HPV6, 3/10 to HPV11, 3/10 to HPV16 and 6/18 to HPV18,
after the fourth 4vHPV dose. There was an increase in antibody GMC for
HPV 6, 16, 18, with highest increase for HPV16 (from 6.02 to 44.63
International Units). There was no increase of anti-HPV-11. Within seven
days after vaccination, only three of the 23 vaccinees reported any
adverse event, none of which were classified as
serious.ConclusionsAlthough safe, the fourth 4vHPV vaccine dose led to
seroconversion in only few immunosuppressed women who had not
seroconverted after three doses.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451308"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451308</a></p>
<hr />
<h2
id="localized-provoked-vulvodynia-as-an-immune-mediated-inflammatory-disease-rationale-for-a-new-line-of-research">Localized
provoked vulvodynia as an immune-mediated inflammatory disease:
rationale for a new line of research</h2>
<p>date: 2024-12-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Localized provoked vulvodynia (LPV), also called vulvar vestibulitis
or provoked vestibulodynia, is a major cause of dyspareunia that
severely impacts sexual health. At the tissue level, lymphocytic
inflammation and hyperinnervation are characteristic pathological
features, explaining the main symptoms and signs. A recent experimental
animal study suggests that the histopathological findings of LPV may be
due to mucosal CD4 Th17 immune responses to microbial antigens. We
hypothesize that LPV is an immune-mediated inflammatory disease and
challenge the concept of LPV as a chronic pain syndrome of unknown
cause. Since most treatment modalities currently used in LPV are no
better than placebo, we therefore warrant future research investigating
the possible presence of CD4 Th17 cells and IL17 cytokine in affected
tissues together with treatment trials that include inhibitors of the
IL17 pathway.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1505845"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1505845</a></p>
<hr />
<h2
id="helicobacter-pylori-resistance-in-hainan-province-china-investigating-phenotypes-and-genotypes-through-whole-genome-sequencing">Helicobacter
pylori resistance in Hainan Province, China: investigating phenotypes
and genotypes through whole-genome sequencing</h2>
<p>date: 2024-12-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Helicobacter pylori is increasingly resistant to antibiotics,
significantly lowering eradication rates and posing a major public
health challenge. This study investigated the distribution of
antibiotic-resistant phenotypes and genotypes of H. pylori in Hainan
Province. It determined the minimum inhibitory concentrations (MICs) of
six antibiotics using the E-test method and detected resistance genes
via Sanger sequencing. Furthermore, we compared resistance detection
based on phenotypic analysis and whole genome sequencing (WGS) across 19
clinical isolates of H. pylori. A total of 140 H. pylori strains were
isolated. The resistance rates to levofloxacin (LEV), clarithromycin
(CLA), and metronidazole (MTZ) were 37.9%, 40.0%, and 93.6%,
respectively. Notably, only 3.3% of the strains were susceptible to all
six antibiotics. Multidrug-resistant strains accounted for 25.0% of the
total, with no resistance detected to amoxicillin (AMX), tetracycline
(TET), or furazolidone (FR) during the study period. Genotypic
resistance to CLA and LEV showed near-perfect concordance with
phenotypic resistance, with Kappa values of 0.910 and 0.938,
respectively. Although all isolates were phenotypically sensitive to
TET, 16 exhibited a mutation in the 16S rRNA gene (A926G). All strains
harboring the R16H/C mutation and truncated rdxA were resistant to
metronidazole, demonstrating a specificity of 100%. Therefore, FR, AMX,
and TET are recommended as suitable empirical treatment options for H.
pylori infections in this region. Genotypic analysis provides a reliable
method for predicting resistance to CLA and LEV. WGS proves to be a
valuable tool for identifying novel resistance loci in H. pylori and
contributes to the phylogenetic classification of strains.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1505166"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1505166</a></p>
<hr />
<h2
id="metabarcoding-analysis-of-oral-microbiome-during-pregnancy">Metabarcoding
analysis of oral microbiome during pregnancy</h2>
<p>date: 2024-12-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Pregnancy is a dynamic physiological process involving significant
hormonal, immune, and metabolic changes to support fetal growth and
development. This study investigates the changes in salivary microbiome
and biochemical markers from the second to the third trimester of
pregnancy. Saliva samples were collected from 45 pregnant women enrolled
in the Qatar Birth Cohort study at two time points (second and third
trimesters). DNA was extracted and subjected to 16S rRNA gene sequencing
using Oxford Nanopore Technology. Microbial diversity and taxonomic
analyses were performed, along with correlation analyses between
microbial abundance and clinical parameters. Biochemically, significant
increases in BMI, pulse rate, HbA1c, LDL, total cholesterol, and
triglycerides were observed in the third trimester compared to the
second. Microbial diversity analysis revealed significant changes in
microbial richness and composition. Taxonomy analysis showed a
significant 3-fold increase in Bacteroidota. Also, a significant decline
in Selenomonas and a significant increase in Veillonella, specifically
Veillonella dispar and Veillonella atypica, as well as an increase in
Granulicatella were observed in the third trimester, along with a
significant decrease in Streptococcus sanguinis. Correlation analysis
during the second trimester revealed positive associations between BMI,
cholesterol, LDL, and Selenomonas, and negative correlations with
Streptococcus and Gemella. In the third trimester, BMI was negatively
correlated with Campylobacter, glucose levels were negatively correlated
with Neisseria, and triglyceride levels were negatively correlated with
Prevotella. These findings highlight significant biochemical and
microbial shifts during pregnancy, underscoring the importance of
monitoring oral health and metabolic changes in pregnant women.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477703"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1477703</a></p>
<hr />
<h2
id="polymorphonuclear-myeloid-derived-suppressor-cells-regulates-immune-recovery-during-hiv-infection-through-pd-l1-and-tgf-β-pathways">Polymorphonuclear
myeloid-derived suppressor cells regulates immune recovery during HIV
infection through PD-L1 and TGF-β pathways</h2>
<p>date: 2024-12-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAlthough MDSCs are widely recognized for their
immunoinhibitory effects in pathological conditions, their function
during HIV infection particularly within the mechanisms underlying
incomplete immune recovery remains elusive.MethodsWe conducted a
cross-sectional study in which 30 healthy controls and 62 HIV-1-infected
subjects [31 immunological non-responders (INRs) and 31 immunological
responders (IRs)] were selected. The proportion of MDSCs was determined
in each category of participants. Using flow cytometry and real-time
PCR, immune regulatory molecules (including PD-L1, ARG1, iNOS, IL-10,
TGF-β, and IDO) that are relevant for MDSCs activity were quantified.
Furthermore, we investigated the impact of the blockade of PD-L1 and
TGF-β pathways on MDSCs and their effects on CD4+ T-cells using in vitro
functional experiments.ResultsPMN-MDSCs are more abundant and are
negatively correlated to CD4 counts in HIV-infected individuals. In
addition, PMN-MDSCs suppress CD4+ T-cell proliferation and IFN-γ
production in INRs. Furthermore, correlations were found between PD-L1
expression on PMN-MDSCs and PD-1+ CD4+ T-cells. TGF-β expression on
PMN-MDSCs was likewise enhanced in INRs. Importantly, inhibiting both
PD-L1 and TGF-β pathways had a synergistic impact on restoring CD4+
T-cell activity in vitro.ConclusionsPMN-MDSCs expansion inhibits CD4+
T-cell responses. We suggest that targeting PD-L1 and TGF-β pathways
together may significantly improve immune recovery in INRs.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1516421"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1516421</a></p>
<hr />
<h2
id="the-clinical-indexes-and-immunological-status-of-hivaids-patients-undergoing-different-highly-active-antiretroviral-treatments">The
clinical indexes and immunological status of HIV/AIDS patients
undergoing different highly active antiretroviral treatments</h2>
<p>date: 2024-12-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveThis study aims to investigate the differences of clinical
indices in HIV patients between three different first-line
antiretroviral treatment strategies in Yunnan Province, China.
Furthermore, the hematologic system, liver function, kidney function,
blood lipid levels of HIV patients and its association with CD4+ count,
CD8+ count, CD4/CD8 ratio and antiretroviral treatment were also
assessed.MethodsThis retrospective cohort study included 81 participants
who underwent highly active antiretroviral treatment from September 2009
to September 2019. Baseline sociodemographic and clinical
characteristics were collected from each study participant. Routine
blood tests, liver and renal function, lipid levels as well as
lymphocyte subset counts were measured and recorded for evaluation
before and 3, 6, 9, and 12 months after the treatment. Paired t-test was
used to compare clinical indices changes after antiretroviral treatment.
Univariate linear regression was performed to determine the association
between clinical indices and CD4+ count, CD8+ count, CD4/CD8 ratio and
antiretroviral treatment.ResultThere were no statistical differences in
baseline demographic and clinical characteristics in either treatment
group. Compared with the initiation of HARRT treatment, the CD4+ count(p
&lt; 0.001), CD4/CD8 ratio(p &lt; 0.001) and PLT(p &lt; 0.001) were
increased in the three treatment groups. The TC(p &lt; 0.01) and TG(p
&lt; 0.05) were increased in 3TC+AZT+EFV group after treatment. The
ALT(p &lt; 0.05), AST(p &lt; 0.01) were decreased in 3TC+EFV+TDF group
after treatment. The study indicated statistical differences in CD4+
count (p &lt; 0.001), CD8+ count (p &lt; 0.001), and CD4/CD8 ratio (p
&lt; 0.001) in the three treatment cohorts. Furthermore, a strong
positive correlation was observed between WBC (p &lt; 0.001), platelet
(p &lt; 0.001), Hb (p &lt; 0.001), and CD4+ count in the three treatment
cohorts. Moreover, ALT and AST were negatively associated with CD4+
count in the 3TC + AZT + EFV group. Whereas WBC were positively
correlated with CD8+ count in the three treatment methods. In addition,
platelet and TG were positively correlated with CD8+ count in the 3TC +
EFV + TDF. The study also indicated that TC was positively associated
with CD8+ count in the 3TC + AZT + NVP group. Furthermore, WBC was
negatively related to CD4/CD8 ratio in the 3TC + EFV + TDF group. The
platelet level analysis revealed a positive, while TG indicated a
negative association with CD4/CD8 ratio in the 3TC + AZT + NVP group.
Moreover, ALT and AST were negatively correlated with the CD4/CD8 ratio
in the 3TC + AZT + EFV and 3TC + AZT + NVP groups.ConclusionThe results
showed that HIV/AIDS patients treated with different first-line
antiretroviral treatment strategies had different hematopoietic, liver,
renal and immune system functions. Furthermore, some clinical indicators
such as WBC, PLT, TC, TG, and ALT could predict the CD4+ count, CD8+
count, CD4/CD8 ratio levels and recuperation of HIV/AIDS patients,
therefore, should be monitored by clinicians.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1436123"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1436123</a></p>
<hr />
<h2
id="deciphering-pathogenicity-and-virulence-of-the-first-staphylococcus-debuckii-isolate-from-diabetic-foot-osteomyelitis">Deciphering
pathogenicity and virulence of the first Staphylococcus debuckii isolate
from diabetic foot osteomyelitis</h2>
<p>date: 2024-12-17, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionThis study identifies Staphylococcus debuckii as a new
coagulase-negative staphylococcal species isolated from diabetic foot
osteomyelitis (DFOM) and provides an in-depth analysis of its pathogenic
and virulence profile, as well as demonstrating its potential to cause
infection.MethodsThe S. debuckii NSD001 strain was examined for its
planktonic growth, biofilm production, and phagocytosis rates in murine
macrophages compared to S. aureus NSA739. Additionally, persistence and
replication within human osteoblasts were investigated, while the
zebrafish embryo model was employed to assess virulence. Genomic
sequencing and bioinformatic analysis were also conducted to identify
genes associated with virulent potential.Results and DiscussionS.
debuckii NSD001 exhibited robust planktonic growth and significant
biofilm production, highlighting its capacity to initiate and maintain
an infection, and demonstrated similar rates of phagocytosis as S.
aureus NSA739 in murine macrophages, suggesting a mechanism for evading
initial host defenses. The strain persisted and replicated within human
osteoblasts, indicative of a strategy for intracellular survival and
facilitation of chronic osteomyelitis. The zebrafish embryo model
revealed a slower, yet fatal, virulence profile for S. debuckii NSD001
compared to the rapid lethality induced by S. aureus NSA739. Genomic
sequencing and bioinformatic analysis uncovered various genes
corroborating its virulence. S. debuckii NSD001 poses a significant
concern in DFOM due to its ability to form biofilms and survive within
host cells, presenting challenges for current treatment strategies. This
underscores the need for updated clinical protocols and increased
awareness among healthcare professionals to effectively manage
infections caused by this emerging pathogen.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1489280"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1489280</a></p>
<hr />
<h2 id="peoples-cdc-covid-19-weather-report">People’s CDC COVID-19
Weather Report</h2>
<p>date: 2024-12-16, from: People’s CDC</p>
<p>This is the People’s CDC bi-weekly update for December 16, 2024! This
Weather Report from the People’s CDC sheds light on the ongoing COVID
situation in the US.</p>
<p><br></p>
<p><a
href="https://peoplescdc.org/2024/12/16/peoples-cdc-covid-19-weather-report-88/"
class="uri">https://peoplescdc.org/2024/12/16/peoples-cdc-covid-19-weather-report-88/</a></p>
<hr />
<h2
id="does-smoking-cause-kidney-stones-from-a-kidney-stone-dietitian">Does
Smoking Cause Kidney Stones? [from a kidney stone dietitian]</h2>
<p>date: 2024-12-16, from: The Kidney Dietitian</p>
<p>
Smoking increases the of many chronic diseases such as heart disease,
diabetes, stroke and cancer. It is reasonable to question if smoking
causes kidney stones too! There is some research to show that smoking
might cause kidney stones. Let’s dive into it! What is a Kidney Stone? A
kidney stone is a crystallization of molecules […]
</p>
<p>
The post
<a href="https://www.thekidneydietitian.org/does-smoking-cause-kidney-stones/">Does
Smoking Cause Kidney Stones? [from a kidney stone dietitian]</a>
appeared first on <a href="https://www.thekidneydietitian.org">The
Kidney Dietitian</a>.
</p>
<p><br></p>
<p><a
href="https://www.thekidneydietitian.org/does-smoking-cause-kidney-stones/"
class="uri">https://www.thekidneydietitian.org/does-smoking-cause-kidney-stones/</a></p>
<hr />
<h2
id="new-tools-to-monitor-pseudomonas-aeruginosa-infection-and-biofilms-in-vivo-in-c.-elegans">New
tools to monitor Pseudomonas aeruginosa infection and biofilms in vivo
in C. elegans</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionAntimicrobial resistance is a growing health problem.
Pseudomonas aeruginosa is a pathogen of major concern because of its
multidrug resistance and global threat, especially in health-care
settings. The pathogenesis and drug resistance of P. aeruginosa depends
on its ability to form biofilms, making infections chronic and
untreatable as the biofilm protects against antibiotics and host
immunity. A major barrier to developing new antimicrobials is the lack
of in vivo biofilm models. Standard microbiological testing is usually
performed in vitro using planktonic bacteria, without representation of
biofilms, reducing translatability. Here we develop tools to study both
infection and biofilm formation by P. aeruginosa in vivo to accelerate
development of strategies targeting infection and pathogenic
biofilms.MethodsBiofilms were quantified in vitro using Crystal Violet
staining and fluorescence biofilm assays. For in vivo assays, C. elegans
were infected with P. aeruginosa strains. Pathogenicity was quantified
by measuring healthspan, survival and GFP fluorescence. Healthspan
assays were performed using the WormGazerTM automated imaging
technology.ResultsUsing the nematode Caenorhabditis elegans and P.
aeruginosa reporters combined with in vivo imaging we show that
fluorescent P. aeruginosa reporters that form biofilms in vitro can be
used to visualize tissue infection. Using automated tracking of C.
elegans movement, we find that that the timing of this infection
corresponds with a decline in health endpoints. In a mutant strain of P.
aeruginosa lacking RhlR, a transcription factor that controls quorum
sensing and biofilm formation, we find reduced capacity of P. aeruginosa
to form biofilms, invade host tissues and negatively impact healthspan
and survival.DiscussionOur findings suggest that RhlR could be a new
antimicrobial target to reduce P. aeruginosa biofilms and virulence in
vivo and C. elegans could be used to more effectively screen for new
drugs to combat antimicrobial resistance.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478881"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1478881</a></p>
<hr />
<h2
id="diagnostic-and-prognostic-values-of-nsclc-patients-with-or-without-obstructive-pneumonia-after-sleeve-lobectomy">Diagnostic
and prognostic values of NSCLC patients with or without obstructive
pneumonia after sleeve lobectomy</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveWe aimed to identify the diagnostic value of next-generation
sequencing (NGS) of bronchoalveolar lavage fluid (BALF) from patients
with non-small-cell lung cancer (NSCLC).MethodsForty patients who were
initially diagnosed with pulmonary nodules were enrolled. Frozen section
histology was used to identify the NSCLC cell types. NGS of collected
BALF samples was used for microbial identification. We compared the
bacterial and viral distributions in BALF samples from patients with
NSCLC with and without obstructive pneumonia as well as their NSCLC
drainage times following surgery.ResultsOf the 29 patients with NSCLC,
eight had obstructive pneumonia. Streptococcus pneumoniae, Streptococcus
pseudopneumoniae, and Haemophilus parainfluenzae were the top three
bacteria present in almost 50% of patients, both with and without
obstructive pneumonia. The viral detection rate was higher in the BALF
of patients with NSCLC who did not have obstructive pneumonia. However,
in patients with NSCLC and drain times of &gt;5 days, the human herpes
virus type 7 detection rate was higher following surgery than it was in
patients with NSCLC who had drain times of ≤5 days.ConclusionViral
imbalance in NSCLC is closely related to the occurrence of obstructive
pneumonia and postoperative drainage time.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474998"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474998</a></p>
<hr />
<h2
id="two-genotypes-of-h3n2-swine-influenza-viruses-identified-in-pigs-from-shandong-province-china">Two
genotypes of H3N2 swine influenza viruses identified in pigs from
Shandong Province, China</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Swine influenza virus (SIV) is a highly contagious pathogen that
poses significant economic challenges to the swine industry and carries
zoonotic potential, underscoring the need for vigilant surveillance. In
this study, we performed a comprehensive genetic and molecular analysis
of H3N2 SIV isolates obtained from 372 swine samples collected in
Shandong Province, China. Phylogenetic analysis revealed two distinct
genotypes. The surface genes of both genotypes clustered with the
human-like H3N2 lineage, while the internal genes of one genotype
clustered with the 2009 pandemic H1N1 (pdm/09) lineage. In the second
genotype, the NS gene clustered with the classical swine (CS) H1N1
lineage, while the remaining internal genes clustered with pdm/09,
suggesting stable integration of pdm/09 gene segments into H3N2 SIV.
Homology analysis showed over 96% genetic similarity between the
isolates and reference strains from China and Brazil, suggesting
potential transmission through swine trade or human movement. Molecular
characterization identified amino acid substitutions in the HA protein
(190D, 226I, and 228S), potentially enhancing the virus’s affinity for
human-like receptors, thereby increasing the zoonotic risk. Key
mutations in the PB2 (271A, 591R), PA (336M, 356R, 409N), and M2 (S31N)
proteins, along with novel drug resistance mutations, indicate the
potential for enhanced virulence and drug resistance in these isolates.
Moreover, glycosylation site analysis revealed four differences, and
antigenic site analysis showed 13 differences between the HA proteins of
the isolates and the WHO-recommended vaccine strain
A/Cambodia/E0826360/2020 for the 2021–2022 season, which may reduce
vaccine efficacy. Serological analysis revealed that 11 out of the
tested serum samples were positive for H3N2 antibodies, resulting in an
overall positivity rate of 0.42%. These findings emphasize the urgent
need for strengthened SIV surveillance in China to monitor the risk of
human transmission and ensure better preparedness for future influenza
outbreaks.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1517023"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1517023</a></p>
<hr />
<h2
id="identification-and-characterization-of-a-relish-type-nf-κb-dvrelish-in-dermacentor-variabilis-in-response-to-rickettsia-rickettsii-infection">Identification
and characterization of a Relish-type NF-κB, DvRelish, in Dermacentor
variabilis in response to Rickettsia rickettsii infection</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>Ixodid ticks serve as hosts and transmission vectors for several
obligate intracellular bacteria, including members of the spotted fever
group (SFG) of Rickettsia. Although ticks generate an immune response to
bacterial insults, many of the signaling molecules associated with the
response and how they may contribute to vector competence for Rickettsia
are undefined. In this study, we isolated a full-length dvrelish
transcript from Dermacentor variabilis, which encoded a Relish-type
NF-κB. The presence of a canonical Rel homology domain (RHD) consistent
with NF-κB proteins suggested a role in tick immune response for
DvRelish. The expression of DvRelish was confirmed in tick tissues and
fluorescent microscopy of tick hemocytes indicated increased expression
following infection with Rickettsia as compared to a non-tick-borne
bacterial pathogen. To further determine the effect of dvRelish gene
knockdown on rickettsial infection, we used RNA interference-mediated
gene knockdown in D. variabilis and demonstrated that transcription of
dvRelish was decreased after 24 h post-injection of siRNA. We then
assessed the response of D. variabilis when exposed to Rickettsia
rickettsii and determined that transcription of dvRelish was inversely
associated with rickettsial loads at 48 h post-exposure. Further studies
are required to broaden the understanding of differential immune
responses in ticks to SFG Rickettsia infection and elucidate the role
played by the arthropod immune system in vector competence.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494450"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1494450</a></p>
<hr />
<h2
id="transcriptomic-profiling-of-brain-eating-amoeba-naegleria-fowleri-infection-in-mice-the-host-and-the-protozoa-perspectives">Transcriptomic
profiling of “brain-eating amoeba” Naegleria fowleri infection in mice:
the host and the protozoa perspectives</h2>
<p>date: 2024-12-16, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>The free-living amoeba Naegleria fowleri (NF) causes a rare but
lethal parasitic meningoencephalitis (PAM) in humans. Currently, this
disease lacks effective treatments and the specific molecular mechanisms
that govern NF pathogenesis and host brain response remain unknown. To
address some of these issues, we sought to explore naturally existing
virulence diversity within environmental NF isolates. Herein, we
purified two new NF environmental isolates (NF45 and NF1) and tested
their in vivo virulence using experimental infection in mice. We found
that NF45 was highly virulent (NF45_HV) compared with NF1 (low
virulence, NF1_LV), based on in vivo amoeba growth kinetics and mouse
survival. To identify underlying differences, we conducted RNA-seq and
bioinformatics analyses from the infected mouse brains. Our results
showed that NF1_LV and NF45_HV modulated the expression of their genes
during mouse brain infection. Differentially expressed genes (DEGs) in
NF1_LV were mostly involved in Translational protein, Protein-binding
activity modulator, Protein modifying enzyme, while DEGs in NF45_HV were
related to DNA metabolism, Cytoskeletal protein, Protein-binding
activity modulator. Proteases (namely the virulence factor Cathepsin B)
were upregulated in NF1_LV, while downregulated in NF45_HV. When
analyzing the host response against infection by these two NF strains,
enrichment analyses uncovered genes and mechanisms related to the host
immune responses and nervous systems. We detected more DEGs in NF1_LV
infected mice compared to NF45_HV, related to blood brain barrier
leakage, immune cell recruitment, cytokine production (including IL-6,
IFN-Ɣ and TNFα), inflammation of astrocytes and microglia, and
oligodendrocyte and neurons degeneration. Increased expression of
neuromotor-related genes such as Adam22, Cacnb4 and Zic1 (activated by
NF1_LV infection) and ChAt (activated by NF45_LV infection) could
explain PAM symptoms such as muscle weakness and seizures. Globally, our
results showed that NF isolated from the environment can have different
levels of virulence and differentially modulate their gene expression
during brain infection. We also provided, for the first time, a
comprehensive information for the molecular mechanisms of neuro-immune
and host–pathogen interactions during PAM disease. As the host and the
protozoa are strongly implicated in PAM lethality, new therapies
targeting both the parasite, and the host should be considered to treat
PAM infection.</p>
<p><br></p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1490280"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1490280</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a>.
</footer>
</body>
</html>
