<!doctype html>
<html lang="en-US">
<head>
  <meta charset="utf-8" >
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" >
  <title>health 2024.35</title>
<!--  <link rel="stylesheet" type="text/css"  href="../webfonts/fonts.css" media="screen" > -->
  <link rel="stylesheet" type="text/css"  href="../css/site.css" media="screen" >
</head>
<body>
<header>
	<img class="logo" 
	src="https://upload.wikimedia.org/wikipedia/commons/9/9c/Antenna_1_-_The_Noun_Project.svg"
	alt="line art showing an antenna"
	height="80" width="60" >
	<h1>The Antenna</h1> 
	<h2>finding signal in the noise</h2>
</header>
<nav>
<ul>
	<li><a href="../../">The Antenna</a></li>
	<li><a href="../../../socal_north.html" title="news from the Northern side of Southern California" >SoCal North!</a></li>
	<li><a href="../../../pacific.html" title="News from Micronesia and Pacific">Pacific</a></li>
	<li><a href="../../../ham_radio.html" title="news from  Ham Radio">Ham Radio</a></li>
	<li><a href="../../../mid_central.html" title="news from Mid Central California">Mid Central</a></li>
	<li><a href="../../columns.html" title="blogs I read like columnists">Columns</a></li>
	<li><a href="../../weather.html" title="U.S. National Weather">Weather</a></li>
	<li><a href="../../snapshots/" title="Somedays daily">Snapshots</a></li>
	<li><a href="../">Archives</a></li>
	<li><a href="../../search.html">Search</a></li>
	<li><a href="../../about.html">About</a></li>
</ul>
</nav>
<section>
<div class="description-for-items">
<h2>health 2024.35</h2>
An experiment in personal news aggregation.
</div>
<h1 id="health-2024.35">health 2024.35</h1>
<p>(date: 2024-08-28 08:29:12)</p>
<hr />
<h2
id="editorial-role-of-microbes-in-ocular-surface-health-and-diseases">Editorial:
Role of microbes in ocular surface health and diseases</h2>
<p>date: 2024-08-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1462752"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1462752</a></p>
<hr />
<h2
id="editorial-designing-metal-complexes-metallo-drugs-against-infectious-diseases-and-understanding-their-metal-toxicity">Editorial:
Designing metal-complexes (metallo-drugs) against infectious diseases
and understanding their metal toxicity</h2>
<p>date: 2024-08-28, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1463451"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1463451</a></p>
<hr />
<h2 id="is-orange-juice-good-for-kidney-stones">Is Orange Juice Good For
Kidney Stones?</h2>
<p>date: 2024-08-27, from: The Kidney Dietitian</p>
<p>
Is orange juice actually good for kidney stones? Or, is this an old
wives tale? Orange juice can impact our health in more ways than one,
especially if we have kidney stones.  In this blog post, we’ll explore
the nutritional profile of orange juice, discuss the benefits and
potential concerns related to kidney stones, investigate […]
</p>
<p>
The post
<a href="https://www.thekidneydietitian.org/is-orange-juice-good-for-kidney-stones/">Is
Orange Juice Good For Kidney Stones?</a> appeared first on
<a href="https://www.thekidneydietitian.org">The Kidney Dietitian</a>.
</p>
<p><a
href="https://www.thekidneydietitian.org/is-orange-juice-good-for-kidney-stones/"
class="uri">https://www.thekidneydietitian.org/is-orange-juice-good-for-kidney-stones/</a></p>
<hr />
<h2
id="unveiling-microbial-dynamics-in-lung-adenocarcinoma-and-adjacent-nontumor-tissues-insights-from-nicotine-exposure-and-diverse-clinical-stages-via-nanopore-sequencing-technology">Unveiling
microbial dynamics in lung adenocarcinoma and adjacent nontumor tissues:
insights from nicotine exposure and diverse clinical stages via nanopore
sequencing technology</h2>
<p>date: 2024-08-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundLung is the largest mucosal area of the human body and
directly connected to the external environment, facing microbial
exposure and environmental stimuli. Therefore, studying the internal
microorganisms of the lung is crucial for a deeper understanding of the
relationship between microorganisms and the occurrence and progression
of lung cancer.MethodsTumor and adjacent nontumor tissues were collected
from 38 lung adenocarcinoma patients and used nanopore sequencing
technology to sequence the 16s full-length sequence of bacteria, and
combining bioinformatics methods to identify and quantitatively analyze
microorganisms in tissues, as well as to enrich the metabolic pathways
of microorganisms.Resultsthe microbial composition in lung
adenocarcinoma tissues is highly similar to that in adjacent tissues,
but the alpha diversity is significantly lower than that in adjacent
tissues. The difference analysis results show that the bacterial
communities of Streptococcaceae, Lactobacillaceae, and Neisseriales were
significantly enriched in cancer tissues. The results of metabolic
pathway analysis indicate that pathways related to cellular
communication, transcription, and protein synthesis were significantly
enriched in cancer tissue. In addition, clinical staging analysis of
nicotine exposure and lung cancer found that Haemophilus,
paralinfluenzae, Streptococcus gordonii were significantly enriched in
the nicotine exposure group, while the microbiota of Cardiobactereae and
Cardiobacterales were significantly enriched in stage II tumors. The
microbiota significantly enriched in IA-II stages were Neisseriaeae,
Enterobacteriales, and Cardiobacterales, respectively.ConclusionNanopore
sequencing technology was performed on the full length 16s sequence,
which preliminarily depicted the microbial changes and enrichment of
microbial metabolic pathways in tumor and adjacent nontumor tissues. The
relationship between nicotine exposure, tumor progression, and
microorganisms was explored, providing a theoretical basis for the
treatment of lung cancer through microbial targets.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397989"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1397989</a></p>
<hr />
<h2
id="immunogenicity-and-transmission-blocking-potential-of-quiescin-sulfhydryl-oxidase-in-plasmodium-vivax">Immunogenicity
and transmission-blocking potential of quiescin sulfhydryl oxidase in
Plasmodium vivax</h2>
<p>date: 2024-08-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundTransmission-blocking vaccines (TBVs) can effectively
prevent the community’s spread of malaria by targeting the antigens of
mosquito sexual stage parasites. At present, only a few candidate
antigens have demonstrated transmission-blocking activity (TBA)
potential in P. vivax. Quiescin-sulfhydryl oxidase (QSOX) is a sexual
stage protein in the rodent malaria parasite Plasmodium berghei and is
associated with a critical role in protein folding by introducing
disulfides into unfolded reduced proteins. Here, we reported the
immunogenicity and transmission-blocking potency of the PvQSOX in P.
vivax.Methods and findingsThe full-length recombinant PvQSOX protein
(rPvQSOX) was expressed in the Escherichia coli expression system. The
anti-rPvQSOX antibodies were generated following immunization with the
rPvQSOX in rabbits. A parasite integration of the pvqsox gene into the
P. berghei pbqsox gene knockout genome was developed to express
full-length PvQSOX protein in P. berghei (Pv-Tr-PbQSOX). In western
blot, the anti-rPvQSOX antibodies recognized the native PvQSOX protein
expressed in transgenic P. berghei gametocyte and ookinete. In indirect
immunofluorescence assays, the fluorescence signal was detected in the
sexual stages, including gametocyte, gamete, zygote, and ookinete.
Anti-rPvQSOX IgGs obviously inhibited the ookinetes and oocysts
development both in vivo and in vitro using transgenic parasites. Direct
membrane feeding assays of anti-rPvQSOX antibodies were conducted using
four field P. vivax isolates (named isolates #1–4) in Thailand. Oocyst
density in mosquitoes was significantly reduced by 32.00, 85.96, 43.52,
and 66.03% with rabbit anti-rPvQSOX antibodies, respectively. The
anti-rPvQSOX antibodies also showed a modest reduction of infection
prevalence by 15, 15, 20, and 22.22%, respectively, as compared to the
control, while the effect was insignificant. The variation in the DMFA
results may be unrelated to the genetic polymorphisms. Compared to the
P.vivax Salvador (Sal) I strain sequences, the pvqsox in isolate #1
showed no amino acid substitution, whereas isolates #2, #3, and #4 all
had the M361I substitution.ConclusionsOur results suggest that PvQSOX
could serve as a potential P. vivax TBVs candidate, which warrants
further evaluation and optimization.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451063"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451063</a></p>
<hr />
<h2
id="comparative-analysis-of-metagenomic-and-targeted-next-generation-sequencing-for-pathogens-diagnosis-in-bronchoalveolar-lavage-fluid-specimens">Comparative
analysis of metagenomic and targeted next-generation sequencing for
pathogens diagnosis in bronchoalveolar lavage fluid specimens</h2>
<p>date: 2024-08-27, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>BackgroundAlthough the emerging NGS-based assays, metagenomic
next-generation sequencing (mNGS) and targeted next-generation
sequencing (tNGS), have been extensively utilized for the identification
of pathogens in pulmonary infections, there have been limited studies
systematically evaluating differences in the efficacy of mNGS and
multiplex PCR-based tNGS in bronchoalveolar lavage fluid (BALF)
specimens.MethodsIn this study, 85 suspected infectious BALF specimens
were collected. Parallel mNGS and tNGS workflows to each sample were
performed; then, we comparatively compared their consistency in
detecting pathogens. The differential results for clinically key
pathogens were confirmed using PCR.ResultsThe microbial detection rates
of BALF specimens by the mNGS and tNGS workflows were 95.18% (79/83) and
92.77% (77/83), respectively, with no significant difference. mNGS
identified 55 different microorganisms, whereas tNGS detected 49
pathogens. The comparative analysis of mNGS and tNGS revealed that
86.75% (72/83) of the specimens were complete or partial concordance.
Particularly, mNGS and tNGS differed significantly in detection rates
for some of the human herpesviruses only, including Human
gammaherpesvirus 4 (P&lt;0.001), Human betaherpesvirus 7 (P&lt;0.001),
Human betaherpesvirus 5 (P&lt;0.05) and Human betaherpesvirus 6
(P&lt;0.01), in which tNGS always had higher detection rates. Orthogonal
testing of clinically critical pathogens showed a total coincidence rate
of 50% for mNGS and PCR, as well as for tNGS and PCR.ConclusionsOverall,
the performance of mNGS and multiplex PCR-based tNGS assays was similar
for bacteria and fungi, and tNGS may be superior to mNGS for the
detection of DNA viruses. No significant differences were seen between
the two NGS assays compared to PCR.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451440"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1451440</a></p>
<hr />
<h2
id="editorial-exploring-genetic-characteristics-and-molecular-mechanisms-of-host-adaptation-of-viruses-with-artificial-intelligence-ai-or-and-biological-bio-approaches">Editorial:
Exploring genetic characteristics and molecular mechanisms of host
adaptation of viruses with artificial intelligence (AI) or (and)
biological (BIO) approaches</h2>
<p>date: 2024-08-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474097"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1474097</a></p>
<hr />
<h2
id="proven-anti-virulence-therapies-in-combating-methicillin--and-vancomycin-resistant-staphylococcus-aureus-infections">Proven
anti-virulence therapies in combating methicillin- and
vancomycin-resistant Staphylococcus aureus infections</h2>
<p>date: 2024-08-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>IntroductionDespite years of efforts to develop new antibiotics for
eradicating multidrug-resistant (MDR) and multi-virulent
Methicillin-Resistant Staphylococcus aureus (MRSA) and
Vancomycin-Resistant Staphylococcus aureus (VRSA) infections, treatment
failures and poor prognoses in most cases have been common. Therefore,
there is an urgent need for new therapeutic approaches targeting
virulence arrays. Our aim is to discover new anti-virulence therapies
targeting MRSA and VRSA virulence arrays.MethodologyWe employed
phenotypic, molecular docking, and genetic studies to screen for
anti-virulence activities among selected promising compounds: Coumarin,
Simvastatin, and Ibuprofen.ResultsWe found that nearly all detected MRSA
and VRSA strains exhibited MDR and multi-virulent profiles. The
molecular docking results aligned with the phenotypic and genetic
assessments of virulence production. Biofilm and hemolysin productions
were inhibited, and all virulence genes were downregulated upon
treatment with sub-minimum inhibitory concentration (sub-MIC) of these
promising compounds. Ibuprofen was the most active compound, exhibiting
the highest inhibition and downregulation of virulence gene products.
Moreover, in vivo and histopathological studies confirmed these results.
Interestingly, we observed a significant decrease in wound area and
improvements in re-epithelialization and tissue organization in the
Ibuprofen and antimicrobial treated group compared with the group
treated with antimicrobial alone. These findings support the idea that a
combination of Ibuprofen and antimicrobial drugs may offer a promising
new therapy for MRSA and VRSA infections.ConclusionWe hope that our
findings can be implemented in clinical practice to assist physicians in
making the most suitable treatment decisions.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1403219"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1403219</a></p>
<hr />
<h2
id="metagenomic-next-generation-sequencing-promotes-pathogen-detection-over-culture-in-joint-infections-with-previous-antibiotic-exposure">Metagenomic
next-generation sequencing promotes pathogen detection over culture in
joint infections with previous antibiotic exposure</h2>
<p>date: 2024-08-26, from: Frontiers in Cellular and Infection
Microbiology</p>
<p>ObjectiveTo investigate the diagnostic value of metagenomic
next-generation sequencing (mNGS) in detecting pathogens from joint
infection (JI) synovial fluid (SF) samples with previous antibiotic
exposure.MethodsFrom January 2019 to January 2022, 59 cases with
suspected JI were enrolled. All cases had antibiotic exposure within 2
weeks before sample collection. mNGS and conventional culture were
performed on SF samples. JI was diagnosed based on history and clinical
symptoms in conjunction with MSIS criteria. The diagnostic values,
including sensitivity, specificity, positive/negative predictive values
(PPV/NPV), and accuracy, were in comparison with mNGS and
culture.ResultsThere were 47 of the 59 cases diagnosed with JI, while
the remaining 12 were diagnosed with non-infectious diseases. The
sensitivity of mNGS was 68.1%, which was significantly higher than that
of culture (25.5%, p&lt;0.01). The accuracy of mNGS was significantly
higher at 71.2% compared to the culture at 39.0% (p &lt;0.01). Eleven
pathogenic strains were detected by mNGS but not by microbiological
culture, which included Staphylococcus lugdunensis, Staphylococcus
cohnii, Finegoldia magna, Enterococcus faecalis, Staphylococcus
saprophytics, Escherichia coli, Salmonella enterica, Pseudomonas
aeruginosa, Acinetobacter pittii, Brucella ovis, andCoxiella burnetii.
Antibiotic therapy was adjusted based on the mNGS results in 32 (68.1%)
patients, including 12 (25.5%) and 20 (42.6%) patients, in whom
treatment was upgraded and changed, respectively. All JI patients
underwent surgery and received subsequent antibiotic therapy. They were
followed up for an average of 23 months (20-27 months), and the success
rate of treatment was 89.4%. Out of the 33 patients who had positive
results for pathogens, reoperation was performed in 1 case (3.03%),
while out of the 14 cases with negative results for both mNGS and
cultures, reoperation was performed in 4 cases (28.6%).ConclusionsmNGS
has advantages over conventional culture in detecting pathogens in SF
samples from JI patients previously treated with antibiotics,
potentially improving clinical outcomes.</p>
<p><a
href="https://www.frontiersin.org/articles/10.3389/fcimb.2024.1388765"
class="uri">https://www.frontiersin.org/articles/10.3389/fcimb.2024.1388765</a></p>
</section>
<footer>
Antenna is a personal aggregation of items found around the web.
Curated with <a href="https://rsdoiel.github.io/skimmer">skimmer</a> and <a href="https://sqlite.org">sqlite</a> then rendered with <a href="https://pandoc.org">Pandoc</a> with search provided by <a href="https://pagefind.app">PageFind</a>.
</footer>
<script type="module">
    await import('/pagefind/pagefind-highlight.js');
    new PagefindHighlight({ highlightParam: "highlight" });
</script>
</body>
</html>
