---
title: health
updated: 2024-09-15 10:48:57
---

# health

(date: 2024-09-15 10:48:57)

---

## Can Stress Cause Kidney Stones?

date: 2024-09-14, from: The Kidney Dietitian

<p>Your emotional health is equally important to your physical health. The &#8220;mind-body&#8221; connection doesn&#8217;t get as much credit as it should. Stress is rampant in our lives and contributes to a host of health conditions, including kidney stones. As you may already know, stress can also impact your eating habits. Whether you find yourself drinking [&#8230;]</p>
<p>The post <a href="https://www.thekidneydietitian.org/kidney-stones-and-stress/">Can Stress Cause Kidney Stones?</a> appeared first on <a href="https://www.thekidneydietitian.org">The Kidney Dietitian</a>.</p>
 

<https://www.thekidneydietitian.org/kidney-stones-and-stress/>

---

## Combination of aloe emodin, emodin, and rhein from Aloe with EDTA sensitizes the resistant Acinetobacter baumannii to polymyxins

date: 2024-09-13, from: Frontiers in Cellular and Infection Microbiology

BackgroundThe continuous emergence and spread of polymyxin-resistant Acinetobacter baumannii pose a significant global health challenge, necessitating the development of novel therapeutic strategies. Aloe, with its long-standing history of medicinal use, has recently been the subject of substantial research for its efficacy against pathogenic infections.MethodsThis study investigates the potential application of anthraquinone components in aloe against polymyxin-resistant A. baumannii by liquid chromatography-mass spectrometry, in vitro activity assessment, and construction of animal infection models.ResultsThe findings demonstrate that aloe emodin, emodin, rhein, and their mixtures in equal mass ratios (EAR) exhibit strain-specific antibacterial activities against polymyxin-resistant A. baumannii. Co-administration of EAR with EDTA synergistically and universally enhanced the antibacterial activity and bactericidal efficacy of polymyxins against polymyxin-resistant A. baumannii, while also reducing the frequency of polymyxin-resistant mutations in polymyxinssensitive A. baumannii. Following toxicity assessment on human hepatic and renal cell lines, the combination therapy was applied to skin wounds in mice infected with polymyxin-resistant A. baumannii. Compared to monotherapy, the combination therapy significantly accelerated wound healing and reduced bacterial burden.ConclusionsThe combination of EAR and EDTA with polymyxins offers a novel therapeutic approach for managing skin infections caused by polymyxinresistant A. baumannii. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1467607>

---

## Mechanism of emodin in treating hepatitis B virus-associated hepatocellular carcinoma: network pharmacology and cell experiments

date: 2024-09-13, from: Frontiers in Cellular and Infection Microbiology

IntroductionHepatocellular carcinoma (HCC) is a pressing global issue, with Hepatitis B virus (HBV) infection remaining the primary. Emodin, an anthraquinone compound extracted from the natural plant’s. This study investigates the molecular targets and possible mechanisms of emodin in treating HBV-related HCC based on network pharmacology and molecular docking and validates the screened molecular targets through in vitro experiments.MethodsPotential targets related to emodin were obtained through PubChem, CTD, PharmMapper, SuperPred, and TargetNet databases. Potential disease targets for HBV and HCC were identified using the DisGeNET, GeneCards, OMIM, and TTD databases. A Venn diagram was used to determine overlapping genes between the drug and the diseases. Enrichment analysis of these genes was performed using GO and KEGG via bioinformatics websites. The overlapping genes were imported into STRING to construct a protein-protein interaction network. Cytoscape 3.9.1 software was used for visualizing and analyzing the core targets. Molecular docking analysis of the drug and core targets was performed using Schrodinger. The regulatory effects of emodin on these core targets were validate through in vitro experiments.ResultsA total of 43 overlapping genes were identified. GO analysis recognized 926 entries, and KEGG analysis identified 135 entries. The main pathways involved in the KEGG analysis included cancer, human cytomegalovirus infection and prostate cancer. The binding energies of emodin with HSP90AA1, PTGS2, GSTP1, SOD2, MAPK3, and PCNA were all less than -5 kcal/mol. Compared to normal liver tissue, the mRNA levels of XRCC1, MAPK3, and PCNA were significantly elevated in liver cancer tissue. The expression levels of XRCC1, HIF1A, MAPK3, and PCNA genes were closely related to HCC progression. High expressions of HSP90AA1, TGFB1, HIF1A, MAPK3, and PCNA were all closely associated with poor prognosis in HCC. In vitro experiments demonstrated that emodin significantly downregulated the expression of HSP90AA1, MAPK3, XRCC1, PCNA, and SOD2, while significantly upregulating the expression of PTGS2 and GSTP1.ConclusionThis study, based on network pharmacology and molecular docking validation, suggests that emodin may exert therapeutic effects on HBV-related HCC by downregulating the expression of XRCC1, MAPK3, PCNA, HSP90AA1, and SOD2, and upregulating the expression of PTGS2 and GSTP1. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1458913>

---

## A causal association between esophageal cancer and the oral microbiome: a Mendelian randomization study based on an Asian population

date: 2024-09-13, from: Frontiers in Cellular and Infection Microbiology

BackgroundPrevious studies have suggested a crosstalk between the oral microbiome and esophageal cancer (EC), but the exact relationship is unclear. This study aimed to investigate the causal relationship between changes in the oral microbiome and EC by Mendelian randomization (MR).Materials and methodsIn the study, bidirectional MR analyses were conducted using genome-wide association study data from the oral microbiomes from the 4D-SZ cohort and EC data from the BioBank Japan cohort. Multiple sensitivity tests, including Cochrane’s Q statistic, MR-Egger intercept, and MR-PRESSO, were used to assess and validate the relative stability of the resulting data at various levels.ResultsAmong the 3,117 samples studied, 73 oral microbiomes were found to be statistically causally associated with EC, 38 of which were considered protective factors. According to species analyses, positive results were concentrated in three phyla: Firmicutes (29 species), Patescibacteria (18 species), and Actinobacteria (9 species). It was also determined that Parvimonas micra, Aggregatibacter, and Clostridia had a negative causal relationship, implying that EC caused a decrease in the counts. Following p-value correction, periodonticum_C, unclassified_mgs_3234, and unclassified_mgs_45 were identified as having a strong evidence-grade causal relationship with EC. There was no strong evidence in the results of the inverse MR analyses of EC to the oral microbiome. The sensitivity analysis confirmed the robustness of the findings.ConclusionThis study discovered a bidirectional causal relationship between the oral microbiome and EC, which may provide new insights into the future use of the microbiome for early screening and probiotic therapy. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1420625>

---

## Effects of four potent entomopathogenic fungal isolates on the survival and performance of Telenomus remus, an egg parasitoid of fall armyworm

date: 2024-09-12, from: Frontiers in Cellular and Infection Microbiology

Fall armyworm (FAW), Spodoptera frugiperda is a generalist pest known to feed on more than 300 plant species, including major staple crops such as rice, maize and sorghum. Biological control of FAW using a combination of a major indigenous egg parasitoid Telenomus remus and entomopathogenic fungi was explored in this study. Metarhizium anisopliae strains (ICIPE 7, ICIPE 41, and ICIPE 78) and Beauveria bassiana ICIPE 621 which demonstrated effectiveness to combat the pest, were evaluated through direct and indirect fungal infection to assess their pathogenicity and virulence against T. remus adults, S. frugiperda eggs and their effects on T. remus parasitism rates. Metarhizium anisopliae ICIPE 7 and ICIPE 78 exhibited the highest virulence against T. remus adults with LT50 values >2 days. ICIPE 7 induced the highest T. remus mortality rate (81.40 ± 4.17%) following direct infection with dry conidia. Direct fungal infection also had a significant impact on parasitoid emergence, with the highest emergence rate recorded in the M. anisopliae ICIPE 7 treatment (42.50 ± 5.55%), compared to the control ± (83.25 ± 5.94%). In the indirect infection, the highest concentration of 1 x 109 conidia ml-1 of ICIPE 78 induced the highest mortality (100 ± 0.00%) of T. remus adults, and the highest mortality (51.25%) of FAW eggs, whereas the least FAW egg mortality (15.25%) was recorded in the lowest concentration 1 x 105 conidia ml-1 of ICIPE 41. The number of parasitoids that emerged and their sex ratios were not affected by the different fungal strain concentrations except in ICIPE 7 at high dose. This study showed that potential combination of both M. anisopliae and B. bassiana with T. remus parasitoid can effectively suppress FAW populations. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445156>

---

## Specific cell subclusters of dental pulp stem cells respond to distinct pathogens through the ROS pathway

date: 2024-09-12, from: Frontiers in Cellular and Infection Microbiology

IntroductionMicrobial pathogens invade various human organs, including the oral cavity. Candida albicans (C.a) and Streptococcus mutans (S.m) served respectively as representative oral pathogenic fungi and bacteria to stimulate dental pulp stem cells (DPSCs) and to screen the DPSC subcluster that specifically responded to fungal infection.MethodsDPSCs were obtained from the impacted third molars of six healthy subjects. Then, cells were mixed and divided into three samples, two of which were stimulated with C.a and S.m, respectively; the third sample was exposed to cell medium only (Ctrl). Single-cell mRNA sequencing analysis of treated DPSCs was performed.ResultsDPSCs were composed of four major clusters of which one, DPSC.7, exhibited unique changes compared to those of other subclusters. The DPSC.7 cell percentage of the C.a sample was twice those of the Ctrl and S.m samples. DPSC.7 cells expressed genes associated with the response to reactive oxygen species (ROS) response. DPSC.7 subgroup cells established characteristic aggregation under the stimulation of different pathogens in UMAP. The MAPK/ERK1/2 and NF-κB pathways were up-regulated, DUSP1/5/6 expressions were suppressed, FOS synthesis was activated, the immune-related pathway was induced, and the levels of cytokines, including IL-6 and CCL2, were up-regulated in DPSC.7 cells when stimulated with C.a.ConclusionsOur study analyzed the cellular and molecular properties of DPSCs infected by oral fungi and bacteria with single-cell RNA sequencing. A subcluster of DPSCs responded specifically to infections with different pathogens, activating the MAPK and NF-κB pathways to induce immune responses via the ROS pathway. This suggests novel treatment strategies for fungal infections. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1452124>

---

## Comparing NGS-Based identification of bloodstream infections to traditional culture methods for enhanced ICU care: a comprehensive study

date: 2024-09-12, from: Frontiers in Cellular and Infection Microbiology

BackgroundAccurate identification of infectious diseases using molecular techniques, such as PCR and NGS, is well-established. This study aims to assess the utility of Bactfast and Fungifast in diagnosing bloodstream infections in ICU settings, comparing them against traditional culture methods. The objectives include evaluating sensitivity and specificity and identifying a wide range of pathogens, including non-culturable species.MethodsWe collected 500 non-duplicate blood samples from ICU patients between January 2023 and December 2023. Specimens underwent traditional culture, MALDI-TOF, VITEK®2 compact system, and NGS-based Bactfast and Fungifast analyses.ResultsOut of the 500 samples, 26.8% (n=134) showed bacterial growth via traditional culture methods, while 4.8% (n=24) were positive for fungal growth. MALDI-TOF and VITEK®2 compact system yielded comparable results, identifying 26.4% (n=132) of specimens with bacterial growth. NGS-based Bactfast detected bacterial presence in 38.2% (n=191) of samples, including non-culturable bacteria missed by traditional methods. However, NGS-based Fungifast showed concordant fungal detection rates with culture methods. Among identified pathogens by culture method included Klebsiella pneumoniae 20.89% (n=28), Enterococcus faecalis 18.65% (n=25), Escherichia coli 15.67% (n=21), Pseudomonas aeruginosa 12.68% (n=17), Acinetobacter baumannii 10.44% (n=14), various Streptococcus species 7.46% (n=10), Mycobacterium tuberculosis 6.71% (n=9), Mycobacterium abscessus 4.47% (n=6), and Salmonella spp 2.98% (n=4). Non-culture-based NGS identified additional (n=33) pathogens, including Klebsiella pneumoniae 27.27% (n=9), Bacteroides fragilis 21.21% (n=7), Aerococcus viridans 15.15% (n=5), Elizabethkingia anopheles 12.12% (n=4), Aeromonas salmonicida 9% (n=3), Clostridium 9% (n=3), and Bacteroides vulgatus 6% (n=2). Candida albicans was reported in 5% (n=24) of samples by both methods.ConclusionNGS-based Bactfast and Fungifast demonstrate high sensitivity in identifying a wide array of bacterial and fungal pathogens in ICU patients, outperforming traditional culture methods in detecting non-culturable organisms. These molecular assays offer rapid and comprehensive diagnostic capabilities, potentially improving clinical outcomes through timely and accurate pathogen identification. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1454549>

---

## Genetic landscape of ESBL producing international clone ST410 of Escherichia coli from pediatric infections in Shenzhen, China

date: 2024-09-11, from: Frontiers in Cellular and Infection Microbiology

BackgroundThe emergence of ESBLs producing cephalosporin-resistant Escherichia coli isolates poses a threat to public health. This study aims to decipher the genetic landscape and gain insights into ESBL-producing E. coli strains belonging to the high-risk clone ST410 from pediatric patients.Methods29 E. coli ST410 isolates were collected from young children and subjected to antimicrobial susceptibility testing, Whole-genome sequencing (WGS), serotype analysis, MLST, ESBL genes, virulence genes, and plasmid profiling.ResultsAntimicrobial susceptibility testing demonstrated a high level of resistance to cephalosporins followed by aminoglycoside, sulfonamide, carbapenem and penicillin group of antibiotics. However, n=20/29 shows MDR phenotype. Phylogenetic group B2 (n=15) dominated, followed by group D (n=7), group A (n=4), and group B1 (n=3). Serotyping analysis identified O1:H7 (n=8), O2:H1 (n=6), O8:H4 (n=5), O16:H5 (n=4), and O25:H4 (n=3). Other serotypes identified included O6:H1, O15:H5, and O18:H7 (n=1 each). The most commonly detected ESBL genes were blaCTX-M, (n=26), followed by blaTEM (n=23), and blaSHV (n=18). Additionally, blaOXA-1 (n=10), blaOXA-48 (n=5), blaKPC-2 (n=3), blaKPC-3 (n=2), blaNDM-1 (n=4), blaNDM-5 (n=1), blaGES-1 (n=2), blaGES-5 (n=1), and blaCYM-1 (n=3). Notable virulence genes identified within the ST410 isolates included fimH (n=29), papC (n=24), hlyA (n=22), and cnf1 (n=18), among others. Diverse plasmids were observed including IncFIS, IncX4, IncFIA, IncCol, IncI2 and IncFIC with transmission frequency ranges from 1.3X10-2 to 2.7X10-3.ConclusionThe ST410 clone exhibited a complex resistance profile, diverse serotypes, the presence of specific resistance genes (ESBL genes), virulence gene repertoire, and diverse plasmids. The blaCTX-M was the most prevalent ESBL gene detected. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1403234>

---

## Editorial: Global excellence in fungal pathogenesis: Central and South America

date: 2024-09-11, from: Frontiers in Cellular and Infection Microbiology

 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1481806>

---

## Insights into the role of legionella effectors on host metabolic perturbations

date: 2024-09-11, from: Frontiers in Cellular and Infection Microbiology

Legionella infection, the causative agent of Legionnaires’ disease, represents a significant threat to human health. The pathogenesis of this infection is intricately linked to the complex interactions between the bacterium and its host, resulting in profound metabolic perturbations. Central to these metabolic shifts is the bacterium’s modulation of lipid metabolism, with changes in lipid synthesis and breakdown modifying membrane composition and function. These alterations can influence cellular signaling and immune responses, further contributing to disease progression. It also disrupts glucose utilization and lipid metabolism, altering cellular energy production and immune responses. Additionally, Legionella infection perturbs amino acid and protein metabolism, affecting protein synthesis and degradation, leading to changes in cellular functions and immune responses. This mini-review underscores the complexity of metabolic perturbations in Legionella infection and their significance in host-pathogen interactions. Understanding these metabolic shifts provides valuable insights into the pathogenesis of Legionnaires’ disease and could lead to the development of novel therapeutic strategies. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1458276>

---

## Effects of probiotic treatment on patients and animals with chronic obstructive pulmonary disease: a systematic review and meta-analysis of randomized control trials

date: 2024-09-11, from: Frontiers in Cellular and Infection Microbiology

ObjectiveIn recent years, the lung-gut axis has received increasing attention. The oxidative stress and systemic hypoxia occurring in chronic obstructive pulmonary disease (COPD) are related to gut dysfunction. That suggests probiotics have a potential therapeutic role in COPD. In this study, we therefore evaluated the ameliorative effects of probiotics on COPD.MethodsSearches were conducted in four electronic databases, including PubMed, Cochrane Library, the NIH clinical registry Clinical Trials. Gov and EMBASE. The data extracted was analyzed statistically in this study using StataMP17 software, with mean difference (MD) chosen as the effect size for continuous variables, and the results expressed as effect sizes and their 95% confidence intervals (CIs). Standardized Mean Difference (SMD) was used if the data units were different.ResultsWe included three randomized, controlled, double-blind clinical trials and five randomized controlled animal studies. The results show that for lung function, probiotics improved %FEV1 in COPD patients (MD = 3.02, 95%CI: 1.10, 4.93). Additionally, in inflammation, probiotics increased IL-10 (SMD = 1.99, 95%CI: 1.02, 2.96) and decreased inflammatory markers such as TNF-α (SMD= -2.64, 95%Cl: -3.38, -1.90), IL-1β (SMD= -3.49, 95%Cl: -4.58, -2.40), and IL-6 (SMD= -6.54, 95%Cl: -8.36, -4.73) in COPD animals, while having no significant effect on C-reactive protein (MD = 0.30, 95%CI: -0.71, 1.32) in COPD patients. For lung structure, probiotics significantly reduced the degree of pulmonary collagen fibers deposition in COPD animals (SMD = -2.25, 95%CI: -3.08, -1.41).ConclusionOverall, probiotics may be an additional approach that can improve COPD. Further clinical trials are needed to evaluate the efficacy, safety, and impact factors of probiotics for COPD.Systematic Review Registrationhttps://inplasy.com/inplasy-2023-4-0023/, identifier INPLASY202340023. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1411222>

---

## First report of a blaNDM-producing extensively drug resistant Klebsiella pneumoniae ST437 in Italy

date: 2024-09-11, from: Frontiers in Cellular and Infection Microbiology

Carbapenemase-producing Klebsiella pneumoniae strains (CP-Kps) have recently been observed to spread rapidly worldwide. New Delhi metallo-β-lactamase (NDM) producing clones of Klebsiella pneumoniae (K. pneumoniae) cause a significant healthcare burden, particularly in Indian sub-continent, where this clone is circulating widely. However, in Italy, data on the incidence of these new clones is limited, and an ST437 NDM-producing K. pneumoniae strain has not been reported to date. A sacral ulcer infection caused by a K. pneumoniae strain was identified in an 85-year-old Italian male patient with several comorbidities. Antimicrobial susceptibility testing revealed an extensive resistance to a wide range of antimicrobials, including novel agents such as cefiderocol and ceftazidime/avibactam. Genomic analysis identified the pathogen as an ST437 K. pneumoniae strain harboring blaNDM-5, blaOXA-232 and blaCTX-M-15 genes. Following the identification of this first case, several infection control measures were implemented in healthcare settings, including direct precautions and reinforcement of standard cross-transmission control measures. The emergence of pathogenic microbial clones carrying new genetic determinants, particularly in a little city, requires prompt diagnosis and therapeutic protocols. An effective infection control system for the early detection and/or control of the transmission of NDM-producing Enterobacteriaceae is also needed. Further investigations are required to better understand the potential transmission routes and evolution of these clones. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1426817>

---

## Fine mapping-based multi-omics analysis interprets the gut-lung axis function of SGLT2 inhibitors

date: 2024-09-10, from: Frontiers in Cellular and Infection Microbiology

BackgroundCurrently, Sodium-glucose cotransporter 2 (SGLT2) inhibitors demonstrate additional effects beyond glucose control on the gut microbiota and circulating metabolites. The gut microbiota and metabolites have been found to be useful in elucidating potential biological mechanisms of pulmonary diseases. Therefore, our study aims to investigate the effects of gut microbiota and metabolites mediating SGLT2 inhibition in 10 pulmonary diseases through Mendelian randomization (MR) research.MethodsWe conducted a two-sample, two-step MR study to assess the association between SGLT2 inhibition and 10 pulmonary diseases and to investigate the mediating effects of gut microbiota and metabolite. Gene-fine mapping and annotation of mediators by FUMA and Magma analyses were performed, and causal associations of mapped genes with diseases were assessed by muti-omics MR analyses. Possible side effects of SGLT2 inhibition were assessed by PheWAS analysis.ResultsSGLT2 inhibition was linked to a reduced risk of T2DM, Interstitial lung disease (ILD), Pneumoconiosis, Pulmonary tuberculosis, and Asthma(OR=0.457, 0.054, 0.002, 0.280, 0.706). The family Enterobacteriaceae and order Enterobacteriales were associated with SGLT2 inhibition and ILD(95% CI:0.079–0.138). The family Alcaligenaceae and X-12719 were linked to pneumoconiosis (95% CI: 0.042–0.120, 0.050–0.099). The genus Phascolarctobacterium was connected to pulmonary tuberculosis (95% CI: 0.236–0.703).The degree of unsaturation (Fatty Acids), ratio of docosahexaenoic acid to total fatty acids, and 4-androsten-3beta,17beta-diol disulfate 2, were associated with asthma(95% CI: 0.042–0.119, 0.039–0.101, 0.181–0.473). Furthermore, Fuma and Magma analyses identified target genes for the four diseases, and proteomic MR analysis revealed six overlapping target genes in asthma. PheWAS analysis also highlighted potential side effects of SGLT2 inhibition.ConclusionsThis comprehensive study strongly supports a multi-omics association between SGLT2 inhibition and reduced risk of interstitial lung disease, tuberculosis, pneumoconiosis, and asthma. Four identified gut microbiota, four metabolites, sixteen metabolic pathways, and six target genes appear to play a potential role in this association. The results of the comprehensive phenome-wide association analysis also identified the full effect of SGLT2 inhibitors. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1447327>

---

## Expression of goat poxvirus P32 protein and monoclonal antibody preparation

date: 2024-09-10, from: Frontiers in Cellular and Infection Microbiology

P32 protein serves as a crucial structural component of Goat pox virus (GTPV), which causes a highly virulent infectious disease in sheep and goats. Despite the fact that P32 has been widely expressed in the previous studies, it is difficult to obtain recombinant P32 efficiently. This study aimed to achieve soluble expression of P32 recombinant protein and to develop its specific monoclonal antibody. The gene fragment of P32Δ (GP32Δ) was synthesized by optimizing the coding sequence of amino acids 1-246 of the known goatpox P32 protein. Subsequently, GP32Δ was cloned into a prokaryotic expression vector for expression and purification, resulting in the successful production of soluble recombinant protein rP32Δ. Utilizing rP32Δ, an indirect ELISA method was established by immunizing 6-week-old BALB/c mice with inactivated GTPV as the antigen. Through hybridoma technology, three monoclonal antibody hybridoma cell lines secreting anti-goat pox virus rP32Δ were screened, designated as 2F3, 3E8, and 4H5, respectively. These monoclonal antibodies, classified as IgG1, IgG2a, and IgG2b, respectively, with κappa light chains, were characterized following ascites preparation and purification. Indirect ELISA results demonstrated that the ELISA potency of the three monoclonal antibodies exceeded 1:12800. Furthermore, Western blot analysis revealed specific reactivity of both 3E8 and 4H5 with rP32Δ, while immunofluorescence assays confirmed 3E8's ability to specifically recognize GTPV in cells. The preceding findings demonstrate the successful acquisition of the soluble expressed recombinant P32 protein and its specific monoclonal antibody 3E8 in this study, thereby laying a foundational material basis for the establishment of a GTPV detection method. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1427588>

---

## Recombinase-aided amplification assay for rapid detection of imipenem-resistant Pseudomonas aeruginosa and rifampin-resistant Pseudomonas aeruginosa

date: 2024-09-10, from: Frontiers in Cellular and Infection Microbiology

The indiscriminate use of antibiotics has resulted in a growing resistance to drugs in Pseudomonas aeruginosa. The identification of antibiotic resistance genes holds considerable clinical significance for prompt diagnosis. In this study, we established and optimized a Recombinase-Aided Amplification (RAA) assay to detect two genes associated with drug resistance, oprD and arr, in 101 clinically collected P. aeruginosa isolates. Through screening for the detection or absence of oprD and arr, the results showed that there were 52 Imipenem-resistant P. aeruginosa (IRPA) strains and 23 Rifampin-resistant P. aeruginosa (RRPA) strains. This method demonstrated excellent detection performance even when the sample concentration is 10 copies/μL at isothermal conditions and the results could be obtained within 20 minutes. The detection results were in accordance with the results of conventional PCR and Real-time PCR. The detection outcomes of the arr gene were consistently with the resistance spectrum. However, the antimicrobial susceptibility results revealed that 65 strains were resistant to imipenem, while 49 strains sensitive to imipenem with oprD were identified. This discrepancy could be attributed to genetic mutations. In summary, the RAA has higher sensitivity, shorter time, and lower-cost instrument requirements than traditional detection methods. In addition, to analyze the epidemiological characteristics of the aforementioned drug-resistant strains, we conducted Multilocus Sequence Typing (MLST), virulence gene, and antimicrobial susceptibility testing. MLST analysis showed a strong correlation between the sequence types ST-1639, ST-639, ST-184 and IRPA, while ST-261 was the main subtype of RRPA. It was observed that these drug-resistant strains all possess five or more virulence genes, among which exoS and exoU do not coexist, and they are all multidrug-resistant strains. The non-coexistence of exoU and exoS in P.aeruginosa is related to various factors including bacterial regulatory mechanisms and pathogenic mechanisms. This indicates that the relationship between the presence of virulence genes and the severity of patient infection is worthy of attention. In conclusion, we have developed a rapid and efficient RAA (Recombinase-Aided Amplification) detection method that offers significant advantages in terms of speed, simplicity, and cost-effectiveness (especially in time and equipment aspect). This novel approach is designed to meet the demands of clinical diagnostics. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1428827>

---

## Head-to-head comparison of two human papillomavirus vaccines for efficacy against cervical intraepithelial neoplasia grade 3 and adenocarcinoma in situ—population-based follow-up of two cluster-randomized trials

date: 2024-09-09, from: Frontiers in Cellular and Infection Microbiology

IntroductionWe report head-to-head comparison of the bivalent and quadrivalent HPV vaccine efficacies against immediate precursors of cervical cancer from 15 years’ country-wide cancer registry follow-up of phase III trial cohorts and an age-aligned cohort of unvaccinated women.MethodsThese individually and/or clusterrandomized cohorts of HPV6/11/16/18- and HPV16/18-vaccinated and unvaccinated women were enrolled, respectively, in 2002, 2004, and 2003/2005. The trial cohorts comprised initially 16- to 17-year-old HPV6/11/16/18-vaccinated FUTURE II (NCT00092534) participants (866) and HPV16/18-vaccinated PATRICIA (NCT00122681) and 012 trial (NCT00169494) participants (2,465), and 16,526 initially 16- to 19-year-old unvaccinated controls. After active 4-year clinical follow-up, passive, country-wide Finnish Cancer Registry (FCR) follow-up for cervical intraepithelial neoplasia grade 3 (CIN3) and adenocarcinoma in situ (AIS) was based on consented use of unique personal identifiers and started 6 months after the end of the FUTURE II and PATRICIA trials in 2007 and 2009, and ended at the end of 2019. The follow-up with altogether 229,020 follow-up years was age-aligned to ensure that similarly aged cohorts were passively followed up for 15 years post=vaccination for the intention-to-treat analyses of vaccine efficacy.ResultsOverall, we identified 5 and 16 CIN3 (no AIS) cases in the HPV6/11/16/18 and HPV16/18 cohorts, respectively, during the FCR-based follow-up. In the unvaccinated cohort, we identified 281 CIN3 cases, 20 AIS cases, and 13 cases with invasive cervical cancer. Vaccine efficacies against CIN3+ were 68.4% and 64.5% for the quadrivalent and the bivalent vaccines, respectively, with overlapping confidence intervals.DiscussionLong-term follow-up of randomized, initially adolescent HPV-vaccinated and unvaccinated cohorts shows, in this head-to-head setting, that the bivalent and quadrivalent HPV vaccines are equally effective against immediate precursors of cervical cancer. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1437704>

---

## Vaginal microbiota of pregnant women with Ureaplasma urealyticum and Mycoplasma hominis infections

date: 2024-09-09, from: Frontiers in Cellular and Infection Microbiology

BackgroundThe association between preterm birth and Mycoplasma species such as Mycoplasma hominis and Ureaplasma urealyticum has been extensively investigated. In a clinical setting, conventional diagnostic methods for them involve culture methods for Mycoplasma spp. and Ureaplasma spp., along with PCR tests. However, the clinical utility of these tests remains controversial, highlighting the necessity for more robust and reliable methods for identifying and understanding Mycoplasma infections.ObjectiveThis study aimed to assess the distribution of microbiota in pregnant women with Mycoplasma hominis and Ureaplasma urealyticum infection by the comparison of conventional diagnostic methods with vaginal microbial community analysis.Study DesignThis prospective case–control study involved 228 Korean pregnant women and utilized vaginal microbial community analysis, Ureaplasma/Mycoplasma culture, and 12-multiplex PCR for sexually transmitted diseases. Cross-correlation analysis in SPSS 27 compared the results of two conventional methods with vaginal microbial community analysis. R software generated box plots depicting the relative abundance of microorganisms. Network analysis was conducted using Cytoscape.ResultsPositive Ureaplasma urealyticum culture findings were observed in 60.2% of patients, with 76.4% positive for Ureaplasma parvum PCR and 13.2% positive for Ureaplasma urealyticum PCR. Mycoplasma hominis culture was positive only in two patients, while Mycoplasma hominis PCR was positive in eight women. Vaginal microbial community analysis identified significant differences in relative abundances of Gardnerella species type I and Fannyhessea vaginae between the Ureaplasma urealyticum PCR positive and negative groups. Mycoplasma hominis PCR positive patients exhibited significant differences in 11 bacterial species, including Gardnerella species I and Fannyhessea vaginae.ConclusionThis study suggests that STD-PCR may be more accurate than Ureaplasma/Mycoplasma culture for the diagnosis of Mycoplasma hominis and Ureaplasma urealyticum infection. Also, the presence of Gardnerella species I and Fannyhessea vaginae implies their potential influences on Ureaplasma urealyticum and Mycoplasma hominis infections based on results of vaginal microbial community analysis. Therefore, vaginal microbial community analysis may give the more information of their pathophysiology. 

<https://www.frontiersin.org/articles/10.3389/fcimb.2024.1445300>

